.. _list_software:

List of supported software
==========================

EasyBuild |version| supports 1770 different software packages (incl. toolchains, bundles):

:ref:`list_software_letter_a` - :ref:`list_software_letter_b` - :ref:`list_software_letter_c` - :ref:`list_software_letter_d` - :ref:`list_software_letter_e` - :ref:`list_software_letter_f` - :ref:`list_software_letter_g` - :ref:`list_software_letter_h` - :ref:`list_software_letter_i` - :ref:`list_software_letter_j` - :ref:`list_software_letter_k` - :ref:`list_software_letter_l` - :ref:`list_software_letter_m` - :ref:`list_software_letter_n` - :ref:`list_software_letter_o` - :ref:`list_software_letter_p` - :ref:`list_software_letter_q` - :ref:`list_software_letter_r` - :ref:`list_software_letter_s` - :ref:`list_software_letter_t` - :ref:`list_software_letter_u` - :ref:`list_software_letter_v` - :ref:`list_software_letter_w` - :ref:`list_software_letter_x` - :ref:`list_software_letter_y` - :ref:`list_software_letter_z`


.. _list_software_letter_3:

*3*
---


:ref:`list_software_3to2_328`


.. _list_software_3to2_328:

*3to2*
++++++

lib3to2 is a set of fixers that are intended to backport code written for Python version 3.x  into Python version 2.x.

*homepage*: https://pypi.python.org/pypi/3to2

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.1.1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``1.1.1``    ``-Python-2.7.13``    ``intel/2017a``                
=========    ==================    ===============================


.. _list_software_letter_4:

*4*
---


:ref:`list_software_4ti2_323`


.. _list_software_4ti2_323:

*4ti2*
++++++

A software package for algebraic, geometric and combinatorial problems on linear spaces

*homepage*: https://4ti2.github.io/

=========    ===============
version      toolchain      
=========    ===============
``1.6.9``    ``intel/2018b``
=========    ===============


.. _list_software_letter_a:

*A*
---


:ref:`list_software_ABAQUS_445` - :ref:`list_software_ABINIT_439` - :ref:`list_software_ABySS_418` - :ref:`list_software_ack_303` - :ref:`list_software_ACT_216` - :ref:`list_software_ACTC_283` - :ref:`list_software_ada_294` - :ref:`list_software_AdapterRemoval_1431` - :ref:`list_software_ADDA_266` - :ref:`list_software_ADF_203` - :ref:`list_software_adjustText_1072` - :ref:`list_software_ADMIXTURE_691` - :ref:`list_software_ADOL-C_400` - :ref:`list_software_Advisor_728` - :ref:`list_software_AFNI_286` - :ref:`list_software_AGFusion_764` - :ref:`list_software_aiohttp_761` - :ref:`list_software_ALADIN_425` - :ref:`list_software_Albacore_793` - :ref:`list_software_ALFA_276` - :ref:`list_software_alleleCount_1144` - :ref:`list_software_Allinea_694` - :ref:`list_software_ALLPATHS-LG_793` - :ref:`list_software_ALPS_304` - :ref:`list_software_Amara_482` - :ref:`list_software_amask_525` - :ref:`list_software_Amber_487` - :ref:`list_software_AmberMini_884` - :ref:`list_software_AmberTools_1016` - :ref:`list_software_AMD-LibM_611` - :ref:`list_software_AMD-RNG_486` - :ref:`list_software_AMD-SecureRNG_1101` - :ref:`list_software_AMOS_304` - :ref:`list_software_AMPL-MP_500` - :ref:`list_software_Anaconda2_839` - :ref:`list_software_Anaconda3_840` - :ref:`list_software_angsd_525` - :ref:`list_software_Annif_492` - :ref:`list_software_annovar_757` - :ref:`list_software_ANSYS_398` - :ref:`list_software_ANSYS_CFD_698` - :ref:`list_software_ant_323` - :ref:`list_software_ANTLR_385` - :ref:`list_software_ANTs_342` - :ref:`list_software_anvio_541` - :ref:`list_software_APBS_294` - :ref:`list_software_APR_227` - :ref:`list_software_APR-util_718` - :ref:`list_software_ARAGORN_522` - :ref:`list_software_Arb_277` - :ref:`list_software_ARCH_286` - :ref:`list_software_ARGoS_412` - :ref:`list_software_argtable_834` - :ref:`list_software_Armadillo_917` - :ref:`list_software_ARPACK++_520` - :ref:`list_software_arpack-ng_884` - :ref:`list_software_ArrayFire_901` - :ref:`list_software_Arriba_593` - :ref:`list_software_Arrow_523` - :ref:`list_software_ART_231` - :ref:`list_software_ARWEN_381` - :ref:`list_software_ASAP3_344` - :ref:`list_software_ASE_217` - :ref:`list_software_ASHS_303` - :ref:`list_software_Aspera-CLI_865` - :ref:`list_software_Aspera-Connect_1363` - :ref:`list_software_Assimulo_845` - :ref:`list_software_ASTRID_455` - :ref:`list_software_astropy_786` - :ref:`list_software_at-spi2-atk_1005` - :ref:`list_software_at-spi2-core_1110` - :ref:`list_software_ATK_224` - :ref:`list_software_ATLAS_373` - :ref:`list_software_atomate_747` - :ref:`list_software_AtomPAW_633` - :ref:`list_software_atools_658` - :ref:`list_software_ATSAS_380` - :ref:`list_software_attr_443` - :ref:`list_software_AUGUSTUS_641` - :ref:`list_software_Autoconf_831` - :ref:`list_software_AutoDock_794` - :ref:`list_software_AutoDock_Vina_1287` - :ref:`list_software_AutoGrid_799` - :ref:`list_software_Automake_823` - :ref:`list_software_Autotools_970` - :ref:`list_software_awscli_643`


.. _list_software_ABAQUS_445:

*ABAQUS*
++++++++

Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit  dynamics FEA.

*homepage*: http://www.simulia.com/products/abaqus_fea.html

==========    =================    ==========
version       versionsuffix        toolchain 
==========    =================    ==========
``6.12.1``    ``-linux-x86_64``    ``system``
``6.13.5``    ``-linux-x86_64``    ``system``
``6.14.1``    ``-linux-x86_64``    ``system``
``2017``      ``-hotfix-1721``     ``system``
``2018``      ``-hotfix-1806``     ``system``
==========    =================    ==========


.. _list_software_ABINIT_439:

*ABINIT*
++++++++

ABINIT is a package whose main program allows one to find the total energy,  charge density and  electronic structure of systems made of electrons and nuclei (molecules  and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a  planewave or wavelet basis.

*homepage*: http://www.abinit.org/

==========    ========================    ===============================
version       versionsuffix               toolchain                      
==========    ========================    ===============================
``7.0.3``     ``-x86_64_linux_gnu4.5``    ``system``                     
``7.0.5``     ``-x86_64_linux_gnu4.5``    ``system``                     
``7.2.1``     ``-x86_64_linux_gnu4.5``    ``system``                     
``7.10.5``                                ``intel/2016.02-GCC-4.9``      
``7.10.5``    ``-libxc``                  ``intel/2016.02-GCC-4.9``      
``8.0.8``                                 ``intel/2016a``                
``8.0.8b``                                ``foss/2016b``, ``intel/2016b``
``8.2.2``                                 ``foss/2016b``, ``intel/2016b``
``8.4.4``                                 ``intel/2017b``                
``8.6.3``                                 ``intel/2018a``                
``8.10.2``                                ``intel/2018b``                
``8.10.3``                                ``intel/2018b``                
==========    ========================    ===============================


.. _list_software_ABySS_418:

*ABySS*
+++++++

Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler

*homepage*: http://www.bcgsc.ca/platform/bioinfo/software/abyss

=========    ================================================================
version      toolchain                                                       
=========    ================================================================
``1.9.0``    ``foss/2016a``                                                  
``2.0.2``    ``foss/2016b``, ``foss/2018a``, ``gompi/2019a``, ``intel/2016b``
``2.0.3``    ``foss/2017b``, ``intel/2017b``                                 
=========    ================================================================


.. _list_software_ack_303:

*ack*
+++++

ack is a tool like grep, optimized for programmers

*homepage*: http://beyondgrep.com/

========    ==========
version     toolchain 
========    ==========
``2.14``    ``system``
========    ==========


.. _list_software_ACT_216:

*ACT*
+++++

ACT is a Java application for displaying pairwise comparisons between two or more DNA sequences.  It can be used to identify and analyse regions of similarity and difference between genomes  and to explore conservation of synteny, in the context of the entire sequences and their annotation.  It can read complete EMBL, GENBANK and GFF entries or sequences in FASTA or raw format. 

*homepage*: https://sanger-pathogens.github.io/Artemis/ACT/

==========    =============    ==========
version       versionsuffix    toolchain 
==========    =============    ==========
``18.0.2``    ``-Java-11``     ``system``
``18.0.3``    ``-Java-11``     ``system``
==========    =============    ==========


.. _list_software_ACTC_283:

*ACTC*
++++++

ACTC converts independent triangles into triangle strips or fans.

*homepage*: https://sourceforge.net/projects/actc

=======    =====================================================
version    toolchain                                            
=======    =====================================================
``1.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``intel/2016b``
=======    =====================================================


.. _list_software_ada_294:

*ada*
+++++

Performs discrete, real, and gentle boost under both exponential and logistic loss on a given data set.

*homepage*: https://cran.r-project.org/web/packages/ada/index.html

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``2.0-5``    ``-R-3.4.0``     ``intel/2017a``
=========    =============    ===============


.. _list_software_AdapterRemoval_1431:

*AdapterRemoval*
++++++++++++++++

AdapterRemoval searches for and removes remnant adapter sequences  from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases  from the 3' end of reads following adapter removal.

*homepage*: https://github.com/%/(github_account)s/%(namelower)s

=========    ==============
version      toolchain     
=========    ==============
``2.2.0``    ``foss/2016b``
``2.2.2``    ``foss/2018b``
``2.3.1``    ``foss/2018b``
=========    ==============


.. _list_software_ADDA_266:

*ADDA*
++++++

ADDA is an open-source parallel implementation of the discrete dipole approximation, capable to simulate light scattering by particles of arbitrary shape and composition in a wide range of particle sizes.

*homepage*: https://github.com/adda-team/adda/wiki

=========    ==============
version      toolchain     
=========    ==============
``1.3b4``    ``foss/2019a``
=========    ==============


.. _list_software_ADF_203:

*ADF*
+++++

ADF is an accurate, parallelized, powerful computational chemistry program to understand and  predict chemical structure and reactivity with density functional theory (DFT).

*homepage*: http://www.scm.com/ADF/

================================    ===============
version                             toolchain      
================================    ===============
``2009.01a.pc64_linux.intelmpi``    ``system``     
``2014.02``                         ``system``     
``2014.11.r48287``                  ``intel/2016a``
``2016.101``                        ``system``     
================================    ===============


.. _list_software_adjustText_1072:

*adjustText*
++++++++++++

A small library for automatically adjustment of text position in matplotlib plots to minimize overlaps.

*homepage*: https://github.com/Phlya/adjustText

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.7.3``    ``-Python-3.7.2``    ``intel/2019a``
=========    =================    ===============


.. _list_software_ADMIXTURE_691:

*ADMIXTURE*
+++++++++++

ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from  multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much  more rapidly using a fast numerical optimization algorithm.

*homepage*: http://software.genetics.ucla.edu/admixture

=========    ==========
version      toolchain 
=========    ==========
``1.3.0``    ``system``
=========    ==========


.. _list_software_ADOL-C_400:

*ADOL-C*
++++++++

The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitates the evaluation of first and higher derivatives of vector functions that are defined by computer programs written in C or C++. The resulting derivative evaluation routines may be called from C/C++, Fortran, or any other language that can be linked with C.  

*homepage*: https://projects.coin-or.org/ADOL-C

=========    ===============
version      toolchain      
=========    ===============
``2.7.0``    ``gompi/2019a``
=========    ===============


.. _list_software_Advisor_728:

*Advisor*
+++++++++

Vectorization Optimization and Thread Prototyping  - Vectorize & thread code or performance “dies”  - Easy workflow + data + tips = faster code faster  - Prioritize, Prototype & Predict performance gain  

*homepage*: https://software.intel.com/intel-advisor-xe

================    ==========
version             toolchain 
================    ==========
``2016_update2``    ``system``
``2017_update1``    ``system``
``2018_update1``    ``system``
``2018_update3``    ``system``
``2019_update2``    ``system``
``2019_update3``    ``system``
``2019_update5``    ``system``
================    ==========


.. _list_software_AFNI_286:

*AFNI*
++++++

AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data -  a technique for mapping human brain activity.

*homepage*: http://afni.nimh.nih.gov/

============    ==================    ===============================
version         versionsuffix         toolchain                      
============    ==================    ===============================
``18.1.09``     ``-Python-3.6.4``     ``intel/2018a``                
``18.3.00``     ``-Python-3.6.6``     ``foss/2018b``                 
``19.0.01``     ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``20160329``    ``-Python-2.7.11``    ``intel/2016a``                
============    ==================    ===============================


.. _list_software_AGFusion_764:

*AGFusion*
++++++++++

AGFusion is a python package for annotating gene fusions  from the human or mouse genomes.

*homepage*: https://github.com/murphycj/AGFusion

=======    =================    ==============
version    versionsuffix        toolchain     
=======    =================    ==============
``1.2``    ``-Python-3.7.2``    ``foss/2019a``
=======    =================    ==============


.. _list_software_aiohttp_761:

*aiohttp*
+++++++++

" Async http client/server framework 

*homepage*: https://github.com/aio-libs/aiohttp

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``3.5.4``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_ALADIN_425:

*ALADIN*
++++++++

ALADIN was entirely built on the notion of compatibility with its mother system, IFS/ARPEG.  The latter, a joint development between the European Centre for Medium-Range Weather Forecasts (ECMWF) and  Meteo-France, was only meant to consider global Numerical Weather Prediction applications; hence the idea,  for ALADIN, to complement the IFS/ARPEGE project with a limited area model (LAM) version, while keeping the  differences between the two softwares as small as possible.

*homepage*: http://www.cnrm.meteo.fr/aladin/

===============    ===============
version            toolchain      
===============    ===============
``36t1_op2bf1``    ``intel/2016a``
===============    ===============


.. _list_software_Albacore_793:

*Albacore*
++++++++++

Albacore is a software project that provides an entry point to the Oxford Nanopore basecalling  algorithms.

*homepage*: https://community.nanoporetech.com/protocols/albacore-offline-basecalli/v/abec_2003_v1_revz_29nov2016

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``2.0.2``    ``-Python-3.6.1``    ``intel/2017a``
=========    =================    ===============


.. _list_software_ALFA_276:

*ALFA*
++++++

ALFA provides a global overview of features distribution composing NGS dataset(s). Given a set of  aligned reads (BAM files) and an annotation file (GTF format), the tool produces plots of the raw and normalized  distributions of those reads among genomic categories (stop codon, 5'-UTR, CDS, intergenic, etc.) and biotypes  (protein coding genes, miRNA, tRNA, etc.). Whatever the sequencing technique, whatever the organism.

*homepage*: https://github.com/biocompibens/ALFA

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.1.1``    ``-Python-3.7.2``    ``foss/2019a``
=========    =================    ==============


.. _list_software_alleleCount_1144:

*alleleCount*
+++++++++++++

The alleleCount package primarily exists to prevent code duplication between some other projects, specifically AscatNGS and Battenberg. As of v4 the perl  code wraps the C implementation of allele counting code for BAM/CRAM processing. 

*homepage*: http://cancerit.github.io/alleleCount/

=========    =================
version      toolchain        
=========    =================
``4.0.0``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_Allinea_694:

*Allinea*
+++++++++

The Allinea environment is an essential toolkit for developers and computational scientists looking to get results faster.

*homepage*: http://www.allinea.com

===============================    ==========
version                            toolchain 
===============================    ==========
``4.1-32834-Redhat-5.7-x86_64``    ``system``
``4.1-32834-Redhat-6.0-x86_64``    ``system``
``6.1.1-Ubuntu-14.04-x86_64``      ``system``
===============================    ==========


.. _list_software_ALLPATHS-LG_793:

*ALLPATHS-LG*
+++++++++++++

ALLPATHS-LG, the new short read genome assembler.

*homepage*: http://www.broadinstitute.org/software/allpaths-lg/blog/

=========    ==============
version      toolchain     
=========    ==============
``52488``    ``foss/2016a``
=========    ==============


.. _list_software_ALPS_304:

*ALPS*
++++++

The ALPS project (Algorithms and Libraries for Physics Simulations) is an open source effort  aiming at providing high-end simulation codes for strongly correlated quantum mechanical systems as well as   C++ libraries for simplifying the development of such code. 

*homepage*: http://alps.comp-phys.org/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``2.2.b4``    ``-Python-2.7.11``    ``intel/2016a``
``2.3.0``     ``-Python-2.7.12``    ``foss/2016b`` 
``2.3.0``     ``-Python-3.5.2``     ``foss/2016b`` 
==========    ==================    ===============


.. _list_software_Amara_482:

*Amara*
+++++++

Library for XML processing in Python, designed to balance the native idioms of Python  with the native character of XML.

*homepage*: https://pypi.org/project/Amara

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``1.2.0.2``    ``-Python-2.7.15``    ``foss/2019a``, ``intel/2019a``
===========    ==================    ===============================


.. _list_software_amask_525:

*amask*
+++++++

amask is a set of tools to to determine the affinity of MPI processes and OpenMP threads  in a parallel environment.

*homepage*: https://github.com/TACC/amask

============    ==============
version         toolchain     
============    ==============
``20171106``    ``foss/2018a``
``20190404``    ``foss/2018b``
============    ==============


.. _list_software_Amber_487:

*Amber*
+++++++

Amber (originally Assisted Model Building with Energy  Refinement) is software for performing molecular dynamics and structure  prediction.

*homepage*: http://ambermd.org/amber.html

=======    =================================================    ========================================================================
version    versionsuffix                                        toolchain                                                               
=======    =================================================    ========================================================================
``14``     ``-AmberTools-15-patchlevel-13-13``                  ``intel/2016a``                                                         
``16``     ``-AmberTools-16-patchlevel-5-14``                   ``iomkl/2016.07``                                                       
``16``     ``-AmberTools-16-patchlevel-5-14-CUDA``              ``iomkl/2016.09-GCC-4.9.3-2.25``                                        
``16``     ``-AmberTools-16-patchlevel-5-14-serial``            ``iomkl/2016.07``                                                       
``16``     ``-AmberTools-17-patchlevel-10-15``                  ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                     
``16``     ``-AmberTools-17-patchlevel-10-15-Python-2.7.14``    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``16``     ``-AmberTools-17-patchlevel-8-12``                   ``intel/2017b``                                                         
``18``     ``-AmberTools-18-patchlevel-10-8``                   ``foss/2018b``, ``fosscuda/2018b``, ``intel/2017b``                     
=======    =================================================    ========================================================================


.. _list_software_AmberMini_884:

*AmberMini*
+++++++++++

A stripped-down set of just antechamber, sqm, and tleap.

*homepage*: https://github.com/choderalab/ambermini

===========    ===============
version        toolchain      
===========    ===============
``16.16.0``    ``intel/2017b``
===========    ===============


.. _list_software_AmberTools_1016:

*AmberTools*
++++++++++++

AmberTools consists of several independently developed packages that work well by themselves,  and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations,  with either explicit water or generalized Born solvent models.

*homepage*: http://ambermd.org/

=======    ================================
version    toolchain                       
=======    ================================
``17``     ``intel/2017b``, ``intel/2018a``
=======    ================================


.. _list_software_AMD-LibM_611:

*AMD-LibM*
++++++++++

AMD LibM is a software library containing a collection of basic math functions  optimized for x86-64 processor based machines.

*homepage*: https://developer.amd.com/amd-cpu-libraries/amd-math-library-libm/

=========    ==================
version      toolchain         
=========    ==================
``3.2.2``    ``GCC/7.3.0-2.30``
=========    ==================


.. _list_software_AMD-RNG_486:

*AMD-RNG*
+++++++++

AMD Random Number Generator Library is a pseudorandom number generator library.

*homepage*: https://developer.amd.com/amd-cpu-libraries/rng-library/

=======    ==================
version    toolchain         
=======    ==================
``1.0``    ``GCC/7.3.0-2.30``
=======    ==================


.. _list_software_AMD-SecureRNG_1101:

*AMD-SecureRNG*
+++++++++++++++

The AMD Secure Random Number Generator (RNG) is a library that provides APIs to access the  cryptographically secure random numbers generated by AMD’s hardware-based random number generator implementation.

*homepage*: https://developer.amd.com/amd-cpu-libraries/rng-library/

=======    ==================
version    toolchain         
=======    ==================
``1.0``    ``GCC/7.3.0-2.30``
=======    ==================


.. _list_software_AMOS_304:

*AMOS*
++++++

The AMOS consortium is committed to the development of open-source whole genome assembly software

*homepage*: http://amos.sourceforge.net

=========    ==============
version      toolchain     
=========    ==============
``3.1.0``    ``foss/2018b``
=========    ==============


.. _list_software_AMPL-MP_500:

*AMPL-MP*
+++++++++

An open-source library for mathematical programming. 

*homepage*: https://github.com/ampl/mp

=========    =================
version      toolchain        
=========    =================
``3.1.0``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_Anaconda2_839:

*Anaconda2*
+++++++++++

Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. 

*homepage*: https://www.anaconda.com

===========    ==========
version        toolchain 
===========    ==========
``4.0.0``      ``system``
``4.2.0``      ``system``
``4.4.0``      ``system``
``5.0.1``      ``system``
``5.1.0``      ``system``
``5.3.0``      ``system``
``2018.12``    ``system``
``2019.03``    ``system``
``2019.07``    ``system``
===========    ==========


.. _list_software_Anaconda3_840:

*Anaconda3*
+++++++++++

Built to complement the rich, open source Python community, the Anaconda platform provides an enterprise-ready data analytics platform  that empowers companies to adopt a modern open data science analytics architecture. 

*homepage*: https://www.anaconda.com

===========    ==========
version        toolchain 
===========    ==========
``4.0.0``      ``system``
``4.2.0``      ``system``
``4.4.0``      ``system``
``5.0.1``      ``system``
``5.1.0``      ``system``
``5.3.0``      ``system``
``2018.12``    ``system``
``2019.03``    ``system``
``2019.07``    ``system``
===========    ==========


.. _list_software_angsd_525:

*angsd*
+++++++

Program for analysing NGS data.

*homepage*: http://www.popgen.dk/angsd

=========    ==============
version      toolchain     
=========    ==============
``0.910``    ``foss/2016a``
``0.921``    ``foss/2018a``
``0.925``    ``foss/2018b``
=========    ==============


.. _list_software_Annif_492:

*Annif*
+++++++

Annif is a multi-algorithm automated subject indexing tool for libraries, archives and museums.

*homepage*: https://github.com/NatLibFi/Annif

==========    =================    ===============================
version       versionsuffix        toolchain                      
==========    =================    ===============================
``0.40.0``    ``-Python-3.7.2``    ``foss/2019a``, ``intel/2019a``
==========    =================    ===============================


.. _list_software_annovar_757:

*annovar*
+++++++++

ANNOVAR is an efficient software tool to utilize update-to-date information   to functionally annotate genetic variants detected from diverse genomes (including human   genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others).

*homepage*: http://annovar.openbioinformatics.org/en/latest/

=============    ================    ==============
version          versionsuffix       toolchain     
=============    ================    ==============
``2016Feb01``    ``-Perl-5.22.1``    ``foss/2016a``
=============    ================    ==============


.. _list_software_ANSYS_398:

*ANSYS*
+++++++

ANSYS simulation software enables organizations to confidently predict      how their products will operate in the real world. We believe that every product is      a promise of something greater. 

*homepage*: http://www.ansys.com

========    ==========
version     toolchain 
========    ==========
``15.0``    ``system``
========    ==========


.. _list_software_ANSYS_CFD_698:

*ANSYS_CFD*
+++++++++++

ANSYS computational fluid dynamics (CFD) simulation software allows you to predict, with confidence,  the impact of fluid flows on your product throughout design and manufacturing as well as during end use.  ANSYS renowned CFD analysis tools include the widely used and well-validated ANSYS Fluent and ANSYS CFX.

*homepage*: http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics

========    ==========
version     toolchain 
========    ==========
``16.2``    ``system``
``17.0``    ``system``
========    ==========


.. _list_software_ant_323:

*ant*
+++++

Apache Ant is a Java library and command-line tool whose mission is to drive processes described in  build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of  Java applications.

*homepage*: http://ant.apache.org/

==========    ===================    ==========
version       versionsuffix          toolchain 
==========    ===================    ==========
``1.8.4``     ``-Java-1.7.0_10``     ``system``
``1.8.4``     ``-Java-1.7.0_21``     ``system``
``1.9.0``     ``-Java-1.7.0_15``     ``system``
``1.9.0``     ``-Java-1.7.0_21``     ``system``
``1.9.3``     ``-Java-1.7.0_60``     ``system``
``1.9.3``     ``-Java-1.7.0_79``     ``system``
``1.9.6``     ``-Java-1.7.0_75``     ``system``
``1.9.6``     ``-Java-1.7.0_79``     ``system``
``1.9.6``     ``-Java-1.7.0_80``     ``system``
``1.9.6``     ``-Java-1.8.0_66``     ``system``
``1.9.6``     ``-Java-1.8.0_72``     ``system``
``1.9.6``     ``-Java-1.8.0_77``     ``system``
``1.9.7``     ``-Java-1.8.0_92``     ``system``
``1.10.0``    ``-Java-1.8.0_112``    ``system``
``1.10.1``    ``-Java-1.8``          ``system``
``1.10.1``    ``-Java-1.8.0_121``    ``system``
``1.10.1``    ``-Java-1.8.0_144``    ``system``
``1.10.1``    ``-Java-1.8.0_152``    ``system``
``1.10.1``    ``-Java-1.8.0_162``    ``system``
``1.10.5``    ``-Java-1.8``          ``system``
``1.10.6``    ``-Java-1.8``          ``system``
``1.10.7``    ``-Java-11``           ``system``
==========    ===================    ==========


.. _list_software_ANTLR_385:

*ANTLR*
+++++++

ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)  is a language tool that provides a framework for constructing recognizers,  compilers, and translators from grammatical descriptions containing  Java, C#, C++, or Python actions.

*homepage*: http://www.antlr2.org/

=========    ==================    =====================================================================================================
version      versionsuffix         toolchain                                                                                            
=========    ==================    =====================================================================================================
``2.7.7``                          ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2017b``, ``foss/2018b``, ``foss/2019a``, ``intel/2017b``
``2.7.7``    ``-Python-2.7.11``    ``foss/2016a``                                                                                       
``2.7.7``    ``-Python-2.7.13``    ``intel/2017a``                                                                                      
``2.7.7``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2017b``                                                                      
``2.7.7``    ``-Python-3.6.4``     ``intel/2018a``                                                                                      
=========    ==================    =====================================================================================================


.. _list_software_ANTs_342:

*ANTs*
++++++

ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing,  interpreting and visualizing multidimensional data.

*homepage*: http://stnava.github.io/ANTs/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``2.2.0``    ``-Python-2.7.12``    ``foss/2016b``                 
``2.3.0``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``2.3.1``    ``-Python-3.6.6``     ``foss/2018b``                 
=========    ==================    ===============================


.. _list_software_anvio_541:

*anvio*
+++++++

An analysis and visualization platform for 'omics data.

*homepage*: https://merenlab.org/software/anvio/

=======    =================    ===============
version    versionsuffix        toolchain      
=======    =================    ===============
``6.1``    ``-Python-3.7.4``    ``intel/2019b``
=======    =================    ===============


.. _list_software_APBS_294:

*APBS*
++++++

APBS is a software package for modeling biomolecular solvation   through solution of the Poisson-Boltzmann equation (PBE), one of the most popular   continuum models for describing electrostatic interactions between molecular solutes   in salty, aqueous media. 

*homepage*: http://www.poissonboltzmann.org/apbs

=======    ========================    ==========
version    versionsuffix               toolchain 
=======    ========================    ==========
``1.4``    ``-linux-static-x86_64``    ``system``
=======    ========================    ==========


.. _list_software_APR_227:

*APR*
+++++

Apache Portable Runtime (APR) libraries.

*homepage*: http://apr.apache.org/

=========    =====================================================
version      toolchain                                            
=========    =====================================================
``1.6.3``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``iomkl/2018a``
``1.7.0``    ``GCCcore/8.2.0``                                    
=========    =====================================================


.. _list_software_APR-util_718:

*APR-util*
++++++++++

Apache Portable Runtime (APR) util libraries.

*homepage*: http://apr.apache.org/

=========    ========================================================================
version      toolchain                                                               
=========    ========================================================================
``1.6.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``iomkl/2018a``
=========    ========================================================================


.. _list_software_ARAGORN_522:

*ARAGORN*
+++++++++

a program to detect tRNA genes and tmRNA genes in nucleotide sequences

*homepage*: http://mbio-serv2.mbioekol.lu.se/ARAGORN/

==========    ==============
version       toolchain     
==========    ==============
``1.2.38``    ``foss/2016b``
==========    ==============


.. _list_software_Arb_277:

*Arb*
+++++

Arb is a C library for arbitrary-precision interval arithmetic.  It has full support for both real and complex numbers. The library is thread-safe,  portable, and extensively tested.

*homepage*: http://arblib.org

==========    ==========================================================
version       toolchain                                                 
==========    ==========================================================
``2.16.0``    ``GCC/7.3.0-2.30``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
==========    ==========================================================


.. _list_software_ARCH_286:

*ARCH*
++++++

Autoregressive Conditional Heteroskedasticity (ARCH) and other tools for financial econometrics,  written in Python (with Cython and/or Numba used to improve performance).

*homepage*: https://pypi.org/project/arch

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``4.5.0``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_ARGoS_412:

*ARGoS*
+++++++

A parallel, multi-engine simulator for heterogeneous swarm robotics

*homepage*: http://www.argos-sim.info

================    ==============    ==============
version             versionsuffix     toolchain     
================    ==============    ==============
``3.0.0-beta53``    ``-Lua-5.2.4``    ``foss/2018b``
================    ==============    ==============


.. _list_software_argtable_834:

*argtable*
++++++++++

Argtable is an ANSI C library for parsing GNU style  command line options with a minimum of fuss. 

*homepage*: http://argtable.sourceforge.net/

========    ===================================================================================
version     toolchain                                                                          
========    ===================================================================================
``2.13``    ``GCCcore/8.2.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
========    ===================================================================================


.. _list_software_Armadillo_917:

*Armadillo*
+++++++++++

Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards  a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,  as well as a subset of trigonometric and statistics functions.

*homepage*: https://arma.sourceforge.net/

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``7.600.2``    ``-Python-2.7.12``    ``foss/2016b``                 
``7.800.0``    ``-Python-2.7.12``    ``intel/2016b``                
``7.950.1``    ``-Python-2.7.12``    ``intel/2016b``                
``8.300.1``                          ``foss/2017b``, ``intel/2017b``
``8.400.0``                          ``foss/2018a``                 
``9.700.2``                          ``foss/2019a``                 
===========    ==================    ===============================


.. _list_software_ARPACK++_520:

*ARPACK++*
++++++++++

Arpackpp is a C++ interface to the ARPACK Fortran package,  which implements the implicit restarted Arnoldi method for  iteratively solving large-scale sparse eigenvalue problems.

*homepage*: https://github.com/m-reuter/arpackpp

==============    ==============
version           toolchain     
==============    ==============
``2018.03.26``    ``foss/2017b``
==============    ==============


.. _list_software_arpack-ng_884:

*arpack-ng*
+++++++++++

ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.

*homepage*: https://github.com/opencollab/arpack-ng

=========    ================================================================================
version      toolchain                                                                       
=========    ================================================================================
``3.3.0``    ``foss/2016a``, ``intel/2016a``                                                 
``3.4.0``    ``foss/2016b``, ``foss/2017a``, ``intel/2016b``, ``intel/2017a``                
``3.5.0``    ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``
``3.6.2``    ``intel/2018a``                                                                 
``3.7.0``    ``foss/2019a``                                                                  
=========    ================================================================================


.. _list_software_ArrayFire_901:

*ArrayFire*
+++++++++++

ArrayFire is a general-purpose library that simplifies the process of  developing software that targets parallel and massively-parallel architectures  including CPUs, GPUs, and other hardware acceleration devices. 

*homepage*: https://arrayfire.com/

=========    ================    ==============
version      versionsuffix       toolchain     
=========    ================    ==============
``3.6.4``                        ``foss/2018b``
``3.6.4``    ``-CUDA-9.2.88``    ``foss/2018b``
=========    ================    ==============


.. _list_software_Arriba_593:

*Arriba*
++++++++

Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.  It was developed for the use in a clinical research setting. Therefore, short runtimes and high  sensitivity were important design criteria.

*homepage*: https://github.com/suhrig/arriba

=========    ==============
version      toolchain     
=========    ==============
``1.1.0``    ``foss/2018b``
=========    ==============


.. _list_software_Arrow_523:

*Arrow*
+++++++

Apache Arrow is a cross-language development platform for in-memory data.

*homepage*: https://arrow.apache.org

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``0.7.1``     ``-Python-3.6.3``     ``intel/2017b``                
``0.12.0``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
``0.12.0``    ``-Python-3.6.6``     ``intel/2018b``                
==========    ==================    ===============================


.. _list_software_ART_231:

*ART*
+++++

ART is a set of simulation tools to generate synthetic next-generation sequencing reads

*homepage*: http://www.niehs.nih.gov/research/resources/software/biostatistics/art/

==============    ==================================
version           toolchain                         
==============    ==================================
``2016.06.05``    ``GCCcore/6.4.0``, ``intel/2016b``
==============    ==================================


.. _list_software_ARWEN_381:

*ARWEN*
+++++++

ARWEN, tRNA detection in metazoan mitochondrial sequences

*homepage*: http://mbio-serv2.mbioekol.lu.se/ARWEN

=========    =================
version      toolchain        
=========    =================
``1.2.3``    ``GCCcore/7.3.0``
=========    =================


.. _list_software_ASAP3_344:

*ASAP3*
+++++++

ASAP is a calculator for doing large-scale classical molecular dynamics within the Campos Atomic Simulation Environment (ASE).

*homepage*: https://wiki.fysik.dtu.dk/asap/

===========    =================    ================================================
version        versionsuffix        toolchain                                       
===========    =================    ================================================
``3.10.7``     ``-Python-3.5.2``    ``foss/2016b``                                  
``3.10.7``     ``-Python-3.6.2``    ``foss/2017b``                                  
``3.10.8``     ``-Python-3.5.2``    ``foss/2016b``                                  
``3.10.8``     ``-Python-3.6.2``    ``foss/2017b``                                  
``3.10.8``     ``-Python-3.6.3``    ``foss/2017b``                                  
``3.10.10``    ``-Python-3.6.6``    ``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.11.10``    ``-Python-3.7.2``    ``foss/2019a``, ``intel/2019a``                 
===========    =================    ================================================


.. _list_software_ASE_217:

*ASE*
+++++

ASE is a python package providing an open source Atomic Simulation Environment  in the Python scripting language.

*homepage*: https://wiki.fysik.dtu.dk/ase/

==============    ==================    ================================================
version           versionsuffix         toolchain                                       
==============    ==================    ================================================
``3.9.1.4567``    ``-Python-2.7.11``    ``intel/2016a``                                 
``3.10.0``        ``-Python-2.7.11``    ``intel/2016.02-GCC-4.9``                       
``3.11.0``        ``-Python-2.7.12``    ``intel/2016b``                                 
``3.13.0``        ``-Python-2.7.12``    ``foss/2016b``                                  
``3.13.0``        ``-Python-2.7.13``    ``intel/2017a``                                 
``3.15.0``        ``-Python-2.7.12``    ``foss/2016b``                                  
``3.15.0``        ``-Python-2.7.14``    ``intel/2017b``                                 
``3.15.0``        ``-Python-3.5.2``     ``foss/2016b``                                  
``3.15.0``        ``-Python-3.6.2``     ``foss/2017b``                                  
``3.15.0``        ``-Python-3.6.3``     ``foss/2017b``                                  
``3.16.2``        ``-Python-3.6.4``     ``iomkl/2018.02``, ``iomkl/2018a``              
``3.16.2``        ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.17.0``        ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
``3.17.0``        ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``                 
``3.18.0``        ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                 
``3.18.0``        ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``                 
==============    ==================    ================================================


.. _list_software_ASHS_303:

*ASHS*
++++++

Automatic Segmentation of Hippocampal Subfields (ASHS) 

*homepage*: https://sites.google.com/site/hipposubfields/home

===================    ==========
version                toolchain 
===================    ==========
``rev103_20140612``    ``system``
===================    ==========


.. _list_software_Aspera-CLI_865:

*Aspera-CLI*
++++++++++++

IBM Aspera Command-Line Interface (the Aspera CLI) is a collection of Aspera tools for performing high-speed, secure data transfers from the command line. The Aspera CLI is for users and organizations who want to automate their transfer workflows.

*homepage*: http://asperasoft.com

=========    =================    ==========
version      versionsuffix        toolchain 
=========    =================    ==========
``3.7.2``    ``.354.010c3b8``     ``system``
``3.9.0``    ``.1326.6985b21``    ``system``
=========    =================    ==========


.. _list_software_Aspera-Connect_1363:

*Aspera-Connect*
++++++++++++++++

Connect is an install-on-demand Web browser plug-in that facilitates high-speed uploads and  downloads with an Aspera transfer server.

*homepage*: http://downloads.asperasoft.com/connect2/

=========    ==========
version      toolchain 
=========    ==========
``3.6.1``    ``system``
``3.9.6``    ``system``
=========    ==========


.. _list_software_Assimulo_845:

*Assimulo*
++++++++++

Assimulo is a simulation package for solving ordinary differential equations.

*homepage*: https://jmodelica.org/assimulo/

=======    ==================    ===============
version    versionsuffix         toolchain      
=======    ==================    ===============
``2.9``    ``-Python-2.7.15``    ``intel/2018b``
=======    ==================    ===============


.. _list_software_ASTRID_455:

*ASTRID*
++++++++

ASTRID-2 is a method for estimating species trees from gene trees.

*homepage*: https://github.com/pranjalv123/ASTRID

=========    ===============
version      toolchain      
=========    ===============
``2.2.1``    ``gompi/2019a``
=========    ===============


.. _list_software_astropy_786:

*astropy*
+++++++++

The Astropy Project is a community effort to develop   a single core package for Astronomy in Python and foster interoperability   between Python astronomy packages.

*homepage*: http://www.astropy.org/

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``2.0.12``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
``2.0.14``                          ``foss/2019a``                 
==========    ==================    ===============================


.. _list_software_at-spi2-atk_1005:

*at-spi2-atk*
+++++++++++++

AT-SPI 2 toolkit bridge 

*homepage*: https://wiki.gnome.org/Accessibility

==========    ==================
version       toolchain         
==========    ==================
``2.26.3``    ``fosscuda/2018b``
``2.32.0``    ``GCCcore/8.2.0`` 
==========    ==================


.. _list_software_at-spi2-core_1110:

*at-spi2-core*
++++++++++++++

Assistive Technology Service Provider Interface. 

*homepage*: https://wiki.gnome.org/Accessibility

==========    ==================
version       toolchain         
==========    ==================
``2.26.3``    ``fosscuda/2018b``
``2.32.0``    ``GCCcore/8.2.0`` 
==========    ==================


.. _list_software_ATK_224:

*ATK*
+++++

ATK provides the set of accessibility interfaces that are implemented by other  toolkits and applications. Using the ATK interfaces, accessibility tools have  full access to view and control running applications. 

*homepage*: https://developer.gnome.org/ATK/stable/

==========    ===================================================================
version       toolchain                                                          
==========    ===================================================================
``2.18.0``    ``intel/2016a``                                                    
``2.20.0``    ``foss/2016a``, ``intel/2016a``                                    
``2.22.0``    ``foss/2016b``, ``intel/2016b``                                    
``2.26.0``    ``intel/2017a``                                                    
``2.26.1``    ``foss/2018b``, ``intel/2017b``                                    
``2.27.1``    ``foss/2017b``, ``intel/2017b``                                    
``2.28.1``    ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``
``2.32.0``    ``GCCcore/8.2.0``                                                  
==========    ===================================================================


.. _list_software_ATLAS_373:

*ATLAS*
+++++++

ATLAS (Automatically Tuned Linear Algebra Software) is the application of  the AEOS (Automated Empirical Optimization of Software) paradigm, with the present emphasis   on the Basic Linear Algebra Subprograms (BLAS), a widely used, performance-critical, linear   algebra kernel library.

*homepage*: http://math-atlas.sourceforge.net

==========    =================    ==================
version       versionsuffix        toolchain         
==========    =================    ==================
``3.10.2``    ``-LAPACK-3.6.1``    ``GCC/5.4.0-2.26``
==========    =================    ==================


.. _list_software_atomate_747:

*atomate*
+++++++++

atomate has implementations of FireWorks workflows for Materials Science.

*homepage*: https://pythonhosted.org/atomate/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.4.4``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_AtomPAW_633:

*AtomPAW*
+++++++++

AtomPAW is a Projector-Augmented Wave Dataset Generator that  can be used both as a standalone program and a library.

*homepage*: http://users.wfu.edu/natalie/papers/pwpaw/man.html

===========    ===============
version        toolchain      
===========    ===============
``4.1.0.5``    ``intel/2018b``
``4.1.0.6``    ``intel/2018b``
===========    ===============


.. _list_software_atools_658:

*atools*
++++++++

Tools to make using job arrays a lot more convenient.

*homepage*: https://github.com/gjbex/atools

=========    ==================    =================
version      versionsuffix         toolchain        
=========    ==================    =================
``1.4.2``    ``-Python-2.7.12``    ``intel/2016b``  
``1.4.6``    ``-Python-2.7.16``    ``GCCcore/8.3.0``
=========    ==================    =================


.. _list_software_ATSAS_380:

*ATSAS*
+++++++

ATSAS is a program suite for small-angle scattering data analysis from biological macromolecules.

*homepage*: http://www.embl-hamburg.de/ExternalInfo/Research/Sax/software.html

===========    ===============    ==========
version        versionsuffix      toolchain 
===========    ===============    ==========
``2.5.1-1``    ``.el6.x86_64``    ``system``
``2.5.1-1``    ``.sl5.x86_64``    ``system``
``2.7.1-1``    ``.el7.x86_64``    ``system``
===========    ===============    ==========


.. _list_software_attr_443:

*attr*
++++++

Commands for Manipulating Filesystem Extended Attributes

*homepage*: https://savannah.nongnu.org/projects/attr

==========    =================
version       toolchain        
==========    =================
``2.4.47``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_AUGUSTUS_641:

*AUGUSTUS*
++++++++++

AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences

*homepage*: http://bioinf.uni-greifswald.de/augustus/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``3.2.3``    ``-Python-2.7.13``    ``intel/2017a``                
``3.3``                            ``foss/2018a``                 
``3.3.2``                          ``intel/2019a``                
``3.3.2``    ``-Python-2.7.13``    ``intel/2017a``                
``3.3.2``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``3.3.2``    ``-Python-2.7.15``    ``intel/2018b``                
=========    ==================    ===============================


.. _list_software_Autoconf_831:

*Autoconf*
++++++++++

Autoconf is an extensible package of M4 macros that produce shell scripts  to automatically configure software source code packages. These scripts can adapt the  packages to many kinds of UNIX-like systems without manual user intervention. Autoconf  creates a configuration script for a package from a template file that lists the  operating system features that the package can use, in the form of M4 macro calls.

*homepage*: http://www.gnu.org/software/autoconf/

========    ===================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version     toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
========    ===================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``2.68``    ``foss/2016b``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
``2.69``    ``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
========    ===================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_AutoDock_794:

*AutoDock*
++++++++++

AutoDock is a suite of automated docking tools. It is designed to   predict how small molecules, such as substrates or drug candidates, bind to   a receptor of known 3D structure.

*homepage*: http://autodock.scripps.edu/

===========    =============
version        toolchain    
===========    =============
``4.2.5.1``    ``GCC/5.2.0``
===========    =============


.. _list_software_AutoDock_Vina_1287:

*AutoDock_Vina*
+++++++++++++++

AutoDock Vina is an open-source program for doing molecular docking. 

*homepage*: http://vina.scripps.edu/index.html

=========    ==============    ==========
version      versionsuffix     toolchain 
=========    ==============    ==========
``1.1.2``    ``_linux_x86``    ``system``
=========    ==============    ==========


.. _list_software_AutoGrid_799:

*AutoGrid*
++++++++++

AutoDock is a suite of automated docking tools. It is designed to   predict how small molecules, such as substrates or drug candidates, bind to   a receptor of known 3D structure.

*homepage*: http://autodock.scripps.edu/

===========    =============
version        toolchain    
===========    =============
``4.2.5.1``    ``GCC/5.2.0``
===========    =============


.. _list_software_Automake_823:

*Automake*
++++++++++

Automake: GNU Standards-compliant Makefile generator

*homepage*: http://www.gnu.org/software/automake/automake.html

==========    =====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version       toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
==========    =====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``1.11.3``    ``foss/2016b``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
``1.14``      ``GCC/4.8.2``, ``intel/2016a``                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
``1.14.1``    ``GCC/4.8.2``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
``1.15``      ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.15.1``    ``GCCcore/6.4.0``, ``GCCcore/7.2.0``                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
``1.16.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                                                                                                                                                                                                                                                                                                                                                           
==========    =====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_Autotools_970:

*Autotools*
+++++++++++

This bundle collect the standard GNU build tools: Autoconf, Automake  and libtool 

*homepage*: http://autotools.io

============    ======================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version         toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
============    ======================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``20150119``    ``GCC/4.9.2``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
``20150215``    ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``20170619``    ``GCCcore/6.4.0``, ``GCCcore/7.2.0``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
``20180311``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                                                                                                                                                                                                                                                                                                                                                                                               
============    ======================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_awscli_643:

*awscli*
++++++++

Universal Command Line Environment for AWS

*homepage*: https://pypi.python.org/pypi/awscli

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``1.11.1``      ``-Python-2.7.12``    ``intel/2016b``
``1.11.56``     ``-Python-2.7.12``    ``foss/2016b`` 
``1.16.290``    ``-Python-3.6.6``     ``foss/2018b`` 
============    ==================    ===============


.. _list_software_letter_b:

*B*
---


:ref:`list_software_Bader_478` - :ref:`list_software_BAGEL_347` - :ref:`list_software_bam-readcount_1314` - :ref:`list_software_BAMM_285` - :ref:`list_software_BamM_349` - :ref:`list_software_BamTools_801` - :ref:`list_software_BamUtil_686` - :ref:`list_software_Bandage_674` - :ref:`list_software_barrnap_742` - :ref:`list_software_basemap_729` - :ref:`list_software_Bash_382` - :ref:`list_software_bat_311` - :ref:`list_software_BayesAss_795` - :ref:`list_software_BayeScan_774` - :ref:`list_software_BayeScEnv_864` - :ref:`list_software_BayesTraits_1131` - :ref:`list_software_Bazel_494` - :ref:`list_software_bbcp_407` - :ref:`list_software_bbFTP_430` - :ref:`list_software_BBMap_418` - :ref:`list_software_bc_197` - :ref:`list_software_BCALM_351` - :ref:`list_software_BCEL_278` - :ref:`list_software_BCFtools_764` - :ref:`list_software_bcgTree_700` - :ref:`list_software_bcl2fastq2_948` - :ref:`list_software_bcolz_538` - :ref:`list_software_BDBag_400` - :ref:`list_software_beagle-lib_964` - :ref:`list_software_Beast_495` - :ref:`list_software_BeautifulSoup_1352` - :ref:`list_software_BEDOPS_445` - :ref:`list_software_BEDTools_732` - :ref:`list_software_behave_619` - :ref:`list_software_BerkeleyGW_977` - :ref:`list_software_BFAST_368` - :ref:`list_software_BFC_203` - :ref:`list_software_bibtexparser_1291` - :ref:`list_software_binutils_874` - :ref:`list_software_Bio-DB-HTS_745` - :ref:`list_software_Bio-SamTools_1145` - :ref:`list_software_bioawk_637` - :ref:`list_software_biobambam2_972` - :ref:`list_software_biomart-perl_1230` - :ref:`list_software_BioPerl_685` - :ref:`list_software_Biopython_956` - :ref:`list_software_biscuit_755` - :ref:`list_software_Bismark_713` - :ref:`list_software_Bison_507` - :ref:`list_software_bitarray_862` - :ref:`list_software_BLACS_357` - :ref:`list_software_BLASR_372` - :ref:`list_software_blasr_libcpp_1261` - :ref:`list_software_BLAST_374` - :ref:`list_software_BLAST+_417` - :ref:`list_software_BLAT_291` - :ref:`list_software_Blender_700` - :ref:`list_software_BLIS_298` - :ref:`list_software_Blitz++_603` - :ref:`list_software_BlobTools_912` - :ref:`list_software_Blosc_499` - :ref:`list_software_bmtagger_841` - :ref:`list_software_bnpy_441` - :ref:`list_software_BOINC_363` - :ref:`list_software_bokeh_521` - :ref:`list_software_BoltzTraP_898` - :ref:`list_software_Bonmin_611` - :ref:`list_software_Bonnie++_689` - :ref:`list_software_Boost_519` - :ref:`list_software_Boost.Python_1207` - :ref:`list_software_Bottleneck_1035` - :ref:`list_software_Bowtie_618` - :ref:`list_software_Bowtie2_668` - :ref:`list_software_Bpipe_496` - :ref:`list_software_bpp-core_792` - :ref:`list_software_bpp-phyl_812` - :ref:`list_software_bpp-seq_696` - :ref:`list_software_BRAKER_439` - :ref:`list_software_BreakDancer_1074` - :ref:`list_software_bsddb3_562` - :ref:`list_software_buildenv_857` - :ref:`list_software_Bullet_616` - :ref:`list_software_BUSCO_380` - :ref:`list_software_BWA_218` - :ref:`list_software_bwa-meth_789` - :ref:`list_software_bwakit_642` - :ref:`list_software_bwidget_742` - :ref:`list_software_BWISE_378` - :ref:`list_software_bx-python_937` - :ref:`list_software_BXH_XCEDE_TOOLS_1178` - :ref:`list_software_byacc_514` - :ref:`list_software_bzip2_487`


.. _list_software_Bader_478:

*Bader*
+++++++

A fast algorithm for doing Bader's analysis on a charge density grid.

*homepage*: http://theory.cm.utexas.edu/henkelman/code/bader/

========    ===============
version     toolchain      
========    ===============
``1.02``    ``intel/2018a``
``1.03``    ``intel/2018b``
========    ===============


.. _list_software_BAGEL_347:

*BAGEL*
+++++++

BAGEL (Brilliantly Advanced General Electronic-structure Library) is a parallel electronic-structure program.

*homepage*: http://www.nubakery.org

=========    ===============
version      toolchain      
=========    ===============
``1.1.1``    ``intel/2016b``
=========    ===============


.. _list_software_bam-readcount_1314:

*bam-readcount*
+++++++++++++++

Count DNA sequence reads in BAM files

*homepage*: https://github.com/genome/bam-readcount

=========    ==============
version      toolchain     
=========    ==============
``0.8.0``    ``foss/2018b``
=========    ==============


.. _list_software_BAMM_285:

*BAMM*
++++++

BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates.  It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of  lineage diversification and trait evolution. 

*homepage*: http://bamm-project.org/

=========    ==============
version      toolchain     
=========    ==============
``2.5.0``    ``foss/2016b``
=========    ==============


.. _list_software_BamM_349:

*BamM*
++++++

BamM is a c library, wrapped in python, that parses BAM files 

*homepage*: http://ecogenomics.github.io/BamM/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.7.3``    ``-Python-2.7.12``    ``foss/2016b``
``1.7.3``    ``-Python-2.7.14``    ``foss/2018a``
=========    ==================    ==============


.. _list_software_BamTools_801:

*BamTools*
++++++++++

BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.

*homepage*: https://github.com/pezmaster31/bamtools

=========    ================================================================================================================================================
version      toolchain                                                                                                                                       
=========    ================================================================================================================================================
``2.4.0``    ``foss/2016b``                                                                                                                                  
``2.4.1``    ``intel/2017a``                                                                                                                                 
``2.5.0``    ``foss/2016b``, ``intel/2017b``                                                                                                                 
``2.5.1``    ``GCC/8.2.0-2.31.1``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017b``, ``intel/2018b``
=========    ================================================================================================================================================


.. _list_software_BamUtil_686:

*BamUtil*
+++++++++

BamUtil is a repository that contains several programs   that perform operations on SAM/BAM files. All of these programs   are built into a single executable, bam.

*homepage*: http://genome.sph.umich.edu/wiki/BamUtil

==========    ===============
version       toolchain      
==========    ===============
``1.0.13``    ``intel/2016b``
``1.0.14``    ``intel/2018a``
==========    ===============


.. _list_software_Bandage_674:

*Bandage*
+++++++++

Bandage is a program for visualising de novo assembly graphs

*homepage*: http://rrwick.github.io/Bandage/

=========    =============    ==========
version      versionsuffix    toolchain 
=========    =============    ==========
``0.8.1``    ``_Centos``      ``system``
``0.8.1``    ``_Ubuntu``      ``system``
=========    =============    ==========


.. _list_software_barrnap_742:

*barrnap*
+++++++++

Barrnap (BAsic Rapid Ribosomal RNA Predictor) predicts the location of ribosomal RNA genes in genomes.

*homepage*: https://github.com/tseemann/barrnap

=======    ==============
version    toolchain     
=======    ==============
``0.9``    ``foss/2018b``
=======    ==============


.. _list_software_basemap_729:

*basemap*
+++++++++

The matplotlib basemap toolkit is a library for plotting 2D data on maps in Python

*homepage*: http://matplotlib.org/basemap/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.0.7``    ``-Python-2.7.13``    ``intel/2017a``
``1.0.7``    ``-Python-3.6.3``     ``intel/2017b``
``1.0.7``    ``-Python-3.6.4``     ``intel/2018a``
``1.2.0``    ``-Python-3.6.6``     ``intel/2018b``
=========    ==================    ===============


.. _list_software_Bash_382:

*Bash*
++++++

Bash is an sh-compatible command language interpreter that executes commands    read from the standard input or from a file.  Bash also incorporates useful features from the    Korn and C shells (ksh and csh).

*homepage*: http://www.gnu.org/software/bash

=======    =============
version    toolchain    
=======    =============
``4.3``    ``GCC/4.9.2``
=======    =============


.. _list_software_bat_311:

*bat*
+++++

The BAT Python package supports the processing and analysis of Bro data with Pandas, scikit-learn, and Spark

*homepage*: https://pypi.python.org/pypi/bat

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.3.3``    ``-Python-3.6.3``    ``intel/2017b``
=========    =================    ===============


.. _list_software_BayesAss_795:

*BayesAss*
++++++++++

BayesAss: Bayesian Inference of Recent Migration Using Multilocus Genotypes

*homepage*: http://www.rannala.org/?page_id=245

=========    ==============
version      toolchain     
=========    ==============
``3.0.4``    ``foss/2016a``
=========    ==============


.. _list_software_BayeScan_774:

*BayeScan*
++++++++++

BayeScan aims at identifying candidate loci under natural selection from genetic data,  using differences in allele frequencies between populations.

*homepage*: http://cmpg.unibe.ch/software/BayeScan/

=======    ===============================================
version    toolchain                                      
=======    ===============================================
``2.1``    ``foss/2016a``, ``foss/2018a``, ``intel/2018a``
=======    ===============================================


.. _list_software_BayeScEnv_864:

*BayeScEnv*
+++++++++++

BayeScEnv is a Fst-based, genome-scan method that uses environmental variables to detect local adaptation.

*homepage*: https://github.com/devillemereuil/bayescenv

=======    ==============
version    toolchain     
=======    ==============
``1.1``    ``foss/2016a``
=======    ==============


.. _list_software_BayesTraits_1131:

*BayesTraits*
+++++++++++++

BayesTraits is a computer package for performing analyses of trait   evolution among groups of species for which a phylogeny or sample of phylogenies is   available. This new package incoporates our earlier and separate programes Multistate,   Discrete and Continuous. BayesTraits can be applied to the analysis of traits that adopt   a finite number of discrete states, or to the analysis of continuously varying traits.   Hypotheses can be tested about models of evolution, about ancestral states and about   correlations among pairs of traits. 

*homepage*: http://www.evolution.reading.ac.uk/BayesTraitsV1.html

===============    =================    ==========
version            versionsuffix        toolchain 
===============    =================    ==========
``1.0-linux32``                         ``system``
``2.0``            ``-Beta-Linux64``    ``system``
===============    =================    ==========


.. _list_software_Bazel_494:

*Bazel*
+++++++

Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software.

*homepage*: https://bazel.io/

==========    ====================================
version       toolchain                           
==========    ====================================
``0.4.4``     ``system``                          
``0.7.0``     ``GCCcore/6.4.0``                   
``0.10.0``    ``GCCcore/6.4.0``                   
``0.11.0``    ``GCCcore/6.4.0``                   
``0.11.1``    ``GCCcore/6.4.0``                   
``0.12.0``    ``GCCcore/6.4.0``                   
``0.16.0``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``0.18.0``    ``GCCcore/7.3.0``                   
``0.20.0``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``0.25.2``    ``GCCcore/8.2.0``                   
``0.26.1``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``0.29.1``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
``1.1.0``     ``GCCcore/8.3.0``                   
==========    ====================================


.. _list_software_bbcp_407:

*bbcp*
++++++

BBCP is an alternative to Gridftp when transferring large amounts of data,  capable of breaking up your transfer into multiple simultaneous transferring streams,  thereby transferring data much faster than single-streaming utilities such as SCP and SFTP.  See details at http://pcbunn.cithep.caltech.edu/bbcp/using_bbcp.htm  or http://www.nics.tennessee.edu/computing-resources/data-transfer/bbcp

*homepage*: http://www.slac.stanford.edu/~abh/bbcp/

=================    ==================    ==========
version              versionsuffix         toolchain 
=================    ==================    ==========
``12.01.30.00.0``    ``-amd64_linux26``    ``system``
=================    ==================    ==========


.. _list_software_bbFTP_430:

*bbFTP*
+++++++

bbFTP is a file transfer software. It implements its own transfer protocol,   which is optimized for large files (larger than 2GB) and secure as it does not read the   password in a file and encrypts the connection information. bbFTP main features are:   * Encoded username and password at connection * SSH and Certificate authentication modules   * Multi-stream transfer * Big windows as defined in RFC1323 * On-the-fly data compression   * Automatic retry * Customizable time-outs * Transfer simulation   * AFS authentication integration * RFIO interface

*homepage*: http://doc.in2p3.fr/bbftp/

=========    ===============
version      toolchain      
=========    ===============
``3.2.1``    ``intel/2016a``
=========    ===============


.. _list_software_BBMap_418:

*BBMap*
+++++++

BBMap short read aligner, and other bioinformatic tools.

*homepage*: https://sourceforge.net/projects/bbmap/

==========    ===================    ===============================
version       versionsuffix          toolchain                      
==========    ===================    ===============================
``36.62``     ``-Java-1.8.0_112``    ``intel/2016b``                
``37.93``                            ``foss/2018a``, ``intel/2018a``
``38.26``                            ``foss/2018b``                 
``38.50b``                           ``GCC/8.2.0-2.31.1``           
==========    ===================    ===============================


.. _list_software_bc_197:

*bc*
++++

bc is an arbitrary precision numeric processing language.

*homepage*: https://www.gnu.org/software/bc/

===========    =============
version        toolchain    
===========    =============
``1.06.95``    ``GCC/4.8.2``
===========    =============


.. _list_software_BCALM_351:

*BCALM*
+++++++

de Bruijn graph compaction in low memory

*homepage*: https://github.com/GATB/bcalm

=========    ==============
version      toolchain     
=========    ==============
``2.2.0``    ``foss/2018a``
=========    ==============


.. _list_software_BCEL_278:

*BCEL*
++++++

The Byte Code Engineering Library (Apache Commons BCEL™) is intended to give  users a convenient way to analyze, create, and manipulate (binary) Java class  files (those ending with .class). 

*homepage*: http://commons.apache.org/proper/commons-bcel/

=======    =============    ==========
version    versionsuffix    toolchain 
=======    =============    ==========
``5.2``    ``-Java-1.8``    ``system``
=======    =============    ==========


.. _list_software_BCFtools_764:

*BCFtools*
++++++++++

Samtools is a suite of programs for interacting with high-throughput sequencing data.  BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence  variants

*homepage*: http://www.htslib.org/

=======    =========================================================================================
version    toolchain                                                                                
=======    =========================================================================================
``1.3``    ``foss/2016a``, ``intel/2016a``                                                          
``1.6``    ``foss/2016b``, ``foss/2017b``, ``intel/2017b``                                          
``1.9``    ``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
=======    =========================================================================================


.. _list_software_bcgTree_700:

*bcgTree*
+++++++++

Automatized phylogenetic tree building from bacterial core genomes.

*homepage*: https://github.com/molbiodiv/bcgTree

==========    ================    ===============
version       versionsuffix       toolchain      
==========    ================    ===============
``1.0.10``    ``-Perl-5.26.1``    ``intel/2018a``
==========    ================    ===============


.. _list_software_bcl2fastq2_948:

*bcl2fastq2*
++++++++++++

bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by  Illumina sequencing systems to standard FASTQ file formats for downstream analysis.

*homepage*: https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``2.19.1``    ``-Python-2.7.12``    ``foss/2016b``                 
``2.20.0``                          ``foss/2018b``, ``intel/2019a``
``2.20.0``    ``-Python-2.7.12``    ``foss/2016b``                 
``2.20.0``    ``-Python-2.7.14``    ``intel/2017b``                
==========    ==================    ===============================


.. _list_software_bcolz_538:

*bcolz*
+++++++

bcolz provides columnar, chunked data containers that can be compressed either in-memory and on-disk.   Column storage allows for efficiently querying tables, as well as for cheap column addition and removal.   It is based on NumPy, and uses it as the standard data container to communicate with bcolz objects,   but it also comes with support for import/export facilities to/from HDF5/PyTables tables and pandas dataframes.

*homepage*: http://bcolz.blosc.org/en/latest/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.1.1``    ``-Python-2.7.13``    ``foss/2017a``
=========    ==================    ==============


.. _list_software_BDBag_400:

*BDBag*
+++++++

The bdbag utilities are a collection of software programs for working with BagIt packages that conform to the Bagit and Bagit/RO profiles.

*homepage*: https://github.com/fair-research/bdbag

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.4.1``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_beagle-lib_964:

*beagle-lib*
++++++++++++

beagle-lib is a high-performance library that can perform the core calculations at the heart of most  Bayesian and Maximum Likelihood phylogenetics packages.

*homepage*: https://github.com/beagle-dev/beagle-lib

=========    ================    ===============================
version      versionsuffix       toolchain                      
=========    ================    ===============================
``2.1.2``                        ``foss/2016a``, ``foss/2017a`` 
``3.0.1``                        ``foss/2018a``, ``intel/2018a``
``3.0.2``                        ``foss/2018b``                 
``3.0.2``    ``-CUDA-9.2.88``    ``foss/2018b``                 
``3.1.2``                        ``GCC/8.2.0-2.31.1``           
=========    ================    ===============================


.. _list_software_Beast_495:

*Beast*
+++++++

BEAST is a cross-platform program for Bayesian MCMC analysis of molecular   sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using   strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies   but is also a framework for testing evolutionary hypotheses without conditioning on a single   tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted   proportional to its posterior probability. 

*homepage*: http://beast2.org/

==========    ====================
version       toolchain           
==========    ====================
``1.8.4``     ``system``          
``1.10.1``    ``intel/2018a``     
``2.4.0``     ``foss/2016a``      
``2.4.7``     ``foss/2017a``      
``2.5.0``     ``foss/2018a``      
``2.5.1``     ``foss/2018b``      
``2.5.2``     ``GCC/8.2.0-2.31.1``
==========    ====================


.. _list_software_BeautifulSoup_1352:

*BeautifulSoup*
+++++++++++++++

Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping.

*homepage*: https://www.crummy.com/software/BeautifulSoup

=========    =================    =================
version      versionsuffix        toolchain        
=========    =================    =================
``4.6.0``    ``-Python-3.6.3``    ``intel/2017b``  
``4.6.3``    ``-Python-3.6.4``    ``intel/2018a``  
``4.7.1``    ``-Python-3.6.6``    ``intel/2018b``  
``4.8.0``                         ``GCCcore/8.2.0``
=========    =================    =================


.. _list_software_BEDOPS_445:

*BEDOPS*
++++++++

BEDOPS is an open-source command-line toolkit that performs highly efficient and  scalable Boolean and other set operations, statistical calculations, archiving, conversion and  other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for  distributing whole-genome analyses across a computational cluster.

*homepage*: http://bedops.readthedocs.io/en/latest/index.html

==========    ===============================
version       toolchain                      
==========    ===============================
``2.4.1``     ``GCC/4.8.4``                  
``2.4.2``     ``GCC/4.8.2``                  
``2.4.20``    ``system``                     
``2.4.26``    ``system``                     
``2.4.30``    ``foss/2016b``                 
``2.4.32``    ``foss/2018a``, ``intel/2018a``
``2.4.35``    ``foss/2018b``                 
==========    ===============================


.. _list_software_BEDTools_732:

*BEDTools*
++++++++++

The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps  and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,  and SAM/BAM.

*homepage*: https://github.com/arq5x/bedtools2

==========    ====================================================================================
version       toolchain                                                                           
==========    ====================================================================================
``2.25.0``    ``foss/2016a``                                                                      
``2.26.0``    ``GCCcore/6.4.0``, ``foss/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``2.27.1``    ``GCCcore/6.4.0``, ``foss/2016b``, ``foss/2018b``, ``intel/2017a``, ``intel/2018a`` 
``2.28.0``    ``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``                      
==========    ====================================================================================


.. _list_software_behave_619:

*behave*
++++++++

behave: Behavior-driven development (or BDD) is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.

*homepage*: http://pythonhosted.org/behave

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.2.5``    ``-Python-2.7.12``    ``foss/2016b`` 
``1.2.6``    ``-Python-3.6.4``     ``intel/2018a``
=========    ==================    ===============


.. _list_software_BerkeleyGW_977:

*BerkeleyGW*
++++++++++++

The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle  properties and the optical responses of a large variety of materials from bulk periodic crystals to  nanostructures such as slabs, wires and molecules.

*homepage*: http://www.berkeleygw.org

=============    ================================================================
version          toolchain                                                       
=============    ================================================================
``1.0.6``        ``intel/2016.02-GCC-4.9``                                       
``1.1-beta2``    ``intel/2016.02-GCC-4.9``                                       
``1.2.0``        ``intel/2017a``, ``intel/2018a``                                
``2.0.0``        ``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
=============    ================================================================


.. _list_software_BFAST_368:

*BFAST*
+++++++

BFAST facilitates the fast and accurate mapping of short reads to reference sequences.  Some advantages of BFAST include:  1) Speed: enables billions of short reads to be mapped quickly.   2) Accuracy: A priori probabilities for mapping reads with defined set of variants.   3) An easy way to measurably tune accuracy at the expense of speed.

*homepage*: http://bfast.sourceforge.net/

==========    ==============
version       toolchain     
==========    ==============
``0.7.0a``    ``foss/2016b``
==========    ==============


.. _list_software_BFC_203:

*BFC*
+++++

BFC is a standalone high-performance tool for correcting sequencing errors from Illumina sequencing data. It is specifically designed for high-coverage whole-genome human data, though also performs well for small genomes.

*homepage*: https://github.com/lh3/bfc

=======    ==============
version    toolchain     
=======    ==============
``1``      ``foss/2018a``
=======    ==============


.. _list_software_bibtexparser_1291:

*bibtexparser*
++++++++++++++

Bibtex parser in Python 2.7 and 3.x

*homepage*: https://github.com/sciunto-org/python-bibtexparser

=========    =================
version      toolchain        
=========    =================
``1.1.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_binutils_874:

*binutils*
++++++++++

binutils: GNU binary utilities

*homepage*: https://directory.fsf.org/project/binutils/

==========    ========================================================================================================================================================================================
version       toolchain                                                                                                                                                                               
==========    ========================================================================================================================================================================================
``2.25``      ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``system``
``2.25.1``    ``system``                                                                                                                                                                              
``2.26``      ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``system``                                                                                                  
``2.27``      ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``system``                                                                                                                     
``2.28``      ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``system``                                                                                                                     
``2.29``      ``GCCcore/7.2.0``, ``GCCcore/system``, ``system``                                                                                                                                       
``2.30``      ``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``system``                                                                                                                                        
``2.31.1``    ``GCCcore/7.4.0``, ``GCCcore/8.2.0``, ``system``                                                                                                                                        
``2.32``      ``GCCcore/8.3.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``system``                                                                                                                     
==========    ========================================================================================================================================================================================


.. _list_software_Bio-DB-HTS_745:

*Bio-DB-HTS*
++++++++++++

Read files using HTSlib including BAM/CRAM, Tabix and BCF database files

*homepage*: https://metacpan.org/release/Bio-DB-HTS

========    ================    ===============================
version     versionsuffix       toolchain                      
========    ================    ===============================
``2.11``    ``-Perl-5.26.0``    ``foss/2017b``, ``intel/2017b``
``2.11``    ``-Perl-5.26.1``    ``intel/2018a``                
``2.11``    ``-Perl-5.28.0``    ``foss/2018b``                 
``3.01``    ``-Perl-5.28.1``    ``GCC/8.2.0-2.31.1``           
========    ================    ===============================


.. _list_software_Bio-SamTools_1145:

*Bio-SamTools*
++++++++++++++

This is a Perl interface to the SAMtools sequence alignment interface.

*homepage*: http://search.cpan.org/~lds/Bio-SamTools/

========    ================    ===============
version     versionsuffix       toolchain      
========    ================    ===============
``1.43``    ``-Perl-5.24.1``    ``intel/2017a``
========    ================    ===============


.. _list_software_bioawk_637:

*bioawk*
++++++++

Bioawk is an extension to Brian Kernighan's awk,  adding the support of several common biological data formats,  including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names. 

*homepage*: https://github.com/lh3/bioawk

=======    ==============
version    toolchain     
=======    ==============
``1.0``    ``foss/2018b``
=======    ==============


.. _list_software_biobambam2_972:

*biobambam2*
++++++++++++

Tools for processing BAM files

*homepage*: https://github.com/gt1/biobambam2

==========    ===============
version       toolchain      
==========    ===============
``2.0.87``    ``intel/2018a``
==========    ===============


.. _list_software_biomart-perl_1230:

*biomart-perl*
++++++++++++++

The BioMart Perl API allows you to go a step further with BioMart   and integrate BioMart Perl Code into custom Perl scripts.

*homepage*: https://useast.ensembl.org/info/data/biomart/biomart_perl_api.html

===============    ================    =================
version            versionsuffix       toolchain        
===============    ================    =================
``0.7_e6db561``    ``-Perl-5.26.0``    ``GCCcore/6.4.0``
===============    ================    =================


.. _list_software_BioPerl_685:

*BioPerl*
+++++++++

Bioperl is the product of a community effort to produce Perl code which is useful in biology.  Examples include Sequence objects, Alignment objects and database searching objects.

*homepage*: http://www.bioperl.org/

===========    ================    ===============================
version        versionsuffix       toolchain                      
===========    ================    ===============================
``1.6.924``    ``-Perl-5.20.3``    ``intel/2016a``                
``1.6.924``    ``-Perl-5.22.1``    ``foss/2016a``                 
``1.7.0``      ``-Perl-5.24.0``    ``foss/2016b``                 
``1.7.1``      ``-Perl-5.24.0``    ``intel/2016b``                
``1.7.1``      ``-Perl-5.24.1``    ``intel/2017a``                
``1.7.2``                          ``GCCcore/8.3.0``              
``1.7.2``      ``-Perl-5.26.0``    ``foss/2017b``, ``intel/2017b``
``1.7.2``      ``-Perl-5.26.1``    ``intel/2018a``                
``1.7.2``      ``-Perl-5.28.0``    ``foss/2018b``, ``intel/2018b``
``1.7.2``      ``-Perl-5.28.1``    ``GCCcore/8.2.0``              
===========    ================    ===============================


.. _list_software_Biopython_956:

*Biopython*
+++++++++++

Biopython is a set of freely available tools for biological  computation written in Python by an international team of developers. It is  a distributed collaborative effort to develop Python libraries and  applications which address the needs of current and future work in  bioinformatics. 

*homepage*: https://www.biopython.org

========    ==================    ===================================================
version     versionsuffix         toolchain                                          
========    ==================    ===================================================
``1.65``    ``-Python-2.7.11``    ``foss/2016a``                                     
``1.68``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                    
``1.68``    ``-Python-3.5.2``     ``foss/2016b``, ``intel/2016b``                    
``1.70``    ``-Python-2.7.13``    ``foss/2017a``                                     
``1.70``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``                    
``1.70``    ``-Python-3.6.1``     ``intel/2017a``                                    
``1.70``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``                    
``1.71``    ``-Python-2.7.14``    ``intel/2018a``                                    
``1.71``    ``-Python-3.6.4``     ``foss/2018a``                                     
``1.72``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``                    
``1.72``    ``-Python-3.6.6``     ``foss/2018b``                                     
``1.73``                          ``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
``1.73``    ``-Python-3.6.6``     ``foss/2018b``                                     
``1.74``                          ``foss/2019a``                                     
``1.75``    ``-Python-3.7.4``     ``foss/2019b``, ``intel/2019b``                    
========    ==================    ===================================================


.. _list_software_biscuit_755:

*biscuit*
+++++++++

Utilities to help analyze bisulfite-treated sequence data  

*homepage*: https://github.com/zwdzwd/biscuit

=========    ==============
version      toolchain     
=========    ==============
``0.1.4``    ``foss/2016a``
=========    ==============


.. _list_software_Bismark_713:

*Bismark*
+++++++++

A tool to map bisulfite converted sequence reads and determine cytosine methylation states

*homepage*: http://www.bioinformatics.babraham.ac.uk/projects/bismark/

==========    ===============================
version       toolchain                      
==========    ===============================
``0.19.0``    ``intel/2017b``                
``0.20.1``    ``foss/2018b``, ``intel/2018b``
==========    ===============================


.. _list_software_Bison_507:

*Bison*
+++++++

Bison is a general-purpose parser generator that converts an annotated  context-free grammar into a deterministic LR or generalized LR (GLR) parser  employing LALR(1) parser tables. 

*homepage*: http://www.gnu.org/software/bison

=========    ==========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
=========    ==========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``2.7``      ``GCC/4.8.1``, ``GCC/4.8.4``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``system``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
``3.0.2``    ``GCC/4.8.2``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
``3.0.3``    ``GCC/4.9.2``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
``3.0.4``    ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``GCCcore/system``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``3.0.5``    ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/system``, ``system``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
``3.2.2``    ``GCCcore/7.4.0``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
``3.3.2``    ``GCCcore/8.3.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``system``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
=========    ==========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_bitarray_862:

*bitarray*
++++++++++

bitarray provides an object type which efficiently represents an array of booleans

*homepage*: https://github.com/ilanschnell/bitarray

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.8.3``    ``-Python-2.7.15``    ``intel/2018b``
``0.8.3``    ``-Python-3.6.6``     ``intel/2018b``
=========    ==================    ===============


.. _list_software_BLACS_357:

*BLACS*
+++++++

The BLACS (Basic Linear Algebra Communication Subprograms) project is  an ongoing investigation whose purpose is to create a linear algebra oriented message passing interface  that may be implemented efficiently and uniformly across a large range of distributed memory platforms.

*homepage*: http://www.netlib.org/blacs/

=======    ===================
version    toolchain          
=======    ===================
``1.1``    ``gmvapich2/2016a``
=======    ===================


.. _list_software_BLASR_372:

*BLASR*
+++++++

The PacBio® long read aligner

*homepage*: https://github.com/PacificBiosciences/blasr

============    ===============
version         toolchain      
============    ===============
``2.2``         ``intel/2016b``
``5.3.3``       ``gompi/2019a``
``20170330``    ``intel/2017a``
============    ===============


.. _list_software_blasr_libcpp_1261:

*blasr_libcpp*
++++++++++++++

Blasr_libcpp is a library used by blasr and other executables such as samtoh5, loadPulses for  analyzing PacBio sequences

*homepage*: https://github.com/PacificBiosciences/blasr_libcpp

============    ===============
version         toolchain      
============    ===============
``20170426``    ``intel/2017a``
============    ===============


.. _list_software_BLAST_374:

*BLAST*
+++++++

Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological  sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.

*homepage*: http://blast.ncbi.nlm.nih.gov/

==========    =================    ==========
version       versionsuffix        toolchain 
==========    =================    ==========
``2.2.26``    ``-Linux_x86_64``    ``system``
``2.8.1``     ``-Linux_x86_64``    ``system``
==========    =================    ==========


.. _list_software_BLAST+_417:

*BLAST+*
++++++++

Basic Local Alignment Search Tool, or BLAST, is an algorithm  for comparing primary biological sequence information, such as the amino-acid  sequences of different proteins or the nucleotides of DNA sequences.

*homepage*: http://blast.ncbi.nlm.nih.gov/

==========    ==================    ================================================================
version       versionsuffix         toolchain                                                       
==========    ==================    ================================================================
``2.2.31``                          ``system``                                                      
``2.3.0``     ``-Python-2.7.11``    ``foss/2016a``                                                  
``2.6.0``     ``-Python-2.7.12``    ``foss/2016b``                                                  
``2.6.0``     ``-Python-2.7.13``    ``foss/2017a``, ``intel/2017a``                                 
``2.6.0``     ``-Python-2.7.14``    ``intel/2017b``                                                 
``2.7.1``                           ``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``2.7.1``     ``-Python-2.7.14``    ``intel/2017b``                                                 
``2.8.1``                           ``foss/2018b``                                                  
``2.9.0``                           ``gompi/2019a``, ``gompi/2019b``, ``iimpi/2019a``               
==========    ==================    ================================================================


.. _list_software_BLAT_291:

*BLAT*
++++++

BLAT on DNA is designed to quickly find sequences of 95% and greater similarity  of length 25 bases or more.

*homepage*: http://genome.ucsc.edu/FAQ/FAQblat.html

=======    ======================================================================================
version    toolchain                                                                             
=======    ======================================================================================
``3.5``    ``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2018b``, ``intel/2016b``, ``intel/2017a``
=======    ======================================================================================


.. _list_software_Blender_700:

*Blender*
+++++++++

Blender is the free and open source 3D creation suite. It supports  the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,  compositing and motion tracking, even video editing and game creation.

*homepage*: https://www.blender.org/

=========    =============================    ===============================
version      versionsuffix                    toolchain                      
=========    =============================    ===============================
``2.77a``    ``-Python-3.5.2``                ``intel/2016b``                
``2.79``     ``-Python-3.6.1``                ``intel/2017a``                
``2.79b``    ``-Python-3.6.6``                ``intel/2018b``                
``2.79b``    ``-Python-3.6.6-CUDA-9.2.88``    ``foss/2018b``                 
``2.81``     ``-Python-3.7.4``                ``foss/2019b``, ``intel/2019b``
=========    =============================    ===============================


.. _list_software_BLIS_298:

*BLIS*
++++++

AMD fork of BLIS. BLIS is a portable software framework for instantiating high-performance  BLAS-like dense linear algebra libraries.

*homepage*: https://developer.amd.com/amd-cpu-libraries/blas-library/

=========    =============    ==================
version      versionsuffix    toolchain         
=========    =============    ==================
``0.3.2``                     ``GCC/7.3.0-2.30``
``0.6.0``                     ``GCC/8.3.0-2.32``
``1.2``      ``-amd``         ``GCC/7.3.0-2.30``
=========    =============    ==================


.. _list_software_Blitz++_603:

*Blitz++*
+++++++++

Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation in C++  with a speed comparable to Fortran implementations, while preserving an object-oriented interface

*homepage*: http://blitz.sourceforge.net/

========    =================================
version     toolchain                        
========    =================================
``0.10``    ``GCCcore/6.4.0``, ``foss/2016a``
========    =================================


.. _list_software_BlobTools_912:

*BlobTools*
+++++++++++

A modular command-line solution for visualisation,  quality control and taxonomic partitioning of genome datasets. 

*homepage*: https://blobtools.readme.io/docs

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20180528``    ``-Python-2.7.15``    ``foss/2018b``
============    ==================    ==============


.. _list_software_Blosc_499:

*Blosc*
+++++++

Blosc, an extremely fast, multi-threaded, meta-compressor library

*homepage*: http://www.blosc.org/

==========    ==================================================================
version       toolchain                                                         
==========    ==================================================================
``1.11.1``    ``intel/2016b``                                                   
``1.12.1``    ``GCCcore/6.4.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2017a``
``1.14.2``    ``GCCcore/6.4.0``, ``foss/2016a``                                 
``1.14.4``    ``GCCcore/7.3.0``                                                 
``1.17.0``    ``GCCcore/8.2.0``                                                 
==========    ==================================================================


.. _list_software_bmtagger_841:

*bmtagger*
++++++++++

Best Match Tagger for removing human reads from metagenomics datasets

*homepage*: ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/

=========    ===============================
version      toolchain                      
=========    ===============================
``3.101``    ``foss/2018b``, ``gompi/2019a``
=========    ===============================


.. _list_software_bnpy_441:

*bnpy*
++++++

Bayesian nonparametric machine learning for python provides  code for training popular clustering models on large datasets. The focus is  on Bayesian nonparametric models based on the Dirichlet process, but it also  provides parametric counterparts.

*homepage*: https://github.com/bnpy/bnpy

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``0.1.6``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_BOINC_363:

*BOINC*
+++++++

BOINC is a program that lets you donate your idle computer time to science projects  like SETI@home, Climateprediction.net, Rosetta@home, World Community Grid, and many others.

*homepage*: https://boinc.berkeley.edu

==========    =============    =============
version       versionsuffix    toolchain    
==========    =============    =============
``7.2.42``    ``-client``      ``GCC/4.8.2``
==========    =============    =============


.. _list_software_bokeh_521:

*bokeh*
+++++++

Statistical and novel interactive HTML plots for Python

*homepage*: https://github.com/bokeh/bokeh

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``0.12.3``     ``-Python-2.7.12``    ``intel/2016b``                
``0.12.3``     ``-Python-3.5.2``     ``intel/2016b``                
``0.12.15``    ``-Python-3.6.4``     ``intel/2018a``                
``1.0.4``      ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
``1.3.4``      ``-Python-3.7.2``     ``foss/2019a``                 
``1.4.0``      ``-Python-3.7.4``     ``foss/2019b``                 
===========    ==================    ===============================


.. _list_software_BoltzTraP_898:

*BoltzTraP*
+++++++++++

Boltzmann Transport Properties (BoltzTraP) is a program for calculating the semi-classic transport  coefficients.

*homepage*: http://www.icams.de/content/departments/cmat/boltztrap/

=========    ===============
version      toolchain      
=========    ===============
``1.2.5``    ``intel/2016a``
=========    ===============


.. _list_software_Bonmin_611:

*Bonmin*
++++++++

Ipopt (Interior Point OPTimizer, pronounced eye-pea-Opt) is a software package for  large-scale nonlinear optimization.

*homepage*: https://coin-or.github.io/Ipopt

=========    ===============
version      toolchain      
=========    ===============
``1.8.7``    ``intel/2019a``
=========    ===============


.. _list_software_Bonnie++_689:

*Bonnie++*
++++++++++

Bonnie++-1.97: Enhanced performance Test of Filesystem I/O

*homepage*: http://www.coker.com.au/bonnie++/

========    ==============
version     toolchain     
========    ==============
``1.97``    ``foss/2016a``
========    ==============


.. _list_software_Boost_519:

*Boost*
+++++++

Boost provides free peer-reviewed portable C++ source libraries.

*homepage*: https://www.boost.org/

==========    ==================    ====================================================================
version       versionsuffix         toolchain                                                           
==========    ==================    ====================================================================
``1.54.0``    ``-Python-2.7.12``    ``foss/2016b``                                                      
``1.55.0``                          ``system``                                                          
``1.55.0``    ``-Python-2.7.11``    ``foss/2016a``                                                      
``1.57.0``    ``-Python-2.7.10``    ``gimkl/2.11.5``                                                    
``1.58.0``                          ``intel/2017a``                                                     
``1.58.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                                     
``1.58.0``    ``-serial``           ``GCC/4.9.2``                                                       
``1.59.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                                     
``1.60.0``                          ``foss/2016a``, ``intel/2016a``                                     
``1.60.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                                     
``1.60.0``    ``-Python-3.5.1``     ``foss/2016a``                                                      
``1.61.0``                          ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``    
``1.61.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                                     
``1.61.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                     
``1.61.0``    ``-Python-3.5.1``     ``foss/2016a``                                                      
``1.62.0``    ``-Python-2.7.12``    ``intel/2016b``                                                     
``1.63.0``                          ``foss/2017a``                                                      
``1.63.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                     
``1.63.0``    ``-Python-2.7.13``    ``foss/2017a``, ``intel/2017a``                                     
``1.63.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2017b``                                     
``1.63.0``    ``-Python-3.5.2``     ``foss/2016b``                                                      
``1.64.0``                          ``gompi/2019a``, ``gompic/2019a``, ``intel/2017a``                  
``1.64.0``    ``-Python-2.7.13``    ``intel/2017a``                                                     
``1.65.0``    ``-Python-2.7.13``    ``intel/2017a``                                                     
``1.65.1``                          ``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``    
``1.65.1``    ``-Python-2.7.13``    ``intel/2017a``                                                     
``1.65.1``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``                                     
``1.65.1``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``                                     
``1.66.0``                          ``foss/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a`` 
``1.66.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2017b``, ``intel/2018a``                    
``1.66.0``    ``-Python-3.6.3``     ``intel/2018.01``                                                   
``1.66.0``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                                     
``1.66.0``    ``-no_mpi``           ``GCCcore/6.4.0``                                                   
``1.67.0``                          ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``
``1.67.0``    ``-Python-2.7.14``    ``foss/2018a``                                                      
``1.68.0``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``                                     
``1.68.0``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                                     
``1.69.0``                          ``intel/2019.01``                                                   
``1.70.0``                          ``gompi/2019a``, ``iimpi/2019a``, ``iimpic/2019a``                  
``1.71.0``                          ``gompi/2019b``, ``iimpi/2019b``                                    
==========    ==================    ====================================================================


.. _list_software_Boost.Python_1207:

*Boost.Python*
++++++++++++++

Boost.Python is a C++ library which enables seamless interoperability between C++  and the Python programming language.

*homepage*: http://boostorg.github.io/python

==========    ==================    ===================================================
version       versionsuffix         toolchain                                          
==========    ==================    ===================================================
``1.64.0``                          ``gompi/2019a``, ``gompic/2019a``                  
``1.65.1``    ``-Python-2.7.14``    ``intel/2017b``                                    
``1.66.0``    ``-Python-2.7.14``    ``intel/2018a``                                    
``1.66.0``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                    
``1.67.0``    ``-Python-2.7.15``    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.67.0``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                    
``1.70.0``                          ``gompi/2019a``, ``iimpi/2019a``                   
==========    ==================    ===================================================


.. _list_software_Bottleneck_1035:

*Bottleneck*
++++++++++++

Fast NumPy array functions written in C

*homepage*: https://kwgoodman.github.io/bottleneck-doc

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.2.1``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_Bowtie_618:

*Bowtie*
++++++++

Bowtie is an ultrafast, memory-efficient short read aligner.  It aligns short DNA sequences (reads) to the human genome.

*homepage*: http://bowtie-bio.sourceforge.net/index.shtml

===========    =======================================================================================================
version        toolchain                                                                                              
===========    =======================================================================================================
``1.1.2``      ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``
``1.2.1.1``    ``foss/2016b``, ``intel/2017b``                                                                        
``1.2.2``      ``foss/2018b``, ``intel/2017b``, ``intel/2018a``                                                       
``1.2.3``      ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``                         
===========    =======================================================================================================


.. _list_software_Bowtie2_668:

*Bowtie2*
+++++++++

Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads  to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s  of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.  Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,  its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.

*homepage*: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml

===========    ==============================================================
version        toolchain                                                     
===========    ==============================================================
``2.2.8``      ``foss/2016a``                                                
``2.2.9``      ``foss/2016a``, ``intel/2016b``                               
``2.3.2``      ``foss/2016b``, ``foss/2017a``, ``intel/2017a``               
``2.3.3.1``    ``intel/2017b``                                               
``2.3.4``      ``intel/2017b``                                               
``2.3.4.1``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``              
``2.3.4.2``    ``foss/2018b``, ``intel/2018b``                               
``2.3.4.3``    ``foss/2017b``                                                
``2.3.5.1``    ``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
===========    ==============================================================


.. _list_software_Bpipe_496:

*Bpipe*
+++++++

Bpipe - a tool for running and managing bioinformatics pipelines

*homepage*: http://docs.bpipe.org/

===========    ===============
version        toolchain      
===========    ===============
``0.9.9.2``    ``intel/2017a``
===========    ===============


.. _list_software_bpp-core_792:

*bpp-core*
++++++++++

Bio++ Core Library

*homepage*: https://github.com/BioPP/bpp-core

=========    ====================
version      toolchain           
=========    ====================
``2.4.1``    ``GCC/8.2.0-2.31.1``
=========    ====================


.. _list_software_bpp-phyl_812:

*bpp-phyl*
++++++++++

Bio++ Phylogenetic Library

*homepage*: https://github.com/BioPP/bpp-phyl

=========    ====================
version      toolchain           
=========    ====================
``2.4.1``    ``GCC/8.2.0-2.31.1``
=========    ====================


.. _list_software_bpp-seq_696:

*bpp-seq*
+++++++++

Bio++ Sequence Library

*homepage*: https://github.com/BioPP/bpp-seq

=========    ====================
version      toolchain           
=========    ====================
``2.4.1``    ``GCC/8.2.0-2.31.1``
=========    ====================


.. _list_software_BRAKER_439:

*BRAKER*
++++++++

BRAKER is a pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET  and AUGUSTUS in novel eukaryotic genomes.

*homepage*: https://github.com/Gaius-Augustus/BRAKER

=========    ===============
version      toolchain      
=========    ===============
``2.1.2``    ``intel/2019a``
=========    ===============


.. _list_software_BreakDancer_1074:

*BreakDancer*
+++++++++++++

BreakDancer is a Perl/C++ package that provides genome-wide detection of structural variants from  next generation paired-end sequencing reads

*homepage*: http://gmt.genome.wustl.edu/packages/breakdancer

=========    ===============
version      toolchain      
=========    ===============
``1.4.5``    ``intel/2017a``
=========    ===============


.. _list_software_bsddb3_562:

*bsddb3*
++++++++

bsddb3 is a nearly complete Python binding of the Oracle/Sleepycat C API for the Database Environment, Database, Cursor, Log Cursor, Sequence and Transaction objects.

*homepage*: https://pypi.org/project/bsddb3/

=========    ==================    ==================
version      versionsuffix         toolchain         
=========    ==================    ==================
``6.2.6``                          ``GCCcore/8.2.0`` 
``6.2.6``    ``-Python-2.7.15``    ``fosscuda/2018b``
=========    ==================    ==================


.. _list_software_buildenv_857:

*buildenv*
++++++++++

This module sets a group of environment variables for compilers, linkers, maths libraries, etc., that  you can use to easily transition between toolchains when building your software. To query the variables being set  please use: module show <this module name>

*homepage*: None

===========    =================================================================================================
version        toolchain                                                                                        
===========    =================================================================================================
``default``    ``foss/2017b``, ``foss/2018b``, ``foss/2019b``, ``intel/2016b``, ``intel/2017a``, ``intel/2019b``
===========    =================================================================================================


.. _list_software_Bullet_616:

*Bullet*
++++++++

Bullet professional 3D Game Multiphysics Library provides state  of the art collision detection, soft body and rigid body dynamics.

*homepage*: http://bulletphysics.org/

==========    ===============================
version       toolchain                      
==========    ===============================
``2.83.7``    ``foss/2016a``, ``intel/2016a``
==========    ===============================


.. _list_software_BUSCO_380:

*BUSCO*
+++++++

BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs

*homepage*: http://busco.ezlab.org/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.22``     ``-Python-2.7.13``    ``intel/2017a``
``2.0.1``    ``-Python-2.7.13``    ``intel/2017a``
``3.0.2``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_BWA_218:

*BWA*
+++++

Burrows-Wheeler Aligner (BWA) is an efficient program that aligns  relatively short nucleotide sequences against a long reference sequence such as the human genome.

*homepage*: http://bio-bwa.sourceforge.net/

===========    =================================================================================================================================================================
version        toolchain                                                                                                                                                        
===========    =================================================================================================================================================================
``0.7.12``     ``foss/2016b``                                                                                                                                                   
``0.7.13``     ``foss/2016a``, ``intel/2016a``                                                                                                                                  
``0.7.15``     ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``                                                           
``0.7.16a``    ``foss/2016b``, ``intel/2017a``                                                                                                                                  
``0.7.17``     ``GCC/8.2.0-2.31.1``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
===========    =================================================================================================================================================================


.. _list_software_bwa-meth_789:

*bwa-meth*
++++++++++

Fast and accurante alignment of BS-Seq reads.

*homepage*: https://github.com/brentp/bwa-meth

=========    ========================================
version      toolchain                               
=========    ========================================
``0.2.2``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
=========    ========================================


.. _list_software_bwakit_642:

*bwakit*
++++++++

Bwakit is a self-consistent installation-free package of scripts and precompiled binaries, providing an end-to-end solution to read mapping.

*homepage*: https://github.com/lh3/bwa/tree/master/bwakit

==========    ==============    ==========
version       versionsuffix     toolchain 
==========    ==============    ==========
``0.7.15``    ``_x64-linux``    ``system``
==========    ==============    ==========


.. _list_software_bwidget_742:

*bwidget*
+++++++++

The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces.

*homepage*: https://core.tcl-lang.org/bwidget/home

==========    =================
version       toolchain        
==========    =================
``1.9.13``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_BWISE_378:

*BWISE*
+++++++

de Bruijn Workflow using Integral information of Short pair End reads

*homepage*: https://github.com/Malfoy/BWISE

============    =================    ==============
version         versionsuffix        toolchain     
============    =================    ==============
``20180820``    ``-Python-3.6.4``    ``foss/2018a``
============    =================    ==============


.. _list_software_bx-python_937:

*bx-python*
+++++++++++

The bx-python project is a Python library and associated set of scripts to allow for rapid  implementation of genome scale analyses.

*homepage*: https://github.com/bxlab/bx-python

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.7.4``    ``-Python-2.7.12``    ``foss/2016b`` 
``0.7.4``    ``-Python-2.7.13``    ``intel/2017a``
``0.8.1``    ``-Python-2.7.14``    ``intel/2018a``
``0.8.2``    ``-Python-3.6.6``     ``foss/2018b`` 
=========    ==================    ===============


.. _list_software_BXH_XCEDE_TOOLS_1178:

*BXH_XCEDE_TOOLS*
+++++++++++++++++

A collection of data processing and image analysis  tools for data in BXH or XCEDE format. This includes data format  encapsulation/conversion, event-related analysis, QA tools, and more.  These tools form the basis of the fBIRN QA procedures and are also  distributed as part of the fBIRN Data Upload Scripts.

*homepage*: http://www.nitrc.org/projects/bxh_xcede_tools/

==========    ==========
version       toolchain 
==========    ==========
``1.11.1``    ``system``
==========    ==========


.. _list_software_byacc_514:

*byacc*
+++++++

Berkeley Yacc (byacc) is generally conceded to be the best yacc variant  available.  In contrast to bison, it is written to avoid dependencies  upon a particular compiler. 

*homepage*: http://invisible-island.net/byacc/byacc.html

============    ===============================
version         toolchain                      
============    ===============================
``20160324``    ``intel/2016a``                
``20160606``    ``foss/2016b``, ``intel/2016b``
``20170709``    ``GCCcore/6.4.0``              
============    ===============================


.. _list_software_bzip2_487:

*bzip2*
+++++++

bzip2 is a freely available, patent free, high-quality data compressor. It  typically compresses files to within 10% to 15% of the best available  techniques (the PPM family of statistical compressors), whilst being around  twice as fast at compression and six times faster at decompression. 

*homepage*: https://sourceware.org/bzip2

=========    ==============================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
=========    ==============================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``1.0.6``    ``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.0.8``    ``GCCcore/8.3.0``                                                                                                                                                                                                                                                                                                                                                                                                                                                             
=========    ==============================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_letter_c:

*C*
---


:ref:`list_software_C3D_186` - :ref:`list_software_cadaver_726` - :ref:`list_software_Caffe_469` - :ref:`list_software_cairo_526` - :ref:`list_software_cairomm_744` - :ref:`list_software_Calendrical_1106` - :ref:`list_software_Cantera_702` - :ref:`list_software_canu_423` - :ref:`list_software_Canvas_604` - :ref:`list_software_CAP3_263` - :ref:`list_software_CapnProto_918` - :ref:`list_software_Cargo_492` - :ref:`list_software_CastXML_636` - :ref:`list_software_Catch2_533` - :ref:`list_software_CatMAP_502` - :ref:`list_software_causalml_850` - :ref:`list_software_CaVEMan_603` - :ref:`list_software_CBLAS_357` - :ref:`list_software_ccache_599` - :ref:`list_software_CCL_210` - :ref:`list_software_cclib_509` - :ref:`list_software_CD-HIT_409` - :ref:`list_software_cDNA_Cupcake_1105` - :ref:`list_software_CDO_214` - :ref:`list_software_cdsapi_628` - :ref:`list_software_CellMix_686` - :ref:`list_software_CellRanger_991` - :ref:`list_software_Centrifuge_1036` - :ref:`list_software_CESM-deps_769` - :ref:`list_software_CFITSIO_529` - :ref:`list_software_cftime_632` - :ref:`list_software_CGAL_279` - :ref:`list_software_cget_419` - :ref:`list_software_CGNS_299` - :ref:`list_software_CharLS_541` - :ref:`list_software_charmm_632` - :ref:`list_software_CHARMM_440` - :ref:`list_software_CHASE_356` - :ref:`list_software_CheckM_555` - :ref:`list_software_Cheetah_690` - :ref:`list_software_CheMPS2_562` - :ref:`list_software_Chimera_697` - :ref:`list_software_ChimPipe_783` - :ref:`list_software_Chromaprint_1159` - :ref:`list_software_CIRCexplorer_1170` - :ref:`list_software_CIRCexplorer2_1220` - :ref:`list_software_Circos_611` - :ref:`list_software_CIRI_295` - :ref:`list_software_cisTEM_549` - :ref:`list_software_CITE-seq-Count_1233` - :ref:`list_software_Clang_485` - :ref:`list_software_Clang-Python-bindings_2063` - :ref:`list_software_CLAPACK_495` - :ref:`list_software_CLHEP_364` - :ref:`list_software_CLISP_379` - :ref:`list_software_ClonalFrameML_1245` - :ref:`list_software_CLooG_436` - :ref:`list_software_Clustal-Omega_1262` - :ref:`list_software_ClustalW2_865` - :ref:`list_software_Cluster-Buster_1412` - :ref:`list_software_ClusterShell_1242` - :ref:`list_software_CMake_449` - :ref:`list_software_CNVkit_559` - :ref:`list_software_CNVnator_779` - :ref:`list_software_CODEX2_421` - :ref:`list_software_Commet_613` - :ref:`list_software_Con3F_409` - :ref:`list_software_CONCOCT_521` - :ref:`list_software_configparser_1283` - :ref:`list_software_configurable-http-proxy_2381` - :ref:`list_software_ConnectomeWorkbench_1966` - :ref:`list_software_Control-FREEC_1139` - :ref:`list_software_CoordgenLibs_1211` - :ref:`list_software_Coot_405` - :ref:`list_software_core-counter_1238` - :ref:`list_software_Coreutils_954` - :ref:`list_software_corner_649` - :ref:`list_software_coverage_844` - :ref:`list_software_cowsay_662` - :ref:`list_software_CP2K_272` - :ref:`list_software_CPB_213` - :ref:`list_software_CPLEX_380` - :ref:`list_software_CppUnit_707` - :ref:`list_software_cram_419` - :ref:`list_software_cramtools_980` - :ref:`list_software_CrayCCE_602` - :ref:`list_software_CrayGNU_633` - :ref:`list_software_CrayIntel_907` - :ref:`list_software_CrayPGI_623` - :ref:`list_software_crb-blast_890` - :ref:`list_software_CRF++_305` - :ref:`list_software_CrossTalkZ_1008` - :ref:`list_software_CryptoMiniSat_1334` - :ref:`list_software_cscope_637` - :ref:`list_software_csvkit_660` - :ref:`list_software_ctags_530` - :ref:`list_software_ctffind_734` - :ref:`list_software_Cube_383` - :ref:`list_software_CubeGUI_612` - :ref:`list_software_CubeLib_662` - :ref:`list_software_CubeWriter_1020` - :ref:`list_software_CUDA_285` - :ref:`list_software_cuDNN_440` - :ref:`list_software_Cufflinks_933` - :ref:`list_software_CUnit_483` - :ref:`list_software_cURL_342` - :ref:`list_software_custodian_970` - :ref:`list_software_cutadapt_854` - :ref:`list_software_CVS_236` - :ref:`list_software_CVXOPT_484` - :ref:`list_software_CVXPY_410` - :ref:`list_software_cysignals_973` - :ref:`list_software_Cython_629` - :ref:`list_software_cytosim_776` - :ref:`list_software_cyvcf2_589`


.. _list_software_C3D_186:

*C3D*
+++++

Convert3D Medical Image Processing Tool

*homepage*: https://sourceforge.net/projects/c3d/

=========    ==========
version      toolchain 
=========    ==========
``1.0.0``    ``system``
=========    ==========


.. _list_software_cadaver_726:

*cadaver*
+++++++++

cadaver is a command-line WebDAV client for Unix.

*homepage*: http://www.webdav.org/cadaver/

==========    ===============
version       toolchain      
==========    ===============
``0.23.3``    ``intel/2017a``
==========    ===============


.. _list_software_Caffe_469:

*Caffe*
+++++++

Caffe is a deep learning framework made with expression, speed,  and modularity in mind. It is developed by the Berkeley Vision  and Learning Center (BVLC) and community contributors. 

*homepage*: https://github.com/BVLC/caffe

=======    ==============================    ===============
version    versionsuffix                     toolchain      
=======    ==============================    ===============
``1.0``    ``-CUDA-9.1.85-Python-2.7.14``    ``intel/2017b``
``1.0``    ``-Python-2.7.13``                ``intel/2017a``
``1.0``    ``-Python-2.7.14``                ``intel/2017b``
``rc3``    ``-CUDA-7.5.18-Python-2.7.11``    ``foss/2016a`` 
=======    ==============================    ===============


.. _list_software_cairo_526:

*cairo*
+++++++

Cairo is a 2D graphics library with support for multiple output devices.  Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,  PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB

*homepage*: http://cairographics.org

===========    ================    ================================================================
version        versionsuffix       toolchain                                                       
===========    ================    ================================================================
``1.14.6``                         ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``1.14.6``     ``-GLib-2.48.0``    ``foss/2016a``, ``intel/2016a``                                 
``1.14.8``                         ``intel/2017a``                                                 
``1.14.10``                        ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``intel/2017b``           
``1.14.12``                        ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                            
``1.16.0``                         ``GCCcore/8.2.0``                                               
===========    ================    ================================================================


.. _list_software_cairomm_744:

*cairomm*
+++++++++

The Cairomm package provides a C++ interface to Cairo.

*homepage*: http://cairographics.org

==========    ====================================
version       toolchain                           
==========    ====================================
``1.12.2``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
==========    ====================================


.. _list_software_Calendrical_1106:

*Calendrical*
+++++++++++++

Calendrical module is for calendrical calculations.

*homepage*: https://www.funaba.org/code#calendrical

==========    =================    ===============
version       versionsuffix        toolchain      
==========    =================    ===============
``2.0.1``     ``-Python-3.6.3``    ``intel/2017b``
``2.0.2a``    ``-Python-3.6.4``    ``intel/2018a``
``2.0.2a``    ``-Python-3.6.6``    ``intel/2018b``
==========    =================    ===============


.. _list_software_Cantera_702:

*Cantera*
+++++++++

Chemical kinetics, thermodynamics, and transport tool suite

*homepage*: https://github.com/Cantera/cantera

=========    ==================    ================================================
version      versionsuffix         toolchain                                       
=========    ==================    ================================================
``2.2.1``    ``-Python-2.7.12``    ``intel/2016b``                                 
``2.3.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                 
``2.3.0``    ``-Python-2.7.13``    ``intel/2017a``                                 
``2.3.0``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``2.4.0``    ``-Python-2.7.14``    ``intel/2018a``                                 
=========    ==================    ================================================


.. _list_software_canu_423:

*canu*
++++++

Canu is a fork of the Celera Assembler designed for high-noise single-molecule sequencing

*homepage*: http://canu.readthedocs.io

=======    ================    ===============================
version    versionsuffix       toolchain                      
=======    ================    ===============================
``1.4``                        ``foss/2016b``                 
``1.7``                        ``intel/2018a``                
``1.8``    ``-Perl-5.26.0``    ``foss/2017b``, ``intel/2017b``
``1.8``    ``-Perl-5.28.0``    ``foss/2018b``                 
``1.8``    ``-Perl-5.28.1``    ``GCCcore/8.2.0``              
=======    ================    ===============================


.. _list_software_Canvas_604:

*Canvas*
++++++++

Copy number variant (CNV) calling from DNA sequencing data

*homepage*: https://github.com/Illumina/canvas

===============    ==========
version            toolchain 
===============    ==========
``1.39.0.1598``    ``system``
===============    ==========


.. _list_software_CAP3_263:

*CAP3*
++++++

CAP3 assembly program 

*homepage*: http://seq.cs.iastate.edu/

=========================    ==========
version                      toolchain 
=========================    ==========
``20071221-intel-x86``       ``system``
``20071221-intel-x86_64``    ``system``
``20071221-opteron``         ``system``
=========================    ==========


.. _list_software_CapnProto_918:

*CapnProto*
+++++++++++

Cap’n Proto is an insanely fast data interchange format and capability-based RPC system.

*homepage*: https://capnproto.org

=========    =================
version      toolchain        
=========    =================
``0.6.1``    ``GCCcore/6.4.0``
``0.7.0``    ``GCCcore/7.3.0``
=========    =================


.. _list_software_Cargo_492:

*Cargo*
+++++++

The Rust package manager

*homepage*: https://crates.io/

==========    ==============
version       toolchain     
==========    ==============
``0.13.0``    ``foss/2016b``
==========    ==============


.. _list_software_CastXML_636:

*CastXML*
+++++++++

CastXML is a C-family abstract syntax tree XML output tool.

*homepage*: https://github.com/CastXML/CastXML

============    ==============
version         toolchain     
============    ==============
``20160617``    ``foss/2016a``
``20180806``    ``foss/2018a``
============    ==============


.. _list_software_Catch2_533:

*Catch2*
++++++++

A modern, C++-native, header-only,   test framework for unit-tests, TDD and BDD   - using C++11, C++14, C++17 and later     (or C++03 on the Catch1.x branch) 

*homepage*: https://github.com/catchorg/Catch2

==========    ==========
version       toolchain 
==========    ==========
``2.9.1``     ``system``
``2.11.0``    ``system``
==========    ==========


.. _list_software_CatMAP_502:

*CatMAP*
++++++++

Catalyst Micro-kinetic Analysis Package for automated creation of micro-kinetic models used  in catalyst screening.

*homepage*: http://catmap.readthedocs.io/

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20170927``    ``-Python-2.7.14``    ``intel/2017b``
============    ==================    ===============


.. _list_software_causalml_850:

*causalml*
++++++++++

Causal ML: A Python Package for Uplift Modeling and Causal Inference with ML 

*homepage*: https://github.com/uber/causalml

==================    =================    ==============
version               versionsuffix        toolchain     
==================    =================    ==============
``0.3.0-20180610``    ``-Python-3.7.2``    ``foss/2019a``
==================    =================    ==============


.. _list_software_CaVEMan_603:

*CaVEMan*
+++++++++

SNV expectation maximisation based mutation calling algorithm   aimed at detecting somatic mutations in paired (tumour/normal) cancer samples.  Supports both bam and cram format via htslib

*homepage*: http://cancerit.github.io/CaVEMan/

==========    ==============
version       toolchain     
==========    ==============
``1.13.2``    ``foss/2018a``
==========    ==============


.. _list_software_CBLAS_357:

*CBLAS*
+++++++

C interface to the BLAS

*homepage*: http://www.netlib.org/blas/

============    ==============
version         toolchain     
============    ==============
``20110120``    ``foss/2016b``
============    ==============


.. _list_software_ccache_599:

*ccache*
++++++++

Cache for C/C++ compilers

*homepage*: http://ccache.samba.org/

=========    =============    ==========
version      versionsuffix    toolchain 
=========    =============    ==========
``3.2.5``                     ``system``
``3.3.1``                     ``system``
``3.3.3``                     ``system``
``3.3.4``    ``-f90``         ``system``
=========    =============    ==========


.. _list_software_CCL_210:

*CCL*
+++++

Clozure CL (often called CCL for short) is a free Common Lisp  implementation with a long history. Some distinguishing features of the implementation  include fast compilation speed, native threads, a precise, generational, compacting  garbage collector, and a convenient foreign-function interface.

*homepage*: https://ccl.clozure.com/

==========    ==========
version       toolchain 
==========    ==========
``1.11.5``    ``system``
==========    ==========


.. _list_software_cclib_509:

*cclib*
+++++++

cclib is a Python library that provides parsers for computational chemistry log files.  It alsoprovides a platform to implement algorithms in a package-independent manner. 

*homepage*: http://cclib.github.io/

=======    =================    ==============
version    versionsuffix        toolchain     
=======    =================    ==============
``1.5``    ``-Python-3.5.2``    ``foss/2016b``
=======    =================    ==============


.. _list_software_CD-HIT_409:

*CD-HIT*
++++++++

CD-HIT is a very widely used program for clustering and   comparing protein or nucleotide sequences.

*homepage*: http://weizhong-lab.ucsd.edu/cd-hit/

=========    ==============    ================================================
version      versionsuffix     toolchain                                       
=========    ==============    ================================================
``4.6.4``    ``-2015-0603``    ``GNU/4.9.3-2.25``                              
``4.6.6``                      ``foss/2016b``                                  
``4.6.8``                      ``foss/2018b``, ``intel/2017a``, ``intel/2018a``
``4.8.1``                      ``foss/2018b``                                  
=========    ==============    ================================================


.. _list_software_cDNA_Cupcake_1105:

*cDNA_Cupcake*
++++++++++++++

cDNA_Cupcake is a miscellaneous collection of Python and R scripts used for analyzing sequencing data.

*homepage*: https://github.com/Magdoll/cDNA_Cupcake

=======    ==================    ===============
version    versionsuffix         toolchain      
=======    ==================    ===============
``5.8``    ``-Python-2.7.14``    ``intel/2018a``
=======    ==================    ===============


.. _list_software_CDO_214:

*CDO*
+++++

CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.

*homepage*: https://code.zmaw.de/projects/cdo

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.7.2``    ``intel/2016b``                                  
``1.8.1``    ``intel/2017a``                                  
``1.9.1``    ``intel/2017b``                                  
``1.9.2``    ``intel/2017b``                                  
``1.9.5``    ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
=========    =================================================


.. _list_software_cdsapi_628:

*cdsapi*
++++++++

Climate Data Store API

*homepage*: https://pypi.org/project/cdsapi

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.1.4``                         ``foss/2019a``
``0.1.4``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_CellMix_686:

*CellMix*
+++++++++

A Comprehensive Toolbox for Gene Expression Deconvolution

*homepage*: http://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``1.6.2``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_CellRanger_991:

*CellRanger*
++++++++++++

Cell Ranger is a set of analysis pipelines that process Chromium  single-cell RNA-seq output to align reads, generate gene-cell matrices and perform  clustering and gene expression analysis.

*homepage*: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger

=========    ==========
version      toolchain 
=========    ==========
``3.0.0``    ``system``
``3.0.2``    ``system``
``3.1.0``    ``system``
=========    ==========


.. _list_software_Centrifuge_1036:

*Centrifuge*
++++++++++++

Classifier for metagenomic sequences

*homepage*: https://ccb.jhu.edu/software/centrifuge/

==============    ==============
version           toolchain     
==============    ==============
``1.0.3``         ``foss/2018b``
``1.0.4-beta``    ``foss/2018b``
==============    ==============


.. _list_software_CESM-deps_769:

*CESM-deps*
+++++++++++

CESM is a fully-coupled, community, global climate model that provides state-of-the-art  computer simulations of the Earth's past, present, and future climate states.

*homepage*: http://www.cesm.ucar.edu/models/cesm2/

=======    ================================================
version    toolchain                                       
=======    ================================================
``2``      ``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
=======    ================================================


.. _list_software_CFITSIO_529:

*CFITSIO*
+++++++++

CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format.

*homepage*: https://heasarc.gsfc.nasa.gov/fitsio/

========    =====================================================
version     toolchain                                            
========    =====================================================
``3.38``    ``foss/2016a``, ``intel/2016a``                      
``3.41``    ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``intel/2016b``
``3.42``    ``GCCcore/6.4.0``, ``intel/2017b``                   
``3.45``    ``GCCcore/7.3.0``, ``intel/2018b``                   
``3.47``    ``GCCcore/8.2.0``                                    
========    =====================================================


.. _list_software_cftime_632:

*cftime*
++++++++

Time-handling functionality from netcdf4-python

*homepage*: https://github.com/Unidata/cftime

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``1.0.0``      ``-Python-3.6.4``     ``intel/2018a``                
``1.0.0b1``    ``-Python-3.6.2``     ``foss/2017b``                 
``1.0.1``      ``-Python-2.7.15``    ``intel/2018b``                
``1.0.1``      ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
===========    ==================    ===============================


.. _list_software_CGAL_279:

*CGAL*
++++++

The goal of the CGAL Open Source Project is to provide easy access to efficient  and reliable geometric algorithms in the form of a C++ library.

*homepage*: http://www.cgal.org/

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``4.8``       ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``4.8.1``                           ``foss/2016b``, ``intel/2016b``
``4.8.1``     ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``4.9``       ``-Python-2.7.12``    ``intel/2016b``                
``4.9``       ``-Python-2.7.13``    ``intel/2017a``                
``4.11``      ``-Python-2.7.13``    ``intel/2017a``                
``4.11``      ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``4.11``      ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``4.11.1``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``
``4.11.1``    ``-Python-2.7.15``    ``foss/2018b``                 
``4.11.1``    ``-Python-3.6.4``     ``foss/2018a``                 
``4.11.1``    ``-Python-3.6.6``     ``foss/2018b``                 
``4.14``      ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``
``4.14.1``    ``-Python-3.7.4``     ``foss/2019b``                 
==========    ==================    ===============================


.. _list_software_cget_419:

*cget*
++++++

Cmake package retrieval. This can be used to download and install cmake packages

*homepage*: https://cget.readthedocs.io/en/latest/index.html

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.1.6``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_CGNS_299:

*CGNS*
++++++

The CGNS system is designed to facilitate the exchange of data between sites and applications,  and to help stabilize the archiving of aerodynamic data.

*homepage*: https://cgns.github.io/

=========    ==============
version      toolchain     
=========    ==============
``3.3.1``    ``foss/2016b``
=========    ==============


.. _list_software_CharLS_541:

*CharLS*
++++++++

CharLS is a C++ implementation of the JPEG-LS standard for lossless and near-lossless image compression and decompression. JPEG-LS is a low-complexity image compression standard that matches JPEG 2000 compression ratios.

*homepage*: https://github.com/team-charls/charls

=========    ====================================
version      toolchain                           
=========    ====================================
``2.0.0``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
=========    ====================================


.. _list_software_charmm_632:

*charmm*
++++++++

CHARMM (Chemistry at HARvard Macromolecular Mechanics) is a versatile and widely used molecular simulation program with broad application to many-particle systems. charmm provides all the functionality of CHARMM except its performance enhancements.

*homepage*: http://www.charmm.org

========    ===============================
version     toolchain                      
========    ===============================
``43b2``    ``foss/2017b``, ``intel/2017b``
========    ===============================


.. _list_software_CHARMM_440:

*CHARMM*
++++++++

CHARMM (Chemistry at HARvard Macromolecular Mechanics) is a versatile and widely used molecular simulation program with broad application to many-particle systems.

*homepage*: http://www.charmm.org

========    ===============================
version     toolchain                      
========    ===============================
``37b2``    ``foss/2016a``, ``intel/2016a``
========    ===============================


.. _list_software_CHASE_356:

*CHASE*
+++++++

Case-control HAplotype Sharing analyses.   Haplotype sharing analyses for genome-wide association studies.

*homepage*: http://people.duke.edu/~asallen/Software.html

============    ==========
version         toolchain 
============    ==========
``20130626``    ``system``
============    ==========


.. _list_software_CheckM_555:

*CheckM*
++++++++

CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,  single cells, or metagenomes.

*homepage*: https://github.com/Ecogenomics/CheckM

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``1.0.13``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``1.0.13``    ``-Python-2.7.15``    ``foss/2018b``                 
``1.0.13``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``1.0.18``    ``-Python-2.7.15``    ``foss/2019a``                 
==========    ==================    ===============================


.. _list_software_Cheetah_690:

*Cheetah*
+++++++++

Cheetah is an open source template engine and code generation tool.

*homepage*: http://cheetahtemplate.org

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.4.4``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_CheMPS2_562:

*CheMPS2*
+++++++++

CheMPS2 is a scientific library which contains a spin-adapted implementation of the density matrix renormalization group (DMRG) for ab initio quantum chemistry.

*homepage*: https://github.com/SebWouters/CheMPS2

===========    ================================================================
version        toolchain                                                       
===========    ================================================================
``1.6``        ``intel/2016a``                                                 
``1.7.1``      ``intel/2016a``                                                 
``1.7.2``      ``intel/2016a``                                                 
``1.7-rc2``    ``intel/2016a``                                                 
``1.8``        ``intel/2016b``                                                 
``1.8.8``      ``intel/2018b``                                                 
``1.8.9``      ``foss/2018b``, ``foss/2019a``, ``intel/2018b``, ``intel/2019a``
===========    ================================================================


.. _list_software_Chimera_697:

*Chimera*
+++++++++

UCSF Chimera is a highly extensible program for interactive visualization   and analysis of molecular structures and related data, including density maps, supramolecular   assemblies, sequence alignments, docking results, trajectories, and conformational ensembles. 

*homepage*: https://www.cgl.ucsf.edu/chimera/

========    =================    ==========
version     versionsuffix        toolchain 
========    =================    ==========
``1.10``    ``-linux_x86_64``    ``system``
========    =================    ==========


.. _list_software_ChimPipe_783:

*ChimPipe*
++++++++++

ChimPipe is a computational method for the detection of novel transcription-induced  chimeric transcripts and fusion genes from Illumina Paired-End RNA-seq data. It combines junction  spanning and paired-end read information to accurately detect chimeric splice junctions at base-pair  resolution.

*homepage*: https://chimpipe.readthedocs.org/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``0.9.5``                          ``foss/2018b``
``0.9.5``    ``-Python-2.7.12``    ``foss/2016b``
=========    ==================    ==============


.. _list_software_Chromaprint_1159:

*Chromaprint*
+++++++++++++

Chromaprint is the core component of the AcoustID project. It's a client-side library  that implements a custom algorithm for extracting fingerprints from any audio source.

*homepage*: https://acoustid.org/chromaprint

=========    =================
version      toolchain        
=========    =================
``1.4.3``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_CIRCexplorer_1170:

*CIRCexplorer*
++++++++++++++

CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.

*homepage*: http://circexplorer2.readthedocs.io/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.1.10``    ``-Python-2.7.14``    ``intel/2017b``
==========    ==================    ===============


.. _list_software_CIRCexplorer2_1220:

*CIRCexplorer2*
+++++++++++++++

CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.

*homepage*: http://circexplorer2.readthedocs.io/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.3.2``    ``-Python-2.7.14``    ``intel/2017b``
``2.3.3``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_Circos_611:

*Circos*
++++++++

Circos is a software package for visualizing data and information.  It visualizes data in a circular layout - this makes Circos ideal for exploring  relationships between objects or positions.

*homepage*: http://www.circos.ca/

==========    ================    =================
version       versionsuffix       toolchain        
==========    ================    =================
``0.69-5``    ``-Perl-5.24.0``    ``foss/2016b``   
``0.69-6``    ``-Perl-5.26.1``    ``GCCcore/6.4.0``
``0.69-6``    ``-Perl-5.28.0``    ``GCCcore/7.3.0``
==========    ================    =================


.. _list_software_CIRI_295:

*CIRI*
++++++

CircRNA Identifier. A de novo circular RNA identification tool

*homepage*: https://sourceforge.net/projects/ciri/

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``2.0.6``    ``-Perl-5.26.0``    ``intel/2017b``
=========    ================    ===============


.. _list_software_cisTEM_549:

*cisTEM*
++++++++

cisTEM is user-friendly software to process cryo-EM images of macromolecular complexes  and obtain high-resolution 3D reconstructions from them. 

*homepage*: https://cistem.org/

==============    ==============
version           toolchain     
==============    ==============
``1.0.0-beta``    ``foss/2018a``
==============    ==============


.. _list_software_CITE-seq-Count_1233:

*CITE-seq-Count*
++++++++++++++++

A python package that allows to count antibody TAGS from a CITE-seq and/or cell hashing experiment.

*homepage*: https://github.com/Hoohm/CITE-seq-Count

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.4.3``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_Clang_485:

*Clang*
+++++++

C, C++, Objective-C compiler, based on LLVM.  Does not  include C++ standard library -- use libstdc++ from GCC.

*homepage*: http://clang.llvm.org/

=========    ==================    ======================================
version      versionsuffix         toolchain                             
=========    ==================    ======================================
``3.3``                            ``GCC/4.8.1``                         
``3.4``                            ``GCC/4.8.2``                         
``3.4.1``                          ``GCC/4.8.2``                         
``3.4.2``                          ``GCC/4.8.2``                         
``3.6.0``                          ``GCC/4.9.2``                         
``3.6.1``                          ``GCC/4.9.2``                         
``3.7.0``                          ``GNU/4.9.3-2.25``                    
``3.7.1``                          ``GCC/4.9.3-2.25``, ``foss/2016a``    
``3.8.0``                          ``GCC/4.9.3-2.25``                    
``3.8.1``                          ``GCC/5.4.0-2.26``, ``foss/2016b``    
``5.0.0``                          ``GCC/6.4.0-2.28``                    
``5.0.1``                          ``GCC/6.4.0-2.28``                    
``6.0.1``                          ``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``
``7.0.0``                          ``GCC/6.4.0-2.28``                    
``7.0.1``                          ``GCC/7.3.0-2.30``                    
``8.0.0``                          ``GCCcore/8.2.0``                     
``8.0.0``    ``-CUDA-10.1.105``    ``GCCcore/8.2.0``                     
=========    ==================    ======================================


.. _list_software_Clang-Python-bindings_2063:

*Clang-Python-bindings*
+++++++++++++++++++++++

Python bindings for libclang

*homepage*: https://clang.llvm.org

=========    ==================    =================
version      versionsuffix         toolchain        
=========    ==================    =================
``8.0.0``    ``-Python-2.7.15``    ``GCCcore/8.2.0``
=========    ==================    =================


.. _list_software_CLAPACK_495:

*CLAPACK*
+++++++++

C version of LAPACK

*homepage*: http://www.netlib.org/clapack

=========    ===========================================================================
version      toolchain                                                                  
=========    ===========================================================================
``3.2.1``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017a``
=========    ===========================================================================


.. _list_software_CLHEP_364:

*CLHEP*
+++++++

The CLHEP project is intended to be a set of HEP-specific foundation and  utility classes such as random generators, physics vectors, geometry and linear algebra.  CLHEP is structured in a set of packages independent of any external package.

*homepage*: http://proj-clhep.web.cern.ch/proj-clhep/

===========    ================================================================
version        toolchain                                                       
===========    ================================================================
``2.1.1.0``    ``intel/2016a``                                                 
``2.1.3.1``    ``intel/2016a``                                                 
``2.2.0.8``    ``intel/2016a``                                                 
``2.3.1.1``    ``intel/2016a``                                                 
``2.3.4.3``    ``foss/2017b``, ``intel/2017b``                                 
``2.4.0.0``    ``intel/2017b``                                                 
``2.4.1.0``    ``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018b``
===========    ================================================================


.. _list_software_CLISP_379:

*CLISP*
+++++++

Common Lisp is a high-level, general-purpose, object-oriented, dynamic,  functional programming language. 

*homepage*: http://www.clisp.org/

========    =================
version     toolchain        
========    =================
``2.49``    ``GCCcore/6.4.0``
========    =================


.. _list_software_ClonalFrameML_1245:

*ClonalFrameML*
+++++++++++++++

Efficient Inference of Recombination in Whole Bacterial Genomes

*homepage*: https://github.com/xavierdidelot/ClonalFrameML

========    ==============
version     toolchain     
========    ==============
``1.11``    ``foss/2016b``
========    ==============


.. _list_software_CLooG_436:

*CLooG*
+++++++

CLooG is a free software and library to generate code for scanning Z-polyhedra. That is, it finds a  code (e.g. in C, FORTRAN...) that reaches each integral point of one or more parameterized polyhedra. CLooG has been  originally written to solve the code generation problem for optimizing compilers based on the polytope model.  Nevertheless it is used now in various area e.g. to build control automata for high-level synthesis or to find the  best polynomial approximation of a function. CLooG may help in any situation where scanning polyhedra matters. While  the user has full control on generated code quality, CLooG is designed to avoid control overhead and to produce a  very effective code.

*homepage*: http://www.bastoul.net/cloog/index.php

==========    =============
version       toolchain    
==========    =============
``0.18.1``    ``GCC/4.8.2``
==========    =============


.. _list_software_Clustal-Omega_1262:

*Clustal-Omega*
+++++++++++++++

Clustal Omega is a multiple sequence alignment  program for proteins. It produces biologically meaningful multiple  sequence alignments of divergent sequences. Evolutionary relationships  can be seen via viewing Cladograms or Phylograms 

*homepage*: http://www.clustal.org/omega/

=========    ================================================
version      toolchain                                       
=========    ================================================
``1.2.0``    ``foss/2016b``                                  
``1.2.4``    ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
=========    ================================================


.. _list_software_ClustalW2_865:

*ClustalW2*
+++++++++++

ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.

*homepage*: http://www.ebi.ac.uk/Tools/msa/clustalw2/

=======    ==========================================================================================================
version    toolchain                                                                                                 
=======    ==========================================================================================================
``2.1``    ``foss/2016b``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017b``, ``intel/2018b``
=======    ==========================================================================================================


.. _list_software_Cluster-Buster_1412:

*Cluster-Buster*
++++++++++++++++

Cluster-Buster: Finding dense clusters of motifs in DNA sequences

*homepage*: https://github.com/weng-lab/cluster-buster/

============    ===============
version         toolchain      
============    ===============
``20160106``    ``intel/2016a``
============    ===============


.. _list_software_ClusterShell_1242:

*ClusterShell*
++++++++++++++

ClusterShell is an event-driven open source Python library, designed to run local or distant commands  in parallel on server farms or on large Linux clusters.

*homepage*: http://cea-hpc.github.io/clustershell/

=========    ==========
version      toolchain 
=========    ==========
``1.7.3``    ``system``
=========    ==========


.. _list_software_CMake_449:

*CMake*
+++++++

CMake, the cross-platform, open-source build system.  CMake is a family of tools designed to build, test and package software.

*homepage*: http://www.cmake.org

==========    =================================================================================================================================================
version       toolchain                                                                                                                                        
==========    =================================================================================================================================================
``2.8.11``    ``GCC/4.8.1``                                                                                                                                    
``2.8.12``    ``GCC/4.8.1``, ``GCC/4.8.2``                                                                                                                     
``3.0.0``     ``GCC/4.8.3``                                                                                                                                    
``3.1.0``     ``GCC/4.9.2``                                                                                                                                    
``3.1.3``     ``GCC/4.9.2``, ``system``                                                                                                                        
``3.2.1``     ``GCC/4.9.2``, ``GNU/4.9.3-2.25``                                                                                                                
``3.3.1``     ``system``                                                                                                                                       
``3.3.2``     ``GNU/4.9.3-2.25``, ``gimkl/2.11.5``                                                                                                             
``3.4.1``     ``GCC/4.9.2``, ``GCCcore/4.9.3``, ``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``3.4.3``     ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``                                                                                
``3.5.1``     ``intel/2016a``                                                                                                                                  
``3.5.2``     ``GCC/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``system``                                                 
``3.6.1``     ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``foss/2016b``, ``intel/2016b``, ``system``                                                               
``3.6.2``     ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``                                                                                               
``3.7.1``     ``GCCcore/5.4.0``, ``GCCcore/6.2.0``, ``foss/2016b``, ``intel/2016b``                                                                            
``3.7.2``     ``GCCcore/6.3.0``, ``foss/2016b``, ``intel/2016b``                                                                                               
``3.8.0``     ``GCCcore/6.3.0``                                                                                                                                
``3.8.1``     ``GCCcore/6.3.0``                                                                                                                                
``3.8.2``     ``GCCcore/6.3.0``                                                                                                                                
``3.9.1``     ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``system``                                                                                                 
``3.9.4``     ``GCCcore/6.4.0``                                                                                                                                
``3.9.5``     ``GCCcore/6.4.0``                                                                                                                                
``3.9.6``     ``system``                                                                                                                                       
``3.10.0``    ``GCCcore/6.4.0``                                                                                                                                
``3.10.1``    ``GCCcore/6.4.0``                                                                                                                                
``3.10.2``    ``GCCcore/6.4.0``, ``GCCcore/7.2.0``                                                                                                             
``3.10.3``    ``GCCcore/6.4.0``, ``GCCcore/7.2.0``                                                                                                             
``3.11.1``    ``GCCcore/6.4.0``                                                                                                                                
``3.11.4``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                                                                                                             
``3.12.1``    ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``system``                                                                              
``3.13.3``    ``GCCcore/8.2.0``                                                                                                                                
``3.15.3``    ``GCCcore/8.3.0``                                                                                                                                
==========    =================================================================================================================================================


.. _list_software_CNVkit_559:

*CNVkit*
++++++++

A command-line toolkit and Python library for detecting copy  number variants and alterations genome-wide from high-throughput sequencing.

*homepage*: http://github.com/etal/cnvkit

=========    =========================    ===============
version      versionsuffix                toolchain      
=========    =========================    ===============
``0.9.2``    ``-Python-2.7.14``           ``intel/2017b``
``0.9.6``    ``-Python-3.7.2-R-3.6.0``    ``foss/2019a`` 
=========    =========================    ===============


.. _list_software_CNVnator_779:

*CNVnator*
++++++++++

a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads 

*homepage*: https://github.com/abyzovlab/CNVnator

=========    ==============
version      toolchain     
=========    ==============
``0.3.3``    ``foss/2016b``
=========    ==============


.. _list_software_CODEX2_421:

*CODEX2*
++++++++

Full-spectrum copy number variation detection by high-throughput DNA sequencing

*homepage*: https://github.com/yuchaojiang/CODEX2

============    =============    ===============
version         versionsuffix    toolchain      
============    =============    ===============
``20180227``    ``-R-3.4.3``     ``intel/2017b``
============    =============    ===============


.. _list_software_Commet_613:

*Commet*
++++++++

COMMET ("COmpare Multiple METagenomes") provides a global similarity overview between all datasets of  a large metagenomic project. 

*homepage*: https://colibread.inria.fr/software/commet/

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20150415``    ``-Python-2.7.11``    ``foss/2016a``
============    ==================    ==============


.. _list_software_Con3F_409:

*Con3F*
+++++++

Con3F is a Python package to read, manipulate and convert force field files

*homepage*: https://github.ugent.be/CMM/con3f

================    =================    ===============
version             versionsuffix        toolchain      
================    =================    ===============
``1.0-20190329``    ``-Python-3.7.2``    ``intel/2019a``
================    =================    ===============


.. _list_software_CONCOCT_521:

*CONCOCT*
+++++++++

Clustering cONtigs with COverage and ComposiTion (CONCOCT) is a program for unsupervised binning  of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data  from paired end reads.

*homepage*: https://concoct.readthedocs.io

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.0.0``    ``-Python-2.7.14``    ``foss/2017b``
``1.0.0``    ``-Python-2.7.15``    ``foss/2018b``
``1.0.0``    ``-Python-3.6.3``     ``foss/2017b``
``1.1.0``    ``-Python-2.7.15``    ``foss/2019a``
=========    ==================    ==============


.. _list_software_configparser_1283:

*configparser*
++++++++++++++

configparser is a Python library that brings the updated configparser from Python 3.5 to Python 2.6-3.5

*homepage*: http://docs.python.org/3/library/configparser.html

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``3.5.0``    ``-Python-2.7.11``    ``foss/2016a``                 
``3.5.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``3.5.0``    ``-Python-3.5.2``     ``intel/2016b``                
``3.5.0``    ``-Python-3.6.3``     ``intel/2017b``                
=========    ==================    ===============================


.. _list_software_configurable-http-proxy_2381:

*configurable-http-proxy*
+++++++++++++++++++++++++

HTTP proxy for node.js including a REST API for updating the routing table.  Developed as a part of the Jupyter Hub multi-user server.

*homepage*: https://github.com/jupyterhub/configurable-http-proxy

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.3.0``    ``-nodejs-4.4.7``    ``foss/2016a``
``3.1.1``    ``-nodejs-8.9.4``    ``foss/2017a``
=========    =================    ==============


.. _list_software_ConnectomeWorkbench_1966:

*ConnectomeWorkbench*
+++++++++++++++++++++

Connectome Workbench is an open source, freely available visualization  and discovery tool used to map neuroimaging data, especially data generated by the  Human Connectome Project.

*homepage*: https://www.humanconnectome.org/software/connectome-workbench

=========    ===============================
version      toolchain                      
=========    ===============================
``1.2.2``    ``system``                     
``1.3.2``    ``foss/2017b``, ``intel/2017b``
=========    ===============================


.. _list_software_Control-FREEC_1139:

*Control-FREEC*
+++++++++++++++

Copy number and genotype annotation from whole  genome and whole exome sequencing data.

*homepage*: https://github.com/BoevaLab/FREEC

========    ====================
version     toolchain           
========    ====================
``11.5``    ``GCC/8.2.0-2.31.1``
========    ====================


.. _list_software_CoordgenLibs_1211:

*CoordgenLibs*
++++++++++++++

Schrodinger-developed 2D Coordinate Generation

*homepage*: https://github.com/schrodinger/coordgenlibs

=========    ================================
version      toolchain                       
=========    ================================
``1.3.2``    ``gompi/2019a``, ``iimpi/2019a``
=========    ================================


.. _list_software_Coot_405:

*Coot*
++++++

Coot is for macromolecular model building, model completion  and validation, particularly suitable for protein modelling using X-ray data.

*homepage*: http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot

=========    ===========================================    ==========
version      versionsuffix                                  toolchain 
=========    ===========================================    ==========
``0.8.1``    ``-binary-Linux-x86_64-rhel-6-python-gtk2``    ``system``
=========    ===========================================    ==========


.. _list_software_core-counter_1238:

*core-counter*
++++++++++++++

Tool to check available cores and OMP threads

*homepage*: https://github.com/gjbex/core-counter

=========    ==========
version      toolchain 
=========    ==========
``1.1.1``    ``system``
=========    ==========


.. _list_software_Coreutils_954:

*Coreutils*
+++++++++++

The GNU Core Utilities are the basic file, shell and text manipulation utilities of the  GNU operating system.  These are the core utilities which are expected to exist on every operating system. 

*homepage*: http://www.gnu.org/software/coreutils/

========    =================
version     toolchain        
========    =================
``8.23``    ``GCC/4.9.2``    
``8.27``    ``GCCcore/5.4.0``
``8.29``    ``GCCcore/6.4.0``
========    =================


.. _list_software_corner_649:

*corner*
++++++++

Make some beautiful corner plots.

*homepage*: https://corner.readthedocs.io/en/latest/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.0.1``    ``-Python-2.7.15``    ``foss/2019a``
``2.0.1``    ``-Python-3.7.2``     ``foss/2019a``
=========    ==================    ==============


.. _list_software_coverage_844:

*coverage*
++++++++++

Coverage.py is a tool for measuring code coverage of Python programs.  It monitors your program, noting which parts of the code have been executed,  then analyzes the source to identify code that could have been executed but was not. 

*homepage*: https://coverage.readthedocs.io

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``4.5.1``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_cowsay_662:

*cowsay*
++++++++

Configurable talking characters in ASCII art

*homepage*: https://github.com/tnalpgge/rank-amateur-cowsay

========    ==========
version     toolchain 
========    ==========
``3.04``    ``system``
========    ==========


.. _list_software_CP2K_272:

*CP2K*
++++++

CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular  simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different  methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and  classical pair and many-body potentials. 

*homepage*: http://www.cp2k.org/

=======    =============    ==================================================================
version    versionsuffix    toolchain                                                         
=======    =============    ==================================================================
``3.0``                     ``intel/2016a``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``
``3.0``    ``-psmp``        ``intel/2016b``                                                   
``4.1``                     ``intel/2016b``                                                   
``4.1``    ``-psmp``        ``foss/2016b``                                                    
``5.1``                     ``foss/2018a``, ``intel/2017b``, ``intel/2018a``                  
``6.1``                     ``foss/2019a``, ``intel/2018a``                                   
=======    =============    ==================================================================


.. _list_software_CPB_213:

*CPB*
+++++

CPB is a novel two-step Pearson correlation based biclustering approach to mine genes that are co-regulated with   a given reference gene in order to discover genes that function in a common biological process.   In the first step, the algorithm identifies subsets of genes with high correlation, reducing false negatives   with a nonparametric filtering scheme.   In the second step, biclusters from multiple datasets are used to extract and rank gene correlation information.

*homepage*: http://tda.gatech.edu/software/cpb/index.html

=============    ==================    ==============
version          versionsuffix         toolchain     
=============    ==================    ==============
``11-4-2011``    ``-Python-2.7.13``    ``foss/2017a``
=============    ==================    ==============


.. _list_software_CPLEX_380:

*CPLEX*
+++++++

IBM ILOG CPLEX Optimizer's mathematical programming technology enables  analytical decision support for improving efficiency,  reducing costs, and increasing profitability.

*homepage*: https://www.ibm.com/analytics/cplex-optimizer

========    =================
version     toolchain        
========    =================
``12.9``    ``GCCcore/8.2.0``
========    =================


.. _list_software_CppUnit_707:

*CppUnit*
+++++++++

CppUnit is the C++ port of the famous JUnit framework for unit testing.

*homepage*: http://sourceforge.net/projects/cppunit/

==========    ====================================================
version       toolchain                                           
==========    ====================================================
``1.12.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``
==========    ====================================================


.. _list_software_cram_419:

*cram*
++++++

Cram is a functional testing framework for command line applications.

*homepage*: https://bitheap.org/cram

=======    =================
version    toolchain        
=======    =================
``0.7``    ``GCCcore/8.2.0``
=======    =================


.. _list_software_cramtools_980:

*cramtools*
+++++++++++

CRAMTools is a set of Java tools and APIs for efficient compression of sequence  read data. Although this is intended as a stable version the code is released as early access. Parts of the CRAMTools are experimental and may not be supported in the future.

*homepage*: https://github.com/enasequence/cramtools/

=======    ==================    ==========
version    versionsuffix         toolchain 
=======    ==================    ==========
``2.0``    ``-Java-1.7.0_80``    ``system``
``3.0``    ``-Java-1.7.0_80``    ``system``
=======    ==================    ==========


.. _list_software_CrayCCE_602:

*CrayCCE*
+++++++++

Toolchain using Cray compiler wrapper, using PrgEnv-cray (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

=========    ==========
version      toolchain 
=========    ==========
``19.06``    ``system``
=========    ==========


.. _list_software_CrayGNU_633:

*CrayGNU*
+++++++++

Toolchain using Cray compiler wrapper, using PrgEnv-gnu module (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

=========    ==========
version      toolchain 
=========    ==========
``19.06``    ``system``
=========    ==========


.. _list_software_CrayIntel_907:

*CrayIntel*
+++++++++++

Toolchain using Cray compiler wrapper, using PrgEnv-intel (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

=========    ==========
version      toolchain 
=========    ==========
``19.06``    ``system``
=========    ==========


.. _list_software_CrayPGI_623:

*CrayPGI*
+++++++++

Toolchain using Cray compiler wrapper, PrgEnv-pgi compiler (PE release: June 2019). 

*homepage*: https://pubs.cray.com/discover

=========    ==========
version      toolchain 
=========    ==========
``19.06``    ``system``
=========    ==========


.. _list_software_crb-blast_890:

*crb-blast*
+++++++++++

Conditional Reciprocal Best BLAST - high confidence ortholog assignment.  CRB-BLAST is a novel method for finding orthologs between one set of sequences and another.  This is particularly useful in genome and transcriptome annotation.

*homepage*: https://github.com/cboursnell/crb-blast

=========    ===============
version      toolchain      
=========    ===============
``0.6.9``    ``intel/2017a``
=========    ===============


.. _list_software_CRF++_305:

*CRF++*
+++++++

CRF++ is a simple, customizable, and open source implementation of  Conditional Random Fields (CRFs) for segmenting/labeling sequential data. CRF++ is  designed for generic purpose and will be applied to a variety of NLP tasks, such as  Named Entity Recognition, Information Extraction and Text Chunking. 

*homepage*: https://taku910.github.io/crfpp/

========    =========================================================
version     toolchain                                                
========    =========================================================
``0.58``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
========    =========================================================


.. _list_software_CrossTalkZ_1008:

*CrossTalkZ*
++++++++++++

CrossTalkZ is a statistical method and software to assess the significance of crosstalk enrichment  between pairs of gene or protein groups in large biological networks. 

*homepage*: http://sonnhammer.sbc.su.se/download/software/CrossTalkZ/

=======    ==============
version    toolchain     
=======    ==============
``1.4``    ``foss/2016a``
=======    ==============


.. _list_software_CryptoMiniSat_1334:

*CryptoMiniSat*
+++++++++++++++

CryptoMiniSat is an advanced SAT solver

*homepage*: https://github.com/msoos/cryptominisat

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``5.0.1``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_cscope_637:

*cscope*
++++++++

Cscope is a developer's tool for browsing source code.

*homepage*: http://cscope.sourceforge.net/

========    ==========
version     toolchain 
========    ==========
``15.9``    ``system``
========    ==========


.. _list_software_csvkit_660:

*csvkit*
++++++++

csvkit is a suite of command-line tools for converting to and working with CSV,  the king of tabular file formats.

*homepage*: https://github.com/wireservice/csvkit

=========    =================
version      toolchain        
=========    =================
``1.0.4``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_ctags_530:

*ctags*
+++++++

Ctags generates an index (or tag) file of language objects found in source files that allows these  items to be quickly and easily located by a text editor or other utility.

*homepage*: http://ctags.sourceforge.net/

=======    ==========
version    toolchain 
=======    ==========
``5.8``    ``system``
=======    ==========


.. _list_software_ctffind_734:

*ctffind*
+++++++++

Program for finding CTFs of electron micrographs.

*homepage*: http://grigoriefflab.janelia.org/ctffind4

==========    ==================================
version       toolchain                         
==========    ==================================
``4.1.13``    ``foss/2019a``, ``fosscuda/2019a``
==========    ==================================


.. _list_software_Cube_383:

*Cube*
++++++

Cube, which is used as performance report explorer for Scalasca and  Score-P, is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and (iii) system  resource. Each dimension can be represented as a tree, where non-leaf nodes of the tree  can be collapsed or expanded to achieve the desired level of granularity.

*homepage*: http://www.scalasca.org/software/cube-4.x/download.html

=========    ==============
version      toolchain     
=========    ==============
``4.3.4``    ``foss/2016a``
=========    ==============


.. _list_software_CubeGUI_612:

*CubeGUI*
+++++++++

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube graphical report explorer. 

*homepage*: https://www.scalasca.org/software/cube-4.x/download.html

=========    =================
version      toolchain        
=========    =================
``4.4.4``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_CubeLib_662:

*CubeLib*
+++++++++

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube general purpose C++ library component and  command-line tools. 

*homepage*: https://www.scalasca.org/software/cube-4.x/download.html

=========    =================
version      toolchain        
=========    =================
``4.4.4``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_CubeWriter_1020:

*CubeWriter*
++++++++++++

Cube, which is used as performance report explorer for Scalasca and Score-P,  is a generic tool for displaying a multi-dimensional performance space  consisting of the dimensions (i) performance metric, (ii) call path, and  (iii) system resource. Each dimension can be represented as a tree, where  non-leaf nodes of the tree can be collapsed or expanded to achieve the  desired level of granularity.   This module provides the Cube high-performance C writer library component. 

*homepage*: https://www.scalasca.org/software/cube-4.x/download.html

=========    =================
version      toolchain        
=========    =================
``4.4.3``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_CUDA_285:

*CUDA*
++++++

CUDA (formerly Compute Unified Device Architecture) is a parallel  computing platform and programming model created by NVIDIA and implemented by the  graphics processing units (GPUs) that they produce. CUDA gives developers access  to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs.

*homepage*: https://developer.nvidia.com/cuda-toolkit

=================    ==========================================================================
version              toolchain                                                                 
=================    ==========================================================================
``5.5.22``           ``GCC/4.8.2``, ``system``                                                 
``6.0.37``           ``system``                                                                
``6.5.14``           ``system``                                                                
``7.0.28``           ``system``                                                                
``7.5.18``           ``GCC/4.9.4-2.25``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``system``    
``8.0.44``           ``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``system``    
``8.0.61``           ``system``                                                                
``8.0.61_375.26``    ``GCC/5.4.0-2.26``                                                        
``9.0.176``          ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``system``    
``9.1.85``           ``GCC/6.4.0-2.28``, ``system``                                            
``9.2.88``           ``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``, ``system``                        
``9.2.148.1``        ``system``                                                                
``10.0.130``         ``system``                                                                
``10.1.105``         ``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``system``
``10.1.168``         ``system``                                                                
``10.1.243``         ``GCC/8.3.0``, ``iccifort/2019.5.281``, ``system``                        
``10.2.89``          ``GCC/8.3.0``                                                             
=================    ==========================================================================


.. _list_software_cuDNN_440:

*cuDNN*
+++++++

The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.

*homepage*: https://developer.nvidia.com/cudnn

============    ==================    ===========================================================
version         versionsuffix         toolchain                                                  
============    ==================    ===========================================================
``4.0``                               ``system``                                                 
``5.0``         ``-CUDA-7.5.18``      ``system``                                                 
``5.0-rc``                            ``system``                                                 
``5.1``         ``-CUDA-8.0.44``      ``system``                                                 
``6.0``         ``-CUDA-8.0.61``      ``system``                                                 
``7.0.2``       ``-CUDA-9.0.176``     ``system``                                                 
``7.0.5``       ``-CUDA-9.1.85``      ``system``                                                 
``7.0.5.15``                          ``fosscuda/2017b``, ``fosscuda/2018a``, ``intelcuda/2017b``
``7.1.4.18``                          ``fosscuda/2018b``                                         
``7.4.2.24``                          ``gcccuda/2019a``                                          
``7.4.2.24``    ``-CUDA-10.0.130``    ``system``                                                 
``7.6.2.24``    ``-CUDA-10.1.243``    ``system``                                                 
``7.6.4.38``                          ``gcccuda/2019a``, ``gcccuda/2019b``                       
============    ==================    ===========================================================


.. _list_software_Cufflinks_933:

*Cufflinks*
+++++++++++

Transcript assembly, differential expression, and differential regulation for RNA-Seq

*homepage*: http://cole-trapnell-lab.github.io/cufflinks/

============    ================================================================================
version         toolchain                                                                       
============    ================================================================================
``2.2.1``       ``foss/2016a``, ``foss/2016b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``20190706``    ``gompi/2019a``                                                                 
============    ================================================================================


.. _list_software_CUnit_483:

*CUnit*
+++++++

Automated testing framework for C.

*homepage*: https://sourceforge.net/projects/cunit/

=========    =================
version      toolchain        
=========    =================
``2.1-3``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_cURL_342:

*cURL*
++++++

libcurl is a free and easy-to-use client-side URL transfer library,  supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,  LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.  libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP  form based upload, proxies, cookies, user+password authentication (Basic,  Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling  and more. 

*homepage*: https://curl.haxx.se

==========    ===================================================================================
version       toolchain                                                                          
==========    ===================================================================================
``7.33.0``    ``GCC/4.8.2``                                                                      
``7.34.0``    ``GCC/4.8.2``                                                                      
``7.40.0``    ``GCC/4.9.2``                                                                      
``7.46.0``    ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``                                
``7.47.0``    ``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``                         
``7.49.1``    ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``7.52.1``    ``gimkl/2017a``, ``intel/2016b``                                                   
``7.53.1``    ``GCCcore/6.3.0``                                                                  
``7.54.0``    ``GCCcore/6.3.0``                                                                  
``7.55.1``    ``GCCcore/6.4.0``                                                                  
``7.56.0``    ``GCCcore/6.4.0``                                                                  
``7.56.1``    ``GCCcore/6.4.0``                                                                  
``7.58.0``    ``GCCcore/6.4.0``                                                                  
``7.59.0``    ``GCCcore/6.4.0``                                                                  
``7.60.0``    ``GCCcore/7.2.0``, ``GCCcore/7.3.0``                                               
``7.63.0``    ``GCCcore/8.2.0``                                                                  
``7.66.0``    ``GCCcore/8.3.0``                                                                  
==========    ===================================================================================


.. _list_software_custodian_970:

*custodian*
+++++++++++

A simple JIT job management framework in Python.

*homepage*: https://pypi.python.org/pypi/custodian

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.1.0``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_cutadapt_854:

*cutadapt*
++++++++++

Cutadapt finds and removes adapter sequences, primers, poly-A tails and  other types of unwanted sequence from your high-throughput sequencing reads.

*homepage*: http://opensource.scilifelab.se/projects/cutadapt/

=========    ==================    ================================================
version      versionsuffix         toolchain                                       
=========    ==================    ================================================
``1.9.1``    ``-Python-2.7.11``    ``foss/2016a``                                  
``1.9.1``    ``-Python-2.7.12``    ``foss/2016b``                                  
``1.14``     ``-Python-2.7.13``    ``foss/2017a``, ``intel/2017a``                 
``1.15``     ``-Python-3.5.2``     ``foss/2016b``                                  
``1.16``     ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``1.16``     ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``                 
``1.16``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                 
``1.18``                           ``GCCcore/8.2.0``                               
``1.18``     ``-Python-2.7.15``    ``foss/2018b``                                  
``1.18``     ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                 
``2.1``      ``-Python-3.6.6``     ``foss/2018b``                                  
=========    ==================    ================================================


.. _list_software_CVS_236:

*CVS*
+++++

CVS is a version control system, an important component of  Source Configuration Management (SCM).

*homepage*: https://savannah.nongnu.org/projects/cvs

===========    ===================================================
version        toolchain                                          
===========    ===================================================
``1.11.23``    ``GCC/4.8.2``, ``GCCcore/4.9.3``, ``GCCcore/6.4.0``
===========    ===================================================


.. _list_software_CVXOPT_484:

*CVXOPT*
++++++++

CVXOPT is a free software package for convex optimization based on the Python programming language.  Its main purpose is to make the development of software for convex optimization applications straightforward by  building on Python's extensive standard library and on the strengths of Python as a high-level programming language. 

*homepage*: http://cvxopt.org

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.1.9``    ``-Python-2.7.13``    ``intel/2017a``
``1.2.1``    ``-Python-3.6.4``     ``intel/2018a``
``1.2.3``                          ``foss/2019a`` 
=========    ==================    ===============


.. _list_software_CVXPY_410:

*CVXPY*
+++++++

CVXPY is a Python-embedded modeling language for convex optimization problems.  It allows you to express your problem in a natural way that follows the math,  rather than in the restrictive standard form required by solvers. 

*homepage*: https://www.cvxpy.org/

==========    ==============
version       toolchain     
==========    ==============
``1.0.24``    ``foss/2019a``
==========    ==============


.. _list_software_cysignals_973:

*cysignals*
+++++++++++

The cysignals package provides mechanisms to handle  interrupts (and other signals and errors) in Cython code.

*homepage*: https://pypi.org/project/cysignals/

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``1.10.2``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``1.10.2``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
==========    ==================    ===============================


.. _list_software_Cython_629:

*Cython*
++++++++

The Cython language makes writing C extensions for the Python language as easy as Python itself. Cython is a source code translator based on the well-known Pyrex,  but supports more cutting edge functionality and optimizations.

*homepage*: https://pypi.python.org/pypi/Cython/

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``0.23.4``     ``-Python-2.7.10``    ``gimkl/2.11.5``               
``0.24.1``     ``-Python-2.7.11``    ``foss/2016a``                 
``0.25.2``     ``-Python-2.7.12``    ``foss/2016b``                 
``0.25.2``     ``-Python-3.6.4``     ``intel/2018a``                
``0.27.3``     ``-Python-2.7.15``    ``GCCcore/8.2.0``              
``0.29.10``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``0.29.10``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
===========    ==================    ===============================


.. _list_software_cytosim_776:

*cytosim*
+++++++++

Cytosim is a cytoskeleton simulation engine written in  C++ working on Mac OS, GNU/Linux and Windows (with Cygwin).

*homepage*: https://github.com/nedelec/cytosim

============    =============    ===============
version         versionsuffix    toolchain      
============    =============    ===============
``20190117``    ``-mkl``         ``gomkl/2019a``
============    =============    ===============


.. _list_software_cyvcf2_589:

*cyvcf2*
++++++++

cython + htslib == fast VCF and BCF processing

*homepage*: https://github.com/brentp/cyvcf2

===========    =================    ===============================
version        versionsuffix        toolchain                      
===========    =================    ===============================
``0.10.10``    ``-Python-3.6.6``    ``foss/2018b``                 
``0.11.5``                          ``foss/2019a``, ``intel/2019a``
===========    =================    ===============================


.. _list_software_letter_d:

*D*
---


:ref:`list_software_dadi_402` - :ref:`list_software_dagitty_758` - :ref:`list_software_damageproto_1171` - :ref:`list_software_dammit_636` - :ref:`list_software_DANPOS2_503` - :ref:`list_software_DAS_Tool_725` - :ref:`list_software_dask_419` - :ref:`list_software_datamash_835` - :ref:`list_software_davix_540` - :ref:`list_software_DB_134` - :ref:`list_software_DB_File_613` - :ref:`list_software_DBD-mysql_813` - :ref:`list_software_DBG2OLC_477` - :ref:`list_software_DBus_366` - :ref:`list_software_dbus-glib_889` - :ref:`list_software_dcm2niix_798` - :ref:`list_software_DCMTK_371` - :ref:`list_software_deal.II_598` - :ref:`list_software_deap_410` - :ref:`list_software_deconf_623` - :ref:`list_software_DeconICA_694` - :ref:`list_software_deepdiff_823` - :ref:`list_software_DeepSurv_814` - :ref:`list_software_deepTools_943` - :ref:`list_software_Delly_506` - :ref:`list_software_DeMixT_555` - :ref:`list_software_DendroPy_805` - :ref:`list_software_destiny_768` - :ref:`list_software_detonate_852` - :ref:`list_software_DFT-D3_386` - :ref:`list_software_DFTB+_331` - :ref:`list_software_DIAL_282` - :ref:`list_software_DIAMOND_508` - :ref:`list_software_DicomBrowser_1232` - :ref:`list_software_Diffutils_938` - :ref:`list_software_dill_421` - :ref:`list_software_distributed_1187` - :ref:`list_software_DL_POLY_Classic_1364` - :ref:`list_software_DMTCP_376` - :ref:`list_software_DOLFIN_444` - :ref:`list_software_Doris_513` - :ref:`list_software_dotNET-Core-Runtime_1781` - :ref:`list_software_double-conversion_1774` - :ref:`list_software_Doxygen_734` - :ref:`list_software_Drake_487` - :ref:`list_software_dropEst_737` - :ref:`list_software_DSA_216` - :ref:`list_software_DSRC_300` - :ref:`list_software_Dsuite_622` - :ref:`list_software_dtcmp_536` - :ref:`list_software_dxpy_453`


.. _list_software_dadi_402:

*dadi*
++++++

∂a∂i implements methods for demographic history and selection inference from genetic data,  based on diffusion approximations to the allele frequency spectrum.

*homepage*: https://bitbucket.org/gutenkunstlab/dadi

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.7.0``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_dagitty_758:

*dagitty*
+++++++++

A port of the web-based software 'DAGitty', available at <http://dagitty.net>, for  analyzing structural causal models (also known as directed acyclic graphs or DAGs). This package  computes covariate adjustment sets for estimating causal effects, enumerates instrumental variables,  derives testable implications (d-separation and vanishing tetrads), generates equivalent models, and  includes a simple facility for data simulation.

*homepage*: https://cran.r-project.org/web/packages/dagitty/

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``0.2-2``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_damageproto_1171:

*damageproto*
+++++++++++++

X protocol and ancillary headers for xinerama

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.2.1``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_dammit_636:

*dammit*
++++++++

dammit is a simple de novo transcriptome annotator. It was born out of  the observations that annotation is mundane and annoying,  all the individual pieces of the process exist already, and the existing  solutions are overly complicated or rely on crappy non-free software.

*homepage*: http://www.camillescott.org/dammit/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.3.2``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_DANPOS2_503:

*DANPOS2*
+++++++++

A toolkit for Dynamic Analysis of Nucleosome and Protein Occupancy by Sequencing, version 2

*homepage*: https://sites.google.com/site/danposdoc

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.2.2``    ``-Python-2.7.12``    ``foss/2016b``
=========    ==================    ==============


.. _list_software_DAS_Tool_725:

*DAS_Tool*
++++++++++

DAS Tool is an automated method that integrates the results of a flexible number of binning  algorithms to calculate an optimized, non-redundant set of bins from a single assembly.

*homepage*: https://github.com/cmks/DAS_Tool

=========    ==========================    ==============
version      versionsuffix                 toolchain     
=========    ==========================    ==============
``1.1.1``    ``-R-3.5.1-Python-2.7.15``    ``foss/2018b``
=========    ==========================    ==============


.. _list_software_dask_419:

*dask*
++++++

Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance  at scale for the tools you love.

*homepage*: https://dask.org/

==========    ==================    ===================================================
version       versionsuffix         toolchain                                          
==========    ==================    ===================================================
``0.8.2``     ``-Python-2.7.11``    ``intel/2016a``                                    
``0.8.2``     ``-Python-3.5.1``     ``intel/2016a``                                    
``0.11.0``    ``-Python-2.7.11``    ``foss/2016a``                                     
``0.11.0``    ``-Python-2.7.12``    ``intel/2016b``                                    
``0.11.0``    ``-Python-3.5.2``     ``intel/2016b``                                    
``0.12.0``    ``-Python-2.7.12``    ``intel/2016b``                                    
``0.12.0``    ``-Python-3.5.2``     ``foss/2016b``, ``intel/2016b``                    
``0.16.0``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``                    
``0.16.0``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``                    
``0.17.0``    ``-Python-2.7.13``    ``foss/2017a``, ``intel/2017a``                    
``0.17.0``    ``-Python-3.6.1``     ``intel/2017a``                                    
``0.17.2``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                    
``0.19.4``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.0.0``     ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                    
``1.1.4``     ``-Python-2.7.15``    ``fosscuda/2018b``                                 
``2.3.0``     ``-Python-3.7.2``     ``foss/2019a``                                     
``2.8.0``     ``-Python-3.7.4``     ``foss/2019b``                                     
==========    ==================    ===================================================


.. _list_software_datamash_835:

*datamash*
++++++++++

GNU datamash performs basic numeric, textual and statistical operations on input data files

*homepage*: https://www.gnu.org/software/datamash/

=======    ==============
version    toolchain     
=======    ==============
``1.3``    ``foss/2018a``
=======    ==============


.. _list_software_davix_540:

*davix*
+++++++

The davix project aims to make file management over HTTP-based protocols simple.  The focus is on high-performance remote I/O and data management of large collections of files.  Currently, there is support for the WebDav (link is external), Amazon S3 (link is external),  Microsoft Azure (link is external), and HTTP (link is external) protocols.

*homepage*: http://dmc.web.cern.ch/projects/davix/home

=========    ===============
version      toolchain      
=========    ===============
``0.6.6``    ``intel/2017a``
=========    ===============


.. _list_software_DB_134:

*DB*
++++

Berkeley DB enables the development of custom data management solutions,  without the overhead traditionally associated with such custom projects.

*homepage*: http://www.oracle.com/technetwork/products/berkeleydb

===========    ==================================
version        toolchain                         
===========    ==================================
``4.8.30``     ``intel/2016a``                   
``6.2.23``     ``foss/2016a``                    
``6.2.32``     ``GCCcore/6.4.0``, ``intel/2017a``
``18.1.25``    ``GCCcore/7.3.0``                 
``18.1.32``    ``GCCcore/8.2.0``                 
===========    ==================================


.. _list_software_DB_File_613:

*DB_File*
+++++++++

Perl5 access to Berkeley DB version 1.x.

*homepage*: http://perldoc.perl.org/DB_File.html

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``1.835``    ``-Perl-5.20.3``    ``intel/2016a``
``1.835``    ``-Perl-5.22.1``    ``foss/2016a`` 
=========    ================    ===============


.. _list_software_DBD-mysql_813:

*DBD-mysql*
+++++++++++

Perl binding for MySQL

*homepage*: https://metacpan.org/pod/distribution/DBD-mysql/lib/DBD/mysql.pm

=========    ================    ===============================
version      versionsuffix       toolchain                      
=========    ================    ===============================
``4.032``    ``-Perl-5.22.2``    ``intel/2016a``                
``4.033``    ``-Perl-5.24.0``    ``intel/2016b``                
``4.042``    ``-Perl-5.24.1``    ``intel/2017a``                
``4.046``    ``-Perl-5.26.0``    ``foss/2017b``, ``intel/2017b``
``4.046``    ``-Perl-5.26.1``    ``intel/2018a``                
``4.048``    ``-Perl-5.28.0``    ``foss/2018b``                 
``4.050``    ``-Perl-5.28.1``    ``foss/2019a``                 
=========    ================    ===============================


.. _list_software_DBG2OLC_477:

*DBG2OLC*
+++++++++

DBG2OLC:Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation   Sequencing Technologies

*homepage*: https://github.com/yechengxi/DBG2OLC

============    ==========================================================
version         toolchain                                                 
============    ==========================================================
``20170208``    ``intel/2016b``, ``intel/2017a``                          
``20180221``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
============    ==========================================================


.. _list_software_DBus_366:

*DBus*
++++++

D-Bus is a message bus system, a simple way for applications to talk  to one another.  In addition to interprocess communication, D-Bus helps  coordinate process lifecycle; it makes it simple and reliable to code  a "single instance" application or daemon, and to launch applications  and daemons on demand when their services are needed. 

*homepage*: http://dbus.freedesktop.org/

===========    ====================================
version        toolchain                           
===========    ====================================
``1.10.8``     ``foss/2016a``, ``intel/2016a``     
``1.10.12``    ``intel/2016b``                     
``1.10.20``    ``GCCcore/6.4.0``                   
``1.11.20``    ``intel/2017a``                     
``1.13.0``     ``intel/2017b``                     
``1.13.6``     ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.13.8``     ``GCCcore/8.2.0``                   
``1.13.12``    ``GCCcore/8.3.0``                   
===========    ====================================


.. _list_software_dbus-glib_889:

*dbus-glib*
+++++++++++

D-Bus is a message bus system, a simple way for applications to talk to one another.

*homepage*: http://dbus.freedesktop.org/doc/dbus-glib

=========    =====================================================
version      toolchain                                            
=========    =====================================================
``0.106``    ``foss/2016a``, ``intel/2016a``                      
``0.108``    ``intel/2016b``, ``intel/2017a``                     
``0.110``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``intel/2017b``
=========    =====================================================


.. _list_software_dcm2niix_798:

*dcm2niix*
++++++++++

dcm2niix is a designed to convert neuroimaging data from the DICOM format to the NIfTI format.

*homepage*: https://github.com/rordenlab/dcm2niix

================    ====================================
version             toolchain                           
================    ====================================
``1.0.20180622``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.0.20190902``    ``GCCcore/7.3.0``                   
================    ====================================


.. _list_software_DCMTK_371:

*DCMTK*
+++++++

DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers.

*homepage*: https://dicom.offis.de/dcmtk

=========    =================
version      toolchain        
=========    =================
``3.6.3``    ``GCCcore/7.3.0``
=========    =================


.. _list_software_deal.II_598:

*deal.II*
+++++++++

deal.II is a C++ program library targeted at the computational solution of  partial differential equations using adaptive finite elements.

*homepage*: https://www.dealii.org

=========    ===============
version      toolchain      
=========    ===============
``9.1.1``    ``intel/2019a``
=========    ===============


.. _list_software_deap_410:

*deap*
++++++

DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.  It seeks to make algorithms explicit and data structures transparent.

*homepage*: http://deap.readthedocs.org/en/master/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.9.2``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_deconf_623:

*deconf*
++++++++

decomposition (deconfounding) of OMICS datasets in heterogeneous tissues

*homepage*: http://web.cbio.uct.ac.za/~renaud/CRAN/

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``1.0.1``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_DeconICA_694:

*DeconICA*
++++++++++

Deconvolution of transcriptome through Immune Component Analysis (DeconICA) is an R package for identifying immune-related signals in transcriptome through deconvolution or unsupervised source separation methods.

*homepage*: https://urszulaczerwinska.github.io/DeconICA

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``0.1.0``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_deepdiff_823:

*deepdiff*
++++++++++

DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively.

*homepage*: https://deepdiff.readthedocs.io/en/latest/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``3.3.0``    ``-Python-2.7.15``    ``intel/2018b``                
``3.3.0``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
``4.0.6``    ``-Python-3.7.2``     ``GCCcore/8.2.0``              
=========    ==================    ===============================


.. _list_software_DeepSurv_814:

*DeepSurv*
++++++++++

DeepSurv is a deep learning approach to survival analysis. 

*homepage*: https://github.com/jaredleekatzman/DeepSurv

==================    =================    ==================
version               versionsuffix        toolchain         
==================    =================    ==================
``2.0.0-20180922``    ``-Python-3.6.6``    ``fosscuda/2018b``
==================    =================    ==================


.. _list_software_deepTools_943:

*deepTools*
+++++++++++

deepTools is a suite of python tools particularly developed for the efficient analysis of  high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq.

*homepage*: https://deeptools.readthedocs.io/

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``2.5.4``    ``-Python-3.6.3``    ``intel/2017b``
``3.3.1``    ``-Python-3.6.6``    ``foss/2018b`` 
=========    =================    ===============


.. _list_software_Delly_506:

*Delly*
+++++++

Delly is an integrated structural variant (SV) prediction  method that can discover, genotype and visualize deletions, tandem duplications,  inversions and translocations at single-nucleotide resolution in  short-read massively parallel sequencing data. 

*homepage*: https://github.com/dellytools/delly

=========    =================    ==========
version      versionsuffix        toolchain 
=========    =================    ==========
``0.7.8``    ``-linux_x86_64``    ``system``
=========    =================    ==========


.. _list_software_DeMixT_555:

*DeMixT*
++++++++

Cell type-specific deconvolution of heterogeneous tumor samples with two or three components  using expression data from RNAseq or microarray platforms.

*homepage*: https://bioinformatics.mdanderson.org/main/DeMixT

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``0.2.1``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_DendroPy_805:

*DendroPy*
++++++++++

A Python library for phylogenetics and phylogenetic computing:  reading, writing, simulation, processing and manipulation of phylogenetic trees  (phylogenies) and characters.

*homepage*: https://pypi.python.org/pypi/DendroPy/

=========    ==================    ==================================
version      versionsuffix         toolchain                         
=========    ==================    ==================================
``4.4.0``                          ``GCCcore/8.2.0``, ``intel/2019a``
``4.4.0``    ``-Python-2.7.15``    ``intel/2018b``                   
=========    ==================    ==================================


.. _list_software_destiny_768:

*destiny*
+++++++++

R packages to create and plot diffusion maps.

*homepage*: https://bioconductor.org/packages/destiny

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``2.5.6``    ``-R-3.4.0``     ``intel/2017a``
=========    =============    ===============


.. _list_software_detonate_852:

*detonate*
++++++++++

DETONATE (DE novo TranscriptOme rNa-seq Assembly with or without the Truth Evaluation)  consists of two component packages, RSEM-EVAL and REF-EVAL. Both packages are mainly intended to be used  to evaluate de novo transcriptome assemblies, although REF-EVAL can be used to compare sets of any kinds  of genomic sequences.

*homepage*: http://deweylab.biostat.wisc.edu/detonate/

========    ===============
version     toolchain      
========    ===============
``1.11``    ``intel/2017b``
========    ===============


.. _list_software_DFT-D3_386:

*DFT-D3*
++++++++

DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical  quantum chemical methods.

*homepage*: http://www.thch.uni-bonn.de/tc/index.php?section=downloads&subsection=DFT-D3&lang=english

=========    ===============
version      toolchain      
=========    ===============
``3.2.0``    ``intel/2019a``
=========    ===============


.. _list_software_DFTB+_331:

*DFTB+*
+++++++

DFTB+ is a fast and efficient versatile quantum mechanical simulation package.  It is based on the   Density Functional Tight Binding (DFTB) method, containing almost all of the useful extensions which have been  developed for the DFTB framework so far.  Using DFTB+ you can carry out quantum mechanical simulations like with  ab-initio density functional theory based packages, but in an approximate way gaining typically around two order of  magnitude in speed.

*homepage*: https://www.dftb-plus.info

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.3.1``                          ``intel/2017a``
``17.1``     ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_DIAL_282:

*DIAL*
++++++

DIAL (De novo Identification of Alleles) is a collection of programs  to automate the discovery of alleles for a species where we lack a reference sequence.   The SNPs/alleles are specifically selected for a low error rate in genotyping assays.   

*homepage*: http://www.bx.psu.edu/miller_lab/

==============    ==============
version           toolchain     
==============    ==============
``2011.06.06``    ``foss/2016a``
==============    ==============


.. _list_software_DIAMOND_508:

*DIAMOND*
+++++++++

DIAMOND is a sequence aligner for protein and translated DNA searches, designed for high performance  analysis of big sequence data.

*homepage*: https://github.com/bbuchfink/diamond

==========    ================================================================
version       toolchain                                                       
==========    ================================================================
``0.9.22``    ``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``0.9.24``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``                        
==========    ================================================================


.. _list_software_DicomBrowser_1232:

*DicomBrowser*
++++++++++++++

DicomBrowser is an application for inspecting and modifying DICOM metadata in many files at once.

*homepage*: http://nrg.wustl.edu/software/dicom-browser/

===========    ==================    ==========
version        versionsuffix         toolchain 
===========    ==================    ==========
``1.7.0b5``    ``-Java-1.7.0_80``    ``system``
===========    ==================    ==========


.. _list_software_Diffutils_938:

*Diffutils*
+++++++++++

Diffutils: GNU diff utilities - find the differences between files

*homepage*: http://www.gnu.org/software/diffutils/diffutils.html

=======    =============
version    toolchain    
=======    =============
``3.3``    ``GCC/4.8.2``
=======    =============


.. _list_software_dill_421:

*dill*
++++++

dill extends python's pickle module for serializing and de-serializing python objects to the majority  of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse  of which is converting a byte stream back to on python object hierarchy.

*homepage*: https://pypi.org/project/dill/

=========    =================
version      toolchain        
=========    =================
``0.3.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_distributed_1187:

*distributed*
+++++++++++++

Dask.distributed is a lightweight library for distributed computing in Python.  It extends both the concurrent.futures and dask APIs to moderate sized clusters.

*homepage*: https://distributed.readthedocs.io/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.14.3``    ``-Python-2.7.12``    ``intel/2016b``
``1.14.3``    ``-Python-3.5.2``     ``intel/2016b``
``1.21.6``    ``-Python-3.6.4``     ``intel/2018a``
==========    ==================    ===============


.. _list_software_DL_POLY_Classic_1364:

*DL_POLY_Classic*
+++++++++++++++++

DL_POLY Classic is a freely available molecular dynamics program developed  from the DL_POLY_2 package. This version does not install the java gui.

*homepage*: http://ccpforge.cse.rl.ac.uk/gf/project/dl_poly_classic/

========    =================    ===============================
version     versionsuffix        toolchain                      
========    =================    ===============================
``1.9``                          ``intel/2016b``                
``1.9``     ``-PLUMED-2.2.3``    ``intel/2016b``                
``1.10``                         ``foss/2019b``, ``intel/2019b``
========    =================    ===============================


.. _list_software_DMTCP_376:

*DMTCP*
+++++++

DMTCP is a tool to transparently checkpoint the state of multiple simultaneous applications, including multi-threaded and distributed applications. It operates directly on the user binary executable, without any Linux kernel modules or other kernel modifications.

*homepage*: https://github.com/dmtcp/dmtcp

=========    ==============================
version      toolchain                     
=========    ==============================
``2.4.5``    ``system``                    
``2.5.0``    ``foss/2016a``                
``2.5.1``    ``system``                    
``2.5.2``    ``foss/2016b``, ``foss/2018b``
=========    ==============================


.. _list_software_DOLFIN_444:

*DOLFIN*
++++++++

DOLFIN is the C++/Python interface of FEniCS, providing a consistent PSE   (Problem Solving Environment) for ordinary and partial differential equations.

*homepage*: https://bitbucket.org/fenics-project/dolfin

==================    =================    ==============
version               versionsuffix        toolchain     
==================    =================    ==============
``2018.1.0.post1``    ``-Python-3.6.4``    ``foss/2018a``
==================    =================    ==============


.. _list_software_Doris_513:

*Doris*
+++++++

Delft object-oriented radar interferometric software

*homepage*: http://doris.tudelft.nl/

=============    ===============================
version          toolchain                      
=============    ===============================
``4.02``         ``intel/2017a``                
``4.04beta4``    ``foss/2018a``, ``intel/2017a``
``4.06beta2``    ``intel/2017a``                
=============    ===============================


.. _list_software_dotNET-Core-Runtime_1781:

*dotNET-Core-Runtime*
+++++++++++++++++++++

.NET is a free, cross-platform, open source developer platform for building many different types  of applications.

*homepage*: https://www.microsoft.com/net/

=========    =================
version      toolchain        
=========    =================
``2.0.7``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_double-conversion_1774:

*double-conversion*
+++++++++++++++++++

Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.

*homepage*: https://github.com/google/double-conversion

=========    ====================================
version      toolchain                           
=========    ====================================
``3.0.3``    ``foss/2018a``                      
``3.1.4``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
=========    ====================================


.. _list_software_Doxygen_734:

*Doxygen*
+++++++++

Doxygen is a documentation system for C++, C, Java, Objective-C, Python,   IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D.

*homepage*: http://www.doxygen.org

===========    =======================================================================================================================================================
version        toolchain                                                                                                                                              
===========    =======================================================================================================================================================
``1.8.9.1``    ``GCC/4.9.2``                                                                                                                                          
``1.8.10``     ``GNU/4.9.3-2.25``, ``intel/2016.02-GCC-4.9``                                                                                                          
``1.8.11``     ``GCC/4.9.2``, ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.8.13``     ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``gimkl/2017a``                                                                                                  
``1.8.14``     ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``                                                                                                
``1.8.15``     ``GCCcore/8.2.0``                                                                                                                                      
``1.8.16``     ``GCCcore/8.3.0``                                                                                                                                      
===========    =======================================================================================================================================================


.. _list_software_Drake_487:

*Drake*
+++++++

Drake is a simple-to-use, extensible, text-based data workflow tool that organizes command execution around data and its dependencies.

*homepage*: https://github.com/Factual/drake

=========    =============    ==========
version      versionsuffix    toolchain 
=========    =============    ==========
``1.0.3``    ``-Java-1.8``    ``system``
=========    =============    ==========


.. _list_software_dropEst_737:

*dropEst*
+++++++++

Pipeline for initial analysis of droplet-based single-cell RNA-seq data

*homepage*: https://github.com/hms-dbmi/dropEst

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``0.7.1``    ``-R-3.4.3``     ``intel/2017b``
=========    =============    ===============


.. _list_software_DSA_216:

*DSA*
+++++

Digital Sorting Algorithm

*homepage*: https://github.com/zhandong/DSA

=======    =============    ==============
version    versionsuffix    toolchain     
=======    =============    ==============
``1.0``    ``-R-3.5.1``     ``foss/2018b``
=======    =============    ==============


.. _list_software_DSRC_300:

*DSRC*
++++++

DNA Sequence Reads Compression is an application designed for compression of data files  containing reads from DNA sequencing in FASTQ format. The amount of such files can be huge, e.g., a few  (or tens) of gigabytes, so a need for a robust data compression tool is clear. Usually universal compression  programs like gzip or bzip2 are used for this purpose, but it is obvious that a specialized tool can work better.

*homepage*: http://sun.aei.polsl.pl/dsrc

=========    =================    ==========
version      versionsuffix        toolchain 
=========    =================    ==========
``2.0rc``    ``-linux-64-bit``    ``system``
=========    =================    ==========


.. _list_software_Dsuite_622:

*Dsuite*
++++++++

Fast calculation of the ABBA-BABA statistics across many populations/species

*homepage*: https://github.com/millanek/Dsuite

============    ========================================
version         toolchain                               
============    ========================================
``20190713``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
============    ========================================


.. _list_software_dtcmp_536:

*dtcmp*
+++++++

Datatype Compare (DTCMP) Library for sorting and ranking distributed  data using MPI  

*homepage*: https://github.com/llnl/dtcmp

=========    ================================
version      toolchain                       
=========    ================================
``1.1.0``    ``gompi/2019a``, ``iimpi/2019a``
=========    ================================


.. _list_software_dxpy_453:

*dxpy*
++++++

DNAnexus Platform API bindings for Python

*homepage*: http://autodoc.dnanexus.com/

===========    ==================    ===============
version        versionsuffix         toolchain      
===========    ==================    ===============
``0.266.1``    ``-Python-2.7.14``    ``intel/2018a``
===========    ==================    ===============


.. _list_software_letter_e:

*E*
---


:ref:`list_software_ea-utils_804` - :ref:`list_software_earthengine-api_1521` - :ref:`list_software_EasyBuild_898` - :ref:`list_software_EasyQC_550` - :ref:`list_software_ecCodes_694` - :ref:`list_software_ED2_187` - :ref:`list_software_EggLib_554` - :ref:`list_software_eggnog-mapper_1321` - :ref:`list_software_Eigen_488` - :ref:`list_software_EIGENSOFT_676` - :ref:`list_software_elastix_762` - :ref:`list_software_Elk_284` - :ref:`list_software_ELPA_290` - :ref:`list_software_ELPH_297` - :ref:`list_software_Emacs_489` - :ref:`list_software_EMAN2_339` - :ref:`list_software_EMBOSS_457` - :ref:`list_software_emcee_511` - :ref:`list_software_enaBrowserTool_1462` - :ref:`list_software_enchant_737` - :ref:`list_software_entrypoints_1231` - :ref:`list_software_EPD_217` - :ref:`list_software_EPIC_289` - :ref:`list_software_EricScript_1016` - :ref:`list_software_ESMF_299` - :ref:`list_software_Essentia_828` - :ref:`list_software_ETE_222` - :ref:`list_software_ETSF_IO_553` - :ref:`list_software_eudev_537` - :ref:`list_software_EvidentialGene_1412` - :ref:`list_software_evmix_553` - :ref:`list_software_ExaBayes_786` - :ref:`list_software_Exonerate_939` - :ref:`list_software_expat_546` - :ref:`list_software_expect_649` - :ref:`list_software_eXpress_746` - :ref:`list_software_Extrae_617`


.. _list_software_ea-utils_804:

*ea-utils*
++++++++++

Command-line tools for processing biological sequencing data. Barcode demultiplexing, adapter trimming, etc.  Primarily written to support an Illumina based pipeline -  but should work with any FASTQs.

*homepage*: http://expressionanalysis.github.io/ea-utils/

============    ===============================================
version         toolchain                                      
============    ===============================================
``1.04.807``    ``foss/2016a``, ``foss/2016b``, ``intel/2016b``
============    ===============================================


.. _list_software_earthengine-api_1521:

*earthengine-api*
+++++++++++++++++

Python and JavaScript bindings for calling the Earth Engine API

*homepage*: https://github.com/google/earthengine-api

===========    ==================    ===============
version        versionsuffix         toolchain      
===========    ==================    ===============
``0.1.143``    ``-Python-2.7.14``    ``intel/2018a``
===========    ==================    ===============


.. _list_software_EasyBuild_898:

*EasyBuild*
+++++++++++

EasyBuild is a software build and installation framework  written in Python that allows you to install software in a structured,  repeatable and robust way.

*homepage*: https://easybuilders.github.io/easybuild

==========    ==========
version       toolchain 
==========    ==========
``1.0.0``     ``system``
``1.0.1``     ``system``
``1.0.2``     ``system``
``1.1.0``     ``system``
``1.2.0``     ``system``
``1.3.0``     ``system``
``1.4.0``     ``system``
``1.5.0``     ``system``
``1.6.0``     ``system``
``1.7.0``     ``system``
``1.8.0``     ``system``
``1.8.1``     ``system``
``1.8.2``     ``system``
``1.9.0``     ``system``
``1.10.0``    ``system``
``1.11.0``    ``system``
``1.11.1``    ``system``
``1.12.0``    ``system``
``1.12.1``    ``system``
``1.13.0``    ``system``
``1.14.0``    ``system``
``1.15.0``    ``system``
``1.15.1``    ``system``
``1.15.2``    ``system``
``1.16.0``    ``system``
``1.16.1``    ``system``
``1.16.2``    ``system``
``2.0.0``     ``system``
``2.1.0``     ``system``
``2.1.1``     ``system``
``2.2.0``     ``system``
``2.3.0``     ``system``
``2.4.0``     ``system``
``2.5.0``     ``system``
``2.6.0``     ``system``
``2.7.0``     ``system``
``2.8.0``     ``system``
``2.8.1``     ``system``
``2.8.2``     ``system``
``2.9.0``     ``system``
``3.0.0``     ``system``
``3.0.1``     ``system``
``3.0.2``     ``system``
``3.1.0``     ``system``
``3.1.1``     ``system``
``3.1.2``     ``system``
``3.2.0``     ``system``
``3.2.1``     ``system``
``3.3.0``     ``system``
``3.3.1``     ``system``
``3.4.0``     ``system``
``3.4.1``     ``system``
``3.5.0``     ``system``
``3.5.1``     ``system``
``3.5.2``     ``system``
``3.5.3``     ``system``
``3.6.0``     ``system``
``3.6.1``     ``system``
``3.6.2``     ``system``
``3.7.0``     ``system``
``3.7.1``     ``system``
``3.8.0``     ``system``
``3.8.1``     ``system``
``3.9.0``     ``system``
``3.9.1``     ``system``
``3.9.2``     ``system``
``3.9.3``     ``system``
``3.9.4``     ``system``
``4.0.0``     ``system``
``4.0.1``     ``system``
==========    ==========


.. _list_software_EasyQC_550:

*EasyQC*
++++++++

EasyQC is an R-package that provides advanced functionality to  (1) perform file-level QC of single genome-wide association (GWA) data-sets  (2) conduct quality control across several GWA data-sets (meta-level QC)  (3) simplify data-handling of large-scale GWA data-sets.

*homepage*: http://www.uni-regensburg.de/medizin/epidemiologie-praeventivmedizin/genetische-epidemiologie/software/

=======    =============    ===============
version    versionsuffix    toolchain      
=======    =============    ===============
``9.2``    ``-R-3.3.1``     ``intel/2016b``
=======    =============    ===============


.. _list_software_ecCodes_694:

*ecCodes*
+++++++++

ecCodes is a package developed by ECMWF which provides an application programming interface and  a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2,  WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding).

*homepage*: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home

==========    ================================
version       toolchain                       
==========    ================================
``2.7.3``     ``intel/2018a``                 
``2.8.2``     ``intel/2018a``                 
``2.9.2``     ``intel/2018b``, ``iomkl/2018b``
``2.12.5``    ``gompi/2019a``                 
==========    ================================


.. _list_software_ED2_187:

*ED2*
+++++

The Ecosystem Demography Biosphere Model (ED2) is an integrated terrestrial biosphere model  incorporating hydrology, land-surface biophysics, vegetation dynamics, and soil carbon and nitrogen biogeochemistry

*homepage*: https://github.com/EDmodel/ED2

============    =============    ===============
version         versionsuffix    toolchain      
============    =============    ===============
``20170201``                     ``intel/2017a``
``20170201``    ``-serial``      ``intel/2017a``
============    =============    ===============


.. _list_software_EggLib_554:

*EggLib*
++++++++

EggLib is a C++/Python library and program package for evolutionary genetics and genomics.

*homepage*: http://egglib.sourceforge.net/

==========    ===============
version       toolchain      
==========    ===============
``2.1.10``    ``intel/2016a``
==========    ===============


.. _list_software_eggnog-mapper_1321:

*eggnog-mapper*
+++++++++++++++

eggnog-mapper is a tool for fast functional annotation of novel sequences (genes or proteins)  using precomputed eggNOG-based orthology assignments

*homepage*: http://eggnog-mapper.embl.de

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.0.3``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_Eigen_488:

*Eigen*
+++++++

Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers,  and related algorithms.

*homepage*: http://eigen.tuxfamily.org/index.php?title=Main_Page

==========    ===========================================
version       toolchain                                  
==========    ===========================================
``3.2.3``     ``foss/2016a``, ``foss/2016b``             
``3.2.5``     ``system``                                 
``3.2.6``     ``system``                                 
``3.2.7``     ``foss/2016a``, ``intel/2016a``            
``3.2.8``     ``foss/2016a``, ``intel/2016a``, ``system``
``3.2.9``     ``foss/2016b``, ``intel/2016b``            
``3.2.10``    ``intel/2016b``                            
``3.3.2``     ``foss/2016b``, ``intel/2016b``            
``3.3.3``     ``GCCcore/6.3.0``, ``intel/2016b``         
``3.3.4``     ``system``                                 
``3.3.5``     ``system``                                 
``3.3.7``     ``system``                                 
==========    ===========================================


.. _list_software_EIGENSOFT_676:

*EIGENSOFT*
+++++++++++

The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.  2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal  components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of  variation;  the resulting correction is specific to a candidate marker’s variation in frequency across ancestral  populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT  package has a built-in plotting script and supports multiple file formats and quantitative phenotypes.

*homepage*: http://www.hsph.harvard.edu/alkes-price/software/

=========    ===============================
version      toolchain                      
=========    ===============================
``6.0.1``    ``foss/2016a``                 
``6.1.1``    ``foss/2016a``                 
``6.1.4``    ``foss/2016b``                 
``7.2.1``    ``foss/2019a``, ``intel/2019a``
=========    ===============================


.. _list_software_elastix_762:

*elastix*
+++++++++

elastix: a toolbox for rigid and nonrigid registration of images. 

*homepage*: http://elastix.isi.uu.nl/

=========    ==============
version      toolchain     
=========    ==============
``4.9.0``    ``foss/2018a``
=========    ==============


.. _list_software_Elk_284:

*Elk*
+++++

An all-electron full-potential linearised augmented-plane wave (FP-LAPW) code with many advanced features. Written originally at Karl-Franzens-Universität Graz as a milestone of the EXCITING EU Research and Training Network, the code is designed to be as simple as possible so that new developments in the field of density functional theory (DFT) can be added quickly and reliably. 

*homepage*: http://elk.sourceforge.net/

==========    ===============
version       toolchain      
==========    ===============
``4.0.15``    ``intel/2016b``
``4.3.6``     ``intel/2017a``
==========    ===============


.. _list_software_ELPA_290:

*ELPA*
++++++

Eigenvalue SoLvers for Petaflop-Applications .

*homepage*: http://elpa.rzg.mpg.de

===================    ================================================
version                toolchain                                       
===================    ================================================
``2015.02.002``        ``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``2016.05.004``        ``intel/2016b``, ``intel/2017a``                
``2016.11.001.pre``    ``foss/2018b``, ``intel/2018b``                 
``2017.11.001``        ``foss/2018b``, ``intel/2018a``, ``intel/2018b``
``2018.05.001``        ``foss/2018b``, ``intel/2018b``                 
``2018.11.001``        ``intel/2019a``                                 
===================    ================================================


.. _list_software_ELPH_297:

*ELPH*
++++++

ELPH is a general-purpose Gibbs sampler for finding motifs in a set   of DNA or protein sequences. The program takes as input a set containing anywhere from   a few dozen to thousands of sequences, and searches through them for the most common motif,   assuming that each sequence contains one copy of the motif. We have used ELPH to find   patterns such as ribosome binding sites (RBSs) and exon splicing enhancers (ESEs). 

*homepage*: http://ccb.jhu.edu/software/ELPH/index.shtml

=========    ==============
version      toolchain     
=========    ==============
``1.0.1``    ``foss/2018b``
=========    ==============


.. _list_software_Emacs_489:

*Emacs*
+++++++

GNU Emacs is an extensible, customizable text editor--and more.  At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming  language with extensions to support text editing.

*homepage*: http://www.gnu.org/software/emacs/

========    =============    =======================================================
version     versionsuffix    toolchain                                              
========    =============    =======================================================
``24.3``                     ``GCC/4.8.3``                                          
``24.3``    ``-bare``        ``GCC/4.8.3``                                          
``24.4``                     ``GCC/4.9.2``                                          
``24.5``                     ``GCC/4.9.3-2.25``                                     
``25.1``                     ``foss/2016a``                                         
``25.3``                     ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
========    =============    =======================================================


.. _list_software_EMAN2_339:

*EMAN2*
+++++++

EMAN2 is a broadly based greyscale scientific image processing suite with a  primary focus on processing data from transmission electron microscopes. 

*homepage*: https://blake.bcm.edu/emanwiki/EMAN2

=========    ===============================    ==================================
version      versionsuffix                      toolchain                         
=========    ===============================    ==================================
``2.3``                                         ``system``                        
``2.3``      ``-Python-2.7.15``                 ``foss/2019a``, ``fosscuda/2019a``
``2.21a``    ``-Python-2.7.14-Boost-1.63.0``    ``foss/2018a``                    
=========    ===============================    ==================================


.. _list_software_EMBOSS_457:

*EMBOSS*
++++++++

EMBOSS is 'The European Molecular Biology Open Software Suite'.  EMBOSS is a free Open Source software analysis package specially developed for  the needs of the molecular biology (e.g. EMBnet) user community.

*homepage*: http://emboss.sourceforge.net/

=========    =================    ==========================================================================================================
version      versionsuffix        toolchain                                                                                                 
=========    =================    ==========================================================================================================
``6.6.0``                         ``foss/2016b``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017a``, ``intel/2018b``
``6.6.0``    ``-X11-20170314``    ``intel/2017a``                                                                                           
=========    =================    ==========================================================================================================


.. _list_software_emcee_511:

*emcee*
+++++++

Emcee is an extensible, pure-Python implementation of  Goodman & Weare's Affine Invariant Markov chain Monte Carlo (MCMC) Ensemble sampler.  It's designed for Bayesian parameter estimation and it's really sweet! 

*homepage*: https://dfm.io/emcee

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``2.2.1``                          ``foss/2019a``                 
``2.2.1``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
``2.2.1``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
=========    ==================    ===============================


.. _list_software_enaBrowserTool_1462:

*enaBrowserTool*
++++++++++++++++

enaBrowserTools is a set of scripts that interface with the ENA  web services to download data from ENA easily, without any knowledge of scripting required.

*homepage*: https://github.com/enasequence/enaBrowserTools/

=========    =================    =================
version      versionsuffix        toolchain        
=========    =================    =================
``1.5.4``    ``-Python-3.7.2``    ``GCCcore/8.2.0``
=========    =================    =================


.. _list_software_enchant_737:

*enchant*
+++++++++

Enchant is a library (and command-line program) that wraps a number of different spelling libraries  and programs with a consistent interface.

*homepage*: https://abiword.github.io/enchant/

=========    ===============
version      toolchain      
=========    ===============
``1.6.1``    ``intel/2017a``
=========    ===============


.. _list_software_entrypoints_1231:

*entrypoints*
+++++++++++++

Entry points are a way for Python packages to advertise objects with some common interface.

*homepage*: https://github.com/takluyver/entrypoints

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.2.2``    ``-Python-2.7.11``    ``foss/2016a``                 
``0.2.2``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``0.2.2``    ``-Python-3.5.1``     ``foss/2016a``                 
``0.2.2``    ``-Python-3.5.2``     ``intel/2016b``                
=========    ==================    ===============================


.. _list_software_EPD_217:

*EPD*
+++++

The Enthought Python Distribution provides scientists with a comprehensive set of tools to perform  rigorous data analysis and visualization. Python, distinguished by its flexibility, coherence, and ease-of-use,  is rapidly becoming the programming language of choice for researchers worldwide.  EPD extends this capacity with a powerful collection of Python libraries to enable interactive technical computing and  cross-platform rapid application development.

*homepage*: http://www.enthought.com/products/edudownload.php

=============    ==========
version          toolchain 
=============    ==========
``7.3-2-rh5``    ``system``
=============    ==========


.. _list_software_EPIC_289:

*EPIC*
++++++

Package implementing EPIC method to estimate the proportion of immune, stromal, endothelial  and cancer or other cells from bulk gene expression data.

*homepage*: https://gfellerlab.shinyapps.io/EPIC_1-1

=======    =============    ==============
version    versionsuffix    toolchain     
=======    =============    ==============
``1.1``    ``-R-3.5.1``     ``foss/2018b``
=======    =============    ==============


.. _list_software_EricScript_1016:

*EricScript*
++++++++++++

EricScript is a computational framework for the discovery of gene fusions in paired end RNA-seq data.

*homepage*: https://sites.google.com/site/bioericscript/home

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``0.5.5``    ``-R-3.4.0``     ``intel/2017a``
=========    =============    ===============


.. _list_software_ESMF_299:

*ESMF*
++++++

The Earth System Modeling Framework (ESMF) is software for building and coupling weather,  climate, and related models.

*homepage*: http://sourceforge.net/projects/esmf

============    ================    =================================================================
version         versionsuffix       toolchain                                                        
============    ================    =================================================================
``6.3.0rp1``                        ``intel/2017a``                                                  
``6.3.0rp1``    ``-HDF5-1.8.18``    ``intel/2017a``                                                  
``7.0.0``                           ``foss/2016a``                                                   
``7.0.2``                           ``intel/2017b``                                                  
``7.1.0r``                          ``foss/2018b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
============    ================    =================================================================


.. _list_software_Essentia_828:

*Essentia*
++++++++++

Open-source library and tools for audio and music analysis, description and synthesis

*homepage*: https://essentia.upf.edu

=============    ==================    ==============
version          versionsuffix         toolchain     
=============    ==================    ==============
``2.1_beta5``    ``-Python-2.7.15``    ``foss/2019a``
=============    ==================    ==============


.. _list_software_ETE_222:

*ETE*
+++++

A Python framework for the analysis and visualization of trees

*homepage*: http://etetoolkit.org

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``3.0.0b36``    ``-Python-2.7.12``    ``intel/2016b``
============    ==================    ===============


.. _list_software_ETSF_IO_553:

*ETSF_IO*
+++++++++

A library of F90 routines to read/write the ETSF file  format has been written. It is called ETSF_IO and available under LGPL. 

*homepage*: http://www.etsf.eu/resources/software/libraries_and_tools

=========    =================================================================================================
version      toolchain                                                                                        
=========    =================================================================================================
``1.0.4``    ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
=========    =================================================================================================


.. _list_software_eudev_537:

*eudev*
+++++++

eudev is a fork of systemd-udev with the goal of obtaining better  compatibility with existing software such as OpenRC and Upstart,  older kernels, various toolchains and anything else required by  users and various distributions. 

*homepage*: https://wiki.gentoo.org/wiki/Project:Eudev

=========    =================================================
version      toolchain                                        
=========    =================================================
``3.1.5``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``3.2``      ``GCCcore/4.9.3``                                
``3.2.2``    ``GCCcore/6.4.0``                                
=========    =================================================


.. _list_software_EvidentialGene_1412:

*EvidentialGene*
++++++++++++++++

EvidentialGene is a genome informatics project for  "Evidence Directed Gene Construction for Eukaryotes",  for constructing high quality, accurate gene sets for  animals and plants (any eukaryotes), being developed by  Don Gilbert at Indiana University, gilbertd at indiana edu.

*homepage*: http://arthropods.eugenes.org/EvidentialGene/

==============    ================    ===============
version           versionsuffix       toolchain      
==============    ================    ===============
``2018.01.01``    ``-Perl-5.24.1``    ``intel/2017a``
==============    ================    ===============


.. _list_software_evmix_553:

*evmix*
+++++++

evmix: Extreme Value Mixture Modelling,  Threshold Estimation and Boundary Corrected Kernel Density Estimation

*homepage*: http://cran.r-project.org/web/packages/evmix

=======    =============    ===============
version    versionsuffix    toolchain      
=======    =============    ===============
``2.6``    ``-R-3.3.1``     ``intel/2016b``
=======    =============    ===============


.. _list_software_ExaBayes_786:

*ExaBayes*
++++++++++

ExaBayes is a software package for Bayesian tree inference.  It is particularly suitable for large-scale analyses on computer clusters

*homepage*: https://cme.h-its.org/exelixis/web/software/exabayes/

=======    ==============
version    toolchain     
=======    ==============
``1.5``    ``foss/2016b``
=======    ==============


.. _list_software_Exonerate_939:

*Exonerate*
+++++++++++

Exonerate is a generic tool for pairwise sequence comparison.  It allows you to align sequences using a many alignment models, using either   exhaustive dynamic programming, or a variety of heuristics. 

*homepage*: https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate

=========    ====================================================================================================================================================================
version      toolchain                                                                                                                                                           
=========    ====================================================================================================================================================================
``2.4.0``    ``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017a``
=========    ====================================================================================================================================================================


.. _list_software_expat_546:

*expat*
+++++++

Expat is an XML parser library written in C. It is a stream-oriented parser  in which an application registers handlers for things the parser might find  in the XML document (like start tags) 

*homepage*: https://libexpat.github.io

=========    =========================================================================================================================
version      toolchain                                                                                                                
=========    =========================================================================================================================
``2.1.0``    ``GCC/4.9.2``, ``foss/2016a``, ``intel/2016a``                                                                           
``2.1.1``    ``foss/2016a``, ``intel/2016a``                                                                                          
``2.2.0``    ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.2.4``    ``GCCcore/6.4.0``                                                                                                        
``2.2.5``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                                                                                     
``2.2.6``    ``GCCcore/8.2.0``                                                                                                        
``2.2.7``    ``GCCcore/8.3.0``                                                                                                        
=========    =========================================================================================================================


.. _list_software_expect_649:

*expect*
++++++++

Expect is a tool for automating interactive applications  such as telnet, ftp, passwd, fsck, rlogin, tip, etc.  Expect really makes this stuff trivial.  Expect is also useful for testing these same applications.  

*homepage*: https://core.tcl.tk/expect/index

==========    =================
version       toolchain        
==========    =================
``5.45.4``    ``GCCcore/7.3.0``
==========    =================


.. _list_software_eXpress_746:

*eXpress*
+++++++++

Streaming quantification for high-throughput sequencing

*homepage*: https://pachterlab.github.io/eXpress

=========    ===============
version      toolchain      
=========    ===============
``1.5.1``    ``intel/2017b``
=========    ===============


.. _list_software_Extrae_617:

*Extrae*
++++++++

Extrae is the core instrumentation package developed by the Performance Tools group at BSC. Extrae is capable of instrumenting applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1 using different instrumentation approaches. The information gathered by Extrae typically includes timestamped events of runtime calls, performance counters and source code references. Besides, Extrae provides its own API to allow the user to manually instrument his or her application.

*homepage*: http://www.bsc.es/computer-sciences/performance-tools

=========    ===============
version      toolchain      
=========    ===============
``3.4.1``    ``foss/2017a`` 
``3.7.1``    ``intel/2019a``
=========    ===============


.. _list_software_letter_f:

*F*
---


:ref:`list_software_f90cache_707` - :ref:`list_software_faceswap_842` - :ref:`list_software_FALCON_435` - :ref:`list_software_FANN_291` - :ref:`list_software_fast5_483` - :ref:`list_software_FASTA_367` - :ref:`list_software_FastaIndex_999` - :ref:`list_software_FastANI_614` - :ref:`list_software_FastME_544` - :ref:`list_software_fastp_542` - :ref:`list_software_fastPHASE_799` - :ref:`list_software_fastq-tools_1149` - :ref:`list_software_FastQ_Screen_1182` - :ref:`list_software_FastQC_546` - :ref:`list_software_FastQTL_639` - :ref:`list_software_fastqz_665` - :ref:`list_software_FastRFS_633` - :ref:`list_software_fastStructure_1407` - :ref:`list_software_FastTree_798` - :ref:`list_software_FASTX-Toolkit_1177` - :ref:`list_software_fatslim_752` - :ref:`list_software_fbm_309` - :ref:`list_software_FCM_214` - :ref:`list_software_FDS_221` - :ref:`list_software_fdstools_878` - :ref:`list_software_FDTD_Solutions_1361` - :ref:`list_software_feh_307` - :ref:`list_software_Ferret_616` - :ref:`list_software_FFC_207` - :ref:`list_software_FFLAS-FFPACK_836` - :ref:`list_software_FFmpeg_565` - :ref:`list_software_ffnet_531` - :ref:`list_software_FFTW_311` - :ref:`list_software_FGSL_300` - :ref:`list_software_FIAT_292` - :ref:`list_software_file_416` - :ref:`list_software_Filtlong_831` - :ref:`list_software_find_circ_929` - :ref:`list_software_findhap_730` - :ref:`list_software_findutils_978` - :ref:`list_software_fineRADstructure_1642` - :ref:`list_software_fineSTRUCTURE_1139` - :ref:`list_software_Firefox_723` - :ref:`list_software_FireWorks_924` - :ref:`list_software_fixesproto_1107` - :ref:`list_software_FLANN_367` - :ref:`list_software_FLASH_366` - :ref:`list_software_Flask_497` - :ref:`list_software_FLEUR_382` - :ref:`list_software_flex_431` - :ref:`list_software_FlexiDot_799` - :ref:`list_software_FLINT_381` - :ref:`list_software_FLTK_305` - :ref:`list_software_FLUENT_462` - :ref:`list_software_Flye_400` - :ref:`list_software_FMILibrary_945` - :ref:`list_software_FMRIprep_741` - :ref:`list_software_fmt_327` - :ref:`list_software_FoldX_477` - :ref:`list_software_fontconfig_1069` - :ref:`list_software_fontsproto_1118` - :ref:`list_software_foss_443` - :ref:`list_software_fosscuda_856` - :ref:`list_software_FoX_269` - :ref:`list_software_FPM_227` - :ref:`list_software_fqtrim_659` - :ref:`list_software_fqzcomp_768` - :ref:`list_software_FragGeneScan_1156` - :ref:`list_software_FRANz_417` - :ref:`list_software_FreeFem++_752` - :ref:`list_software_freeglut_862` - :ref:`list_software_FreeImage_869` - :ref:`list_software_FreeSurfer_1017` - :ref:`list_software_freetype_868` - :ref:`list_software_FreeXL_550` - :ref:`list_software_FriBidi_665` - :ref:`list_software_FSL_229` - :ref:`list_software_FSLeyes_667` - :ref:`list_software_FTGL_301` - :ref:`list_software_fullrmc_757` - :ref:`list_software_FUNWAVE-TVD_823` - :ref:`list_software_FUSE_307` - :ref:`list_software_future_667` - :ref:`list_software_fxtract_764`


.. _list_software_f90cache_707:

*f90cache*
++++++++++

f90cache is a compiler cache. It acts as a caching pre-processor to Fortran compilers,  using the -E compiler switch and a hash to detect when a compilation can be satisfied from cache.  This often results in a great speedup in common compilations.

*homepage*: http://people.irisa.fr/Edouard.Canot/f90cache/

========    ==========
version     toolchain 
========    ==========
``0.96``    ``system``
========    ==========


.. _list_software_faceswap_842:

*faceswap*
++++++++++

Faceswap is a tool that utilizes deep learning to recognize and swap faces in pictures and videos.

*homepage*: https://github.com/deepfakes/faceswap

============    =================    ==============
version         versionsuffix        toolchain     
============    =================    ==============
``20180212``    ``-Python-3.6.3``    ``foss/2017b``
============    =================    ==============


.. _list_software_FALCON_435:

*FALCON*
++++++++

Falcon: a set of tools for fast aligning long reads for consensus and assembly

*homepage*: https://github.com/PacificBiosciences/FALCON

=========    ===============
version      toolchain      
=========    ===============
``1.8.8``    ``intel/2017b``
=========    ===============


.. _list_software_FANN_291:

*FANN*
++++++

Fast Artificial Neural Network Library is a free open source neural network library,  which implements multilayer artificial neural networks in C with support for both fully connected  and sparsely connected networks.

*homepage*: http://leenissen.dk

=========    ==================================
version      toolchain                         
=========    ==================================
``2.2.0``    ``GCCcore/6.4.0``, ``intel/2018a``
=========    ==================================


.. _list_software_fast5_483:

*fast5*
+++++++

A lightweight C++ library for accessing Oxford Nanopore  Technologies sequencing data.

*homepage*: http://simpsonlab.github.io/2017/02/27/packing_fast5/

=========    ==========
version      toolchain 
=========    ==========
``0.6.5``    ``system``
=========    ==========


.. _list_software_FASTA_367:

*FASTA*
+++++++

The FASTA programs find regions of local or global (new) similarity between  protein or DNA sequences, either by searching Protein or DNA databases, or by identifying  local duplications within a sequence.

*homepage*: http://fasta.bioch.virginia.edu

===========    ==============
version        toolchain     
===========    ==============
``36.3.5e``    ``foss/2016b``
===========    ==============


.. _list_software_FastaIndex_999:

*FastaIndex*
++++++++++++

FastA index (.fai) handler compatible with samtools faidx

*homepage*: https://github.com/lpryszcz/FastaIndex

===========    ==================    ===============
version        versionsuffix         toolchain      
===========    ==================    ===============
``0.11rc7``    ``-Python-2.7.14``    ``intel/2017b``
===========    ==================    ===============


.. _list_software_FastANI_614:

*FastANI*
+++++++++

FastANI is developed for fast alignment-free computation of  whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean  nucleotide identity of orthologous gene pairs shared between two microbial  genomes. FastANI supports pairwise comparison of both complete and draft  genome assemblies.

*homepage*: https://www.iodbc.org/

=======    ==============================================================
version    toolchain                                                     
=======    ==============================================================
``1.1``    ``foss/2018b``, ``intel/2018b``                               
``1.2``    ``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
=======    ==============================================================


.. _list_software_FastME_544:

*FastME*
++++++++

FastME: a comprehensive, accurate and fast distance-based phylogeny inference program.

*homepage*: http://www.atgc-montpellier.fr/fastme/

===========    ================================
version        toolchain                       
===========    ================================
``2.1.5``      ``foss/2016a``                  
``2.1.6.1``    ``intel/2018a``, ``intel/2018b``
===========    ================================


.. _list_software_fastp_542:

*fastp*
+++++++

A tool designed to provide fast all-in-one preprocessing for FastQ files.   This tool is developed in C++ with multithreading supported to afford high performance.

*homepage*: https://github.com/OpenGene/fastp

==========    ==============
version       toolchain     
==========    ==============
``0.19.7``    ``foss/2018b``
==========    ==============


.. _list_software_fastPHASE_799:

*fastPHASE*
+++++++++++

fastPHASE: software for haplotype reconstruction,  and estimating missing genotypes from population data Documentation: http://scheet.org/code/fastphase_doc_1.4.pdf

*homepage*: http://stephenslab.uchicago.edu/software.html#fastphase

=========    ==========
version      toolchain 
=========    ==========
``1.4.8``    ``system``
=========    ==========


.. _list_software_fastq-tools_1149:

*fastq-tools*
+++++++++++++

This package provides a number of small and efficient programs to perform  common tasks with high throughput sequencing data in the FASTQ format. All of the programs  work with typical FASTQ files as well as gzipped FASTQ files.

*homepage*: https://homes.cs.washington.edu/~dcjones/fastq-tools/

=======    ==============================
version    toolchain                     
=======    ==============================
``0.8``    ``foss/2016b``, ``foss/2018b``
=======    ==============================


.. _list_software_FastQ_Screen_1182:

*FastQ_Screen*
++++++++++++++

FastQ Screen allows you to screen a library of sequences in FastQ  format against a set of sequence databases so you can see if the composition of the  library matches with what you expect.

*homepage*: http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/

==========    ================    ===============
version       versionsuffix       toolchain      
==========    ================    ===============
``0.11.3``    ``-Perl-5.24.0``    ``foss/2016b`` 
``0.11.4``    ``-Perl-5.24.0``    ``foss/2016b`` 
``0.12.0``    ``-Perl-5.26.1``    ``intel/2018a``
``0.13.0``    ``-Perl-5.28.0``    ``foss/2018b`` 
==========    ================    ===============


.. _list_software_FastQC_546:

*FastQC*
++++++++

FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline.

*homepage*: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/

==========    ===================    ==========
version       versionsuffix          toolchain 
==========    ===================    ==========
``0.10.1``    ``-Java-1.7.0_80``     ``system``
``0.11.2``    ``-Java-1.7.0_60``     ``system``
``0.11.3``    ``-Java-1.7.0_80``     ``system``
``0.11.4``    ``-Java-1.8.0_66``     ``system``
``0.11.4``    ``-Java-1.8.0_74``     ``system``
``0.11.5``    ``-Java-1.7.0_80``     ``system``
``0.11.5``    ``-Java-1.8.0_144``    ``system``
``0.11.5``    ``-Java-1.8.0_74``     ``system``
``0.11.7``    ``-Java-1.8.0_162``    ``system``
``0.11.8``    ``-Java-1.8``          ``system``
``0.11.8``    ``-Java-11``           ``system``
==========    ===================    ==========


.. _list_software_FastQTL_639:

*FastQTL*
+++++++++

FastQTL is a QTL mapper

*homepage*: http://fastqtl.sourceforge.net/

=========    ==============
version      toolchain     
=========    ==============
``2.184``    ``foss/2018b``
=========    ==============


.. _list_software_fastqz_665:

*fastqz*
++++++++

fastqz is a compressor for FASTQ files. FASTQ is the output of DNA sequencing machines.  It is one of the compressors described in the paper: Bonfield JK, Mahoney MV (2013) Compression of  FASTQ and SAM Format Sequencing Data. (mirror) PLoS ONE 8(3): e59190. doi:10.1371/journal.pone.0059190

*homepage*: http://mattmahoney.net/dc/fastqz/

=======    =============
version    toolchain    
=======    =============
``1.5``    ``GCC/4.8.2``
=======    =============


.. _list_software_FastRFS_633:

*FastRFS*
+++++++++

Fast Robinson Foulds Supertrees

*homepage*: https://github.com/pranjalv123/FastRFS

================    ===============
version             toolchain      
================    ===============
``1.0-20190613``    ``gompi/2019a``
================    ===============


.. _list_software_fastStructure_1407:

*fastStructure*
+++++++++++++++

fastStructure is a fast algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x. 

*homepage*: https://rajanil.github.io/fastStructure/

=======    ==================    ==============
version    versionsuffix         toolchain     
=======    ==================    ==============
``1.0``    ``-Python-2.7.11``    ``foss/2016a``
``1.0``    ``-Python-2.7.12``    ``foss/2016b``
``1.0``    ``-Python-2.7.13``    ``foss/2017a``
``1.0``    ``-Python-2.7.15``    ``foss/2019a``
=======    ==================    ==============


.. _list_software_FastTree_798:

*FastTree*
++++++++++

FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide  or protein sequences. FastTree can handle alignments with up to a million of sequences in a reasonable amount of  time and memory. 

*homepage*: http://www.microbesonline.org/fasttree/

==========    =================================================================
version       toolchain                                                        
==========    =================================================================
``2.1.10``    ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``2.1.11``    ``GCCcore/8.2.0``                                                
==========    =================================================================


.. _list_software_FASTX-Toolkit_1177:

*FASTX-Toolkit*
+++++++++++++++

The FASTX-Toolkit is a collection of command line tools for   Short-Reads FASTA/FASTQ files preprocessing.

*homepage*: http://hannonlab.cshl.edu/fastx_toolkit/

==========    ==================================================================
version       toolchain                                                         
==========    ==================================================================
``0.0.14``    ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2018a``
==========    ==================================================================


.. _list_software_fatslim_752:

*fatslim*
+++++++++

FATSLiM stands for “Fast Analysis Toolbox for Simulations of Lipid Membranes” and its goal is to provide an efficient, yet robust, tool to extract physical parameters from MD trajectories.

*homepage*: https://github.com/FATSLiM/fatslim

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.2.1``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_fbm_309:

*fbm*
+++++

Exact methods for simulating fractional Brownian motion and fractional Gaussian noise in Python

*homepage*: https://pypi.org/project/fbm

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.2.0``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_FCM_214:

*FCM*
+++++

FCM is a set of tools for managing and building source code.

*homepage*: http://www.metoffice.gov.uk/research/collaboration/fcm

=========    ==========
version      toolchain 
=========    ==========
``2.3.1``    ``system``
=========    ==========


.. _list_software_FDS_221:

*FDS*
+++++

Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,  with an emphasis on smoke and heat transport from fires.

*homepage*: https://pages.nist.gov/fds-smv/

=========    =============    ================================
version      versionsuffix    toolchain                       
=========    =============    ================================
``6.0.1``    ``-no-OFED``     ``system``                      
``6.5.2``                     ``intel/2016b``                 
``6.5.3``                     ``intel/2017a``                 
``6.6.0``                     ``intel/2017b``, ``intel/2018a``
``6.7.0``                     ``intel/2018a``                 
=========    =============    ================================


.. _list_software_fdstools_878:

*fdstools*
++++++++++

Forensic DNA Sequencing Tools Tools for characterisation and filtering of PCR stutter artefacts  and other systemic noise in Next Generation Sequencing data of forensic STR markers.

*homepage*: https://git.lumc.nl/jerryhoogenboom/fdstools

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20160322``    ``-Python-2.7.11``    ``foss/2016a``
============    ==================    ==============


.. _list_software_FDTD_Solutions_1361:

*FDTD_Solutions*
++++++++++++++++

High performance FDTD-method Maxwell solver for the design, analysis and optimization of nanophotonic devices, processes and materials.

*homepage*: http://www.lumerical.com/tcad-products/fdtd/

=============    ==========
version          toolchain 
=============    ==========
``8.6.2``        ``system``
``8.11.337``     ``system``
``8.16.982``     ``system``
``8.20.1731``    ``system``
=============    ==========


.. _list_software_feh_307:

*feh*
+++++

feh is an X11 image viewer aimed mostly at console users. Unlike most other viewers, it does not have a fancy GUI, but simply displays images. It is controlled via commandline arguments and configurable key/mouse actions. 

*homepage*: https://feh.finalrewind.org/

========    =================
version     toolchain        
========    =================
``2.26``    ``GCCcore/6.4.0``
========    =================


.. _list_software_Ferret_616:

*Ferret*
++++++++

Ferret is an interactive computer visualization and analysis environment designed to meet the needs of oceanographers and meteorologists analyzing large and complex gridded data sets.

*homepage*: http://ferret.pmel.noaa.gov/

=======    ===============
version    toolchain      
=======    ===============
``7.3``    ``intel/2017b``
=======    ===============


.. _list_software_FFC_207:

*FFC*
+++++

The FEniCS Form Compiler (FFC) is a compiler for finite element variational forms.

*homepage*: https://bitbucket.org/fenics-project/ffc

============    =================    ==============
version         versionsuffix        toolchain     
============    =================    ==============
``2018.1.0``    ``-Python-3.6.4``    ``foss/2018a``
============    =================    ==============


.. _list_software_FFLAS-FFPACK_836:

*FFLAS-FFPACK*
++++++++++++++

Finite Field Linear Algebra Subroutines / Package

*homepage*: https://linbox-team.github.io/fflas-ffpack/

=========    ==============
version      toolchain     
=========    ==============
``2.2.0``    ``foss/2016a``
=========    ==============


.. _list_software_FFmpeg_565:

*FFmpeg*
++++++++

A complete, cross-platform solution to record, convert and stream audio and video.

*homepage*: https://www.ffmpeg.org/

===========    ===================================================
version        toolchain                                          
===========    ===================================================
``0.10.16``    ``gimkl/2.11.5``, ``intel/2016a``                  
``2.8.6``      ``intel/2016a``                                    
``2.8.7``      ``foss/2016a``, ``intel/2016a``                    
``3.0.2``      ``foss/2016a``, ``intel/2016a``                    
``3.1.3``      ``foss/2016b``, ``intel/2016b``                    
``3.2.4``      ``gimkl/2017a``                                    
``3.3.1``      ``foss/2016b``                                     
``3.3.4``      ``intel/2017a``                                    
``3.4``        ``GCCcore/6.4.0``                                  
``3.4.1``      ``foss/2017b``, ``intel/2017b``                    
``3.4.2``      ``foss/2018a``, ``intel/2018a``                    
``3.4.5``      ``foss/2018b``                                     
``4.0``        ``foss/2018a``, ``intel/2018a``                    
``4.0.1``      ``intel/2018a``                                    
``4.1``        ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``4.1.3``      ``GCCcore/8.2.0``                                  
``4.2.1``      ``GCCcore/8.3.0``                                  
===========    ===================================================


.. _list_software_ffnet_531:

*ffnet*
+++++++

Feed-forward neural network solution for python

*homepage*: http://ffnet.sourceforge.net/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.8.3``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_FFTW_311:

*FFTW*
++++++

FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)  in one or more dimensions, of arbitrary input size, and of both real and complex data.

*homepage*: http://www.fftw.org

=========    =============    ========================================================================================================================================================================================
version      versionsuffix    toolchain                                                                                                                                                                               
=========    =============    ========================================================================================================================================================================================
``2.1.5``                     ``intel/2016b``, ``intel/2017a``, ``intel/2018b``                                                                                                                                       
``3.3.4``                     ``gmpich/2016a``, ``gmvapich2/1.7.20``, ``gmvapich2/2016a``, ``gompi/2016.04``, ``gompi/2016.06``, ``gompi/2016.07``, ``gompi/2016a``, ``gompi/2016b``, ``intel/2016a``, ``intel/2016b``
``3.3.5``                     ``gompi/2016.07``, ``gompi/2016.09``, ``gompi/2016b``, ``intel/2016b``                                                                                                                  
``3.3.6``                     ``gimpi/2017b``, ``gimpic/2017b``, ``gompi/2017a``, ``gompi/2017b``, ``gompic/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``                           
``3.3.7``                     ``gimkl/2017a``, ``gimpi/2018a``, ``gmpich/2017.08``, ``gompi/2018a``, ``gompic/2018a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018a``, ``iomkl/2018a``        
``3.3.7``    ``-serial``      ``GCC/6.4.0-2.28``                                                                                                                                                                      
``3.3.8``                     ``gompi/2018.08``, ``gompi/2018b``, ``gompi/2019a``, ``gompi/2019b``, ``gompic/2018b``, ``gompic/2019a``, ``gompic/2019b``, ``intel/2018b``, ``intel/2019a``, ``iomkl/2018b``           
=========    =============    ========================================================================================================================================================================================


.. _list_software_FGSL_300:

*FGSL*
++++++

FGSL: A Fortran interface to the GNU Scientific Library

*homepage*: https://www.lrz.de/services/software/mathematik/gsl/fortran/

=========    ===============
version      toolchain      
=========    ===============
``1.1.0``    ``intel/2016b``
=========    ===============


.. _list_software_FIAT_292:

*FIAT*
++++++

The FInite element Automatic Tabulator (FIAT) supports generation of arbitrary order  instances of the Lagrange elements on lines, triangles, and tetrahedra. It is also capable of generating  arbitrary order instances of Jacobi-type quadrature rules on the same element shapes.

*homepage*: https://bitbucket.org/fenics-project/fiat

============    ==================    ===============================
version         versionsuffix         toolchain                      
============    ==================    ===============================
``1.6.0``       ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``2018.1.0``    ``-Python-3.6.4``     ``foss/2018a``                 
============    ==================    ===============================


.. _list_software_file_416:

*file*
++++++

The file command is 'a file type guesser', that is, a command-line tool  that tells you in words what kind of data a file contains.

*homepage*: http://www.darwinsys.com/file/

========    =================
version     toolchain        
========    =================
``5.17``    ``GCC/4.8.2``    
``5.25``    ``intel/2016a``  
``5.28``    ``foss/2016b``   
``5.30``    ``intel/2017a``  
``5.33``    ``GCCcore/6.4.0``
``5.35``    ``GCCcore/7.3.0``
========    =================


.. _list_software_Filtlong_831:

*Filtlong*
++++++++++

Filtlong is a tool for filtering long reads by quality. It can take a set  of long reads and produce a smaller, better subset. It uses both read length (longer is better)  and read identity (higher is better) when choosing which reads pass the filter

*homepage*: https://github.com/rrwick/Filtlong

=========    ==============
version      toolchain     
=========    ==============
``0.2.0``    ``foss/2016b``
=========    ==============


.. _list_software_find_circ_929:

*find_circ*
+++++++++++

circRNA detection from RNA-seq reads

*homepage*: https://github.com/marvin-jens/find_circ

================    ==================    ===============
version             versionsuffix         toolchain      
================    ==================    ===============
``1.2-20170228``    ``-Python-2.7.14``    ``intel/2017b``
================    ==================    ===============


.. _list_software_findhap_730:

*findhap*
+++++++++

Find haplotypes and impute genotypes using multiple chip sets and sequence data

*homepage*: http://aipl.arsusda.gov/software/findhap/

=======    ==========
version    toolchain 
=======    ==========
``4``      ``system``
=======    ==========


.. _list_software_findutils_978:

*findutils*
+++++++++++

findutils: The GNU find, locate, updatedb, and xargs utilities

*homepage*: http://www.gnu.org/software/findutils/findutils.html

=========    =============
version      toolchain    
=========    =============
``4.4.2``    ``GCC/4.8.2``
=========    =============


.. _list_software_fineRADstructure_1642:

*fineRADstructure*
++++++++++++++++++

A package for population structure inference from RAD-seq data

*homepage*: http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html

============    ===============
version         toolchain      
============    ===============
``20180709``    ``intel/2018a``
============    ===============


.. _list_software_fineSTRUCTURE_1139:

*fineSTRUCTURE*
+++++++++++++++

fineSTRUCTURE is a fast and powerful algorithm for identifying population structure using  dense sequencing data.

*homepage*: https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html

=========    ===============
version      toolchain      
=========    ===============
``2.1.3``    ``intel/2017b``
=========    ===============


.. _list_software_Firefox_723:

*Firefox*
+++++++++

Firefox is a free, open source Web browser for Windows, Linux and Mac OS X. It is based on the  Mozilla code base and offers customization options and features such as its capability to block pop-up windows,  tabbed browsing, privacy and security measures, smart searching, and RSS live bookmarks.

*homepage*: https://www.mozilla.org/en-US/firefox/new/

==========    ==========
version       toolchain 
==========    ==========
``44.0.2``    ``system``
==========    ==========


.. _list_software_FireWorks_924:

*FireWorks*
+++++++++++

FireWorks helps run calculation workflows, with a centralized workflow server  controlling many worker nodes.

*homepage*: https://pypi.python.org/pypi/FireWorks

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.4.2``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_fixesproto_1107:

*fixesproto*
++++++++++++

X.org FixesProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=======    =================================================
version    toolchain                                        
=======    =================================================
``5.0``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=======    =================================================


.. _list_software_FLANN_367:

*FLANN*
+++++++

FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces.

*homepage*: http://www.cs.ubc.ca/research/flann/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.8.4``    ``-Python-2.7.11``    ``intel/2016a``
``1.8.4``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_FLASH_366:

*FLASH*
+++++++

FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to merge pairs of reads when the original DNA fragments are shorter than twice the length of reads. The resulting longer reads can significantly improve genome assemblies. They can also improve transcriptome assembly when FLASH is used to merge RNA-seq data. 

*homepage*: https://ccb.jhu.edu/software/FLASH/

==========    ==============================================
version       toolchain                                     
==========    ==============================================
``1.2.11``    ``foss/2016a``, ``foss/2018a``, ``foss/2018b``
``2.2.00``    ``foss/2018b``                                
==========    ==============================================


.. _list_software_Flask_497:

*Flask*
+++++++

" Flask is a lightweight WSGI web application framework.  It is designed to make getting started quick and easy, with the ability to scale up to complex applications. 

*homepage*: https://www.palletsprojects.com/p/flask/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.0.2``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_FLEUR_382:

*FLEUR*
+++++++

FLEUR is a feature-full, freely available FLAPW (full potential linearized augmented planewave) code,  based on density-functional theory.

*homepage*: http://www.flapw.de/

=========    ===============
version      toolchain      
=========    ===============
``0.26e``    ``intel/2016a``
=========    ===============


.. _list_software_flex_431:

*flex*
++++++

Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,  sometimes called a tokenizer, is a program which recognizes lexical patterns  in text. 

*homepage*: http://flex.sourceforge.net/

==========    =================================================================================================================================================================================================================================================================================================================================================
version       toolchain                                                                                                                                                                                                                                                                                                                                        
==========    =================================================================================================================================================================================================================================================================================================================================================
``2.5.35``    ``system``                                                                                                                                                                                                                                                                                                                                       
``2.5.38``    ``GCC/4.8.2``                                                                                                                                                                                                                                                                                                                                    
``2.5.39``    ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``system``
``2.6.0``     ``GCC/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``                                                             
``2.6.2``     ``intel/2016b``                                                                                                                                                                                                                                                                                                                                  
``2.6.3``     ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/7.1.0``, ``gimkl/2017a``, ``system``                                                                                                                                                                                                                                                             
``2.6.4``     ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/system``, ``system``                                                                                                  
==========    =================================================================================================================================================================================================================================================================================================================================================


.. _list_software_FlexiDot_799:

*FlexiDot*
++++++++++

Highly customizable, ambiguity-aware dotplots for visual sequence analyses 

*homepage*: https://github.com/molbio-dresden/flexidot

========    ==================    ==============
version     versionsuffix         toolchain     
========    ==================    ==============
``1.06``    ``-Python-2.7.15``    ``foss/2018b``
========    ==================    ==============


.. _list_software_FLINT_381:

*FLINT*
+++++++

FLINT (Fast Library for Number Theory) is a C library in support of computations  in number theory. Operations that can be performed include conversions, arithmetic, computing GCDs,  factoring, solving linear systems, and evaluating special functions. In addition, FLINT provides  various low-level routines for fast arithmetic. FLINT is extensively documented and tested.

*homepage*: http://www.flintlib.org

=========    ==========================================================
version      toolchain                                                 
=========    ==========================================================
``2.5.2``    ``GCC/7.3.0-2.30``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``
=========    ==========================================================


.. _list_software_FLTK_305:

*FLTK*
++++++

FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,  and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL  and its built-in GLUT emulation.

*homepage*: http://www.fltk.org

=========    ===================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                          
=========    ===================================================================================================================================================================================================
``1.3.3``    ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                                                                                                                                   
``1.3.4``    ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``
``1.3.5``    ``GCC/8.2.0-2.31.1``                                                                                                                                                                               
=========    ===================================================================================================================================================================================================


.. _list_software_FLUENT_462:

*FLUENT*
++++++++

ANSYS FLUENT software contains the broad physical modeling capabilities needed to model flow, turbulence, heat transfer, and reactions for industrial applications ranging from air flow over an aircraft wing to combustion in a furnace, from bubble columns to oil platforms, from blood flow to semiconductor manufacturing, and from clean room design to wastewater treatment plants.

*homepage*: http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent

==========    ==========
version       toolchain 
==========    ==========
``14.5``      ``system``
``15.0.7``    ``system``
``16.0``      ``system``
``17.1``      ``system``
``18.0``      ``system``
``18.1``      ``system``
==========    ==========


.. _list_software_Flye_400:

*Flye*
++++++

Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio  and Oxford Nanopore Technologies.

*homepage*: https://github.com/fenderglass/Flye

=======    ==================    ===============
version    versionsuffix         toolchain      
=======    ==================    ===============
``2.4``    ``-Python-2.7.15``    ``intel/2018b``
``2.6``    ``-Python-3.7.2``     ``foss/2019a`` 
=======    ==================    ===============


.. _list_software_FMILibrary_945:

*FMILibrary*
++++++++++++

FMI library is intended as a foundation for applications interfacing  FMUs (Functional Mockup Units) that follow FMI Standard. This version of the library supports  FMI 1.0 and FMI2.0. See http://www.fmi-standard.org/

*homepage*: https://jmodelica.org/

=========    ===============
version      toolchain      
=========    ===============
``2.0.3``    ``intel/2018b``
=========    ===============


.. _list_software_FMRIprep_741:

*FMRIprep*
++++++++++

FMRIprep is a functional magnetic resonance imaging (fMRI) data preprocessing pipeline that is  designed to provide an easily accessible, state-of-the-art interface that is robust to variations in scan  acquisition protocols and that requires minimal user input, while providing easily interpretable and comprehensive  error and output reporting.

*homepage*: https://fmriprep.readthedocs.io

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.1.8``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_fmt_327:

*fmt*
+++++

fmt (formerly cppformat) is an open-source formatting library.

*homepage*: http://fmtlib.net/

=========    ====================================
version      toolchain                           
=========    ====================================
``3.0.1``    ``foss/2016b``, ``intel/2016b``     
``3.0.2``    ``GCCcore/6.4.0``, ``intel/2017a``  
``5.3.0``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=========    ====================================


.. _list_software_FoldX_477:

*FoldX*
+++++++

FoldX is used to provide a fast and quantitative estimation of the importance of the interactions  contributing to the stability of proteins and protein complexes.

*homepage*: http://foldx.crg.es/

===============    ==========
version            toolchain 
===============    ==========
``2.5.2``          ``system``
``3.0-beta5.1``    ``system``
``3.0-beta6``      ``system``
``3.0-beta6.1``    ``system``
===============    ==========


.. _list_software_fontconfig_1069:

*fontconfig*
++++++++++++

Fontconfig is a library designed to provide system-wide font configuration,  customization and application access. 

*homepage*: https://www.freedesktop.org/wiki/Software/fontconfig/

===========    ==================    ======================================================================================
version        versionsuffix         toolchain                                                                             
===========    ==================    ======================================================================================
``2.11.94``                          ``foss/2016a``, ``intel/2016a``                                                       
``2.11.95``                          ``foss/2016a``, ``intel/2016a``                                                       
``2.12.1``                           ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.12.1``     ``-libpng-1.6.29``    ``GCCcore/6.3.0``                                                                     
``2.12.4``                           ``GCCcore/6.4.0``                                                                     
``2.12.6``                           ``GCCcore/6.4.0``                                                                     
``2.13.0``                           ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                                                  
``2.13.1``                           ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                  
===========    ==================    ======================================================================================


.. _list_software_fontsproto_1118:

*fontsproto*
++++++++++++

X11 font extension wire protocol

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``2.1.3``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_foss_443:

*foss*
++++++

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain

===========    ==========
version        toolchain 
===========    ==========
``2016.04``    ``system``
``2016.06``    ``system``
``2016.07``    ``system``
``2016.09``    ``system``
``2016a``      ``system``
``2016b``      ``system``
``2017a``      ``system``
``2017b``      ``system``
``2018.08``    ``system``
``2018a``      ``system``
``2018b``      ``system``
``2019a``      ``system``
``2019b``      ``system``
===========    ==========


.. _list_software_fosscuda_856:

*fosscuda*
++++++++++

GCC based compiler toolchain __with CUDA support__, and including  OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

=========    ==========
version      toolchain 
=========    ==========
``2017b``    ``system``
``2018a``    ``system``
``2018b``    ``system``
``2019a``    ``system``
``2019b``    ``system``
=========    ==========


.. _list_software_FoX_269:

*FoX*
+++++

FoX is an XML library written in Fortran 95. It allows software developers to read, write and modify XML documents from Fortran applications without the complications of dealing with multi-language development.

*homepage*: http://homepages.see.leeds.ac.uk/~earawa/FoX/

=========    ================================
version      toolchain                       
=========    ================================
``4.1.2``    ``intel/2017b``, ``intel/2018a``
=========    ================================


.. _list_software_FPM_227:

*FPM*
+++++

Effing package management! Build packages for multiple platforms (deb, rpm, etc) with great ease  and sanity.

*homepage*: https://github.com/jordansissel/fpm

=========    ===============    ==========
version      versionsuffix      toolchain 
=========    ===============    ==========
``1.3.3``    ``-Ruby-2.1.6``    ``system``
=========    ===============    ==========


.. _list_software_fqtrim_659:

*fqtrim*
++++++++

fqtrim is a versatile stand-alone utility that can be used to trim adapters, poly-A tails,  terminal unknown bases (Ns) and low quality 3' regions in reads from high-throughput next-generation sequencing  machines.

*homepage*: http://ccb.jhu.edu/software/fqtrim/

=========    ===============
version      toolchain      
=========    ===============
``0.9.4``    ``intel/2016b``
``0.9.5``    ``intel/2017a``
=========    ===============


.. _list_software_fqzcomp_768:

*fqzcomp*
+++++++++

Fqzcomp is a basic fastq compressor, designed primarily for high performance.  Despite that it is comparable to bzip2 for compression levels.

*homepage*: http://sourceforge.net/projects/fqzcomp/

=======    =============
version    toolchain    
=======    =============
``4.6``    ``GCC/4.8.2``
=======    =============


.. _list_software_FragGeneScan_1156:

*FragGeneScan*
++++++++++++++

FragGeneScan is an application for finding (fragmented) genes in short reads.

*homepage*: http://omics.informatics.indiana.edu/FragGeneScan/

========    =================================
version     toolchain                        
========    =================================
``1.31``    ``GCCcore/8.2.0``, ``foss/2018b``
========    =================================


.. _list_software_FRANz_417:

*FRANz*
+++++++

A fast and flexible parentage inference program for natural populations.

*homepage*: https://www.bioinf.uni-leipzig.de/Software/FRANz

=========    ==============
version      toolchain     
=========    ==============
``2.0.0``    ``foss/2018a``
=========    ==============


.. _list_software_FreeFem++_752:

*FreeFem++*
+++++++++++

FreeFem++ is a partial differential equation solver. It has its own language. freefem scripts can solve multiphysics non linear systems in 2D and 3D. 

*homepage*: http://www.freefem.org/

==========    ====================    ===============
version       versionsuffix           toolchain      
==========    ====================    ===============
``3.58``      ``-downloaded-deps``    ``foss/2017b`` 
``3.60``      ``-downloaded-deps``    ``intel/2018a``
``3.61-1``    ``-downloaded-deps``    ``intel/2018a``
==========    ====================    ===============


.. _list_software_freeglut_862:

*freeglut*
++++++++++

freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library.

*homepage*: http://freeglut.sourceforge.net/

=========    ================    ======================================================================================================================================================================================
version      versionsuffix       toolchain                                                                                                                                                                             
=========    ================    ======================================================================================================================================================================================
``3.0.0``                        ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.0.0``    ``-Mesa-11.2.1``    ``foss/2016a``, ``intel/2016a``                                                                                                                                                       
=========    ================    ======================================================================================================================================================================================


.. _list_software_FreeImage_869:

*FreeImage*
+++++++++++

FreeImage is an Open Source library project for developers who would like to support popular graphics image formats like PNG, BMP, JPEG, TIFF and others as needed by today's multimedia applications. FreeImage is easy to use, fast, multithreading safe.

*homepage*: http://freeimage.sourceforge.net

==========    =================
version       toolchain        
==========    =================
``3.18.0``    ``GCCcore/7.3.0``
==========    =================


.. _list_software_FreeSurfer_1017:

*FreeSurfer*
++++++++++++

FreeSurfer is a set of tools for analysis and visualization of structural and functional brain imaging data. FreeSurfer contains a fully automatic structural imaging stream for processing cross sectional and longitudinal data.

*homepage*: https://surfer.nmr.mgh.harvard.edu/

=========    ===================    ==========
version      versionsuffix          toolchain 
=========    ===================    ==========
``5.3.0``    ``-centos4_x86_64``    ``system``
``5.3.0``    ``-centos6_x86_64``    ``system``
``6.0.0``    ``-centos6_x86_64``    ``system``
``6.0.1``    ``-centos6_x86_64``    ``system``
=========    ===================    ==========


.. _list_software_freetype_868:

*freetype*
++++++++++

FreeType 2 is a software font engine that is designed to be small, efficient,  highly customizable, and portable while capable of producing high-quality  output (glyph images). It can be used in graphics libraries, display servers,  font conversion tools, text image generation tools, and many other products  as well. 

*homepage*: http://freetype.org

==========    ==================    ======================================================================
version       versionsuffix         toolchain                                                             
==========    ==================    ======================================================================
``2.6.2``                           ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                     
``2.6.3``                           ``foss/2016a``, ``intel/2016a``                                       
``2.6.5``                           ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b`` 
``2.7``                             ``foss/2016b``, ``intel/2016b``                                       
``2.7.1``                           ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``gimkl/2017a``, ``intel/2016b``
``2.7.1``     ``-libpng-1.6.29``    ``GCCcore/6.3.0``                                                     
``2.8``                             ``GCCcore/6.4.0``                                                     
``2.8.1``                           ``GCCcore/6.4.0``                                                     
``2.9``                             ``GCCcore/6.4.0``                                                     
``2.9.1``                           ``GCCcore/7.3.0``, ``GCCcore/8.2.0``                                  
``2.10.1``                          ``GCCcore/8.3.0``                                                     
==========    ==================    ======================================================================


.. _list_software_FreeXL_550:

*FreeXL*
++++++++

FreeXL is an open source library to extract valid data from within an  Excel (.xls) spreadsheet. 

*homepage*: https://www.gaia-gis.it/fossil/freexl/index

=========    ====================================
version      toolchain                           
=========    ====================================
``1.0.2``    ``foss/2016b``, ``intel/2016b``     
``1.0.3``    ``GCCcore/6.4.0``                   
``1.0.5``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=========    ====================================


.. _list_software_FriBidi_665:

*FriBidi*
+++++++++

The Free Implementation of the Unicode Bidirectional Algorithm. 

*homepage*: https://github.com/fribidi/fribidi

=========    =======================================================
version      toolchain                                              
=========    =======================================================
``1.0.1``    ``foss/2018a``, ``intel/2018a``                        
``1.0.2``    ``GCCcore/6.4.0``                                      
``1.0.5``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
=========    =======================================================


.. _list_software_FSL_229:

*FSL*
+++++

FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.

*homepage*: http://www.fmrib.ox.ac.uk/fsl/

==========    ==================    ================================================
version       versionsuffix         toolchain                                       
==========    ==================    ================================================
``5.0.9``                           ``intel/2016a``                                 
``5.0.9``     ``-Mesa-11.2.1``      ``intel/2016a``                                 
``5.0.9``     ``-centos6_64``       ``system``                                      
``5.0.10``                          ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``5.0.11``                          ``foss/2018b``                                  
``5.0.11``    ``-Python-3.6.6``     ``foss/2018b``                                  
``6.0.1``     ``-Python-2.7.15``    ``foss/2019a``                                  
``6.0.1``     ``-Python-3.7.2``     ``foss/2019a``                                  
==========    ==================    ================================================


.. _list_software_FSLeyes_667:

*FSLeyes*
+++++++++

FSLeyes is the FSL image viewer.

*homepage*: https://git.fmrib.ox.ac.uk/fsl/fsleyes/fsleyes

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.15.0``    ``-Python-2.7.13``    ``intel/2017a``
==========    ==================    ===============


.. _list_software_FTGL_301:

*FTGL*
++++++

FTGL is a free open source library to enable developers to use arbitrary fonts in their OpenGL (www.opengl.org) applications. 

*homepage*: http://ftgl.sourceforge.net/docs/html/

=============    ======================================================================================
version          toolchain                                                                             
=============    ======================================================================================
``2.1.3-rc5``    ``GCCcore/8.2.0``, ``foss/2017b``, ``foss/2018a``, ``fosscuda/2018b``, ``intel/2017b``
=============    ======================================================================================


.. _list_software_fullrmc_757:

*fullrmc*
+++++++++

Reverse Monte Carlo (RMC) is probably best known for its applications  in condensed matter physics and solid state chemistry. fullrmc which stands for  FUndamental Library Language for Reverse Monte Carlo is different than traditional  RMC but a stochastic modelling method to solve an inverse problem whereby  an atomic/molecular model is adjusted until its atoms position havei  the greatest consistency with a set of experimental data.

*homepage*: http://bachiraoun.github.io/fullrmc

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.2.0``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_FUNWAVE-TVD_823:

*FUNWAVE-TVD*
+++++++++++++

FUNWAVE–TVD is the TVD version of the fully nonlinear Boussinesq wave model (FUNWAVE)  initially developed by Kirby et al.

*homepage*: https://fengyanshi.github.io

================    =============    ===============
version             versionsuffix    toolchain      
================    =============    ===============
``3.1-20170525``                     ``intel/2017a``
``3.1-20170525``    ``-no-storm``    ``intel/2017a``
================    =============    ===============


.. _list_software_FUSE_307:

*FUSE*
++++++

The reference implementation of the Linux FUSE (Filesystem in Userspace) interface

*homepage*: https://github.com/libfuse/libfuse

=========    ===============
version      toolchain      
=========    ===============
``3.2.6``    ``intel/2018a``
``3.4.1``    ``foss/2018a`` 
=========    ===============


.. _list_software_future_667:

*future*
++++++++

python-future is the missing compatibility layer between Python 2 and Python 3.

*homepage*: http://python-future.org/

==========    ==================    ================================================
version       versionsuffix         toolchain                                       
==========    ==================    ================================================
``0.16.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``0.16.0``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``                 
``0.16.0``    ``-Python-3.6.3``     ``intel/2017b``                                 
``0.16.0``    ``-Python-3.6.6``     ``foss/2018b``                                  
==========    ==================    ================================================


.. _list_software_fxtract_764:

*fxtract*
+++++++++

Extract sequences from a fastx (fasta or fastq) file given a subsequence.

*homepage*: https://github.com/ctSkennerton/fxtract

=======    =================
version    toolchain        
=======    =================
``2.3``    ``GCCcore/8.2.0``
=======    =================


.. _list_software_letter_g:

*G*
---


:ref:`list_software_g2clib_563` - :ref:`list_software_g2lib_464` - :ref:`list_software_g2log_475` - :ref:`list_software_Gaia_370` - :ref:`list_software_GAMESS-US_661` - :ref:`list_software_gap_312` - :ref:`list_software_GapCloser_896` - :ref:`list_software_GapFiller_886` - :ref:`list_software_GAT_220` - :ref:`list_software_GATE_289` - :ref:`list_software_GATK_295` - :ref:`list_software_Gblocks_709` - :ref:`list_software_gbs2ploidy_1023` - :ref:`list_software_gc_202` - :ref:`list_software_GC3Pie_475` - :ref:`list_software_GCC_205` - :ref:`list_software_GCCcore_630` - :ref:`list_software_gcccuda_714` - :ref:`list_software_GConf_461` - :ref:`list_software_Gctf_388` - :ref:`list_software_GD_139` - :ref:`list_software_GDAL_280` - :ref:`list_software_GDB_205` - :ref:`list_software_gdbgui_626` - :ref:`list_software_gdc-client_986` - :ref:`list_software_GDCHART_509` - :ref:`list_software_GDCM_283` - :ref:`list_software_GDGraph_637` - :ref:`list_software_gdist_539` - :ref:`list_software_Gdk-Pixbuf_945` - :ref:`list_software_Geant4_547` - :ref:`list_software_gearshifft_1059` - :ref:`list_software_GEM-library_1019` - :ref:`list_software_GEMMA_359` - :ref:`list_software_gencore_variant_detection_2645` - :ref:`list_software_GeneMark-ET_976` - :ref:`list_software_GenomeMapper_1216` - :ref:`list_software_GenomeTester4_1286` - :ref:`list_software_GenomeThreader_1418` - :ref:`list_software_GenotypeHarmonizer_1898` - :ref:`list_software_gensim_643` - :ref:`list_software_geopy_548` - :ref:`list_software_GEOS_302` - :ref:`list_software_Gerris_620` - :ref:`list_software_GETORB_451` - :ref:`list_software_gettext_773` - :ref:`list_software_GffCompare_986` - :ref:`list_software_gffread_719` - :ref:`list_software_gflags_628` - :ref:`list_software_GFOLD_364` - :ref:`list_software_GHC_210` - :ref:`list_software_Ghostscript_1178` - :ref:`list_software_giflib_621` - :ref:`list_software_gifsicle_838` - :ref:`list_software_GIMIC_361` - :ref:`list_software_gimkl_532` - :ref:`list_software_gimpi_534` - :ref:`list_software_gimpic_633` - :ref:`list_software_GIMPS_384` - :ref:`list_software_giolf_529` - :ref:`list_software_giolfc_628` - :ref:`list_software_git_324` - :ref:`list_software_git-lfs_694` - :ref:`list_software_GitPython_934` - :ref:`list_software_Givaro_616` - :ref:`list_software_Giza_395` - :ref:`list_software_GL2PS_360` - :ref:`list_software_Glade_477` - :ref:`list_software_glew_431` - :ref:`list_software_GLib_350` - :ref:`list_software_glibc_513` - :ref:`list_software_GLibmm_568` - :ref:`list_software_GLIMMER_525` - :ref:`list_software_GlimmerHMM_943` - :ref:`list_software_GLM_224` - :ref:`list_software_GlobalArrays_1219` - :ref:`list_software_GlobusConnectPersonal_2170` - :ref:`list_software_glog_425` - :ref:`list_software_GLPK_302` - :ref:`list_software_glproto_775` - :ref:`list_software_GMAP-GSNAP_715` - :ref:`list_software_GMP_228` - :ref:`list_software_gmpich_632` - :ref:`list_software_gmpolf_645` - :ref:`list_software_gmpy2_495` - :ref:`list_software_gmsh_431` - :ref:`list_software_GMT_232` - :ref:`list_software_gmvapich2_897` - :ref:`list_software_gmvolf_651` - :ref:`list_software_GNU_234` - :ref:`list_software_gnuplot_777` - :ref:`list_software_GnuTLS_541` - :ref:`list_software_Go_182` - :ref:`list_software_GObject-Introspection_2108` - :ref:`list_software_golf_424` - :ref:`list_software_gomkl_538` - :ref:`list_software_gompi_540` - :ref:`list_software_gompic_639` - :ref:`list_software_google-java-format_1794` - :ref:`list_software_googletest_1085` - :ref:`list_software_GP2C_268` - :ref:`list_software_GPAW_303` - :ref:`list_software_GPAW-setups_1024` - :ref:`list_software_gperf_532` - :ref:`list_software_gperftools_1093` - :ref:`list_software_gpustat_776` - :ref:`list_software_Grace_482` - :ref:`list_software_grace_514` - :ref:`list_software_Gradle_591` - :ref:`list_software_graph-tool_1021` - :ref:`list_software_Graphene_810` - :ref:`list_software_GraphicsMagick_1405` - :ref:`list_software_Graphviz_843` - :ref:`list_software_graphviz_875` - :ref:`list_software_GRASS_384` - :ref:`list_software_Greenlet_822` - :ref:`list_software_Grep_398` - :ref:`list_software_grib_api_829` - :ref:`list_software_GRIT_310` - :ref:`list_software_GRNBoost_750` - :ref:`list_software_GroIMP_526` - :ref:`list_software_GROMACS_524` - :ref:`list_software_GromacsWrapper_1453` - :ref:`list_software_GroopM_596` - :ref:`list_software_GSL_230` - :ref:`list_software_gSOAP_410` - :ref:`list_software_GST-plugins-base_1509` - :ref:`list_software_GStreamer_906` - :ref:`list_software_GTDB-Tk_525` - :ref:`list_software_gtest_551` - :ref:`list_software_GTK+_273` - :ref:`list_software_GTOOL_389` - :ref:`list_software_GTS_238` - :ref:`list_software_guenomu_768` - :ref:`list_software_Guile_502` - :ref:`list_software_GULP_312` - :ref:`list_software_Gurobi_616` - :ref:`list_software_gzip_442`


.. _list_software_g2clib_563:

*g2clib*
++++++++

Library contains GRIB2 encoder/decoder ('C' version).

*homepage*: http://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/

=========    ==================================================================
version      toolchain                                                         
=========    ==================================================================
``1.6.0``    ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
=========    ==================================================================


.. _list_software_g2lib_464:

*g2lib*
+++++++

Library contains GRIB2 encoder/decoder and search/indexing routines.

*homepage*: http://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.4.0``    ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.1.0``    ``intel/2018b``                                  
=========    =================================================


.. _list_software_g2log_475:

*g2log*
+++++++

g2log, efficient asynchronous logger using C++11

*homepage*: https://sites.google.com/site/kjellhedstrom2//g2log-efficient-background-io-processign-with-c11

=======    ==============
version    toolchain     
=======    ==============
``1.0``    ``foss/2016b``
=======    ==============


.. _list_software_Gaia_370:

*Gaia*
++++++

Gaia is a C++ library with python bindings which implements similarity measures and classiﬁcations  on the results of audio analysis, and generates classiﬁcation models that Essentia can use to compute high-level  description of music.

*homepage*: https://github.com/MTG/gaia

=========    ==================    =================
version      versionsuffix         toolchain        
=========    ==================    =================
``2.4.5``    ``-Python-2.7.15``    ``GCCcore/8.2.0``
=========    ==================    =================


.. _list_software_GAMESS-US_661:

*GAMESS-US*
+++++++++++

The General Atomic and Molecular Electronic Structure System (GAMESS)  is a general ab initio quantum chemistry package. 

*homepage*: http://www.msg.chem.iastate.edu/gamess/index.html

===============    =============    ===============
version            versionsuffix    toolchain      
===============    =============    ===============
``20141205-R1``                     ``intel/2016a``
``20170420-R1``                     ``intel/2016b``
``20170420-R1``    ``-sockets``     ``intel/2016b``
``20180214-R1``                     ``foss/2016b`` 
``20180214-R1``    ``-sockets``     ``foss/2016b`` 
===============    =============    ===============


.. _list_software_gap_312:

*gap*
+++++

GAP is a system for computational discrete algebra, with particular emphasis on Computational Group Theory.

*homepage*: https://www.gap-system.org

=========    ===============
version      toolchain      
=========    ===============
``4.9.3``    ``intel/2018b``
=========    ===============


.. _list_software_GapCloser_896:

*GapCloser*
+++++++++++

GapCloser is designed to close the gaps emerging during the scaffolding process  by SOAPdenovo or other assembler, using the abundant pair relationships of short reads.

*homepage*: https://sourceforge.net/projects/soapdenovo2/files/GapCloser/

===========    ===============================
version        toolchain                      
===========    ===============================
``1.12-r6``    ``foss/2018a``, ``intel/2017b``
===========    ===============================


.. _list_software_GapFiller_886:

*GapFiller*
+++++++++++

GapFiller is a seed-and-extend local assembler to fill the gap within paired reads.  It can be used for both DNA and RNA and it has been tested on Illumina data.

*homepage*: https://sourceforge.net/projects/gapfiller

=========    ===============
version      toolchain      
=========    ===============
``2.1.1``    ``intel/2017a``
=========    ===============


.. _list_software_GAT_220:

*GAT*
+++++

The Genomic Association Tester (GAT) is a tool for computing the significance of overlap between   multiple sets of genomic intervals. GAT estimates significance based on simulation.

*homepage*: https://gat.readthedocs.io/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.2.2``    ``-Python-2.7.11``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_GATE_289:

*GATE*
++++++

GATE is an advanced opensource software developed by the international OpenGATE collaboration and  dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography  (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography

*homepage*: http://www.opengatecollaboration.org/

===========    ===============================    ===============================
version        versionsuffix                      toolchain                      
===========    ===============================    ===============================
``6.2``        ``-Python-2.7.11``                 ``intel/2016a``                
``7.1``        ``-Python-2.7.11``                 ``intel/2016a``                
``7.2``        ``-Python-2.7.11``                 ``intel/2016a``                
``8.0``        ``-Python-2.7.14``                 ``foss/2017b``, ``intel/2017b``
``8.0``        ``-Python-2.7.14-Geant4-10.04``    ``intel/2017b``                
``8.1.p01``    ``-Python-2.7.15``                 ``foss/2018b``                 
``8.2``        ``-Python-2.7.14``                 ``foss/2017b``, ``intel/2017b``
``8.2``        ``-Python-2.7.15``                 ``foss/2018b``                 
===========    ===============================    ===============================


.. _list_software_GATK_295:

*GATK*
++++++

The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute  to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,  with a primary focus on variant discovery and genotyping as well as strong emphasis on  data quality assurance. Its robust architecture, powerful processing engine and  high-performance computing features make it capable of taking on projects of any size.

*homepage*: https://www.broadinstitute.org/gatk/

============    ===================    =================
version         versionsuffix          toolchain        
============    ===================    =================
``1.0.5083``                           ``system``       
``2.5-2``       ``-Java-1.7.0_10``     ``system``       
``2.6-5``       ``-Java-1.7.0_10``     ``system``       
``2.7-4``                              ``system``       
``2.7-4``       ``-Java-1.7.0_10``     ``system``       
``2.8-1``       ``-Java-1.7.0_10``     ``system``       
``3.0-0``       ``-Java-1.7.0_10``     ``system``       
``3.3-0``       ``-Java-1.7.0_21``     ``system``       
``3.3-0``       ``-Java-1.7.0_80``     ``system``       
``3.3-0``       ``-Java-1.8.0_66``     ``system``       
``3.5``         ``-Java-1.8.0_66``     ``system``       
``3.5``         ``-Java-1.8.0_74``     ``system``       
``3.6``         ``-Java-1.8.0_92``     ``system``       
``3.7``         ``-Java-1.8.0_112``    ``system``       
``3.8-0``       ``-Java-1.8.0_144``    ``system``       
``4.0.1.2``     ``-Java-1.8``          ``system``       
``4.0.4.0``     ``-Python-2.7.14``     ``intel/2018a``  
``4.0.4.0``     ``-Python-3.6.4``      ``intel/2018a``  
``4.0.5.1``     ``-Python-3.6.4``      ``foss/2018a``   
``4.0.7.0``     ``-Python-2.7.14``     ``intel/2018a``  
``4.0.7.0``     ``-Python-3.6.4``      ``intel/2018a``  
``4.0.8.1``     ``-Python-2.7.15``     ``foss/2018b``   
``4.0.8.1``     ``-Python-3.6.6``      ``foss/2018b``   
``4.0.10.0``    ``-Python-3.6.6``      ``foss/2018b``   
``4.0.12.0``    ``-Python-3.6.6``      ``foss/2018b``   
``4.1.0.0``     ``-Python-3.6.6``      ``foss/2018b``   
``4.1.2.0``     ``-Java-1.8``          ``GCCcore/8.2.0``
``4.1.3.0``     ``-Java-1.8``          ``GCCcore/8.3.0``
============    ===================    =================


.. _list_software_Gblocks_709:

*Gblocks*
+++++++++

Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis

*homepage*: http://molevol.cmima.csic.es/castresana/Gblocks.html

=========    ==========
version      toolchain 
=========    ==========
``0.91b``    ``system``
=========    ==========


.. _list_software_gbs2ploidy_1023:

*gbs2ploidy*
++++++++++++

Inference of Ploidy from (Genotyping-by-Sequencing) GBS Data

*homepage*: https://cran.r-project.org/web/packages/gbs2ploidy

=======    =============    ===============
version    versionsuffix    toolchain      
=======    =============    ===============
``1.0``    ``-R-3.4.3``     ``intel/2017b``
=======    =============    ===============


.. _list_software_gc_202:

*gc*
++++

The Boehm-Demers-Weiser conservative garbage collector can be used as a  garbage collecting replacement for C malloc or C++ new. 

*homepage*: http://hboehm.info/gc/

==========    ==================================
version       toolchain                         
==========    ==================================
``7.4.4``     ``GCC/4.9.3-2.25``, ``foss/2016a``
``7.6.0``     ``GCCcore/6.4.0``                 
``7.6.4``     ``GCCcore/7.3.0``                 
``7.6.10``    ``GCCcore/8.2.0``                 
==========    ==================================


.. _list_software_GC3Pie_475:

*GC3Pie*
++++++++

GC3Pie is a Python package for running large job campaigns on diverse batch-oriented execution  environments.

*homepage*: https://gc3pie.readthedocs.org

=========    ==========
version      toolchain 
=========    ==========
``2.4.2``    ``system``
``2.5.0``    ``system``
``2.5.2``    ``system``
=========    ==========


.. _list_software_GCC_205:

*GCC*
+++++

The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).

*homepage*: http://gcc.gnu.org/

==========    ===================    ==========
version       versionsuffix          toolchain 
==========    ===================    ==========
``4.8.1``                            ``system``
``4.8.1``     ``-CLooG``             ``system``
``4.8.2``                            ``system``
``4.8.2``     ``-CLooG``             ``system``
``4.8.2``     ``-CLooG-multilib``    ``system``
``4.8.2``     ``-multilib``          ``system``
``4.8.3``                            ``system``
``4.8.3``     ``-CLooG-multilib``    ``system``
``4.8.4``                            ``system``
``4.8.4``     ``-CLooG``             ``system``
``4.8.4``     ``-CLooG-multilib``    ``system``
``4.8.5``                            ``system``
``4.9.0``                            ``system``
``4.9.0``     ``-CLooG``             ``system``
``4.9.0``     ``-CLooG-multilib``    ``system``
``4.9.1``                            ``system``
``4.9.1``     ``-CLooG``             ``system``
``4.9.1``     ``-CLooG-multilib``    ``system``
``4.9.2``                            ``system``
``4.9.2``     ``-CLooG``             ``system``
``4.9.2``     ``-CLooG-multilib``    ``system``
``4.9.2``     ``-binutils-2.25``     ``system``
``4.9.3``                            ``system``
``4.9.3``     ``-2.25``              ``system``
``4.9.3``     ``-binutils-2.25``     ``system``
``4.9.4``     ``-2.25``              ``system``
``5.1.0``                            ``system``
``5.1.0``     ``-binutils-2.25``     ``system``
``5.2.0``                            ``system``
``5.3.0``                            ``system``
``5.3.0``     ``-2.26``              ``system``
``5.4.0``     ``-2.26``              ``system``
``5.5.0``     ``-2.26``              ``system``
``6.1.0``     ``-2.27``              ``system``
``6.2.0``     ``-2.27``              ``system``
``6.3.0``     ``-2.27``              ``system``
``6.3.0``     ``-2.28``              ``system``
``6.4.0``     ``-2.28``              ``system``
``7.1.0``     ``-2.28``              ``system``
``7.2.0``     ``-2.29``              ``system``
``7.3.0``     ``-2.30``              ``system``
``7.4.0``     ``-2.31.1``            ``system``
``8.1.0``     ``-2.30``              ``system``
``8.2.0``     ``-2.31.1``            ``system``
``8.3.0``                            ``system``
``8.3.0``     ``-2.32``              ``system``
``9.1.0``     ``-2.32``              ``system``
``9.2.0``     ``-2.32``              ``system``
``system``                           ``system``
``system``    ``-2.29``              ``system``
==========    ===================    ==========


.. _list_software_GCCcore_630:

*GCCcore*
+++++++++

The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,  as well as libraries for these languages (libstdc++, libgcj,...).

*homepage*: http://gcc.gnu.org/

==========    ==========
version       toolchain 
==========    ==========
``4.9.2``     ``system``
``4.9.3``     ``system``
``4.9.4``     ``system``
``5.3.0``     ``system``
``5.4.0``     ``system``
``5.5.0``     ``system``
``6.1.0``     ``system``
``6.2.0``     ``system``
``6.3.0``     ``system``
``6.4.0``     ``system``
``6.5.0``     ``system``
``7.1.0``     ``system``
``7.2.0``     ``system``
``7.3.0``     ``system``
``7.4.0``     ``system``
``8.1.0``     ``system``
``8.2.0``     ``system``
``8.3.0``     ``system``
``9.1.0``     ``system``
``9.2.0``     ``system``
``system``    ``system``
==========    ==========


.. _list_software_gcccuda_714:

*gcccuda*
+++++++++

GNU Compiler Collection (GCC) based compiler toolchain, along with CUDA toolkit.

*homepage*: (none)

===========    ==========
version        toolchain 
===========    ==========
``2016.08``    ``system``
``2017b``      ``system``
``2018a``      ``system``
``2018b``      ``system``
``2019a``      ``system``
``2019b``      ``system``
===========    ==========


.. _list_software_GConf_461:

*GConf*
+++++++

GConf is a system for storing application preferences.  It is intended for user preferences; not configuration  of something like Apache, or arbitrary data storage.

*homepage*: https://projects.gnome.org/gconf/

=========    =====================================================================================================================
version      toolchain                                                                                                            
=========    =====================================================================================================================
``3.2.6``    ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
=========    =====================================================================================================================


.. _list_software_Gctf_388:

*Gctf*
++++++

Gctf: real-time CTF determination and correction, Kai Zhang, 2016

*homepage*: https://www.mrc-lmb.cam.ac.uk/kzhang/

========    ==========
version     toolchain 
========    ==========
``1.06``    ``system``
========    ==========


.. _list_software_GD_139:

*GD*
++++

GD.pm - Interface to Gd Graphics Library

*homepage*: https://github.com/lstein/Perl-GD

========    ================    =================
version     versionsuffix       toolchain        
========    ================    =================
``2.66``    ``-Perl-5.24.0``    ``foss/2016b``   
``2.68``    ``-Perl-5.26.1``    ``GCCcore/6.4.0``
``2.69``    ``-Perl-5.28.0``    ``GCCcore/7.3.0``
========    ================    =================


.. _list_software_GDAL_280:

*GDAL*
++++++

GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style  Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model  to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for  data translation and processing.

*homepage*: https://www.gdal.org

=========    ==============================    ================================================================
version      versionsuffix                     toolchain                                                       
=========    ==============================    ================================================================
``2.0.2``                                      ``foss/2016a``, ``intel/2016a``                                 
``2.1.0``                                      ``foss/2016a``, ``foss/2016b``, ``intel/2016b``                 
``2.1.1``                                      ``foss/2016a``                                                  
``2.1.1``    ``-Python-2.7.12``                ``intel/2016b``                                                 
``2.1.2``    ``-Python-2.7.12``                ``intel/2016b``                                                 
``2.1.3``    ``-Python-2.7.12``                ``foss/2016b``, ``intel/2016b``                                 
``2.1.3``    ``-Python-2.7.13``                ``intel/2017a``                                                 
``2.1.3``    ``-Python-3.6.1``                 ``intel/2017a``                                                 
``2.2.0``    ``-Python-2.7.13-HDF5-1.8.18``    ``intel/2017a``                                                 
``2.2.0``    ``-Python-2.7.13-HDF5-HDF``       ``intel/2017a``                                                 
``2.2.0``    ``-Python-3.6.1``                 ``intel/2017a``                                                 
``2.2.2``    ``-Python-2.7.14``                ``intel/2017b``                                                 
``2.2.2``    ``-Python-2.7.14-HDF5-1.8.19``    ``intel/2017b``                                                 
``2.2.3``    ``-Python-2.7.14``                ``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``2.2.3``    ``-Python-2.7.15``                ``foss/2018b``                                                  
``2.2.3``    ``-Python-3.6.3``                 ``foss/2017b``                                                  
``2.2.3``    ``-Python-3.6.4``                 ``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``                
``2.2.3``    ``-Python-3.6.6``                 ``foss/2018b``, ``intel/2018b``                                 
``3.0.0``    ``-Python-2.7.15``                ``foss/2019a``                                                  
``3.0.0``    ``-Python-3.7.2``                 ``foss/2019a``, ``intel/2019a``                                 
=========    ==============================    ================================================================


.. _list_software_GDB_205:

*GDB*
+++++

The GNU Project Debugger

*homepage*: http://www.gnu.org/software/gdb/gdb.html

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``7.8.2``                           ``GCC/4.9.2``                  
``7.9``                             ``GCC/4.9.2``                  
``7.10.1``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``7.11``      ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``7.11.1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``8.0.1``     ``-Python-2.7.14``    ``foss/2017b``                 
``8.0.1``     ``-Python-3.6.3``     ``foss/2017b``                 
``8.1``       ``-Python-2.7.14``    ``foss/2018a``                 
``8.1.1``     ``-Python-2.7.14``    ``intel/2018a``                
``8.3``       ``-Python-3.7.2``     ``GCCcore/8.2.0``              
==========    ==================    ===============================


.. _list_software_gdbgui_626:

*gdbgui*
++++++++

Browser-based frontend to gdb (gnu debugger). Add breakpoints, view the stack, visualize data structures, and more in C, C++, Go, Rust, and Fortran. Run gdbgui from the terminal and a new tab will open in your browser.

*homepage*: https://gdbgui.com

============    =================    =================
version         versionsuffix        toolchain        
============    =================    =================
``0.13.1.2``    ``-Python-3.7.2``    ``GCCcore/8.2.0``
============    =================    =================


.. _list_software_gdc-client_986:

*gdc-client*
++++++++++++

The gdc-client provides several convenience functions over the GDC API which provides general  download/upload via HTTPS.

*homepage*: https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.0.1``    ``-Python-2.7.12``    ``intel/2016b``                
``1.3.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2017b``
``1.3.0``    ``-Python-2.7.15``    ``foss/2018b``                 
=========    ==================    ===============================


.. _list_software_GDCHART_509:

*GDCHART*
+++++++++

Easy to use C API, high performance library to create charts  and graphs in PNG, GIF and WBMP format.

*homepage*: http://users.fred.net/brv/chart

=============    =================
version          toolchain        
=============    =================
``0.11.5dev``    ``GCCcore/8.2.0``
=============    =================


.. _list_software_GDCM_283:

*GDCM*
++++++

Grassroots DICOM: Cross-platform DICOM implementation

*homepage*: https://sourceforge.net/projects/gdcm

=========    =================
version      toolchain        
=========    =================
``2.8.8``    ``GCCcore/6.4.0``
``2.8.9``    ``GCCcore/7.3.0``
``3.0.4``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_GDGraph_637:

*GDGraph*
+++++++++

GDGraph is a Perl package to generate charts

*homepage*: https://metacpan.org/release/GDGraph

========    ================    ===============
version     versionsuffix       toolchain      
========    ================    ===============
``1.54``    ``-Perl-5.26.1``    ``intel/2018a``
``1.54``    ``-Perl-5.28.0``    ``foss/2018b`` 
========    ================    ===============


.. _list_software_gdist_539:

*gdist*
+++++++

The gdist module is a Cython interface to a C++ library (http://code.google.com/p/geodesic/) for  computing geodesic distance which is the length of shortest line between two vertices on a triangulated mesh in three  dimensions, such that the line lies on the surface.

*homepage*: https://pypi.python.org/pypi/gdist

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.0.3``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_Gdk-Pixbuf_945:

*Gdk-Pixbuf*
++++++++++++

The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.  It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it  was distributed as part of GTK+ 2 but it was split off into a separate package  in preparation for the change to GTK+ 3. 

*homepage*: https://developer.gnome.org/gdk-pixbuf/stable/

===========    ====================================================================================
version        toolchain                                                                           
===========    ====================================================================================
``2.32.3``     ``intel/2016a``                                                                     
``2.35.1``     ``foss/2016a``, ``intel/2016a``                                                     
``2.36.0``     ``foss/2016b``, ``intel/2016b``                                                     
``2.36.8``     ``intel/2017a``                                                                     
``2.36.10``    ``intel/2017a``                                                                     
``2.36.11``    ``foss/2017b``, ``foss/2018a``, ``fosscuda/2018b``, ``intel/2017b``, ``intel/2018a``
``2.36.12``    ``foss/2018b``, ``fosscuda/2018b``                                                  
``2.38.1``     ``GCCcore/8.2.0``                                                                   
===========    ====================================================================================


.. _list_software_Geant4_547:

*Geant4*
++++++++

Geant4 is a toolkit for the simulation of the passage of particles through matter.  Its areas of application include high energy, nuclear and accelerator physics,   as well as studies in medical and space science.

*homepage*: http://geant4.cern.ch/

=============    ================================================================
version          toolchain                                                       
=============    ================================================================
``9.5.p02``      ``intel/2016a``                                                 
``9.6.p04``      ``intel/2016a``                                                 
``10.01.p03``    ``intel/2016a``                                                 
``10.02.p01``    ``intel/2016a``                                                 
``10.03.p03``    ``foss/2017b``, ``intel/2017b``                                 
``10.04``        ``intel/2017b``                                                 
``10.5``         ``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018b``
=============    ================================================================


.. _list_software_gearshifft_1059:

*gearshifft*
++++++++++++

Benchmark Suite for Heterogenuous FFT Implementations

*homepage*: https://github.com/mpicbg-scicomp/gearshifft

=========    ==============
version      toolchain     
=========    ==============
``0.4.0``    ``foss/2019a``
=========    ==============


.. _list_software_GEM-library_1019:

*GEM-library*
+++++++++++++

Next-generation sequencing platforms (Illumina/Solexa, ABI/SOLiD, etc.)   call for powerful and very optimized tools to index/analyze huge genomes. The GEM library   strives to be a true "next-generation" tool for handling any kind of sequence data, offering   state-of-the-art algorithms and data structures specifically tailored to this demanding task.   At the moment, efficient indexing and searching algorithms based on the Burrows-Wheeler   transform (BWT) have been implemented. The library core is written in C for maximum speed,   with concise interfaces to higher-level programming languages like OCaml and Python.   Many high-performance standalone programs (mapper, splice mapper, etc.) are provided along   with the library; in general, new algorithms and tools can be easily implemented on the top of it. 

*homepage*: http://algorithms.cnag.cat/wiki/The_GEM_library

===================    ===============================    ==========
version                versionsuffix                      toolchain 
===================    ===============================    ==========
``20130406-045632``    ``_pre-release-3_Linux-x86_64``    ``system``
===================    ===============================    ==========


.. _list_software_GEMMA_359:

*GEMMA*
+++++++

Genome-wide Efficient Mixed Model Association

*homepage*: https://github.com/genetics-statistics/GEMMA

==========    ==============================
version       toolchain                     
==========    ==============================
``0.97``      ``foss/2016b``, ``foss/2017a``
``0.98.1``    ``foss/2018b``                
==========    ==============================


.. _list_software_gencore_variant_detection_2645:

*gencore_variant_detection*
+++++++++++++++++++++++++++

This is a bundled install of many software packages for doing variant detection analysis. 

*homepage*: https://nyuad-cgsb.github.io/variant_detection/public/index.html

=======    ==========
version    toolchain 
=======    ==========
``1.0``    ``system``
=======    ==========


.. _list_software_GeneMark-ET_976:

*GeneMark-ET*
+++++++++++++

Eukaryotic gene prediction suite with automatic training

*homepage*: http://exon.gatech.edu/GeneMark

========    =================
version     toolchain        
========    =================
``4.38``    ``GCCcore/8.2.0``
========    =================


.. _list_software_GenomeMapper_1216:

*GenomeMapper*
++++++++++++++

GenomeMapper is a short read mapping tool designed for accurate read alignments.   It quickly aligns millions of reads either with ungapped or gapped alignments.   This version is used to align against a single reference.   If you are unsure which one is the appropriate GenomeMapper, you might want to use this one.

*homepage*: https://1001genomes.org/software/genomemapper_singleref.html

=========    ==============
version      toolchain     
=========    ==============
``0.4.4``    ``foss/2016a``
=========    ==============


.. _list_software_GenomeTester4_1286:

*GenomeTester4*
+++++++++++++++

A toolkit for performing set operations - union, intersection and complement - on k-mer lists.

*homepage*: https://github.com/bioinfo-ut/GenomeTester4

=======    ===============
version    toolchain      
=======    ===============
``4.0``    ``intel/2018a``
=======    ===============


.. _list_software_GenomeThreader_1418:

*GenomeThreader*
++++++++++++++++

GenomeThreader is a software tool to compute gene structure predictions.

*homepage*: http://genomethreader.org

=========    =======================    ==========
version      versionsuffix              toolchain 
=========    =======================    ==========
``1.7.1``    ``-Linux_x86_64-64bit``    ``system``
=========    =======================    ==========


.. _list_software_GenotypeHarmonizer_1898:

*GenotypeHarmonizer*
++++++++++++++++++++

The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data  stored using different file formats with different and potentially unknown strands.

*homepage*: https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer

==========    ==================    ==========
version       versionsuffix         toolchain 
==========    ==================    ==========
``1.4.14``    ``-Java-1.7.0_80``    ``system``
==========    ==================    ==========


.. _list_software_gensim_643:

*gensim*
++++++++

Gensim is a Python library for topic modelling, document indexing and similarity retrieval with  large corpora.

*homepage*: https://pypi.python.org/pypi/gensim

==========    ==================    ==============
version       versionsuffix         toolchain     
==========    ==================    ==============
``0.13.2``    ``-Python-2.7.11``    ``foss/2016a``
==========    ==================    ==============


.. _list_software_geopy_548:

*geopy*
+++++++

geopy is a Python 2 and 3 client for several popular geocoding web services.

*homepage*: https://github.com/geopy/geopy

==========    =================    ===============
version       versionsuffix        toolchain      
==========    =================    ===============
``1.11.0``    ``-Python-3.6.1``    ``intel/2017a``
==========    =================    ===============


.. _list_software_GEOS_302:

*GEOS*
++++++

GEOS (Geometry Engine - Open Source) is a C++ port of the  Java Topology Suite (JTS)

*homepage*: https://trac.osgeo.org/geos

=========    ==================    ================================================================
version      versionsuffix         toolchain                                                       
=========    ==================    ================================================================
``3.5.0``    ``-Python-2.7.11``    ``intel/2016a``                                                 
``3.5.0``    ``-Python-2.7.12``    ``intel/2016b``                                                 
``3.6.1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                 
``3.6.1``    ``-Python-2.7.13``    ``intel/2017a``                                                 
``3.6.1``    ``-Python-3.6.1``     ``intel/2017a``                                                 
``3.6.2``    ``-Python-2.7.14``    ``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``3.6.2``    ``-Python-2.7.15``    ``foss/2018b``                                                  
``3.6.2``    ``-Python-3.6.2``     ``foss/2017b``                                                  
``3.6.2``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``, ``intel/2018.01``              
``3.6.2``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``                
``3.6.2``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                                 
``3.7.2``    ``-Python-2.7.15``    ``foss/2019a``                                                  
``3.7.2``    ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``                                 
=========    ==================    ================================================================


.. _list_software_Gerris_620:

*Gerris*
++++++++

Gerris is a Free Software program for the solution of the partial                  differential equations describing fluid flow

*homepage*: http://gfs.sourceforge.net/wiki/index.php/Main_Page

============    ==============
version         toolchain     
============    ==============
``20131206``    ``foss/2017b``
============    ==============


.. _list_software_GETORB_451:

*GETORB*
++++++++

GETORB software package contains programs to handle the orbital data records (ODRs)

*homepage*: http://www.deos.tudelft.nl/ers/precorbs/tools/getorb_pack.shtml

=========    ===============
version      toolchain      
=========    ===============
``2.3.2``    ``intel/2017a``
=========    ===============


.. _list_software_gettext_773:

*gettext*
+++++++++

GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools and documentation

*homepage*: https://www.gnu.org/software/gettext/

============    ==================    ===================================================================================================
version         versionsuffix         toolchain                                                                                          
============    ==================    ===================================================================================================
``0.18.2``                            ``system``                                                                                         
``0.19.4``                            ``GCC/4.9.2``, ``system``                                                                          
``0.19.6``                            ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``system``                  
``0.19.7``                            ``foss/2016a``, ``intel/2016a``, ``system``                                                        
``0.19.8``                            ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016.04``, ``foss/2016b``, ``intel/2016b``, ``system``
``0.19.8.1``                          ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``system``             
``0.19.8.1``    ``-libxml2-2.9.7``    ``GCCcore/6.4.0``                                                                                  
``0.20.1``                            ``GCCcore/8.3.0``                                                                                  
============    ==================    ===================================================================================================


.. _list_software_GffCompare_986:

*GffCompare*
++++++++++++

GffCompare provides classification and reference annotation mapping and  matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files.

*homepage*: https://github.com/gpertea/gffcompare

==========    =================
version       toolchain        
==========    =================
``0.10.1``    ``foss/2016b``   
``0.10.6``    ``GCCcore/7.3.0``
==========    =================


.. _list_software_gffread_719:

*gffread*
+++++++++

GFF/GTF parsing utility providing format conversions, region filtering, FASTA sequence extraction and  more.

*homepage*: https://github.com/gpertea/gffread

==========    =================
version       toolchain        
==========    =================
``0.9.12``    ``foss/2016b``   
``0.10.6``    ``GCCcore/7.3.0``
==========    =================


.. _list_software_gflags_628:

*gflags*
++++++++

The gflags package contains a C++ library that implements commandline flags processing.  It includes built-in support for standard types such as string and the ability to define flags in the source file in which they are used. 

*homepage*: https://github.com/gflags/gflags

=========    ===================================================
version      toolchain                                          
=========    ===================================================
``2.1.2``    ``foss/2016a``                                     
``2.2.1``    ``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``2.2.2``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``               
=========    ===================================================


.. _list_software_GFOLD_364:

*GFOLD*
+++++++

Generalized fold change for ranking differentially expressed genes from RNA-seq data

*homepage*: http://www.tongji.edu.cn/~zhanglab/GFOLD/index.html

=========    ===============
version      toolchain      
=========    ===============
``1.1.4``    ``intel/2016a``
=========    ===============


.. _list_software_GHC_210:

*GHC*
+++++

The Glorious/Glasgow Haskell Compiler

*homepage*: http://haskell.org/ghc/

==========    ==========
version       toolchain 
==========    ==========
``6.12.3``    ``system``
==========    ==========


.. _list_software_Ghostscript_1178:

*Ghostscript*
+++++++++++++

Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to  different targets. It used to be part of the cups printing stack, but is no longer used for that.

*homepage*: http://ghostscript.com

========    ==================    ==================================================
version     versionsuffix         toolchain                                         
========    ==================    ==================================================
``9.19``                          ``intel/2016a``, ``intel/2016b``                  
``9.20``                          ``foss/2016b``, ``intel/2016b``                   
``9.21``                          ``intel/2017a``                                   
``9.22``                          ``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``
``9.22``    ``-cairo-1.14.12``    ``GCCcore/6.4.0``                                 
``9.23``                          ``GCCcore/6.4.0``, ``GCCcore/7.3.0``              
``9.23``    ``-cairo-1.14.12``    ``GCCcore/6.4.0``                                 
``9.27``                          ``GCCcore/8.2.0``                                 
========    ==================    ==================================================


.. _list_software_giflib_621:

*giflib*
++++++++

giflib is a library for reading and writing gif images. It is API and ABI compatible with libungif which was in wide use while the LZW compression algorithm was patented.

*homepage*: http://libungif.sourceforge.net/

=========    ====================================
version      toolchain                           
=========    ====================================
``5.1.4``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=========    ====================================


.. _list_software_gifsicle_838:

*gifsicle*
++++++++++

Gifsicle is a command-line tool for creating, editing,  and getting information about GIF images and animations.   Making a GIF animation with gifsicle is easy.

*homepage*: https://github.com/kohler/gifsicle

========    =================
version     toolchain        
========    =================
``1.92``    ``GCCcore/8.2.0``
========    =================


.. _list_software_GIMIC_361:

*GIMIC*
+++++++

The GIMIC program calculates magnetically induced currents in molecules.  You need to provide this program with a density matrix in atomic-orbital (AO) basis  and three (effective) magnetically perturbed AO density matrices in the proper format. Currently ACES2, Turbomole, G09, QChem, FERMION++, and LSDalton can produce these matrices.

*homepage*: http://gimic.readthedocs.io

==============    ==================    ===============
version           versionsuffix         toolchain      
==============    ==================    ===============
``2018.04.20``    ``-Python-2.7.14``    ``intel/2018a``
==============    ==================    ===============


.. _list_software_gimkl_532:

*gimkl*
+++++++

GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI and MKL

*homepage*: (none)

==========    ==========
version       toolchain 
==========    ==========
``2.11.5``    ``system``
``2017a``     ``system``
``2018b``     ``system``
==========    ==========


.. _list_software_gimpi_534:

*gimpi*
+++++++

GNU Compiler Collection (GCC) based compiler toolchain with Intel MPI.

*homepage*: (none)

==========    ==========
version       toolchain 
==========    ==========
``2.11.5``    ``system``
``2017a``     ``system``
``2017b``     ``system``
``2018a``     ``system``
``2018b``     ``system``
==========    ==========


.. _list_software_gimpic_633:

*gimpic*
++++++++

GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including IntelMPI for MPI support with CUDA features enabled.

*homepage*: (none)

=========    ==========
version      toolchain 
=========    ==========
``2017b``    ``system``
=========    ==========


.. _list_software_GIMPS_384:

*GIMPS*
+++++++

GIMPS: Great Internet Mersenne Prime Search;  it can be useful for limited stress testing of nodes, in user space

*homepage*: http://www.mersenne.org/

===================    =============
version                toolchain    
===================    =============
``p95v279``            ``GCC/4.8.2``
``p95v279.linux64``    ``system``   
===================    =============


.. _list_software_giolf_529:

*giolf*
+++++++

GNU Compiler Collection (GCC) based compiler toolchain, including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

=========    ==========
version      toolchain 
=========    ==========
``2017b``    ``system``
``2018a``    ``system``
=========    ==========


.. _list_software_giolfc_628:

*giolfc*
++++++++

GCC based compiler toolchain __with CUDA support__, and including  IntelMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

=========    ==========
version      toolchain 
=========    ==========
``2017b``    ``system``
=========    ==========


.. _list_software_git_324:

*git*
+++++

Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.

*homepage*: http://git-scm.com/

===========    =============    =================
version        versionsuffix    toolchain        
===========    =============    =================
``1.8.5.6``                     ``GCC/4.9.2``    
``2.2.2``                       ``GCC/4.9.2``    
``2.4.1``                       ``GCC/4.9.2``    
``2.8.0``                       ``foss/2016a``   
``2.12.2``                      ``foss/2016b``   
``2.13.1``                      ``foss/2016b``   
``2.14.1``                      ``GCCcore/6.4.0``
``2.16.1``                      ``foss/2018a``   
``2.18.0``                      ``GCCcore/7.3.0``
``2.19.1``                      ``GCCcore/7.3.0``
``2.21.0``                      ``GCCcore/8.2.0``
``2.21.0``     ``-nodocs``      ``GCCcore/8.2.0``
``2.23.0``                      ``GCCcore/8.3.0``
``2.23.0``     ``-nodocs``      ``GCCcore/8.3.0``
===========    =============    =================


.. _list_software_git-lfs_694:

*git-lfs*
+++++++++

Git Large File Storage (LFS) replaces large files such as audio  samples, videos, datasets, and graphics with text pointers inside Git, while   storing the file contents on a remote server like GitHub.com

*homepage*: https://git-lfs.github.com

=========    ==========
version      toolchain 
=========    ==========
``1.1.1``    ``system``
``2.7.1``    ``system``
=========    ==========


.. _list_software_GitPython_934:

*GitPython*
+++++++++++

GitPython is a python library used to interact with Git repositories 

*homepage*: https://github.com/gitpython-developers/GitPython

==========    =================    ===============================
version       versionsuffix        toolchain                      
==========    =================    ===============================
``2.1.11``    ``-Python-3.6.6``    ``foss/2018b``, ``intel/2018b``
==========    =================    ===============================


.. _list_software_Givaro_616:

*Givaro*
++++++++

C++ library for arithmetic and algebraic computations

*homepage*: http://givaro.forge.imag.fr/

=========    ==============
version      toolchain     
=========    ==============
``4.0.1``    ``foss/2016a``
=========    ==============


.. _list_software_Giza_395:

*Giza*
++++++

Giza is an open, lightweight scientific plotting library built on  top of cairo that provides uniform output to multiple devices.

*homepage*: https://danieljprice.github.io/giza/

=========    ==============
version      toolchain     
=========    ==============
``1.1.0``    ``foss/2018b``
=========    ==============


.. _list_software_GL2PS_360:

*GL2PS*
+++++++

GL2PS: an OpenGL to PostScript printing library

*homepage*: http://www.geuz.org/gl2ps/

=========    ================    =================================================================================================
version      versionsuffix       toolchain                                                                                        
=========    ================    =================================================================================================
``1.3.9``                        ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                                 
``1.3.9``    ``-Mesa-11.2.1``    ``foss/2016a``, ``intel/2016a``                                                                  
``1.4.0``                        ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
=========    ================    =================================================================================================


.. _list_software_Glade_477:

*Glade*
+++++++

Glade is a RAD tool to enable quick & easy development of user interfaces for the GTK+ toolkit  and the GNOME desktop environment.

*homepage*: https://glade.gnome.org/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.8.5``    ``-Python-2.7.11``    ``intel/2016a``
``3.8.5``    ``-Python-2.7.14``    ``intel/2017b``
``3.8.6``    ``-Python-2.7.14``    ``intel/2018a``
``3.8.6``    ``-Python-2.7.15``    ``foss/2018b`` 
=========    ==================    ===============


.. _list_software_glew_431:

*glew*
++++++

The OpenGL Extension Wrangler Library  The OpenGL Extension Wrangler Library (GLEW) is a cross-platform  open-source C/C++ extension loading library. GLEW provides  efficient run-time mechanisms for determining which OpenGL  extensions are supported on the target platform. OpenGL  core and extension functionality is exposed in a single header  file. GLEW has been tested on a variety of operating systems,  including Windows, Linux, Mac OS X, FreeBSD, Irix, and Solaris.

*homepage*: http://glew.sourceforge.net/

=========    =================================
version      toolchain                        
=========    =================================
``2.1.0``    ``GCCcore/8.2.0``, ``foss/2018b``
=========    =================================


.. _list_software_GLib_350:

*GLib*
++++++

GLib is one of the base libraries of the GTK+ project

*homepage*: https://www.gtk.org/

==========    =================================================
version       toolchain                                        
==========    =================================================
``2.42.1``    ``GCC/4.9.2``                                    
``2.44.0``    ``GCC/4.9.2``                                    
``2.47.5``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.48.0``    ``foss/2016a``, ``intel/2016a``                  
``2.49.5``    ``foss/2016b``, ``intel/2016b``                  
``2.52.0``    ``foss/2017a``, ``intel/2017a``                  
``2.53.5``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``             
``2.54.2``    ``GCCcore/6.4.0``                                
``2.54.3``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``             
``2.60.1``    ``GCCcore/8.2.0``                                
``2.62.0``    ``GCCcore/8.3.0``                                
==========    =================================================


.. _list_software_glibc_513:

*glibc*
+++++++

The GNU C Library project provides the core libraries for the GNU system and GNU/Linux systems,  as well as many other systems that use Linux as the kernel.

*homepage*: https://www.gnu.org/software/libc/

========    =================
version     toolchain        
========    =================
``2.17``    ``GCCcore/6.4.0``
``2.26``    ``GCCcore/6.4.0``
========    =================


.. _list_software_GLibmm_568:

*GLibmm*
++++++++

C++ bindings for Glib

*homepage*: http://www.gtk.org/

==========    =================
version       toolchain        
==========    =================
``2.49.7``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_GLIMMER_525:

*GLIMMER*
+++++++++

Glimmer is a system for finding genes in microbial DNA, especially  the genomes of bacteria, archaea, and viruses.

*homepage*: http://www.cbcb.umd.edu/software/glimmer/

=========    ==============================
version      toolchain                     
=========    ==============================
``3.02b``    ``foss/2016b``, ``foss/2018b``
=========    ==============================


.. _list_software_GlimmerHMM_943:

*GlimmerHMM*
++++++++++++

GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model.  Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally  it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted  from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models.

*homepage*: https://ccb.jhu.edu/software/glimmerhmm

=========    ==============================
version      toolchain                     
=========    ==============================
``3.0.4``    ``foss/2016b``, ``foss/2018b``
=========    ==============================


.. _list_software_GLM_224:

*GLM*
+++++

OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics software based on  the OpenGL Shading Language (GLSL) specifications.

*homepage*: https://github.com/g-truc/glm

===========    =================
version        toolchain        
===========    =================
``0.9.7.6``    ``intel/2016a``  
``0.9.8.3``    ``GCCcore/5.4.0``
``0.9.9.0``    ``GCCcore/6.4.0``
===========    =================


.. _list_software_GlobalArrays_1219:

*GlobalArrays*
++++++++++++++

Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model

*homepage*: https://hpc.pnl.gov/globalarrays

=======    =============    ===============
version    versionsuffix    toolchain      
=======    =============    ===============
``5.7``                     ``intel/2018b``
``5.7``    ``-peigs``       ``intel/2019a``
=======    =============    ===============


.. _list_software_GlobusConnectPersonal_2170:

*GlobusConnectPersonal*
+++++++++++++++++++++++

Globus Connect Personal turns your laptop or other personal computer into a  Globus endpoint with a just a few clicks. With Globus Connect Personal you can  share and transfer files to/from a local machine—campus server, desktop computer or laptop—even if it's behind a firewall and you don't have administrator  privileges. 

*homepage*: https://www.globus.org/globus-connect-personal

=========    ==========
version      toolchain 
=========    ==========
``2.3.6``    ``system``
=========    ==========


.. _list_software_glog_425:

*glog*
++++++

A C++ implementation of the Google logging module.

*homepage*: https://github.com/google/glog

=========    ===================================================
version      toolchain                                          
=========    ===================================================
``0.3.4``    ``foss/2016a``                                     
``0.3.5``    ``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``0.4.0``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``               
=========    ===================================================


.. _list_software_GLPK_302:

*GLPK*
++++++

The GLPK (GNU Linear Programming Kit) package is intended for  solving large-scale linear programming (LP),  mixed integer programming (MIP), and other related problems.  It is a set of routines written in ANSI C   and organized in the form of a callable library.

*homepage*: https://www.gnu.org/software/glpk/

========    =======================================================
version     toolchain                                              
========    =======================================================
``4.58``    ``foss/2016a``, ``intel/2016a``                        
``4.60``    ``GCCcore/5.4.0``, ``intel/2016b``                     
``4.61``    ``intel/2017a``                                        
``4.65``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
========    =======================================================


.. _list_software_glproto_775:

*glproto*
+++++++++

X protocol and ancillary headers

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

==========    =================================================
version       toolchain                                        
==========    =================================================
``1.4.17``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
==========    =================================================


.. _list_software_GMAP-GSNAP_715:

*GMAP-GSNAP*
++++++++++++

GMAP: A Genomic Mapping and Alignment Program for mRNA and EST Sequences  GSNAP: Genomic Short-read Nucleotide Alignment Program

*homepage*: http://research-pub.gene.com/gmap/

==============    ===============
version           toolchain      
==============    ===============
``2016-05-01``    ``foss/2016a`` 
``2016-11-07``    ``foss/2016b`` 
``2018-05-11``    ``intel/2018a``
``2018-07-04``    ``intel/2018a``
``2019-03-15``    ``foss/2018b`` 
==============    ===============


.. _list_software_GMP_228:

*GMP*
+++++

GMP is a free library for arbitrary precision arithmetic,  operating on signed integers, rational numbers, and floating point numbers. 

*homepage*: http://gmplib.org/

==========    =====================================================================================================================================================
version       toolchain                                                                                                                                            
==========    =====================================================================================================================================================
``4.3.2``     ``system``                                                                                                                                           
``5.1.3``     ``GCC/4.8.2``                                                                                                                                        
``6.0.0``     ``GCC/4.9.2``                                                                                                                                        
``6.0.0a``    ``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``                                                                                                     
``6.1.0``     ``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``6.1.1``     ``GCC/5.4.0-2.26``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``                          
``6.1.2``     ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016b``                                        
==========    =====================================================================================================================================================


.. _list_software_gmpich_632:

*gmpich*
++++++++

gcc and GFortran based compiler toolchain,  including MPICH for MPI support.

*homepage*: (none)

===========    ==========
version        toolchain 
===========    ==========
``2016a``      ``system``
``2017.08``    ``system``
===========    ==========


.. _list_software_gmpolf_645:

*gmpolf*
++++++++

gcc and GFortran based compiler toolchain,  MPICH for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

===========    ==========
version        toolchain 
===========    ==========
``2016a``      ``system``
``2017.10``    ``system``
===========    ==========


.. _list_software_gmpy2_495:

*gmpy2*
+++++++

GMP/MPIR, MPFR, and MPC interface to Python 2.6+ and 3.x

*homepage*: https://github.com/aleaxit/gmpy

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``2.0.8``      ``-Python-2.7.13``    ``intel/2017a``                
``2.0.8``      ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``2.0.8``      ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``2.1.0b1``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``2.1.0b1``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
===========    ==================    ===============================


.. _list_software_gmsh_431:

*gmsh*
++++++

Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor.

*homepage*: http://geuz.org/gmsh

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``3.0.6``    ``-Python-2.7.14``    ``foss/2017b``
``3.0.6``    ``-Python-3.6.6``     ``foss/2018b``
``4.2.2``    ``-Python-3.6.6``     ``foss/2018b``
=========    ==================    ==============


.. _list_software_GMT_232:

*GMT*
+++++

GMT is an open source collection of about 80 command-line tools for manipulating  geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,  etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps  to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another  40 more specialized and discipline-specific tools. 

*homepage*: https://gmt.soest.hawaii.edu/

==========    ===============================
version       toolchain                      
==========    ===============================
``4.5.17``    ``foss/2018a``                 
``5.4.1``     ``intel/2017a``                
``5.4.3``     ``foss/2018a``, ``intel/2017b``
``5.4.5``     ``foss/2019a``                 
==========    ===============================


.. _list_software_gmvapich2_897:

*gmvapich2*
+++++++++++

GNU Compiler Collection (GCC) based compiler toolchain,  including MVAPICH2 for MPI support.

*homepage*: (none)

==========    ==========
version       toolchain 
==========    ==========
``1.7.20``    ``system``
``2016a``     ``system``
==========    ==========


.. _list_software_gmvolf_651:

*gmvolf*
++++++++

GNU Compiler Collection (GCC) based compiler toolchain, including  MVAPICH2 for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.

*homepage*: (none)

==========    ==========
version       toolchain 
==========    ==========
``1.7.20``    ``system``
``2016a``     ``system``
==========    ==========


.. _list_software_GNU_234:

*GNU*
+++++

Compiler-only toolchain with GCC and binutils.

*homepage*: http://www.gnu.org/software/

==============    ==========
version           toolchain 
==============    ==========
``4.9.2-2.25``    ``system``
``4.9.3-2.25``    ``system``
``5.1.0-2.25``    ``system``
==============    ==========


.. _list_software_gnuplot_777:

*gnuplot*
+++++++++

Portable interactive, function plotting utility

*homepage*: http://gnuplot.sourceforge.net/

=========    ================================================================
version      toolchain                                                       
=========    ================================================================
``5.0.3``    ``foss/2016a``, ``intel/2016a``                                 
``5.0.5``    ``foss/2016b``, ``intel/2016b``                                 
``5.0.6``    ``intel/2017a``                                                 
``5.2.2``    ``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``5.2.5``    ``foss/2018b``                                                  
``5.2.6``    ``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``           
=========    ================================================================


.. _list_software_GnuTLS_541:

*GnuTLS*
++++++++

GnuTLS is a secure communications library implementing the SSL, TLS  and DTLS protocols and technologies around them. It provides a simple  C language application programming interface (API) to access the secure  communications protocols as well as APIs to parse and write X.509, PKCS #12,  OpenPGP and other required structures. It is aimed to be portable  and efficient with focus on security and interoperability.

*homepage*: http://www.gnutls.org/

==========    ==================
version       toolchain         
==========    ==================
``3.3.21``    ``intel/2016a``   
``3.4.7``     ``GNU/4.9.3-2.25``
``3.4.11``    ``foss/2016a``    
==========    ==================


.. _list_software_Go_182:

*Go*
++++

Go is an open source programming language that makes it easy to build  simple, reliable, and efficient software.

*homepage*: http://www.golang.org

==========    =================
version       toolchain        
==========    =================
``1.2.1``     ``GCC/4.8.2``    
``1.4.2``     ``GCC/4.8.4``    
``1.5``       ``GCC/4.8.4``    
``1.8.1``     ``system``       
``1.11.5``    ``system``       
``1.12``      ``system``       
``1.12.1``    ``GCCcore/7.3.0``
``1.13.1``    ``system``       
==========    =================


.. _list_software_GObject-Introspection_2108:

*GObject-Introspection*
+++++++++++++++++++++++

GObject introspection is a middleware layer between C libraries  (using GObject) and language bindings. The C library can be scanned at  compile time and generate a metadata file, in addition to the actual  native C library. Then at runtime, language bindings can read this  metadata and automatically provide bindings to call into the C library.

*homepage*: https://wiki.gnome.org/GObjectIntrospection/

==========    ==================    ==================================
version       versionsuffix         toolchain                         
==========    ==================    ==================================
``1.47.1``                          ``foss/2016a``, ``intel/2016a``   
``1.48.0``                          ``foss/2016a``, ``intel/2016a``   
``1.49.1``                          ``foss/2016b``, ``intel/2016b``   
``1.52.0``                          ``intel/2017a``                   
``1.53.5``    ``-Python-2.7.13``    ``intel/2017a``                   
``1.53.5``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``   
``1.53.5``    ``-Python-3.6.1``     ``intel/2017a``                   
``1.54.1``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``   
``1.54.1``    ``-Python-2.7.15``    ``foss/2018b``, ``fosscuda/2018b``
``1.54.1``    ``-Python-3.6.6``     ``fosscuda/2018b``                
``1.60.1``    ``-Python-3.7.2``     ``GCCcore/8.2.0``                 
==========    ==================    ==================================


.. _list_software_golf_424:

*golf*
++++++

GNU Compiler Collection (GCC) based compiler toolchain, including  OpenBLAS (BLAS and LAPACK support) and FFTW.

*homepage*: (none)

=========    ==========
version      toolchain 
=========    ==========
``2018a``    ``system``
=========    ==========


.. _list_software_gomkl_538:

*gomkl*
+++++++

GNU Compiler Collection (GCC) based compiler toolchain with OpenMPI and MKL

*homepage*: (none)

=========    ==========
version      toolchain 
=========    ==========
``2018b``    ``system``
``2019a``    ``system``
=========    ==========


.. _list_software_gompi_540:

*gompi*
+++++++

GNU Compiler Collection (GCC) based compiler toolchain,  including OpenMPI for MPI support.

*homepage*: (none)

===========    =============    ==========
version        versionsuffix    toolchain 
===========    =============    ==========
``2016.04``                     ``system``
``2016.06``                     ``system``
``2016.07``                     ``system``
``2016.09``                     ``system``
``2016a``                       ``system``
``2016b``                       ``system``
``2017a``                       ``system``
``2017b``                       ``system``
``2018.08``                     ``system``
``2018a``                       ``system``
``2018b``                       ``system``
``2019a``                       ``system``
``2019b``                       ``system``
``system``     ``-2.29``        ``system``
===========    =============    ==========


.. _list_software_gompic_639:

*gompic*
++++++++

GNU Compiler Collection (GCC) based compiler toolchain along with CUDA toolkit,  including OpenMPI for MPI support with CUDA features enabled.

*homepage*: (none)

=========    ==========
version      toolchain 
=========    ==========
``2017b``    ``system``
``2018a``    ``system``
``2018b``    ``system``
``2019a``    ``system``
``2019b``    ``system``
=========    ==========


.. _list_software_google-java-format_1794:

*google-java-format*
++++++++++++++++++++

Reformats Java source code to comply with Google Java Style.

*homepage*: https://github.com/google/google-java-format

=======    =============    ==========
version    versionsuffix    toolchain 
=======    =============    ==========
``1.7``    ``-Java-1.8``    ``system``
=======    =============    ==========


.. _list_software_googletest_1085:

*googletest*
++++++++++++

Google's C++ test framework

*homepage*: https://github.com/google/googletest

=========    =====================================================================
version      toolchain                                                            
=========    =====================================================================
``1.8.0``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``
=========    =====================================================================


.. _list_software_GP2C_268:

*GP2C*
++++++

The gp2c compiler is a package for translating GP routines into the C programming language, so that  they can  be compiled and used with the PARI system or the GP calculator. 

*homepage*: http://pari.math.u-bordeaux.fr/pub/pari/manuals/gp2c/gp2c.html

============    ==============
version         toolchain     
============    ==============
``0.0.9pl5``    ``foss/2016a``
============    ==============


.. _list_software_GPAW_303:

*GPAW*
++++++

GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)  method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or  atom-centered basis-functions.

*homepage*: https://wiki.fysik.dtu.dk/gpaw/

==========    ============================    ===============================
version       versionsuffix                   toolchain                      
==========    ============================    ===============================
``1.4.0``     ``-Python-3.6.6``               ``foss/2018b``, ``intel/2018b``
``19.8.1``    ``-ASE-3.18.0-Python-3.6.6``    ``foss/2018b``, ``intel/2018b``
``19.8.1``    ``-Python-3.7.2``               ``foss/2019a``, ``intel/2019a``
==========    ============================    ===============================


.. _list_software_GPAW-setups_1024:

*GPAW-setups*
+++++++++++++

PAW setup for the GPAW Density Functional Theory package.   Users can install setups manually using 'gpaw install-data' or use setups from this package.   The versions of GPAW and GPAW-setups can be intermixed.

*homepage*: https://wiki.fysik.dtu.dk/gpaw/

=============    ==========
version          toolchain 
=============    ==========
``0.8.7929``     ``system``
``0.9.9672``     ``system``
``0.9.11271``    ``system``
``0.9.20000``    ``system``
=============    ==========


.. _list_software_gperf_532:

*gperf*
+++++++

GNU gperf is a perfect hash function generator. For a given list of strings,  it produces a hash function and hash table, in form of C or C++ code, for  looking up a value depending on the input string. The hash function is  perfect, which means that the hash table has no collisions, and the hash  table lookup needs a single string comparison only. 

*homepage*: https://www.gnu.org/software/gperf/

=========    =========================================================================================================================
version      toolchain                                                                                                                
=========    =========================================================================================================================
``3.0.4``    ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``3.1``      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                               
=========    =========================================================================================================================


.. _list_software_gperftools_1093:

*gperftools*
++++++++++++

gperftools are for use by developers so that they can create more robust applications.  Especially of use to those developing multi-threaded applications in C++ with templates.  Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler.

*homepage*: http://github.com/gperftools/gperftools

=========    ===============================
version      toolchain                      
=========    ===============================
``2.5``      ``foss/2016a``, ``intel/2016b``
``2.6.3``    ``GCCcore/6.4.0``              
=========    ===============================


.. _list_software_gpustat_776:

*gpustat*
+++++++++

dstat-like utilization monitor for NVIDIA GPUs

*homepage*: https://github.com/wookayin/gpustat

=========    ==================    ==================
version      versionsuffix         toolchain         
=========    ==================    ==================
``0.5.0``    ``-Python-2.7.15``    ``fosscuda/2018b``
=========    ==================    ==================


.. _list_software_Grace_482:

*Grace*
+++++++

Grace is a WYSIWYG tool to make two-dimensional plots of numerical data.

*homepage*: http://plasma-gate.weizmann.ac.il/Grace/

==========    =============    ===============================================
version       versionsuffix    toolchain                                      
==========    =============    ===============================================
``5.1.25``    ``-5build1``     ``foss/2017b``, ``foss/2018a``, ``intel/2017b``
==========    =============    ===============================================


.. _list_software_grace_514:

*grace*
+++++++

Grace is a WYSIWYG 2D plotting tool for X Windows System and Motif.

*homepage*: http://freecode.com/projects/grace

==========    ===============================
version       toolchain                      
==========    ===============================
``5.1.25``    ``foss/2016a``, ``intel/2016a``
==========    ===============================


.. _list_software_Gradle_591:

*Gradle*
++++++++

Complete Gradle install. From mobile apps to microservices, from small startups to big enterprises, Gradle helps teams build, automate and deliver better software, faster.  

*homepage*: https://gradle.org

=========    ==========
version      toolchain 
=========    ==========
``4.5.1``    ``system``
=========    ==========


.. _list_software_graph-tool_1021:

*graph-tool*
++++++++++++

Graph-tool is an efficient Python module for manipulation and  statistical analysis of graphs (a.k.a. networks). Contrary to  most other python modules with similar functionality, the core  data structures and algorithms are implemented in C++, making  extensive use of template metaprogramming, based heavily on  the Boost Graph Library. This confers it a level of  performance that is comparable (both in memory usage and  computation time) to that of a pure C/C++ library.

*homepage*: https://graph-tool.skewed.de/

========    =================    ==============
version     versionsuffix        toolchain     
========    =================    ==============
``2.26``    ``-Python-3.6.3``    ``foss/2017b``
``2.27``    ``-Python-3.6.6``    ``foss/2018b``
========    =================    ==============


.. _list_software_Graphene_810:

*Graphene*
++++++++++

Graphene is a a thin layer of types for graphic libraries

*homepage*: http://ebassi.github.io/graphene/

=========    ===============
version      toolchain      
=========    ===============
``1.6.0``    ``intel/2017a``
=========    ===============


.. _list_software_GraphicsMagick_1405:

*GraphicsMagick*
++++++++++++++++

GraphicsMagick is the swiss army knife of image processing.

*homepage*: http://www.graphicsmagick.org/

==========    ================================
version       toolchain                       
==========    ================================
``1.3.23``    ``foss/2016a``, ``intel/2016a`` 
``1.3.25``    ``intel/2016b``, ``intel/2017a``
``1.3.28``    ``foss/2018a``                  
``1.3.31``    ``foss/2018b``                  
==========    ================================


.. _list_software_Graphviz_843:

*Graphviz*
++++++++++

Graphviz is open source graph visualization software. Graph visualization  is a way of representing structural information as diagrams of  abstract graphs and networks. It has important applications in networking,  bioinformatics,  software engineering, database and web design, machine learning,  and in visual interfaces for other technical domains.

*homepage*: http://www.graphviz.org/

==========    ===============================
version       toolchain                      
==========    ===============================
``2.38.0``    ``foss/2016b``, ``intel/2016b``
``2.40.1``    ``intel/2018a``                
==========    ===============================


.. _list_software_graphviz_875:

*graphviz*
++++++++++

Simple Python interface for Graphviz

*homepage*: https://pypi.python.org/pypi/graphviz

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.5.1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``0.5.1``    ``-Python-3.5.2``     ``intel/2016b``                
``0.8.2``    ``-Python-3.6.4``     ``intel/2018a``                
=========    ==================    ===============================


.. _list_software_GRASS_384:

*GRASS*
+++++++

The Geographic Resources Analysis Support System - used  for geospatial data management and analysis, image processing,  graphics and maps production, spatial modeling, and visualization

*homepage*: http://grass.osgeo.org

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``7.6.0``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_Greenlet_822:

*Greenlet*
++++++++++

The greenlet package is a spin-off of Stackless, a version of CPython that supports micro-threads called "tasklets". Tasklets run pseudo-concurrently (typically in a single or a few OS-level threads) and are synchronized with data exchanges on "channels". A "greenlet", on the other hand, is a still more primitive notion of micro-thread with no implicit scheduling; coroutines, in other words. This is useful when you want to control exactly when your code runs. 

*homepage*: https://github.com/python-greenlet/greenlet

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.4.9``     ``-Python-2.7.11``    ``intel/2016a``
``0.4.9``     ``-Python-3.5.1``     ``intel/2016a``
``0.4.11``    ``-Python-2.7.12``    ``intel/2016b``
``0.4.12``    ``-Python-2.7.14``    ``intel/2017b``
==========    ==================    ===============


.. _list_software_Grep_398:

*Grep*
++++++

The grep command searches one or more input files for lines containing a match to a specified pattern.  By default, grep prints the matching lines.

*homepage*: http://www.gnu.org/software/grep/

========    =============
version     toolchain    
========    =============
``2.21``    ``GCC/4.9.2``
========    =============


.. _list_software_grib_api_829:

*grib_api*
++++++++++

The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python  programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of  command line tools is also provided to give quick access to GRIB messages.

*homepage*: https://software.ecmwf.int/wiki/display/GRIB/Home

==========    ================================================
version       toolchain                                       
==========    ================================================
``1.16.0``    ``intel/2016a``                                 
``1.21.0``    ``foss/2017a``                                  
``1.24.0``    ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
==========    ================================================


.. _list_software_GRIT_310:

*GRIT*
++++++

GRIT - A tool for the integrative analysis of RNA-seq type assays

*homepage*: https://github.com/nboley/grit

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.0.5``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_GRNBoost_750:

*GRNBoost*
++++++++++

XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.

*homepage*: https://github.com/dmlc/xgboost

============    ===================    ===============
version         versionsuffix          toolchain      
============    ===================    ===============
``20171009``    ``-Java-1.8.0_152``    ``intel/2017b``
============    ===================    ===============


.. _list_software_GroIMP_526:

*GroIMP*
++++++++

GroIMP (Growth Grammar-related Interactive Modelling Platform) is a 3D-modelling platform.

*homepage*: http://www.grogra.de/software/groimp

=======    ==========
version    toolchain 
=======    ==========
``1.5``    ``system``
=======    ==========


.. _list_software_GROMACS_524:

*GROMACS*
+++++++++

GROMACS is a versatile package to perform molecular dynamics,  i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.

*homepage*: http://www.gromacs.org

==========    =================    ============================================================================================================
version       versionsuffix        toolchain                                                                                                   
==========    =================    ============================================================================================================
``5.1.2``     ``-hybrid``          ``foss/2016a``, ``intel/2016a``                                                                             
``5.1.2``     ``-hybrid-dp``       ``intel/2016a``                                                                                             
``5.1.2``     ``-mt``              ``foss/2016a``                                                                                              
``5.1.4``     ``-hybrid``          ``foss/2016b``                                                                                              
``5.1.4``     ``-mt``              ``foss/2016b``                                                                                              
``2016``      ``-hybrid``          ``foss/2016b``                                                                                              
``2016``      ``-mt``              ``foss/2016b``                                                                                              
``2016.1``    ``-PLUMED``          ``foss/2017a``                                                                                              
``2016.2``                         ``foss/2017a``                                                                                              
``2016.3``                         ``foss/2017a``, ``intel/2017a``                                                                             
``2016.3``    ``-GPU-enabled``     ``foss/2016b``                                                                                              
``2016.4``                         ``foss/2017b``, ``fosscuda/2017b``, ``giolf/2017b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``  
``2016.5``                         ``intel/2018a``                                                                                             
``2018``                           ``foss/2018a``                                                                                              
``2018.1``    ``-PLUMED``          ``foss/2018b``                                                                                              
``2018.2``                         ``foss/2017b``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018b``, ``intel/2017b``, ``intelcuda/2017b``
``2018.3``                         ``foss/2018b``, ``fosscuda/2018b``                                                                          
``2018.4``    ``-PLUMED-2.5.0``    ``foss/2018b``, ``fosscuda/2018b``                                                                          
``2019``                           ``foss/2018b``, ``fosscuda/2018b``                                                                          
``2019.2``                         ``fosscuda/2019a``                                                                                          
``2019.3``                         ``foss/2019a``, ``fosscuda/2019a``                                                                          
==========    =================    ============================================================================================================


.. _list_software_GromacsWrapper_1453:

*GromacsWrapper*
++++++++++++++++

GromacsWrapper is a python package that wraps system calls to Gromacs tools into thin classes.  This allows for fairly seamless integration of the gromacs tools into python scripts. 

*homepage*: https://gromacswrapper.readthedocs.org

=========    =================    ==================
version      versionsuffix        toolchain         
=========    =================    ==================
``0.8.0``    ``-Python-3.7.2``    ``fosscuda/2019a``
=========    =================    ==================


.. _list_software_GroopM_596:

*GroopM*
++++++++

GroopM is a metagenomic binning toolset. It leverages spatio-temporal   dynamics (differential coverage) to accurately (and almost automatically)   extract population genomes from multi-sample metagenomic datasets.  

*homepage*: http://ecogenomics.github.io/GroopM/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``0.3.4``    ``-Python-2.7.12``    ``foss/2016b``
=========    ==================    ==============


.. _list_software_GSL_230:

*GSL*
+++++

The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.  The library provides a wide range of mathematical routines such as random number generators, special functions  and least-squares fitting.

*homepage*: http://www.gnu.org/software/gsl/

=========    ============================================================================================================================
version      toolchain                                                                                                                   
=========    ============================================================================================================================
``1.16``     ``foss/2016a``, ``intel/2016a``                                                                                             
``2.1``      ``GCC/5.4.0-2.26``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``2.2.1``    ``intel/2016a``, ``intel/2016b``                                                                                            
``2.3``      ``foss/2016b``, ``foss/2017a``, ``intel/2016b``, ``intel/2017a``                                                            
``2.4``      ``GCCcore/6.4.0``                                                                                                           
``2.5``      ``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``  
=========    ============================================================================================================================


.. _list_software_gSOAP_410:

*gSOAP*
+++++++

The gSOAP toolkit is a C and C++ software development toolkit for  SOAP and REST XML Web services and generic C/C++ XML data bindings.  The toolkit analyzes WSDLs and XML schemas (separately or as a combined set) and maps the XML schema types  and the SOAP/REST XML messaging protocols to easy-to-use and efficient C and C++ code.  It also supports exposing (legacy) C and C++ applications as XML Web services  by auto-generating XML serialization code and WSDL specifications.  Or you can simply use it to automatically convert XML to/from C and C++ data.  The toolkit supports options to generate pure ANSI C or C++ with or without STL.

*homepage*: http://www.cs.fsu.edu/~engelen/soap.html

==========    =================
version       toolchain        
==========    =================
``2.8.48``    ``GCCcore/6.3.0``
==========    =================


.. _list_software_GST-plugins-base_1509:

*GST-plugins-base*
++++++++++++++++++

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: http://gstreamer.freedesktop.org/

===========    ==================================================================================================================
version        toolchain                                                                                                         
===========    ==================================================================================================================
``0.10.36``    ``foss/2016a``, ``foss/2017b``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``1.6.4``      ``foss/2016a``                                                                                                    
``1.8.3``      ``foss/2016a``                                                                                                    
``1.16.0``     ``GCC/8.2.0-2.31.1``                                                                                              
===========    ==================================================================================================================


.. _list_software_GStreamer_906:

*GStreamer*
+++++++++++

GStreamer is a library for constructing graphs of media-handling  components. The applications it supports range from simple  Ogg/Vorbis playback, audio/video streaming to complex audio  (mixing) and video (non-linear editing) processing.

*homepage*: http://gstreamer.freedesktop.org/

===========    ==================================================================================================================
version        toolchain                                                                                                         
===========    ==================================================================================================================
``0.10.36``    ``foss/2016a``, ``foss/2017b``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``1.6.4``      ``foss/2016a``                                                                                                    
``1.8.3``      ``foss/2016a``                                                                                                    
``1.15.1``     ``fosscuda/2018b``                                                                                                
``1.16.0``     ``GCC/8.2.0-2.31.1``                                                                                              
===========    ==================================================================================================================


.. _list_software_GTDB-Tk_525:

*GTDB-Tk*
+++++++++

A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.

*homepage*: https://github.com/Ecogenomics/GTDBTk

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.2.2``    ``-Python-2.7.15``    ``intel/2018b``                
``0.3.2``    ``-Python-2.7.15``    ``foss/2019a``, ``intel/2019a``
=========    ==================    ===============================


.. _list_software_gtest_551:

*gtest*
+++++++

Google's framework for writing C++ tests on a variety of platforms

*homepage*: https://github.com/google/googletest

=========    ====================================
version      toolchain                           
=========    ====================================
``1.8.0``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.8.1``    ``GCCcore/8.2.0``                   
=========    ====================================


.. _list_software_GTK+_273:

*GTK+*
++++++

The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications. 

*homepage*: https://developer.gnome.org/gtk+/stable/

===========    ================================================================================
version        toolchain                                                                       
===========    ================================================================================
``2.24.28``    ``intel/2016a``                                                                 
``2.24.30``    ``foss/2016a``, ``intel/2016a``                                                 
``2.24.31``    ``foss/2016b``, ``intel/2016b``, ``intel/2017a``                                
``2.24.32``    ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
``3.22.30``    ``fosscuda/2018b``                                                              
``3.24.8``     ``GCCcore/8.2.0``                                                               
===========    ================================================================================


.. _list_software_GTOOL_389:

*GTOOL*
+++++++

GTOOL is a program for transforming sets of genotype data for use with  the programs SNPTEST and IMPUTE.  

*homepage*: http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html

=========    ==========
version      toolchain 
=========    ==========
``0.7.5``    ``system``
=========    ==========


.. _list_software_GTS_238:

*GTS*
+++++

GTS stands for the GNU Triangulated Surface Library.  It is an Open Source Free Software Library intended to provide a set of useful  functions to deal with 3D surfaces meshed with interconnected triangles.

*homepage*: http://gts.sourceforge.net/

============    =================================================================================================
version         toolchain                                                                                        
============    =================================================================================================
``0.7.6``       ``foss/2016a``, ``foss/2016b``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2018a``
``20121130``    ``foss/2017b``                                                                                   
============    =================================================================================================


.. _list_software_guenomu_768:

*guenomu*
+++++++++

guenomu is a software written in C that estimates the species tree for a given set of gene families.

*homepage*: https://bitbucket.org/leomrtns/guenomu

==============    =============    ===============
version           versionsuffix    toolchain      
==============    =============    ===============
``2019.07.05``    ``-mpi``         ``iimpi/2019a``
==============    =============    ===============


.. _list_software_Guile_502:

*Guile*
+++++++

Guile is a programming language, designed to help programmers create flexible  applications that can be extended by users or other programmers with plug-ins,  modules, or scripts. 

*homepage*: https://www.gnu.org/software/guile/

==========    ================================================================================================================================================================================
version       toolchain                                                                                                                                                                       
==========    ================================================================================================================================================================================
``1.8.8``     ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``
``2.0.11``    ``GCC/4.9.3-2.25``, ``foss/2016a``                                                                                                                                              
``2.2.2``     ``GCCcore/6.4.0``                                                                                                                                                               
``2.2.4``     ``GCCcore/7.3.0``                                                                                                                                                               
==========    ================================================================================================================================================================================


.. _list_software_GULP_312:

*GULP*
++++++

GULP is a program for performing a variety of types of simulation on materials  using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs  and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations

*homepage*: https://gulp.curtin.edu.au/gulp/

=======    ===============
version    toolchain      
=======    ===============
``5.1``    ``intel/2019a``
=======    ===============


.. _list_software_Gurobi_616:

*Gurobi*
++++++++

The Gurobi Optimizer is a state-of-the-art solver for mathematical programming. The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern  architectures and multi-core processors, using the most advanced implementations of the  latest algorithms.

*homepage*: http://www.gurobi.com

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``6.5.1``                         ``system``     
``6.5.2``                         ``system``     
``7.0.1``                         ``system``     
``7.5.2``                         ``system``     
``7.5.2``    ``-Python-3.6.4``    ``intel/2018a``
``8.1.1``                         ``system``     
=========    =================    ===============


.. _list_software_gzip_442:

*gzip*
++++++

gzip (GNU zip) is a popular data compression program as a replacement for compress

*homepage*: http://www.gnu.org/software/gzip/

========    =======================================================
version     toolchain                                              
========    =======================================================
``1.8``     ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``1.9``     ``GCCcore/7.3.0``                                      
``1.10``    ``GCCcore/8.2.0``                                      
========    =======================================================


.. _list_software_letter_h:

*H*
---


:ref:`list_software_h4toh5_540` - :ref:`list_software_H5hut_462` - :ref:`list_software_h5py_390` - :ref:`list_software_Hadoop_603` - :ref:`list_software_hanythingondemand_1808` - :ref:`list_software_HAPGEN2_485` - :ref:`list_software_HarfBuzz_812` - :ref:`list_software_Harminv_725` - :ref:`list_software_HBase_451` - :ref:`list_software_HDDM_285` - :ref:`list_software_HDF_210` - :ref:`list_software_HDF5_263` - :ref:`list_software_hdf5storage_1116` - :ref:`list_software_HDFView_621` - :ref:`list_software_HEALPix_587` - :ref:`list_software_heaptrack_947` - :ref:`list_software_Hello_500` - :ref:`list_software_help2man_791` - :ref:`list_software_HH-suite_743` - :ref:`list_software_HiC-Pro_594` - :ref:`list_software_HiCExplorer_1093` - :ref:`list_software_HIPS_308` - :ref:`list_software_HISAT2_427` - :ref:`list_software_HLAminer_752` - :ref:`list_software_HMMER_377` - :ref:`list_software_hmmlearn_852` - :ref:`list_software_HOME_297` - :ref:`list_software_Horovod_737` - :ref:`list_software_HPCG_290` - :ref:`list_software_HPCX_307` - :ref:`list_software_HPDBSCAN_579` - :ref:`list_software_HPL_228` - :ref:`list_software_htop_443` - :ref:`list_software_HTSeq_453` - :ref:`list_software_HTSlib_550` - :ref:`list_software_hub_319` - :ref:`list_software_hunspell_875` - :ref:`list_software_hwloc_541` - :ref:`list_software_Hyperopt_859` - :ref:`list_software_hyperspy_900` - :ref:`list_software_hypothesis_1104` - :ref:`list_software_Hypre_520`


.. _list_software_h4toh5_540:

*h4toh5*
++++++++

The h4toh5 software consists of the h4toh5 and h5toh4 command-line utilities,  as well as a conversion library for converting between individual HDF4 and HDF5 objects.

*homepage*: http://www.hdfgroup.org/h4toh5/

=========    ========================    ==========
version      versionsuffix               toolchain 
=========    ========================    ==========
``2.2.2``    ``-linux-x86_64-static``    ``system``
=========    ========================    ==========


.. _list_software_H5hut_462:

*H5hut*
+++++++

HDF5 Utility Toolkit: High-Performance I/O Library for Particle-based Simulations

*homepage*: https://amas.psi.ch/H5hut/

===========    ===============
version        toolchain      
===========    ===============
``1.99.13``    ``intel/2016b``
===========    ===============


.. _list_software_h5py_390:

*h5py*
++++++

HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,  version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous  amounts of data.

*homepage*: http://www.h5py.org/

==========    =====================================    ====================================================================================
version       versionsuffix                            toolchain                                                                           
==========    =====================================    ====================================================================================
``2.5.0``     ``-Python-2.7.11-HDF5-1.8.16``           ``foss/2016a``, ``intel/2016a``                                                     
``2.5.0``     ``-Python-2.7.11-HDF5-1.8.16-serial``    ``foss/2016a``, ``intel/2016a``                                                     
``2.5.0``     ``-Python-3.5.1-HDF5-1.8.16``            ``foss/2016a``                                                                      
``2.6.0``     ``-Python-2.7.11``                       ``intel/2016a``                                                                     
``2.6.0``     ``-Python-2.7.12-HDF5-1.10.0-patch1``    ``foss/2016b``, ``intel/2016b``                                                     
``2.6.0``     ``-Python-2.7.12-HDF5-1.8.17``           ``foss/2016b``, ``intel/2016b``                                                     
``2.6.0``     ``-Python-2.7.12-HDF5-1.8.18``           ``foss/2016b``, ``intel/2016b``                                                     
``2.6.0``     ``-Python-3.5.2-HDF5-1.10.0-patch1``     ``foss/2016b``, ``intel/2016b``                                                     
``2.6.0``     ``-Python-3.5.2-HDF5-1.8.17``            ``intel/2016b``                                                                     
``2.6.0``     ``-Python-3.5.2-HDF5-1.8.18``            ``foss/2016b``, ``intel/2016b``                                                     
``2.7.0``     ``-Python-2.7.12``                       ``intel/2016b``                                                                     
``2.7.0``     ``-Python-2.7.13``                       ``intel/2017a``                                                                     
``2.7.0``     ``-Python-2.7.13-HDF5-1.10.1``           ``foss/2017a``                                                                      
``2.7.0``     ``-Python-2.7.13-HDF5-1.8.19``           ``foss/2017a``                                                                      
``2.7.0``     ``-Python-3.6.1``                        ``intel/2017a``                                                                     
``2.7.0``     ``-Python-3.6.1-HDF5-1.10.0-patch1``     ``intel/2017a``                                                                     
``2.7.0``     ``-Python-3.6.1-HDF5-1.10.1``            ``foss/2017a``                                                                      
``2.7.0``     ``-Python-3.6.1-HDF5-1.8.19``            ``foss/2017a``                                                                      
``2.7.1``     ``-Python-2.7.13``                       ``intel/2017a``                                                                     
``2.7.1``     ``-Python-2.7.14``                       ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``
``2.7.1``     ``-Python-2.7.14-serial``                ``intel/2018a``                                                                     
``2.7.1``     ``-Python-3.6.1``                        ``intel/2017a``                                                                     
``2.7.1``     ``-Python-3.6.3``                        ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``                                 
``2.7.1``     ``-Python-3.6.4``                        ``foss/2018a``, ``intel/2018a``                                                     
``2.8.0``     ``-Python-2.7.15``                       ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                                 
``2.8.0``     ``-Python-2.7.15-serial``                ``intel/2018b``                                                                     
``2.8.0``     ``-Python-3.6.6``                        ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                                 
``2.9.0``                                              ``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``            
``2.10.0``    ``-Python-3.7.4``                        ``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``                                 
==========    =====================================    ====================================================================================


.. _list_software_Hadoop_603:

*Hadoop*
++++++++

Hadoop MapReduce by Cloudera

*homepage*: https://archive.cloudera.com/cdh5/cdh/5/

===================    ==========================    =================
version                versionsuffix                 toolchain        
===================    ==========================    =================
``2.4.0``              ``-seagate-722af1-native``    ``system``       
``2.5.0-cdh5.3.1``     ``-native``                   ``system``       
``2.6.0-cdh5.4.5``     ``-native``                   ``system``       
``2.6.0-cdh5.7.0``     ``-native``                   ``system``       
``2.6.0-cdh5.8.0``     ``-native``                   ``system``       
``2.6.0-cdh5.12.0``    ``-native``                   ``system``       
``2.9.2``              ``-native``                   ``GCCcore/7.3.0``
===================    ==========================    =================


.. _list_software_hanythingondemand_1808:

*hanythingondemand*
+++++++++++++++++++

HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster. It uses the Torque resource manager to do node allocation.

*homepage*: https://github.com/hpcugent/hanythingondemand

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.0.0``    ``-cli``              ``system``     
``3.0.1``    ``-cli``              ``system``     
``3.0.2``    ``-cli``              ``system``     
``3.0.3``    ``-cli``              ``system``     
``3.0.4``    ``-cli``              ``system``     
``3.1.0``    ``-Python-2.7.11``    ``intel/2016a``
``3.1.0``    ``-cli``              ``system``     
``3.1.1``    ``-Python-2.7.11``    ``intel/2016a``
``3.1.1``    ``-cli``              ``system``     
``3.1.2``    ``-Python-2.7.11``    ``intel/2016a``
``3.1.2``    ``-cli``              ``system``     
``3.1.3``    ``-Python-2.7.12``    ``intel/2016b``
``3.1.3``    ``-cli``              ``system``     
``3.1.4``    ``-Python-2.7.12``    ``intel/2016b``
``3.1.4``    ``-cli``              ``system``     
``3.2.0``    ``-Python-2.7.12``    ``intel/2016b``
``3.2.0``    ``-cli``              ``system``     
``3.2.2``    ``-Python-2.7.12``    ``intel/2016b``
``3.2.2``    ``-cli``              ``system``     
=========    ==================    ===============


.. _list_software_HAPGEN2_485:

*HAPGEN2*
+++++++++

'HAPGEN2' simulates case control datasets at SNP markers.

*homepage*: https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html

=========    ==========
version      toolchain 
=========    ==========
``2.2.0``    ``system``
=========    ==========


.. _list_software_HarfBuzz_812:

*HarfBuzz*
++++++++++

HarfBuzz is an OpenType text shaping engine.

*homepage*: http://www.freedesktop.org/wiki/Software/HarfBuzz

=========    ================================================
version      toolchain                                       
=========    ================================================
``1.1.3``    ``foss/2016a``, ``intel/2016a``                 
``1.2.7``    ``foss/2016a``, ``intel/2016a``                 
``1.3.1``    ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.5.1``    ``intel/2017a``                                 
``1.7.1``    ``foss/2017b``, ``intel/2017b``                 
``1.7.5``    ``foss/2018a``, ``intel/2018a``                 
``1.9.0``    ``fosscuda/2018b``                              
``2.2.0``    ``foss/2018b``, ``fosscuda/2018b``              
``2.4.0``    ``GCCcore/8.2.0``                               
=========    ================================================


.. _list_software_Harminv_725:

*Harminv*
+++++++++

Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion -  given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially  decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those  sinusoids.

*homepage*: http://ab-initio.mit.edu/wiki/index.php/Harminv

=======    ==============
version    toolchain     
=======    ==============
``1.4``    ``foss/2016a``
=======    ==============


.. _list_software_HBase_451:

*HBase*
+++++++

Apache HBase. is the Hadoop database, a distributed, scalable, big data store. 

*homepage*: http://hbase.apache.org/

=========    ==========
version      toolchain 
=========    ==========
``1.0.2``    ``system``
=========    ==========


.. _list_software_HDDM_285:

*HDDM*
++++++

HDDM is a Puthon toolbox for hierarchical Bayesian parameter estimation  of the Drift Diffusion Model (via PyMC).

*homepage*: http://ski.clps.brown.edu/hddm_docs

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.6.1``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_HDF_210:

*HDF*
+++++

HDF (also known as HDF4) is a library and multi-object file format for  storing and managing data between machines. 

*homepage*: http://www.hdfgroup.org/products/hdf4/

==========    ==============    =======================================================
version       versionsuffix     toolchain                                              
==========    ==============    =======================================================
``4.2.11``                      ``intel/2016a``                                        
``4.2.12``                      ``intel/2017a``                                        
``4.2.13``                      ``GCCcore/6.4.0``                                      
``4.2.13``    ``-no-netcdf``    ``intel/2017a``                                        
``4.2.14``                      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
==========    ==============    =======================================================


.. _list_software_HDF5_263:

*HDF5*
++++++

HDF5 is a data model, library, and file format for storing and managing data.  It supports an unlimited variety of datatypes, and is designed for flexible  and efficient I/O and for high volume and complex data.

*homepage*: https://portal.hdfgroup.org/display/support

=================    =============    ==================================================================================================================================================================================================================
version              versionsuffix    toolchain                                                                                                                                                                                                         
=================    =============    ==================================================================================================================================================================================================================
``1.8.10``           ``-serial``      ``GCC/4.8.1``                                                                                                                                                                                                     
``1.8.11``           ``-serial``      ``GCC/4.8.1``                                                                                                                                                                                                     
``1.8.12``                            ``foss/2018b``, ``intel/2016b``                                                                                                                                                                                   
``1.8.13``                            ``foss/2018b``                                                                                                                                                                                                    
``1.8.16``                            ``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``                                                                                                   
``1.8.16``           ``-serial``      ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                                                                                                                                                                 
``1.8.17``                            ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                                                                                                                                                  
``1.8.17``           ``-serial``      ``GCC/5.4.0-2.26``, ``foss/2016a``, ``intel/2016b``                                                                                                                                                               
``1.8.18``                            ``foss/2016b``, ``foss/2017a``, ``gimkl/2017a``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``                                                                                                              
``1.8.18``           ``-serial``      ``intel/2017a``                                                                                                                                                                                                   
``1.8.19``                            ``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``                                                                                                                                                  
``1.8.20``                            ``foss/2018a``, ``intel/2017b``, ``intel/2018a``                                                                                                                                                                  
``1.10.0-patch1``                     ``foss/2016b``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``                                                                                                                                               
``1.10.1``                            ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2017b``, ``iomkl/2018a``
``1.10.2``                            ``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``                                                                                                                 
``1.10.5``                            ``gompi/2019a``, ``gompi/2019b``, ``gompic/2019a``, ``gompic/2019b``, ``iimpi/2019a``, ``iimpi/2019b``, ``iimpic/2019a``                                                                                          
=================    =============    ==================================================================================================================================================================================================================


.. _list_software_hdf5storage_1116:

*hdf5storage*
+++++++++++++

This Python package provides high level utilities to read/write a variety of Python types to/from  HDF5 (Heirarchal Data Format) formatted files. This package also provides support for MATLAB MAT v7.3 formatted  files, which are just HDF5 files with a different extension and some extra meta-data. All of this is done without  pickling data. Pickling is bad for security because it allows arbitrary code to be executed in the interpreter.  One wants to be able to read possibly HDF5 and MAT files from untrusted sources, so pickling is avoided in this  package.

*homepage*: https://pythonhosted.org/hdf5storage/

==========    ==================================
version       toolchain                         
==========    ==================================
``0.1.15``    ``foss/2019a``, ``fosscuda/2019a``
==========    ==================================


.. _list_software_HDFView_621:

*HDFView*
+++++++++

HDFView is a visual tool for browsing and editing HDF4 and HDF5 files.

*homepage*: https://support.hdfgroup.org/products/java/hdfview/

========    ===========================    ==========
version     versionsuffix                  toolchain 
========    ===========================    ==========
``2.14``    ``-Java-1.8.0_152-centos6``    ``system``
``2.14``    ``-Java-1.8.0_152-centos7``    ``system``
========    ===========================    ==========


.. _list_software_HEALPix_587:

*HEALPix*
+++++++++

Hierarchical Equal Area isoLatitude Pixelation of a sphere.

*homepage*: http://healpix.sourceforge.net/

========    ====================================
version     toolchain                           
========    ====================================
``3.50``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
========    ====================================


.. _list_software_heaptrack_947:

*heaptrack*
+++++++++++

A heap memory profiler for Linux.

*homepage*: http://milianw.de/blog/heaptrack-a-heap-memory-profiler-for-linux

=========    ==============
version      toolchain     
=========    ==============
``1.1.0``    ``foss/2016b``
=========    ==============


.. _list_software_Hello_500:

*Hello*
+++++++

The GNU Hello program produces a familiar, friendly greeting. Yes, this is another implementation of the classic program that prints "Hello, world!" when you run it.  However, unlike the minimal version often seen, GNU Hello processes its argument list to modify its behavior, supports greetings in many languages, and so on. 

*homepage*: https://www.gnu.org/software/hello/

========    =================
version     toolchain        
========    =================
``2.10``    ``GCCcore/8.2.0``
========    =================


.. _list_software_help2man_791:

*help2man*
++++++++++

help2man produces simple manual pages from the '--help' and '--version' output of other commands.

*homepage*: https://www.gnu.org/software/help2man/

===========    ==================================================================================================================================================================================
version        toolchain                                                                                                                                                                         
===========    ==================================================================================================================================================================================
``1.47.4``     ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/system``, ``gimkl/2017a``, ``intel/2016b``, ``system``
``1.47.5``     ``GCCcore/5.5.0``                                                                                                                                                                 
``1.47.6``     ``GCCcore/8.1.0``                                                                                                                                                                 
``1.47.7``     ``GCCcore/8.2.0``                                                                                                                                                                 
``1.47.8``     ``GCCcore/7.4.0``, ``GCCcore/8.3.0``                                                                                                                                              
``1.47.10``    ``GCCcore/9.1.0``, ``GCCcore/9.2.0``                                                                                                                                              
===========    ==================================================================================================================================================================================


.. _list_software_HH-suite_743:

*HH-suite*
++++++++++

HH-suite is an open-source software package for sensitive protein sequence searching.  It contains programs that can search for similar protein sequences in protein sequence databases.

*homepage*: https://github.com/soedinglab/hh-suite

==============    ===============
version           toolchain      
==============    ===============
``3.0-beta.3``    ``intel/2018a``
``3.2.0``         ``foss/2019b`` 
==============    ===============


.. _list_software_HiC-Pro_594:

*HiC-Pro*
+++++++++

HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data)   to the normalized contact maps.

*homepage*: http://nservant.github.io/HiC-Pro

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.9.0``    ``-Python-2.7.12``    ``foss/2016b``
=========    ==================    ==============


.. _list_software_HiCExplorer_1093:

*HiCExplorer*
+++++++++++++

HiCexplorer addresses the common tasks of Hi-C analysis from processing to visualization.

*homepage*: http://hicexplorer.readthedocs.org/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``2.1.1``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_HIPS_308:

*HIPS*
++++++

HIPS (Hierarchical Iterative Parallel Solver) is a scientific library that provides  an efficient parallel iterative solver for very large sparse linear systems. 

*homepage*: http://hips.gforge.inria.fr/

============    ==============
version         toolchain     
============    ==============
``1.2b-rc5``    ``foss/2017b``
============    ==============


.. _list_software_HISAT2_427:

*HISAT2*
++++++++

HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads  (both DNA and RNA) against the general human population (as well as against a single reference genome).

*homepage*: https://ccb.jhu.edu/software/hisat2/index.shtml

==============    =================================================================================
version           toolchain                                                                        
==============    =================================================================================
``2.0.3-beta``    ``intel/2016a``                                                                  
``2.0.4``         ``foss/2016b``                                                                   
``2.0.5``         ``intel/2017a``                                                                  
``2.1.0``         ``foss/2017b``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
==============    =================================================================================


.. _list_software_HLAminer_752:

*HLAminer*
++++++++++

HLAminer is a software for HLA predictions from next-generation shotgun (NGS) sequence read data and supports direct  read alignment and targeted de novo assembly of sequence reads. 

*homepage*: http://www.bcgsc.ca/platform/bioinfo/software/hlaminer

=======    ================    ==============
version    versionsuffix       toolchain     
=======    ================    ==============
``1.4``    ``-Perl-5.28.0``    ``foss/2018b``
=======    ================    ==============


.. _list_software_HMMER_377:

*HMMER*
+++++++

HMMER is used for searching sequence databases for homologs  of protein sequences, and for making protein sequence alignments. It  implements methods using probabilistic models called profile hidden Markov  models (profile HMMs).  Compared to BLAST, FASTA, and other sequence  alignment and database search tools based on older scoring methodology,  HMMER aims to be significantly more accurate and more able to detect remote  homologs because of the strength of its underlying mathematical models. In the  past, this strength came at significant computational expense, but in the new  HMMER3 project, HMMER is now essentially as fast as BLAST.

*homepage*: http://hmmer.org/

=========    ============================================================================================================================================
version      toolchain                                                                                                                                   
=========    ============================================================================================================================================
``3.1b2``    ``GCC/6.4.0-2.28``, ``foss/2016a``, ``foss/2016b``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017a``, ``intel/2018a``
``3.2.1``    ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``gompi/2019b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iimpi/2019b``, ``intel/2018b``           
=========    ============================================================================================================================================


.. _list_software_hmmlearn_852:

*hmmlearn*
++++++++++

hmmlearn is a set of algorithms for unsupervised learning and inference of Hidden Markov Models

*homepage*: https://github.com/hmmlearn/hmmlearn

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.2.0``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_HOME_297:

*HOME*
++++++

HOME (histogram of methylation) is a python package for differential methylation region (DMR) identification. The method uses histogram of methylation features and the linear Support Vector Machine (SVM) to identify DMRs  from whole genome bisulfite sequencing (WGBS) data.

*homepage*: https://github.com/ListerLab/HOME

=======    ==================    ==============
version    versionsuffix         toolchain     
=======    ==================    ==============
``0.9``    ``-Python-2.7.13``    ``foss/2017a``
=======    ==================    ==============


.. _list_software_Horovod_737:

*Horovod*
+++++++++

Horovod is a distributed training framework for TensorFlow.

*homepage*: https://github.com/uber/horovod

==========    ==================================    ==================
version       versionsuffix                         toolchain         
==========    ==================================    ==================
``0.9.10``    ``-Python-3.6.3``                     ``intel/2017b``   
``0.18.1``    ``-Python-3.7.2``                     ``foss/2019a``    
``0.18.2``    ``-TensorFlow-2.0.0-Python-3.7.4``    ``fosscuda/2019b``
==========    ==================================    ==================


.. _list_software_HPCG_290:

*HPCG*
++++++

The HPCG Benchmark project is an effort to create a more relevant metric for ranking HPC systems than  the High Performance LINPACK (HPL) benchmark, that is currently used by the TOP500 benchmark.

*homepage*: https://software.sandia.gov/hpcg

=======    ===============================================
version    toolchain                                      
=======    ===============================================
``3.0``    ``foss/2016b``, ``foss/2018b``, ``intel/2018b``
``3.1``    ``foss/2018b``, ``intel/2018b``                
=======    ===============================================


.. _list_software_HPCX_307:

*HPCX*
++++++

The Mellanox HPC-X Toolkit is a comprehensive MPI and SHMEM/PGAS software suite for high performance computing environments

*homepage*: http://www.mellanox.com/page/products_dyn?product_family=189&mtag=hpc-x

=========    ====================
version      toolchain           
=========    ====================
``2.3.0``    ``GCC/8.2.0-2.31.1``
=========    ====================


.. _list_software_HPDBSCAN_579:

*HPDBSCAN*
++++++++++

Highly parallel density based spatial clustering for application with noise

*homepage*: https://bitbucket.org/markus.goetz/hpdbscan

============    ==============
version         toolchain     
============    ==============
``20171110``    ``foss/2017b``
============    ==============


.. _list_software_HPL_228:

*HPL*
+++++

HPL is a software package that solves a (random) dense linear system in double precision (64 bits)  arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available  implementation of the High Performance Computing Linpack Benchmark.

*homepage*: http://www.netlib.org/benchmark/hpl/

=======    =========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version    toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
=======    =========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``2.1``    ``foss/2016.04``, ``foss/2016.06``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gmpolf/2016a``, ``gmvolf/1.7.20``, ``gmvolf/2016a``, ``intel/2016.00``, ``intel/2016.01``, ``intel/2016.02-GCC-4.9``, ``intel/2016.02-GCC-5.3``, ``intel/2016.03-GCC-4.9``, ``intel/2016.03-GCC-5.3``, ``intel/2016.03-GCC-5.4``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``pomkl/2016.03``, ``pomkl/2016.04``, ``pomkl/2016.09``                                                                                                                                                                                                                                                                                                                                                                   
``2.2``    ``foss/2016.07``, ``foss/2016.09``, ``foss/2017a``, ``foss/2017b``, ``foss/2018.08``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``gimkl/2018b``, ``giolf/2017b``, ``giolf/2018a``, ``giolfc/2017b``, ``gmpolf/2017.10``, ``gomkl/2018b``, ``intel/2017.00``, ``intel/2017.01``, ``intel/2017.02``, ``intel/2017.09``, ``intel/2017a``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``, ``intel/2018.02``, ``intel/2018.04``, ``intel/2018a``, ``intel/2018b``, ``intel/2019.00``, ``intel/2019.01``, ``intelcuda/2016.10``, ``intelcuda/2017b``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``iomkl/2016.09-GCC-5.4.0-2.26``, ``iomkl/2017.01``, ``iomkl/2017a``, ``iomkl/2017b``, ``iomkl/2018.02``, ``iomkl/2018a``, ``iomkl/2018b``, ``pomkl/2016.09``
``2.3``    ``CrayCCE/19.06``, ``CrayGNU/19.06``, ``CrayIntel/19.06``, ``foss/2019a``, ``foss/2019b``, ``fosscuda/2019b``, ``gomkl/2019a``, ``intel/2019.02``, ``intel/2019.03``, ``intel/2019a``, ``intel/2019b``, ``intelcuda/2019b``, ``iomkl/2019.01``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
=======    =========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_htop_443:

*htop*
++++++

An interactive process viewer for Unix

*homepage*: http://hisham.hm/htop/

=========    ==========
version      toolchain 
=========    ==========
``2.0.0``    ``system``
``2.0.1``    ``system``
=========    ==========


.. _list_software_HTSeq_453:

*HTSeq*
+++++++

A framework to process and analyze data from high-throughput sequencing (HTS) assays

*homepage*: http://www-huber.embl.de/users/anders/HTSeq/

===========    ==================    ================================================
version        versionsuffix         toolchain                                       
===========    ==================    ================================================
``0.6.1p1``    ``-Python-2.7.11``    ``foss/2016a``                                  
``0.6.1p1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                 
``0.9.1``      ``-Python-2.7.12``    ``foss/2016b``                                  
``0.9.1``      ``-Python-2.7.13``    ``intel/2017a``                                 
``0.9.1``      ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``0.9.1``      ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``                 
``0.10.0``     ``-Python-2.7.14``    ``foss/2018a``                                  
``0.11.0``     ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``                 
``0.11.2``     ``-Python-3.6.6``     ``foss/2018b``                                  
``0.11.2``     ``-Python-3.7.2``     ``foss/2019a``                                  
===========    ==================    ================================================


.. _list_software_HTSlib_550:

*HTSlib*
++++++++

PacBio fork of C library for reading/writing high-throughput sequencing data.  This package includes the utilities bgzip and tabix

*homepage*: https://github.com/PacificBiosciences/htslib

============    =============    ===========================================================================================================================================================
version         versionsuffix    toolchain                                                                                                                                                  
============    =============    ===========================================================================================================================================================
``1.2.1``                        ``foss/2016b``                                                                                                                                             
``1.3``                          ``foss/2016a``, ``intel/2016a``                                                                                                                            
``1.3.1``                        ``foss/2016a``, ``foss/2016b``, ``intel/2016b``                                                                                                            
``1.3.2``                        ``intel/2016b``                                                                                                                                            
``1.4``                          ``foss/2016b``, ``intel/2016b``                                                                                                                            
``1.4.1``                        ``foss/2016a``, ``intel/2017a``                                                                                                                            
``1.6``                          ``foss/2016b``, ``foss/2017b``, ``intel/2017b``                                                                                                            
``1.7``                          ``intel/2018a``                                                                                                                                            
``1.8``                          ``foss/2018a``, ``intel/2018a``                                                                                                                            
``1.9``                          ``GCC/6.4.0-2.28``, ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
``20160107``    ``-PacBio``      ``intel/2017a``                                                                                                                                            
============    =============    ===========================================================================================================================================================


.. _list_software_hub_319:

*hub*
+++++

hub is a command-line wrapper for git that makes you better at GitHub.

*homepage*: https://hub.github.com/

=========    ================    ==========
version      versionsuffix       toolchain 
=========    ================    ==========
``2.2.2``    ``-linux-amd64``    ``system``
=========    ================    ==========


.. _list_software_hunspell_875:

*hunspell*
++++++++++

Hunspell is a spell checker and morphological analyzer library and program designed for languages  with rich morphology and complex word compounding or character encoding.

*homepage*: http://hunspell.github.io/

=========    =================
version      toolchain        
=========    =================
``1.6.1``    ``intel/2017a``  
``1.7.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_hwloc_541:

*hwloc*
+++++++

The Portable Hardware Locality (hwloc) software package provides a portable  abstraction (across OS, versions, architectures, ...) of the hierarchical  topology of modern architectures, including NUMA memory nodes, sockets, shared  caches, cores and simultaneous multithreading. It also gathers various system  attributes such as cache and memory information as well as the locality of I/O  devices such as network interfaces, InfiniBand HCAs or GPUs. It primarily  aims at helping applications with gathering information about modern computing  hardware so as to exploit it accordingly and efficiently. 

*homepage*: https://www.open-mpi.org/projects/hwloc/

===========    =====================================================================================================================================================================================================================================================
version        toolchain                                                                                                                                                                                                                                            
===========    =====================================================================================================================================================================================================================================================
``1.7.2``      ``GCC/4.8.2``                                                                                                                                                                                                                                        
``1.8.1``      ``GCC/4.8.2``, ``GCC/4.8.3``                                                                                                                                                                                                                         
``1.9``        ``GCC/4.8.3``                                                                                                                                                                                                                                        
``1.10.0``     ``GCC/4.9.2``                                                                                                                                                                                                                                        
``1.10.1``     ``GCC/4.8.4``, ``GNU/4.9.2-2.25``                                                                                                                                                                                                                    
``1.11.0``     ``GNU/4.9.3-2.25``                                                                                                                                                                                                                                   
``1.11.1``     ``GCC/4.9.3``                                                                                                                                                                                                                                        
``1.11.2``     ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``                                                                                                                                                                                                               
``1.11.3``     ``GCC/5.2.0``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``PGI/16.3-GCC-4.9.3-2.25``, ``PGI/16.4-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``1.11.4``     ``GCC/6.2.0-2.27``, ``PGI/16.7-GCC-5.4.0-2.26``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``                                                                                                                                                              
``1.11.5``     ``GCC/5.4.0-2.26``, ``GCC/6.3.0-2.27``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``                                                                                                                                                                       
``1.11.6``     ``GCC/6.3.0-2.28``                                                                                                                                                                                                                                   
``1.11.7``     ``GCCcore/5.4.0``, ``GCCcore/6.4.0``                                                                                                                                                                                                                 
``1.11.8``     ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``intel/2017a``                                                                                                                                                                                                
``1.11.10``    ``GCCcore/7.3.0``                                                                                                                                                                                                                                    
``1.11.11``    ``GCCcore/8.2.0``                                                                                                                                                                                                                                    
``1.11.12``    ``GCCcore/8.3.0``                                                                                                                                                                                                                                    
``2.0.2``      ``GCCcore/8.2.0``                                                                                                                                                                                                                                    
``2.0.3``      ``GCCcore/8.3.0``                                                                                                                                                                                                                                    
===========    =====================================================================================================================================================================================================================================================


.. _list_software_Hyperopt_859:

*Hyperopt*
++++++++++

hyperopt is a Python library for optimizing over awkward search spaces with real-valued,  discrete, and conditional dimensions.

*homepage*: http://hyperopt.github.io/hyperopt/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.1``      ``-Python-2.7.14``    ``intel/2017b``
``0.1.1``    ``-Python-3.6.6``     ``intel/2018b``
=========    ==================    ===============


.. _list_software_hyperspy_900:

*hyperspy*
++++++++++

HyperSpy is an open source Python library which provides tools to facilitate the interactive  data analysis of multi-dimensional datasets that can be described as multi-dimensional arrays of a given signal  (e.g. a 2D array of spectra a.k.a spectrum image)

*homepage*: http://hyperspy.org/

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.1.1``    ``-Python-3.5.2``    ``intel/2016b``
=========    =================    ===============


.. _list_software_hypothesis_1104:

*hypothesis*
++++++++++++

Hypothesis is an advanced testing library for Python. It lets you write tests which are parametrized  by a source of examples, and then generates simple and comprehensible examples that make your tests fail. This lets  you find more bugs in your code with less work.

*homepage*: https://github.com/HypothesisWorks/hypothesis

==========    =================    ==================================
version       versionsuffix        toolchain                         
==========    =================    ==================================
``4.5.0``     ``-Python-3.6.6``    ``foss/2018b``, ``fosscuda/2018b``
``4.23.4``                         ``GCCcore/8.2.0``                 
``4.44.2``    ``-Python-3.7.4``    ``GCCcore/8.3.0``                 
==========    =================    ==================================


.. _list_software_Hypre_520:

*Hypre*
+++++++

Hypre is a library for solving large, sparse linear systems of equations on massively  parallel computers. The problems of interest arise in the simulation codes being developed at LLNL  and elsewhere to study physical phenomena in the defense, environmental, energy, and biological sciences.

*homepage*: https://computation.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods

==========    ===============================
version       toolchain                      
==========    ===============================
``2.11.1``    ``foss/2016a``, ``intel/2016a``
``2.14.0``    ``foss/2018a``, ``intel/2018a``
``2.15.1``    ``foss/2019a``, ``intel/2019a``
==========    ===============================


.. _list_software_letter_i:

*I*
---


:ref:`list_software_i-cisTarget_1084` - :ref:`list_software_i-PI_303` - :ref:`list_software_I-TASSER_584` - :ref:`list_software_ICA-AROMA_618` - :ref:`list_software_icc_303` - :ref:`list_software_iccifort_851` - :ref:`list_software_iccifortcuda_1264` - :ref:`list_software_icmake_618` - :ref:`list_software_iCount_626` - :ref:`list_software_ICU_225` - :ref:`list_software_IDBA-UD_470` - :ref:`list_software_ieeg-cli_767` - :ref:`list_software_ifort_548` - :ref:`list_software_igraph_635` - :ref:`list_software_IGV_230` - :ref:`list_software_IGVTools_759` - :ref:`list_software_iimpi_536` - :ref:`list_software_iimpic_635` - :ref:`list_software_IMa2_297` - :ref:`list_software_IMa2p_409` - :ref:`list_software_imageio_731` - :ref:`list_software_ImageJ_557` - :ref:`list_software_ImageMagick_1071` - :ref:`list_software_imake_519` - :ref:`list_software_IMB_216` - :ref:`list_software_imbalanced-learn_1599` - :ref:`list_software_imgaug_634` - :ref:`list_software_imkl_429` - :ref:`list_software_Imlib2_543` - :ref:`list_software_IMOD_297` - :ref:`list_software_impi_431` - :ref:`list_software_IMPUTE2_518` - :ref:`list_software_Inelastica_1021` - :ref:`list_software_Infernal_815` - :ref:`list_software_Infomap_714` - :ref:`list_software_inputproto_1124` - :ref:`list_software_Inspector_951` - :ref:`list_software_IntaRNA_621` - :ref:`list_software_INTEGRATE_675` - :ref:`list_software_INTEGRATE-Neo_1010` - :ref:`list_software_intel_540` - :ref:`list_software_IntelClusterChecker_1939` - :ref:`list_software_intelcuda_953` - :ref:`list_software_IntelPython_1150` - :ref:`list_software_InterProScan_1208` - :ref:`list_software_intltool_885` - :ref:`list_software_io_lib_622` - :ref:`list_software_iomkl_540` - :ref:`list_software_iompi_542` - :ref:`list_software_IOR_234` - :ref:`list_software_IOzone_596` - :ref:`list_software_Ipopt_524` - :ref:`list_software_ipp_329` - :ref:`list_software_IPy_274` - :ref:`list_software_ipyparallel_1183` - :ref:`list_software_ipyrad_649` - :ref:`list_software_IPython_715` - :ref:`list_software_IQ-TREE_503` - :ref:`list_software_IRkernel_796` - :ref:`list_software_IronPython_1050` - :ref:`list_software_ISL_232` - :ref:`list_software_ispc_431` - :ref:`list_software_itac_417` - :ref:`list_software_ITK_232` - :ref:`list_software_ITSTool_654`


.. _list_software_i-cisTarget_1084:

*i-cisTarget*
+++++++++++++

An integrative genomics method for the prediction of regulatory features and cis-regulatory modules  in Human, Mouse, and Fly

*homepage*: https://gbiomed.kuleuven.be/apps/lcb/i-cisTarget

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20160602``    ``-Python-2.7.11``    ``intel/2016a``
============    ==================    ===============


.. _list_software_i-PI_303:

*i-PI*
++++++

A Python wrapper for (ab initio) (path integrals) molecular dynamics

*homepage*: https://github.com/i-pi/i-pi

================    ==================    ===============
version             versionsuffix         toolchain      
================    ==================    ===============
``1.0-20160213``    ``-Python-2.7.11``    ``intel/2016a``
================    ==================    ===============


.. _list_software_I-TASSER_584:

*I-TASSER*
++++++++++

I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function modelling and comparison.

*homepage*: http://zhanglab.ccmb.med.umich.edu/I-TASSER/

=======    ==========
version    toolchain 
=======    ==========
``4.0``    ``system``
``4.2``    ``system``
``5.1``    ``system``
=======    ==========


.. _list_software_ICA-AROMA_618:

*ICA-AROMA*
+++++++++++

ICA-AROMA (i.e. 'ICA-based Automatic Removal Of Motion Artifacts') concerns a data-driven method  to identify and remove motion-related independent components from fMRI data.

*homepage*: https://github.com/maartenmennes/ICA-AROMA

==============    =================    ==============
version           versionsuffix        toolchain     
==============    =================    ==============
``0.4.4-beta``    ``-Python-3.6.6``    ``foss/2018b``
==============    =================    ==============


.. _list_software_icc_303:

*icc*
+++++

C and C++ compiler from Intel

*homepage*: http://software.intel.com/en-us/intel-compilers/

==============    =====================    ==========
version           versionsuffix            toolchain 
==============    =====================    ==========
``2016.0.109``                             ``system``
``2016.0.109``    ``-GCC-4.9.3-2.25``      ``system``
``2016.1.150``    ``-GCC-4.9.3-2.25``      ``system``
``2016.2.181``    ``-GCC-4.9.3-2.25``      ``system``
``2016.2.181``    ``-GCC-5.3.0-2.26``      ``system``
``2016.3.210``    ``-GCC-4.9.3-2.25``      ``system``
``2016.3.210``    ``-GCC-5.3.0-2.26``      ``system``
``2016.3.210``    ``-GCC-5.4.0-2.26``      ``system``
``2017.0.098``    ``-GCC-5.4.0-2.26``      ``system``
``2017.1.132``    ``-GCC-5.4.0-2.26``      ``system``
``2017.1.132``    ``-GCC-6.3.0-2.27``      ``system``
``2017.2.174``    ``-GCC-6.3.0-2.27``      ``system``
``2017.4.196``    ``-GCC-6.4.0-2.28``      ``system``
``2017.5.239``    ``-GCC-6.4.0-2.28``      ``system``
``2017.6.256``    ``-GCC-6.4.0-2.28``      ``system``
``2017.7.259``    ``-GCC-6.4.0-2.28``      ``system``
``2018.0.128``    ``-GCC-6.4.0-2.28``      ``system``
``2018.1.163``    ``-GCC-6.4.0-2.28``      ``system``
``2018.2.199``    ``-GCC-6.4.0-2.28``      ``system``
``2018.3.222``    ``-GCC-7.3.0-2.30``      ``system``
``2018.5.274``    ``-GCC-7.3.0-2.30``      ``system``
``2019.0.117``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.1.144``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.2.187``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.3.199``    ``-GCC-8.3.0-2.32``      ``system``
``system``        ``-GCC-system-2.29``     ``system``
==============    =====================    ==========


.. _list_software_iccifort_851:

*iccifort*
++++++++++

Intel C, C++ & Fortran compilers

*homepage*: http://software.intel.com/en-us/intel-cluster-toolkit-compiler/

==============    =====================    ==========
version           versionsuffix            toolchain 
==============    =====================    ==========
``2016.0.109``                             ``system``
``2016.0.109``    ``-GCC-4.9.3-2.25``      ``system``
``2016.1.150``    ``-GCC-4.9.3-2.25``      ``system``
``2016.2.181``    ``-GCC-4.9.3-2.25``      ``system``
``2016.2.181``    ``-GCC-5.3.0-2.26``      ``system``
``2016.3.210``    ``-GCC-4.9.3-2.25``      ``system``
``2016.3.210``    ``-GCC-5.3.0-2.26``      ``system``
``2016.3.210``    ``-GCC-5.4.0-2.26``      ``system``
``2017.0.098``    ``-GCC-5.4.0-2.26``      ``system``
``2017.1.132``    ``-GCC-5.4.0-2.26``      ``system``
``2017.1.132``    ``-GCC-6.3.0-2.27``      ``system``
``2017.2.174``    ``-GCC-6.3.0-2.27``      ``system``
``2017.4.196``    ``-GCC-6.4.0-2.28``      ``system``
``2017.5.239``    ``-GCC-6.4.0-2.28``      ``system``
``2018.0.128``    ``-GCC-6.4.0-2.28``      ``system``
``2018.1.163``    ``-GCC-6.4.0-2.28``      ``system``
``2018.2.199``    ``-GCC-6.4.0-2.28``      ``system``
``2018.3.222``    ``-GCC-7.3.0-2.30``      ``system``
``2018.5.274``    ``-GCC-7.3.0-2.30``      ``system``
``2019.0.117``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.1.144``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.2.187``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.3.199``    ``-GCC-8.3.0-2.32``      ``system``
``2019.4.243``                             ``system``
``2019.5.281``                             ``system``
``system``        ``-GCC-system-2.29``     ``system``
==============    =====================    ==========


.. _list_software_iccifortcuda_1264:

*iccifortcuda*
++++++++++++++

Intel C, C++ & Fortran compilers with CUDA toolkit

*homepage*: (none)

==============    ===================    ==========
version           versionsuffix          toolchain 
==============    ===================    ==========
``2016.10``                              ``system``
``2017.4.196``    ``-GCC-6.4.0-2.28``    ``system``
``2019a``                                ``system``
``2019b``                                ``system``
==============    ===================    ==========


.. _list_software_icmake_618:

*icmake*
++++++++

Icmake is a hybrid between a 'make' utility and a 'shell script' language. Originally, it was written  to provide a useful tool for automatic program maintenance and system administrative tasks on old MS-DOS platforms.

*homepage*: http://icmake.sourceforge.net/

===========    ==============
version        toolchain     
===========    ==============
``7.23.02``    ``foss/2016a``
===========    ==============


.. _list_software_iCount_626:

*iCount*
++++++++

iCount: protein-RNA interaction analysis  is a Python module and associated command-line interface (CLI),  which provides all the commands needed to process iCLIP data on protein-RNA interactions.

*homepage*: https://github.com/tomazc/iCount

============    =================    ==============
version         versionsuffix        toolchain     
============    =================    ==============
``20180820``    ``-Python-3.6.6``    ``foss/2018b``
============    =================    ==============


.. _list_software_ICU_225:

*ICU*
+++++

ICU is a mature, widely used set of C/C++ and Java libraries providing Unicode and Globalization  support for software applications.

*homepage*: http://site.icu-project.org/home

========    ====================================
version     toolchain                           
========    ====================================
``61.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``64.2``    ``GCCcore/8.2.0``                   
========    ====================================


.. _list_software_IDBA-UD_470:

*IDBA-UD*
+++++++++

IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads  Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.  IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth  contigs are removed iteratively with cutoff threshold from low to high to reduce the errors  in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled  locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD  can iterate k value of de Bruijn graph to a very large value with less gaps and less branches  to form long contigs in both low-depth and high-depth regions.

*homepage*: http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/

=========    ====================================================
version      toolchain                                           
=========    ====================================================
``1.1.3``    ``GCC/8.2.0-2.31.1``, ``foss/2018a``, ``foss/2018b``
=========    ====================================================


.. _list_software_ieeg-cli_767:

*ieeg-cli*
++++++++++

IEEG.ORG is a collaborative initiative funded by the National Institutes  of Neurological Disorders and Stroke. This initiative seeks to advance research towards  the understanding of epilepsy by providing a platform for sharing data,  tools and expertise between researchers.

*homepage*: https://www.ieeg.org/

===========    ==========
version        toolchain 
===========    ==========
``1.14.56``    ``system``
===========    ==========


.. _list_software_ifort_548:

*ifort*
+++++++

Fortran compiler from Intel

*homepage*: http://software.intel.com/en-us/intel-compilers/

==============    =====================    ==========
version           versionsuffix            toolchain 
==============    =====================    ==========
``2016.0.109``                             ``system``
``2016.0.109``    ``-GCC-4.9.3-2.25``      ``system``
``2016.1.150``    ``-GCC-4.9.3-2.25``      ``system``
``2016.2.181``    ``-GCC-4.9.3-2.25``      ``system``
``2016.2.181``    ``-GCC-5.3.0-2.26``      ``system``
``2016.3.210``    ``-GCC-4.9.3-2.25``      ``system``
``2016.3.210``    ``-GCC-5.3.0-2.26``      ``system``
``2016.3.210``    ``-GCC-5.4.0-2.26``      ``system``
``2017.0.098``    ``-GCC-5.4.0-2.26``      ``system``
``2017.1.132``    ``-GCC-5.4.0-2.26``      ``system``
``2017.1.132``    ``-GCC-6.3.0-2.27``      ``system``
``2017.2.174``    ``-GCC-6.3.0-2.27``      ``system``
``2017.4.196``    ``-GCC-6.4.0-2.28``      ``system``
``2017.5.239``    ``-GCC-6.4.0-2.28``      ``system``
``2017.6.256``    ``-GCC-6.4.0-2.28``      ``system``
``2017.7.259``    ``-GCC-6.4.0-2.28``      ``system``
``2018.0.128``    ``-GCC-6.4.0-2.28``      ``system``
``2018.1.163``    ``-GCC-6.4.0-2.28``      ``system``
``2018.2.199``    ``-GCC-6.4.0-2.28``      ``system``
``2018.3.222``    ``-GCC-7.3.0-2.30``      ``system``
``2018.5.274``    ``-GCC-7.3.0-2.30``      ``system``
``2019.0.117``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.1.144``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.2.187``    ``-GCC-8.2.0-2.31.1``    ``system``
``2019.3.199``    ``-GCC-8.3.0-2.32``      ``system``
``system``        ``-GCC-system-2.29``     ``system``
==============    =====================    ==========


.. _list_software_igraph_635:

*igraph*
++++++++

igraph is a collection of network analysis tools with the emphasis on  efficiency, portability and ease of use. igraph is open source and free. igraph can be  programmed in R, Python and C/C++.

*homepage*: http://igraph.org

=========    ================================================
version      toolchain                                       
=========    ================================================
``0.7.1``    ``foss/2018b``, ``intel/2016b``, ``intel/2017b``
=========    ================================================


.. _list_software_IGV_230:

*IGV*
+++++

This package contains command line utilities for  preprocessing, computing feature count density (coverage),  sorting, and  indexing data files.

*homepage*: http://www.broadinstitute.org/software/igv/

==========    ==================    ==========
version       versionsuffix         toolchain 
==========    ==================    ==========
``2.3.68``    ``-Java-1.7.0_80``    ``system``
``2.3.80``    ``-Java-1.7.0_80``    ``system``
``2.5.0``     ``-Java-11``          ``system``
==========    ==================    ==========


.. _list_software_IGVTools_759:

*IGVTools*
++++++++++

This package contains command line utilities for preprocessing,   computing feature count density (coverage),  sorting, and indexing data files.  See also http://www.broadinstitute.org/software/igv/igvtools_commandline. 

*homepage*: https://software.broadinstitute.org/software/igv/igvtools

==========    ==================    ==========
version       versionsuffix         toolchain 
==========    ==================    ==========
``2.3.68``    ``-Java-1.7.0_80``    ``system``
``2.3.72``    ``-Java-1.7.0_80``    ``system``
``2.3.75``    ``-Java-1.7.0_80``    ``system``
``2.4.18``    ``-Java-1.8``         ``system``
==========    ==================    ==========


.. _list_software_iimpi_536:

*iimpi*
+++++++

Intel C/C++ and Fortran compilers, alongside Intel MPI.

*homepage*: http://software.intel.com/en-us/intel-cluster-toolkit-compiler/

===========    ====================    ==========
version        versionsuffix           toolchain 
===========    ====================    ==========
``8.1.5``      ``-GCC-4.9.3-2.25``     ``system``
``2016.00``    ``-GCC-4.9.3-2.25``     ``system``
``2016.01``    ``-GCC-4.9.3-2.25``     ``system``
``2016.02``    ``-GCC-4.9.3-2.25``     ``system``
``2016.02``    ``-GCC-5.3.0-2.26``     ``system``
``2016.03``    ``-GCC-4.9.3-2.25``     ``system``
``2016.03``    ``-GCC-5.3.0-2.26``     ``system``
``2016.03``    ``-GCC-5.4.0-2.26``     ``system``
``2016b``                              ``system``
``2017.00``    ``-GCC-5.4.0-2.26``     ``system``
``2017.01``    ``-GCC-5.4.0-2.26``     ``system``
``2017.02``    ``-GCC-6.3.0-2.27``     ``system``
``2017.09``                            ``system``
``2017a``                              ``system``
``2017b``                              ``system``
``2018.00``                            ``system``
``2018.01``                            ``system``
``2018.02``                            ``system``
``2018.04``                            ``system``
``2018a``                              ``system``
``2018b``                              ``system``
``2019.00``                            ``system``
``2019.01``                            ``system``
``2019.02``                            ``system``
``2019.03``                            ``system``
``2019a``                              ``system``
``2019b``                              ``system``
``system``     ``-GCC-system-2.29``    ``system``
===========    ====================    ==========


.. _list_software_iimpic_635:

*iimpic*
++++++++

Intel C/C++ and Fortran compilers, alongside Intel MPI and CUDA.

*homepage*: (none)

===========    ==========
version        toolchain 
===========    ==========
``2016.10``    ``system``
``2017b``      ``system``
``2019a``      ``system``
``2019b``      ``system``
===========    ==========


.. _list_software_IMa2_297:

*IMa2*
++++++

IMa2 is a progam for population genetic analysis that can handle two or more populations.

*homepage*: https://bio.cst.temple.edu/~hey/software/software.htm#IMa2

===========    ==============
version        toolchain     
===========    ==============
``8.27.12``    ``foss/2016a``
===========    ==============


.. _list_software_IMa2p_409:

*IMa2p*
+++++++

IMa2p is a parallel implementation of IMa2, using OpenMPI-C++ - a Bayesian MCMC  based method for inferring population demography under the IM (Isolation with  Migration) model. http://dx.doi.org/10.1111/1755-0998.12437 

*homepage*: https://github.com/arunsethuraman/ima2p

============    ===============
version         toolchain      
============    ===============
``20151123``    ``foss/2016a`` 
``20160804``    ``intel/2016b``
============    ===============


.. _list_software_imageio_731:

*imageio*
+++++++++

Imageio is a Python library that provides an easy interface to read and write a wide range of  image data, including animated images, video, volumetric data, and scientific formats.

*homepage*: https://imageio.github.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``2.3.0``    ``-Python-3.6.4``    ``intel/2018a``
``2.5.0``                         ``foss/2019a`` 
=========    =================    ===============


.. _list_software_ImageJ_557:

*ImageJ*
++++++++

Image Processing and Analysis in Java

*homepage*: https://imagej.nih.gov/ij

=========    ==========
version      toolchain 
=========    ==========
``1.51a``    ``system``
``1.51i``    ``system``
``1.51k``    ``system``
=========    ==========


.. _list_software_ImageMagick_1071:

*ImageMagick*
+++++++++++++

ImageMagick is a software suite to create, edit, compose, or convert bitmap images

*homepage*: http://www.imagemagick.org/

============    ===================================    ===============================
version         versionsuffix                          toolchain                      
============    ===================================    ===============================
``6.9.4-8``                                            ``intel/2016a``                
``7.0.1-6``                                            ``intel/2016a``                
``7.0.1-9``                                            ``intel/2016a``                
``7.0.2-9``                                            ``intel/2016a``                
``7.0.3-1``                                            ``intel/2016b``                
``7.0.5-4``                                            ``intel/2017a``                
``7.0.5-10``                                           ``foss/2016b``                 
``7.0.7-8``     ``-JasPer-1.900.1``                    ``intel/2017a``                
``7.0.7-14``                                           ``foss/2017b``, ``intel/2017b``
``7.0.7-15``                                           ``GCCcore/6.4.0``              
``7.0.7-26``                                           ``foss/2018a``                 
``7.0.7-30``                                           ``GCCcore/6.4.0``              
``7.0.7-30``    ``-Ghostscript-9.22-cairo-1.14.12``    ``GCCcore/6.4.0``              
``7.0.7-39``    ``-Ghostscript-9.23-cairo-1.14.12``    ``GCCcore/6.4.0``              
``7.0.8-11``                                           ``GCCcore/7.3.0``              
``7.0.8-46``                                           ``GCCcore/8.2.0``              
============    ===================================    ===============================


.. _list_software_imake_519:

*imake*
+++++++

imake is a Makefile-generator that is intended to make it easier to develop software  portably for multiple systems.

*homepage*: http://www.x.org/

=========    ===============
version      toolchain      
=========    ===============
``1.0.7``    ``intel/2016a``
=========    ===============


.. _list_software_IMB_216:

*IMB*
+++++

The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and  global communication operations for a range of message sizes

*homepage*: https://software.intel.com/en-us/articles/intel-mpi-benchmarks

==========    ==================================================================
version       toolchain                                                         
==========    ==================================================================
``4.1``       ``foss/2016a``, ``foss/2017a``, ``intel/2017.02``, ``intel/2017a``
``2018.1``    ``intel/2017a``                                                   
``2019.3``    ``gompi/2019a``, ``iimpi/2019a``                                  
==========    ==================================================================


.. _list_software_imbalanced-learn_1599:

*imbalanced-learn*
++++++++++++++++++

imbalanced-learn is a Python package offering a number of re-sampling techniques commonly used in  datasets showing strong between-class imbalance.

*homepage*: https://github.com/scikit-learn-contrib/imbalanced-learn

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.2.1``    ``-Python-2.7.12``    ``intel/2016b``
``0.2.1``    ``-Python-3.5.2``     ``intel/2016b``
``0.3.3``    ``-Python-3.6.4``     ``foss/2018a`` 
``0.4.3``    ``-Python-3.6.6``     ``foss/2018b`` 
=========    ==================    ===============


.. _list_software_imgaug_634:

*imgaug*
++++++++

This python library helps you with augmenting images for your machine learning projects.  It converts a set of input images into a new, much larger set of slightly altered images. 

*homepage*: https://imgaug.readthedocs.io/en/latest/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.2.8``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_imkl_429:

*imkl*
++++++

Intel Math Kernel Library is a library of highly optimized,  extensively threaded math routines for science, engineering, and financial  applications that require maximum performance. Core math functions include  BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more.

*homepage*: https://software.intel.com/en-us/intel-mkl/

==============    ======================================================================================================================================================================================================================================================================
version           toolchain                                                                                                                                                                                                                                                             
==============    ======================================================================================================================================================================================================================================================================
``11.2.3.187``    ``gimpi/2.11.5``                                                                                                                                                                                                                                                      
``11.3.0.109``    ``iimpi/2016.00-GCC-4.9.3-2.25``                                                                                                                                                                                                                                      
``11.3.1.150``    ``iimpi/2016.01-GCC-4.9.3-2.25``, ``iimpi/8.1.5-GCC-4.9.3-2.25``                                                                                                                                                                                                      
``11.3.2.181``    ``iimpi/2016.02-GCC-4.9.3-2.25``, ``iimpi/2016.02-GCC-5.3.0-2.26``, ``pompi/2016.03``                                                                                                                                                                                 
``11.3.3.210``    ``iimpi/2016.03-GCC-4.9.3-2.25``, ``iimpi/2016.03-GCC-5.3.0-2.26``, ``iimpi/2016.03-GCC-5.4.0-2.26``, ``iimpi/2016b``, ``iimpic/2016.10``, ``iompi/2016.07``, ``iompi/2016.09-GCC-4.9.3-2.25``, ``iompi/2016.09-GCC-5.4.0-2.26``, ``pompi/2016.04``, ``pompi/2016.09``
``2017.0.098``    ``iimpi/2017.00-GCC-5.4.0-2.26``                                                                                                                                                                                                                                      
``2017.1.132``    ``gimpi/2017a``, ``iimpi/2017.01-GCC-5.4.0-2.26``, ``iimpi/2017a``, ``iompi/2017.01``, ``iompi/2017a``                                                                                                                                                                
``2017.2.174``    ``iimpi/2017.02-GCC-6.3.0-2.27``                                                                                                                                                                                                                                      
``2017.3.196``    ``gompi/2017b``, ``iimpi/2017b``, ``iimpic/2017b``, ``iompi/2017b``                                                                                                                                                                                                   
``2017.4.239``    ``iimpi/2017.09``                                                                                                                                                                                                                                                     
``2018.0.128``    ``iimpi/2018.00``                                                                                                                                                                                                                                                     
``2018.1.163``    ``iimpi/2018.01``, ``iimpi/2018a``, ``iompi/2018a``                                                                                                                                                                                                                   
``2018.2.199``    ``iimpi/2018.02``, ``iompi/2018.02``                                                                                                                                                                                                                                  
``2018.3.222``    ``gimpi/2018b``, ``gompi/2018b``, ``iimpi/2018b``, ``iompi/2018b``                                                                                                                                                                                                    
``2018.4.274``    ``iimpi/2018.04``                                                                                                                                                                                                                                                     
``2019.0.117``    ``iimpi/2019.00``                                                                                                                                                                                                                                                     
``2019.1.144``    ``gompi/2019a``, ``iimpi/2019.01``, ``iimpi/2019a``, ``iimpic/2019a``, ``iompi/2019.01``                                                                                                                                                                              
``2019.2.187``    ``iimpi/2019.02``                                                                                                                                                                                                                                                     
``2019.3.199``    ``iimpi/2019.03``                                                                                                                                                                                                                                                     
``2019.5.281``    ``iimpi/2019b``, ``iimpic/2019b``                                                                                                                                                                                                                                     
==============    ======================================================================================================================================================================================================================================================================


.. _list_software_Imlib2_543:

*Imlib2*
++++++++

This is the Imlib 2 library - a library that does image file loading and saving as well as rendering, manipulation, arbitrary polygon support, etc. It does ALL of these operations FAST. Imlib2 also tries to be highly intelligent about doing them, so writing naive programs can be done easily, without sacrificing speed. 

*homepage*: https://docs.enlightenment.org/api/imlib2/html/

=========    =================
version      toolchain        
=========    =================
``1.5.1``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_IMOD_297:

*IMOD*
++++++

IMOD is a set of image processing, modeling and display programs used for tomographic reconstruction and for 3D reconstruction of EM serial sections and optical sections. The package contains tools for assembling and aligning data within multiple types and sizes of image stacks, viewing 3-D data from any orientation, and modeling and display of the image files. IMOD was developed primarily by David Mastronarde, Rick Gaudette, Sue Held, Jim Kremer, Quanren Xiong, and John Heumann at the University of Colorado.

*homepage*: http://bio3d.colorado.edu/imod/

==========    =====================    ==========
version       versionsuffix            toolchain 
==========    =====================    ==========
``4.7.15``    ``_RHEL6-64_CUDA6.0``    ``system``
==========    =====================    ==========


.. _list_software_impi_431:

*impi*
++++++

The Intel(R) MPI Library for Linux* OS is a multi-fabric message  passing library based on ANL MPICH2 and OSU MVAPICH2. The Intel MPI Library for  Linux OS implements the Message Passing Interface, version 2 (MPI-2) specification.

*homepage*: http://software.intel.com/en-us/intel-mpi-library/

==============    =============    ================================================================================================================================================================================================================================
version           versionsuffix    toolchain                                                                                                                                                                                                                       
==============    =============    ================================================================================================================================================================================================================================
``3.2.2.006``                      ``system``                                                                                                                                                                                                                      
``4.0.0.028``                      ``system``                                                                                                                                                                                                                      
``4.0.0.028``     ``-32bit``       ``system``                                                                                                                                                                                                                      
``4.0.2.003``                      ``system``                                                                                                                                                                                                                      
``4.1.0.027``                      ``system``                                                                                                                                                                                                                      
``4.1.0.030``                      ``system``                                                                                                                                                                                                                      
``4.1.1.036``                      ``system``                                                                                                                                                                                                                      
``4.1.2.040``                      ``system``                                                                                                                                                                                                                      
``4.1.3.045``                      ``system``                                                                                                                                                                                                                      
``4.1.3.049``                      ``GCC/4.8.3``, ``system``                                                                                                                                                                                                       
``5.0.3.048``                      ``GCC/4.9.3``                                                                                                                                                                                                                   
``5.1.1.109``                      ``iccifort/2016.0.109-GCC-4.9.3-2.25``                                                                                                                                                                                          
``5.1.2.150``                      ``iccifort/2016.1.150-GCC-4.9.3-2.25``                                                                                                                                                                                          
``5.1.3.181``                      ``iccifort/2016.2.181-GCC-4.9.3-2.25``, ``iccifort/2016.2.181-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.3.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``iccifortcuda/2016.10``
``2017.0.098``                     ``iccifort/2017.0.098-GCC-5.4.0-2.26``                                                                                                                                                                                          
``2017.1.132``                     ``GCC/5.4.0-2.26``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``                                                                                                                              
``2017.2.174``                     ``iccifort/2017.2.174-GCC-6.3.0-2.27``                                                                                                                                                                                          
``2017.3.196``                     ``GCC/6.4.0-2.28``, ``gcccuda/2017b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifortcuda/2017.4.196-GCC-6.4.0-2.28``                                                                                                       
``2017.4.239``                     ``iccifort/2017.5.239-GCC-6.4.0-2.28``                                                                                                                                                                                          
``2018.0.128``                     ``iccifort/2018.0.128-GCC-6.4.0-2.28``                                                                                                                                                                                          
``2018.1.163``                     ``GCC/6.4.0-2.28``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``                                                                                                                                                                      
``2018.2.199``                     ``iccifort/2018.2.199-GCC-6.4.0-2.28``                                                                                                                                                                                          
``2018.3.222``                     ``GCC/7.3.0-2.30``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``                                                                                                                                                                      
``2018.4.274``                     ``iccifort/2018.5.274-GCC-7.3.0-2.30``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``iccifortcuda/2019a``                                                                                                                        
``2018.5.288``                     ``iccifort/2019.5.281``, ``iccifortcuda/2019b``                                                                                                                                                                                 
``2019.0.117``                     ``iccifort/2019.0.117-GCC-8.2.0-2.31.1``                                                                                                                                                                                        
``2019.1.144``                     ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``                                                                                                                                                                                        
``2019.2.187``                     ``iccifort/2019.2.187-GCC-8.2.0-2.31.1``                                                                                                                                                                                        
``2019.3.199``                     ``iccifort/2019.3.199-GCC-8.3.0-2.32``                                                                                                                                                                                          
``system``                         ``iccifort/system-GCC-system-2.29``                                                                                                                                                                                             
==============    =============    ================================================================================================================================================================================================================================


.. _list_software_IMPUTE2_518:

*IMPUTE2*
+++++++++

IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation   and haplotype phasing program based on ideas from Howie et al. 2009 

*homepage*: http://mathgen.stats.ox.ac.uk/impute/impute_v2.html

=========    ===================    ==========
version      versionsuffix          toolchain 
=========    ===================    ==========
``2.3.0``    ``_x86_64_dynamic``    ``system``
``2.3.0``    ``_x86_64_static``     ``system``
``2.3.2``    ``_x86_64_dynamic``    ``system``
``2.3.2``    ``_x86_64_static``     ``system``
=========    ===================    ==========


.. _list_software_Inelastica_1021:

*Inelastica*
++++++++++++

Python package for eigenchannels, vibrations and inelastic electron transport based on SIESTA/TranSIESTA DFT.

*homepage*: https://github.com/tfrederiksen/inelastica

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.3.5``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_Infernal_815:

*Infernal*
++++++++++

Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases  for RNA structure and sequence similarities.

*homepage*: http://eddylab.org/infernal/

=========    ================================================================
version      toolchain                                                       
=========    ================================================================
``1.1.2``    ``foss/2016b``, ``foss/2018b``, ``intel/2017a``, ``intel/2018b``
=========    ================================================================


.. _list_software_Infomap_714:

*Infomap*
+++++++++

Multi-level network clustering based on the Map equation.

*homepage*: https://www.mapequation.org/code.html#Linux

============    ====================
version         toolchain           
============    ====================
``20190308``    ``GCC/8.2.0-2.31.1``
============    ====================


.. _list_software_inputproto_1124:

*inputproto*
++++++++++++

X.org InputProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``2.3.1``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``2.3.2``    ``intel/2016a``                                  
=========    =================================================


.. _list_software_Inspector_951:

*Inspector*
+++++++++++

Intel Inspector XE is an easy to use memory error checker and thread checker for serial  and parallel applications

*homepage*: https://software.intel.com/en-us/intel-inspector-xe

================    ==========
version             toolchain 
================    ==========
``2013_update6``    ``system``
``2013_update7``    ``system``
``2016_update3``    ``system``
``2017_update1``    ``system``
``2017_update2``    ``system``
``2018_update1``    ``system``
``2018_update2``    ``system``
``2018_update3``    ``system``
``2019_update2``    ``system``
``2019_update5``    ``system``
================    ==========


.. _list_software_IntaRNA_621:

*IntaRNA*
+++++++++

Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites

*homepage*: https://github.com/BackofenLab/IntaRNA

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.3.1``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_INTEGRATE_675:

*INTEGRATE*
+++++++++++

INTEGRATE is a tool calling gene fusions with exact fusion junctions and genomic breakpoints  by combining RNA-Seq and WGS data. It is highly sensitive and accurate by applying a fast split-read  mapping algorithm based on Burrow-Wheeler transform. 

*homepage*: https://sourceforge.net/p/integrate-fusion/wiki/Home/

=========    =================
version      toolchain        
=========    =================
``0.2.6``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_INTEGRATE-Neo_1010:

*INTEGRATE-Neo*
+++++++++++++++

INTEGRATE-Neo is a gene fusion neoantigen discovering tool using next-generation sequencing data.  It is written in C++ and Python. 

*homepage*: 

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.2.1``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_intel_540:

*intel*
+++++++

Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).

*homepage*: https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain

===========    =============    ==========
version        versionsuffix    toolchain 
===========    =============    ==========
``2016.00``                     ``system``
``2016.01``                     ``system``
``2016.02``    ``-GCC-4.9``     ``system``
``2016.02``    ``-GCC-5.3``     ``system``
``2016.03``    ``-GCC-4.9``     ``system``
``2016.03``    ``-GCC-5.3``     ``system``
``2016.03``    ``-GCC-5.4``     ``system``
``2016a``                       ``system``
``2016b``                       ``system``
``2017.00``                     ``system``
``2017.01``                     ``system``
``2017.02``                     ``system``
``2017.09``                     ``system``
``2017a``                       ``system``
``2017b``                       ``system``
``2018.00``                     ``system``
``2018.01``                     ``system``
``2018.02``                     ``system``
``2018.04``                     ``system``
``2018a``                       ``system``
``2018b``                       ``system``
``2019.00``                     ``system``
``2019.01``                     ``system``
``2019.02``                     ``system``
``2019.03``                     ``system``
``2019a``                       ``system``
``2019b``                       ``system``
===========    =============    ==========


.. _list_software_IntelClusterChecker_1939:

*IntelClusterChecker*
+++++++++++++++++++++

Verifies cluster components work together ― for   - better uptime and productivity and lower total  - cost of ownership (TCO)  

*homepage*: https://software.intel.com/en-us/intel-cluster-checker

==============    ==========
version           toolchain 
==============    ==========
``2017.1.016``    ``system``
==============    ==========


.. _list_software_intelcuda_953:

*intelcuda*
+++++++++++

Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers,  Intel MPI & Intel MKL, with CUDA toolkit

*homepage*: (none)

===========    ==========
version        toolchain 
===========    ==========
``2016.10``    ``system``
``2017b``      ``system``
``2019a``      ``system``
``2019b``      ``system``
===========    ==========


.. _list_software_IntelPython_1150:

*IntelPython*
+++++++++++++

Intel® Distribution for Python. Powered by Anaconda.  Accelerating Python* performance on modern architectures from Intel. 

*homepage*: https://software.intel.com/en-us/intel-distribution-for-python

==========    ===============    ==========
version       versionsuffix      toolchain 
==========    ===============    ==========
``2.7.15``    ``-2019.2.066``    ``system``
``3.6.8``     ``-2019.2.066``    ``system``
==========    ===============    ==========


.. _list_software_InterProScan_1208:

*InterProScan*
++++++++++++++

InterProScan is a sequence analysis application (nucleotide and protein sequences) that combines  different protein signature recognition methods into one resource.

*homepage*: http://www.ebi.ac.uk/interpro/

=============    ===============
version          toolchain      
=============    ===============
``5.26-65.0``    ``intel/2017b``
``5.27-66.0``    ``intel/2017b``
``5.28-67.0``    ``intel/2018a``
=============    ===============


.. _list_software_intltool_885:

*intltool*
++++++++++

intltool is a set of tools to centralize translation of  many different file formats using GNU gettext-compatible PO files.

*homepage*: http://freedesktop.org/wiki/Software/intltool/

==========    ================    ======================================================================================
version       versionsuffix       toolchain                                                                             
==========    ================    ======================================================================================
``0.51.0``                        ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                  
``0.51.0``    ``-Perl-5.20.3``    ``intel/2016a``                                                                       
``0.51.0``    ``-Perl-5.22.1``    ``foss/2016a``, ``intel/2016a``                                                       
``0.51.0``    ``-Perl-5.24.0``    ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``0.51.0``    ``-Perl-5.24.1``    ``GCCcore/6.3.0``, ``intel/2017a``                                                    
``0.51.0``    ``-Perl-5.26.0``    ``GCCcore/6.4.0``                                                                     
``0.51.0``    ``-Perl-5.26.1``    ``GCCcore/6.4.0``                                                                     
``0.51.0``    ``-Perl-5.28.0``    ``GCCcore/7.3.0``                                                                     
==========    ================    ======================================================================================


.. _list_software_io_lib_622:

*io_lib*
++++++++

Io_lib is a library of file reading and writing code to provide a general purpose trace file (and  Experiment File) reading interface. The programmer simply calls the (eg) read_reading to create a "Read" C structure  with the data loaded into memory. It has been compiled and tested on a variety of unix systems, MacOS X and MS   Windows.

*homepage*: http://sourceforge.net/projects/staden/files/io_lib/

==========    ==============
version       toolchain     
==========    ==============
``1.14.8``    ``foss/2016a``
==========    ==============


.. _list_software_iomkl_540:

*iomkl*
+++++++

Intel Cluster Toolchain Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MKL &  OpenMPI.

*homepage*: http://software.intel.com/en-us/intel-cluster-toolkit-compiler/

==========================    ==========
version                       toolchain 
==========================    ==========
``2016.07``                   ``system``
``2016.09-GCC-4.9.3-2.25``    ``system``
``2016.09-GCC-5.4.0-2.26``    ``system``
``2017.01``                   ``system``
``2017a``                     ``system``
``2017b``                     ``system``
``2018.02``                   ``system``
``2018a``                     ``system``
``2018b``                     ``system``
``2019.01``                   ``system``
==========================    ==========


.. _list_software_iompi_542:

*iompi*
+++++++

Intel C/C++ and Fortran compilers, alongside Open MPI.

*homepage*: http://software.intel.com/en-us/intel-cluster-toolkit-compiler/

==========================    ==========
version                       toolchain 
==========================    ==========
``2016.07``                   ``system``
``2016.09-GCC-4.9.3-2.25``    ``system``
``2016.09-GCC-5.4.0-2.26``    ``system``
``2017.01``                   ``system``
``2017a``                     ``system``
``2017b``                     ``system``
``2018.02``                   ``system``
``2018a``                     ``system``
``2018b``                     ``system``
``2019.01``                   ``system``
==========================    ==========


.. _list_software_IOR_234:

*IOR*
+++++

The IOR software is used for benchmarking parallel file systems using POSIX, MPIIO,  or HDF5 interfaces. 

*homepage*: https://github.com/IOR-LANL/ior

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``3.0.1``    ``-mpiio``       ``foss/2016a`` 
``3.2.1``                     ``gompi/2019b``
=========    =============    ===============


.. _list_software_IOzone_596:

*IOzone*
++++++++

IOzone is a filesystem benchmark tool. The benchmark generates and measures a variety of file  operations. Iozone has been ported to many machines and runs under many operating systems.

*homepage*: http://www.iozone.org/

=========    ==============
version      toolchain     
=========    ==============
``3.434``    ``foss/2016a``
=========    ==============


.. _list_software_Ipopt_524:

*Ipopt*
+++++++

IPOPT (Interior Point Optimizer, pronounced Eye-Pea-Opt)  is an open source software package for large-scale nonlinear optimization. 

*homepage*: https://projects.coin-or.org/Ipopt

===========    ===============
version        toolchain      
===========    ===============
``3.12.9``     ``foss/2017b`` 
``3.12.13``    ``intel/2019a``
===========    ===============


.. _list_software_ipp_329:

*ipp*
+++++

Intel Integrated Performance Primitives (Intel IPP) is an extensive library  of multicore-ready, highly optimized software functions for multimedia, data processing,  and communications applications. Intel IPP offers thousands of optimized functions  covering frequently used fundamental algorithms.

*homepage*: http://software.intel.com/en-us/articles/intel-ipp/

==============    ==========
version           toolchain 
==============    ==========
``7.0.5.233``     ``system``
``8.1.0.144``     ``system``
``9.0.1.150``     ``system``
``2017.1.132``    ``system``
==============    ==========


.. _list_software_IPy_274:

*IPy*
+++++

Class and tools for handling of IPv4 and IPv6 addresses and networks

*homepage*: https://pypi.python.org/pypi/IPy

========    ==========
version     toolchain 
========    ==========
``0.83``    ``system``
========    ==========


.. _list_software_ipyparallel_1183:

*ipyparallel*
+++++++++++++

ipyparallel is a Python package and collection  of CLI scripts for controlling clusters for Jupyter

*homepage*: https://ipyparallel.readthedocs.io

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``6.2.2``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_ipyrad_649:

*ipyrad*
++++++++

ipyrad is an interactive toolkit for assembly and analysis of restriction-site associated genomic  data sets (e.g., RAD, ddRAD, GBS) for population genetic and phylogenetic studies.

*homepage*: https://ipyrad.readthedocs.io

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.6.15``    ``-Python-2.7.13``    ``intel/2017a``
==========    ==================    ===============


.. _list_software_IPython_715:

*IPython*
+++++++++

IPython provides a rich architecture for interactive computing with:  Powerful interactive shells (terminal and Qt-based).  A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.  Support for interactive data visualization and use of GUI toolkits.  Flexible, embeddable interpreters to load into your own projects.  Easy to use, high performance tools for parallel computing.

*homepage*: https://ipython.org/index.html

=========    ==================    =======================================================================================
version      versionsuffix         toolchain                                                                              
=========    ==================    =======================================================================================
``3.2.3``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                                                        
``4.2.0``    ``-Python-2.7.11``    ``intel/2016a``                                                                        
``5.0.0``    ``-Python-2.7.11``    ``foss/2016a``                                                                         
``5.0.0``    ``-Python-3.5.1``     ``foss/2016a``                                                                         
``5.1.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                                        
``5.1.0``    ``-Python-3.5.2``     ``intel/2016b``                                                                        
``5.2.2``    ``-Python-2.7.12``    ``intel/2016b``                                                                        
``5.3.0``    ``-Python-2.7.13``    ``intel/2017a``                                                                        
``5.7.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``                                                        
``5.8.0``    ``-Python-2.7.14``    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``               
``5.8.0``    ``-Python-2.7.15``    ``foss/2018b``, ``foss/2019a``, ``fosscuda/2018b``, ``fosscuda/2019a``, ``intel/2018b``
``5.8.0``    ``-Python-3.6.3``     ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``               
``6.2.1``    ``-Python-3.6.4``     ``foss/2017a``                                                                         
``6.3.1``    ``-Python-3.6.4``     ``intel/2018a``                                                                        
``6.4.0``    ``-Python-3.6.4``     ``foss/2018a``                                                                         
``7.2.0``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                                    
``7.7.0``    ``-Python-3.7.2``     ``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``                                    
``7.9.0``    ``-Python-3.7.4``     ``foss/2019b``                                                                         
=========    ==================    =======================================================================================


.. _list_software_IQ-TREE_503:

*IQ-TREE*
+++++++++

Efficient phylogenomic software by maximum likelihood

*homepage*: http://www.iqtree.org/

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``1.5.5``    ``-omp-mpi``     ``foss/2016a`` 
``1.6.6``                     ``intel/2018a``
=========    =============    ===============


.. _list_software_IRkernel_796:

*IRkernel*
++++++++++

The R kernel for the 'Jupyter' environment executes R code   which the front-end (Jupyter Notebook or other front-ends) submits to the   kernel via the network.

*homepage*: https://irkernel.github.io

==========    ==========================    ===============================
version       versionsuffix                 toolchain                      
==========    ==========================    ===============================
``0.8.15``    ``-R-3.4.3-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
==========    ==========================    ===============================


.. _list_software_IronPython_1050:

*IronPython*
++++++++++++

IronPython is an open-source implementation of the Python programming language  which is tightly integrated with the .NET Framework. IronPython can use the .NET Framework and Python libraries, and other .NET languages can use Python code just as easily.

*homepage*: http://ironpython.net/

=======    ===============
version    toolchain      
=======    ===============
``2.7``    ``intel/2016b``
=======    ===============


.. _list_software_ISL_232:

*ISL*
+++++

isl is a library for manipulating sets and relations of integer points bounded by linear constraints.

*homepage*: http://isl.gforge.inria.fr/

========    ======================================================
version     toolchain                                             
========    ======================================================
``0.14``    ``GCC/4.9.2``                                         
``0.15``    ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``
``0.16``    ``GCC/4.9.3-2.25``                                    
``0.17``    ``foss/2016a``                                        
========    ======================================================


.. _list_software_ispc_431:

*ispc*
++++++

Intel SPMD Program Compilers; An open-source compiler for high-performance  SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,  with extensions for 'single program, multiple data' (SPMD) programming.  Under the SPMD model, the programmer writes a program that generally appears  to be a regular serial program, though the execution model is actually that  a number of program instances execute in parallel on the hardware.

*homepage*: http://ispc.github.io/ , https://github.com/ispc/ispc/

==========    ==========
version       toolchain 
==========    ==========
``1.6.0``     ``system``
``1.10.0``    ``system``
==========    ==========


.. _list_software_itac_417:

*itac*
++++++

The Intel Trace Collector is a low-overhead tracing library that performs  event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application  activities gathered by the Intel Trace Collector through graphical displays. 

*homepage*: http://software.intel.com/en-us/intel-trace-analyzer/

==============    ==========
version           toolchain 
==============    ==========
``8.0.0.011``     ``system``
``8.1.4.045``     ``system``
``9.0.3.051``     ``system``
``2017.1.024``    ``system``
``2018.1.017``    ``system``
``2018.3.022``    ``system``
``2019.2.026``    ``system``
==============    ==========


.. _list_software_ITK_232:

*ITK*
+++++

Insight Segmentation and Registration Toolkit (ITK) provides  an extensive suite of software tools for registering and segmenting  multidimensional imaging data.

*homepage*: https://itk.org

==========    ==================    ==============
version       versionsuffix         toolchain     
==========    ==================    ==============
``4.12.2``    ``-Python-2.7.12``    ``foss/2016b``
``4.13.0``    ``-Python-2.7.14``    ``foss/2018a``
``4.13.0``    ``-Python-3.6.4``     ``foss/2018a``
``4.13.1``    ``-Python-2.7.14``    ``foss/2018a``
``4.13.1``    ``-Python-2.7.15``    ``foss/2018b``
``4.13.1``    ``-Python-3.6.4``     ``foss/2018a``
``4.13.1``    ``-Python-3.6.6``     ``foss/2018b``
``5.0.1``     ``-Python-3.7.2``     ``foss/2019a``
``5.0b01``    ``-Python-3.6.6``     ``foss/2018b``
==========    ==================    ==============


.. _list_software_ITSTool_654:

*ITSTool*
+++++++++

ITS Tool allows you to translate your XML documents with PO files

*homepage*: http://itstool.org/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.0.5``    ``-Python-2.7.14``    ``intel/2018a``
``2.0.5``    ``-Python-2.7.15``    ``foss/2018b`` 
=========    ==================    ===============


.. _list_software_letter_j:

*J*
---


:ref:`list_software_JAGS_293` - :ref:`list_software_Jansson_732` - :ref:`list_software_JasPer_581` - :ref:`list_software_Java_386` - :ref:`list_software_JAXFrontCE_884` - :ref:`list_software_jbigkit_740` - :ref:`list_software_Jellyfish_938` - :ref:`list_software_jemalloc_839` - :ref:`list_software_jhbuild_738` - :ref:`list_software_JiTCODE_546` - :ref:`list_software_jModelTest_1019` - :ref:`list_software_jq_219` - :ref:`list_software_JsonCpp_701` - :ref:`list_software_JUBE_294` - :ref:`list_software_Judy_412` - :ref:`list_software_Julia_501` - :ref:`list_software_JUnit_490` - :ref:`list_software_jupyterhub_1106` - :ref:`list_software_JWM_238`


.. _list_software_JAGS_293:

*JAGS*
++++++

JAGS is Just Another Gibbs Sampler.  It is a program for analysis   of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  

*homepage*: http://mcmc-jags.sourceforge.net/

=========    ================================================
version      toolchain                                       
=========    ================================================
``4.2.0``    ``foss/2016a``, ``intel/2016a``, ``intel/2017a``
``4.3.0``    ``foss/2017b``, ``foss/2018b``, ``intel/2017b`` 
=========    ================================================


.. _list_software_Jansson_732:

*Jansson*
+++++++++

Jansson is a C library for encoding, decoding and manipulating JSON data.  Its main features and design principles are:  * Simple and intuitive API and data model  * Comprehensive documentation  * No dependencies on other libraries  * Full Unicode support (UTF-8)  * Extensive test suite

*homepage*: http://www.digip.org/jansson/

=======    =============
version    toolchain    
=======    =============
``2.6``    ``GCC/4.8.3``
=======    =============


.. _list_software_JasPer_581:

*JasPer*
++++++++

The JasPer Project is an open-source initiative to provide a free  software-based reference implementation of the codec specified in  the JPEG-2000 Part-1 standard. 

*homepage*: https://www.ece.uvic.ca/~frodo/jasper/

===========    ==========================================================================================================================================================
version        toolchain                                                                                                                                                 
===========    ==========================================================================================================================================================
``1.900.1``    ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``2.0.10``     ``intel/2016b``                                                                                                                                           
``2.0.12``     ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2017a``                                                                                                        
``2.0.14``     ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                
===========    ==========================================================================================================================================================


.. _list_software_Java_386:

*Java*
++++++

Java Platform, Standard Edition (Java SE) lets you develop and deploy  Java applications on desktops and servers.

*homepage*: http://java.com/

=======================    ==========
version                    toolchain 
=======================    ==========
``1.6.0_24``               ``system``
``1.7.0_10``               ``system``
``1.7.0_15``               ``system``
``1.7.0_21``               ``system``
``1.7.0_40``               ``system``
``1.7.0_45``               ``system``
``1.7.0_60``               ``system``
``1.7.0_75``               ``system``
``1.7.0_76``               ``system``
``1.7.0_79``               ``system``
``1.7.0_80``               ``system``
``1.8``                    ``system``
``1.8.0_20``               ``system``
``1.8.0_25``               ``system``
``1.8.0_31``               ``system``
``1.8.0_40``               ``system``
``1.8.0_45``               ``system``
``1.8.0_60``               ``system``
``1.8.0_65``               ``system``
``1.8.0_66``               ``system``
``1.8.0_72``               ``system``
``1.8.0_74``               ``system``
``1.8.0_77``               ``system``
``1.8.0_92``               ``system``
``1.8.0_112``              ``system``
``1.8.0_121``              ``system``
``1.8.0_131``              ``system``
``1.8.0_141``              ``system``
``1.8.0_144``              ``system``
``1.8.0_152``              ``system``
``1.8.0_162``              ``system``
``1.8.0_172``              ``system``
``1.8.0_181``              ``system``
``1.8.0_192``              ``system``
``1.8.0_202``              ``system``
``1.8.0_212``              ``system``
``1.8.0_221``              ``system``
``1.8_191-b26-OpenJDK``    ``system``
``1.9.0.4``                ``system``
``11``                     ``system``
``11.0.2``                 ``system``
=======================    ==========


.. _list_software_JAXFrontCE_884:

*JAXFrontCE*
++++++++++++

JAXFront is a technology to generate graphical user interfaces  on multiple channels (Java Swing, HTML, PDF) on the basis of an XML schema.

*homepage*: http://www.jaxfront.org/pages/free_community_edition.html

========    ==========
version     toolchain 
========    ==========
``2.75``    ``system``
========    ==========


.. _list_software_jbigkit_740:

*jbigkit*
+++++++++

JBIG-KIT is a software implementation of the JBIG1 data  compression standard (ITU-T T.82), which was designed for bi-level image  data, such as scanned documents.

*homepage*: 

=======    ====================================
version    toolchain                           
=======    ====================================
``2.1``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=======    ====================================


.. _list_software_Jellyfish_938:

*Jellyfish*
+++++++++++

Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA.

*homepage*: http://www.genome.umd.edu/jellyfish.html

==========    ===============================
version       toolchain                      
==========    ===============================
``1.1.11``    ``foss/2016a``, ``foss/2016b`` 
``1.1.12``    ``foss/2018b``, ``intel/2018a``
``2.2.6``     ``foss/2016b``, ``intel/2017a``
``2.2.10``    ``foss/2018b``, ``intel/2018a``
==========    ===============================


.. _list_software_jemalloc_839:

*jemalloc*
++++++++++

jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and  scalable concurrency support.

*homepage*: http://jemalloc.net

=========    ===============================
version      toolchain                      
=========    ===============================
``4.1.0``    ``intel/2016a``                
``4.2.0``    ``foss/2016a``, ``intel/2016a``
``4.2.1``    ``intel/2016b``                
``4.5.0``    ``intel/2017a``                
``5.0.1``    ``GCCcore/6.4.0``              
``5.1.0``    ``GCCcore/7.3.0``              
``5.2.0``    ``GCCcore/8.2.0``              
=========    ===============================


.. _list_software_jhbuild_738:

*jhbuild*
+++++++++

JHBuild allows you to automatically download and compile “modules” (i.e. source code packages).  Modules are listed in “module set” files, which also include dependency information so that JHBuild can discover  what modules need to be built and in what order.

*homepage*: https://wiki.gnome.org/action/show/Projects/Jhbuild

===========    =================
version        toolchain        
===========    =================
``3.15.92``    ``GCCcore/4.9.3``
===========    =================


.. _list_software_JiTCODE_546:

*JiTCODE*
+++++++++

Just-in-time compilation for ordinary/delay/stochastic differential equations (DDEs)

*homepage*: https://jitcde-common.readthedocs.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.3.2``    ``-Python-3.6.4``    ``intel/2018a``
``1.4.0``    ``-Python-3.7.2``    ``foss/2019a`` 
=========    =================    ===============


.. _list_software_jModelTest_1019:

*jModelTest*
++++++++++++

jModelTest is a tool to carry out statistical selection of best-fit models of nucleotide substitution.

*homepage*: https://github.com/ddarriba/jmodeltest2

===================    ==================    ==========
version                versionsuffix         toolchain 
===================    ==================    ==========
``2.1.10r20160303``    ``-Java-1.8.0_92``    ``system``
===================    ==================    ==========


.. _list_software_jq_219:

*jq*
++++

jq is a lightweight and flexible command-line JSON processor.

*homepage*: https://stedolan.github.io/jq/

=======    =================
version    toolchain        
=======    =================
``1.5``    ``GCCcore/6.4.0``
=======    =================


.. _list_software_JsonCpp_701:

*JsonCpp*
+++++++++

JsonCpp is a C++ library that allows manipulating JSON values,  including serialization and deserialization to and from strings. It can also preserve existing comment in  unserialization/serialization steps, making it a convenient format to store user input files. 

*homepage*: http://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html

==========    =================
version       toolchain        
==========    =================
``0.10.7``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_JUBE_294:

*JUBE*
++++++

The JUBE benchmarking environment provides a script based framework to easily create benchmark sets,  run those sets on different computer systems and evaluate the results.

*homepage*: http://www.fz-juelich.de/jsc/jube

=========    ==========
version      toolchain 
=========    ==========
``2.0.3``    ``system``
``2.0.4``    ``system``
``2.0.5``    ``system``
=========    ==========


.. _list_software_Judy_412:

*Judy*
++++++

A C library that implements a dynamic array.

*homepage*: http://judy.sourceforge.net/

=========    =================
version      toolchain        
=========    =================
``1.0.5``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_Julia_501:

*Julia*
+++++++

Julia is a high-level, high-performance dynamic programming language for numerical computing

*homepage*: https://julialang.org

=========    =================    ==========
version      versionsuffix        toolchain 
=========    =================    ==========
``1.1.1``    ``-linux-x86_64``    ``system``
``1.2.0``    ``-linux-x86_64``    ``system``
=========    =================    ==========


.. _list_software_JUnit_490:

*JUnit*
+++++++

A programmer-oriented testing framework for Java.

*homepage*: http://sourceforge.net/projects/junit

========    ===================    ==========
version     versionsuffix          toolchain 
========    ===================    ==========
``4.10``    ``-Java-1.7.0_10``     ``system``
``4.10``    ``-Java-1.7.0_21``     ``system``
``4.11``    ``-Java-1.7.0_15``     ``system``
``4.11``    ``-Java-1.7.0_21``     ``system``
``4.11``    ``-Java-1.7.0_60``     ``system``
``4.11``    ``-Java-1.7.0_75``     ``system``
``4.11``    ``-Java-1.7.0_79``     ``system``
``4.12``    ``-Java-1.7.0_80``     ``system``
``4.12``    ``-Java-1.8``          ``system``
``4.12``    ``-Java-1.8.0_112``    ``system``
``4.12``    ``-Java-1.8.0_121``    ``system``
``4.12``    ``-Java-1.8.0_144``    ``system``
``4.12``    ``-Java-1.8.0_152``    ``system``
``4.12``    ``-Java-1.8.0_162``    ``system``
``4.12``    ``-Java-1.8.0_66``     ``system``
``4.12``    ``-Java-1.8.0_72``     ``system``
``4.12``    ``-Java-1.8.0_77``     ``system``
``4.12``    ``-Java-1.8.0_92``     ``system``
========    ===================    ==========


.. _list_software_jupyterhub_1106:

*jupyterhub*
++++++++++++

JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed  for centralized deployments in companies, university classrooms and research labs.

*homepage*: http://jupyter.org

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.6.1``    ``-Python-3.5.1``    ``foss/2016a``
``0.8.1``    ``-Python-3.6.4``    ``foss/2017a``
=========    =================    ==============


.. _list_software_JWM_238:

*JWM*
+++++

JWM is a light-weight window manager for the X11 Window System.

*homepage*: https://joewing.net/projects/jwm/

=========    ===============
version      toolchain      
=========    ===============
``2.3.5``    ``intel/2016a``
=========    ===============


.. _list_software_letter_k:

*K*
---


:ref:`list_software_Kaiju_500` - :ref:`list_software_kallisto_867` - :ref:`list_software_KAT_224` - :ref:`list_software_kbproto_769` - :ref:`list_software_Kent_tools_1058` - :ref:`list_software_Keras_502` - :ref:`list_software_khmer_535` - :ref:`list_software_kim-api_680` - :ref:`list_software_KMC_219` - :ref:`list_software_KmerGenie_887` - :ref:`list_software_KNIME_372` - :ref:`list_software_kpcalg_626` - :ref:`list_software_Kraken_604` - :ref:`list_software_Kraken2_654` - :ref:`list_software_Kratos_628` - :ref:`list_software_KronaTools_1036` - :ref:`list_software_kwant_549` - :ref:`list_software_KWIML_388` - :ref:`list_software_kWIP_347` - :ref:`list_software_KyotoCabinet_1228`


.. _list_software_Kaiju_500:

*Kaiju*
+++++++

Kaiju is a program for sensitive taxonomic classification of high-throughput sequencing reads from metagenomic whole genome sequencing experiments

*homepage*: http://kaiju.binf.ku.dk/

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.5.0``                         ``intel/2016b``
``1.7.2``    ``-Python-3.7.2``    ``iimpi/2019a``
=========    =================    ===============


.. _list_software_kallisto_867:

*kallisto*
++++++++++

kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally  of target sequences using high-throughput sequencing reads.

*homepage*: http://pachterlab.github.io/kallisto/

==========    ================================================
version       toolchain                                       
==========    ================================================
``0.42.5``    ``foss/2016a``                                  
``0.43.0``    ``intel/2016b``                                 
``0.43.1``    ``foss/2016b``, ``intel/2017a``, ``intel/2017b``
``0.44.0``    ``foss/2016b``, ``intel/2018a``                 
``0.45.0``    ``foss/2018b``                                  
``0.45.1``    ``foss/2019a``                                  
``0.46.0``    ``intel/2019a``                                 
==========    ================================================


.. _list_software_KAT_224:

*KAT*
+++++

The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.

*homepage*: https://www.earlham.ac.uk/kat-tools

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``2.4.2``    ``-Python-3.6.4``    ``foss/2018a``
``2.4.2``    ``-Python-3.7.2``    ``foss/2019a``
=========    =================    ==============


.. _list_software_kbproto_769:

*kbproto*
+++++++++

X.org KBProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ==================================================================
version      toolchain                                                         
=========    ==================================================================
``1.0.7``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``
=========    ==================================================================


.. _list_software_Kent_tools_1058:

*Kent_tools*
++++++++++++

Kent tools: collection of tools used by the UCSC genome browser.

*homepage*: http://genome.cse.ucsc.edu/

============    =================    ==========
version         versionsuffix        toolchain 
============    =================    ==========
``20130806``    ``-linux.x86_64``    ``system``
``20171107``    ``-linux.x86_64``    ``system``
``20180716``    ``-linux.x86_64``    ``system``
``20190326``    ``-linux.x86_64``    ``system``
============    =================    ==========


.. _list_software_Keras_502:

*Keras*
+++++++

Keras is a minimalist, highly modular neural networks library, written in Python and capable of running on top of either TensorFlow or Theano.

*homepage*: https://keras.io/

=========    ==================    ==================================
version      versionsuffix         toolchain                         
=========    ==================    ==================================
``1.0.8``    ``-Python-3.5.2``     ``intel/2016b``                   
``1.1.0``    ``-Python-3.5.2``     ``intel/2016b``                   
``2.0.4``    ``-Python-2.7.13``    ``intel/2017a``                   
``2.0.4``    ``-Python-3.6.1``     ``intel/2017a``                   
``2.0.5``    ``-Python-3.6.1``     ``intel/2017a``                   
``2.0.8``    ``-Python-3.6.1``     ``intel/2017a``                   
``2.1.1``    ``-Python-2.7.14``    ``intel/2017b``                   
``2.1.1``    ``-Python-3.6.3``     ``intel/2017b``                   
``2.1.2``    ``-Python-2.7.14``    ``intel/2017b``                   
``2.1.2``    ``-Python-3.6.3``     ``foss/2017b``                    
``2.1.3``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``   
``2.2.0``    ``-Python-2.7.14``    ``fosscuda/2017b``                
``2.2.0``    ``-Python-3.6.3``     ``fosscuda/2017b``                
``2.2.0``    ``-Python-3.6.4``     ``foss/2018a``                    
``2.2.2``    ``-Python-2.7.15``    ``fosscuda/2018b``                
``2.2.4``    ``-Python-3.6.4``     ``intel/2018a``                   
``2.2.4``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``
``2.2.4``    ``-Python-3.7.2``     ``foss/2019a``, ``fosscuda/2019a``
=========    ==================    ==================================


.. _list_software_khmer_535:

*khmer*
+++++++

In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more 

*homepage*: https://github.com/ged-lab/khmer/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.4.1``    ``-Python-2.7.12``    ``foss/2016b`` 
``2.1.1``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_kim-api_680:

*kim-api*
+++++++++

Open Knowledgebase of Interatomic Models.  KIM is an API and OpenKIM is a collection of interatomic models (potentials) for atomistic simulations.  This is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild only installs the API, the models can be installed with the package openkim-models, or the user can install them manually by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  

*homepage*: https://openkim.org/

=========    ===============================
version      toolchain                      
=========    ===============================
``2.1.2``    ``foss/2019a``, ``intel/2019a``
=========    ===============================


.. _list_software_KMC_219:

*KMC*
+++++

KMC is a disk-based programm for counting k-mers from (possibly gzipped) FASTQ/FASTA files.

*homepage*: http://sun.aei.polsl.pl/kmc

=========    ==============================
version      toolchain                     
=========    ==============================
``3.1.0``    ``foss/2018a``, ``foss/2018b``
=========    ==============================


.. _list_software_KmerGenie_887:

*KmerGenie*
+++++++++++

KmerGenie estimates the best k-mer length for genome de novo assembly.

*homepage*: http://kmergenie.bx.psu.edu/

==========    ===============
version       toolchain      
==========    ===============
``1.7044``    ``intel/2017a``
``1.7048``    ``intel/2018a``
==========    ===============


.. _list_software_KNIME_372:

*KNIME*
+++++++

KNIME Analytics Platform is the open source software for creating data science applications and services. KNIME stands for KoNstanz Information MinEr.  

*homepage*: https://www.knime.com/

=========    ==========
version      toolchain 
=========    ==========
``3.6.2``    ``system``
=========    ==========


.. _list_software_kpcalg_626:

*kpcalg*
++++++++

Kernel PC (kPC) algorithm for causal structure learning and  causal inference using graphical models. kPC is a version of PC algorithm that  uses kernel based independence criteria in order to be able to deal with  non-linear relationships and non-Gaussian noise.  Includes pcalg: Functions for causal structure learning and causal inference  using graphical models.

*homepage*: https://cran.r-project.org/package=kpcalg

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``1.0.1``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_Kraken_604:

*Kraken*
++++++++

Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.

*homepage*: https://ccb.jhu.edu/software/kraken/

===============    ================    =================
version            versionsuffix       toolchain        
===============    ================    =================
``0.10.5-beta``    ``-Perl-5.22.1``    ``foss/2016a``   
``0.10.5-beta``    ``-Perl-5.24.0``    ``foss/2016b``   
``1.0``            ``-Perl-5.26.1``    ``intel/2018a``  
``1.1``            ``-Perl-5.28.0``    ``foss/2018b``   
``1.1.1``          ``-Perl-5.28.1``    ``GCCcore/8.2.0``
===============    ================    =================


.. _list_software_Kraken2_654:

*Kraken2*
+++++++++

Kraken is a system for assigning taxonomic labels to short DNA sequences,  usually obtained through metagenomic studies. Previous attempts by other  bioinformatics software to accomplish this task have often used sequence  alignment or machine learning techniques that were quite slow, leading to  the development of less sensitive but much faster abundance estimation  programs. Kraken aims to achieve high sensitivity and high speed by  utilizing exact alignments of k-mers and a novel classification algorithm.

*homepage*: http://www.ccb.jhu.edu/software/kraken2/

==============    ================    ==============
version           versionsuffix       toolchain     
==============    ================    ==============
``2.0.6-beta``    ``-Perl-5.26.1``    ``foss/2018a``
``2.0.7-beta``    ``-Perl-5.28.0``    ``foss/2018b``
==============    ================    ==============


.. _list_software_Kratos_628:

*Kratos*
++++++++

Kratos Multiphysics (A.K.A Kratos) is a framework for building parallel multi-disciplinary  simulation software.

*homepage*: http://www.cimne.com/kratos

=======    =================    ==============
version    versionsuffix        toolchain     
=======    =================    ==============
``6.0``    ``-Python-3.6.4``    ``foss/2018a``
=======    =================    ==============


.. _list_software_KronaTools_1036:

*KronaTools*
++++++++++++

Krona Tools is a set of scripts to create Krona charts from  several Bioinformatics tools as well as from text and XML files.

*homepage*: https://github.com/marbl/Krona/wiki/KronaTools

=========    =================
version      toolchain        
=========    =================
``2.7``      ``GCCcore/7.3.0``
``2.7.1``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_kwant_549:

*kwant*
+++++++

Kwant is a free (open source), powerful, and easy to use Python package for   numerical calculations on tight-binding models with a strong focus on quantum transport.

*homepage*: https://kwant-project.org/

=========    =================    ===============================
version      versionsuffix        toolchain                      
=========    =================    ===============================
``1.4.1``    ``-Python-3.7.2``    ``foss/2019a``, ``intel/2019a``
=========    =================    ===============================


.. _list_software_KWIML_388:

*KWIML*
+++++++

The Kitware Information Macro Library

*homepage*: https://gitlab.kitware.com/utils/kwiml

============    =================
version         toolchain        
============    =================
``20180201``    ``GCCcore/6.4.0``
============    =================


.. _list_software_kWIP_347:

*kWIP*
++++++

This software implements a de novo, alignment free measure of sample  genetic dissimilarity which operates upon raw sequencing reads.  It is able to calculate the genetic dissimilarity between samples without any reference genome,  and without assembling one.

*homepage*: https://github.com/kdmurray91/kWIP

=========    =================
version      toolchain        
=========    =================
``0.2.0``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_KyotoCabinet_1228:

*KyotoCabinet*
++++++++++++++

Kyoto Cabinet is a library of routines for managing a database.

*homepage*: https://fallabs.com/kyotocabinet

==========    =================
version       toolchain        
==========    =================
``1.2.77``    ``GCCcore/7.3.0``
==========    =================


.. _list_software_letter_l:

*L*
---


:ref:`list_software_L_RNA_scaffolder_1540` - :ref:`list_software_LAME_287` - :ref:`list_software_LAST_308` - :ref:`list_software_LASTZ_398` - :ref:`list_software_lavaan_627` - :ref:`list_software_LCov_372` - :ref:`list_software_LeadIT_531` - :ref:`list_software_Leptonica_927` - :ref:`list_software_less_439` - :ref:`list_software_LevelDB_638` - :ref:`list_software_lftp_438` - :ref:`list_software_libaio_624` - :ref:`list_software_libarchive_1049` - :ref:`list_software_libbitmask_1058` - :ref:`list_software_libcerf_727` - :ref:`list_software_libcircle_937` - :ref:`list_software_libcmaes_832` - :ref:`list_software_libconfig_941` - :ref:`list_software_libcpuset_971` - :ref:`list_software_libcroco_845` - :ref:`list_software_libctl_634` - :ref:`list_software_libdap_620` - :ref:`list_software_libdrm_634` - :ref:`list_software_libdwarf_843` - :ref:`list_software_libedit_733` - :ref:`list_software_libelf_622` - :ref:`list_software_libepoxy_876` - :ref:`list_software_libevent_857` - :ref:`list_software_libffcall_927` - :ref:`list_software_libffi_620` - :ref:`list_software_libFLAME_668` - :ref:`list_software_libfontenc_1060` - :ref:`list_software_libgcrypt_976` - :ref:`list_software_libgd_514` - :ref:`list_software_libgeotiff_1051` - :ref:`list_software_libglade_820` - :ref:`list_software_libGLU_543` - :ref:`list_software_libglvnd_850` - :ref:`list_software_libgpg-error_1228` - :ref:`list_software_libgpuarray_1186` - :ref:`list_software_libgtextutils_1428` - :ref:`list_software_libharu_743` - :ref:`list_software_libibmad_820` - :ref:`list_software_libibumad_937` - :ref:`list_software_libICE_520` - :ref:`list_software_libiconv_854` - :ref:`list_software_libidn_626` - :ref:`list_software_Libint_610` - :ref:`list_software_libjpeg-turbo_1334` - :ref:`list_software_libmatheval_1161` - :ref:`list_software_libmaus2_799` - :ref:`list_software_libMemcached_1198` - :ref:`list_software_libpciaccess_1253` - :ref:`list_software_libpng_636` - :ref:`list_software_libpsl_646` - :ref:`list_software_libpsml_755` - :ref:`list_software_libpsortb_977` - :ref:`list_software_libpthread-stubs_1661` - :ref:`list_software_libQGLViewer_1165` - :ref:`list_software_libreadline_1147` - :ref:`list_software_libRmath_819` - :ref:`list_software_librsvg_761` - :ref:`list_software_libsamplerate_1381` - :ref:`list_software_libsigc++_819` - :ref:`list_software_libsigsegv_1071` - :ref:`list_software_libSM_471` - :ref:`list_software_libsndfile_1052` - :ref:`list_software_libsodium_968` - :ref:`list_software_LibSoup_702` - :ref:`list_software_libspatialindex_1597` - :ref:`list_software_libspatialite_1383` - :ref:`list_software_libssh_645` - :ref:`list_software_LIBSVM_461` - :ref:`list_software_libtar_638` - :ref:`list_software_libtasn1_798` - :ref:`list_software_LibTIFF_576` - :ref:`list_software_libtirpc_857` - :ref:`list_software_libtool_757` - :ref:`list_software_libunistring_1306` - :ref:`list_software_libunwind_972` - :ref:`list_software_libutempter_1197` - :ref:`list_software_LibUUID_590` - :ref:`list_software_libvdwxc_867` - :ref:`list_software_libwebp_741` - :ref:`list_software_libX11_497` - :ref:`list_software_libXau_613` - :ref:`list_software_libxc_530` - :ref:`list_software_libxcb_628` - :ref:`list_software_libXcursor_1069` - :ref:`list_software_libXdamage_1006` - :ref:`list_software_libXdmcp_819` - :ref:`list_software_libXext_736` - :ref:`list_software_libXfixes_942` - :ref:`list_software_libXfont_838` - :ref:`list_software_libXft_617` - :ref:`list_software_libXi_504` - :ref:`list_software_libXinerama_1132` - :ref:`list_software_libxkbcommon_1285` - :ref:`list_software_libxml++_734` - :ref:`list_software_libxml2_698` - :ref:`list_software_libxml2-python_1417` - :ref:`list_software_libXmu_625` - :ref:`list_software_libXp_511` - :ref:`list_software_libXpm_620` - :ref:`list_software_libXrandr_934` - :ref:`list_software_libXrender_1039` - :ref:`list_software_libxslt_770` - :ref:`list_software_libxsmm_764` - :ref:`list_software_libXt_515` - :ref:`list_software_libXxf86vm_958` - :ref:`list_software_libyaml_746` - :ref:`list_software_libzip_650` - :ref:`list_software_lifelines_955` - :ref:`list_software_Lighter_719` - :ref:`list_software_likwid_644` - :ref:`list_software_LinBox_588` - :ref:`list_software_LittleCMS_849` - :ref:`list_software_LLVM_315` - :ref:`list_software_LMDB_287` - :ref:`list_software_LMfit_476` - :ref:`list_software_Lmod_396` - :ref:`list_software_LocARNA_576` - :ref:`list_software_LoFreq_585` - :ref:`list_software_Log-Log4perl_1112` - :ref:`list_software_LOHHLA_440` - :ref:`list_software_Loki_399` - :ref:`list_software_Longshot_846` - :ref:`list_software_lpsolve_773` - :ref:`list_software_lrslib_648` - :ref:`list_software_LS-PrePost_921` - :ref:`list_software_LSMS_319` - :ref:`list_software_Lua_290` - :ref:`list_software_LuaJIT_521` - :ref:`list_software_Lucene-Geo-Gazetteer_1916` - :ref:`list_software_LUMPY_407` - :ref:`list_software_LUSCUS_479` - :ref:`list_software_lwgrp_556` - :ref:`list_software_lxml_445` - :ref:`list_software_lynx_459` - :ref:`list_software_lz4_282` - :ref:`list_software_LZO_245`


.. _list_software_L_RNA_scaffolder_1540:

*L_RNA_scaffolder*
++++++++++++++++++

L_RNA_scaffolder is a novel scaffolding tool using long trancriptome reads to scaffold genome fragments

*homepage*: http://www.fishbrowser.org/software/L_RNA_scaffolder/

============    ================    ===============
version         versionsuffix       toolchain      
============    ================    ===============
``20141124``    ``-Perl-5.24.0``    ``intel/2016b``
============    ================    ===============


.. _list_software_LAME_287:

*LAME*
++++++

LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.

*homepage*: http://lame.sourceforge.net/

==========    ===========================================================================================
version       toolchain                                                                                  
==========    ===========================================================================================
``3.99.5``    ``foss/2016b``, ``intel/2017a``                                                            
``3.100``     ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017b``
==========    ===========================================================================================


.. _list_software_LAST_308:

*LAST*
++++++

LAST finds similar regions between sequences.

*homepage*: http://last.cbrc.jp/

=======    ===============
version    toolchain      
=======    ===============
``869``    ``intel/2017a``
``914``    ``intel/2017b``
=======    ===============


.. _list_software_LASTZ_398:

*LASTZ*
+++++++

LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle  sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS   sequencing technologies such as Roche 454. 

*homepage*: http://www.bx.psu.edu/~rsharris/lastz/

===========    ==============
version        toolchain     
===========    ==============
``1.02.00``    ``foss/2016a``
===========    ==============


.. _list_software_lavaan_627:

*lavaan*
++++++++

lavaan is a free, open source R package for latent variable analysis

*homepage*: http://lavaan.org

==============    =============    ==============
version           versionsuffix    toolchain     
==============    =============    ==============
``0.6-4.1433``    ``-R-3.6.0``     ``foss/2019a``
==============    =============    ==============


.. _list_software_LCov_372:

*LCov*
++++++

LCOV - the LTP GCOV extension

*homepage*: http://ltp.sourceforge.net/coverage/lcov.php

========    =================
version     toolchain        
========    =================
``1.13``    ``GCCcore/7.2.0``
========    =================


.. _list_software_LeadIT_531:

*LeadIT*
++++++++

Visually Informed LeadOpt

*homepage*: http://www.biosolveit.de/LeadIT/index.html

=========    ==========
version      toolchain 
=========    ==========
``2.1.9``    ``system``
=========    ==========


.. _list_software_Leptonica_927:

*Leptonica*
+++++++++++

Leptonica is a collection of pedagogically-oriented open source software  that is broadly useful for image processing and image analysis applications.

*homepage*: http://www.leptonica.org

==========    =================
version       toolchain        
==========    =================
``1.77.0``    ``GCCcore/7.3.0``
``1.78.0``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_less_439:

*less*
++++++

Less is a free, open-source file pager. It can be found on most versions of Linux,  Unix and Mac OS, as well as on many other operating systems.

*homepage*: http://www.greenwoodsoftware.com/less/

=======    =============
version    toolchain    
=======    =============
``458``    ``GCC/4.8.2``
=======    =============


.. _list_software_LevelDB_638:

*LevelDB*
+++++++++

LevelDB is a fast key-value storage library written at Google that provides an  ordered mapping from string keys to string values.

*homepage*: https://github.com/google/leveldb

========    ================================================
version     toolchain                                       
========    ================================================
``1.18``    ``foss/2016a``, ``intel/2017a``, ``intel/2017b``
``1.20``    ``GCCcore/7.3.0``                               
``1.22``    ``GCCcore/8.2.0``                               
========    ================================================


.. _list_software_lftp_438:

*lftp*
++++++

LFTP is a sophisticated ftp/http client, and a file transfer program supporting   a number of network protocols. Like BASH, it has job control and uses the readline library for   input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel.   It was designed with reliability in mind.

*homepage*: http://lftp.yar.ru

=========    ==================
version      toolchain         
=========    ==================
``4.6.4``    ``GNU/4.9.3-2.25``
``4.8.4``    ``GCCcore/6.4.0`` 
=========    ==================


.. _list_software_libaio_624:

*libaio*
++++++++

Asynchronous input/output library that uses the kernels native interface.

*homepage*: https://pagure.io/libaio

===========    =================
version        toolchain        
===========    =================
``0.3.111``    ``GCCcore/8.2.0``
===========    =================


.. _list_software_libarchive_1049:

*libarchive*
++++++++++++

Multi-format archive and compression library 

*homepage*: https://www.libarchive.org/

=========    =================
version      toolchain        
=========    =================
``3.4.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_libbitmask_1058:

*libbitmask*
++++++++++++

libbitmask provides a convenient, powerful bitmask data type

*homepage*: http://oss.sgi.com/projects/cpusets/

=======    ==========
version    toolchain 
=======    ==========
``2.0``    ``system``
=======    ==========


.. _list_software_libcerf_727:

*libcerf*
+++++++++

libcerf is a self-contained numeric library that provides an efficient and  accurate implementation of complex error functions, along with Dawson,  Faddeeva, and Voigt functions. 

*homepage*: http://apps.jcns.fz-juelich.de/doku/sc/libcerf

========    =========================================================================================================
version     toolchain                                                                                                
========    =========================================================================================================
``1.4``     ``foss/2016a``, ``foss/2016b``, ``intel/2016a``                                                          
``1.5``     ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``
``1.7``     ``GCCcore/7.3.0``                                                                                        
``1.11``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``                                                                     
========    =========================================================================================================


.. _list_software_libcircle_937:

*libcircle*
+++++++++++

An API to provide an efficient distributed queue on a cluster. libcircle is an  API for distributing embarrassingly parallel workloads using self-stabilization. 

*homepage*: https://github.com/hpc/libcircle/

==============    ================================
version           toolchain                       
==============    ================================
``0.2.1-rc.1``    ``gompi/2019a``, ``iimpi/2019a``
==============    ================================


.. _list_software_libcmaes_832:

*libcmaes*
++++++++++

libcmaes is a multithreaded C++11 library for high performance blackbox stochastic optimization using the CMA-ES algorithm for Covariance Matrix Adaptation Evolution Strategy.

*homepage*: http://beniz.github.io/libcmaes/

=========    ==============
version      toolchain     
=========    ==============
``0.9.5``    ``foss/2016a``
=========    ==============


.. _list_software_libconfig_941:

*libconfig*
+++++++++++

Libconfig is a simple library for processing structured configuration files

*homepage*: https://hyperrealm.github.io/libconfig

=========    =================
version      toolchain        
=========    =================
``1.5``      ``intel/2016b``  
``1.7.1``    ``GCCcore/6.4.0``
``1.7.2``    ``GCCcore/7.3.0``
=========    =================


.. _list_software_libcpuset_971:

*libcpuset*
+++++++++++

libcpuset provides full access to cpuset capabilities

*homepage*: http://oss.sgi.com/projects/cpusets/

=======    ==========
version    toolchain 
=======    ==========
``1.0``    ``system``
=======    ==========


.. _list_software_libcroco_845:

*libcroco*
++++++++++

Libcroco is a standalone css2 parsing and manipulation library.

*homepage*: https://github.com/GNOME/libcroco

==========    ===============
version       toolchain      
==========    ===============
``0.6.11``    ``intel/2016a``
==========    ===============


.. _list_software_libctl_634:

*libctl*
++++++++

libctl is a free Guile-based library implementing flexible control files for scientific simulations.

*homepage*: http://ab-initio.mit.edu/libctl

=========    ==============
version      toolchain     
=========    ==============
``3.2.2``    ``foss/2016a``
=========    ==============


.. _list_software_libdap_620:

*libdap*
++++++++

A C++ SDK which contains an implementation of DAP 2.0 and  DAP4.0. This includes both Client- and Server-side support classes.

*homepage*: https://www.opendap.org/software/libdap

==========    ==================    ==================================================
version       versionsuffix         toolchain                                         
==========    ==================    ==================================================
``3.18.1``                          ``intel/2017a``                                   
``3.18.1``    ``-Python-2.7.11``    ``foss/2016a``                                    
``3.19.1``                          ``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``
``3.20.3``                          ``GCCcore/7.3.0``                                 
``3.20.4``                          ``GCCcore/8.2.0``                                 
==========    ==================    ==================================================


.. _list_software_libdrm_634:

*libdrm*
++++++++

Direct Rendering Manager runtime library.

*homepage*: http://dri.freedesktop.org

==========    ==================================================
version       toolchain                                         
==========    ==================================================
``2.4.67``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a`` 
``2.4.68``    ``foss/2016a``, ``intel/2016a``                   
``2.4.70``    ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``2.4.76``    ``GCCcore/6.3.0``, ``intel/2017a``                
``2.4.88``    ``GCCcore/6.4.0``                                 
``2.4.91``    ``GCCcore/6.4.0``                                 
``2.4.92``    ``GCCcore/7.3.0``                                 
``2.4.97``    ``GCCcore/8.2.0``                                 
``2.4.99``    ``GCCcore/8.3.0``                                 
==========    ==================================================


.. _list_software_libdwarf_843:

*libdwarf*
++++++++++

The DWARF Debugging Information Format is of interest to programmers working on compilers and debuggers (and anyone interested in reading or writing DWARF information))

*homepage*: http://www.prevanders.net/dwarf.html

============    ===================================================
version         toolchain                                          
============    ===================================================
``20140805``    ``GCC/4.9.2``                                      
``20150310``    ``GCC/4.9.2``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``
``20190529``    ``GCCcore/8.2.0``                                  
============    ===================================================


.. _list_software_libedit_733:

*libedit*
+++++++++

This BSD-style licensed command line editor library provides generic line editing, history, and tokenization functions, similar to those found in GNU Readline. 

*homepage*: http://thrysoee.dk/editline/

============    ==================
version         toolchain         
============    ==================
``20150325``    ``GNU/4.9.3-2.25``
``20180525``    ``GCCcore/6.4.0`` 
============    ==================


.. _list_software_libelf_622:

*libelf*
++++++++

libelf is a free ELF object file access library

*homepage*: http://www.mr511.de/software/english.html

==========    =====================================================================================
version       toolchain                                                                            
==========    =====================================================================================
``0.8.13``    ``GCC/4.8.3``, ``GCC/4.9.2``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/8.2.0``
==========    =====================================================================================


.. _list_software_libepoxy_876:

*libepoxy*
++++++++++

Epoxy is a library for handling OpenGL function pointer management for you

*homepage*: https://github.com/anholt/libepoxy

=========    =====================================
version      toolchain                            
=========    =====================================
``1.5.3``    ``GCCcore/8.2.0``, ``fosscuda/2018b``
=========    =====================================


.. _list_software_libevent_857:

*libevent*
++++++++++

The libevent API provides a mechanism to execute a callback function when a specific  event occurs on a file descriptor or after a timeout has been reached.  Furthermore, libevent also support callbacks due to signals or regular timeouts.

*homepage*: http://libevent.org/

==========    ========================================================================
version       toolchain                                                               
==========    ========================================================================
``2.0.22``    ``GCC/4.9.2``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GNU/4.9.3-2.25``
``2.1.8``     ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``system``     
``2.1.11``    ``GCCcore/8.3.0``                                                       
==========    ========================================================================


.. _list_software_libffcall_927:

*libffcall*
+++++++++++

GNU Libffcall is a collection of four libraries which can be used to build  foreign function call interfaces in embedded interpreters 

*homepage*: https://www.gnu.org/software/libffcall/

========    =================
version     toolchain        
========    =================
``1.13``    ``GCCcore/6.4.0``
========    =================


.. _list_software_libffi_620:

*libffi*
++++++++

The libffi library provides a portable, high level programming interface to  various calling conventions. This allows a programmer to call any function  specified by a call interface description at run-time. 

*homepage*: http://sourceware.org/libffi/

=========    =============================================================================================================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                                                                                                                    
=========    =============================================================================================================================================================================================================================================================================================================
``3.2.1``    ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``system``
=========    =============================================================================================================================================================================================================================================================================================================


.. _list_software_libFLAME_668:

*libFLAME*
++++++++++

AMD fork of libFLAME. libFLAME is a portable library for dense matrix computations,  providing much of the functionality present in LAPACK.

*homepage*: https://developer.amd.com/amd-cpu-libraries/blas-library/#libflame

=======    =============    ==================
version    versionsuffix    toolchain         
=======    =============    ==================
``1.0``    ``-amd``         ``GCC/7.3.0-2.30``
=======    =============    ==================


.. _list_software_libfontenc_1060:

*libfontenc*
++++++++++++

X11 font encoding library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs/

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.1.3``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libgcrypt_976:

*libgcrypt*
+++++++++++

Libgpg-error is a small library that defines common error values for all GnuPG components.

*homepage*: https://gnupg.org/related_software/libgcrypt/index.html

=========    ====================================
version      toolchain                           
=========    ====================================
``1.6.5``    ``intel/2016a``                     
``1.8.4``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=========    ====================================


.. _list_software_libgd_514:

*libgd*
+++++++

GD is an open source code library for the dynamic creation of images by programmers.

*homepage*: https://libgd.github.io/

=========    =========================================================================================
version      toolchain                                                                                
=========    =========================================================================================
``2.1.1``    ``foss/2016a``, ``intel/2016a``                                                          
``2.2.3``    ``foss/2016b``, ``intel/2016b``                                                          
``2.2.4``    ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2017a``                                       
``2.2.5``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``intel/2017b``, ``intel/2018a``
=========    =========================================================================================


.. _list_software_libgeotiff_1051:

*libgeotiff*
++++++++++++

Library for reading and writing coordinate system information from/to GeoTIFF files

*homepage*: https://directory.fsf.org/wiki/Libgeotiff

=========    ===============================================
version      toolchain                                      
=========    ===============================================
``1.4.2``    ``foss/2018a``, ``foss/2018b``, ``intel/2018b``
``1.5.1``    ``GCCcore/8.2.0``                              
=========    ===============================================


.. _list_software_libglade_820:

*libglade*
++++++++++

Libglade is a library for constructing user interfaces dynamically from XML descriptions.

*homepage*: https://developer.gnome.org/libglade/

=========    =================================================================
version      toolchain                                                        
=========    =================================================================
``2.6.4``    ``foss/2018b``, ``intel/2016a``, ``intel/2017b``, ``intel/2018a``
=========    =================================================================


.. _list_software_libGLU_543:

*libGLU*
++++++++

The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. 

*homepage*: ftp://ftp.freedesktop.org/pub/mesa/glu/

=========    ================    ===========================================================================================================================================================================================================================================================================================================================================
version      versionsuffix       toolchain                                                                                                                                                                                                                                                                                                                                  
=========    ================    ===========================================================================================================================================================================================================================================================================================================================================
``9.0.0``                        ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``fosscuda/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``, ``iomkl/2018a``
``9.0.0``    ``-Mesa-11.2.1``    ``foss/2016a``, ``intel/2016a``                                                                                                                                                                                                                                                                                                            
``9.0.1``                        ``GCCcore/8.3.0``                                                                                                                                                                                                                                                                                                                          
=========    ================    ===========================================================================================================================================================================================================================================================================================================================================


.. _list_software_libglvnd_850:

*libglvnd*
++++++++++

libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors.

*homepage*: https://github.com/NVIDIA/libglvnd

=========    ====================================
version      toolchain                           
=========    ====================================
``1.2.0``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
=========    ====================================


.. _list_software_libgpg-error_1228:

*libgpg-error*
++++++++++++++

Libgpg-error is a small library that defines common error values for all GnuPG components.

*homepage*: https://gnupg.org/related_software/libgpg-error/index.html

========    =================
version     toolchain        
========    =================
``1.21``    ``intel/2016a``  
``1.35``    ``GCCcore/7.3.0``
``1.36``    ``GCCcore/8.2.0``
========    =================


.. _list_software_libgpuarray_1186:

*libgpuarray*
+++++++++++++

Library to manipulate tensors on the GPU.

*homepage*: http://deeplearning.net/software/libgpuarray/

=========    ==================    ========================================================
version      versionsuffix         toolchain                                               
=========    ==================    ========================================================
``0.7.5``    ``-Python-2.7.14``    ``fosscuda/2017b``, ``intelcuda/2017b``                 
``0.7.5``    ``-Python-3.6.3``     ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.7.6``                          ``fosscuda/2019a``                                      
``0.7.6``    ``-Python-2.7.15``    ``fosscuda/2018b``                                      
``0.7.6``    ``-Python-3.6.6``     ``fosscuda/2018b``                                      
=========    ==================    ========================================================


.. _list_software_libgtextutils_1428:

*libgtextutils*
+++++++++++++++

ligtextutils is a dependency of fastx-toolkit and is provided via the same upstream

*homepage*: http://hannonlab.cshl.edu/fastx_toolkit/

=======    ==================================================================
version    toolchain                                                         
=======    ==================================================================
``0.7``    ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2018a``
=======    ==================================================================


.. _list_software_libharu_743:

*libharu*
+++++++++

libHaru is a free, cross platform, open source library for generating PDF files.

*homepage*: http://libharu.org/

=========    ======================================================================================================
version      toolchain                                                                                             
=========    ======================================================================================================
``2.3.0``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2017a``, ``intel/2018b``
=========    ======================================================================================================


.. _list_software_libibmad_820:

*libibmad*
++++++++++

libibmad is a convenience library to encode, decode, and dump IB MAD packets. It  is implemented on top of and in conjunction with libibumad (the umad kernel  interface library.)

*homepage*: http://www.openfabrics.org

==========    ==================
version       toolchain         
==========    ==================
``1.3.12``    ``GCC/4.9.3-2.25``
==========    ==================


.. _list_software_libibumad_937:

*libibumad*
+++++++++++

libibumad is the umad kernel interface library.

*homepage*: http://www.openfabrics.org

============    ==================
version         toolchain         
============    ==================
``1.3.10.2``    ``GCC/4.9.3-2.25``
============    ==================


.. _list_software_libICE_520:

*libICE*
++++++++

X Inter-Client Exchange library for freedesktop.org

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.0.9``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libiconv_854:

*libiconv*
++++++++++

Libiconv converts from one character encoding to another through Unicode conversion

*homepage*: https://www.gnu.org/software/libiconv

========    =======================================================
version     toolchain                                              
========    =======================================================
``1.15``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
========    =======================================================


.. _list_software_libidn_626:

*libidn*
++++++++

GNU Libidn is a fully documented implementation of the Stringprep, Punycode and IDNA specifications. Libidn's purpose is to encode and decode internationalized domain names.

*homepage*: http://www.gnu.org/software/libidn

========    ======================================================================================
version     toolchain                                                                             
========    ======================================================================================
``1.32``    ``GCCcore/5.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``
``1.34``    ``GCCcore/6.4.0``                                                                     
``1.35``    ``GCCcore/7.3.0``                                                                     
========    ======================================================================================


.. _list_software_Libint_610:

*Libint*
++++++++

Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body  matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory.

*homepage*: https://github.com/evaleev/libint

=========    ==================================================================================================================================================================
version      toolchain                                                                                                                                                         
=========    ==================================================================================================================================================================
``1.1.4``    ``intel/2016a``                                                                                                                                                   
``1.1.6``    ``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2018a``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``2.0.3``    ``gompi/2019a``                                                                                                                                                   
``2.1.0``    ``intel/2016b``                                                                                                                                                   
``2.4.2``    ``intel/2018a``                                                                                                                                                   
``2.5.0``    ``gompi/2019a``, ``iimpi/2019a``                                                                                                                                  
=========    ==================================================================================================================================================================


.. _list_software_libjpeg-turbo_1334:

*libjpeg-turbo*
+++++++++++++++

libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to  accelerate baseline JPEG compression and decompression. libjpeg is a library  that implements JPEG image encoding, decoding and transcoding. 

*homepage*: https://sourceforge.net/projects/libjpeg-turbo/

=========    =================    ==================================================================
version      versionsuffix        toolchain                                                         
=========    =================    ==================================================================
``1.4.2``                         ``foss/2016a``, ``foss/2016b``, ``intel/2016a``                   
``1.4.2``    ``-NASM-2.12.01``    ``foss/2016a``, ``intel/2016a``                                   
``1.5.0``                         ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``1.5.1``                         ``foss/2016b``, ``intel/2016b``, ``intel/2017a``                  
``1.5.2``                         ``GCCcore/6.3.0``, ``GCCcore/6.4.0``                              
``1.5.3``                         ``GCCcore/6.4.0``                                                 
``2.0.0``                         ``GCCcore/7.3.0``                                                 
``2.0.2``                         ``GCCcore/7.3.0``, ``GCCcore/8.2.0``                              
``2.0.3``                         ``GCCcore/8.3.0``                                                 
=========    =================    ==================================================================


.. _list_software_libmatheval_1161:

*libmatheval*
+++++++++++++

GNU libmatheval is a library (callable from C and Fortran) to parse  and evaluate symbolic expressions input as text.

*homepage*: http://www.gnu.org/software/libmatheval/

==========    =========================================================================================================================
version       toolchain                                                                                                                
==========    =========================================================================================================================
``1.1.11``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``
==========    =========================================================================================================================


.. _list_software_libmaus2_799:

*libmaus2*
++++++++++

libmaus2 is a collection of data structures and algorithms.

*homepage*: https://github.com/gt1/libmaus2

===========    ===============
version        toolchain      
===========    ===============
``2.0.453``    ``intel/2018a``
===========    ===============


.. _list_software_libMemcached_1198:

*libMemcached*
++++++++++++++

libMemcached is an open source C/C++ client library and tools for  the memcached server (http://danga.com/memcached). It has been designed to be light  on memory usage, thread safe, and provide full access to server side methods.

*homepage*: https://memcached.org

==========    =================
version       toolchain        
==========    =================
``1.0.18``    ``GCCcore/6.4.0``
==========    =================


.. _list_software_libpciaccess_1253:

*libpciaccess*
++++++++++++++

Generic PCI access library.

*homepage*: http://cgit.freedesktop.org/xorg/lib/libpciaccess/

==========    =============================================================================================
version       toolchain                                                                                    
==========    =============================================================================================
``0.13.4``    ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``           
``0.14``      ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
==========    =============================================================================================


.. _list_software_libpng_636:

*libpng*
++++++++

libpng is the official PNG reference library

*homepage*: http://www.libpng.org/pub/png/libpng.html

==========    =====================================================================
version       toolchain                                                            
==========    =====================================================================
``1.2.58``    ``system``                                                           
``1.6.21``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                    
``1.6.23``    ``foss/2016a``, ``foss/2016b``, ``intel/2016b``                      
``1.6.24``    ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``1.6.26``    ``foss/2016b``, ``intel/2016b``                                      
``1.6.27``    ``intel/2016b``                                                      
``1.6.28``    ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``gimkl/2017a``                
``1.6.29``    ``GCCcore/6.3.0``                                                    
``1.6.32``    ``GCCcore/6.4.0``                                                    
``1.6.34``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                                 
``1.6.36``    ``GCCcore/8.2.0``                                                    
``1.6.37``    ``GCCcore/8.3.0``                                                    
==========    =====================================================================


.. _list_software_libpsl_646:

*libpsl*
++++++++

C library for the Public Suffix List

*homepage*: https://rockdaboot.github.io/libpsl

==========    =================
version       toolchain        
==========    =================
``0.20.2``    ``GCCcore/7.3.0``
``0.21.0``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_libpsml_755:

*libpsml*
+++++++++

LibPSML provides a Fortran API to parse files in the  PSeudopotential Markup Language (PSML) format.

*homepage*: https://launchpad.net/libpsml

=========    ==============================
version      toolchain                     
=========    ==============================
``1.1.7``    ``foss/2016b``, ``foss/2017a``
=========    ==============================


.. _list_software_libpsortb_977:

*libpsortb*
+++++++++++

PSORT family of programs for subcellular localization prediction as well as other datasets and  resources relevant to localization prediction.

*homepage*: http://psort.org/

=======    ==============
version    toolchain     
=======    ==============
``1.0``    ``foss/2016a``
=======    ==============


.. _list_software_libpthread-stubs_1661:

*libpthread-stubs*
++++++++++++++++++

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring  a small footprint, latency hiding, direct access to the protocol, improved  threading support, and extensibility. 

*homepage*: http://xcb.freedesktop.org/

=======    =====================================================================================================
version    toolchain                                                                                            
=======    =====================================================================================================
``0.3``    ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``0.4``    ``GCCcore/6.4.0``                                                                                    
=======    =====================================================================================================


.. _list_software_libQGLViewer_1165:

*libQGLViewer*
++++++++++++++

libQGLViewer is a C++ library based on Qt that eases the creation of OpenGL 3D viewers.

*homepage*: http://libqglviewer.com/

=========    ================    ===============================================
version      versionsuffix       toolchain                                      
=========    ================    ===============================================
``2.6.3``                        ``foss/2016a``, ``foss/2016b``, ``intel/2016b``
``2.6.3``    ``-Mesa-11.2.1``    ``foss/2016a``, ``intel/2016a``                
``2.6.4``                        ``intel/2016b``                                
``2.7.1``                        ``intel/2018a``                                
=========    ================    ===============================================


.. _list_software_libreadline_1147:

*libreadline*
+++++++++++++

The GNU Readline library provides a set of functions for use by applications  that allow users to edit command lines as they are typed in. Both Emacs and  vi editing modes are available. The Readline library includes additional  functions to maintain a list of previously-entered command lines, to recall  and perhaps reedit those lines, and perform csh-like history expansion on  previous commands. 

*homepage*: http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html

=======    ===================================================================================================================================================================================================================================================================================================================================================================================
version    toolchain                                                                                                                                                                                                                                                                                                                                                                          
=======    ===================================================================================================================================================================================================================================================================================================================================================================================
``6.3``    ``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``7.0``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``                                                                                                                                                                                                                                                                                                         
``8.0``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``system``                                                                                                                                                                                                                                                                                                                                   
=======    ===================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_libRmath_819:

*libRmath*
++++++++++

The routines supporting the distribution and special functions in R and a few others are declared  in C header file Rmath.h. These can be compiled into a standalone library for linking to other applications.

*homepage*: https://cran.r-project.org/doc/manuals/r-release/R-admin.html#The-standalone-Rmath-library

=========    ==============
version      toolchain     
=========    ==============
``3.6.0``    ``foss/2018b``
=========    ==============


.. _list_software_librsvg_761:

*librsvg*
+++++++++

librsvg is a library to render SVG files using cairo.

*homepage*: https://wiki.gnome.org/action/show/Projects/LibRsvg

===========    ===============
version        toolchain      
===========    ===============
``2.40.15``    ``intel/2016a``
===========    ===============


.. _list_software_libsamplerate_1381:

*libsamplerate*
+++++++++++++++

Secret Rabbit Code (aka libsamplerate) is a Sample Rate Converter for audio.

*homepage*: http://www.mega-nerd.com/libsamplerate

=========    =================
version      toolchain        
=========    =================
``0.1.9``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_libsigc++_819:

*libsigc++*
+++++++++++

The libsigc++ package implements a typesafe callback system for standard C++.

*homepage*: https://libsigcplusplus.github.io/libsigcplusplus/

==========    =================
version       toolchain        
==========    =================
``2.10.0``    ``GCCcore/6.4.0``
``2.10.1``    ``GCCcore/7.3.0``
``2.10.2``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_libsigsegv_1071:

*libsigsegv*
++++++++++++

GNU libsigsegv is a library for handling page faults in user mode.

*homepage*: https://www.gnu.org/software/libsigsegv/

========    =================
version     toolchain        
========    =================
``2.11``    ``GCCcore/6.4.0``
========    =================


.. _list_software_libSM_471:

*libSM*
+++++++

X11 Session Management library, which allows for applications to both manage sessions,  and make use of session managers to save and restore their state for later use.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.2.2``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libsndfile_1052:

*libsndfile*
++++++++++++

Libsndfile is a C library for reading and writing files containing sampled sound  (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.

*homepage*: http://www.mega-nerd.com/libsndfile

==========    ========================================================================
version       toolchain                                                               
==========    ========================================================================
``1.0.28``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``intel/2017a``
==========    ========================================================================


.. _list_software_libsodium_968:

*libsodium*
+++++++++++

Sodium is a modern, easy-to-use software library for encryption, decryption, signatures,  password hashing and more.

*homepage*: http://doc.libsodium.org/

==========    ====================================
version       toolchain                           
==========    ====================================
``1.0.6``     ``intel/2016a``                     
``1.0.8``     ``foss/2016a``                      
``1.0.11``    ``foss/2016b``, ``intel/2016b``     
``1.0.12``    ``GCCcore/6.4.0``, ``intel/2017a``  
``1.0.13``    ``GCCcore/6.4.0``, ``foss/2017a``   
``1.0.16``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``1.0.17``    ``GCCcore/8.2.0``                   
``1.0.18``    ``GCCcore/8.3.0``                   
==========    ====================================


.. _list_software_LibSoup_702:

*LibSoup*
+++++++++

libsoup is an HTTP client/server library for GNOME. It uses GObjects and the glib main loop, to integrate well with GNOME applications, and also has a synchronous API, for use in threaded applications.

*homepage*: https://wiki.gnome.org/Projects/libsoup

==========    =================
version       toolchain        
==========    =================
``2.66.1``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_libspatialindex_1597:

*libspatialindex*
+++++++++++++++++

C++ implementation of R*-tree, an MVR-tree and a TPR-tree with C API

*homepage*: http://libspatialindex.github.io

=========    ======================================================================================
version      toolchain                                                                             
=========    ======================================================================================
``1.8.5``    ``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2018a``
=========    ======================================================================================


.. _list_software_libspatialite_1383:

*libspatialite*
+++++++++++++++

SpatiaLite is an open source library intended to extend the SQLite core to support  fully fledged Spatial SQL capabilities.

*homepage*: https://www.gaia-gis.it/fossil/libspatialite/home

==========    =================    ===============================================
version       versionsuffix        toolchain                                      
==========    =================    ===============================================
``4.3.0a``                         ``foss/2016b``, ``foss/2018b``, ``intel/2016b``
``4.3.0a``    ``-Python-3.7.2``    ``foss/2019a``                                 
==========    =================    ===============================================


.. _list_software_libssh_645:

*libssh*
++++++++

Multiplatform C library implementing the SSHv2 protocol on client and server side

*homepage*: https://www.libssh.org

=========    =================
version      toolchain        
=========    =================
``0.9.0``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_LIBSVM_461:

*LIBSVM*
++++++++

LIBSVM is an integrated software for support vector classification, (C-SVC, nu-SVC), regression  (epsilon-SVR, nu-SVR) and distribution estimation (one-class SVM). It supports multi-class classification.

*homepage*: http://www.csie.ntu.edu.tw/~cjlin/libsvm/

========    ================================
version     toolchain                       
========    ================================
``3.22``    ``intel/2016b``, ``intel/2017b``
``3.23``    ``intel/2018b``                 
========    ================================


.. _list_software_libtar_638:

*libtar*
++++++++

C library for manipulating POSIX tar files

*homepage*: https://repo.or.cz/libtar.git

==========    ====================================
version       toolchain                           
==========    ====================================
``1.2.20``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
==========    ====================================


.. _list_software_libtasn1_798:

*libtasn1*
++++++++++

Libtasn1 is the ASN.1 library used by GnuTLS, GNU Shishi and some other packages.  It was written by Fabio Fiorina, and has been shipped as part of GnuTLS  for some time but is now a proper GNU package.

*homepage*: https://www.gnu.org/software/libtasn1/

========    ===================================================
version     toolchain                                          
========    ===================================================
``4.7``     ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``4.12``    ``GCCcore/5.4.0``                                  
``4.13``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``               
========    ===================================================


.. _list_software_LibTIFF_576:

*LibTIFF*
+++++++++

tiff: Library and tools for reading and writing TIFF data files

*homepage*: https://www.remotesensing.org/libtiff/

==========    =========================================================================================================
version       toolchain                                                                                                
==========    =========================================================================================================
``4.0.6``     ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                      
``4.0.7``     ``foss/2016b``, ``intel/2017a``                                                                          
``4.0.8``     ``intel/2017a``, ``intel/2017b``                                                                         
``4.0.9``     ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018b``
``4.0.10``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                     
==========    =========================================================================================================


.. _list_software_libtirpc_857:

*libtirpc*
++++++++++

Libtirpc is a port of Suns Transport-Independent RPC library to Linux.

*homepage*: https://sourceforge.net/projects/libtirpc/

=========    ====================================
version      toolchain                           
=========    ====================================
``1.1.4``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=========    ====================================


.. _list_software_libtool_757:

*libtool*
+++++++++

GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries  behind a consistent, portable interface.

*homepage*: http://www.gnu.org/software/libtool

=========    ====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
=========    ====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``2.4.2``    ``GCC/4.8.2``, ``GCC/4.9.2``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
``2.4.5``    ``GCC/4.8.4``, ``GCC/4.9.2``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
``2.4.6``    ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
=========    ====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_libunistring_1306:

*libunistring*
++++++++++++++

This library provides functions for manipulating Unicode strings and for  manipulating C strings according to the Unicode standard. 

*homepage*: http://www.gnu.org/software/libunistring/

==========    =======================================================================
version       toolchain                                                              
==========    =======================================================================
``0.9.3``     ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
``0.9.6``     ``GCCcore/5.4.0``, ``foss/2016b``, ``foss/2017a``, ``intel/2016b``     
``0.9.7``     ``GCCcore/6.4.0``                                                      
``0.9.10``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``                                   
==========    =======================================================================


.. _list_software_libunwind_972:

*libunwind*
+++++++++++

The primary goal of libunwind is to define a portable and efficient C programming interface  (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the  preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain  (non-local goto). The API supports both local (same-process) and remote (across-process) operation.  As such, the API is useful in a number of applications

*homepage*: https://www.nongnu.org/libunwind/

=========    =======================================================================
version      toolchain                                                              
=========    =======================================================================
``1.1``      ``GCC/4.9.2``, ``foss/2016a``, ``intel/2016b``                         
``1.2.1``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016b``
``1.3.1``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                   
=========    =======================================================================


.. _list_software_libutempter_1197:

*libutempter*
+++++++++++++

libutempter is library that provides an interface for terminal  emulators such as screen and xterm to record user sessions to utmp and wtmp files.

*homepage*: http://git.altlinux.org/people/ldv/packages/?p=libutempter.git

===========    ==================
version        toolchain         
===========    ==================
``1.1.6.2``    ``GCC/6.4.0-2.28``
===========    ==================


.. _list_software_LibUUID_590:

*LibUUID*
+++++++++

Portable uuid C library

*homepage*: http://sourceforge.net/projects/libuuid/

=========    =========================================================================================================
version      toolchain                                                                                                
=========    =========================================================================================================
``1.0.3``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2016a``, ``intel/2017a``, ``intel/2017b``
=========    =========================================================================================================


.. _list_software_libvdwxc_867:

*libvdwxc*
++++++++++

libvdwxc is a general library for evaluating energy and potential for exchange-correlation (XC) functionals from the vdW-DF family that can be used with various of density functional theory (DFT) codes.

*homepage*: http://libvdwxc.org

=========    ==============
version      toolchain     
=========    ==============
``0.3.2``    ``foss/2018b``
``0.4.0``    ``foss/2019a``
=========    ==============


.. _list_software_libwebp_741:

*libwebp*
+++++++++

WebP is a modern image format that provides superior lossless and lossy compression for images on the web. Using WebP, webmasters and web developers can create smaller, richer images that make the web faster.

*homepage*: https://developers.google.com/speed/webp/

=========    ====================================
version      toolchain                           
=========    ====================================
``1.0.0``    ``foss/2018b``                      
``1.0.2``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=========    ====================================


.. _list_software_libX11_497:

*libX11*
++++++++

X11 client-side library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.6.3``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libXau_613:

*libXau*
++++++++

The libXau package contains a library implementing the X11 Authorization Protocol. This is useful for restricting client access to the display.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.0.8``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libxc_530:

*libxc*
+++++++

Libxc is a library of exchange-correlation functionals for density-functional theory.  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals.

*homepage*: http://www.tddft.org/programs/octopus/wiki/index.php/Libxc

=========    =======================================================================================================================================
version      toolchain                                                                                                                              
=========    =======================================================================================================================================
``2.2.2``    ``GCC/8.2.0-2.31.1``                                                                                                                   
``2.2.3``    ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017b``, ``intel/2018a``                                                     
``3.0.0``    ``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``         
``3.0.1``    ``GCC/8.2.0-2.31.1``, ``foss/2016b``, ``foss/2017a``, ``foss/2018a``, ``foss/2018b``, ``gimkl/2017a``, ``intel/2018a``, ``intel/2018b``
``4.0.1``    ``foss/2017b``, ``intel/2017b``                                                                                                        
``4.0.3``    ``foss/2016b``, ``foss/2017a``                                                                                                         
``4.2.3``    ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``gimkl/2017a``, ``intel/2018a``, ``intel/2018b``                                      
``4.3.4``    ``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``                                                                         
=========    =======================================================================================================================================


.. _list_software_libxcb_628:

*libxcb*
++++++++

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.

*homepage*: http://xcb.freedesktop.org/

==========    =================================================
version       toolchain                                        
==========    =================================================
``1.11.1``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
==========    =================================================


.. _list_software_libXcursor_1069:

*libXcursor*
++++++++++++

X Cursor management library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

==========    ===============================
version       toolchain                      
==========    ===============================
``1.1.14``    ``foss/2016a``, ``intel/2016a``
==========    ===============================


.. _list_software_libXdamage_1006:

*libXdamage*
++++++++++++

X Damage extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.1.4``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libXdmcp_819:

*libXdmcp*
++++++++++

The libXdmcp package contains a library implementing the X Display Manager Control Protocol. This is useful for allowing clients to interact with the X Display Manager. 

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.1.2``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libXext_736:

*libXext*
+++++++++

Common X Extensions library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.3.3``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libXfixes_942:

*libXfixes*
+++++++++++

X Fixes extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``5.0.1``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``5.0.2``    ``intel/2016a``                                  
=========    =================================================


.. _list_software_libXfont_838:

*libXfont*
++++++++++

X font libary

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===================    =================================================
version      versionsuffix          toolchain                                        
=========    ===================    =================================================
``1.5.1``                           ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.5.1``    ``-freetype-2.6.3``    ``foss/2016a``, ``intel/2016a``                  
=========    ===================    =================================================


.. _list_software_libXft_617:

*libXft*
++++++++

X11 client-side library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =======================    ===============================
version      versionsuffix              toolchain                      
=========    =======================    ===============================
``2.3.2``                               ``foss/2016a``, ``intel/2016a``
``2.3.2``    ``-fontconfig-2.11.95``    ``intel/2016a``                
``2.3.2``    ``-freetype-2.6.3``        ``foss/2016a``                 
=========    =======================    ===============================


.. _list_software_libXi_504:

*libXi*
+++++++

LibXi provides an X Window System client interface to the XINPUT extension to the X protocol.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.7.6``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_libXinerama_1132:

*libXinerama*
+++++++++++++

Xinerama multiple monitor library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.1.3``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_libxkbcommon_1285:

*libxkbcommon*
++++++++++++++

xkbcommon is a library to handle keyboard descriptions,  including loading them from disk, parsing them and handling their state.  It's mainly meant for client toolkits, window systems,  and other system applications.

*homepage*: http://xkbcommon.org/

=========    ===============================
version      toolchain                      
=========    ===============================
``0.6.1``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_libxml++_734:

*libxml++*
++++++++++

libxml++ is a C++ wrapper for the libxml XML parser library.

*homepage*: http://libxmlplusplus.sourceforge.net

==========    =================
version       toolchain        
==========    =================
``2.40.1``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_libxml2_698:

*libxml2*
+++++++++

Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). 

*homepage*: http://xmlsoft.org/

=========    ==================    ==================================================================================================================================================================================
version      versionsuffix         toolchain                                                                                                                                                                         
=========    ==================    ==================================================================================================================================================================================
``2.9.2``                          ``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``                                                                                               
``2.9.3``                          ``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                                                                                                             
``2.9.3``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                                                                                                                                                   
``2.9.4``                          ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.9.4``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                                                                                                                                   
``2.9.4``    ``-Python-2.7.13``    ``intel/2017a``                                                                                                                                                                   
``2.9.5``                          ``GCCcore/6.3.0``                                                                                                                                                                 
``2.9.6``                          ``GCCcore/6.4.0``                                                                                                                                                                 
``2.9.7``                          ``GCCcore/6.4.0``                                                                                                                                                                 
``2.9.8``                          ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``                                                                                                        
``2.9.9``                          ``GCCcore/8.3.0``                                                                                                                                                                 
=========    ==================    ==================================================================================================================================================================================


.. _list_software_libxml2-python_1417:

*libxml2-python*
++++++++++++++++

Libxml2 is the XML C parser and toolchain developed for the Gnome project  (but usable outside of the Gnome platform). This is the Python binding.

*homepage*: http://xmlsoft.org/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.9.7``    ``-Python-2.7.14``    ``intel/2018a``
``2.9.8``    ``-Python-2.7.15``    ``foss/2018b`` 
=========    ==================    ===============


.. _list_software_libXmu_625:

*libXmu*
++++++++

libXmu provides a set of miscellaneous utility convenience functions for X libraries to use.  libXmuu is a lighter-weight version that does not depend on libXt or libXext

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.1.2``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_libXp_511:

*libXp*
+++++++

libXp provides the X print library.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============
version      toolchain      
=========    ===============
``1.0.3``    ``intel/2016a``
=========    ===============


.. _list_software_libXpm_620:

*libXpm*
++++++++

libXp provides the X print library.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

==========    ===============================
version       toolchain                      
==========    ===============================
``3.5.11``    ``foss/2016a``, ``intel/2016a``
==========    ===============================


.. _list_software_libXrandr_934:

*libXrandr*
+++++++++++

X Resize, Rotate and Reflection extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.5.0``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_libXrender_1039:

*libXrender*
++++++++++++

X11 client-side library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``0.9.9``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libxslt_770:

*libxslt*
+++++++++

Libxslt is the XSLT C library developed for the GNOME project  (but usable outside of the Gnome platform).

*homepage*: http://xmlsoft.org/

==========    ==================    =================================================================
version       versionsuffix         toolchain                                                        
==========    ==================    =================================================================
``1.1.28``                          ``foss/2016a``, ``intel/2016a``                                  
``1.1.28``    ``-Python-2.7.11``    ``intel/2016a``                                                  
``1.1.29``                          ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``1.1.30``                          ``GCCcore/6.3.0``                                                
``1.1.32``                          ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                             
``1.1.33``                          ``GCCcore/8.2.0``                                                
==========    ==================    =================================================================


.. _list_software_libxsmm_764:

*libxsmm*
+++++++++

LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications targeting Intel Architecture (x86).

*homepage*: https://github.com/hfp/libxsmm

=========    ===============================================================================
version      toolchain                                                                      
=========    ===============================================================================
``1.4``      ``intel/2016a``                                                                
``1.4.4``    ``foss/2016b``, ``intel/2016b``                                                
``1.6.4``    ``foss/2016b``, ``intel/2016b``                                                
``1.8.2``    ``intel/2017b``                                                                
``1.8.3``    ``foss/2018a``, ``intel/2018a``                                                
``1.10``     ``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
=========    ===============================================================================


.. _list_software_libXt_515:

*libXt*
+++++++

libXt provides the X Toolkit Intrinsics, an abstract widget library upon which other toolkits are  based. Xt is the basis for many toolkits, including the Athena widgets (Xaw), and LessTif (a Motif implementation).

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.1.5``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_libXxf86vm_958:

*libXxf86vm*
++++++++++++

X11 XFree86 video mode extension library

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.1.4``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_libyaml_746:

*libyaml*
+++++++++

LibYAML is a YAML parser and emitter written in C.

*homepage*: https://pyyaml.org/wiki/LibYAML

=========    ===================================================================
version      toolchain                                                          
=========    ===================================================================
``0.1.6``    ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``0.1.7``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``system``
``0.2.1``    ``GCCcore/7.3.0``, ``system``                                      
``0.2.2``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                               
=========    ===================================================================


.. _list_software_libzip_650:

*libzip*
++++++++

libzip is a C library for reading, creating, and modifying zip archives.

*homepage*: https://libzip.org/

=========    =================
version      toolchain        
=========    =================
``1.5.2``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_lifelines_955:

*lifelines*
+++++++++++

lifelines is a pure Python implementation of the best parts of survival analysis

*homepage*: https://lifelines.readthedocs.io

==========    =================    ==================
version       versionsuffix        toolchain         
==========    =================    ==================
``0.22.8``    ``-Python-3.7.2``    ``fosscuda/2019a``
==========    =================    ==================


.. _list_software_Lighter_719:

*Lighter*
+++++++++

Fast and memory-efficient sequencing error corrector

*homepage*: https://github.com/mourisl/Lighter

=========    ==============
version      toolchain     
=========    ==============
``1.1.1``    ``foss/2018a``
``1.1.2``    ``foss/2018b``
=========    ==============


.. _list_software_likwid_644:

*likwid*
++++++++

Likwid stands for Like I knew what I am doing. This project contributes easy  to use command line tools for Linux to support programmers in developing high  performance multi threaded programs. 

*homepage*: https://github.com/RRZE-HPC/likwid

=========    =====================================================================
version      toolchain                                                            
=========    =====================================================================
``4.0.1``    ``GNU/4.9.3-2.25``                                                   
``4.1.0``    ``GCCcore/4.9.3``                                                    
``4.2.0``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2017a``, ``intel/2017a``
``4.3.2``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                                 
=========    =====================================================================


.. _list_software_LinBox_588:

*LinBox*
++++++++

C++ library for exact, high-performance linear algebra

*homepage*: http://linalg.org/

=========    ==============
version      toolchain     
=========    ==============
``1.4.0``    ``foss/2016a``
=========    ==============


.. _list_software_LittleCMS_849:

*LittleCMS*
+++++++++++

Little CMS intends to be an OPEN SOURCE small-footprint color management engine,  with special focus on accuracy and performance. 

*homepage*: http://www.littlecms.com/

=======    ========================================================================================================
version    toolchain                                                                                               
=======    ========================================================================================================
``2.7``    ``intel/2016a``                                                                                         
``2.8``    ``GCCcore/6.4.0``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``                                     
``2.9``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2017b``, ``foss/2018a``, ``intel/2017b``
=======    ========================================================================================================


.. _list_software_LLVM_315:

*LLVM*
++++++

The LLVM Core libraries provide a modern source- and target-independent  optimizer, along with code generation support for many popular CPUs  (as well as some less common ones!) These libraries are built around a well  specified code representation known as the LLVM intermediate representation  ("LLVM IR"). The LLVM Core libraries are well documented, and it is  particularly easy to invent your own language (or port an existing compiler)  to use LLVM as an optimizer and code generator.

*homepage*: http://llvm.org/

=========    ========================================================================
version      toolchain                                                               
=========    ========================================================================
``3.7.1``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                       
``3.8.0``    ``foss/2016a``, ``intel/2016a``                                         
``3.8.1``    ``GCCcore/4.9.3``, ``foss/2016b``, ``intel/2016b``                      
``3.9.0``    ``foss/2016b``, ``intel/2016b``                                         
``3.9.1``    ``foss/2017a``                                                          
``4.0.0``    ``foss/2017a``, ``intel/2017a``                                         
``4.0.1``    ``intel/2017a``, ``intel/2017b``                                        
``5.0.0``    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``5.0.1``    ``GCCcore/6.4.0``                                                       
``6.0.0``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                                    
``7.0.0``    ``GCCcore/7.2.0``, ``GCCcore/7.3.0``                                    
``7.0.1``    ``GCCcore/8.2.0``                                                       
``9.0.0``    ``GCCcore/8.3.0``                                                       
=========    ========================================================================


.. _list_software_LMDB_287:

*LMDB*
++++++

LMDB is a fast, memory-efficient database. With memory-mapped files, it has the read performance  of a pure in-memory database while retaining the persistence of standard disk-based databases.

*homepage*: https://symas.com/lmdb

==========    ==================================
version       toolchain                         
==========    ==================================
``0.9.18``    ``foss/2016a``                    
``0.9.21``    ``GCCcore/6.4.0``, ``intel/2017a``
``0.9.22``    ``GCCcore/7.3.0``                 
``0.9.23``    ``GCCcore/8.2.0``                 
``0.9.24``    ``GCCcore/8.3.0``                 
==========    ==================================


.. _list_software_LMfit_476:

*LMfit*
+++++++

Lmfit provides a high-level interface to non-linear optimization and curve fitting problems for Python

*homepage*: https://lmfit.github.io/lmfit-py

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.9.9``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_Lmod_396:

*Lmod*
++++++

Lmod is a Lua based module system. Modules allow for dynamic modification  of a user's environment under Unix systems. See www.tacc.utexas.edu/tacc-projects/lmod  for a complete description. Lmod is a new implementation that easily handles the MODULEPATH  Hierarchical problem. It is drop-in replacement for TCL/C modules and reads TCL modulefiles directly.

*homepage*: http://sourceforge.net/projects/lmod/

=========    ============================
version      toolchain                   
=========    ============================
``5.2``      ``GCC/4.8.2``               
``5.2.5``    ``GCC/4.8.2``               
``5.3``      ``GCC/4.8.2``               
``5.4``      ``GCC/4.8.2``               
``5.4.2``    ``GCC/4.8.2``               
``5.5``      ``GCC/4.8.2``               
``5.5.1``    ``GCC/4.8.2``               
``5.6``      ``GCC/4.8.2``               
``5.7``      ``GCC/4.8.2``               
``5.8``      ``GCC/4.8.2``               
``5.8.5``    ``GCC/4.8.2``               
``5.9``      ``GCC/4.8.2``, ``GCC/4.8.4``
``6.4.2``    ``system``                  
``7.3``      ``system``                  
=========    ============================


.. _list_software_LocARNA_576:

*LocARNA*
+++++++++

LocARNA is a collection of alignment tools for the structural analysis of RNA.  Given a set of RNA sequences, LocARNA simultaneously aligns and predicts common structures for  your RNAs. In this way, LocARNA performs Sankoff-like alignment and is in particular suited for  analyzing sets of related RNAs without known common structure.

*homepage*: http://www.bioinf.uni-freiburg.de/Software/LocARNA/

===========    =================    ==============
version        versionsuffix        toolchain     
===========    =================    ==============
``1.9.2``                           ``foss/2016b``
``1.9.2.2``    ``-Python-3.6.6``    ``foss/2018b``
===========    =================    ==============


.. _list_software_LoFreq_585:

*LoFreq*
++++++++

Fast and sensitive variant calling from next-gen sequencing data

*homepage*: http://csb5.github.io/lofreq

===========    ==================    ================================================
version        versionsuffix         toolchain                                       
===========    ==================    ================================================
``2.1.2``      ``-Python-2.7.12``    ``intel/2016b``                                 
``2.1.3.1``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
===========    ==================    ================================================


.. _list_software_Log-Log4perl_1112:

*Log-Log4perl*
++++++++++++++

Log4perl

*homepage*: https://metacpan.org/pod/Log::Log4perl

========    ==============
version     toolchain     
========    ==============
``1.47``    ``foss/2016a``
========    ==============


.. _list_software_LOHHLA_440:

*LOHHLA*
++++++++

LOHHLA, Loss Of Heterozygosity in Human Leukocyte Antigen, a computational tool to evaluate HLA loss using next-generation sequencing data.

*homepage*: https://bitbucket.org/mcgranahanlab/lohhla

==============    =============    ==============
version           versionsuffix    toolchain     
==============    =============    ==============
``2018.11.05``    ``-R-3.5.1``     ``foss/2018b``
==============    =============    ==============


.. _list_software_Loki_399:

*Loki*
++++++

Loki is a C++ library of designs, containing flexible implementations of common design patterns and   idioms. 

*homepage*: http://loki-lib.sourceforge.net/

=========    =================================================================================
version      toolchain                                                                        
=========    =================================================================================
``0.1.7``    ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2018a``
=========    =================================================================================


.. _list_software_Longshot_846:

*Longshot*
++++++++++

Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific  Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs  a phased VCF file with variants and haplotype information. It can also output haplotype-separated BAM files that can  be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs).

*homepage*: https://github.com/pjedge/longshot

=========    =================
version      toolchain        
=========    =================
``0.3.4``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_lpsolve_773:

*lpsolve*
+++++++++

Mixed Integer Linear Programming (MILP) solver

*homepage*: https://sourceforge.net/projects/lpsolve/

===========    ======================================================================================================================================================
version        toolchain                                                                                                                                             
===========    ======================================================================================================================================================
``5.5.2.5``    ``GCC/6.4.0-2.28``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2017a``, ``intel/2018b``
===========    ======================================================================================================================================================


.. _list_software_lrslib_648:

*lrslib*
++++++++

lrslib is a self-contained ANSI C implementation of the reverse search algorithm for vertex enumeration/convex hull problems

*homepage*: http://cgm.cs.mcgill.ca/~avis/C/lrs.html

=======    ===============
version    toolchain      
=======    ===============
``6.2``    ``intel/2018b``
=======    ===============


.. _list_software_LS-PrePost_921:

*LS-PrePost*
++++++++++++

LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA.

*homepage*: http://lstc.com/products/ls-prepost

=======    =============    ==========
version    versionsuffix    toolchain 
=======    =============    ==========
``4.6``    ``-centos6``     ``system``
``4.6``    ``-centos7``     ``system``
=======    =============    ==========


.. _list_software_LSMS_319:

*LSMS*
++++++

LSMS benchmark, part of CORAL suite

*homepage*: https://asc.llnl.gov/CORAL-benchmarks/#lsms

============    ==============
version         toolchain     
============    ==============
``3_rev237``    ``foss/2016a``
============    ==============


.. _list_software_Lua_290:

*Lua*
+++++

Lua is a powerful, fast, lightweight, embeddable scripting language.  Lua combines simple procedural syntax with powerful data description constructs based  on associative arrays and extensible semantics. Lua is dynamically typed,  runs by interpreting bytecode for a register-based virtual machine,  and has automatic memory management with incremental garbage collection,  making it ideal for configuration, scripting, and rapid prototyping.

*homepage*: http://www.lua.org/

===========    ========================================
version        toolchain                               
===========    ========================================
``5.1.4-5``    ``GCC/4.8.2``                           
``5.1.4-8``    ``GCC/4.8.2``, ``GCC/4.8.4``, ``system``
``5.1.5``      ``GCCcore/7.3.0``                       
``5.2.4``      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``    
``5.3.4``      ``GCCcore/7.2.0``, ``system``           
``5.3.5``      ``GCCcore/8.2.0``, ``system``           
===========    ========================================


.. _list_software_LuaJIT_521:

*LuaJIT*
++++++++

LuaJIT is a Just-In-Time Compiler (JIT) for the Lua programming language. Lua is a powerful, dynamic and light-weight programming language. It may be embedded or used as a general-purpose, stand-alone language. 

*homepage*: http://luajit.org/

=========    =============
version      toolchain    
=========    =============
``2.0.2``    ``GCC/4.9.2``
=========    =============


.. _list_software_Lucene-Geo-Gazetteer_1916:

*Lucene-Geo-Gazetteer*
++++++++++++++++++++++

A command line gazetteer built around the Geonames.org dataset, that uses the Apache Lucene library  to create a searchable gazetteer.

*homepage*: https://github.com/chrismattmann/lucene-geo-gazetteer

============    ==========
version         toolchain 
============    ==========
``20170718``    ``system``
============    ==========


.. _list_software_LUMPY_407:

*LUMPY*
+++++++

A probabilistic framework for structural variant discovery. 

*homepage*: https://github.com/arq5x/lumpy-sv

==========    ==============
version       toolchain     
==========    ==============
``0.2.13``    ``foss/2016b``
==========    ==============


.. _list_software_LUSCUS_479:

*LUSCUS*
++++++++

Luscus is the program for graphical display and editing of molecular systems.

*homepage*: https://sourceforge.net/projects/luscus/

=========    ===============================
version      toolchain                      
=========    ===============================
``0.8.6``    ``foss/2018b``, ``intel/2018a``
=========    ===============================


.. _list_software_lwgrp_556:

*lwgrp*
+++++++

The Light-weight Group Library provides methods for MPI codes to quickly create  and destroy process groups  

*homepage*: https://github.com/llnl/lwgrp

=========    ================================
version      toolchain                       
=========    ================================
``1.0.2``    ``gompi/2019a``, ``iimpi/2019a``
=========    ================================


.. _list_software_lxml_445:

*lxml*
++++++

The lxml XML toolkit is a Pythonic binding for the C libraries libxml2 and libxslt.

*homepage*: http://lxml.de/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``3.5.0``    ``-Python-2.7.11``    ``intel/2016a``                
``3.6.0``    ``-Python-2.7.11``    ``intel/2016a``                
``3.6.4``    ``-Python-2.7.12``    ``intel/2016b``                
``4.0.0``    ``-Python-2.7.13``    ``intel/2017a``                
``4.1.1``    ``-Python-2.7.14``    ``intel/2017b``                
``4.2.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``
``4.2.0``    ``-Python-3.6.4``     ``intel/2018a``                
``4.2.5``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
``4.2.5``    ``-Python-3.6.6``     ``foss/2018b``                 
``4.3.3``                          ``GCCcore/8.2.0``              
=========    ==================    ===============================


.. _list_software_lynx_459:

*lynx*
++++++

lynx is an alphanumeric display oriented World-Wide Web Client

*homepage*: http://lynx.browser.org/

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``2.8.9``    ``-develop``     ``foss/2016b``
=========    =============    ==============


.. _list_software_lz4_282:

*lz4*
+++++

LZ4 is lossless compression algorithm, providing compression speed at 400 MB/s per core.   It features an extremely fast decoder, with speed in multiple GB/s per core.

*homepage*: https://lz4.github.io/lz4/

=========    ====================================
version      toolchain                           
=========    ====================================
``1.8.2``    ``GCCcore/5.4.0``, ``GCCcore/6.4.0``
``1.9.0``    ``GCCcore/7.3.0``                   
``1.9.1``    ``GCCcore/8.2.0``                   
``1.9.2``    ``GCCcore/8.3.0``                   
=========    ====================================


.. _list_software_LZO_245:

*LZO*
+++++

Portable lossless data compression library

*homepage*: http://www.oberhumer.com/opensource/lzo/

========    =========================================================================================================================================================
version     toolchain                                                                                                                                                
========    =========================================================================================================================================================
``2.09``    ``intel/2016b``, ``intel/2017b``                                                                                                                         
``2.10``    ``GCCcore/6.4.0``, ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2017a``
========    =========================================================================================================================================================


.. _list_software_letter_m:

*M*
---


:ref:`list_software_M4_129` - :ref:`list_software_MACH_281` - :ref:`list_software_MACS2_342` - :ref:`list_software_maeparser_960` - :ref:`list_software_MAFFT_366` - :ref:`list_software_magick_620` - :ref:`list_software_magma_515` - :ref:`list_software_MAGMA_355` - :ref:`list_software_MagresPython_1249` - :ref:`list_software_mahotas_749` - :ref:`list_software_MAJIQ_370` - :ref:`list_software_make_414` - :ref:`list_software_makedepend_1038` - :ref:`list_software_makedepf90_934` - :ref:`list_software_Mako_392` - :ref:`list_software_Maple_495` - :ref:`list_software_MapSplice_894` - :ref:`list_software_Maq_287` - :ref:`list_software_MariaDB_624` - :ref:`list_software_MariaDB-connector-c_1784` - :ref:`list_software_Mash_393` - :ref:`list_software_MaSuRCA_588` - :ref:`list_software_Mathematica_1118` - :ref:`list_software_MathGL_541` - :ref:`list_software_MATIO_378` - :ref:`list_software_MATLAB_433` - :ref:`list_software_MATLAB-Engine_1076` - :ref:`list_software_matplotlib_1080` - :ref:`list_software_MATSim_523` - :ref:`list_software_Maven_503` - :ref:`list_software_mawk_432` - :ref:`list_software_MaxBin_575` - :ref:`list_software_MaxQuant_815` - :ref:`list_software_mayavi_647` - :ref:`list_software_mc_208` - :ref:`list_software_MCL_220` - :ref:`list_software_MCR_226` - :ref:`list_software_MDBM_288` - :ref:`list_software_MDSplus_680` - :ref:`list_software_mdtest_657` - :ref:`list_software_MDTraj_546` - :ref:`list_software_medaka_611` - :ref:`list_software_Meep_391` - :ref:`list_software_MEGA_282` - :ref:`list_software_MEGACC_416` - :ref:`list_software_MEGAHIT_511` - :ref:`list_software_MEME_292` - :ref:`list_software_memkind_741` - :ref:`list_software_memory-profiler_1577` - :ref:`list_software_meRanTK_658` - :ref:`list_software_MERCKX_458` - :ref:`list_software_Mercurial_932` - :ref:`list_software_Mesa_390` - :ref:`list_software_meshio_645` - :ref:`list_software_Meson_514` - :ref:`list_software_Mesquite_845` - :ref:`list_software_MetaBAT_606` - :ref:`list_software_MetaboAnalystR_1414` - :ref:`list_software_MetaGeneAnnotator_1724` - :ref:`list_software_Metal_499` - :ref:`list_software_MetaPhlAn2_908` - :ref:`list_software_metaWRAP_737` - :ref:`list_software_Metaxa2_658` - :ref:`list_software_MethylDackel_1207` - :ref:`list_software_methylpy_892` - :ref:`list_software_METIS_386` - :ref:`list_software_MiGEC_389` - :ref:`list_software_MIGRATE-N_644` - :ref:`list_software_MINC_295` - :ref:`list_software_Miniconda2_964` - :ref:`list_software_Miniconda3_965` - :ref:`list_software_minieigen_949` - :ref:`list_software_Minimac4_754` - :ref:`list_software_minimap2_797` - :ref:`list_software_MinPath_689` - :ref:`list_software_MIRA_297` - :ref:`list_software_miRDeep2_728` - :ref:`list_software_MiXCR_419` - :ref:`list_software_mkl-dnn_689` - :ref:`list_software_mkl-service_1122` - :ref:`list_software_MLC_220` - :ref:`list_software_Mmg_289` - :ref:`list_software_MMSEQ_387` - :ref:`list_software_MMseqs2_648` - :ref:`list_software_modred_635` - :ref:`list_software_Molcas_607` - :ref:`list_software_Molden_607` - :ref:`list_software_Molekel_713` - :ref:`list_software_molmod_648` - :ref:`list_software_Molpro_633` - :ref:`list_software_Mono_409` - :ref:`list_software_mordecai_836` - :ref:`list_software_mosdepth_868` - :ref:`list_software_Mothur_639` - :ref:`list_software_motif_543` - :ref:`list_software_MotionCor2_972` - :ref:`list_software_MoviePy_713` - :ref:`list_software_MPC_224` - :ref:`list_software_MPFR_309` - :ref:`list_software_mpi4py_611` - :ref:`list_software_MPICH_369` - :ref:`list_software_MPICH2_419` - :ref:`list_software_mpifileutils_1303` - :ref:`list_software_mpiP_406` - :ref:`list_software_MPJ-Express_1022` - :ref:`list_software_mpmath_647` - :ref:`list_software_MrBayes_691` - :ref:`list_software_MRIcron_666` - :ref:`list_software_MRtrix_614` - :ref:`list_software_MSM_237` - :ref:`list_software_MSPC_307` - :ref:`list_software_msprime_765` - :ref:`list_software_MTL4_289` - :ref:`list_software_MultiNest_933` - :ref:`list_software_MultiQC_671` - :ref:`list_software_Multiwfn_854` - :ref:`list_software_MUMmer_563` - :ref:`list_software_MUMPS_402` - :ref:`list_software_muParser_847` - :ref:`list_software_MuPeXI_536` - :ref:`list_software_MUSCLE_457` - :ref:`list_software_MuSiC_449` - :ref:`list_software_MuTect_594` - :ref:`list_software_mutil_555` - :ref:`list_software_MVAPICH2_570` - :ref:`list_software_MXNet_460` - :ref:`list_software_mygene_645` - :ref:`list_software_MyMediaLite_1076` - :ref:`list_software_mympingpong_1205` - :ref:`list_software_mypy_463` - :ref:`list_software_MySQL_438` - :ref:`list_software_MySQL-python_1157` - :ref:`list_software_mysqlclient_1205`


.. _list_software_M4_129:

*M4*
++++

GNU M4 is an implementation of the traditional Unix macro processor. It is  mostly SVR4 compatible although it has some extensions (for example, handling  more than 9 positional parameters to macros). GNU M4 also has built-in  functions for including files, running shell commands, doing arithmetic, etc. 

*homepage*: http://www.gnu.org/software/m4/m4.html

==========    ===========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version       toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
==========    ===========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``1.4.16``    ``GCC/4.8.1``, ``GCC/4.8.2``, ``system``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   
``1.4.17``    ``GCC/4.8.2``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCC/5.2.0``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``, ``GNU/5.1.0-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.4.18``    ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/system``, ``system``                                                                                                                                                                                                                                                                                                   
==========    ===========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_MACH_281:

*MACH*
++++++

MACH 1.0 is a Markov Chain based haplotyper  that can resolve long haplotypes or infer missing genotypes  in samples of unrelated individuals.

*homepage*: http://csg.sph.umich.edu/abecasis/MACH/

==========    ==========
version       toolchain 
==========    ==========
``1.0.18``    ``system``
==========    ==========


.. _list_software_MACS2_342:

*MACS2*
+++++++

Model Based Analysis for ChIP-Seq data

*homepage*: https://github.com/taoliu/MACS/

==================    ==================    ===============================
version               versionsuffix         toolchain                      
==================    ==================    ===============================
``2.1.1.20160309``    ``-Python-2.7.14``    ``intel/2017b``                
``2.1.2.1``           ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``2.1.2.1``           ``-Python-2.7.15``    ``intel/2019a``                
==================    ==================    ===============================


.. _list_software_maeparser_960:

*maeparser*
+++++++++++

maeparser is a parser for Schrodinger Maestro files.

*homepage*: https://github.com/schrodinger/maeparser

=========    ================================
version      toolchain                       
=========    ================================
``1.2.2``    ``gompi/2019a``, ``iimpi/2019a``
=========    ================================


.. _list_software_MAFFT_366:

*MAFFT*
+++++++

MAFFT is a multiple sequence alignment program   for unix-like operating systems.  It offers a range of multiple  alignment methods, L-INS-i (accurate; for alignment of <∼200 sequences),   FFT-NS-2 (fast; for alignment of <∼10,000 sequences), etc.

*homepage*: https://mafft.cbrc.jp/alignment/software/source.html

=========    ====================    ====================
version      versionsuffix           toolchain           
=========    ====================    ====================
``7.305``    ``-with-extensions``    ``foss/2016b``      
``7.397``    ``-with-extensions``    ``intel/2018a``     
``7.427``    ``-with-extensions``    ``intel/2018b``     
``7.429``    ``-with-extensions``    ``GCC/8.2.0-2.31.1``
=========    ====================    ====================


.. _list_software_magick_620:

*magick*
++++++++

R bindings to the open-source image processing library ImageMagick

*homepage*: https://cran.r-project.org/web/packages/magick/

=======    =============    ==============
version    versionsuffix    toolchain     
=======    =============    ==============
``2.0``    ``-R-3.5.1``     ``foss/2018b``
=======    =============    ==============


.. _list_software_magma_515:

*magma*
+++++++

The MAGMA project aims to develop a dense linear algebra library similar to  LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems.

*homepage*: https://icl.cs.utk.edu/magma/

=========    ================    =======================================
version      versionsuffix       toolchain                              
=========    ================    =======================================
``2.3.0``                        ``fosscuda/2017b``, ``intelcuda/2017b``
``2.3.0``    ``-CUDA-9.1.85``    ``foss/2018a``                         
``2.4.0``                        ``fosscuda/2018b``                     
``2.5.0``                        ``fosscuda/2018b``                     
``2.5.1``                        ``fosscuda/2019a``, ``fosscuda/2019b`` 
=========    ================    =======================================


.. _list_software_MAGMA_355:

*MAGMA*
+++++++

MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous  GWAS or meta-analysis.

*homepage*: https://ctg.cncr.nl/software/magma

=========    ==============
version      toolchain     
=========    ==============
``1.07b``    ``foss/2018b``
=========    ==============


.. _list_software_MagresPython_1249:

*MagresPython*
++++++++++++++

MagresPython is a Python library for parsing the CCP-NC ab-initio magnetic resonance file format.  This is used in the latest version of the CASTEP and Quantum ESPRESSO (PWSCF) codes. 

*homepage*: http://tfgg.me/magres-python

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20160329``    ``-Python-2.7.15``    ``foss/2018b``
============    ==================    ==============


.. _list_software_mahotas_749:

*mahotas*
+++++++++

Mahotas is a computer vision and image processing library for Python

*homepage*: http://mahotas.readthedocs.io/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.4.3``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_MAJIQ_370:

*MAJIQ*
+++++++

MAJIQ and Voila are two software packages that together detect, quantify, and visualize  local splicing variations (LSV) from RNA-Seq data.

*homepage*: https://majiq.biociphers.org

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.1.1``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_make_414:

*make*
++++++

GNU version of make utility

*homepage*: http://www.gnu.org/software/make/make.html

=========    ==================
version      toolchain         
=========    ==================
``3.82``     ``GCC/4.8.2``     
``4.1``      ``GCC/4.9.2``     
``4.2.1``    ``GCC/7.3.0-2.30``
=========    ==================


.. _list_software_makedepend_1038:

*makedepend*
++++++++++++

The makedepend package contains a C-preprocessor like utility to determine build-time dependencies.

*homepage*: http://www.linuxfromscratch.org/blfs/view/svn/x/makedepend.html

=========    =======================================================================================
version      toolchain                                                                              
=========    =======================================================================================
``1.0.5``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
``1.0.6``    ``GCCcore/7.3.0``                                                                      
=========    =======================================================================================


.. _list_software_makedepf90_934:

*makedepf90*
++++++++++++

Makedepf90 is a program for automatic creation of  Makefile-style dependency lists for Fortran source code. 

*homepage*: https://linux.die.net/man/1/makedepf90

=========    ==============
version      toolchain     
=========    ==============
``2.8.8``    ``foss/2017a``
=========    ==============


.. _list_software_Mako_392:

*Mako*
++++++

A super-fast templating language that borrows the best ideas from the existing templating languages

*homepage*: http://www.makotemplates.org

=========    ==================    =================================================================================================================================================================
version      versionsuffix         toolchain                                                                                                                                                        
=========    ==================    =================================================================================================================================================================
``1.0.4``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                                                                                                                  
``1.0.6``    ``-Python-2.7.13``    ``foss/2017a``, ``intel/2017a``                                                                                                                                  
``1.0.7``    ``-Python-2.7.14``    ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2018a``
``1.0.7``    ``-Python-2.7.15``    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                                                                                                              
``1.0.7``    ``-Python-3.6.4``     ``intel/2018a``                                                                                                                                                  
``1.0.7``    ``-Python-3.6.6``     ``fosscuda/2018b``                                                                                                                                               
``1.0.8``                          ``GCCcore/8.2.0``                                                                                                                                                
``1.1.0``                          ``GCCcore/8.3.0``                                                                                                                                                
=========    ==================    =================================================================================================================================================================


.. _list_software_Maple_495:

*Maple*
+++++++

Maple combines the world's most powerful mathematical computation engine with an intuitive,    'clickable' user interface.

*homepage*: http://www.maplesoft.com/products/maple/

==========    ==========
version       toolchain 
==========    ==========
``15``        ``system``
``2017.2``    ``system``
==========    ==========


.. _list_software_MapSplice_894:

*MapSplice*
+++++++++++

MapSplice is a software for mapping RNA-seq data to reference genome for splice  junction discovery that depends only on reference genome, and not on any further annotations.

*homepage*: http://www.netlab.uky.edu/p/bioinfo/MapSplice2

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.2.1``    ``-Python-2.7.12``    ``foss/2016b`` 
``2.2.1``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_Maq_287:

*Maq*
+++++

Maq is a software that builds mapping assemblies from short reads generated by the  next-generation sequencing  machines.

*homepage*: http://maq.sourceforge.net/maq-man.shtml

=========    ==========
version      toolchain 
=========    ==========
``0.7.0``    ``system``
=========    ==========


.. _list_software_MariaDB_624:

*MariaDB*
+++++++++

MariaDB An enhanced, drop-in replacement for MySQL.

*homepage*: https://mariadb.org/

===========    ===============================
version        toolchain                      
===========    ===============================
``10.0.21``    ``GNU/4.9.3-2.25``             
``10.1.13``    ``intel/2016a``                
``10.1.14``    ``foss/2016a``, ``intel/2016a``
``10.1.17``    ``intel/2016b``                
``10.1.24``    ``intel/2017a``                
``10.2.11``    ``foss/2017b``, ``intel/2017b``
``10.3.7``     ``intel/2018a``                
``10.3.10``    ``foss/2018b``                 
``10.3.14``    ``foss/2019a``                 
===========    ===============================


.. _list_software_MariaDB-connector-c_1784:

*MariaDB-connector-c*
+++++++++++++++++++++

MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases.

*homepage*: https://downloads.mariadb.org/connector-c/

=========    =================================
version      toolchain                        
=========    =================================
``2.3.4``    ``foss/2017b``, ``intel/2017b``  
``2.3.5``    ``intel/2018a``, ``iomkl/2018a`` 
``2.3.7``    ``GCCcore/8.2.0``, ``foss/2018b``
=========    =================================


.. _list_software_Mash_393:

*Mash*
++++++

Fast genome and metagenome distance estimation using MinHash

*homepage*: http://mash.readthedocs.org

=======    ==============
version    toolchain     
=======    ==============
``2.0``    ``foss/2018a``
``2.1``    ``foss/2018b``
=======    ==============


.. _list_software_MaSuRCA_588:

*MaSuRCA*
+++++++++

MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph    and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing   only short reads from Illumina sequencing or a mixture of short reads and long reads   (Sanger, 454, Pacbio and Nanopore).

*homepage*: http://www.genome.umd.edu/masurca.html

=========    ================    ==============
version      versionsuffix       toolchain     
=========    ================    ==============
``3.2.2``                        ``foss/2016a``
``3.2.2``    ``-Perl-5.26.0``    ``foss/2017b``
``3.2.4``                        ``foss/2018a``
``3.2.5``    ``-Perl-5.26.0``    ``foss/2017b``
``3.3.1``    ``-Perl-5.28.0``    ``foss/2018b``
=========    ================    ==============


.. _list_software_Mathematica_1118:

*Mathematica*
+++++++++++++

Mathematica is a computational software program used in many scientific, engineering, mathematical and computing fields.

*homepage*: http://www.wolfram.com/mathematica

==========    ==========
version       toolchain 
==========    ==========
``9.0.1``     ``system``
``10.0.2``    ``system``
``10.1.0``    ``system``
``10.4.1``    ``system``
``11.0.1``    ``system``
``11.1.1``    ``system``
``11.3.0``    ``system``
``12.0.0``    ``system``
==========    ==========


.. _list_software_MathGL_541:

*MathGL*
++++++++

MathGL is ...  a library for making high-quality scientific graphics under Linux and Windows;  a library for the fast data plotting and data processing of large data arrays;  a library for working in window and console modes and for easy embedding into other programs;  a library with large and growing set of graphics. 

*homepage*: http://mathgl.sourceforge.net/doc_en/Main.html

=========    ==============
version      toolchain     
=========    ==============
``2.4.1``    ``foss/2018a``
=========    ==============


.. _list_software_MATIO_378:

*MATIO*
+++++++

matio is an C library for reading and writing Matlab MAT files.

*homepage*: http://sourceforge.net/projects/matio/

==========    =================
version       toolchain        
==========    =================
``1.5.9``     ``GCCcore/5.4.0``
``1.5.11``    ``foss/2017b``   
``1.5.12``    ``GCCcore/6.4.0``
==========    =================


.. _list_software_MATLAB_433:

*MATLAB*
++++++++

MATLAB is a high-level language and interactive environment  that enables you to perform computationally intensive tasks faster than with  traditional programming languages such as C, C++, and Fortran.

*homepage*: http://www.mathworks.com/products/matlab

=========    ==========
version      toolchain 
=========    ==========
``2012b``    ``system``
``2013b``    ``system``
``2015a``    ``system``
``2016a``    ``system``
``2017a``    ``system``
``2018b``    ``system``
=========    ==========


.. _list_software_MATLAB-Engine_1076:

*MATLAB-Engine*
+++++++++++++++

The MATLAB Engine API for Python provides a package for Python  to call MATLAB as a computational engine.

*homepage*: https://www.mathworks.com/help/matlab/matlab-engine-for-python.html

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``2018b``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``2018b``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
=========    ==================    ===============================


.. _list_software_matplotlib_1080:

*matplotlib*
++++++++++++

matplotlib is a python 2D plotting library which produces publication quality figures in a variety of  hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python  and ipython shell, web application servers, and six graphical user interface toolkits.

*homepage*: https://matplotlib.org

=========    =================================    ========================================================================
version      versionsuffix                        toolchain                                                               
=========    =================================    ========================================================================
``1.4.3``    ``-Python-2.7.12``                   ``intel/2016b``                                                         
``1.5.1``    ``-Python-2.7.11``                   ``foss/2016a``, ``intel/2016a``                                         
``1.5.1``    ``-Python-2.7.11-freetype-2.6.3``    ``foss/2016a``, ``intel/2016a``                                         
``1.5.1``    ``-Python-2.7.12``                   ``intel/2016b``                                                         
``1.5.1``    ``-Python-3.5.1``                    ``foss/2016a``, ``intel/2016a``                                         
``1.5.1``    ``-Python-3.5.2``                    ``intel/2016b``                                                         
``1.5.2``    ``-Python-2.7.12``                   ``foss/2016b``, ``intel/2016b``                                         
``1.5.2``    ``-Python-3.5.2``                    ``intel/2016b``                                                         
``1.5.3``    ``-Python-2.7.12``                   ``foss/2016b``, ``intel/2016b``                                         
``1.5.3``    ``-Python-3.5.2``                    ``foss/2016b``, ``intel/2016b``                                         
``2.0.0``    ``-Python-2.7.12``                   ``foss/2016b``, ``intel/2016b``                                         
``2.0.0``    ``-Python-2.7.13``                   ``intel/2017a``                                                         
``2.0.1``    ``-Python-3.6.1``                    ``intel/2017a``                                                         
``2.0.2``    ``-Python-2.7.13``                   ``foss/2017a``, ``intel/2017a``                                         
``2.0.2``    ``-Python-2.7.13-Qt-4.8.7``          ``intel/2017a``                                                         
``2.0.2``    ``-Python-2.7.13-libpng-1.6.29``     ``intel/2017a``                                                         
``2.0.2``    ``-Python-3.6.1``                    ``foss/2017a``                                                          
``2.0.2``    ``-Python-3.6.1-libpng-1.6.29``      ``intel/2017a``                                                         
``2.1.0``    ``-Python-2.7.14``                   ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``2.1.0``    ``-Python-3.6.2``                    ``foss/2017b``                                                          
``2.1.0``    ``-Python-3.6.3``                    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``2.1.1``    ``-Python-2.7.14``                   ``intel/2017b``                                                         
``2.1.1``    ``-Python-3.6.3``                    ``foss/2017b``, ``intel/2017b``                                         
``2.1.2``    ``-Python-2.7.14``                   ``foss/2018a``, ``intel/2018a``                                         
``2.1.2``    ``-Python-3.6.4``                    ``foss/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``     
``2.2.3``    ``-Python-2.7.15``                   ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                     
``2.2.3``    ``-Python-3.6.6``                    ``intel/2018b``                                                         
``2.2.4``    ``-Python-2.7.15``                   ``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``3.0.0``    ``-Python-3.6.6``                    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``    
``3.0.2``    ``-Python-3.6.6``                    ``foss/2018b``                                                          
``3.0.3``    ``-Python-3.7.2``                    ``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``3.1.1``    ``-Python-3.7.4``                    ``foss/2019b``, ``intel/2019b``                                         
=========    =================================    ========================================================================


.. _list_software_MATSim_523:

*MATSim*
++++++++

MATSim is an open-source framework to implement large-scale agent-based transport simulations.

*homepage*: http://www.matsim.org/

=========    ===================    ===============
version      versionsuffix          toolchain      
=========    ===================    ===============
``0.8.1``    ``-Java-1.8.0_112``    ``intel/2016b``
=========    ===================    ===============


.. _list_software_Maven_503:

*Maven*
+++++++

Binary maven install, Apache Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information. 

*homepage*: https://maven.apache.org/index.html

=========    ==========
version      toolchain 
=========    ==========
``3.2.3``    ``system``
``3.3.3``    ``system``
``3.3.9``    ``system``
``3.5.0``    ``system``
``3.5.2``    ``system``
``3.6.0``    ``system``
=========    ==========


.. _list_software_mawk_432:

*mawk*
++++++

mawk is an interpreter for the AWK Programming Language.

*homepage*: https://invisible-island.net/mawk/

==================    =================================================================================================
version               toolchain                                                                                        
==================    =================================================================================================
``1.3.4-20141206``    ``GCC/4.9.2``                                                                                    
``1.3.4-20171017``    ``foss/2018a``, ``foss/2018b``, ``foss/2019a``, ``intel/2018a``, ``intel/2018b``, ``intel/2019a``
==================    =================================================================================================


.. _list_software_MaxBin_575:

*MaxBin*
++++++++

MaxBin is software for binning assembled metagenomic sequences based on  an Expectation-Maximization algorithm.

*homepage*: https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html

=========    ================    ====================
version      versionsuffix       toolchain           
=========    ================    ====================
``2.2.6``    ``-Perl-5.28.0``    ``foss/2018b``      
``2.2.7``    ``-Perl-5.28.1``    ``GCC/8.2.0-2.31.1``
=========    ================    ====================


.. _list_software_MaxQuant_815:

*MaxQuant*
++++++++++

MaxQuant is a quantitative proteomics software package designed for analyzing large  mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling  techniques as well as label-free quantification are supported.

*homepage*: https://maxquant.org/maxquant/

=============    ==============
version          toolchain     
=============    ==============
``1.6.10.43``    ``foss/2018b``
=============    ==============


.. _list_software_mayavi_647:

*mayavi*
++++++++

The Mayavi scientific data 3-dimensional visualizer

*homepage*: http://code.enthought.com/projects/mayavi/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``4.4.4``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_mc_208:

*mc*
++++

mc-4.6.1: User-friendly file manager and visual shell

*homepage*: https://www.midnight-commander.org/

==========    =============
version       toolchain    
==========    =============
``4.8.13``    ``GCC/4.9.2``
==========    =============


.. _list_software_MCL_220:

*MCL*
+++++

The MCL algorithm is short for the Markov Cluster Algorithm, a fast and scalable unsupervised cluster algorithm for graphs (also known as networks) based on simulation of (stochastic) flow in graphs. 

*homepage*: http://micans.org/mcl/

==========    ================    ===============================
version       versionsuffix       toolchain                      
==========    ================    ===============================
``02.063``                        ``intel/2016b``                
``14.137``                        ``foss/2016a``, ``intel/2016b``
``14.137``    ``-Perl-5.26.1``    ``GCCcore/6.4.0``              
``14.137``    ``-Perl-5.28.0``    ``GCCcore/7.3.0``              
==========    ================    ===============================


.. _list_software_MCR_226:

*MCR*
+++++

The MATLAB Runtime is a standalone set of shared libraries  that enables the execution of compiled MATLAB applications  or components on computers that do not have MATLAB installed.

*homepage*: http://www.mathworks.com/products/compiler/mcr/

==========    ==========
version       toolchain 
==========    ==========
``R2013a``    ``system``
``R2013b``    ``system``
``R2014a``    ``system``
``R2014b``    ``system``
``R2015a``    ``system``
``R2015b``    ``system``
``R2016a``    ``system``
``R2016b``    ``system``
``R2018a``    ``system``
``R2018b``    ``system``
``R2019a``    ``system``
==========    ==========


.. _list_software_MDBM_288:

*MDBM*
++++++

MDBM is a super-fast memory-mapped key/value store

*homepage*: https://github.com/yahoo/mdbm

==========    =================
version       toolchain        
==========    =================
``4.13.0``    ``GCCcore/6.4.0``
==========    =================


.. _list_software_MDSplus_680:

*MDSplus*
+++++++++

MDSplus is a set of software tools for data acquisition and storage and a methodology  for management of complex scientific data.

*homepage*: http://mdsplus.org/

==========    ================================    ==============
version       versionsuffix                       toolchain     
==========    ================================    ==============
``7.0.67``    ``-Java-1.7.0_79-Python-2.7.11``    ``foss/2016a``
``7.46.1``                                        ``foss/2018a``
==========    ================================    ==============


.. _list_software_mdtest_657:

*mdtest*
++++++++

mdtest is an MPI-coordinated metadata benchmark test that performs   open/stat/close operations on files and directories and then reports the performance.

*homepage*: http://sourceforge.net/projects/mdtest/

=========    ===============
version      toolchain      
=========    ===============
``1.9.3``    ``intel/2017a``
=========    ===============


.. _list_software_MDTraj_546:

*MDTraj*
++++++++

Read, write and analyze MD trajectories with only a few lines of Python code.

*homepage*: http://mdtraj.org

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.9.1``    ``-Python-3.6.3``    ``intel/2017b``
``1.9.2``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_medaka_611:

*medaka*
++++++++

medaka is a tool to create a consensus sequence of nanopore sequencing data.

*homepage*: https://nanoporetech.github.io/medaka

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.4.3``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_Meep_391:

*Meep*
++++++

Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package  developed at MIT to model electromagnetic systems.

*homepage*: http://ab-initio.mit.edu/wiki/index.php/Meep

=======    ==============
version    toolchain     
=======    ==============
``1.3``    ``foss/2016a``
=======    ==============


.. _list_software_MEGA_282:

*MEGA*
++++++

MEGA-CC (Molecular Evolutionary Genetics Analysis Computational Core) is an integrated suite of tools  for statistics-based comparative analysis of molecular sequence data based on evolutionary principles.

*homepage*: http://www.megasoftware.net/

============    ==========
version         toolchain 
============    ==========
``7.0.20-1``    ``system``
============    ==========


.. _list_software_MEGACC_416:

*MEGACC*
++++++++

MEGA-Computing Core - Sophisticated and user-friendly software suite for analyzing DNA and   protein sequence data from species and populations.

*homepage*: http://www.megasoftware.net

============    ==========
version         toolchain 
============    ==========
``7.0.18-1``    ``system``
============    ==========


.. _list_software_MEGAHIT_511:

*MEGAHIT*
+++++++++

An ultra-fast single-node solution for large and complex  metagenomics assembly via succinct de Bruijn graph

*homepage*: https://github.com/voutcn/megahit

=========    ==================    ==============================
version      versionsuffix         toolchain                     
=========    ==================    ==============================
``1.1.2``    ``-Python-2.7.14``    ``foss/2018a``                
``1.1.3``    ``-Python-2.7.14``    ``foss/2017b``, ``foss/2018a``
``1.1.3``    ``-Python-3.6.3``     ``foss/2017b``                
``1.1.4``    ``-Python-2.7.15``    ``foss/2018b``                
``1.1.4``    ``-Python-3.6.6``     ``foss/2018b``                
``1.2.8``                          ``GCCcore/8.2.0``             
=========    ==================    ==============================


.. _list_software_MEME_292:

*MEME*
++++++

The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or  GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using  MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate  motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment  using SpaMo or CentriMo.

*homepage*: http://meme-suite.org

=========    ==============================    ===============================
version      versionsuffix                     toolchain                      
=========    ==============================    ===============================
``5.0.4``    ``-Perl-5.26.0-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``5.0.4``    ``-Perl-5.26.0-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
=========    ==============================    ===============================


.. _list_software_memkind_741:

*memkind*
+++++++++

User Extensible Heap Manager built on top of jemalloc which enables control of memory characteristics  and a partitioning of the heap between kinds of memory. 

*homepage*: http://memkind.github.io

=========    =================
version      toolchain        
=========    =================
``1.5.0``    ``GCCcore/5.4.0``
=========    =================


.. _list_software_memory-profiler_1577:

*memory-profiler*
+++++++++++++++++

memory-profiler is a Python module for monitoring memory consumption of a process  as well as line-by-line analysis of memory consumption for python programs.

*homepage*: https://pypi.org/project/memory-profiler

==========    ===============================
version       toolchain                      
==========    ===============================
``0.55.0``    ``foss/2019a``, ``intel/2019a``
==========    ===============================


.. _list_software_meRanTK_658:

*meRanTK*
+++++++++

meRanTK is a versatile high performance toolkit for complete analysis of methylated RNA data.

*homepage*: http://www.icbi.at/index.html

==========    ==========
version       toolchain 
==========    ==========
``1.1.1b``    ``system``
==========    ==========


.. _list_software_MERCKX_458:

*MERCKX*
++++++++

Multilingual Entity/Resource Combiner & Knowledge eXtractor

*homepage*: https://github.com/madewild/MERCKX

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20170330``    ``-Python-2.7.13``    ``intel/2017a``
============    ==================    ===============


.. _list_software_Mercurial_932:

*Mercurial*
+++++++++++

Mercurial is a free, distributed source control management tool. It efficiently handles projects of any size and offers an easy and intuitive interface. 

*homepage*: http://mercurial.selenic.com/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``3.8.3``    ``-Python-2.7.11``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_Mesa_390:

*Mesa*
++++++

Mesa is an open-source implementation of the OpenGL specification -  a system for rendering interactive 3D graphics.

*homepage*: https://www.mesa3d.org/

==========    ====================================================================
version       toolchain                                                           
==========    ====================================================================
``11.1.2``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                   
``11.2.1``    ``foss/2016a``, ``intel/2016a``                                     
``12.0.2``    ``foss/2016b``, ``intel/2016b``                                     
``17.0.2``    ``foss/2017a``, ``intel/2017a``                                     
``17.2.4``    ``intel/2017b``, ``intelcuda/2017b``                                
``17.2.5``    ``foss/2017b``, ``fosscuda/2017b``                                  
``17.3.6``    ``foss/2018a``, ``fosscuda/2018a``, ``intel/2018a``, ``iomkl/2018a``
``18.1.1``    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                 
``19.0.1``    ``GCCcore/8.2.0``                                                   
``19.1.7``    ``GCCcore/8.3.0``                                                   
``19.2.1``    ``GCCcore/8.3.0``                                                   
==========    ====================================================================


.. _list_software_meshio_645:

*meshio*
++++++++

meshio is a tool for reading/writing various mesh formats representing unstructured meshes

*homepage*: https://github.com/nschloe/meshio

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.7.1``    ``-Python-2.7.12``    ``intel/2016b``
``2.0.2``    ``-Python-2.7.14``    ``intel/2018a``
``2.0.2``    ``-Python-3.6.4``     ``intel/2018a``
=========    ==================    ===============


.. _list_software_Meson_514:

*Meson*
+++++++

Meson is a cross-platform build system designed to be both as fast and as user friendly as possible.

*homepage*: https://mesonbuild.com

==========    =================    ==================================
version       versionsuffix        toolchain                         
==========    =================    ==================================
``0.43.0``    ``-Python-3.6.3``    ``intel/2017b``                   
``0.46.1``    ``-Python-3.6.4``    ``foss/2018a``                    
``0.48.1``    ``-Python-3.6.4``    ``foss/2018a``, ``intel/2018a``   
``0.48.1``    ``-Python-3.6.6``    ``foss/2018b``, ``fosscuda/2018b``
``0.50.0``    ``-Python-3.7.2``    ``GCCcore/8.2.0``                 
``0.51.2``    ``-Python-3.7.4``    ``GCCcore/8.3.0``                 
==========    =================    ==================================


.. _list_software_Mesquite_845:

*Mesquite*
++++++++++

Mesh-Quality Improvement Library

*homepage*: https://software.sandia.gov/mesquite/

=========    =====================================================================
version      toolchain                                                            
=========    =====================================================================
``2.3.0``    ``GCCcore/6.4.0``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017a``
=========    =====================================================================


.. _list_software_MetaBAT_606:

*MetaBAT*
+++++++++

An efficient tool for accurately reconstructing single genomes from complex microbial communities

*homepage*: https://bitbucket.org/berkeleylab/metabat

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``2.12.1``    ``-Python-2.7.15``    ``foss/2018b`` 
``2.14``                            ``gompi/2019a``
==========    ==================    ===============


.. _list_software_MetaboAnalystR_1414:

*MetaboAnalystR*
++++++++++++++++

MetaboAnalystR contains the R functions and libraries underlying the popular MetaboAnalyst web  server, including > 500 functions for metabolomic data analysis, visualization, and functional interpretation.

*homepage*: https://www.metaboanalyst.ca

==================    =============    ==============
version               versionsuffix    toolchain     
==================    =============    ==============
``2.0.1-20190827``    ``-R-3.6.0``     ``foss/2019a``
==================    =============    ==============


.. _list_software_MetaGeneAnnotator_1724:

*MetaGeneAnnotator*
+++++++++++++++++++

MetaGeneAnnotator is a gene-finding program for prokaryote and phage.

*homepage*: http://metagene.nig.ac.jp/

============    =============    ==========
version         versionsuffix    toolchain 
============    =============    ==========
``20080819``    ``-x86-64``      ``system``
============    =============    ==========


.. _list_software_Metal_499:

*Metal*
+++++++

Metal - Meta Analysis Helper. The METAL software is designed to facilitate meta-analysis of large  datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner.

*homepage*: http://csg.sph.umich.edu/abecasis/Metal/

==============    ==============
version           toolchain     
==============    ==============
``2011-03-25``    ``foss/2016a``
==============    ==============


.. _list_software_MetaPhlAn2_908:

*MetaPhlAn2*
++++++++++++

MetaPhlAn is a computational tool for profiling the composition of microbial  communities (Bacteria, Archaea, Eukaryotes and Viruses) from metagenomic shotgun sequencing  data (i.e. not 16S) with species-level. With the newly added StrainPhlAn module, it is now  possible to perform accurate strain-level microbial profiling.

*homepage*: https://bitbucket.org/biobakery/metaphlan2/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.7.8``    ``-Python-2.7.12``    ``foss/2016b``
``2.7.8``    ``-Python-3.6.6``     ``foss/2018b``
=========    ==================    ==============


.. _list_software_metaWRAP_737:

*metaWRAP*
++++++++++

MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of  metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling,  extracting draft genomes (binning), and functional annotation.

*homepage*: https://github.com/bxlab/metaWRAP

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.2``      ``-Python-2.7.15``    ``foss/2018b``
``1.2.2``    ``-Python-2.7.15``    ``foss/2019a``
=========    ==================    ==============


.. _list_software_Metaxa2_658:

*Metaxa2*
+++++++++

Metaxa2 -- Identifies Small Subunit (SSU) rRNAs and classifies them taxonomically

*homepage*: https://microbiology.se/software/metaxa2/

=======    ===============
version    toolchain      
=======    ===============
``2.2``    ``gompi/2019a``
=======    ===============


.. _list_software_MethylDackel_1207:

*MethylDackel*
++++++++++++++

A (mostly) universal methylation extractor for BS-seq experiments.

*homepage*: https://github.com/dpryan79/MethylDackel

=========    ========================================
version      toolchain                               
=========    ========================================
``0.4.0``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
=========    ========================================


.. _list_software_methylpy_892:

*methylpy*
++++++++++

Bisulfite sequencing data processing and differential methylation analysis.

*homepage*: https://pypi.python.org/pypi/methylpy

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.1.9``    ``-Python-2.7.13``    ``foss/2017a``
=========    ==================    ==============


.. _list_software_METIS_386:

*METIS*
+++++++

METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes.

*homepage*: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview

=========    =============    ===============================================================================================================================================================================================================
version      versionsuffix    toolchain                                                                                                                                                                                                      
=========    =============    ===============================================================================================================================================================================================================
``5.0.2``                     ``gimkl/2.11.5``                                                                                                                                                                                               
``5.1.0``                     ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``5.1.0``    ``-32bitIDX``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                                                                                                                                                              
=========    =============    ===============================================================================================================================================================================================================


.. _list_software_MiGEC_389:

*MiGEC*
+++++++

MIGEC is a software pipeline that facilitates processing and analysis of   immune repertoire sequencing data generated using molecular barcoding technique

*homepage*: https://milaboratory.com/software/migec/

=========    ===================    ==========
version      versionsuffix          toolchain 
=========    ===================    ==========
``1.2.8``    ``-Java-1.8.0_162``    ``system``
=========    ===================    ==========


.. _list_software_MIGRATE-N_644:

*MIGRATE-N*
+++++++++++

Migrate estimates population parameters, effective population sizes and migration rates of n populations, using genetic data.  It  uses a coalescent theory approach taking into account history of  mutations and uncertainty of the genealogy.  

*homepage*: http://popgen.sc.fsu.edu/Migrate/Migrate-n.html

==========    ==============
version       toolchain     
==========    ==============
``4.2.8``     ``foss/2016a``
``4.2.14``    ``foss/2018a``
==========    ==============


.. _list_software_MINC_295:

*MINC*
++++++

Medical Image NetCDF or MINC isn't netCDF.

*homepage*: https://github.com/BIC-MNI/libminc

==========    ===============================================
version       toolchain                                      
==========    ===============================================
``2.4.03``    ``foss/2017b``, ``foss/2018a``, ``intel/2017b``
==========    ===============================================


.. _list_software_Miniconda2_964:

*Miniconda2*
++++++++++++

Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.

*homepage*: https://docs.conda.io/en/latest/miniconda.html

==========    ==========
version       toolchain 
==========    ==========
``4.3.21``    ``system``
``4.6.14``    ``system``
``4.7.10``    ``system``
==========    ==========


.. _list_software_Miniconda3_965:

*Miniconda3*
++++++++++++

Miniconda is a free minimal installer for conda. It is a small,  bootstrap version of Anaconda that includes only conda, Python, the packages they  depend on, and a small number of other useful packages.

*homepage*: https://docs.conda.io/en/latest/miniconda.html

==========    ==========
version       toolchain 
==========    ==========
``4.4.10``    ``system``
``4.5.12``    ``system``
``4.6.14``    ``system``
``4.7.10``    ``system``
==========    ==========


.. _list_software_minieigen_949:

*minieigen*
+++++++++++

A small wrapper for core parts of EIgen, c++ library for linear algebra.

*homepage*: https://launchpad.net/minieigen/

=========    ===============================    ===============================
version      versionsuffix                      toolchain                      
=========    ===============================    ===============================
``0.5.3``    ``-Python-2.7.11``                 ``foss/2016a``, ``intel/2016a``
``0.5.3``    ``-Python-2.7.12``                 ``foss/2016b``, ``intel/2016b``
``0.5.3``    ``-Python-2.7.12-Boost-1.63.0``    ``intel/2016b``                
``0.5.4``    ``-Python-2.7.14``                 ``intel/2018a``                
=========    ===============================    ===============================


.. _list_software_Minimac4_754:

*Minimac4*
++++++++++

Minimac4 is a latest version in the series of genotype imputation software  - preceded by Minimac3 (2015), Minimac2 (2014), minimac (2012) and MaCH (2010).  Minimac4 is a lower memory and more computationally efficient implementation  of the original algorithms with comparable imputation quality.

*homepage*: https://genome.sph.umich.edu/wiki/Minimac4

=========    ==============
version      toolchain     
=========    ==============
``1.0.0``    ``foss/2018a``
=========    ==============


.. _list_software_minimap2_797:

*minimap2*
++++++++++

Minimap2 is a fast sequence mapping and alignment program that can find overlaps between long noisy reads, or map long reads or their assemblies to a reference genome optionally with detailed alignment (i.e. CIGAR). At present, it works efficiently with query sequences from a few kilobases to ~100 megabases in length at an error rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited test data sets, minimap2 is over 20 times faster than most other long-read aligners. It will replace BWA-MEM for long reads and contig alignment.

*homepage*: https://github.com/lh3/minimap2

==========    ====================
version       toolchain           
==========    ====================
``2.0rc1``    ``foss/2016b``      
``2.10``      ``foss/2018a``      
``2.11``      ``intel/2018a``     
``2.12``      ``foss/2018a``      
``2.13``      ``foss/2018b``      
``2.17``      ``GCC/8.2.0-2.31.1``
==========    ====================


.. _list_software_MinPath_689:

*MinPath*
+++++++++

MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions  using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological  pathways for a query dataset.

*homepage*: http://omics.informatics.indiana.edu/MinPath

=======    ==================    ===============================
version    versionsuffix         toolchain                      
=======    ==================    ===============================
``1.4``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
=======    ==================    ===============================


.. _list_software_MIRA_297:

*MIRA*
++++++

MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),  IonTorrent data and PacBio (the latter at the moment only CCS and error-corrected CLR reads).

*homepage*: https://sourceforge.net/projects/mira-assembler/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``4.0.2``                          ``intel/2017b``
``4.0.2``    ``-Python-2.7.11``    ``foss/2016a`` 
``4.9.6``                          ``intel/2017b``
=========    ==================    ===============


.. _list_software_miRDeep2_728:

*miRDeep2*
++++++++++

miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs

*homepage*: https://www.mdc-berlin.de/8551903/en/

===========    =================    ===============
version        versionsuffix        toolchain      
===========    =================    ===============
``0.1.1``      ``-Python-3.6.6``    ``foss/2018b`` 
``2.0.0.8``                         ``intel/2016b``
===========    =================    ===============


.. _list_software_MiXCR_419:

*MiXCR*
+++++++

MiXCR processes big immunome data from raw sequences  to quantitated clonotypes 

*homepage*: https://milaboratory.com/software/mixcr/

=========    ===================    ==========
version      versionsuffix          toolchain 
=========    ===================    ==========
``2.1.9``    ``-Java-1.8.0_162``    ``system``
``3.0.3``    ``-Java-1.8``          ``system``
=========    ===================    ==========


.. _list_software_mkl-dnn_689:

*mkl-dnn*
+++++++++

Intel(R) Math Kernel Library for Deep Neural Networks (Intel(R) MKL-DNN)

*homepage*: https://01.org/mkl-dnn

==========    ===============================
version       toolchain                      
==========    ===============================
``0.11``      ``intel/2017b``                
``0.13``      ``intel/2018a``                
``0.16``      ``foss/2018b``, ``intel/2018b``
``0.17.2``    ``foss/2018a``                 
==========    ===============================


.. _list_software_mkl-service_1122:

*mkl-service*
+++++++++++++

Python hooks for Intel(R) Math Kernel Library runtime control settings.

*homepage*: https://github.com/IntelPython/mkl-service

=========    ===============
version      toolchain      
=========    ===============
``2.0.2``    ``intel/2019a``
=========    ===============


.. _list_software_MLC_220:

*MLC*
+++++

Intel Memory Latency Checker (Intel MLC) is a tool used to measure memory latencies and b/w,  and how they change with increasing load on the system.

*homepage*: https://software.intel.com/en-us/articles/intelr-memory-latency-checker

=======    ==========
version    toolchain 
=======    ==========
``3.0``    ``system``
=======    ==========


.. _list_software_Mmg_289:

*Mmg*
+++++

Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries:     the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional triangulation         and generation of a triangulation from a set of points or from given boundary edges     the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation         and isovalue discretization     the mmg3d application and the libmmg3d library: adaptation and optimization of a tetrahedral mesh         and implicit domain meshing     the libmmg library gathering the libmmg2d, libmmgs and libmmg3d libraries 

*homepage*: https://www.mmgtools.org/

=========    ==============
version      toolchain     
=========    ==============
``5.3.9``    ``foss/2017b``
=========    ==============


.. _list_software_MMSEQ_387:

*MMSEQ*
+++++++

The MMSEQ package contains a collection of statistical tools  for analysing RNA-seq expression data. 

*homepage*: https://github.com/eturro/mmseq

=========    ===================    ==========
version      versionsuffix          toolchain 
=========    ===================    ==========
``1.0.8``    ``-linux64-static``    ``system``
=========    ===================    ==========


.. _list_software_MMseqs2_648:

*MMseqs2*
+++++++++

MMseqs2: ultra fast and sensitive search and clustering suite

*homepage*: https://mmseqs.com

===========    ===============
version        toolchain      
===========    ===============
``1-c7a89``    ``foss/2016b`` 
``5-9375b``    ``intel/2018a``
``8-fac81``    ``intel/2018b``
===========    ===============


.. _list_software_modred_635:

*modred*
++++++++

Compute modal decompositions and reduced-order models, easily, efficiently, and in parallel.

*homepage*: https://pypi.python.org/pypi/modred/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``2.0.2``    ``-Python-3.5.2``    ``foss/2016b``
=========    =================    ==============


.. _list_software_Molcas_607:

*Molcas*
++++++++

Molcas is an ab initio quantum chemistry software package developed by scientists to be used by scientists. The basic philosophy is is to be able to treat general electronic structures for molecules consisting of atoms from most of the periodic table. As such, the primary focus of the package is on multiconfigurational methods with applications typically connected to the treatment of highly degenerate states.

*homepage*: http://www.molcas.org

================    ======================    ==========
version             versionsuffix             toolchain 
================    ======================    ==========
``8.0-15.06.18``    ``_CentOS_6.6_x86_64``    ``system``
``8.2``             ``-centos-mkl-par``       ``system``
``8.2``             ``-centos-par``           ``system``
================    ======================    ==========


.. _list_software_Molden_607:

*Molden*
++++++++

Molden is a package for displaying Molecular Density from the  Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical  packages Mopac/Ampac

*homepage*: https://www.cmbi.ru.nl/molden/

=======    =================
version    toolchain        
=======    =================
``5.6``    ``foss/2016a``   
``5.7``    ``intel/2016b``  
``5.8``    ``foss/2018a``   
``6.1``    ``GCCcore/8.2.0``
=======    =================


.. _list_software_Molekel_713:

*Molekel*
+++++++++

Molekel is an open-source multi-platform molecular visualization program.

*homepage*: http://molekel.cscs.ch/

=========    =================    ==========
version      versionsuffix        toolchain 
=========    =================    ==========
``5.4.0``    ``-Linux_x86_64``    ``system``
=========    =================    ==========


.. _list_software_molmod_648:

*molmod*
++++++++

MolMod is a Python library with many compoments that are useful to write molecular modeling programs.

*homepage*: http://molmod.github.io/molmod/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.1``      ``-Python-2.7.11``    ``intel/2016a``                
``1.1``      ``-Python-2.7.12``    ``intel/2016b``                
``1.1``      ``-Python-2.7.13``    ``intel/2017a``                
``1.4.3``    ``-Python-2.7.14``    ``intel/2017b``                
``1.4.3``    ``-Python-3.6.3``     ``intel/2017b``                
``1.4.4``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``
``1.4.4``    ``-Python-2.7.15``    ``intel/2018b``                
``1.4.4``    ``-Python-3.7.2``     ``intel/2019a``                
=========    ==================    ===============================


.. _list_software_Molpro_633:

*Molpro*
++++++++

Molpro is a complete system of ab initio programs for molecular electronic structure calculations.

*homepage*: https://www.molpro.net

=============    ========================    ==========
version          versionsuffix               toolchain 
=============    ========================    ==========
``2010.1.23``    ``.Linux_x86_64``           ``system``
``2015.1.0``     ``.linux_x86_64_intel``     ``system``
``2015.1.3``     ``.linux_x86_64_openmp``    ``system``
``2015.1.10``    ``.linux_x86_64_openmp``    ``system``
=============    ========================    ==========


.. _list_software_Mono_409:

*Mono*
++++++

An open source, cross-platform, implementation of C# and the CLR that is  binary compatible with Microsoft.NET.

*homepage*: https://www.mono-project.com/

==============    ===============================
version           toolchain                      
==============    ===============================
``2.10.6``        ``intel/2016b``                
``4.6.2.7``       ``intel/2016b``, ``system``    
``4.8.0.495``     ``intel/2017a``                
``5.4.1.6``       ``foss/2017b``, ``intel/2017b``
``5.10.0.160``    ``foss/2018a``                 
``5.18.1.0``      ``foss/2018a``                 
``6.4.0.198``     ``foss/2018b``                 
==============    ===============================


.. _list_software_mordecai_836:

*mordecai*
++++++++++

mordecai is a full text geoparsing as a Python library. Extract the place names from a piece of text, resolve them to the correct place, and return their coordinates and structured geographic information.

*homepage*: https://github.com/openeventdata/mordecai

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``2.0.1``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_mosdepth_868:

*mosdepth*
++++++++++

Fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing

*homepage*: https://github.com/brentp/mosdepth

=========    ===============
version      toolchain      
=========    ===============
``0.2.2``    ``intel/2018a``
``0.2.3``    ``intel/2018a``
``0.2.4``    ``foss/2018b`` 
=========    ===============


.. _list_software_Mothur_639:

*Mothur*
++++++++

Mothur is a single piece of open-source, expandable software  to fill the bioinformatics needs of the microbial ecology community.

*homepage*: http://www.mothur.org/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.39.5``    ``-Python-2.7.13``    ``intel/2017a``
``1.41.0``    ``-Python-2.7.15``    ``foss/2018b`` 
==========    ==================    ===============


.. _list_software_motif_543:

*motif*
+++++++

Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building  applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.  It was the standard toolkit for the Common Desktop Environment and thus for Unix.

*homepage*: http://motif.ics.com/

=========    ================================================================================
version      toolchain                                                                       
=========    ================================================================================
``2.3.5``    ``foss/2016a``, ``intel/2016a``                                                 
``2.3.7``    ``intel/2017a``                                                                 
``2.3.8``    ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``
=========    ================================================================================


.. _list_software_MotionCor2_972:

*MotionCor2*
++++++++++++

MotionCor2 correct anisotropic image motion at the single pixel level across the whole frame, suitable for both single particle and tomographic images. Iterative, patch-based motion detection is combined with spatial and temporal constraints and dose weighting.  Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache, Yifan Cheng and David A. Agard (2016) Anisotropic Correction of Beam-induced Motion for Improved Single-particle Electron Cryo-microscopy, Nature Methods, submitted. BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960 

*homepage*: https://msg.ucsf.edu/

=========    =================
version      toolchain        
=========    =================
``1.2.6``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_MoviePy_713:

*MoviePy*
+++++++++

MoviePy (full documentation) is a Python library for video editing: cutting, concatenations,  title insertions, video compositing (a.k.a. non-linear editing), video processing, and creation of custom effects.

*homepage*: https://zulko.github.io/moviepy/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.0.1``    ``-Python-3.7.2``    ``foss/2019a``
=========    =================    ==============


.. _list_software_MPC_224:

*MPC*
+++++

Gnu Mpc is a C library for the arithmetic of  complex numbers with arbitrarily high precision and correct  rounding of the result. It extends the principles of the IEEE-754  standard for fixed precision real floating point numbers to  complex numbers, providing well-defined semantics for every  operation. At the same time, speed of operation at high precision  is a major design goal.

*homepage*: http://www.multiprecision.org/

=========    ===============    ===============================
version      versionsuffix      toolchain                      
=========    ===============    ===============================
``1.0.3``                       ``intel/2017a``                
``1.0.3``    ``-MPFR-3.1.6``    ``foss/2017b``, ``intel/2017b``
=========    ===============    ===============================


.. _list_software_MPFR_309:

*MPFR*
++++++

The MPFR library is a C library for multiple-precision floating-point  computations with correct rounding. 

*homepage*: https://www.mpfr.org

=========    ================================================================
version      toolchain                                                       
=========    ================================================================
``2.4.2``    ``system``                                                      
``3.1.4``    ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``3.1.5``    ``GCCcore/6.4.0``, ``intel/2017a``                              
``3.1.6``    ``GCCcore/6.4.0``                                               
``4.0.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                            
``4.0.2``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                            
=========    ================================================================


.. _list_software_mpi4py_611:

*mpi4py*
++++++++

MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for  the Python programming language, allowing any Python program to exploit multiple processors.

*homepage*: https://bitbucket.org/mpi4py/mpi4py

=========    =================================    ================================
version      versionsuffix                        toolchain                       
=========    =================================    ================================
``1.3.1``    ``-Python-2.7.11-timed-pingpong``    ``intel/2016a``                 
``1.3.1``    ``-Python-2.7.12-timed-pingpong``    ``intel/2016b``                 
``2.0.0``    ``-Python-2.7.12``                   ``intel/2016b``                 
``2.0.0``    ``-Python-2.7.13-timed-pingpong``    ``intel/2017a``                 
``3.0.0``    ``-Python-2.7.14-timed-pingpong``    ``intel/2018a``                 
``3.0.0``    ``-Python-3.6.3``                    ``intel/2017b``                 
``3.0.1``    ``-Python-3.6.6``                    ``intel/2018b``                 
``3.0.2``    ``-timed-pingpong``                  ``gompi/2019a``, ``iimpi/2019a``
=========    =================================    ================================


.. _list_software_MPICH_369:

*MPICH*
+++++++

MPICH v3.x is an open source high-performance MPI 3.0 implementation. It does not support InfiniBand (use MVAPICH2 with InfiniBand devices).

*homepage*: http://www.mpich.org/

=========    ======================================
version      toolchain                             
=========    ======================================
``3.0.4``    ``GCC/4.8.1``                         
``3.2``      ``GCC/4.9.3-2.25``, ``GCC/7.2.0-2.29``
``3.2.1``    ``GCC/7.2.0-2.29``                    
=========    ======================================


.. _list_software_MPICH2_419:

*MPICH2*
++++++++

MPICH v3.x is an open source high-performance MPI 3.0 implementation. It does not support InfiniBand (use MVAPICH2 with InfiniBand devices).

*homepage*: http://www.mpich.org/

=======    =============
version    toolchain    
=======    =============
``1.1``    ``GCC/4.8.1``
=======    =============


.. _list_software_mpifileutils_1303:

*mpifileutils*
++++++++++++++

MPI-Based File Utilities For Distributed Systems 

*homepage*: https://hpc.github.io/mpifileutils/

=========    ================================
version      toolchain                       
=========    ================================
``0.9.1``    ``gompi/2019a``, ``iimpi/2019a``
=========    ================================


.. _list_software_mpiP_406:

*mpiP*
++++++

mpiP is a lightweight profiling library for MPI applications.  Because it  only collects statistical information about MPI functions, mpiP generates  considerably less overhead and much less data than tracing tools.  All the  information captured by mpiP is task-local. It only uses communication during  report generation, typically at the end of the experiment, to merge results  from all of the tasks into one output file. 

*homepage*: http://mpip.sourceforge.net/

=========    ===================================================
version      toolchain                                          
=========    ===================================================
``3.4.1``    ``gompi/2019a``, ``iimpi/2019a``, ``iompi/2019.01``
=========    ===================================================


.. _list_software_MPJ-Express_1022:

*MPJ-Express*
+++++++++++++

MPJ Express is an open source Java message passing library that allows application   developers to write and execute parallel applications for multicore processors and compute clusters/clouds.

*homepage*: http://mpj-express.org/

========    ==================    ==============
version     versionsuffix         toolchain     
========    ==================    ==============
``0.44``    ``-Java-1.8.0_92``    ``foss/2016a``
========    ==================    ==============


.. _list_software_mpmath_647:

*mpmath*
++++++++

mpmath can be used as an arbitrary-precision substitute for Python's float/complex  types and math/cmath modules, but also does much more advanced mathematics. Almost any calculation  can be performed just as well at 10-digit or 1000-digit precision, with either real or complex  numbers, and in many cases mpmath implements efficient algorithms that scale well for extremely  high precision work.

*homepage*: http://mpmath.org/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.19``     ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``1.0.0``    ``-Python-2.7.14``    ``intel/2018a``                
``1.0.0``    ``-Python-2.7.15``    ``foss/2018b``                 
=========    ==================    ===============================


.. _list_software_MrBayes_691:

*MrBayes*
+++++++++

MrBayes is a program for the Bayesian estimation of phylogeny.

*homepage*: http://mrbayes.csit.fsu.edu

=========    ==============
version      toolchain     
=========    ==============
``3.2.6``    ``foss/2016a``
=========    ==============


.. _list_software_MRIcron_666:

*MRIcron*
+++++++++

MRIcron allows viewing of medical images. It  includes tools to complement SPM and FSL. Native format is NIFTI  but includes a conversion program (see dcm2nii) for converting DICOM images.  Features layers, ROIs, and volume rendering.  

*homepage*: http://www.mccauslandcenter.sc.edu/mricro/mricron/

================    ==========
version             toolchain 
================    ==========
``1.0.20180614``    ``system``
``20150601``        ``system``
================    ==========


.. _list_software_MRtrix_614:

*MRtrix*
++++++++

MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner  robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines.

*homepage*: http://www.brain.org.au/software/index.html#mrtrix

===========    ==================    ===============
version        versionsuffix         toolchain      
===========    ==================    ===============
``0.3.14``     ``-Python-2.7.11``    ``intel/2016a``
``0.3.15``     ``-Python-2.7.12``    ``intel/2016b``
``3.0_RC2``    ``-Python-2.7.13``    ``foss/2017a`` 
``3.0_RC3``    ``-Python-2.7.14``    ``intel/2018a``
===========    ==================    ===============


.. _list_software_MSM_237:

*MSM*
+++++

Multimodal Surface Matching with Higher order Clique Reduction

*homepage*: https://github.com/ecr05/MSM_HOCR

=======    ===============================
version    toolchain                      
=======    ===============================
``1.0``    ``foss/2017b``, ``intel/2017b``
=======    ===============================


.. _list_software_MSPC_307:

*MSPC*
++++++

Using combined evidence from replicates to evaluate ChIP-seq peaks

*homepage*: https://genometric.github.io/MSPC/

=========    ==========
version      toolchain 
=========    ==========
``3.3.1``    ``system``
=========    ==========


.. _list_software_msprime_765:

*msprime*
+++++++++

msprime is a coalescent simulator and library for processing tree-based genetic data.

*homepage*: https://msprime.readthedocs.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.7.0``    ``-Python-3.7.2``    ``intel/2019a``
=========    =================    ===============


.. _list_software_MTL4_289:

*MTL4*
++++++

The Matrix Template Library 4 incorporates the most modern programming techniques  to provide an easy and intuitive interface to users while enabling optimal performance. The natural  mathematical notation in MTL4 empowers all engineers and scientists to implement their algorithms and  models in minimal time. All technical aspects are encapsulated in the library.

*homepage*: http://www.simunova.com/mtl4

============    ==========
version         toolchain 
============    ==========
``4.0.8878``    ``system``
``4.0.9555``    ``system``
============    ==========


.. _list_software_MultiNest_933:

*MultiNest*
+++++++++++

MultiNest is a Bayesian inference tool which calculates the evidence and explores the parameter space  which may contain multiple posterior modes and pronounced (curving) degeneracies in moderately high dimensions.

*homepage*: https://ccpforge.cse.rl.ac.uk/gf/project/multinest/

========    ===============
version     toolchain      
========    ===============
``3.10``    ``intel/2016a``
========    ===============


.. _list_software_MultiQC_671:

*MultiQC*
+++++++++

Aggregate results from bioinformatics analyses across many samples into a single   report.   MultiQC searches a given directory for analysis logs and compiles a HTML report. It's a general  use tool, perfect for summarising the output from numerous bioinformatics tools.

*homepage*: http://multiqc.info

=======    ==================    ===============================
version    versionsuffix         toolchain                      
=======    ==================    ===============================
``0.9``    ``-Python-2.7.12``    ``foss/2016b``                 
``1.2``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``1.2``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``1.6``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``1.6``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``1.7``    ``-Python-2.7.15``    ``intel/2018b``                
``1.7``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
=======    ==================    ===============================


.. _list_software_Multiwfn_854:

*Multiwfn*
++++++++++

Multiwfn is an extremely powerful program for realizingi  electronic wavefunction analysis, which is a key ingredient of quantum  chemistry. Multiwfn is free, open-source, high-efficient, very user-friendly  and flexible, it supports almost all of the most important wavefunction  analysis methods.

*homepage*: http://sobereva.com/multiwfn/

=========    ===============
version      toolchain      
=========    ===============
``3.4.1``    ``intel/2017b``
``3.6``      ``intel/2019a``
=========    ===============


.. _list_software_MUMmer_563:

*MUMmer*
++++++++

MUMmer is a system for rapidly aligning entire genomes,  whether in complete or draft form. AMOS makes use of it.

*homepage*: http://mummer.sourceforge.net/

==============    ==============
version           toolchain     
==============    ==============
``3.23``          ``foss/2016b``
``4.0.0beta2``    ``foss/2018b``
==============    ==============


.. _list_software_MUMPS_402:

*MUMPS*
+++++++

A parallel sparse direct solver

*homepage*: http://graal.ens-lyon.fr/MUMPS/

=========    ==============    ===============================
version      versionsuffix     toolchain                      
=========    ==============    ===============================
``5.1.2``    ``-metis``        ``foss/2017b``                 
``5.2.1``    ``-metis``        ``foss/2019a``, ``intel/2019a``
``5.2.1``    ``-metis-seq``    ``foss/2019a``, ``intel/2019a``
=========    ==============    ===============================


.. _list_software_muParser_847:

*muParser*
++++++++++

muParser is an extensible high performance math expression parser library  written in C++. It works by transforming a mathematical expression into  bytecode and precalculating constant parts of the expression. 

*homepage*: http://beltoforion.de/article.php?a=muparser

=========    =================
version      toolchain        
=========    =================
``2.2.5``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_MuPeXI_536:

*MuPeXI*
++++++++

MuPeXI: Mutant Peptide eXtractor and Informer. Given a list of somatic mutations (VCF file) as input,  MuPeXI returns a table containing all mutated peptides (neo-peptides) of user-defined lengths,  along with several pieces of information relevant for identifying  which of these neo-peptides are likely to serve as neo-epitopes.

*homepage*: https://github.com/ambj/MuPeXI

=========    ==============================    ==============
version      versionsuffix                     toolchain     
=========    ==============================    ==============
``1.2.0``    ``-Perl-5.28.0-Python-2.7.15``    ``foss/2018b``
=========    ==============================    ==============


.. _list_software_MUSCLE_457:

*MUSCLE*
++++++++

MUSCLE is one of the best-performing multiple alignment programs   according to published benchmark tests, with accuracy and speed that are consistently   better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users   learn everything they need to know about MUSCLE in a few minutes-only a handful of  command-line options are needed to perform common alignment tasks.

*homepage*: http://drive5.com/muscle/

============    ===============    ==================================================================================================================
version         versionsuffix      toolchain                                                                                                         
============    ===============    ==================================================================================================================
``3.8.31``                         ``foss/2016a``, ``foss/2017b``, ``foss/2018a``, ``intel/2016a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``3.8.31``      ``-i86linux64``    ``system``                                                                                                        
``3.8.1551``                       ``GCC/8.2.0-2.31.1``                                                                                              
============    ===============    ==================================================================================================================


.. _list_software_MuSiC_449:

*MuSiC*
+++++++

Multi-subject Single Cell deconvolution (MuSiC) is a deconvolution method that utilizes  cross-subject scRNA-seq to estimate cell type proportions in bulk RNA-seq data.

*homepage*: https://github.com/xuranw/MuSiC

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``1.6.2``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_MuTect_594:

*MuTect*
++++++++

MuTect is a method developed at the Broad Institute for the reliable  and accurate identification of somatic point mutations in next generation sequencing  data of cancer genomes. 

*homepage*: http://www.broadinstitute.org/cancer/cga/mutect

=========    ==================    ==========
version      versionsuffix         toolchain 
=========    ==================    ==========
``1.1.4``    ``-Java-1.7.0_76``    ``system``
``1.1.4``    ``-Java-1.7.0_80``    ``system``
``1.1.7``    ``-Java-1.7.0_80``    ``system``
=========    ==================    ==========


.. _list_software_mutil_555:

*mutil*
+++++++

Mutil is a set of standard utilities that have been parallelized to maximize performance on  modern file systems. These currently include multi-threaded drop-in replacements for cp and md5sum from GNU coreutils, which have achieved 10/30x rates on one/many nodes. 

*homepage*: http://people.nas.nasa.gov/~kolano/projects/mutil.html

===========    ===============
version        toolchain      
===========    ===============
``1.822.3``    ``intel/2016a``
===========    ===============


.. _list_software_MVAPICH2_570:

*MVAPICH2*
++++++++++

This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH.

*homepage*: http://mvapich.cse.ohio-state.edu/overview/mvapich2/

=========    ==================
version      toolchain         
=========    ==================
``2.0.1``    ``GCC/4.8.4``     
``2.1``      ``GCC/4.9.3-2.25``
``2.2b``     ``GCC/4.9.3-2.25``
=========    ==================


.. _list_software_MXNet_460:

*MXNet*
+++++++

Flexible and Efficient Library for Deep Learning

*homepage*: http://mxnet.io/

=========    ==========================    ==============
version      versionsuffix                 toolchain     
=========    ==========================    ==============
``0.9.3``    ``-Python-2.7.12-R-3.3.3``    ``foss/2016b``
=========    ==========================    ==============


.. _list_software_mygene_645:

*mygene*
++++++++

Python Client for MyGene.Info services.

*homepage*: https://github.com/biothings/mygene.py

=========    ===============
version      toolchain      
=========    ===============
``3.1.0``    ``intel/2019a``
=========    ===============


.. _list_software_MyMediaLite_1076:

*MyMediaLite*
+++++++++++++

MyMediaLite is a lightweight, multi-purpose library of recommender system algorithms.

*homepage*: http://www.ismll.uni-hildesheim.de/mymedialite/

========    ===============
version     toolchain      
========    ===============
``3.10``    ``intel/2016b``
``3.11``    ``intel/2016b``
``3.12``    ``intel/2017a``
========    ===============


.. _list_software_mympingpong_1205:

*mympingpong*
+++++++++++++

A mpi4py based random pair pingpong network stress test.

*homepage*: https://github.com/hpcugent/mympingpong

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.7.0``    ``-Python-2.7.11``    ``intel/2016a``                
``0.7.1``    ``-Python-2.7.12``    ``intel/2016b``                
``0.8.0``    ``-Python-2.7.13``    ``intel/2017a``                
``0.8.0``    ``-Python-2.7.14``    ``intel/2018a``                
``0.8.0``    ``-Python-2.7.15``    ``foss/2019a``, ``intel/2019a``
=========    ==================    ===============================


.. _list_software_mypy_463:

*mypy*
++++++

Optional static typing for Python

*homepage*: http://www.mypy-lang.org/

=========    ===============
version      toolchain      
=========    ===============
``0.4.5``    ``intel/2016b``
=========    ===============


.. _list_software_MySQL_438:

*MySQL*
+++++++

MySQL is one of the world's most widely used open-source  relational database management system (RDBMS).

*homepage*: http://www.mysql.com/

==========    ===============    ==================
version       versionsuffix      toolchain         
==========    ===============    ==================
``5.6.26``    ``-clientonly``    ``GNU/4.9.3-2.25``
``5.7.21``    ``-clientonly``    ``GCCcore/6.4.0`` 
==========    ===============    ==================


.. _list_software_MySQL-python_1157:

*MySQL-python*
++++++++++++++

MySQL database connector for Python

*homepage*: https://github.com/farcepest/MySQLdb1

=========    ==================================    ===============
version      versionsuffix                         toolchain      
=========    ==================================    ===============
``1.2.5``    ``-Python-2.7.11``                    ``intel/2016a``
``1.2.5``    ``-Python-2.7.11-MariaDB-10.1.14``    ``intel/2016a``
=========    ==================================    ===============


.. _list_software_mysqlclient_1205:

*mysqlclient*
+++++++++++++

Python interface to MySQL

*homepage*: https://github.com/PyMySQL/mysqlclient-python

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.3.7``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
=========    ==================    ===============================


.. _list_software_letter_n:

*N*
---


:ref:`list_software_NAG_214` - :ref:`list_software_NAGfor_541` - :ref:`list_software_NAMD_288` - :ref:`list_software_nanofilt_859` - :ref:`list_software_nanonet_755` - :ref:`list_software_nanopolish_1083` - :ref:`list_software_NASM_303` - :ref:`list_software_NBO_223` - :ref:`list_software_NCBI-Toolkit_1071` - :ref:`list_software_ncbi-vdb_773` - :ref:`list_software_NCCL_288` - :ref:`list_software_ncdf4_463` - :ref:`list_software_ncdu_426` - :ref:`list_software_NCL_221` - :ref:`list_software_NCO_224` - :ref:`list_software_ncompress_986` - :ref:`list_software_ncurses_771` - :ref:`list_software_ncview_652` - :ref:`list_software_nd2reader_887` - :ref:`list_software_ne_211` - :ref:`list_software_NEdit_468` - :ref:`list_software_Net-core_765` - :ref:`list_software_netCDF_532` - :ref:`list_software_netCDF-C++_730` - :ref:`list_software_netCDF-C++4_782` - :ref:`list_software_netCDF-Fortran_1309` - :ref:`list_software_netcdf4-python_1399` - :ref:`list_software_netloc_645` - :ref:`list_software_NetLogo_696` - :ref:`list_software_netMHC_543` - :ref:`list_software_netMHCIIpan_1008` - :ref:`list_software_netMHCpan_862` - :ref:`list_software_NetPIPE_597` - :ref:`list_software_nettle_652` - :ref:`list_software_networkx_898` - :ref:`list_software_NeuroKit_817` - :ref:`list_software_NEURON_471` - :ref:`list_software_Nextflow_855` - :ref:`list_software_NextGenMap_983` - :ref:`list_software_NFFT_302` - :ref:`list_software_nglview_764` - :ref:`list_software_NGS_232` - :ref:`list_software_NGS-Python_919` - :ref:`list_software_NGSadmix_763` - :ref:`list_software_NiBabel_653` - :ref:`list_software_NIfTI_410` - :ref:`list_software_Nilearn_713` - :ref:`list_software_Nim_292` - :ref:`list_software_NIMBLE_439` - :ref:`list_software_Ninja_496` - :ref:`list_software_Nipype_629` - :ref:`list_software_NLopt_493` - :ref:`list_software_NLTK_313` - :ref:`list_software_Node-RED_654` - :ref:`list_software_nodejs_643` - :ref:`list_software_Normaliz_844` - :ref:`list_software_nose-parameterized_1871` - :ref:`list_software_novoalign_973` - :ref:`list_software_NOVOPlasty_959` - :ref:`list_software_npstat_666` - :ref:`list_software_NRGLjubljana_1146` - :ref:`list_software_NSPR_323` - :ref:`list_software_NSS_244` - :ref:`list_software_numactl_756` - :ref:`list_software_numba_531` - :ref:`list_software_numexpr_783` - :ref:`list_software_numpy_569` - :ref:`list_software_nvtop_567` - :ref:`list_software_NWChem_546` - :ref:`list_software_NxTrim_610`


.. _list_software_NAG_214:

*NAG*
+++++

The worlds largest collection of robust, documented, tested and maintained numerical algorithms.

*homepage*: http://www.nag.co.uk

=======    ==================================
version    toolchain                         
=======    ==================================
``24``     ``PGI/17.4-GCC-6.4.0-2.28``       
``26``     ``GCCcore/6.4.0``, ``intel/2018a``
=======    ==================================


.. _list_software_NAGfor_541:

*NAGfor*
++++++++

The checking compiler for improved code portability and detailed error reporting.

*homepage*: http://www.nag.co.uk

==========    ==========
version       toolchain 
==========    ==========
``6.2.14``    ``system``
==========    ==========


.. _list_software_NAMD_288:

*NAMD*
++++++

NAMD is a parallel molecular dynamics code designed for high-performance simulation of  large biomolecular systems.

*homepage*: http://www.ks.uiuc.edu/Research/namd/

========    =============    ================================================================
version     versionsuffix    toolchain                                                       
========    =============    ================================================================
``2.11``    ``-mpi``         ``intel/2016a``                                                 
``2.12``    ``-mpi``         ``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.13``                     ``fosscuda/2018b``                                              
``2.13``    ``-mpi``         ``foss/2018b``, ``intel/2018b``                                 
========    =============    ================================================================


.. _list_software_nanofilt_859:

*nanofilt*
++++++++++

Filtering and trimming of long read sequencing data.

*homepage*: https://github.com/wdecoster/nanofilt

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``2.5.0``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_nanonet_755:

*nanonet*
+++++++++

Nanonet provides recurrent neural network basecalling for Oxford Nanopore MinION data.

*homepage*: https://github.com/nanoporetech/nanonet

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.0.0``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_nanopolish_1083:

*nanopolish*
++++++++++++

Software package for signal-level analysis of Oxford Nanopore sequencing data.

*homepage*: https://github.com/jts/nanopolish

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``0.9.2``                           ``intel/2018a``                
``0.10.2``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
==========    ==================    ===============================


.. _list_software_NASM_303:

*NASM*
++++++

NASM: General-purpose x86 assembler

*homepage*: https://www.nasm.us/

===========    ====================================================================================
version        toolchain                                                                           
===========    ====================================================================================
``2.11.08``    ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``
``2.12.01``    ``foss/2016a``, ``intel/2016a``                                                     
``2.12.02``    ``foss/2016a``, ``foss/2016b``, ``intel/2016b``, ``intel/2017a``, ``system``        
``2.13.01``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``                                                
``2.13.03``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                                                
``2.14.02``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                
===========    ====================================================================================


.. _list_software_NBO_223:

*NBO*
+++++

The Natural Bond Orbital (NBO) program is a discovery tool for chemical insights from complex wavefunctions. 

*homepage*: http://nbo.chem.wisc.edu/

=======    ===============
version    toolchain      
=======    ===============
``1.1``    ``intel/2016a``
``7.0``    ``intel/2017b``
=======    ===============


.. _list_software_NCBI-Toolkit_1071:

*NCBI-Toolkit*
++++++++++++++

The NCBI Toolkit is a collection of utilities developed for the   production and distribution of GenBank, Entrez, BLAST, and related services   by the National Center for Biotechnology Information.

*homepage*: http://ncbi.github.io/cxx-toolkit/

==========    ===============
version       toolchain      
==========    ===============
``18.0.0``    ``intel/2017a``
==========    ===============


.. _list_software_ncbi-vdb_773:

*ncbi-vdb*
++++++++++

The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for  using data in the INSDC Sequence Read Archives.

*homepage*: https://github.com/ncbi/ncbi-vdb

===========    ================================================
version        toolchain                                       
===========    ================================================
``2.5.8-1``    ``foss/2016b``, ``intel/2016a``                 
``2.7.0``      ``foss/2016b``                                  
``2.8.2``      ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.9.1-1``    ``foss/2018a``, ``intel/2018a``                 
``2.9.3``      ``foss/2018b``                                  
===========    ================================================


.. _list_software_NCCL_288:

*NCCL*
++++++

The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective communication primitives that are performance optimized for NVIDIA GPUs.

*homepage*: https://developer.nvidia.com/nccl

==========    ===================    ==================
version       versionsuffix          toolchain         
==========    ===================    ==================
``2.1.4``     ``-CUDA-9.0.176``      ``system``        
``2.2.13``    ``-CUDA-9.2.148.1``    ``system``        
``2.3.7``                            ``fosscuda/2018b``
``2.4.2``                            ``gcccuda/2019a`` 
``2.4.8``                            ``gcccuda/2019b`` 
==========    ===================    ==================


.. _list_software_ncdf4_463:

*ncdf4*
+++++++

ncdf4: Interface to Unidata netCDF (version 4 or earlier) format data files

*homepage*: https://cran.r-project.org/web/packages/ncdf4

==========    =============    ===============================
version       versionsuffix    toolchain                      
==========    =============    ===============================
``1.16``      ``-R-3.4.0``     ``intel/2017a``                
``1.16``      ``-R-3.4.3``     ``intel/2017b``                
``1.16``      ``-R-3.4.4``     ``intel/2018a``                
``1.16.1``    ``-R-3.5.1``     ``foss/2018b``, ``intel/2018b``
``1.16.1``    ``-R-3.6.0``     ``foss/2019a``                 
==========    =============    ===============================


.. _list_software_ncdu_426:

*ncdu*
++++++

Ncdu is a disk usage analyzer with an ncurses interface. It is designed to find space hogs on a  remote server where you don't have an entire graphical setup available, but it is a useful tool even on regular  desktop systems. Ncdu aims to be fast, simple and easy to use, and should be able to run in any minimal POSIX-like  environment with ncurses installed.

*homepage*: https://dev.yorhel.nl/ncdu

========    =================
version     toolchain        
========    =================
``1.13``    ``GCCcore/7.3.0``
``1.14``    ``GCCcore/7.3.0``
========    =================


.. _list_software_NCL_221:

*NCL*
+++++

NCL is an interpreted language designed specifically for scientific data analysis and visualization.

*homepage*: http://www.ncl.ucar.edu

=========    =================================================
version      toolchain                                        
=========    =================================================
``6.4.0``    ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``6.5.0``    ``intel/2018a``                                  
``6.6.2``    ``intel/2018b``                                  
=========    =================================================


.. _list_software_NCO_224:

*NCO*
+++++

manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5

*homepage*: http://nco.sourceforge.net

=========    ===============================
version      toolchain                      
=========    ===============================
``4.6.0``    ``foss/2016a``                 
``4.6.6``    ``intel/2017a``                
``4.7.0``    ``intel/2017b``                
``4.7.1``    ``intel/2017b``                
``4.7.4``    ``foss/2017b``                 
``4.7.6``    ``intel/2018a``                
``4.7.9``    ``foss/2018b``, ``intel/2018b``
``4.8.1``    ``foss/2019a``                 
=========    ===============================


.. _list_software_ncompress_986:

*ncompress*
+++++++++++

Compress is a fast, simple LZW file compressor. Compress does not have the  highest compression rate, but it is one of the fastest programs to compress  data. Compress is the defacto standard in the UNIX community for compressing  files. 

*homepage*: http://ncompress.sourceforge.net/

===========    =================
version        toolchain        
===========    =================
``4.2.4.4``    ``GCCcore/6.4.0``
===========    =================


.. _list_software_ncurses_771:

*ncurses*
+++++++++

The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,  and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and  function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses.

*homepage*: http://www.gnu.org/software/ncurses/

=======    =============================================================================================================================================================================================================================================================================================================================================================================
version    toolchain                                                                                                                                                                                                                                                                                                                                                                    
=======    =============================================================================================================================================================================================================================================================================================================================================================================
``5.9``    ``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``, ``gimkl/2.11.5``, ``system``                                                                                                                                                                                                                                                  
``6.0``    ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``6.1``    ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``system``                                                                                                                                                                                                                                                                    
=======    =============================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_ncview_652:

*ncview*
++++++++

Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc.

*homepage*: http://meteora.ucsd.edu/~pierce/ncview_home_page.html

=========    =====================================================================================================================================
version      toolchain                                                                                                                            
=========    =====================================================================================================================================
``2.1.7``    ``foss/2017b``, ``gompi/2019a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
=========    =====================================================================================================================================


.. _list_software_nd2reader_887:

*nd2reader*
+++++++++++

nd2reader is a pure-Python package that reads images produced by NIS Elements 4.0+.

*homepage*: http://www.lighthacking.nl/nd2reader/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.0.6``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_ne_211:

*ne*
++++

ne is a free (GPL'd) text editor based on the POSIX standard that runs (we hope) on almost any UN*X machine. ne is easy to use for the beginner, but powerful and fully configurable for the wizard, and most sparing in its resource usage.

*homepage*: http://ne.di.unimi.it/

=========    ===============
version      toolchain      
=========    ===============
``3.0.1``    ``gimkl/2017a``
=========    ===============


.. _list_software_NEdit_468:

*NEdit*
+++++++

NEdit is a multi-purpose text editor for the X Window System,  which combines a standard, easy to use, graphical user interface with the thorough  functionality and stability required by users who edit text eight hours a day.

*homepage*: http://www.nedit.org

=======    ==============    ==========
version    versionsuffix     toolchain 
=======    ==============    ==========
``5.5``    ``-Linux-x86``    ``system``
=======    ==============    ==========


.. _list_software_Net-core_765:

*Net-core*
++++++++++

.NET Core is a free and open-source managed computer software framework for the Windows,  Linux, and macOS operating systems .NET Core fully supports C# and F# and partially supports Visual Basic

*homepage*: https://dotnet.microsoft.com/

=========    ==========
version      toolchain 
=========    ==========
``2.1.8``    ``system``
``2.2.5``    ``system``
``3.0.0``    ``system``
=========    ==========


.. _list_software_netCDF_532:

*netCDF*
++++++++

NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.

*homepage*: https://www.unidata.ucar.edu/software/netcdf/

===========    ================    ==========================================================================================================================
version        versionsuffix       toolchain                                                                                                                 
===========    ================    ==========================================================================================================================
``4.3.3.1``                        ``foss/2016a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``                                                                
``4.4.0``                          ``foss/2016a``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``                                      
``4.4.1``                          ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                                                          
``4.4.1.1``                        ``foss/2016b``, ``foss/2017b``, ``fosscuda/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``4.4.1.1``    ``-HDF5-1.10.1``    ``foss/2017a``, ``intel/2017a``                                                                                           
``4.4.1.1``    ``-HDF5-1.8.18``    ``intel/2017a``                                                                                                           
``4.4.1.1``    ``-HDF5-1.8.19``    ``foss/2017a``, ``foss/2017b``, ``intel/2017b``                                                                           
``4.5.0``                          ``foss/2017b``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01``                                                     
``4.6.0``                          ``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``                                                                          
``4.6.1``                          ``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``                         
``4.6.2``                          ``gompi/2019a``, ``iimpi/2019a``                                                                                          
``4.7.1``                          ``gompi/2019b``                                                                                                           
===========    ================    ==========================================================================================================================


.. _list_software_netCDF-C++_730:

*netCDF-C++*
++++++++++++

NetCDF (network Common Data Form) is a set of software libraries   and machine-independent data formats that support the creation, access, and sharing of array-oriented   scientific data.

*homepage*: http://www.unidata.ucar.edu/software/netcdf/

=======    ==============
version    toolchain     
=======    ==============
``4.2``    ``foss/2016a``
=======    ==============


.. _list_software_netCDF-C++4_782:

*netCDF-C++4*
+++++++++++++

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: http://www.unidata.ucar.edu/software/netcdf/

=========    ================    ======================================================================================================================================================
version      versionsuffix       toolchain                                                                                                                                             
=========    ================    ======================================================================================================================================================
``4.3.0``                        ``foss/2018b``, ``gompi/2019a``, ``iimpi/2019a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
``4.3.0``    ``-HDF5-1.8.18``    ``intel/2017a``                                                                                                                                       
``4.3.0``    ``-HDF5-1.8.19``    ``intel/2017b``                                                                                                                                       
=========    ================    ======================================================================================================================================================


.. _list_software_netCDF-Fortran_1309:

*netCDF-Fortran*
++++++++++++++++

NetCDF (network Common Data Form) is a set of software libraries  and machine-independent data formats that support the creation, access, and sharing of array-oriented  scientific data.

*homepage*: http://www.unidata.ucar.edu/software/netcdf/

=========    ================    ====================================================================================================================================================================================================================================================================================================================================
version      versionsuffix       toolchain                                                                                                                                                                                                                                                                                                                           
=========    ================    ====================================================================================================================================================================================================================================================================================================================================
``4.4.1``                        ``foss/2016b``, ``intel/2016b``                                                                                                                                                                                                                                                                                                     
``4.4.2``                        ``intel/2016.02-GCC-4.9``                                                                                                                                                                                                                                                                                                           
``4.4.3``                        ``foss/2016a``, ``intel/2016a``                                                                                                                                                                                                                                                                                                     
``4.4.4``                        ``PGI/18.4-GCC-6.4.0-2.28``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2017b``, ``fosscuda/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``intelcuda/2017b``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``iomkl/2018b``
``4.4.4``    ``-HDF5-1.8.18``    ``intel/2017a``                                                                                                                                                                                                                                                                                                                     
``4.4.4``    ``-HDF5-1.8.19``    ``foss/2017b``, ``intel/2017b``                                                                                                                                                                                                                                                                                                     
``4.4.5``                        ``gompi/2019a``, ``iimpi/2019a``                                                                                                                                                                                                                                                                                                    
=========    ================    ====================================================================================================================================================================================================================================================================================================================================


.. _list_software_netcdf4-python_1399:

*netcdf4-python*
++++++++++++++++

Python/numpy interface to netCDF.

*homepage*: https://unidata.github.io/netcdf4-python/

=========    =============================    ===============
version      versionsuffix                    toolchain      
=========    =============================    ===============
``1.2.9``    ``-Python-2.7.13``               ``intel/2017a``
``1.3.1``    ``-Python-3.6.3``                ``intel/2017b``
``1.3.1``    ``-Python-3.6.3-HDF5-1.8.19``    ``intel/2017b``
``1.4.0``    ``-Python-3.6.2-HDF5-1.8.19``    ``foss/2017b`` 
``1.4.0``    ``-Python-3.6.4``                ``intel/2018a``
``1.4.1``    ``-Python-2.7.15``               ``intel/2018b``
``1.4.1``    ``-Python-3.6.6``                ``intel/2018b``
``1.4.2``    ``-Python-3.6.4``                ``foss/2018a`` 
``1.4.3``    ``-Python-3.6.6``                ``foss/2018b`` 
``1.5.2``                                     ``intel/2019a``
=========    =============================    ===============


.. _list_software_netloc_645:

*netloc*
++++++++

The Portable Network Locality (netloc) software package provides  network topology discovery tools, and an abstract representation of those networks  topologies for a range of network types and configurations.  It is provided as a companion to the Portable Hardware Locality (hwloc) package.

*homepage*: http://www.open-mpi.org/software/netloc

=======    =============
version    toolchain    
=======    =============
``0.5``    ``GCC/4.8.3``
=======    =============


.. _list_software_NetLogo_696:

*NetLogo*
+++++++++

NetLogo is a multi-agent programmable modeling environment. It is used by tens of thousands of students, teachers and researchers worldwide. It also powers HubNet participatory simulations. It is authored by Uri Wilensky and developed at the CCL.

*homepage*: http://ccl.northwestern.edu/netlogo/

=========    =============    ==========
version      versionsuffix    toolchain 
=========    =============    ==========
``5.3.1``    ``-64``          ``system``
``6.0.4``    ``-64``          ``system``
=========    =============    ==========


.. _list_software_netMHC_543:

*netMHC*
++++++++

NetMHC 4.0 software  predicts binding of peptides to a number of different  HLA alleles  using artificial neural networks (ANN). 

*homepage*: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHC

========    ==========
version     toolchain 
========    ==========
``4.0a``    ``system``
========    ==========


.. _list_software_netMHCIIpan_1008:

*netMHCIIpan*
+++++++++++++

The NetMHCIIpan software predicts binding of peptides to MHC class II molecules.  The predictions are available for the three human MHC class II isotypes HLA-DR, HLA-DP and HLA-DQ,  as well as mouse molecules (H-2). 

*homepage*: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHCIIpan

=======    =================
version    toolchain        
=======    =================
``3.2``    ``GCCcore/7.3.0``
=======    =================


.. _list_software_netMHCpan_862:

*netMHCpan*
+++++++++++

The NetMHCpan software predicts  binding of peptides  to any known MHC  molecule using artificial neural networks (ANNs). 

*homepage*: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHCpan

========    ==========
version     toolchain 
========    ==========
``4.0a``    ``system``
========    ==========


.. _list_software_NetPIPE_597:

*NetPIPE*
+++++++++

NetPIPE is a protocol independent communication performance benchmark that visually represents  the network performance under a variety of conditions.

*homepage*: http://netpipe.cs.ksu.edu/

=======    ===============
version    toolchain      
=======    ===============
``5.1``    ``intel/2018a``
=======    ===============


.. _list_software_nettle_652:

*nettle*
++++++++

Nettle is a cryptographic library that is designed to fit easily  in more or less any context: In crypto toolkits for object-oriented  languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,  or even in kernel space.

*homepage*: http://www.lysator.liu.se/~nisse/nettle/

=========    ===========================================================================================================================================
version      toolchain                                                                                                                                  
=========    ===========================================================================================================================================
``3.1.1``    ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``                                                                                        
``3.2``      ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``                                                                                         
``3.3``      ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``intel/2017a``                                                                                      
``3.4``      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018a``
``3.4.1``    ``GCCcore/8.2.0``                                                                                                                          
``3.5.1``    ``GCCcore/8.3.0``                                                                                                                          
=========    ===========================================================================================================================================


.. _list_software_networkx_898:

*networkx*
++++++++++

NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,  and functions of complex networks.

*homepage*: https://pypi.python.org/pypi/networkx

========    ==================    ===============================
version     versionsuffix         toolchain                      
========    ==================    ===============================
``1.1``     ``-Python-2.7.12``    ``intel/2016b``                
``1.11``    ``-Python-2.7.11``    ``foss/2016a``                 
``1.11``    ``-Python-2.7.14``    ``intel/2017b``                
``1.11``    ``-Python-3.5.1``     ``foss/2016a``                 
``2.0``     ``-Python-2.7.14``    ``intel/2017b``                
``2.1``     ``-Python-2.7.14``    ``intel/2017b``                
``2.2``     ``-Python-2.7.15``    ``intel/2018b``                
``2.2``     ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
``2.3``     ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``
``2.4``     ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``
========    ==================    ===============================


.. _list_software_NeuroKit_817:

*NeuroKit*
++++++++++

NeuroKit is a Python module that provides high-level integrative functions with good and  flexible defaults, allowing users to focus on what’s important.

*homepage*: https://neurokit.readthedocs.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.2.7``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_NEURON_471:

*NEURON*
++++++++

Empirically-based simulations of neurons and networks of neurons.

*homepage*: http://www.neuron.yale.edu/neuron

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``7.4``                            ``intel/2016b``
``7.6.5``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_Nextflow_855:

*Nextflow*
++++++++++

Nextflow is a reactive workflow framework and a programming DSL   that eases writing computational pipelines with complex data

*homepage*: https://www.nextflow.io/

===========    ==========
version        toolchain 
===========    ==========
``19.04.0``    ``system``
``19.07.0``    ``system``
===========    ==========


.. _list_software_NextGenMap_983:

*NextGenMap*
++++++++++++

NextGenMap is a flexible highly sensitive short read mapping tool that  handles much higher mismatch rates than comparable algorithms while still outperforming  them in terms of runtime.

*homepage*: http://cibiv.github.io/NextGenMap/

=========    ==============
version      toolchain     
=========    ==============
``0.5.5``    ``foss/2016b``
=========    ==============


.. _list_software_NFFT_302:

*NFFT*
++++++

The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine  library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more  dimensions, of arbitrary input size, and of complex data.

*homepage*: https://www-user.tu-chemnitz.de/~potts/nfft/

=========    ==============================
version      toolchain                     
=========    ==============================
``3.5.1``    ``foss/2018b``, ``foss/2019a``
=========    ==============================


.. _list_software_nglview_764:

*nglview*
+++++++++

IPython widget to interactively view molecular structures and trajectories.

*homepage*: https://github.com/arose/nglview

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``2.7.0``    ``-Python-3.7.2``    ``intel/2019a``
=========    =================    ===============


.. _list_software_NGS_232:

*NGS*
+++++

NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.

*homepage*: https://github.com/ncbi/ngs

==========    ===================    ==================================
version       versionsuffix          toolchain                         
==========    ===================    ==================================
``1.2.3``                            ``foss/2016b``, ``intel/2016a``   
``1.2.5``                            ``foss/2016b``                    
``1.3.0``                            ``GCCcore/6.4.0``, ``intel/2017a``
``2.9.1``     ``-Java-1.8.0_162``    ``foss/2018a``, ``intel/2018a``   
``2.9.3``     ``-Java-1.8``          ``foss/2018b``                    
``2.10.0``    ``-Java-1.8``          ``GCCcore/8.2.0``                 
``2.10.0``    ``-Java-11``           ``GCCcore/8.2.0``                 
==========    ===================    ==================================


.. _list_software_NGS-Python_919:

*NGS-Python*
++++++++++++

NGS is a new, domain-specific API for accessing reads, alignments and pileups produced from  Next Generation Sequencing.

*homepage*: https://github.com/ncbi/ngs

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.2.3``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_NGSadmix_763:

*NGSadmix*
++++++++++

NGSadmix is a tool for finding admixture proportions  from NGS data, based on genotype likelihoods.

*homepage*: http://www.popgen.dk/software/index.php/NgsAdmix

=======    ==================
version    toolchain         
=======    ==================
``32``     ``GCC/7.3.0-2.30``
=======    ==================


.. _list_software_NiBabel_653:

*NiBabel*
+++++++++

NiBabel provides read/write access to some common medical and neuroimaging file formats,  including: ANALYZE (plain, SPM99, SPM2 and later), GIFTI, NIfTI1, NIfTI2, MINC1, MINC2, MGH and ECAT  as well as Philips PAR/REC. We can read and write Freesurfer geometry, and read Freesurfer morphometry and  annotation files. There is some very limited support for DICOM. NiBabel is the successor of PyNIfTI.

*homepage*: https://nipy.github.io/nibabel

=========    =================================    ===============================
version      versionsuffix                        toolchain                      
=========    =================================    ===============================
``2.0.2``    ``-Python-2.7.11``                   ``intel/2016a``                
``2.0.2``    ``-Python-2.7.11-freetype-2.6.3``    ``intel/2016a``                
``2.1.0``    ``-Python-2.7.13``                   ``intel/2017a``                
``2.2.1``    ``-Python-3.6.4``                    ``intel/2018a``                
``2.3.0``    ``-Python-2.7.14``                   ``foss/2017b``, ``intel/2017b``
``2.3.0``    ``-Python-3.6.3``                    ``foss/2017b``, ``intel/2017b``
``2.3.0``    ``-Python-3.6.6``                    ``foss/2018b``                 
``2.4.0``                                         ``foss/2019a``, ``intel/2019a``
``2.5.1``    ``-Python-3.6.6``                    ``foss/2018b``                 
=========    =================================    ===============================


.. _list_software_NIfTI_410:

*NIfTI*
+++++++

Niftilib is a set of i/o libraries for reading and writing files in the nifti-1 data format.

*homepage*: http://niftilib.sourceforge.net/

=========    =================
version      toolchain        
=========    =================
``2.0.0``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_Nilearn_713:

*Nilearn*
+++++++++

Nilearn is a Python module for fast and easy statistical learning on NeuroImaging data.

*homepage*: http://nilearn.github.io/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.5.2``                          ``foss/2019a``, ``intel/2019a``
``0.5.2``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``0.5.2``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
=========    ==================    ===============================


.. _list_software_Nim_292:

*Nim*
+++++

Nim is a systems and applications programming language.

*homepage*: https://nim-lang.org/

==========    =================
version       toolchain        
==========    =================
``0.18.0``    ``GCCcore/6.4.0``
``0.19.2``    ``GCCcore/7.3.0``
``1.0.0``     ``GCCcore/8.3.0``
==========    =================


.. _list_software_NIMBLE_439:

*NIMBLE*
++++++++

NIMBLE is a system for building and sharing analysis methods for statistical models,  especially for hierarchical models and computationally-intensive methods.

*homepage*: https://r-nimble.org

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``0.7.0``    ``-R-3.5.1``     ``foss/2018b``
=========    =============    ==============


.. _list_software_Ninja_496:

*Ninja*
+++++++

Ninja is a small build system with a focus on speed.

*homepage*: https://ninja-build.org/

=========    ====================================================================================
version      toolchain                                                                           
=========    ====================================================================================
``1.8.2``    ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2017b``, ``intel/2018a``
``1.9.0``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2018b``                                
=========    ====================================================================================


.. _list_software_Nipype_629:

*Nipype*
++++++++

Nipype is a Python project that provides a uniform interface to existing neuroimaging software and  facilitates interaction between these packages within a single workflow.

*homepage*: http://nipy.org/nipype

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.0.2``    ``-Python-3.6.4``    ``intel/2018a``
``1.1.3``    ``-Python-3.6.6``    ``foss/2018b`` 
=========    =================    ===============


.. _list_software_NLopt_493:

*NLopt*
+++++++

NLopt is a free/open-source library for nonlinear optimization,   providing a common interface for a number of different free optimization routines   available online as well as original implementations of various other algorithms. 

*homepage*: http://ab-initio.mit.edu/wiki/index.php/NLopt

=========    ===================================================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                                                          
=========    ===================================================================================================================================================================================================================================================
``2.4.2``    ``GCC/5.4.0-2.26``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``iomkl/2018a``
``2.6.1``    ``GCCcore/8.2.0``                                                                                                                                                                                                                                  
=========    ===================================================================================================================================================================================================================================================


.. _list_software_NLTK_313:

*NLTK*
++++++

NLTK is a leading platform for building Python programs to work with human language data.

*homepage*: http://www.nltk.org/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.2.2``    ``-Python-2.7.13``    ``intel/2017a``
``3.2.4``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_Node-RED_654:

*Node-RED*
++++++++++

Node-RED is a programming tool for wiring together hardware devices, APIs and online services  in new and interesting ways.

*homepage*: http://nodered.org

==========    ==============
version       toolchain     
==========    ==============
``0.16.2``    ``foss/2017a``
==========    ==============


.. _list_software_nodejs_643:

*nodejs*
++++++++

Node.js is a platform built on Chrome's JavaScript runtime   for easily building fast, scalable network applications. Node.js uses an   event-driven, non-blocking I/O model that makes it lightweight and efficient,   perfect for data-intensive real-time applications that run across distributed devices.

*homepage*: http://nodejs.org

===========    =================
version        toolchain        
===========    =================
``4.4.7``      ``foss/2016a``   
``6.10.3``     ``foss/2017a``   
``8.9.4``      ``foss/2017a``   
``10.15.1``    ``foss/2018b``   
``10.15.3``    ``GCCcore/8.2.0``
===========    =================


.. _list_software_Normaliz_844:

*Normaliz*
++++++++++

Normaliz is an open source tool for computations in affine monoids, vector configurations, lattice polytopes, and rational cones.

*homepage*: https://www.normaliz.uni-osnabrueck.de/

=========    ===============
version      toolchain      
=========    ===============
``3.6.3``    ``intel/2018b``
=========    ===============


.. _list_software_nose-parameterized_1871:

*nose-parameterized*
++++++++++++++++++++

Parameterized testing with any Python test framework.

*homepage*: hmat://github.com/wolever/nose-parameterized

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.5.0``    ``-Python-2.7.11``    ``foss/2016a`` 
``0.5.0``    ``-Python-3.5.1``     ``foss/2016a`` 
``0.5.0``    ``-Python-3.5.2``     ``intel/2016b``
``0.6.0``    ``-Python-3.6.1``     ``intel/2017a``
=========    ==================    ===============


.. _list_software_novoalign_973:

*novoalign*
+++++++++++

NovoCraft is a software bundle.  NovoAlign:  Market’s leading aligner with fully packed features designed for mapping  of short reads onto a reference genome from Illumina, Ion Torrent, 454,  and Color Spance NGS platforms.  NovoAlignCS:  Leading aligner packed with features designed to fully support reads from  ABI SOLiD Color Space.  NovoSort:  Custom designed multi-threaded sort/merge tools for BAM files.  NovoMethyl:  It can analyse a set of alignments to identify methylated cytosine’s.  WARNING!  You can only use the sofware without a license (with some features disabled)  if you are a non-profit organisation and use is for your own research or for  use by students as part of their course. A license is required for use where  these programs are part of a service where a third party is billed.

*homepage*: http://www.novocraft.com/

===========    =============    ==============
version        versionsuffix    toolchain     
===========    =============    ==============
``3.09.00``                     ``system``    
``3.09.01``    ``-R-3.5.1``     ``foss/2018b``
===========    =============    ==============


.. _list_software_NOVOPlasty_959:

*NOVOPlasty*
++++++++++++

NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.

*homepage*: https://github.com/ndierckx/NOVOPlasty

=======    =================
version    toolchain        
=======    =================
``3.7``    ``GCCcore/8.3.0``
=======    =================


.. _list_software_npstat_666:

*npstat*
++++++++

npstat implements some population genetics tests and estimators that can be applied  to pooled sequences from Next Generation Sequencing experiments.

*homepage*: https://code.google.com/archive/p/npstat/

========    ===============================
version     toolchain                      
========    ===============================
``0.99``    ``foss/2016a``, ``intel/2016a``
========    ===============================


.. _list_software_NRGLjubljana_1146:

*NRGLjubljana*
++++++++++++++

NRG Ljubljana is an efficient implementation of the numerical renormalization group (NRG)   technique for solving quantum impurity problems that arise as simplified models of   magnetic impurities and as effective models in the dynamical mean field theory (DMFT)   approach to bulk correlated materials. 

*homepage*: http://nrgljubljana.ijs.si/

============    ==============
version         toolchain     
============    ==============
``2.4.3.23``    ``foss/2016b``
============    ==============


.. _list_software_NSPR_323:

*NSPR*
++++++

Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level  and libc-like functions.

*homepage*: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR

========    ====================================
version     toolchain                           
========    ====================================
``4.20``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``4.21``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
========    ====================================


.. _list_software_NSS_244:

*NSS*
+++++

Network Security Services (NSS) is a set of libraries designed to support cross-platform development  of security-enabled client and server applications.

*homepage*: https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS

==========    ====================================
version       toolchain                           
==========    ====================================
``3.39``      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``3.42.1``    ``GCCcore/8.2.0``                   
``3.45``      ``GCCcore/8.3.0``                   
==========    ====================================


.. _list_software_numactl_756:

*numactl*
+++++++++

The numactl program allows you to run your application program on specific cpu's and memory nodes. It does this by supplying a NUMA memory policy to the operating system before running your program. The libnuma library provides convenient ways for you to add NUMA memory policies into your own program.

*homepage*: http://oss.sgi.com/projects/libnuma/

==========    ==============================================================================================================================================================================================================================================================================================================================================================================================================================================
version       toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                     
==========    ==============================================================================================================================================================================================================================================================================================================================================================================================================================================
``2.0.9``     ``GCC/4.8.3``                                                                                                                                                                                                                                                                                                                                                                                                                                 
``2.0.10``    ``GCC/4.8.4``, ``GCC/4.9.2``, ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``                                                                                                                                                                                                                                                                                                                                                                          
``2.0.11``    ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/5.2.0``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``GCC/6.2.0-2.27``, ``GCC/6.3.0-2.27``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016a``, ``intel/2016b``
``2.0.12``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                                                                                                                                                                                                                                                                                                                                          
==========    ==============================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_numba_531:

*numba*
+++++++

Numba is an Open Source NumPy-aware optimizing compiler for Python sponsored by Continuum Analytics, Inc. It uses the remarkable LLVM compiler infrastructure to compile Python syntax to machine code.

*homepage*: https://numba.pydata.org/

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``0.24.0``    ``-Python-2.7.11``    ``intel/2016a``                
``0.24.0``    ``-Python-3.5.1``     ``intel/2016a``                
``0.26.0``    ``-Python-2.7.11``    ``intel/2016a``                
``0.32.0``    ``-Python-2.7.13``    ``intel/2017a``                
``0.37.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2017b``
``0.37.0``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``
``0.43.1``                          ``intel/2019a``                
``0.46.0``                          ``foss/2019a``                 
==========    ==================    ===============================


.. _list_software_numexpr_783:

*numexpr*
+++++++++

The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.  It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into  code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a  compiler at runtime.

*homepage*: https://numexpr.readthedocs.io

=========    ==================    ==================================
version      versionsuffix         toolchain                         
=========    ==================    ==================================
``2.5.2``    ``-Python-2.7.11``    ``intel/2016a``                   
``2.6.1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``   
``2.6.1``    ``-Python-3.5.2``     ``intel/2016b``                   
``2.6.4``    ``-Python-2.7.13``    ``foss/2017a``                    
``2.6.4``    ``-Python-2.7.14``    ``intel/2018a``                   
``2.6.4``    ``-Python-3.5.1``     ``foss/2016a``                    
``2.6.4``    ``-Python-3.6.1``     ``intel/2017a``                   
``2.6.4``    ``-Python-3.6.3``     ``intel/2017b``                   
``2.6.4``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``   
``2.6.5``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``   
``2.6.5``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``
``2.7.0``                          ``intel/2019a``                   
=========    ==================    ==================================


.. _list_software_numpy_569:

*numpy*
+++++++

NumPy is the fundamental package for scientific computing with Python. It contains among other things:  a powerful N-dimensional array object, sophisticated (broadcasting) functions, tools for integrating C/C++ and Fortran  code, useful linear algebra, Fourier transform, and random number capabilities. Besides its obvious scientific uses,  NumPy can also be used as an efficient multi-dimensional container of generic data. Arbitrary data-types can be   defined. This allows NumPy to seamlessly and speedily integrate with a wide variety of databases.

*homepage*: http://www.numpy.org

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``1.8.2``     ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``1.9.2``     ``-Python-2.7.12``    ``intel/2016b``                
``1.10.4``    ``-Python-2.7.11``    ``intel/2016a``                
``1.11.0``    ``-Python-2.7.11``    ``intel/2016a``                
``1.12.1``    ``-Python-3.5.2``     ``intel/2016b``                
``1.13.0``    ``-Python-2.7.13``    ``intel/2017a``                
``1.13.1``    ``-Python-3.6.1``     ``intel/2017a``                
==========    ==================    ===============================


.. _list_software_nvtop_567:

*nvtop*
+++++++

htop-like GPU usage monitor

*homepage*: https://github.com/Syllo/nvtop

=========    ==================
version      toolchain         
=========    ==================
``1.0.0``    ``fosscuda/2018b``
=========    ==================


.. _list_software_NWChem_546:

*NWChem*
++++++++

NWChem aims to provide its users with computational chemistry tools that are scalable both in  their ability to treat large scientific computational chemistry problems efficiently, and in their use of available  parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters.  NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all  combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties  and relativity.

*homepage*: http://www.nwchem-sw.org

=====================    ==============================================    ================================
version                  versionsuffix                                     toolchain                       
=====================    ==============================================    ================================
``6.6.revision27746``    ``-2015-10-20-Python-2.7.12``                     ``iomkl/2017a``                 
``6.6.revision27746``    ``-2015-10-20-patches-20170814-Python-2.7.13``    ``intel/2017a``                 
``6.8.revision47``       ``-2017-12-14-Python-2.7.14``                     ``intel/2017b``, ``intel/2018a``
=====================    ==============================================    ================================


.. _list_software_NxTrim_610:

*NxTrim*
++++++++

NxTrim is a software to remove Nextera Mate Pair junction adapters and categorise reads according to the orientation implied by the adapter location.

*homepage*: https://github.com/sequencing/NxTrim

=========    ==============
version      toolchain     
=========    ==============
``0.4.3``    ``foss/2018a``
=========    ==============


.. _list_software_letter_o:

*O*
---


:ref:`list_software_Oases_507` - :ref:`list_software_OBITools_747` - :ref:`list_software_OCaml_460` - :ref:`list_software_Octave_610` - :ref:`list_software_OMA_221` - :ref:`list_software_OOMPA_380` - :ref:`list_software_OPARI2_429` - :ref:`list_software_OpenBabel_872` - :ref:`list_software_OpenBLAS_692` - :ref:`list_software_OpenCoarrays_1238` - :ref:`list_software_OpenColorIO_1065` - :ref:`list_software_OpenCV_555` - :ref:`list_software_opencv_contrib_1499` - :ref:`list_software_OpenEXR_641` - :ref:`list_software_OpenFOAM_693` - :ref:`list_software_OpenFOAM-Extend_1354` - :ref:`list_software_OpenImageIO_1037` - :ref:`list_software_OpenJPEG_696` - :ref:`list_software_OpenKIM-API_890` - :ref:`list_software_openkim-models_1444` - :ref:`list_software_OpenMM_556` - :ref:`list_software_OpenMolcas_1009` - :ref:`list_software_OpenMPI_632` - :ref:`list_software_OpenNLP_636` - :ref:`list_software_OpenPGM_630` - :ref:`list_software_openpyxl_895` - :ref:`list_software_OpenRefine_1003` - :ref:`list_software_OpenSSL_644` - :ref:`list_software_OPERA_375` - :ref:`list_software_OptiType_830` - :ref:`list_software_OptiX_500` - :ref:`list_software_OR-Tools_735` - :ref:`list_software_ORCA_293` - :ref:`list_software_OrfM_372` - :ref:`list_software_OrthoFinder_1124` - :ref:`list_software_OrthoMCL_744` - :ref:`list_software_OSU-Micro-Benchmarks_1865` - :ref:`list_software_OTF2_283` - :ref:`list_software_ownCloud_843` - :ref:`list_software_oxford_asl_1073`


.. _list_software_Oases_507:

*Oases*
+++++++

Oases is a de novo transcriptome assembler designed to produce transcripts from  short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly.

*homepage*: http://www.ebi.ac.uk/~zerbino/oases/

==========    =============    ===============
version       versionsuffix    toolchain      
==========    =============    ===============
``0.2.08``                     ``foss/2016b`` 
``0.2.08``    ``-kmer_101``    ``intel/2017b``
==========    =============    ===============


.. _list_software_OBITools_747:

*OBITools*
++++++++++

The OBITools programs aims to help you to manipulate various data and sequence files in a  convenient way using the Unix command line interface. They follow the standard Unix interface for command line  program, allowing to chain a set of commands using the pipe mecanism.

*homepage*: http://www.grenoble.prabi.fr/trac/OBITools

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.2.9``    ``-Python-2.7.11``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_OCaml_460:

*OCaml*
+++++++

OCaml is a general purpose industrial-strength programming language  with an emphasis on expressiveness and safety. Developed for more than 20 years at Inria  it benefits from one of the most advanced type systems and supports functional,  imperative and object-oriented styles of programming.

*homepage*: http://ocaml.org/

==========    ==============
version       toolchain     
==========    ==============
``4.02.3``    ``foss/2016a``
``4.07.1``    ``foss/2018b``
==========    ==============


.. _list_software_Octave_610:

*Octave*
++++++++

GNU Octave is a high-level interpreted language, primarily intended for numerical computations.

*homepage*: http://www.gnu.org/software/octave/

=========    =============    ================================================
version      versionsuffix    toolchain                                       
=========    =============    ================================================
``4.0.0``                     ``foss/2016a``                                  
``4.0.3``                     ``intel/2016b``                                 
``4.2.1``                     ``foss/2018a``, ``intel/2016b``, ``intel/2017a``
``4.2.1``    ``-mt``          ``intel/2017a``                                 
``4.2.2``                     ``foss/2018a``                                  
``4.4.1``                     ``foss/2018b``                                  
=========    =============    ================================================


.. _list_software_OMA_221:

*OMA*
+++++

The OMA ('Orthologous MAtrix') project is a method and database for the inference   of orthologs among complete genomes

*homepage*: https://omabrowser.org/standalone/

=========    ==========
version      toolchain 
=========    ==========
``2.1.1``    ``system``
=========    ==========


.. _list_software_OOMPA_380:

*OOMPA*
+++++++

OOMPA is a suite of R packages for the analysis of gene expression (RNA), proteomics profiling,  and other high throughput molecular biology data.

*homepage*: https://r-forge.r-project.org/projects/oompa

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``3.1.2``    ``-R-3.3.1``     ``intel/2016b``
=========    =============    ===============


.. _list_software_OPARI2_429:

*OPARI2*
++++++++

OPARI2, the successor of Forschungszentrum Juelich's OPARI, is a  source-to-source instrumentation tool for OpenMP and hybrid codes.  It surrounds OpenMP directives and runtime library calls with calls  to the POMP2 measurement interface. 

*homepage*: https://www.score-p.org

=========    =================
version      toolchain        
=========    =================
``2.0``      ``foss/2016a``   
``2.0.5``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_OpenBabel_872:

*OpenBabel*
+++++++++++

Open Babel is a chemical toolbox designed to speak the many  languages of chemical data. It's an open, collaborative project allowing anyone  to search, convert, analyze, or store data from molecular modeling, chemistry,  solid-state materials, biochemistry, or related areas.

*homepage*: http://openbabel.org

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.3.2``    ``-Python-2.7.11``    ``foss/2016a`` 
``2.4.1``    ``-Python-2.7.14``    ``intel/2017b``
``2.4.1``    ``-Python-2.7.15``    ``intel/2018b``
``2.4.1``    ``-Python-3.6.6``     ``intel/2018b``
=========    ==================    ===============


.. _list_software_OpenBLAS_692:

*OpenBLAS*
++++++++++

OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.

*homepage*: http://xianyi.github.com/OpenBLAS/

==========    =================    ==========================================================
version       versionsuffix        toolchain                                                 
==========    =================    ==========================================================
``0.2.9``     ``-LAPACK-3.5.0``    ``GCC/4.8.3``                                             
``0.2.12``    ``-LAPACK-3.5.0``    ``GCC/4.9.2``                                             
``0.2.13``    ``-LAPACK-3.5.0``    ``GCC/4.8.4``, ``GCC/4.9.2``                              
``0.2.14``    ``-LAPACK-3.5.0``    ``GNU/4.9.2-2.25``, ``GNU/4.9.3-2.25``                    
``0.2.15``    ``-LAPACK-3.6.0``    ``GCC/4.9.3-2.25``                                        
``0.2.18``    ``-LAPACK-3.6.0``    ``GCC/4.9.4-2.25``, ``GCC/5.3.0-2.26``, ``GCC/5.4.0-2.26``
``0.2.18``    ``-LAPACK-3.6.1``    ``GCC/5.4.0-2.26``, ``gompi/2016.07``                     
``0.2.19``    ``-LAPACK-3.6.1``    ``gompi/2016.09``                                         
``0.2.19``    ``-LAPACK-3.7.0``    ``GCC/5.4.0-2.26``, ``GCC/6.3.0-2.27``                    
``0.2.20``                         ``GCC/5.4.0-2.26``, ``GCC/6.4.0-2.28``, ``GCC/7.2.0-2.29``
``0.3.0``                          ``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``                    
``0.3.1``                          ``GCC/7.3.0-2.30``                                        
``0.3.3``                          ``GCC/8.2.0-2.31.1``                                      
``0.3.4``                          ``GCC/8.2.0-2.31.1``                                      
``0.3.5``                          ``GCC/8.2.0-2.31.1``                                      
``0.3.6``                          ``GCC/8.3.0-2.32``                                        
``0.3.7``                          ``GCC/8.3.0``                                             
==========    =================    ==========================================================


.. _list_software_OpenCoarrays_1238:

*OpenCoarrays*
++++++++++++++

OpenCoarrays is an open-source software project that supports the coarray Fortran (CAF) parallel programming features of the Fortran 2008 standard and several features proposed for Fortran 2015 in the draft Technical Specification TS 18508 Additional Parallel Features in Fortran.

*homepage*: https://github.com/sourceryinstitute/opencoarrays

=========    ===============
version      toolchain      
=========    ===============
``1.9.0``    ``gompi/2017a``
``2.2.0``    ``gompi/2018b``
``2.8.0``    ``gompi/2019b``
=========    ===============


.. _list_software_OpenColorIO_1065:

*OpenColorIO*
+++++++++++++

OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production  with an emphasis on visual effects and computer animation.

*homepage*: http://opencolorio.org/

=========    ==============
version      toolchain     
=========    ==============
``1.1.0``    ``foss/2018b``
=========    ==============


.. _list_software_OpenCV_555:

*OpenCV*
++++++++

OpenCV (Open Source Computer Vision Library) is an open source computer vision  and machine learning software library. OpenCV was built to provide  a common infrastructure for computer vision applications and to accelerate  the use of machine perception in the commercial products.

*homepage*: http://opencv.org/

==========    ==================    ================================================================
version       versionsuffix         toolchain                                                       
==========    ==================    ================================================================
``2.4.12``                          ``intel/2016a``                                                 
``3.1.0``                           ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``3.3.0``     ``-Python-2.7.13``    ``intel/2017a``                                                 
``3.3.0``     ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``                                 
``3.3.0``     ``-Python-3.6.1``     ``intel/2017a``                                                 
``3.3.0``     ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``                                 
``3.4.1``     ``-Python-2.7.14``    ``foss/2018a``                                                  
``3.4.1``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                                 
``3.4.5``     ``-Python-2.7.15``    ``foss/2018b``                                                  
``3.4.7``     ``-Python-2.7.15``    ``foss/2019a``, ``fosscuda/2019a``                              
``3.4.7``     ``-Python-3.7.2``     ``foss/2019a``, ``fosscuda/2019a``                              
``4.0.1``     ``-Python-2.7.15``    ``foss/2018b``                                                  
``4.0.1``     ``-Python-3.6.6``     ``foss/2018b``                                                  
==========    ==================    ================================================================


.. _list_software_opencv_contrib_1499:

*opencv_contrib*
++++++++++++++++

OpenCV (Open Source Computer Vision Library) is an open source computer vision  and machine learning software library. OpenCV was built to provide  a common infrastructure for computer vision applications and to accelerate  the use of machine perception in the commercial products.

*homepage*: http://opencv.org/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``3.4.1``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_OpenEXR_641:

*OpenEXR*
+++++++++

OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic  for use in computer imaging applications

*homepage*: https://www.openexr.com/

=========    ================================
version      toolchain                       
=========    ================================
``2.2.0``    ``intel/2016b``, ``intel/2017a``
``2.3.0``    ``foss/2018b``, ``intel/2018b`` 
``2.4.0``    ``GCCcore/8.3.0``               
=========    ================================


.. _list_software_OpenFOAM_693:

*OpenFOAM*
++++++++++

OpenFOAM is a free, open source CFD software package.  OpenFOAM has an extensive range of features to solve anything from complex fluid flows  involving chemical reactions, turbulence and heat transfer,  to solid dynamics and electromagnetics.

*homepage*: https://www.openfoam.com/

================    ================================================
version             toolchain                                       
================    ================================================
``2.2.2``           ``intel/2016a``, ``intel/2017a``                
``2.2.x``           ``intel/2019a``                                 
``2.3.1``           ``intel/2017a``                                 
``2.4.0``           ``intel/2017a``, ``intel/2019a``                
``3.0.0``           ``foss/2016a``                                  
``3.0.1``           ``intel/2016b``, ``intel/2018a``                
``4.0``             ``foss/2016b``, ``intel/2016b``                 
``4.1``             ``foss/2016b``, ``intel/2017a``                 
``5.0``             ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``5.0-20180108``    ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``6``               ``foss/2018b``, ``intel/2018a``                 
``v1606+``          ``foss/2018b``                                  
``v1612+``          ``foss/2018b``                                  
``v1712``           ``foss/2017b``, ``intel/2017b``                 
``v1806``           ``foss/2018b``                                  
``v1812``           ``foss/2018b``                                  
``v1906``           ``foss/2019b``                                  
================    ================================================


.. _list_software_OpenFOAM-Extend_1354:

*OpenFOAM-Extend*
+++++++++++++++++

OpenFOAM is a free, open source CFD software package.  OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer,  to solid dynamics and electromagnetics.

*homepage*: http://www.extend-project.de/

=======    =================================
version    toolchain                        
=======    =================================
``3.1``    ``gimkl/2.11.5``                 
``3.2``    ``gimkl/2.11.5``, ``intel/2016a``
``4.0``    ``intel/2017a``                  
=======    =================================


.. _list_software_OpenImageIO_1037:

*OpenImageIO*
+++++++++++++

OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities,  and applications.

*homepage*: https://openimageio.org/

==========    ================================
version       toolchain                       
==========    ================================
``1.6.17``    ``intel/2016b``                 
``1.7.17``    ``intel/2017a``                 
``1.8.16``    ``foss/2018b``, ``intel/2018b`` 
``2.0.12``    ``gompi/2019b``, ``iimpi/2019b``
==========    ================================


.. _list_software_OpenJPEG_696:

*OpenJPEG*
++++++++++

OpenJPEG is an open-source JPEG 2000 codec written in  C language. It has been developed in order to promote the use of JPEG 2000,  a still-image compression standard from the Joint Photographic Experts Group  (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as  a JPEG 2000 Reference Software.

*homepage*: http://www.openjpeg.org/

=========    ====================================
version      toolchain                           
=========    ====================================
``2.1``      ``GCCcore/6.4.0``                   
``2.3.0``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.3.1``    ``GCCcore/8.2.0``                   
=========    ====================================


.. _list_software_OpenKIM-API_890:

*OpenKIM-API*
+++++++++++++

Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild only installs the API, the models have to be installed by the user by running     kim-api-collections-management install user MODELNAME or     kim-api-collections-management install user OpenKIM to install them all.  

*homepage*: https://openkim.org/

=========    ================================================
version      toolchain                                       
=========    ================================================
``1.9.2``    ``foss/2016b``, ``foss/2017b``                  
``1.9.7``    ``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
=========    ================================================


.. _list_software_openkim-models_1444:

*openkim-models*
++++++++++++++++

Open Knowledgebase of Interatomic Models.  OpenKIM is an API and a collection of interatomic models (potentials) for atomistic simulations.  It is a library that can be used by simulation programs to get access to the models in the OpenKIM database.  This EasyBuild installs the models.  The API itself is in the kim-api package.  

*homepage*: https://openkim.org/

============    ===============================
version         toolchain                      
============    ===============================
``20190725``    ``foss/2019a``, ``intel/2019a``
============    ===============================


.. _list_software_OpenMM_556:

*OpenMM*
++++++++

OpenMM is a toolkit for molecular simulation.

*homepage*: https://simtk.org/home/openmm

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``7.1.1``    ``-Python-3.6.3``    ``intel/2017b``
=========    =================    ===============


.. _list_software_OpenMolcas_1009:

*OpenMolcas*
++++++++++++

OpenMolcas is a quantum chemistry software package

*homepage*: https://gitlab.com/Molcas/OpenMolcas

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``18.09``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_OpenMPI_632:

*OpenMPI*
+++++++++

The Open MPI Project is an open source MPI implementation.

*homepage*: http://www.open-mpi.org/

==========    =============    ====================================================================================================================
version       versionsuffix    toolchain                                                                                                           
==========    =============    ====================================================================================================================
``1.6.5``                      ``GCC/4.8.1``, ``GCC/4.8.2``, ``GCC/4.8.3``                                                                         
``1.6.5``     ``-no-OFED``     ``GCC/4.8.1``, ``GCC/4.8.2``                                                                                        
``1.7.3``                      ``GCC/4.8.2``                                                                                                       
``1.8.1``                      ``GCC/4.8.3``                                                                                                       
``1.8.3``                      ``GCC/4.9.2``                                                                                                       
``1.8.4``                      ``GCC/4.8.4``, ``GCC/4.9.2``                                                                                        
``1.8.5``                      ``GNU/4.9.2-2.25``                                                                                                  
``1.8.6``                      ``GNU/4.9.3-2.25``                                                                                                  
``1.8.8``                      ``GNU/4.9.3-2.25``                                                                                                  
``1.10.1``                     ``GCC/4.9.3-2.25``                                                                                                  
``1.10.2``                     ``GCC/4.9.3-2.25``, ``GCC/5.3.0-2.26``, ``GCC/6.1.0-2.27``, ``PGI/16.3-GCC-4.9.3-2.25``, ``PGI/16.4-GCC-5.3.0-2.26``
``1.10.3``                     ``GCC/5.4.0-2.26``, ``GCC/6.1.0-2.27``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``                                      
``1.10.4``                     ``PGI/16.7-GCC-5.4.0-2.26``, ``iccifort/2016.3.210-GCC-4.9.3-2.25``                                                 
``2.0.0``                      ``GCC/5.2.0``                                                                                                       
``2.0.1``                      ``GCC/6.2.0-2.27``, ``iccifort/2017.1.132-GCC-5.4.0-2.26``                                                          
``2.0.2``                      ``GCC/6.3.0-2.27``, ``iccifort/2017.1.132-GCC-6.3.0-2.27``                                                          
``2.0.2``     ``-opa``         ``GCC/6.3.0-2.27``                                                                                                  
``2.1.0``                      ``GCC/6.3.0-2.28``                                                                                                  
``2.1.1``                      ``GCC/6.4.0-2.28``, ``gcccuda/2017b``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``                                       
``2.1.2``                      ``GCC/6.4.0-2.28``, ``gcccuda/2018a``, ``iccifort/2018.1.163-GCC-6.4.0-2.28``                                       
``2.1.3``                      ``iccifort/2018.2.199-GCC-6.4.0-2.28``                                                                              
``3.0.0``                      ``GCC/7.2.0-2.29``                                                                                                  
``3.1.0``                      ``GCC/7.3.0-2.30``                                                                                                  
``3.1.1``                      ``GCC/7.3.0-2.30``, ``gcccuda/2018b``, ``iccifort/2018.3.222-GCC-7.3.0-2.30``                                       
``3.1.2``                      ``GCC/8.2.0-2.31.1``                                                                                                
``3.1.3``                      ``GCC/8.2.0-2.31.1``, ``gcccuda/2019a``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``                                   
``3.1.4``                      ``GCC/8.3.0``, ``GCC/8.3.0-2.32``, ``gcccuda/2019b``                                                                
``4.0.0``                      ``GCC/8.2.0-2.31.1``                                                                                                
``4.0.0``     ``-hpcx``        ``GCC/8.2.0-2.31.1``                                                                                                
``4.0.1``                      ``GCC/8.3.0-2.32``                                                                                                  
``system``                     ``GCC/system-2.29``                                                                                                 
==========    =============    ====================================================================================================================


.. _list_software_OpenNLP_636:

*OpenNLP*
+++++++++

The Apache OpenNLP library is a machine learning based toolkit for the processing of  natural language text.

*homepage*: https://opennlp.apache.org/

=========    ==========
version      toolchain 
=========    ==========
``1.8.1``    ``system``
=========    ==========


.. _list_software_OpenPGM_630:

*OpenPGM*
+++++++++

OpenPGM is an open source implementation of the Pragmatic General Multicast (PGM)  specification in RFC 3208 available at www.ietf.org. PGM is a reliable and scalable multicast protocol  that enables receivers to detect loss, request retransmission of lost data, or notify an application  of unrecoverable loss. PGM is a receiver-reliable protocol, which means the receiver is responsible  for ensuring all data is received, absolving the sender of reception responsibility.

*homepage*: http://code.google.com/p/openpgm/

===========    =============================================================================================================================================================================
version        toolchain                                                                                                                                                                    
===========    =============================================================================================================================================================================
``5.2.122``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
===========    =============================================================================================================================================================================


.. _list_software_openpyxl_895:

*openpyxl*
++++++++++

A Python library to read/write Excel 2010 xlsx/xlsm files

*homepage*: https://openpyxl.readthedocs.io

=========    =================
version      toolchain        
=========    =================
``2.6.2``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_OpenRefine_1003:

*OpenRefine*
++++++++++++

OpenRefine is a power tool that allows you to load data, understand it, clean it up, reconcile it,  and augment it with data coming from the web.

*homepage*: https://github.com/OpenRefine/OpenRefine

=======    ===================    ==========
version    versionsuffix          toolchain 
=======    ===================    ==========
``2.7``    ``-Java-1.8.0_144``    ``system``
=======    ===================    ==========


.. _list_software_OpenSSL_644:

*OpenSSL*
+++++++++

The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,  and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1)   protocols as well as a full-strength general purpose cryptography library. 

*homepage*: http://www.openssl.org/

==========    =====================================================================
version       toolchain                                                            
==========    =====================================================================
``1.0.1f``    ``GCC/4.8.2``                                                        
``1.0.1k``    ``GCC/4.9.2``                                                        
``1.0.1s``    ``foss/2016a``, ``intel/2016a``                                      
``1.0.2g``    ``GCCcore/4.9.3``                                                    
``1.0.2h``    ``foss/2016.04``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``1.1.0c``    ``GCC/5.4.0-2.26``                                                   
``1.1.0e``    ``intel/2016b``                                                      
``1.1.0h``    ``GCCcore/7.3.0``                                                    
``1.1.1b``    ``GCCcore/8.2.0``                                                    
``1.1.1d``    ``GCCcore/8.3.0``                                                    
==========    =====================================================================


.. _list_software_OPERA_375:

*OPERA*
+++++++

An optimal genome scaffolding program

*homepage*: https://sourceforge.net/projects/operasf

=========    ================    ===============================
version      versionsuffix       toolchain                      
=========    ================    ===============================
``2.0.6``    ``-Perl-5.28.0``    ``foss/2018b``, ``intel/2018b``
=========    ================    ===============================


.. _list_software_OptiType_830:

*OptiType*
++++++++++

OptiType is a novel HLA genotyping algorithm based on integer linear programming,  capable of producing accurate 4-digit HLA genotyping predictions from NGS data  by simultaneously selecting all major and minor HLA Class I alleles. 

*homepage*: https://github.com/FRED-2/OptiType

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.3.2``    ``-Python-2.7.15``    ``foss/2018b``
``1.3.2``    ``-Python-3.6.6``     ``foss/2018b``
=========    ==================    ==============


.. _list_software_OptiX_500:

*OptiX*
+++++++

OptiX is NVIDIA SDK for easy ray tracing performance.  It provides a simple framework for accessing the GPU’s massive ray tracing  power using state-of-the-art GPU algorithms.

*homepage*: https://developer.nvidia.com/optix

=========    ==================
version      toolchain         
=========    ==================
``3.8.0``    ``GNU/4.9.3-2.25``
``3.9.0``    ``GNU/4.9.3-2.25``
=========    ==================


.. _list_software_OR-Tools_735:

*OR-Tools*
++++++++++

Google Optimization Tools (a.k.a., OR-Tools) is an open-source, fast and portable software suite for solving combinatorial optimization problems.

*homepage*: https://developers.google.com/optimization/

=======    =================    ==============
version    versionsuffix        toolchain     
=======    =================    ==============
``7.1``    ``-Python-3.7.2``    ``foss/2019a``
=======    =================    ==============


.. _list_software_ORCA_293:

*ORCA*
++++++

ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry  with specific emphasis on spectroscopic properties of open-shell molecules.  It features a wide variety of standard quantum chemical methods ranging from semiempirical methods to DFT to single-  and multireference correlated ab initio methods.  It can also treat environmental and relativistic effects.

*homepage*: https://orcaforum.kofo.mpg.de

======================    ==================    ===============
version                   versionsuffix         toolchain      
======================    ==================    ===============
``3_0_2-linux_x86-64``    ``-OpenMPI-1.8.1``    ``system``     
``4.0.0.2``               ``-OpenMPI-2.0.2``    ``system``     
``4.0.1``                 ``-OpenMPI-2.0.2``    ``system``     
``4.1.0``                 ``-OpenMPI-3.1.3``    ``system``     
``4.2.0``                                       ``gompi/2019b``
======================    ==================    ===============


.. _list_software_OrfM_372:

*OrfM*
++++++

A simple and not slow open reading frame (ORF) caller.

*homepage*: https://github.com/wwood/OrfM

=========    ========================================
version      toolchain                               
=========    ========================================
``0.6.1``    ``foss/2016b``                          
``0.7.1``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
=========    ========================================


.. _list_software_OrthoFinder_1124:

*OrthoFinder*
+++++++++++++

OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics

*homepage*: https://github.com/davidemms/OrthoFinder

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.2.7``    ``-Python-2.7.14``    ``intel/2018a``
``2.3.3``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_OrthoMCL_744:

*OrthoMCL*
++++++++++

OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences.

*homepage*: http://orthomcl.org/

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``1.4``      ``-Perl-5.24.0``    ``intel/2016b``
``2.0.9``    ``-Perl-5.24.0``    ``intel/2016b``
=========    ================    ===============


.. _list_software_OSU-Micro-Benchmarks_1865:

*OSU-Micro-Benchmarks*
++++++++++++++++++++++

OSU Micro-Benchmarks

*homepage*: http://mvapich.cse.ohio-state.edu/benchmarks/

=========    ==============================
version      toolchain                     
=========    ==============================
``5.3.2``    ``foss/2016a``, ``foss/2017a``
``5.6.2``    ``gompi/2019a``               
=========    ==============================


.. _list_software_OTF2_283:

*OTF2*
++++++

The Open Trace Format 2 is a highly scalable, memory efficient event trace  data format plus support library. It is the new standard trace format for  Scalasca, Vampir, and TAU and is open for other tools.  

*homepage*: https://www.score-p.org

=======    ==============================
version    toolchain                     
=======    ==============================
``2.0``    ``foss/2016a``, ``foss/2017a``
``2.2``    ``GCCcore/8.2.0``             
=======    ==============================


.. _list_software_ownCloud_843:

*ownCloud*
++++++++++

The ownCloud Desktop Client is a tool to synchronize files from ownCloud Server with your computer.

*homepage*: https://github.com/owncloud/client

=========    ==============
version      toolchain     
=========    ==============
``2.4.3``    ``foss/2018b``
=========    ==============


.. _list_software_oxford_asl_1073:

*oxford_asl*
++++++++++++

A command line tool for quantification of perfusion from ASL data

*homepage*: https://github.com/ibme-qubic/oxford_asl

=========    ==========================    ===============
version      versionsuffix                 toolchain      
=========    ==========================    ===============
``3.9.6``    ``-centos7-Python-2.7.13``    ``intel/2017a``
=========    ==========================    ===============


.. _list_software_letter_p:

*P*
---


:ref:`list_software_p11-kit_583` - :ref:`list_software_p4est_496` - :ref:`list_software_p4vasp_606` - :ref:`list_software_p7zip_506` - :ref:`list_software_packmol_743` - :ref:`list_software_PAML_298` - :ref:`list_software_pandas_631` - :ref:`list_software_pandas-datareader_1713` - :ref:`list_software_PANDAseq_685` - :ref:`list_software_Pandoc_597` - :ref:`list_software_Pango_501` - :ref:`list_software_PAPI_298` - :ref:`list_software_parallel_845` - :ref:`list_software_parallel-fastq-dump_1916` - :ref:`list_software_ParallelIO_965` - :ref:`list_software_parasail_845` - :ref:`list_software_Paraver_721` - :ref:`list_software_ParaView_799` - :ref:`list_software_PARI-GP_496` - :ref:`list_software_ParmEd_569` - :ref:`list_software_ParMETIS_677` - :ref:`list_software_ParMGridGen_1040` - :ref:`list_software_pasta_537` - :ref:`list_software_PaStiX_569` - :ref:`list_software_patchelf_839` - :ref:`list_software_path.py_708` - :ref:`list_software_pbbam_514` - :ref:`list_software_pbcopper_859` - :ref:`list_software_pbdagcon_830` - :ref:`list_software_pbmm2_478` - :ref:`list_software_pbs_python_1094` - :ref:`list_software_PBSuite_668` - :ref:`list_software_PCAngsd_640` - :ref:`list_software_PCC_214` - :ref:`list_software_PCL_223` - :ref:`list_software_PCMSolver_859` - :ref:`list_software_PCRaster_772` - :ref:`list_software_PCRE_298` - :ref:`list_software_PCRE2_348` - :ref:`list_software_PDT_232` - :ref:`list_software_PEAR_296` - :ref:`list_software_Perl_403` - :ref:`list_software_perl-app-cpanminus_1820` - :ref:`list_software_Perl4-CoreLibs_1287` - :ref:`list_software_PETSc_415` - :ref:`list_software_petsc4py_828` - :ref:`list_software_pftoolsV3_912` - :ref:`list_software_pFUnit_598` - :ref:`list_software_PGDSpider_834` - :ref:`list_software_PGI_224` - :ref:`list_software_PHASE_369` - :ref:`list_software_PHAST_384` - :ref:`list_software_PheWAS_520` - :ref:`list_software_PhiPack_672` - :ref:`list_software_PHLAT_377` - :ref:`list_software_phono3py_832` - :ref:`list_software_phonopy_781` - :ref:`list_software_PHYLIP_470` - :ref:`list_software_PhyloBayes-MPI_1299` - :ref:`list_software_phylokit_884` - :ref:`list_software_phylonaut_996` - :ref:`list_software_PhyML_458` - :ref:`list_software_phyx_457` - :ref:`list_software_picard_627` - :ref:`list_software_pigz_442` - :ref:`list_software_PIL_229` - :ref:`list_software_PileOMeth_871` - :ref:`list_software_Pillow_631` - :ref:`list_software_Pillow-SIMD_977` - :ref:`list_software_Pilon_514` - :ref:`list_software_PIMS_313` - :ref:`list_software_Pindel_604` - :ref:`list_software_pip_329` - :ref:`list_software_Pisces_615` - :ref:`list_software_piSvM_495` - :ref:`list_software_piSvM-JSC_764` - :ref:`list_software_pixman_653` - :ref:`list_software_pizzly_690` - :ref:`list_software_pkg-config_997` - :ref:`list_software_pkgconfig_952` - :ref:`list_software_PlaScope_791` - :ref:`list_software_PLAST_388` - :ref:`list_software_Platanus_840` - :ref:`list_software_Platypus_866` - :ref:`list_software_plc_319` - :ref:`list_software_PLINK_382` - :ref:`list_software_PLINKSEQ_615` - :ref:`list_software_Ploticus_851` - :ref:`list_software_plotly_676` - :ref:`list_software_plotly.py_955` - :ref:`list_software_PLplot_603` - :ref:`list_software_PLUMED_455` - :ref:`list_software_PLY_245` - :ref:`list_software_PMIx_350` - :ref:`list_software_PnetCDF_612` - :ref:`list_software_pocl_430` - :ref:`list_software_pomkl_547` - :ref:`list_software_pompi_549` - :ref:`list_software_poppler_770` - :ref:`list_software_popt_451` - :ref:`list_software_Porechop_832` - :ref:`list_software_poretools_999` - :ref:`list_software_Postgres-XL_1064` - :ref:`list_software_PostgreSQL_980` - :ref:`list_software_POT_243` - :ref:`list_software_POV-Ray_590` - :ref:`list_software_PPfold_581` - :ref:`list_software_ppl_332` - :ref:`list_software_pplacer_743` - :ref:`list_software_pplpy_565` - :ref:`list_software_PRANK_380` - :ref:`list_software_PRC_229` - :ref:`list_software_preseq_656` - :ref:`list_software_Primer3_674` - :ref:`list_software_PRINSEQ_546` - :ref:`list_software_printproto_1121` - :ref:`list_software_PRISMS-PF_673` - :ref:`list_software_ProbABEL_679` - :ref:`list_software_prodigal_850` - :ref:`list_software_PROJ_315` - :ref:`list_software_ProjectQ_808` - :ref:`list_software_prokka_648` - :ref:`list_software_prompt-toolkit_1493` - :ref:`list_software_proovread_978` - :ref:`list_software_propy_570` - :ref:`list_software_Proteinortho_1293` - :ref:`list_software_protobuf_881` - :ref:`list_software_protobuf-python_1600` - :ref:`list_software_pscom_546` - :ref:`list_software_PSI_236` - :ref:`list_software_PSI4_288` - :ref:`list_software_psmc_435` - :ref:`list_software_psmpi_553` - :ref:`list_software_psmpi2_603` - :ref:`list_software_PSolver_715` - :ref:`list_software_PSORTb_506` - :ref:`list_software_psrecord_866` - :ref:`list_software_pstoedit_876` - :ref:`list_software_psutil_673` - :ref:`list_software_psycopg2_823` - :ref:`list_software_ptemcee_739` - :ref:`list_software_PTESFinder_916` - :ref:`list_software_pubtcrs_771` - :ref:`list_software_pullseq_774` - :ref:`list_software_py_233` - :ref:`list_software_py-cpuinfo_1034` - :ref:`list_software_PyAMG_414` - :ref:`list_software_PyAPS3_480` - :ref:`list_software_pybedtools_1093` - :ref:`list_software_pyBigWig_802` - :ref:`list_software_pybind11_744` - :ref:`list_software_PyCairo_695` - :ref:`list_software_PyCharm_692` - :ref:`list_software_pycma_538` - :ref:`list_software_PyCogent_809` - :ref:`list_software_PyCUDA_486` - :ref:`list_software_PyDatastream_1231` - :ref:`list_software_pydicom_757` - :ref:`list_software_pydlpoly_893` - :ref:`list_software_pyEGA3_489` - :ref:`list_software_pyenchant_970` - :ref:`list_software_PyFFmpeg_766` - :ref:`list_software_pyFFTW_544` - :ref:`list_software_pyfits_671` - :ref:`list_software_PyFMI_421` - :ref:`list_software_PyFR_353` - :ref:`list_software_pygccxml_871` - :ref:`list_software_pyGIMLi_635` - :ref:`list_software_Pygments_855` - :ref:`list_software_PyGObject_871` - :ref:`list_software_pygrib_653` - :ref:`list_software_PyGTK_431` - :ref:`list_software_PyGTS_439` - :ref:`list_software_PyGWAS_507` - :ref:`list_software_pyhdf_539` - :ref:`list_software_pyiron_673` - :ref:`list_software_Pyke3_460` - :ref:`list_software_Pylint_640` - :ref:`list_software_pymatgen_869` - :ref:`list_software_pymatgen-db_1112` - :ref:`list_software_pymbar_651` - :ref:`list_software_pymemcache_1052` - :ref:`list_software_PyNAST_511` - :ref:`list_software_Pyomo_532` - :ref:`list_software_PyOpenGL_750` - :ref:`list_software_pyplusplus_1137` - :ref:`list_software_pyproj_676` - :ref:`list_software_pyqstem_787` - :ref:`list_software_PyQt_398` - :ref:`list_software_PyQt5_451` - :ref:`list_software_PyQtGraph_896` - :ref:`list_software_PyRETIS_592` - :ref:`list_software_pyringe_766` - :ref:`list_software_Pyro4_478` - :ref:`list_software_Pysam_522` - :ref:`list_software_pyScaf_614` - :ref:`list_software_PySCF_421` - :ref:`list_software_pyshp_564` - :ref:`list_software_Pysolar_746` - :ref:`list_software_pysqlite_891` - :ref:`list_software_PyStan_607` - :ref:`list_software_pystran_785` - :ref:`list_software_PyTables_804` - :ref:`list_software_pytest_681` - :ref:`list_software_PYTHIA_463` - :ref:`list_software_Python_642` - :ref:`list_software_python-igraph_1354` - :ref:`list_software_python-Levenshtein_1876` - :ref:`list_software_python-parasail_1564` - :ref:`list_software_PyTorch_713` - :ref:`list_software_PyYAML_508` - :ref:`list_software_PyZMQ_449`


.. _list_software_p11-kit_583:

*p11-kit*
+++++++++

Provides a way to load and enumerate PKCS#11 modules.  Provides a standard configuration setup for installing  PKCS#11 modules in such a way that they're discoverable.  Also solves problems with coordinating the use of PKCS#11  by different components or libraries living in the same process.

*homepage*: http://p11-glue.freedesktop.org/p11-kit.html

==========    ======================================================================
version       toolchain                                                             
==========    ======================================================================
``0.23.2``    ``GCCcore/5.4.0``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``intel/2016a``
==========    ======================================================================


.. _list_software_p4est_496:

*p4est*
+++++++

p4est is a C library to manage a collection (a forest) of multiple connected adaptive quadtrees or octrees in parallel.

*homepage*: http://www.p4est.org

=======    ===============
version    toolchain      
=======    ===============
``2.2``    ``intel/2019a``
=======    ===============


.. _list_software_p4vasp_606:

*p4vasp*
++++++++

Visualization suite for VASP

*homepage*: http://www.p4vasp.at/

==========    ==================    ================================
version       versionsuffix         toolchain                       
==========    ==================    ================================
``0.3.29``    ``-Python-2.7.11``    ``intel/2016a``                 
``0.3.30``    ``-Python-2.7.14``    ``intel/2017b``, ``intel/2018a``
``0.3.30``    ``-Python-2.7.15``    ``foss/2018b``                  
==========    ==================    ================================


.. _list_software_p7zip_506:

*p7zip*
+++++++

p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip) for Unix.  7-Zip is a file archiver with highest compression ratio.

*homepage*: http://p7zip.sourceforge.net/

==========    ===========================================================================
version       toolchain                                                                  
==========    ===========================================================================
``9.38.1``    ``GCC/4.9.2``, ``system``                                                  
``16.02``     ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2018a``
==========    ===========================================================================


.. _list_software_packmol_743:

*packmol*
+++++++++

Packing Optimization for Molecular Dynamics Simulations

*homepage*: http://m3g.iqm.unicamp.br/packmol

==========    ===============================
version       toolchain                      
==========    ===============================
``16.103``    ``intel/2016a``                
``18.013``    ``foss/2018a``, ``intel/2018a``
==========    ===============================


.. _list_software_PAML_298:

*PAML*
++++++

PAML is a package of programs for phylogenetic   analyses of DNA or protein sequences using maximum likelihood.

*homepage*: http://abacus.gene.ucl.ac.uk/software/paml.html

========    ==========================================================
version     toolchain                                                 
========    ==========================================================
``4.9i``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
========    ==========================================================


.. _list_software_pandas_631:

*pandas*
++++++++

pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures  and data analysis tools for the Python programming language.

*homepage*: https://pypi.python.org/pypi/pandas/

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``0.18.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``0.18.0``    ``-Python-3.5.1``     ``foss/2016a``, ``intel/2016a``
``0.18.1``    ``-Python-2.7.12``    ``intel/2016b``                
``0.18.1``    ``-Python-3.5.2``     ``intel/2016b``                
``0.19.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``0.19.0``    ``-Python-3.5.2``     ``foss/2016b``, ``intel/2016b``
``0.19.1``    ``-Python-2.7.12``    ``intel/2016b``                
``0.19.1``    ``-Python-3.5.2``     ``intel/2016b``                
``0.20.1``    ``-Python-3.6.1``     ``intel/2017a``                
``0.21.0``    ``-Python-2.7.13``    ``intel/2017a``                
``0.21.0``    ``-Python-3.6.3``     ``intel/2017b``                
==========    ==================    ===============================


.. _list_software_pandas-datareader_1713:

*pandas-datareader*
+++++++++++++++++++

Up to date remote data access for pandas, works for multiple versions of pandas.

*homepage*: https://pypi.org/project/pandas-datareader

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.7.0``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_PANDAseq_685:

*PANDAseq*
++++++++++

PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded     in the sequence, and reconstruct an overlapping sequence.

*homepage*: https://github.com/neufeld/pandaseq

========    ==========================================================
version     toolchain                                                 
========    ==========================================================
``2.10``    ``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``
``2.11``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``          
========    ==========================================================


.. _list_software_Pandoc_597:

*Pandoc*
++++++++

If you need to convert files from one markup format into another, pandoc is your swiss-army knife

*homepage*: http://pandoc.org

=========    ==========
version      toolchain 
=========    ==========
``2.1.3``    ``system``
``2.5``      ``system``
=========    ==========


.. _list_software_Pango_501:

*Pango*
+++++++

Pango is a library for laying out and rendering of text, with an emphasis on internationalization. Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x.

*homepage*: http://www.pango.org/

===========    ==================================
version        toolchain                         
===========    ==================================
``1.39.0``     ``foss/2016a``, ``intel/2016a``   
``1.40.1``     ``foss/2016a``, ``intel/2016a``   
``1.40.3``     ``foss/2016b``, ``intel/2016b``   
``1.40.5``     ``intel/2017a``                   
``1.40.12``    ``intel/2017a``                   
``1.40.14``    ``foss/2017b``, ``intel/2017b``   
``1.41.0``     ``foss/2017b``, ``intel/2017b``   
``1.41.1``     ``foss/2018a``, ``intel/2018a``   
``1.42.4``     ``foss/2018b``, ``fosscuda/2018b``
``1.43.0``     ``GCCcore/8.2.0``                 
===========    ==================================


.. _list_software_PAPI_298:

*PAPI*
++++++

PAPI provides the tool designer and application engineer with a consistent  interface and methodology for use of the performance counter hardware found  in most major microprocessors. PAPI enables software engineers to see, in near  real time, the relation between software performance and processor events.  In addition Component PAPI provides access to a collection of components  that expose performance measurement opportunites across the hardware and  software stack. 

*homepage*: http://icl.cs.utk.edu/projects/papi/

=========    ====================================
version      toolchain                           
=========    ====================================
``5.4.3``    ``foss/2016a``                      
``5.5.1``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``
``5.6.0``    ``GCCcore/6.4.0``                   
``5.7.0``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=========    ====================================


.. _list_software_parallel_845:

*parallel*
++++++++++

parallel: Build and execute shell commands in parallel

*homepage*: https://savannah.gnu.org/projects/parallel/

============    ===============================
version         toolchain                      
============    ===============================
``20141122``    ``GCC/4.9.2``                  
``20150322``    ``GCC/4.9.2``                  
``20150822``    ``GCC/4.9.2``                  
``20160622``    ``foss/2016a``                 
``20170822``    ``intel/2017a``                
``20171022``    ``intel/2017b``                
``20171122``    ``foss/2017b``, ``intel/2017b``
``20180422``    ``intel/2018a``                
``20180822``    ``foss/2018b``                 
``20181222``    ``intel/2018b``                
``20190222``    ``GCCcore/7.3.0``              
``20190622``    ``GCCcore/8.2.0``              
``20190922``    ``GCCcore/8.3.0``              
============    ===============================


.. _list_software_parallel-fastq-dump_1916:

*parallel-fastq-dump*
+++++++++++++++++++++

parallel fastq-dump wrapper

*homepage*: https://github.com/rvalieris/parallel-fastq-dump

=========    =================    =================
version      versionsuffix        toolchain        
=========    =================    =================
``0.6.5``    ``-Python-3.7.2``    ``GCCcore/8.2.0``
=========    =================    =================


.. _list_software_ParallelIO_965:

*ParallelIO*
++++++++++++

A high-level Parallel I/O Library for structured grid applications

*homepage*: https://github.com/NCAR/ParallelIO

==========    ===============
version       toolchain      
==========    ===============
``2.2.2a``    ``intel/2017a``
==========    ===============


.. _list_software_parasail_845:

*parasail*
++++++++++

parasail is a SIMD C (C99) library containing implementations  of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global  pairwise sequence alignment algorithms. 

*homepage*: https://github.com/jeffdaily/parasail

=======    ===============
version    toolchain      
=======    ===============
``2.2``    ``intel/2018a``
``2.4``    ``foss/2018b`` 
=======    ===============


.. _list_software_Paraver_721:

*Paraver*
+++++++++

A very powerful performance visualization and analysis tool based on  traces that can be used to analyse any information that is expressed on its input trace format.  Traces for parallel MPI, OpenMP and other programs can be genereated with Extrae.

*homepage*: https://tools.bsc.es/paraver

=========    ==============
version      toolchain     
=========    ==============
``4.8.1``    ``foss/2019a``
=========    ==============


.. _list_software_ParaView_799:

*ParaView*
++++++++++

ParaView is a scientific parallel visualizer.

*homepage*: https://www.paraview.org

=========    ======================    =================================================================================================
version      versionsuffix             toolchain                                                                                        
=========    ======================    =================================================================================================
``4.4.0``                              ``foss/2016a``, ``intel/2016a``                                                                  
``4.4.0``    ``-mpi``                  ``gimkl/2.11.5``                                                                                 
``5.1.2``    ``-mpi``                  ``foss/2016b``, ``intel/2016b``                                                                  
``5.2.0``    ``-mpi``                  ``foss/2016b``, ``intel/2017a``                                                                  
``5.3.0``    ``-mpi``                  ``foss/2016b``                                                                                   
``5.4.1``    ``-mpi``                  ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``5.5.2``    ``-Python-2.7.15-mpi``    ``foss/2018b``                                                                                   
``5.6.2``    ``-Python-3.7.4-mpi``     ``foss/2019b``                                                                                   
=========    ======================    =================================================================================================


.. _list_software_PARI-GP_496:

*PARI-GP*
+++++++++

PARI/GP is a widely used computer algebra system designed for fast computations in number theory  (factorizations, algebraic  number theory, elliptic curves...), but also contains a large number of other useful  functions to compute with mathematical entities such as matrices, polynomials, power series, algebraic numbers etc.,  and a lot of transcendental functions. PARI is  also available as a C library to allow for faster computations. 

*homepage*: http://pari.math.u-bordeaux.fr

=========    ==============
version      toolchain     
=========    ==============
``2.7.6``    ``foss/2016a``
=========    ==============


.. _list_software_ParmEd_569:

*ParmEd*
++++++++

ParmEd is a general tool for aiding in investigations of biomolecular systems using popular molecular simulation packages, like Amber, CHARMM, and OpenMM written in Python.

*homepage*: http://parmed.github.io/ParmEd/html/index.html

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``2.7.3``    ``-Python-3.6.3``    ``intel/2017b``
=========    =================    ===============


.. _list_software_ParMETIS_677:

*ParMETIS*
++++++++++

ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning  unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the  functionality provided by METIS and includes routines that are especially suited for parallel AMR computations and  large scale numerical simulations. The algorithms implemented in ParMETIS are based on the parallel multilevel k-way  graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning schemes.

*homepage*: http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview

=========    ========================================================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                                                               
=========    ========================================================================================================================================================================================================================================================
``4.0.3``    ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``gimkl/2.11.5``, ``gompi/2019a``, ``iimpi/2019a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
=========    ========================================================================================================================================================================================================================================================


.. _list_software_ParMGridGen_1040:

*ParMGridGen*
+++++++++++++

ParMGridGen is an MPI-based parallel library that is based on the serial package MGridGen,  that implements (serial) algorithms for obtaining a sequence of successive coarse grids that are well-suited  for geometric multigrid methods.

*homepage*: http://www-users.cs.umn.edu/~moulitsa/software.html

=======    ==================================================
version    toolchain                                         
=======    ==================================================
``1.0``    ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017a``
=======    ==================================================


.. _list_software_pasta_537:

*pasta*
+++++++

PASTA (Practical Alignment using SATe and Transitivity)

*homepage*: https://github.com/smirarab/pasta

=========    =================    ====================
version      versionsuffix        toolchain           
=========    =================    ====================
``1.8.5``    ``-Python-3.7.2``    ``GCC/8.2.0-2.31.1``
=========    =================    ====================


.. _list_software_PaStiX_569:

*PaStiX*
++++++++

PaStiX (Parallel Sparse matriX package) is a scientific library that provides a high performance  parallel solver for very large sparse linear systems based on direct methods.

*homepage*: http://pastix.gforge.inria.fr/

=========    ==============
version      toolchain     
=========    ==============
``5.2.3``    ``foss/2017b``
=========    ==============


.. _list_software_patchelf_839:

*patchelf*
++++++++++

PatchELF is a small utility to modify the dynamic linker and RPATH of ELF executables.

*homepage*: http://nixos.org/patchelf.html

========    =================================
version     toolchain                        
========    =================================
``0.8``     ``GNU/4.9.3-2.25``               
``0.9``     ``GCCcore/6.4.0``, ``foss/2016a``
``0.10``    ``GCCcore/7.2.0``                
========    =================================


.. _list_software_path.py_708:

*path.py*
+++++++++

path.py is a Python library implementing path objects as first-class entities,  allowing common operations on files to be invoked on those path objects directly.

*homepage*: https://github.com/jaraco/path.py

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``8.2.1``    ``-Python-2.7.11``    ``foss/2016a``                 
``8.2.1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``8.2.1``    ``-Python-3.5.1``     ``foss/2016a``                 
``8.2.1``    ``-Python-3.5.2``     ``intel/2016b``                
``10.1``     ``-Python-2.7.12``    ``intel/2016b``                
=========    ==================    ===============================


.. _list_software_pbbam_514:

*pbbam*
+++++++

The pbbam software package provides components to create, query, & edit PacBio BAM files  and associated indices.

*homepage*: http://pbbam.readthedocs.io/

============    ===============
version         toolchain      
============    ===============
``1.0.6``       ``gompi/2019a``
``20170508``    ``intel/2017a``
============    ===============


.. _list_software_pbcopper_859:

*pbcopper*
++++++++++

The pbcopper library provides a suite of data structures, algorithms,  and utilities for C++ applications.

*homepage*: https://github.com/PacificBiosciences/pbcopper

=========    ===============
version      toolchain      
=========    ===============
``1.3.0``    ``gompi/2019a``
=========    ===============


.. _list_software_pbdagcon_830:

*pbdagcon*
++++++++++

pbdagcon is a tool that implements DAGCon (Directed Acyclic Graph Consensus) which is a sequence  consensus algorithm based on using directed acyclic graphs to encode multiple sequence alignment.

*homepage*: https://github.com/PacificBiosciences/pbdagcon

============    ===============
version         toolchain      
============    ===============
``20170330``    ``intel/2017a``
============    ===============


.. _list_software_pbmm2_478:

*pbmm2*
+++++++

A minimap2 frontend for PacBio native data formats

*homepage*: https://github.com/PacificBiosciences/pbmm2

=========    ===============
version      toolchain      
=========    ===============
``1.1.0``    ``gompi/2019a``
=========    ===============


.. _list_software_pbs_python_1094:

*pbs_python*
++++++++++++

The pbs_python package is a wrapper class for the Torque C library. With this package you now can  write utilities/extensions in Python instead of C. We developed this package because we want to replace xpbsmon by  an ascii version named pbsmon. PBSQuery is also included in this package. This is a python module build on top of  the pbs python module to simplify querying the batch server, eg: how many jobs, how many nodes, ...

*homepage*: https://oss.trac.surfsara.nl/pbs_python

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``4.6.0``                          ``system``     
``4.6.0``    ``-Python-2.7.11``    ``intel/2016a``
``4.6.0``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_PBSuite_668:

*PBSuite*
+++++++++

PBJelly is a highly automated pipeline that aligns long sequencing reads (such as PacBio RS reads or  long 454 reads in fasta format) to high-confidence draft assembles.

*homepage*: https://sourceforge.net/p/pb-jelly/wiki/Home/

===========    ==================    ===============
version        versionsuffix         toolchain      
===========    ==================    ===============
``15.8.24``    ``-Python-2.7.12``    ``intel/2016b``
===========    ==================    ===============


.. _list_software_PCAngsd_640:

*PCAngsd*
+++++++++

PCAngsd, which estimates the covariance matrix for low depth  NGS data in an iterative procedure based on genotype likelihoods and is able  to perform multiple population genetic analyses in heterogeneous populations.

*homepage*: http://www.popgen.dk/software/index.php/PCAngsd

========    ==================    ==============
version     versionsuffix         toolchain     
========    ==================    ==============
``0.97``    ``-Python-2.7.14``    ``foss/2018a``
========    ==================    ==============


.. _list_software_PCC_214:

*PCC*
+++++

The compiler is based on the original Portable C Compiler by S. C. Johnson,  written in the late 70's. About 50% of the frontend code and 80% of the backend code has been modified.

*homepage*: http://pcc.ludd.ltu.se/

============    ==========
version         toolchain 
============    ==========
``20131024``    ``system``
============    ==========


.. _list_software_PCL_223:

*PCL*
+++++

The Point Cloud Library (PCL) is a standalone, large scale, open project for 2D/3D image and  point cloud processing.

*homepage*: http://pointclouds.org/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.7.2``    ``-Python-2.7.11``    ``intel/2016a``
``1.8.1``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_PCMSolver_859:

*PCMSolver*
+++++++++++

An API for the Polarizable Continuum Model.

*homepage*: https://pcmsolver.readthedocs.org

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``1.1.4``       ``-Python-2.7.11``    ``intel/2016a``
``1.2.3``       ``-Python-3.6.6``     ``foss/2018b`` 
``1.2.3``       ``-Python-3.7.2``     ``gompi/2019a``
``20160205``    ``-Python-2.7.11``    ``intel/2016a``
============    ==================    ===============


.. _list_software_PCRaster_772:

*PCRaster*
++++++++++

PCRaster Is a collection of software targeted at the development  and deployment of spatio-temporal environmental models.

*homepage*: http://pcraster.geo.uu.nl/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``4.1.0``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_PCRE_298:

*PCRE*
++++++

The PCRE library is a set of functions that implement regular expression  pattern matching using the same syntax and semantics as Perl 5. 

*homepage*: https://www.pcre.org/

========    ==================================================================================
version     toolchain                                                                         
========    ==================================================================================
``8.38``    ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``8.39``    ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``                                
``8.40``    ``GCCcore/6.3.0``, ``gimkl/2017a``, ``intel/2016b``, ``intel/2017a``              
``8.41``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                           
``8.43``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                              
========    ==================================================================================


.. _list_software_PCRE2_348:

*PCRE2*
+++++++

The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax  and semantics as Perl 5. 

*homepage*: https://www.pcre.org/

=========    ====================================
version      toolchain                           
=========    ====================================
``10.21``    ``foss/2016a``                      
``10.31``    ``foss/2018b``                      
``10.33``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
=========    ====================================


.. _list_software_PDT_232:

*PDT*
+++++

Program Database Toolkit (PDT) is a framework for analyzing source code  written in several programming languages and for making rich program  knowledge accessible to developers of static and dynamic analysis tools.  PDT implements a standard program representation, the program database  (PDB), that can be accessed in a uniform way through a class library  supporting common PDB operations. 

*homepage*: http://www.cs.uoregon.edu/research/pdt/

========    =================
version     toolchain        
========    =================
``3.22``    ``foss/2016a``   
``3.25``    ``GCCcore/8.2.0``
========    =================


.. _list_software_PEAR_296:

*PEAR*
++++++

PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger.   It is fully parallelized and can run with as low as just a few kilobytes of memory.

*homepage*: http://sco.h-its.org/exelixis/web/software/pear/

==========    ==========================================================
version       toolchain                                                 
==========    ==========================================================
``0.9.8``     ``foss/2016b``, ``intel/2016b``                           
``0.9.10``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``0.9.11``    ``GCCcore/7.3.0``, ``foss/2018a``                         
==========    ==========================================================


.. _list_software_Perl_403:

*Perl*
++++++

Larry Wall's Practical Extraction and Report Language

*homepage*: https://www.perl.org/

==========    =============    =========================================================================================
version       versionsuffix    toolchain                                                                                
==========    =============    =========================================================================================
``5.20.1``    ``-bare``        ``GCC/4.8.2``, ``GCC/4.9.2``                                                             
``5.20.2``    ``-bare``        ``GCC/4.9.2``                                                                            
``5.20.3``                     ``foss/2016a``, ``intel/2016a``                                                          
``5.22.0``    ``-bare``        ``GCC/4.9.2``                                                                            
``5.22.1``                     ``foss/2016a``, ``foss/2016b``, ``intel/2016a``                                          
``5.22.1``    ``-bare``        ``foss/2016a``, ``intel/2016a``                                                          
``5.22.2``                     ``intel/2016a``                                                                          
``5.24.0``                     ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``5.24.0``    ``-bare``        ``foss/2016b``                                                                           
``5.24.1``                     ``GCCcore/6.3.0``, ``foss/2017a``, ``intel/2017a``                                       
``5.26.0``                     ``GCCcore/6.4.0``, ``foss/2017b``, ``intel/2017b``, ``intel/2018.00``, ``intel/2018.01`` 
``5.26.1``                     ``GCCcore/6.4.0``, ``foss/2018a``                                                        
``5.26.1``    ``-bare``        ``foss/2018a``                                                                           
``5.28.0``                     ``GCCcore/7.3.0``                                                                        
``5.28.1``                     ``GCCcore/8.2.0``                                                                        
``5.30.0``                     ``GCCcore/8.3.0``                                                                        
==========    =============    =========================================================================================


.. _list_software_perl-app-cpanminus_1820:

*perl-app-cpanminus*
++++++++++++++++++++

cpanm - get, unpack build and install modules from CPAN 

*homepage*: https://github.com/miyagawa/cpanminus

==========    ==========
version       toolchain 
==========    ==========
``1.7039``    ``system``
==========    ==========


.. _list_software_Perl4-CoreLibs_1287:

*Perl4-CoreLibs*
++++++++++++++++

Libraries historically supplied with Perl 4

*homepage*: https://metacpan.org/pod/Perl4::CoreLibs

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``0.003``    ``-Perl-5.24.1``    ``intel/2017a``
=========    ================    ===============


.. _list_software_PETSc_415:

*PETSc*
+++++++

PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the  scalable (parallel) solution of scientific applications modeled by partial differential equations.

*homepage*: http://www.mcs.anl.gov/petsc

==========    ====================    ===============================
version       versionsuffix           toolchain                      
==========    ====================    ===============================
``3.7.2``     ``-Python-2.7.11``      ``intel/2016a``                
``3.7.3``     ``-Python-2.7.11``      ``foss/2016a``                 
``3.7.5``     ``-downloaded-deps``    ``intel/2016b``                
``3.8.3``     ``-downloaded-deps``    ``foss/2017b``                 
``3.9.1``     ``-downloaded-deps``    ``foss/2018a``                 
``3.9.3``                             ``foss/2018a``, ``intel/2018a``
``3.11.0``    ``-downloaded-deps``    ``foss/2018b``                 
``3.11.1``                            ``foss/2019a``, ``intel/2019a``
==========    ====================    ===============================


.. _list_software_petsc4py_828:

*petsc4py*
++++++++++

petsc4py are Python bindings for PETSc, the Portable, Extensible Toolchain for Scientific Computation.

*homepage*: https://bitbucket.org/petsc/petsc4py

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``3.9.1``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_pftoolsV3_912:

*pftoolsV3*
+++++++++++

The pftools package contains all the software necessary to build protein and DNA generalized profiles and use  them to scan and align sequences, and search databases. 

*homepage*: http://web.expasy.org/pftools/

============    ==============
version         toolchain     
============    ==============
``20160324``    ``foss/2016a``
============    ==============


.. _list_software_pFUnit_598:

*pFUnit*
++++++++

pFUnit is a unit testing framework enabling JUnit-like testing of serial  and MPI-parallel software written in Fortran.

*homepage*: http://pfunit.sourceforge.net

=========    ===============
version      toolchain      
=========    ===============
``3.2.9``    ``gompi/2018b``
=========    ===============


.. _list_software_PGDSpider_834:

*PGDSpider*
+++++++++++

An automated data conversion tool for connecting population genetics and genomics programs

*homepage*: http://cmpg.unibe.ch/software/PGDSpider/

===========    ==================    ==========
version        versionsuffix         toolchain 
===========    ==================    ==========
``2.1.0.3``    ``-Java-1.7.0_80``    ``system``
===========    ==================    ==========


.. _list_software_PGI_224:

*PGI*
+++++

C, C++ and Fortran compilers from The Portland Group - PGI

*homepage*: https://www.pgroup.com/

=========    =======================    ==========
version      versionsuffix              toolchain 
=========    =======================    ==========
``15.7``     ``-GNU-4.9.2-2.25``        ``system``
``15.7``     ``-GNU-4.9.3-2.25``        ``system``
``15.10``    ``-GCC-4.9.3-2.25``        ``system``
``16.1``     ``-CDK-GCC-4.9.2-2.25``    ``system``
``16.3``     ``-GCC-4.9.3-2.25``        ``system``
``16.4``     ``-GCC-5.3.0-2.26``        ``system``
``16.7``     ``-GCC-5.4.0-2.26``        ``system``
``16.10``    ``-GCC-5.4.0-2.26``        ``system``
``17.1``     ``-GCC-6.3.0-2.27``        ``system``
``17.3``     ``-GCC-6.3.0-2.28``        ``system``
``17.4``     ``-GCC-6.4.0-2.28``        ``system``
``17.10``    ``-GCC-6.4.0-2.28``        ``system``
``18.1``     ``-GCC-7.2.0-2.29``        ``system``
``18.4``     ``-GCC-6.4.0-2.28``        ``system``
``18.7``     ``-GCC-7.3.0-2.30``        ``system``
``18.10``    ``-GCC-6.4.0-2.28``        ``system``
``19.1``     ``-GCC-8.2.0-2.31.1``      ``system``
``19.4``     ``-GCC-8.2.0-2.31.1``      ``system``
``19.7``     ``-GCC-8.3.0-2.32``        ``system``
=========    =======================    ==========


.. _list_software_PHASE_369:

*PHASE*
+++++++

The program PHASE implements a Bayesian statistical  method for reconstructing haplotypes from population genotype data. Documentation: http://stephenslab.uchicago.edu/assets/software/phase/instruct2.1.pdf

*homepage*: http://stephenslab.uchicago.edu/phase/download.html

=========    ==========
version      toolchain 
=========    ==========
``2.1.1``    ``system``
=========    ==========


.. _list_software_PHAST_384:

*PHAST*
+++++++

PHAST is a freely available software package for comparative and evolutionary genomics.

*homepage*: http://compgen.cshl.edu/phast/

=======    ==========================================================
version    toolchain                                                 
=======    ==========================================================
``1.4``    ``intel/2017a``                                           
``1.5``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
=======    ==========================================================


.. _list_software_PheWAS_520:

*PheWAS*
++++++++

Provides an accessible R interface to the phenome wide association study.

*homepage*: https://www.vumc.org/cpm/center-precision-medicine-blog/phewas-r-package

============    =============    ===============================
version         versionsuffix    toolchain                      
============    =============    ===============================
``0.12``        ``-R-3.3.3``     ``foss/2016b``, ``intel/2016b``
``0.99.5-2``    ``-R-3.6.0``     ``foss/2019a``, ``intel/2019a``
============    =============    ===============================


.. _list_software_PhiPack_672:

*PhiPack*
+++++++++

The PhiPack software package implements (in C) a few tests for recombination and can produce refined incompatibility matrices as well.

*homepage*: http://www.maths.otago.ac.nz/~dbryant/software.html

==============    ========================================
version           toolchain                               
==============    ========================================
``2016.06.14``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
==============    ========================================


.. _list_software_PHLAT_377:

*PHLAT*
+++++++

PHLAT is a bioinformatics algorithm that offers HLA typing at four-digit resolution (or higher)  using genome-wide transcriptome and exome sequencing data over a wide range of read lengths and sequencing depths.

*homepage*: https://sites.google.com/site/phlatfortype/

=======    ==================    ==============
version    versionsuffix         toolchain     
=======    ==================    ==============
``1.1``    ``-Python-2.7.15``    ``foss/2018b``
=======    ==================    ==============


.. _list_software_phono3py_832:

*phono3py*
++++++++++

phono3py calculates phonon-phonon interaction and related properties using the supercell approach.

*homepage*: http://atztogo.github.io/phono3py/

=============    ==================    ===============================
version          versionsuffix         toolchain                      
=============    ==================    ===============================
``1.12.5.35``    ``-Python-2.7.14``    ``intel/2017b``                
``1.12.7.55``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``
=============    ==================    ===============================


.. _list_software_phonopy_781:

*phonopy*
+++++++++

Phonopy is an open source package of phonon calculations based on the supercell approach.

*homepage*: http://atztogo.github.io/phonopy/

=============    ==================    ===============================
version          versionsuffix         toolchain                      
=============    ==================    ===============================
``1.10.1``       ``-Python-2.7.11``    ``intel/2016a``                
``1.12.2.20``    ``-Python-2.7.14``    ``intel/2017b``                
``1.12.6.66``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``
``1.13.0.64``    ``-Python-2.7.14``    ``intel/2018a``                
``1.14.2``       ``-Python-2.7.15``    ``intel/2018b``                
``2.0.0``        ``-Python-2.7.14``    ``intel/2018a``                
``2.2.0``        ``-Python-3.7.2``     ``intel/2019a``                
=============    ==================    ===============================


.. _list_software_PHYLIP_470:

*PHYLIP*
++++++++

PHYLIP is a free package of programs for inferring phylogenies.

*homepage*: http://evolution.genetics.washington.edu/phylip

=========    ==========================================================
version      toolchain                                                 
=========    ==========================================================
``3.696``    ``foss/2016a``, ``intel/2016a``                           
``3.697``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
=========    ==========================================================


.. _list_software_PhyloBayes-MPI_1299:

*PhyloBayes-MPI*
++++++++++++++++

A Bayesian software for phylogenetic reconstruction using mixture models

*homepage*: https://github.com/bayesiancook/pbmpi

============    ===============
version         toolchain      
============    ===============
``20161021``    ``intel/2016b``
============    ===============


.. _list_software_phylokit_884:

*phylokit*
++++++++++

C++ library for high performance phylogenetics

*homepage*: https://github.com/pranjalv123/phylokit

=======    ====================
version    toolchain           
=======    ====================
``1.0``    ``GCC/8.2.0-2.31.1``
=======    ====================


.. _list_software_phylonaut_996:

*phylonaut*
+++++++++++

Dynamic programming for phylogenetics applications

*homepage*: https://github.com/pranjalv123/phylonaut

============    ===============
version         toolchain      
============    ===============
``20190626``    ``gompi/2019a``
============    ===============


.. _list_software_PhyML_458:

*PhyML*
+++++++

Phylogenetic estimation using (Maximum) Likelihood

*homepage*: https://github.com/stephaneguindon/phyml

================    ==============
version             toolchain     
================    ==============
``3.3.20190321``    ``foss/2018b``
================    ==============


.. _list_software_phyx_457:

*phyx*
++++++

phyx performs phylogenetics analyses on trees and sequences.

*homepage*: https://github.com/FePhyFoFum/phyx

========    ==============
version     toolchain     
========    ==============
``1.01``    ``foss/2019a``
========    ==============


.. _list_software_picard_627:

*picard*
++++++++

A set of tools (in Java) for working with next generation sequencing data in the BAM format.

*homepage*: http://sourceforge.net/projects/picard

===========    ===================    ==========
version        versionsuffix          toolchain 
===========    ===================    ==========
``1.39``                              ``system``
``1.100``                             ``system``
``1.109``                             ``system``
``1.119``                             ``system``
``1.119``      ``-Java-1.7.0_80``     ``system``
``1.120``      ``-Java-1.8.0_66``     ``system``
``1.141``      ``-Java-1.8.0_74``     ``system``
``2.0.1``      ``-Java-1.8.0_66``     ``system``
``2.1.0``                             ``system``
``2.1.0``      ``-Java-1.8.0_74``     ``system``
``2.1.1``      ``-Java-1.8.0_112``    ``system``
``2.1.1``      ``-Java-1.8.0_74``     ``system``
``2.2.4``      ``-Java-1.8.0_92``     ``system``
``2.6.0``      ``-Java-1.8.0_131``    ``system``
``2.10.1``     ``-Java-1.8.0_131``    ``system``
``2.18.5``     ``-Java-1.8.0_162``    ``system``
``2.18.11``    ``-Java-1.8.0_162``    ``system``
``2.18.14``    ``-Java-1.8``          ``system``
``2.18.17``    ``-Java-1.8``          ``system``
``2.18.27``    ``-Java-1.8``          ``system``
``2.21.1``     ``-Java-11``           ``system``
===========    ===================    ==========


.. _list_software_pigz_442:

*pigz*
++++++

pigz, which stands for parallel implementation of gzip, is a fully   functional replacement for gzip that exploits multiple processors and multiple cores   to the hilt when compressing data. pigz was written by Mark Adler, and uses the zlib   and pthread libraries. 

*homepage*: https://zlib.net/pigz/

=========    =======================================================================
version      toolchain                                                              
=========    =======================================================================
``2.3.3``    ``foss/2016b``                                                         
``2.3.4``    ``GCCcore/6.4.0``                                                      
``2.4``      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2018a``
=========    =======================================================================


.. _list_software_PIL_229:

*PIL*
+++++

The Python Imaging Library (PIL) adds image processing capabilities to your Python interpreter.  This library supports many file formats, and provides powerful image processing and graphics capabilities.

*homepage*: http://www.pythonware.com/products/pil

=========    =================================    ===============================
version      versionsuffix                        toolchain                      
=========    =================================    ===============================
``1.1.7``    ``-Python-2.7.11``                   ``intel/2016a``                
``1.1.7``    ``-Python-2.7.11-freetype-2.6.3``    ``intel/2016a``                
``1.1.7``    ``-Python-2.7.12``                   ``foss/2016b``, ``intel/2016b``
``1.1.7``    ``-Python-2.7.13``                   ``intel/2017a``                
``1.1.7``    ``-Python-2.7.15``                   ``foss/2019a``                 
=========    =================================    ===============================


.. _list_software_PileOMeth_871:

*PileOMeth*
+++++++++++

PileOMeth processes a coordinate-sorted and indexed BAM or CRAM file containing some form of BS-seq alignments.  PileOMeth extracts per-base methylation metrics from them.  PileOMeth requires an indexed fasta file containing the reference genome as well. 

*homepage*: https://github.com/dpryan79/PileOMeth

==========    ==============
version       toolchain     
==========    ==============
``0.1.11``    ``foss/2016b``
==========    ==============


.. _list_software_Pillow_631:

*Pillow*
++++++++

Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.

*homepage*: https://pillow.readthedocs.org/

=========    =================================    ===================================================
version      versionsuffix                        toolchain                                          
=========    =================================    ===================================================
``3.2.0``    ``-Python-2.7.11``                   ``intel/2016a``                                    
``3.2.0``    ``-Python-2.7.11-freetype-2.6.3``    ``foss/2016a``, ``intel/2016a``                    
``3.4.2``    ``-Python-2.7.12``                   ``intel/2016b``                                    
``3.4.2``    ``-Python-3.5.2``                    ``foss/2016b``, ``intel/2016b``                    
``3.4.2``    ``-Python-3.5.2-freetype-2.6.5``     ``intel/2016b``                                    
``4.1.0``    ``-Python-2.7.13``                   ``intel/2017a``                                    
``4.1.1``    ``-Python-3.6.1``                    ``intel/2017a``                                    
``4.2.1``    ``-Python-3.6.1``                    ``intel/2017a``                                    
``4.3.0``    ``-Python-2.7.13``                   ``intel/2017a``                                    
``4.3.0``    ``-Python-2.7.14``                   ``intel/2017b``                                    
``4.3.0``    ``-Python-3.6.3``                    ``foss/2017b``                                     
``5.0.0``    ``-Python-2.7.14``                   ``foss/2017b``, ``intel/2017b``, ``intel/2018a``   
``5.0.0``    ``-Python-3.6.3``                    ``foss/2017b``, ``intel/2017b``                    
``5.0.0``    ``-Python-3.6.4``                    ``foss/2018a``, ``intel/2018a``                    
``5.3.0``    ``-Python-2.7.15``                   ``foss/2018b``                                     
``5.3.0``    ``-Python-3.6.6``                    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``6.0.0``                                         ``GCCcore/8.2.0``                                  
``6.2.1``                                         ``GCCcore/8.3.0``                                  
=========    =================================    ===================================================


.. _list_software_Pillow-SIMD_977:

*Pillow-SIMD*
+++++++++++++

Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.  PIL is the Python Imaging Library by Fredrik Lundh and Contributors.

*homepage*: https://github.com/uploadcare/pillow-simd

===============    =================    ===================================================
version            versionsuffix        toolchain                                          
===============    =================    ===================================================
``5.0.0``          ``-Python-3.6.4``    ``foss/2018a``, ``intel/2018a``                    
``5.3.0.post0``    ``-Python-3.6.6``    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``6.0.x.post0``                         ``GCCcore/8.2.0``                                  
===============    =================    ===================================================


.. _list_software_Pilon_514:

*Pilon*
+++++++

Pilon is an automated genome assembly improvement and variant detection tool

*homepage*: https://github.com/broadinstitute/pilon

========    ===================    ==========
version     versionsuffix          toolchain 
========    ===================    ==========
``1.22``    ``-Java-1.8``          ``system``
``1.22``    ``-Java-1.8.0_162``    ``system``
``1.23``    ``-Java-1.8``          ``system``
``1.23``    ``-Java-11``           ``system``
========    ===================    ==========


.. _list_software_PIMS_313:

*PIMS*
++++++

PIMS is a lazy-loading interface to sequential data with numpy-like slicing.

*homepage*: http://soft-matter.github.io/pims

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.4.1``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_Pindel_604:

*Pindel*
++++++++

Pindel can detect breakpoints of large deletions, medium sized  insertions, inversions, tandem duplications and other structural variants at single-based  resolution from next-gen sequence data. It uses a pattern growth approach to identify the  breakpoints of these variants from paired-end short reads. 

*homepage*: http://gmt.genome.wustl.edu/packages/pindel/

====================    ==================
version                 toolchain         
====================    ==================
``0.2.5b8``             ``foss/2016b``    
``0.2.5b9-20170508``    ``GCC/6.4.0-2.28``
====================    ==================


.. _list_software_pip_329:

*pip*
+++++

The PyPA recommended tool for installing Python packages.

*homepage*: https://pip.pypa.io

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``8.0.2``    ``-Python-2.7.11``    ``intel/2016a``                
``8.1.2``    ``-Python-2.7.11``    ``foss/2016a``                 
``8.1.2``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
=========    ==================    ===============================


.. _list_software_Pisces_615:

*Pisces*
++++++++

Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows.

*homepage*: https://github.com/Illumina/Pisces

============    =================
version         toolchain        
============    =================
``5.2.7.47``    ``GCCcore/6.4.0``
============    =================


.. _list_software_piSvM_495:

*piSvM*
+++++++

piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system.

*homepage*: http://pisvm.sourceforge.net/

=======    ===============
version    toolchain      
=======    ===============
``1.3``    ``intel/2017b``
=======    ===============


.. _list_software_piSvM-JSC_764:

*piSvM-JSC*
+++++++++++

piSvM is a parallel implementation of the Support Vector Machine (SVM) algorithm that allows  efficient training and testing on a multiprocessor system. This version is a fork of the original PiSvM to increase scalability.

*homepage*: https://github.com/mricherzhagen/pisvm

================    ===============
version             toolchain      
================    ===============
``1.2-20150622``    ``intel/2017b``
================    ===============


.. _list_software_pixman_653:

*pixman*
++++++++

Pixman is a low-level software library for pixel manipulation, providing  features such as image compositing and trapezoid rasterization. Important  users of pixman are the cairo graphics library and the X server. 

*homepage*: http://www.pixman.org/

==========    ============================================================================================================================================
version       toolchain                                                                                                                                   
==========    ============================================================================================================================================
``0.34.0``    ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``0.38.0``    ``GCCcore/8.2.0``                                                                                                                           
==========    ============================================================================================================================================


.. _list_software_pizzly_690:

*pizzly*
++++++++

Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.

*homepage*: https://github.com/pmelsted/pizzly

==========    ==============
version       toolchain     
==========    ==============
``0.37.3``    ``foss/2018b``
==========    ==============


.. _list_software_pkg-config_997:

*pkg-config*
++++++++++++

pkg-config is a helper tool used when compiling applications and libraries.  It helps you insert the correct compiler options on the command line so an  application can use gcc -o test test.c `pkg-config --libs --cflags glib-2.0`  for instance, rather than hard-coding values on where to find glib (or other  libraries). 

*homepage*: http://www.freedesktop.org/wiki/Software/pkg-config/

==========    ===========================================================================================================================================================================
version       toolchain                                                                                                                                                                  
==========    ===========================================================================================================================================================================
``0.28``      ``GCC/4.8.2``, ``GCC/4.9.2``, ``GNU/4.9.3-2.25``                                                                                                                           
``0.29``      ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``                                                                                                                          
``0.29.1``    ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``0.29.2``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017a``, ``system``                                                 
==========    ===========================================================================================================================================================================


.. _list_software_pkgconfig_952:

*pkgconfig*
+++++++++++

pkgconfig is a Python module to interface with the pkg-config command line tool

*homepage*: https://github.com/matze/pkgconfig

=========    ==================    ===================================================
version      versionsuffix         toolchain                                          
=========    ==================    ===================================================
``1.1.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                    
``1.1.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                    
``1.1.0``    ``-Python-3.5.1``     ``foss/2016a``                                     
``1.1.0``    ``-Python-3.5.2``     ``foss/2016b``, ``intel/2016b``                    
``1.2.2``    ``-Python-2.7.13``    ``foss/2017a``, ``intel/2017a``                    
``1.2.2``    ``-Python-2.7.14``    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``1.2.2``    ``-Python-3.6.1``     ``foss/2017a``, ``intel/2017a``                    
``1.2.2``    ``-Python-3.6.3``     ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``
``1.3.1``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``                    
``1.3.1``    ``-Python-2.7.15``    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.3.1``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                    
``1.3.1``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``1.5.1``    ``-Python-3.7.4``     ``GCCcore/8.3.0``                                  
``1.5.1``    ``-python``           ``GCCcore/8.2.0``                                  
=========    ==================    ===================================================


.. _list_software_PlaScope_791:

*PlaScope*
++++++++++

Plasmid exploration of bacterial genomes

*homepage*: https://github.com/GuilhemRoyer/PlaScope

=========    ==============
version      toolchain     
=========    ==============
``1.3.1``    ``foss/2018b``
=========    ==============


.. _list_software_PLAST_388:

*PLAST*
+++++++

PLAST is a parallel alignment search tool for comparing large protein banks

*homepage*: http://www.irisa.fr/symbiose/projects/plast/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.3.1``    ``-Java-1.8.0_92``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_Platanus_840:

*Platanus*
++++++++++

PLATform for Assembling NUcleotide Sequences

*homepage*: http://platanus.bio.titech.ac.jp/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.2.1``    ``-linux-x86_64``    ``system``    
``1.2.4``                         ``foss/2017a``
=========    =================    ==============


.. _list_software_Platypus_866:

*Platypus*
++++++++++

Platypus is a tool designed for efficient and accurate variant-detection  in high-throughput sequencing data.

*homepage*: http://www.well.ox.ac.uk/platypus

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.8.1``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_plc_319:

*plc*
+++++

plc is the public Planck Likelihood Code.  It provides C and Fortran libraries that allow   users to compute the log likelihoods of the temperature,   polarization, and lensing maps.  Optionally, it also provides a python version of this library,   as well as tools to modify the predetermined options for some likelihoods   (e.g. changing the high-ell and low-ell lmin and lmax values of the temperature). 

*homepage*: http://pla.esac.esa.int/pla/#home

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``3.0.1``    ``-Python-2.7.15``    ``foss/2019a``, ``intel/2018b``
=========    ==================    ===============================


.. _list_software_PLINK_382:

*PLINK*
+++++++

Whole-genome association analysis toolset

*homepage*: https://www.cog-genomics.org/plink/2.0/

===========================    ==============================
version                        toolchain                     
===========================    ==============================
``1.07``                       ``foss/2016a``, ``foss/2016b``
``1.07-x86_64``                ``system``                    
``1.9b_4.1-x86_64``            ``system``                    
``2.00-alpha1-x86_64``         ``system``                    
``2.00-alpha2-x86_64``         ``system``                    
``2.00-alpha2-x86_64_avx2``    ``system``                    
===========================    ==============================


.. _list_software_PLINKSEQ_615:

*PLINKSEQ*
++++++++++

PLINK/SEQ is an open-source C/C++ library for working with human   genetic variation data. The specific focus is to provide a platform for analytic tool   development for variation data from large-scale resequencing and genotyping projects,   particularly whole-exome and whole-genome studies. It is independent of (but designed   to be complementary to) the existing PLINK package. 

*homepage*: https://atgu.mgh.harvard.edu/plinkseq/

========    ==========================================================
version     toolchain                                                 
========    ==========================================================
``0.10``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
========    ==========================================================


.. _list_software_Ploticus_851:

*Ploticus*
++++++++++

Ploticus is a free GPL software utility that can produce various types of plots and graphs

*homepage*: http://ploticus.sourceforge.net/doc/welcome.html

========    =================
version     toolchain        
========    =================
``2.42``    ``GCCcore/7.3.0``
========    =================


.. _list_software_plotly_676:

*plotly*
++++++++

Easily translate 'ggplot2' graphs to an interactive web-based version and/or  create custom web-based visualizations directly from R.

*homepage*: https://cran.r-project.org/web/packages/plotly

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``4.7.1``    ``-R-3.4.0``     ``intel/2017a``
``4.8.0``    ``-R-3.4.4``     ``intel/2018a``
=========    =============    ===============


.. _list_software_plotly.py_955:

*plotly.py*
+++++++++++

An open-source, interactive graphing library for Python

*homepage*: https://plot.ly/python

=========    ===============
version      toolchain      
=========    ===============
``4.1.0``    ``intel/2019a``
=========    ===============


.. _list_software_PLplot_603:

*PLplot*
++++++++

PLplot is a cross-platform software package for creating scientific plots whose (UTF-8) plot symbols  and text are limited in practice only by what Unicode-aware system fonts are installed on a user's computer.

*homepage*: http://plplot.sourceforge.net

==========    ================================    ===============
version       versionsuffix                       toolchain      
==========    ================================    ===============
``5.11.1``    ``-Java-1.7.0_80-Python-2.7.11``    ``foss/2016a`` 
``5.11.1``    ``-Java-1.7.0_80-Python-2.7.12``    ``intel/2016b``
==========    ================================    ===============


.. _list_software_PLUMED_455:

*PLUMED*
++++++++

PLUMED is an open source library for free energy calculations in molecular systems which  works together with some of the most popular molecular dynamics engines. Free energy calculations can be  performed as a function of many order parameters with a particular  focus on biological problems, using  state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.  The software, written in C++, can be easily interfaced with both fortran and C/C++ codes. 

*homepage*: http://www.plumed-code.org

=========    ==================    ================================================
version      versionsuffix         toolchain                                       
=========    ==================    ================================================
``2.2.2``                          ``intel/2016a``                                 
``2.2.3``                          ``foss/2016b``, ``intel/2016b``                 
``2.3.0``                          ``foss/2016b``, ``foss/2017a``, ``intel/2016b`` 
``2.3.4``                          ``intel/2017b``                                 
``2.4.0``                          ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``2.4.0``    ``-PathCV``           ``intel/2018a``                                 
``2.4.1``                          ``iomkl/2018a``                                 
``2.4.2``                          ``foss/2018b``, ``intel/2018b``                 
``2.5.0``                          ``intel/2018b``                                 
``2.5.0``    ``-Python-2.7.15``    ``foss/2018b``, ``fosscuda/2018b``              
``2.5.1``                          ``foss/2019a``, ``intel/2018b``                 
``2.5.1``    ``-PathCV``           ``intel/2018b``                                 
``2.5.2``    ``-Python-3.7.2``     ``intel/2019a``                                 
``2.5b``     ``-Python-2.7.14``    ``intel/2018a``                                 
=========    ==================    ================================================


.. _list_software_PLY_245:

*PLY*
+++++

PLY is yet another implementation of lex and yacc for Python.

*homepage*: http://www.dabeaz.com/ply/

========    =================    ==============
version     versionsuffix        toolchain     
========    =================    ==============
``3.11``    ``-Python-3.6.4``    ``foss/2018a``
========    =================    ==============


.. _list_software_PMIx_350:

*PMIx*
++++++

Process Management for Exascale Environments PMI Exascale (PMIx) represents an attempt to provide an extended version of the PMI standard specifically designed to support clusters up to and including exascale sizes. The overall objective of the project is not to branch the existing pseudo-standard definitions - in fact, PMIx fully supports both of the existing PMI-1 and PMI-2 APIs - but rather to (a) augment and extend those APIs to eliminate some current restrictions that impact scalability, and (b) provide a reference implementation of the PMI-server that demonstrates the desired level of scalability. 

*homepage*: https://pmix.org/

=========    =======================================================
version      toolchain                                              
=========    =======================================================
``1.2.5``    ``GCCcore/6.4.0``                                      
``2.1.3``    ``GCCcore/7.3.0``                                      
``3.0.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                   
``3.0.2``    ``GCCcore/8.2.0``                                      
``3.1.1``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``3.1.4``    ``GCCcore/8.3.0``                                      
=========    =======================================================


.. _list_software_PnetCDF_612:

*PnetCDF*
+++++++++

Parallel netCDF: A Parallel I/O Library for NetCDF File Access

*homepage*: https://trac.mcs.anl.gov/projects/parallel-netcdf

==========    ===============================
version       toolchain                      
==========    ===============================
``1.8.1``     ``intel/2017a``                
``1.9.0``     ``intel/2018a``                
``1.10.0``    ``foss/2018b``, ``intel/2018b``
==========    ===============================


.. _list_software_pocl_430:

*pocl*
++++++

Pocl is a portable open source (MIT-licensed) implementation of the OpenCL standard

*homepage*: http://portablecl.org

=======    ====================
version    toolchain           
=======    ====================
``1.2``    ``GCC/7.3.0-2.30``  
``1.3``    ``GCC/8.2.0-2.31.1``
=======    ====================


.. _list_software_pomkl_547:

*pomkl*
+++++++

Toolchain with PGI C, C++ and Fortran compilers, alongside Intel MKL & OpenMPI.

*homepage*: http://www.pgroup.com/index.htm

===========    ==========
version        toolchain 
===========    ==========
``2016.03``    ``system``
``2016.04``    ``system``
``2016.09``    ``system``
===========    ==========


.. _list_software_pompi_549:

*pompi*
+++++++

Toolchain with PGI C, C++ and Fortran compilers, alongside OpenMPI.

*homepage*: http://www.pgroup.com/index.htm

===========    ==========
version        toolchain 
===========    ==========
``2016.03``    ``system``
``2016.04``    ``system``
``2016.09``    ``system``
===========    ==========


.. _list_software_poppler_770:

*poppler*
+++++++++

Poppler is a PDF rendering library based on the xpdf-3.0 code  base.

*homepage*: https://poppler.freedesktop.org/

==========    ==============
version       toolchain     
==========    ==============
``0.70.1``    ``foss/2018b``
==========    ==============


.. _list_software_popt_451:

*popt*
++++++

Popt is a C library for parsing command line parameters.

*homepage*: http://freecode.com/projects/popt

========    =========================
version     toolchain                
========    =========================
``1.14``    ``GCC/4.8.2``            
``1.16``    ``GCC/4.9.2``, ``system``
========    =========================


.. _list_software_Porechop_832:

*Porechop*
++++++++++

Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.  Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,  it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments  to effectively find adapters, even at low sequence identity

*homepage*: https://github.com/rrwick/Porechop

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.2.3``    ``-Python-3.5.2``    ``foss/2016b``
``0.2.4``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_poretools_999:

*poretools*
+++++++++++

A toolkit for working with nanopore sequencing data from Oxford Nanopore.

*homepage*: http://poretools.readthedocs.io/en/latest/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.6.0``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_Postgres-XL_1064:

*Postgres-XL*
+++++++++++++

Postgres-XL is a horizontally scalable open source SQL database cluster, flexible enough to handle varying database workloads:

*homepage*: http://www.postgres-xl.org

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``9.5r1``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_PostgreSQL_980:

*PostgreSQL*
++++++++++++

PostgreSQL is a powerful, open source object-relational database system.  It is fully ACID compliant, has full support for foreign keys,  joins, views, triggers, and stored procedures (in multiple languages).  It includes most SQL:2008 data types, including INTEGER,  NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.  It also supports storage of binary large objects, including pictures,  sounds, or video. It has native programming interfaces for C/C++, Java,  .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation.

*homepage*: https://www.postgresql.org/

=========    ==================    ================================================================
version      versionsuffix         toolchain                                                       
=========    ==================    ================================================================
``9.4.7``    ``-Python-2.7.11``    ``intel/2016a``                                                 
``9.5.2``    ``-Python-2.7.11``    ``intel/2016a``                                                 
``9.6.0``    ``-Python-2.7.12``    ``intel/2016b``                                                 
``9.6.2``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                 
``10.2``     ``-Python-2.7.14``    ``intel/2018a``                                                 
``10.3``                           ``foss/2018b``                                                  
``10.3``     ``-Python-2.7.14``    ``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``11.3``     ``-Python-2.7.15``    ``GCCcore/8.2.0``                                               
``11.3``     ``-Python-3.7.2``     ``GCCcore/8.2.0``                                               
=========    ==================    ================================================================


.. _list_software_POT_243:

*POT*
+++++

POT (Python Optimal Transport) is a Python library provide several solvers for optimization problems  related to Optimal Transport for signal, image processing and machine learning.

*homepage*: https://github.com/rflamary/POT

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.5.1``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_POV-Ray_590:

*POV-Ray*
+++++++++

The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program  which generates images from a text-based scene description, and is available for a variety  of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning  three-dimensional graphics. The source code is available for those wanting to do their own ports.

*homepage*: http://www.povray.org/

===========    =================================================================================
version        toolchain                                                                        
===========    =================================================================================
``3.7.0.0``    ``intel/2016b``                                                                  
``3.7.0.7``    ``foss/2017b``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
===========    =================================================================================


.. _list_software_PPfold_581:

*PPfold*
++++++++

PPfold is a new implementation of pfold, written in Java 6.0. It can predict the consensus secondary  structure of RNA alignments through a stochastic context-free grammar coupled to an evolutionary model.  It can also use data from chemical probing experiments to predict RNA secondary structure. PPfold is multithreaded,  and can solve the structure of much longer alignments than pfold.

*homepage*: ihttp://www.daimi.au.dk/~compbio/pfold/downloads.html

=========    ==================    ==========
version      versionsuffix         toolchain 
=========    ==================    ==========
``3.1.1``    ``-Java-1.8.0_66``    ``system``
=========    ==================    ==========


.. _list_software_ppl_332:

*ppl*
+++++

The Parma Polyhedra Library (PPL) provides numerical abstractions especially  targeted at applications in the field of analysis and verification of complex systems.

*homepage*: https://www.bugseng.com/parma-polyhedra-library

=======    =================
version    toolchain        
=======    =================
``1.2``    ``GCCcore/6.4.0``
=======    =================


.. _list_software_pplacer_743:

*pplacer*
+++++++++

Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic  likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give  useful information about uncertainty, and to offer advanced visualization and downstream analysis.

*homepage*: http://matsen.fhcrc.org/pplacer/

===============    ==========
version            toolchain 
===============    ==========
``1.1.alpha19``    ``system``
===============    ==========


.. _list_software_pplpy_565:

*pplpy*
+++++++

This Python package provides a wrapper to the C++ Parma Polyhedra Library (PPL).

*homepage*: https://pypi.org/project/pplpy/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.8.4``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``0.8.4``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
=========    ==================    ===============================


.. _list_software_PRANK_380:

*PRANK*
+++++++

PRANK is a probabilistic multiple alignment program for DNA,   codon and amino-acid sequences. PRANK is based on a novel algorithm that treats   insertions correctly and avoids over-estimation of the number of deletion events.

*homepage*: http://wasabiapp.org/software/prank/

==========    ===============
version       toolchain      
==========    ===============
``170427``    ``intel/2018a``
==========    ===============


.. _list_software_PRC_229:

*PRC*
+++++

PRC is a stand-alone program for aligning and scoring two profile hidden Markov models.  This can be used to detect remote relationships between profiles more effectively than by doing simple  profile-sequence comparisons. PRC takes into account all transition and emission probabilities in both  hidden Markov models. 

*homepage*: http//supfam.org/PRC/

=========    ===============
version      toolchain      
=========    ===============
``1.5.6``    ``intel/2018a``
=========    ===============


.. _list_software_preseq_656:

*preseq*
++++++++

Software for predicting library complexity and genome coverage in high-throughput sequencing.

*homepage*: http://smithlabresearch.org/software/preseq

=========    ===============================
version      toolchain                      
=========    ===============================
``2.0.2``    ``foss/2016b``                 
``2.0.3``    ``foss/2018b``, ``intel/2018a``
=========    ===============================


.. _list_software_Primer3_674:

*Primer3*
+++++++++

Primer3 is a widely used program for designing PCR primers (PCR = 'Polymerase Chain Reaction').  PCR is an essential and ubiquitous tool in genetics and molecular biology.  Primer3 can also design hybridization probes and sequencing primers.

*homepage*: http://primer3.sourceforge.net

=========    =========================================================
version      toolchain                                                
=========    =========================================================
``2.3.7``    ``intel/2017b``                                          
``2.4.0``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
=========    =========================================================


.. _list_software_PRINSEQ_546:

*PRINSEQ*
+++++++++

A bioinformatics tool to PRe-process and show INformation of SEQuence data.

*homepage*: http://prinseq.sourceforge.net

==========    ================    ==============
version       versionsuffix       toolchain     
==========    ================    ==============
``0.20.4``    ``-Perl-5.28.0``    ``foss/2018b``
==========    ================    ==============


.. _list_software_printproto_1121:

*printproto*
++++++++++++

X.org PrintProto protocol headers.

*homepage*: http://xorg.freedesktop.org/

=========    ===============
version      toolchain      
=========    ===============
``1.0.5``    ``intel/2016a``
=========    ===============


.. _list_software_PRISMS-PF_673:

*PRISMS-PF*
+++++++++++

PRISMS-PF is a powerful, massively parallel finite element code for conducting phase field and  other related simulations of microstructural evolution.

*homepage*: https://prisms-center.github.io/phaseField

=========    ===============
version      toolchain      
=========    ===============
``2.1.1``    ``intel/2019a``
=========    ===============


.. _list_software_ProbABEL_679:

*ProbABEL*
++++++++++

Tool for genome-wide association  analysis of imputed genetic data.

*homepage*: http://www.genabel.org/packages/ProbABEL

=========    ==========
version      toolchain 
=========    ==========
``0.5.0``    ``system``
=========    ==========


.. _list_software_prodigal_850:

*prodigal*
++++++++++

Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm)    is a microbial (bacterial and archaeal) gene finding program developed    at Oak Ridge National Laboratory and the University of Tennessee.

*homepage*: https://github.com/hyattpd/Prodigal/

=========    ==========================================================================
version      toolchain                                                                 
=========    ==========================================================================
``2.6.2``    ``GCC/4.9.3-binutils-2.25``                                               
``2.6.3``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``
=========    ==========================================================================


.. _list_software_PROJ_315:

*PROJ*
++++++

Program proj is a standard Unix filter function which converts  geographic longitude and latitude coordinates into cartesian coordinates

*homepage*: http://trac.osgeo.org/proj/

=========    =================================================================================
version      toolchain                                                                        
=========    =================================================================================
``4.9.2``    ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                 
``4.9.3``    ``foss/2016b``, ``foss/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``5.0.0``    ``foss/2018a``, ``foss/2018b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018a``
``6.0.0``    ``GCCcore/8.2.0``                                                                
=========    =================================================================================


.. _list_software_ProjectQ_808:

*ProjectQ*
++++++++++

An open source software framework for quantum computing

*homepage*: https://projectq.ch

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.4.2``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_prokka_648:

*prokka*
++++++++

Prokka is a software tool for the rapid annotation of prokaryotic genomes.

*homepage*: https://www.vicbioinformatics.com/software.prokka.shtml

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.11``      ``-BioPerl-1.7.0``    ``foss/2016b`` 
``1.13``      ``-BioPerl-1.7.2``    ``intel/2018a``
``1.13.4``                          ``foss/2018b`` 
``1.13.7``                          ``gompi/2019a``
``1.14.5``                          ``gompi/2019b``
==========    ==================    ===============


.. _list_software_prompt-toolkit_1493:

*prompt-toolkit*
++++++++++++++++

prompt_toolkit is a Python library for building powerful interactive command lines and  terminal applications.

*homepage*: https://github.com/jonathanslenders/python-prompt-toolkit

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``1.0.3``     ``-Python-2.7.11``    ``foss/2016a``                 
``1.0.3``     ``-Python-3.5.1``     ``foss/2016a``                 
``1.0.6``     ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``1.0.6``     ``-Python-3.5.2``     ``intel/2016b``                
``1.0.13``    ``-Python-2.7.12``    ``intel/2016b``                
==========    ==================    ===============================


.. _list_software_proovread_978:

*proovread*
+++++++++++

PacBio hybrid error correction through iterative short read consensus

*homepage*: https://github.com/BioInf-Wuerzburg/proovread

==========    ===============
version       toolchain      
==========    ===============
``2.14.1``    ``intel/2017b``
==========    ===============


.. _list_software_propy_570:

*propy*
+++++++

Propy is a protein description software. It allows analyzing sequence-derived structural and physicochemical features, which are very useful in representing and distinguishing proteins or peptides of different structural, functional and interaction properties. These have been widely used in developing methods and software for predicting protein structural and functional classes, protein-protein interactions, drug-target interactions, protein substrates, among others.

*homepage*: https://code.google.com/archive/p/protpy

=======    ==================    ==============
version    versionsuffix         toolchain     
=======    ==================    ==============
``1.0``    ``-Python-2.7.13``    ``foss/2017a``
=======    ==================    ==============


.. _list_software_Proteinortho_1293:

*Proteinortho*
++++++++++++++

Proteinortho is a tool to detect orthologous genes within different species.

*homepage*: http://www.bioinf.uni-leipzig.de/Software/proteinortho/

=========    =============================    ==============
version      versionsuffix                    toolchain     
=========    =============================    ==============
``5.16b``    ``-Python-3.6.4-Perl-5.26.1``    ``foss/2018a``
=========    =============================    ==============


.. _list_software_protobuf_881:

*protobuf*
++++++++++

Google Protocol Buffers

*homepage*: https://github.com/google/protobuf/

===========    ===================================================
version        toolchain                                          
===========    ===================================================
``2.5.0``      ``GCCcore/7.3.0``, ``system``                      
``2.6.1``      ``system``                                         
``3.0.2``      ``foss/2016a``                                     
``3.2.0``      ``foss/2016b``, ``intel/2016b``                    
``3.3.0``      ``foss/2016b``, ``intel/2017a``                    
``3.4.0``      ``GCCcore/6.4.0``, ``intel/2017a``, ``intel/2017b``
``3.5.1``      ``intel/2017b``                                    
``3.6.0``      ``GCCcore/7.3.0``                                  
``3.6.1``      ``GCCcore/7.3.0``                                  
``3.6.1.2``    ``GCCcore/8.2.0``                                  
``3.7.1``      ``GCCcore/8.2.0``, ``GCCcore/8.3.0``               
``3.10.0``     ``GCCcore/8.3.0``                                  
===========    ===================================================


.. _list_software_protobuf-python_1600:

*protobuf-python*
+++++++++++++++++

Python Protocol Buffers runtime library.

*homepage*: https://github.com/google/protobuf/

=========    ==================    ==================================
version      versionsuffix         toolchain                         
=========    ==================    ==================================
``3.0.2``    ``-Python-2.7.11``    ``foss/2016a``                    
``3.0.2``    ``-Python-3.5.1``     ``foss/2016a``                    
``3.2.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``   
``3.2.0``    ``-Python-3.5.2``     ``foss/2016b``, ``intel/2016b``   
``3.3.0``    ``-Python-2.7.13``    ``intel/2017a``                   
``3.3.0``    ``-Python-3.5.2``     ``foss/2016b``                    
``3.3.0``    ``-Python-3.6.1``     ``intel/2017a``                   
``3.4.0``    ``-Python-2.7.13``    ``intel/2017a``                   
``3.4.0``    ``-Python-2.7.14``    ``intel/2017b``                   
``3.4.0``    ``-Python-3.6.1``     ``intel/2017a``                   
``3.4.0``    ``-Python-3.6.3``     ``intel/2017b``                   
``3.6.0``    ``-Python-2.7.15``    ``fosscuda/2018b``                
``3.6.0``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``
=========    ==================    ==================================


.. _list_software_pscom_546:

*pscom*
+++++++

ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.

*homepage*: http://www.par-tec.com

============    =============
version         toolchain    
============    =============
``5.0.43``      ``GCC/4.8.2``
``5.0.44-1``    ``GCC/4.9.2``
``5.0.48-1``    ``system``   
============    =============


.. _list_software_PSI_236:

*PSI*
+++++

PSI4 is an open-source suite of ab initio quantum chemistry programs designed for  efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform  computations with more than 2500 basis functions running serially or in parallel.

*homepage*: http://www.psicode.org/

==================    =====================    ===============
version               versionsuffix            toolchain      
==================    =====================    ===============
``4.0b6-20160201``    ``-mt-Python-2.7.11``    ``intel/2016a``
==================    =====================    ===============


.. _list_software_PSI4_288:

*PSI4*
++++++

PSI4 is an open-source suite of ab initio quantum chemistry programs designed for  efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform  computations with more than 2500 basis functions running serially or in parallel.

*homepage*: http://www.psicode.org/

=========    =========================    ===============
version      versionsuffix                toolchain      
=========    =========================    ===============
``1.0``      ``-Python-2.7.11``           ``intel/2016a``
``1.0``      ``-mt-Python-2.7.11``        ``intel/2016a``
``1.2.1``    ``-Python-2.7.15``           ``intel/2018b``
``1.2.1``    ``-Python-2.7.15-maxam8``    ``intel/2018b``
=========    =========================    ===============


.. _list_software_psmc_435:

*psmc*
++++++

This software package infers population size history from a diploid sequence using the Pairwise Sequentially Markovian Coalescent (PSMC) model.

*homepage*: https://github.com/lh3/psmc

=========    ==============================
version      toolchain                     
=========    ==============================
``0.6.5``    ``foss/2016a``, ``foss/2018a``
=========    ==============================


.. _list_software_psmpi_553:

*psmpi*
+++++++

ParaStation MPI is an open source high-performance MPI 3.0 implementation, based on MPICH v3. It provides extra low level communication libraries and integration with  various batch systems for tighter process control. 

*homepage*: https://github.com/ParaStation/psmpi2

===========    =============    =============
version        versionsuffix    toolchain    
===========    =============    =============
``5.1.0-1``                     ``GCC/4.9.2``
``5.1.5-1``                     ``GCC/4.9.3``
``5.1.5-1``    ``-mt``          ``GCC/4.9.3``
===========    =============    =============


.. _list_software_psmpi2_603:

*psmpi2*
++++++++

ParaStation is a robust and efficient cluster middleware, consisting  of a high-performance communication layer (MPI) and a sophisticated management layer.

*homepage*: http://www.par-tec.com

==========    =============    =============
version       versionsuffix    toolchain    
==========    =============    =============
``5.0.29``                     ``GCC/4.8.2``
``5.0.29``    ``-mt``          ``GCC/4.8.2``
==========    =============    =============


.. _list_software_PSolver_715:

*PSolver*
+++++++++

Poisson Solver from the BigDFT code compiled as a standalone library.

*homepage*: http://bigdft.org/Wiki/index.php?title=BigDFT_website

=========    =================================================================================================
version      toolchain                                                                                        
=========    =================================================================================================
``1.7.6``    ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
=========    =================================================================================================


.. _list_software_PSORTb_506:

*PSORTb*
++++++++

PSORTb v3.0.4 is the most precise bacterial localization prediction tool available.

*homepage*: http://psort.org/psortb/index.html

=========    ================    ==============
version      versionsuffix       toolchain     
=========    ================    ==============
``3.0.4``    ``-Perl-5.22.1``    ``foss/2016a``
=========    ================    ==============


.. _list_software_psrecord_866:

*psrecord*
++++++++++

psrecord is a small utility that uses the psutil library to record the CPU and  memory activity of a process.

*homepage*: https://github.com/astrofrog/psrecord

=======    ==================    ===============
version    versionsuffix         toolchain      
=======    ==================    ===============
``1.1``    ``-Python-2.7.14``    ``intel/2018a``
``1.1``    ``-Python-2.7.15``    ``intel/2018b``
``1.1``    ``-Python-3.6.4``     ``intel/2018a``
``1.1``    ``-Python-3.6.6``     ``intel/2018b``
=======    ==================    ===============


.. _list_software_pstoedit_876:

*pstoedit*
++++++++++

pstoedit translates PostScript and PDF graphics into other vector formats

*homepage*: http://pstoedit.net/

========    ====================================
version     toolchain                           
========    ====================================
``3.70``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``
========    ====================================


.. _list_software_psutil_673:

*psutil*
++++++++

A cross-platform process and system utilities module for Python

*homepage*: https://github.com/giampaolo/psutil

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``4.2.0``    ``-Python-2.7.11``    ``intel/2016a``                
``4.3.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``5.4.3``    ``-Python-2.7.14``    ``intel/2017b``                
``5.4.7``    ``-Python-2.7.15``    ``foss/2018b``                 
``5.4.7``    ``-Python-3.6.6``     ``foss/2018b``                 
``5.6.1``    ``-Python-2.7.15``    ``fosscuda/2018b``             
``5.6.3``                          ``GCCcore/8.2.0``              
=========    ==================    ===============================


.. _list_software_psycopg2_823:

*psycopg2*
++++++++++

Psycopg is the most popular PostgreSQL adapter for the Python programming language.

*homepage*: http://initd.org/psycopg/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``2.7``      ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``2.8.3``    ``-Python-3.7.2``     ``foss/2019a``                 
=========    ==================    ===============================


.. _list_software_ptemcee_739:

*ptemcee*
+++++++++

ptemcee, pronounced "tem-cee", is fork of Daniel Foreman-Mackey's wonderful emcee to implement  parallel tempering more robustly. If you're trying to characterise awkward, multi-model probability  distributions, then ptemcee is your friend.

*homepage*: https://github.com/willvousden/ptemcee

=========    ==============
version      toolchain     
=========    ==============
``1.0.0``    ``foss/2019a``
=========    ==============


.. _list_software_PTESFinder_916:

*PTESFinder*
++++++++++++

Post-Transcriptional Exon Shuffling (PTES) Identification Pipeline

*homepage*: https://sourceforge.net/projects/ptesfinder-v1/

=======    ===============
version    toolchain      
=======    ===============
``1``      ``intel/2017b``
=======    ===============


.. _list_software_pubtcrs_771:

*pubtcrs*
+++++++++

This repository contains C++ source code for the TCR clustering  and correlation analyses described in the manuscript "Human T cell receptor  occurrence patterns encode immune history, genetic background, and receptor  specificity" by William S DeWitt III, Anajane Smith, Gary Schoch, John A Hansen,  Frederick A Matsen IV and Philip Bradley, available on bioRxiv.

*homepage*: https://github.com/phbradley/pubtcrs

============    ===============
version         toolchain      
============    ===============
``20180622``    ``intel/2019a``
============    ===============


.. _list_software_pullseq_774:

*pullseq*
+++++++++

Utility program for extracting sequences from a fasta/fastq file

*homepage*: https://github.com/bcthomas/pullseq

=========    =================
version      toolchain        
=========    =================
``1.0.2``    ``GCCcore/7.3.0``
=========    =================


.. _list_software_py_233:

*py*
++++

library with cross-python path, ini-parsing, io, code, log facilities

*homepage*:  http://pylib.readthedocs.org/

==========    ==================    ==============
version       versionsuffix         toolchain     
==========    ==================    ==============
``1.4.31``    ``-Python-2.7.11``    ``foss/2016a``
``1.4.31``    ``-Python-3.5.1``     ``foss/2016a``
==========    ==================    ==============


.. _list_software_py-cpuinfo_1034:

*py-cpuinfo*
++++++++++++

py-cpuinfo gets CPU info with pure Python.

*homepage*: https://github.com/workhorsy/py-cpuinfo

=========    ==========
version      toolchain 
=========    ==========
``5.0.0``    ``system``
=========    ==========


.. _list_software_PyAMG_414:

*PyAMG*
+++++++

PyAMG is a library of Algebraic Multigrid (AMG) solvers with a convenient Python interface.

*homepage*: http://pyamg.org

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.0.1``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_PyAPS3_480:

*PyAPS3*
++++++++

Python 3 Atmospheric Phase Screen

*homepage*: https://github.com/AngeliqueBenoit/pyaps3

============    =================    ==============
version         versionsuffix        toolchain     
============    =================    ==============
``20190407``    ``-Python-3.7.2``    ``foss/2019a``
============    =================    ==============


.. _list_software_pybedtools_1093:

*pybedtools*
++++++++++++

pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.

*homepage*: https://daler.github.io/pybedtools

==========    ==================    ================================
version       versionsuffix         toolchain                       
==========    ==================    ================================
``0.7.10``    ``-Python-2.7.14``    ``intel/2017b``, ``intel/2018a``
``0.7.10``    ``-Python-3.6.6``     ``foss/2018b``                  
``0.8.0``                           ``foss/2019a``, ``intel/2019a`` 
==========    ==================    ================================


.. _list_software_pyBigWig_802:

*pyBigWig*
++++++++++

A python extension, written in C, for quick access to bigBed files and access to and creation of  bigWig files.

*homepage*: https://github.com/deeptools/pyBigWig

==========    =================    ==============
version       versionsuffix        toolchain     
==========    =================    ==============
``0.3.13``    ``-Python-3.6.6``    ``foss/2018b``
==========    =================    ==============


.. _list_software_pybind11_744:

*pybind11*
++++++++++

pybind11 is a lightweight header-only library that exposes C++ types in Python and vice versa,  mainly to create Python bindings of existing C++ code.

*homepage*: https://pybind11.readthedocs.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``2.2.4``    ``-Python-3.6.4``    ``foss/2018a`` 
``2.2.4``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_PyCairo_695:

*PyCairo*
+++++++++

Python bindings for the cairo library

*homepage*: https://cairographics.org/pycairo/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.10.0``    ``-Python-2.7.11``    ``intel/2016a``
``1.16.1``    ``-Python-3.6.3``     ``foss/2017b`` 
``1.16.2``    ``-Python-2.7.14``    ``intel/2017b``
``1.18.0``    ``-Python-2.7.14``    ``intel/2018a``
``1.18.0``    ``-Python-2.7.15``    ``foss/2018b`` 
``1.18.0``    ``-Python-3.6.6``     ``foss/2018b`` 
==========    ==================    ===============


.. _list_software_PyCharm_692:

*PyCharm*
+++++++++

PyCharm Community Edition: Python IDE for Professional Developers

*homepage*: https://www.jetbrains.com/pycharm/

============    ==========
version         toolchain 
============    ==========
``2017.2.3``    ``system``
============    ==========


.. _list_software_pycma_538:

*pycma*
+++++++

A stochastic numerical optimization algorithm for difficult (non-convex, ill-conditioned,  multi-modal, rugged, noisy) optimization problems in continuous search spaces, implemented in Python.

*homepage*: https://github.com/CMA-ES/pycma

=========    ===============
version      toolchain      
=========    ===============
``2.7.0``    ``intel/2019a``
=========    ===============


.. _list_software_PyCogent_809:

*PyCogent*
++++++++++

PyCogent is a software library for genomic biology. It is a fully  integrated and thoroughly tested framework for: controlling third-party applications;  devising workflows; querying databases; conducting novel probabilistic analyses of  biological sequence evolution; and generating publication quality graphics.

*homepage*: http://pycogent.org

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.5.3``    ``-Python-2.7.12``    ``intel/2016b``                
``1.9``      ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``1.9``      ``-Python-2.7.15``    ``foss/2018b``                 
=========    ==================    ===============================


.. _list_software_PyCUDA_486:

*PyCUDA*
++++++++

PyCUDA lets you access Nvidia’s CUDA parallel computation API from Python.

*homepage*: http://mathema.tician.de/software/pycuda

============    =============================    ===============
version         versionsuffix                    toolchain      
============    =============================    ===============
``2016.1.2``    ``-Python-2.7.12``               ``intel/2016b``
``2017.1.1``    ``-Python-2.7.14``               ``intel/2018a``
``2018.1``      ``-Python-3.6.4-CUDA-9.1.85``    ``intel/2018a``
============    =============================    ===============


.. _list_software_PyDatastream_1231:

*PyDatastream*
++++++++++++++

Lightweight SOAP client

*homepage*: https://pypi.python.org/pypi/suds-py3

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.5.1``    ``-Python-3.6.4``    ``intel/2018a``
=========    =================    ===============


.. _list_software_pydicom_757:

*pydicom*
+++++++++

Read, modify and write DICOM files with python code

*homepage*: https://github.com/darcymason/pydicom

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.9.9``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_pydlpoly_893:

*pydlpoly*
++++++++++

Pydlpoly is a molecular dynamics simulation package which is a modified version of DL-POLY with  a Python language interface.

*homepage*: http://cmc.aci.ruhr-uni-bochum.de/cmc/

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20150225``    ``-Python-2.7.12``    ``intel/2016b``
``20150225``    ``-Python-2.7.13``    ``intel/2017a``
============    ==================    ===============


.. _list_software_pyEGA3_489:

*pyEGA3*
++++++++

A basic Python-based EGA download client 

*homepage*: https://github.com/EGA-archive/ega-download-client

==========    =================    =================
version       versionsuffix        toolchain        
==========    =================    =================
``3.0.33``    ``-Python-3.7.2``    ``GCCcore/8.2.0``
==========    =================    =================


.. _list_software_pyenchant_970:

*pyenchant*
+++++++++++

PyEnchant is a spellchecking library for Python, based on the excellent Enchant library.

*homepage*: https://pythonhosted.org/pyenchant/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.6.8``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_PyFFmpeg_766:

*PyFFmpeg*
++++++++++

Python FFmpeg wrapper

*homepage*: https://github.com/mhaller/pyffmpeg

===========    ==================    ================
version        versionsuffix         toolchain       
===========    ==================    ================
``2.1beta``    ``-Python-2.7.10``    ``gimkl/2.11.5``
``2.1beta``    ``-Python-2.7.11``    ``intel/2016a`` 
===========    ==================    ================


.. _list_software_pyFFTW_544:

*pyFFTW*
++++++++

A pythonic wrapper around FFTW, the FFT library, presenting a unified interface  for all the supported transforms.

*homepage*: https://github.com/pyFFTW/pyFFTW

==========    ===============
version       toolchain      
==========    ===============
``0.11.1``    ``intel/2019a``
==========    ===============


.. _list_software_pyfits_671:

*pyfits*
++++++++

The PyFITS module is a Python library providing access to FITS (Flexible Image Transport System)

*homepage*: https://pythonhosted.org/pyfits/

=======    ==================    ===============
version    versionsuffix         toolchain      
=======    ==================    ===============
``3.5``    ``-Python-2.7.15``    ``intel/2018b``
=======    ==================    ===============


.. _list_software_PyFMI_421:

*PyFMI*
+++++++

PyFMI is a package for loading and interacting with Functional Mock-Up Units (FMUs),  which are compiled dynamic models compliant with the Functional Mock-Up Interface (FMI)

*homepage*: https://pypi.org/project/PyFMI/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.4.0``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_PyFR_353:

*PyFR*
++++++

PyFR is an open-source Python based framework for solving advection-diffusion type problems on streaming architectures using the Flux Reconstruction approach of Huynh. The framework is designed to solve a range of governing systems on mixed unstructured grids containing various element types. It is also designed to target a range of hardware platforms via use of an in-built domain specific language derived from the Mako templating engine.

*homepage*: http://http://www.pyfr.org

=========    =============================    ===============
version      versionsuffix                    toolchain      
=========    =============================    ===============
``1.7.6``    ``-Python-3.6.4-CUDA-9.1.85``    ``intel/2018a``
=========    =============================    ===============


.. _list_software_pygccxml_871:

*pygccxml*
++++++++++

Python package for easy C++ declarations navigation.

*homepage*: https://pypi.python.org/pypi/pygccxml

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20160706``    ``-Python-2.7.11``    ``foss/2016a``
``20160706``    ``-Python-3.5.1``     ``foss/2016a``
============    ==================    ==============


.. _list_software_pyGIMLi_635:

*pyGIMLi*
+++++++++

pyGIMLi is an open-source multi-method library for solving inverse  and forward tasks related to geophysical problems. Written in C++ and Python,  it offers both efficiency and flexibility allowing you to quickly build  your own robust inversion applications for the geophysical problem at hand.

*homepage*: http://www.pygimli.org/

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20160803``    ``-Python-2.7.11``    ``foss/2016a``
``20160803``    ``-Python-3.5.1``     ``foss/2016a``
============    ==================    ==============


.. _list_software_Pygments_855:

*Pygments*
++++++++++

Generic syntax highlighter suitable for use in code hosting, forums, wikis or other applications  that need to prettify source code.

*homepage*: http://pygments.org/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.1.3``    ``-Python-2.7.11``    ``foss/2016a``
``2.1.3``    ``-Python-3.5.1``     ``foss/2016a``
=========    ==================    ==============


.. _list_software_PyGObject_871:

*PyGObject*
+++++++++++

Python Bindings for GLib/GObject/GIO/GTK+

*homepage*: https://pygobject.readthedocs.io/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``2.28.6``    ``-Python-2.7.11``    ``intel/2016a``
``2.28.6``    ``-Python-2.7.14``    ``intel/2017b``
``2.28.7``    ``-Python-2.7.14``    ``intel/2018a``
``2.28.7``    ``-Python-2.7.15``    ``foss/2018b`` 
==========    ==================    ===============


.. _list_software_pygrib_653:

*pygrib*
++++++++

Python interface for reading and writing GRIB data

*homepage*: https://jswhit.github.io/pygrib

=========    ==============
version      toolchain     
=========    ==============
``2.0.4``    ``foss/2019a``
=========    ==============


.. _list_software_PyGTK_431:

*PyGTK*
+++++++

PyGTK lets you to easily create programs with a graphical user interface  using the Python programming language.

*homepage*: http://www.pygtk.org/

==========    ==================    ================================
version       versionsuffix         toolchain                       
==========    ==================    ================================
``2.24.0``    ``-Python-2.7.11``    ``intel/2016a``                 
``2.24.0``    ``-Python-2.7.14``    ``intel/2017b``, ``intel/2018a``
``2.24.0``    ``-Python-2.7.15``    ``foss/2018b``                  
==========    ==================    ================================


.. _list_software_PyGTS_439:

*PyGTS*
+++++++

PyGTS is a python package used to construct, manipulate,   and perform computations on triangulated surfaces.   It is a hand-crafted and pythonic binding for the GNU Triangulated Surface (GTS) Library. 

*homepage*: https://sourceforge.net/projects/pygts/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.3.1``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``0.3.1``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``0.3.1``    ``-Python-2.7.14``    ``intel/2018a``                
=========    ==================    ===============================


.. _list_software_PyGWAS_507:

*PyGWAS*
++++++++

PyGWAS is a library for running Genome Wide Association studies.

*homepage*: https://github.com/timeu/pygwas

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.2.0``    ``-Python-2.7.11``    ``foss/2016a``                 
``1.3.1``    ``-Python-2.7.11``    ``foss/2016a``                 
``1.4.0``    ``-Python-2.7.11``    ``foss/2016a``                 
``1.5.0``    ``-Python-2.7.11``    ``foss/2016a``                 
``1.6.1``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``1.7.1``    ``-Python-2.7.13``    ``foss/2017a``                 
=========    ==================    ===============================


.. _list_software_pyhdf_539:

*pyhdf*
+++++++

Python wrapper around the NCSA HDF version 4 library

*homepage*: https://github.com/fhs/pyhdf

==========    ==============
version       toolchain     
==========    ==============
``0.10.1``    ``foss/2019a``
==========    ==============


.. _list_software_pyiron_673:

*pyiron*
++++++++

An integrated development environment (IDE) for computational materials science.

*homepage*: https://github.com/pyiron/pyiron

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.2.5``    ``-Python-3.7.2``    ``intel/2019a``
=========    =================    ===============


.. _list_software_Pyke3_460:

*Pyke3*
+++++++

Pyke introduces a form of Logic Programming (inspired by Prolog)  to the Python community by providing a knowledge-based inference engine  (expert system) written in 100% Python.

*homepage*: http://sourceforge.net/projects/pyke/

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.1.1``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_Pylint_640:

*Pylint*
++++++++

Pylint is a tool that checks for errors in Python code, tries to enforce  a coding standard and looks for code smells. It can also look for certain type errors,  it can recommend suggestions about how particular blocks can be refactored and  can offer you details about the code's complexity.

*homepage*: https://www.pylint.org/

=========    ==================    ================================================
version      versionsuffix         toolchain                                       
=========    ==================    ================================================
``1.9.3``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``
=========    ==================    ================================================


.. _list_software_pymatgen_869:

*pymatgen*
++++++++++

Python Materials Genomics is a robust materials analysis code that defines core object  representations for structures and molecules with support for many electronic structure codes.

*homepage*: https://pypi.python.org/pypi/pymatgen

==============    ==================    =========================
version           versionsuffix         toolchain                
==============    ==================    =========================
``3.5.0``         ``-Python-2.7.11``    ``intel/2016.02-GCC-4.9``
``4.1.1``         ``-Python-2.7.12``    ``intel/2016b``          
``4.3.2``         ``-Python-2.7.12``    ``intel/2016b``          
``4.7.3``         ``-Python-2.7.13``    ``intel/2017a``          
``2017.10.16``    ``-Python-2.7.14``    ``intel/2017b``          
``2017.10.16``    ``-Python-3.6.3``     ``intel/2017b``          
==============    ==================    =========================


.. _list_software_pymatgen-db_1112:

*pymatgen-db*
+++++++++++++

Pymatgen-db is a database add-on for the Python Materials Genomics (pymatgen)  materials analysis library.

*homepage*: https://pypi.python.org/pypi/pymatgen-db

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.6.5``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_pymbar_651:

*pymbar*
++++++++

The pymbar package contains the pymbar suite of tools for the analysis of simulated and experimental data with the multistate Bennett acceptance ratio (MBAR) estimator.

*homepage*: http://pymbar.readthedocs.io/en/master/

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``3.0.3``    ``-Python-3.6.3``    ``intel/2017b``
=========    =================    ===============


.. _list_software_pymemcache_1052:

*pymemcache*
++++++++++++

A comprehensive, fast, pure-Python memcached client.

*homepage*: https://github.com/pinterest/pymemcache

=========    =================    ===============================
version      versionsuffix        toolchain                      
=========    =================    ===============================
``2.1.1``    ``-Python-3.6.4``    ``foss/2018a``, ``intel/2018a``
=========    =================    ===============================


.. _list_software_PyNAST_511:

*PyNAST*
++++++++

PyNAST is a reimplementation of the NAST sequence aligner, which has  become a popular tool for adding new 16s rRNA sequences to existing 16s rRNA alignments.  This reimplementation is more flexible, faster, and easier to install and maintain than  the original NAST implementation.

*homepage*: https://biocore.github.io/pynast

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.2.2``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``1.2.2``    ``-Python-2.7.15``    ``foss/2018b``                 
=========    ==================    ===============================


.. _list_software_Pyomo_532:

*Pyomo*
+++++++

Pyomo is a Python-based open-source software package that supports a diverse set of optimization  capabilities for formulating and analyzing optimization models. 

*homepage*: http://www.pyomo.org/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``5.5.0``    ``-Python-2.7.15``    ``foss/2018b``
``5.5.0``    ``-Python-3.6.6``     ``foss/2018b``
=========    ==================    ==============


.. _list_software_PyOpenGL_750:

*PyOpenGL*
++++++++++

PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs.

*homepage*: http://pyopengl.sourceforge.net

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``3.1.1a1``                          ``GCCcore/8.2.0``              
``3.1.1a1``    ``-Python-2.7.11``    ``intel/2016a``                
``3.1.1a1``    ``-Python-2.7.12``    ``foss/2016b``                 
``3.1.1a1``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2017b``
``3.1.3b2``    ``-Python-2.7.14``    ``intel/2018a``                
``3.1.3b2``    ``-Python-2.7.15``    ``foss/2018b``                 
===========    ==================    ===============================


.. _list_software_pyplusplus_1137:

*pyplusplus*
++++++++++++

Py++ is a code generator for Boost.Python that simplifies  writing Python bindings of a C/C++ library  The tool is implemented as a Python module which is controlled by a user script.

*homepage*: https://bitbucket.org/ompl/pyplusplus

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20160707``    ``-Python-2.7.11``    ``foss/2016a``
``20160707``    ``-Python-3.5.1``     ``foss/2016a``
============    ==================    ==============


.. _list_software_pyproj_676:

*pyproj*
++++++++

Python interface to PROJ4 library for cartographic transformations

*homepage*: https://pyproj4.github.io/pyproj

=========    =================
version      toolchain        
=========    =================
``2.1.3``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_pyqstem_787:

*pyqstem*
+++++++++

QSTEM is a program for quantitative image simulation in electron microscopy, including TEM, STEM and CBED image simulation.  This project interfaces the QSTEM code with Python and the Atomic Simulation Environment (ASE) to provide a single environment for building models, simulating and analysing images.

*homepage*: https://github.com/jacobjma/PyQSTEM

=========    =================    ===============================
version      versionsuffix        toolchain                      
=========    =================    ===============================
``1.0.3``    ``-Python-3.6.6``    ``foss/2018b``, ``intel/2018b``
=========    =================    ===============================


.. _list_software_PyQt_398:

*PyQt*
++++++

PyQt is a set of Python v2 and v3 bindings for Digia's Qt application framework.

*homepage*: http://www.riverbankcomputing.co.uk/software/pyqt

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``4.11.4``    ``-Python-2.7.11``    ``intel/2016a``                
``4.11.4``    ``-Python-2.7.12``    ``intel/2016b``                
``4.12``      ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``4.12``      ``-Python-2.7.13``    ``intel/2017a``                
``4.12.1``    ``-Python-2.7.14``    ``foss/2018a``                 
``4.12.3``    ``-Python-2.7.15``    ``fosscuda/2018b``             
==========    ==================    ===============================


.. _list_software_PyQt5_451:

*PyQt5*
+++++++

PyQt5 is a set of Python bindings for v5 of the Qt application framework from The Qt Company.

*homepage*: http://www.riverbankcomputing.co.uk/software/pyqt

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``5.7``       ``-Python-2.7.11``    ``foss/2016a``                 
``5.7.1``     ``-Python-2.7.12``    ``intel/2016b``                
``5.8.2``     ``-Python-2.7.13``    ``intel/2017a``                
``5.9.2``     ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``5.9.2``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``
``5.11.3``    ``-Python-3.6.6``     ``foss/2018b``                 
``5.12.1``    ``-Python-2.7.15``    ``GCCcore/8.2.0``              
``5.12.1``    ``-Python-3.7.2``     ``GCCcore/8.2.0``              
==========    ==================    ===============================


.. _list_software_PyQtGraph_896:

*PyQtGraph*
+++++++++++

PyQtGraph is a pure-python graphics and GUI library built on PyQt4/PySide and numpy.

*homepage*: http://www.pyqtgraph.org/

==========    =================    ===============
version       versionsuffix        toolchain      
==========    =================    ===============
``0.10.0``    ``-Python-3.6.4``    ``intel/2018a``
``0.10.0``    ``-Python-3.7.2``    ``intel/2019a``
==========    =================    ===============


.. _list_software_PyRETIS_592:

*PyRETIS*
+++++++++

PyRETIS is a Python library for rare event molecular simulations with emphasis on methods based  on transition interface sampling and replica exchange transition interface sampling.

*homepage*: http://www.pyretis.org

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``2.1.0``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_pyringe_766:

*pyringe*
+++++++++

Debugger capable of attaching to and injecting code into python processes.

*homepage*: https://github.com/google/pyringe

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.0.2``    ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_Pyro4_478:

*Pyro4*
+++++++

Pyro means PYthon Remote Objects. It is a library that enables you to build applications in which  objects can talk to eachother over the network, with minimal programming effort.

*homepage*: https://pypi.python.org/pypi/Pyro4

========    ==================    ==============
version     versionsuffix         toolchain     
========    ==================    ==============
``4.47``    ``-Python-2.7.11``    ``foss/2016a``
========    ==================    ==============


.. _list_software_Pysam_522:

*Pysam*
+++++++

Pysam is a python module for reading and manipulating Samfiles.   It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.

*homepage*: https://github.com/pysam-developers/pysam

============    ==================    ==============================================================
version         versionsuffix         toolchain                                                     
============    ==================    ==============================================================
``0.8.4``       ``-Python-2.7.12``    ``intel/2016b``                                               
``0.9.1.4``     ``-Python-2.7.12``    ``foss/2016b``                                                
``0.10.0``      ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                               
``0.12.0.1``    ``-Python-2.7.13``    ``intel/2017a``                                               
``0.12.0.1``    ``-Python-2.7.14``    ``intel/2017b``                                               
``0.12.0.1``    ``-Python-3.6.3``     ``intel/2017b``                                               
``0.13``        ``-Python-2.7.14``    ``intel/2017b``                                               
``0.13.0``      ``-Python-3.6.3``     ``intel/2017b``                                               
``0.14``        ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``                               
``0.14``        ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``                               
``0.14``        ``-Python-3.6.4``     ``intel/2018a``                                               
``0.14.1``      ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``                               
``0.14.1``      ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                               
``0.15.1``      ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``                               
``0.15.1``      ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                               
``0.15.2``                            ``GCC/8.2.0-2.31.1``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
``0.15.3``                            ``GCC/8.3.0``, ``iccifort/2019.5.281``                        
============    ==================    ==============================================================


.. _list_software_pyScaf_614:

*pyScaf*
++++++++

pyScaf orders contigs from genome assemblies utilising several types of information

*homepage*: https://github.com/lpryszcz/pyScaf

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.12a4``    ``-Python-2.7.14``    ``intel/2017b``
==========    ==================    ===============


.. _list_software_PySCF_421:

*PySCF*
+++++++

PySCF is an open-source collection of electronic structure modules powered by Python.

*homepage*: http://www.pyscf.org

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.6.3``    ``-Python-3.7.2``    ``foss/2019a``
=========    =================    ==============


.. _list_software_pyshp_564:

*pyshp*
+++++++

Pure Python read/write support for ESRI Shapefile format

*homepage*: https://github.com/GeospatialPython/pyshp

==========    =================    ==============
version       versionsuffix        toolchain     
==========    =================    ==============
``1.2.12``    ``-Python-3.6.2``    ``foss/2017b``
==========    =================    ==============


.. _list_software_Pysolar_746:

*Pysolar*
+++++++++

Pysolar is a collection of Python libraries for simulating the irradiation  of any point on earth by the sun.

*homepage*: http://pysolar.org/

=======    =================    =================
version    versionsuffix        toolchain        
=======    =================    =================
``0.7``    ``-Python-3.6.3``    ``intel/2017b``  
``0.7``    ``-Python-3.6.4``    ``intel/2018a``  
``0.8``    ``-Python-3.6.6``    ``intel/2018b``  
``0.8``    ``-Python-3.7.2``    ``GCCcore/8.2.0``
=======    =================    =================


.. _list_software_pysqlite_891:

*pysqlite*
++++++++++

pysqlite is an interface to the SQLite 3.x embedded relational database engine.  It is almost fully compliant with the Python database API version 2.0 also exposes the unique features of SQLite.

*homepage*: https://pypi.python.org/pypi/pysqlite

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.8.2``    ``-Python-2.7.11``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_PyStan_607:

*PyStan*
++++++++

Python interface to Stan, a package for Bayesian inference  using the No-U-Turn sampler, a variant of Hamiltonian Monte Carlo.

*homepage*: https://github.com/stan-dev/pystan

============    =================    ===============
version         versionsuffix        toolchain      
============    =================    ===============
``2.19.0.0``    ``-Python-3.6.4``    ``intel/2018a``
============    =================    ===============


.. _list_software_pystran_785:

*pystran*
+++++++++

Toolset of dynamical model STRucture ANalysis algorithms

*homepage*: http://stijnvanhoey.github.io/pystran/

==============    ==================    ===============
version           versionsuffix         toolchain      
==============    ==================    ===============
``2017.04.20``    ``-Python-2.7.14``    ``intel/2017b``
==============    ==================    ===============


.. _list_software_PyTables_804:

*PyTables*
++++++++++

PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope  with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the  NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical  parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browse,  process and search very large amounts of data. One important feature of PyTables is that it optimizes memory and disk  resources so that data takes much less space (specially if on-flight compression is used) than other solutions such as  relational or object oriented databases.

*homepage*: http://www.pytables.org

===========    ==================    ==================================
version        versionsuffix         toolchain                         
===========    ==================    ==================================
``3.2.2``      ``-Python-2.7.12``    ``foss/2016b``                    
``3.2.3.1``    ``-Python-2.7.12``    ``intel/2016b``                   
``3.3.0``      ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``   
``3.3.0``      ``-Python-3.5.2``     ``intel/2016b``                   
``3.4.2``      ``-Python-2.7.13``    ``foss/2017a``                    
``3.4.2``      ``-Python-3.6.1``     ``intel/2017a``                   
``3.4.2``      ``-Python-3.6.3``     ``intel/2017b``                   
``3.4.2``      ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``   
``3.4.4``      ``-Python-2.7.15``    ``foss/2018b``                    
``3.4.4``      ``-Python-3.5.1``     ``foss/2016a``                    
``3.4.4``      ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``
``3.5.2``                            ``intel/2019a``                   
``3.5.2``      ``-Python-2.7.14``    ``intel/2018a``                   
===========    ==================    ==================================


.. _list_software_pytest_681:

*pytest*
++++++++

pytest: simple powerful testing with Python

*homepage*: http://pytest.org

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``3.0.1``    ``-Python-2.7.11``    ``foss/2016a``                 
``3.0.1``    ``-Python-3.5.1``     ``foss/2016a``                 
``3.8.0``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``
``3.8.2``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``3.8.2``    ``-Python-2.7.15``    ``intel/2018b``                
``3.8.2``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``3.8.2``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
``4.3.0``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
``4.4.0``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
``4.4.0``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
=========    ==================    ===============================


.. _list_software_PYTHIA_463:

*PYTHIA*
++++++++

PYTHIA is a standard tool for the generation of events in high-energy collisions,  comprising a coherent set of physics models for the evolution from a few-body hard  process to a complex multiparticle final state.

*homepage*: http://home.thep.lu.se/~torbjorn/Pythia.html

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``8.226``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_Python_642:

*Python*
++++++++

Python is a programming language that lets you work more quickly and integrate your systems  more effectively.

*homepage*: https://python.org/

==========    =================    =================================================================================================================================================================
version       versionsuffix        toolchain                                                                                                                                                        
==========    =================    =================================================================================================================================================================
``2.7.9``     ``-bare``            ``GCC/4.8.4``, ``GCC/4.9.2``                                                                                                                                     
``2.7.10``                         ``gimkl/2.11.5``                                                                                                                                                 
``2.7.10``    ``-bare``            ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``                                                                                                                           
``2.7.11``                         ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``                                
``2.7.11``    ``-bare``            ``GCC/4.9.3-2.25``                                                                                                                                               
``2.7.11``    ``-libX11-1.6.3``    ``intel/2016a``                                                                                                                                                  
``2.7.12``                         ``foss/2016b``, ``intel/2016b``, ``iomkl/2017a``                                                                                                                 
``2.7.12``    ``-bare``            ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``                                                                                    
``2.7.13``                         ``foss/2017a``, ``intel/2017a``                                                                                                                                  
``2.7.13``    ``-bare``            ``GCCcore/6.3.0``                                                                                                                                                
``2.7.14``                         ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intel/2018.01``, ``intel/2018a``, ``intelcuda/2017b``, ``iomkl/2018a``
``2.7.14``    ``-bare``            ``GCCcore/6.4.0``                                                                                                                                                
``2.7.15``                         ``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``                                                                          
``2.7.15``    ``-bare``            ``GCCcore/7.2.0``, ``GCCcore/7.3.0``                                                                                                                             
``2.7.16``                         ``GCCcore/8.3.0``                                                                                                                                                
``3.5.1``                          ``foss/2016a``, ``intel/2016a``                                                                                                                                  
``3.5.2``                          ``foss/2016.04``, ``foss/2016b``, ``intel/2016b``                                                                                                                
``3.5.2``     ``-bare``            ``GCC/5.4.0-2.26``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``                                                                                                       
``3.6.1``                          ``foss/2017a``, ``intel/2017a``                                                                                                                                  
``3.6.2``                          ``foss/2017b``, ``intel/2017b``, ``intel/2018.00``                                                                                                               
``3.6.3``                          ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018.01``, ``intelcuda/2017b``                                                                      
``3.6.4``                          ``foss/2017a``, ``foss/2018a``, ``fosscuda/2018a``, ``golf/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``                                          
``3.6.6``                          ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``                                                                                             
``3.7.0``                          ``foss/2018b``, ``intel/2018b``, ``iomkl/2018b``                                                                                                                 
``3.7.2``                          ``GCCcore/8.2.0``                                                                                                                                                
``3.7.4``                          ``GCCcore/8.3.0``                                                                                                                                                
==========    =================    =================================================================================================================================================================


.. _list_software_python-igraph_1354:

*python-igraph*
+++++++++++++++

Python interface to the igraph high performance graph library, primarily aimed at complex network  research and analysis.

*homepage*: http://igraph.org/python

===============    ==================    ===============
version            versionsuffix         toolchain      
===============    ==================    ===============
``0.7.1.post6``    ``-Python-2.7.14``    ``intel/2017b``
``0.7.1.post6``    ``-Python-3.6.6``     ``foss/2018b`` 
===============    ==================    ===============


.. _list_software_python-Levenshtein_1876:

*python-Levenshtein*
++++++++++++++++++++

Python extension for computing string edit distances and similarities.

*homepage*: https://pypi.org/project/python-Levenshtein/

==========    =================    ==============
version       versionsuffix        toolchain     
==========    =================    ==============
``0.12.0``    ``-Python-3.6.6``    ``foss/2018b``
==========    =================    ==============


.. _list_software_python-parasail_1564:

*python-parasail*
+++++++++++++++++

This package contains Python bindings for parasail. 

*homepage*: https://pypi.org/project/parasail/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.1.12``    ``-Python-2.7.14``    ``intel/2018a``
``1.1.16``    ``-Python-3.6.6``     ``foss/2018b`` 
==========    ==================    ===============


.. _list_software_PyTorch_713:

*PyTorch*
+++++++++

Tensors and Dynamic neural networks in Python with strong GPU acceleration. PyTorch is a deep learning framework that puts Python first.

*homepage*: https://pytorch.org/

=========    =============================    ==================================
version      versionsuffix                    toolchain                         
=========    =============================    ==================================
``0.3.1``    ``-Python-2.7.14``               ``fosscuda/2017b``                
``0.3.1``    ``-Python-3.6.3``                ``fosscuda/2017b``                
``0.3.1``    ``-Python-3.6.4``                ``intel/2018a``                   
``0.3.1``    ``-Python-3.6.4-CUDA-9.1.85``    ``foss/2018a``                    
``0.4.1``    ``-Python-3.6.4``                ``intel/2018a``                   
``1.0.1``    ``-Python-3.6.6``                ``foss/2018b``, ``fosscuda/2018b``
``1.1.0``    ``-Python-3.7.2``                ``foss/2019a``                    
``1.2.0``    ``-Python-3.7.2``                ``foss/2019a``, ``fosscuda/2019a``
``1.3.1``    ``-Python-3.7.4``                ``foss/2019b``, ``fosscuda/2019b``
=========    =============================    ==================================


.. _list_software_PyYAML_508:

*PyYAML*
++++++++

PyYAML is a YAML parser and emitter for the Python programming language.

*homepage*: https://github.com/yaml/pyyaml

=========    ==================    ====================================================================================
version      versionsuffix         toolchain                                                                           
=========    ==================    ====================================================================================
``3.11``     ``-Python-2.7.11``    ``intel/2016a``                                                                     
``3.12``                           ``system``                                                                          
``3.12``     ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                                                     
``3.12``     ``-Python-2.7.13``    ``intel/2017a``                                                                     
``3.12``     ``-Python-2.7.14``    ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``
``3.12``     ``-Python-3.5.2``     ``intel/2016b``                                                                     
``3.12``     ``-Python-3.6.1``     ``intel/2017a``                                                                     
``3.12``     ``-Python-3.6.3``     ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``                                 
``3.12``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                                                     
``3.13``                           ``system``                                                                          
``3.13``     ``-Python-2.7.15``    ``fosscuda/2018b``, ``intel/2018b``                                                 
``3.13``     ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                                 
``5.1``                            ``GCCcore/8.2.0``                                                                   
``5.1.2``                          ``GCCcore/8.3.0``                                                                   
=========    ==================    ====================================================================================


.. _list_software_PyZMQ_449:

*PyZMQ*
+++++++

Python bindings for ZeroMQ

*homepage*: http://www.zeromq.org/bindings:python

==========    =======================    ===============================
version       versionsuffix              toolchain                      
==========    =======================    ===============================
``15.2.0``    ``-Python-2.7.11-zmq4``    ``foss/2016a``, ``intel/2016a``
``15.2.0``    ``-Python-3.5.1-zmq4``     ``intel/2016a``                
``15.3.0``    ``-Python-2.7.11-zmq4``    ``foss/2016a``                 
``15.3.0``    ``-Python-3.5.1-zmq4``     ``foss/2016a``                 
``15.4.0``    ``-Python-2.7.12-zmq4``    ``intel/2016b``                
``15.4.0``    ``-Python-3.5.2-zmq4``     ``intel/2016b``                
``16.0.2``    ``-Python-2.7.12-zmq4``    ``foss/2016b``, ``intel/2016b``
``16.0.2``    ``-Python-2.7.13-zmq4``    ``foss/2017a``, ``intel/2017a``
``16.0.2``    ``-Python-3.5.2-zmq4``     ``foss/2016b``, ``intel/2016b``
``16.0.3``    ``-Python-2.7.14-zmq4``    ``intel/2017b``                
``17.0.0``    ``-Python-2.7.14-zmq4``    ``foss/2018a``                 
``17.0.0``    ``-Python-3.6.4-zmq4``     ``foss/2018a``                 
==========    =======================    ===============================


.. _list_software_letter_q:

*Q*
---


:ref:`list_software_Q6_135` - :ref:`list_software_QCA_213` - :ref:`list_software_qcint_543` - :ref:`list_software_QDD_217` - :ref:`list_software_QEMU_312` - :ref:`list_software_QGIS_308` - :ref:`list_software_Qhull_518` - :ref:`list_software_QIIME_373` - :ref:`list_software_QIIME2_423` - :ref:`list_software_Qiskit_629` - :ref:`list_software_QJson_491` - :ref:`list_software_QML_234` - :ref:`list_software_qpth_445` - :ref:`list_software_qrupdate_870` - :ref:`list_software_QScintilla_1012` - :ref:`list_software_Qt_197` - :ref:`list_software_Qt5_250` - :ref:`list_software_Qt5Webkit_864` - :ref:`list_software_QtKeychain_1009` - :ref:`list_software_QTLtools_802` - :ref:`list_software_qtop_452` - :ref:`list_software_Quandl_613` - :ref:`list_software_QuantumESPRESSO_1375` - :ref:`list_software_QUAST_398` - :ref:`list_software_QuaZIP_538` - :ref:`list_software_QuickFF_649` - :ref:`list_software_Quip_415` - :ref:`list_software_Quorum_649` - :ref:`list_software_QuTiP_467` - :ref:`list_software_Qwt_316` - :ref:`list_software_QwtPolar_826`


.. _list_software_Q6_135:

*Q6*
++++

EVB, FEP and LIE simulator.

*homepage*: https://github.com/qusers/Q6

============    ===============
version         toolchain      
============    ===============
``20180205``    ``gompi/2019a``
============    ===============


.. _list_software_QCA_213:

*QCA*
+++++

Taking a hint from the similarly-named Java Cryptography Architecture,   QCA aims to provide a straightforward and cross-platform crypto API, using Qt datatypes   and conventions. QCA separates the API from the implementation, using plugins known as   Providers. The advantage of this model is to allow applications to avoid linking to or   explicitly depending on any particular cryptographic library. This allows one to easily   change or upgrade crypto implementations without even needing to recompile the   application! QCA should work everywhere Qt does, including Windows/Unix/MacOSX.

*homepage*: http://delta.affinix.com/qca/

=========    ==================================================
version      toolchain                                         
=========    ==================================================
``2.1.0``    ``foss/2016a``, ``intel/2016b``                   
``2.1.3``    ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``
=========    ==================================================


.. _list_software_qcint_543:

*qcint*
+++++++

libcint is an open source library for analytical Gaussian integrals. qcint is an optimized libcint branch for the x86-64 platform.

*homepage*: http://wiki.sunqm.net/libcint

==========    ==============
version       toolchain     
==========    ==============
``3.0.18``    ``foss/2019a``
==========    ==============


.. _list_software_QDD_217:

*QDD*
+++++

A user-friendly program to select microsatellite markers and design primers  from large sequencing projects.

*homepage*: http://net.imbe.fr/~emeglecz/qdd.html

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``3.1.2``    ``-Perl-5.28.0``    ``intel/2018b``
=========    ================    ===============


.. _list_software_QEMU_312:

*QEMU*
++++++

QEMU is a generic and open source machine emulator and virtualizer.

*homepage*: https://www.qemu.org/

==========    ===============
version       toolchain      
==========    ===============
``2.10.1``    ``intel/2017b``
==========    ===============


.. _list_software_QGIS_308:

*QGIS*
++++++

QGIS is a user friendly Open Source Geographic Information System (GIS)

*homepage*: http://www.qgis.org/

===========    ==================    ===============
version        versionsuffix         toolchain      
===========    ==================    ===============
``2.14.12``    ``-Python-2.7.12``    ``intel/2016b``
``2.18.4``     ``-Python-2.7.12``    ``foss/2016b`` 
``3.4.12``     ``-Python-3.7.2``     ``foss/2019a`` 
===========    ==================    ===============


.. _list_software_Qhull_518:

*Qhull*
+++++++

Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point, furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull. 

*homepage*: http://www.qhull.org

==========    =============================================================================================================================================================================
version       toolchain                                                                                                                                                                    
==========    =============================================================================================================================================================================
``2015.2``    ``GCCcore/5.4.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
==========    =============================================================================================================================================================================


.. _list_software_QIIME_373:

*QIIME*
+++++++

QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.

*homepage*: http://qiime.org/

=========    ==========
version      toolchain 
=========    ==========
``1.9.1``    ``system``
=========    ==========


.. _list_software_QIIME2_423:

*QIIME2*
++++++++

QIIME is an open-source bioinformatics pipeline for performing microbiome analysis  from raw DNA sequencing data.

*homepage*: http://qiime2.org/

===========    ==========
version        toolchain 
===========    ==========
``2017.10``    ``system``
``2018.2``     ``system``
``2019.4``     ``system``
``2019.7``     ``system``
===========    ==========


.. _list_software_Qiskit_629:

*Qiskit*
++++++++

Qiskit is an open-source framework for working with noisy quantum computers  at the level of pulses, circuits, and algorithms.

*homepage*: https://qiskit.org

==========    =================    ==============
version       versionsuffix        toolchain     
==========    =================    ==============
``0.11.1``    ``-Python-3.7.2``    ``foss/2019a``
``0.12.0``    ``-Python-3.7.2``    ``foss/2019a``
==========    =================    ==============


.. _list_software_QJson_491:

*QJson*
+++++++

QJson is a Qt-based library that maps JSON data to QVariant objects and vice versa.

*homepage*: http://qjson.sourceforge.net/

=========    ==================================================
version      toolchain                                         
=========    ==================================================
``0.9.0``    ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``
=========    ==================================================


.. _list_software_QML_234:

*QML*
+++++

QML is a Python2/3-compatible toolkit for representation learning of properties  of molecules and solids.

*homepage*: http://www.qmlcode.org

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.2.10``    ``-Python-2.7.13``    ``intel/2017a``
==========    ==================    ===============


.. _list_software_qpth_445:

*qpth*
++++++

A fast and differentiable QP solver for PyTorch. 

*homepage*: https://locuslab.github.io/qpth/

===================    =================    ==============
version                versionsuffix        toolchain     
===================    =================    ==============
``0.0.13-20190626``    ``-Python-3.7.2``    ``foss/2019a``
===================    =================    ==============


.. _list_software_qrupdate_870:

*qrupdate*
++++++++++

qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions.

*homepage*: https://sourceforge.net/projects/qrupdate/

=========    ========================================================================================================================================
version      toolchain                                                                                                                               
=========    ========================================================================================================================================
``1.1.2``    ``GCC/5.4.0-2.26``, ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2018a``, ``foss/2018b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
=========    ========================================================================================================================================


.. _list_software_QScintilla_1012:

*QScintilla*
++++++++++++

QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control

*homepage*: https://www.riverbankcomputing.com/software/qscintilla

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``2.9.4``     ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``2.10``      ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``2.11.2``    ``-Python-3.7.2``     ``GCCcore/8.2.0``              
==========    ==================    ===============================


.. _list_software_Qt_197:

*Qt*
++++

Qt is a comprehensive cross-platform C++ application framework.

*homepage*: http://qt.io/

=========    ================    ============================================================================================================================================================================================================================================
version      versionsuffix       toolchain                                                                                                                                                                                                                                   
=========    ================    ============================================================================================================================================================================================================================================
``3.3.8``                        ``intel/2016a``                                                                                                                                                                                                                             
``4.8.6``                        ``system``                                                                                                                                                                                                                                  
``4.8.7``                        ``GCCcore/8.2.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``4.8.7``    ``-GLib-2.48.0``    ``foss/2016a``, ``intel/2016a``                                                                                                                                                                                                             
=========    ================    ============================================================================================================================================================================================================================================


.. _list_software_Qt5_250:

*Qt5*
+++++

Qt is a comprehensive cross-platform C++ application framework.

*homepage*: http://qt.io/

==========    =================================================================================
version       toolchain                                                                        
==========    =================================================================================
``5.6.0``     ``foss/2016a``, ``intel/2016a``                                                  
``5.7.0``     ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                 
``5.7.1``     ``intel/2016b``                                                                  
``5.8.0``     ``foss/2017a``, ``foss/2017b``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``5.9.3``     ``foss/2017b``                                                                   
``5.9.8``     ``fosscuda/2018b``                                                               
``5.10.1``    ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``              
``5.11.2``    ``foss/2018b``                                                                   
``5.12.3``    ``GCCcore/8.2.0``                                                                
``5.13.1``    ``GCCcore/8.3.0``                                                                
==========    =================================================================================


.. _list_software_Qt5Webkit_864:

*Qt5Webkit*
+++++++++++

Qt Port of WebKit. WebKit is an open source web browser engine.

*homepage*: https://github.com/qt/qtwebkit

==================    =================
version               toolchain        
==================    =================
``5.212.0-alpha3``    ``GCCcore/8.2.0``
==================    =================


.. _list_software_QtKeychain_1009:

*QtKeychain*
++++++++++++

Platform-independent Qt API for storing passwords securely.

*homepage*: https://github.com/frankosterfeld/qtkeychain

=========    =================================
version      toolchain                        
=========    =================================
``0.9.1``    ``GCCcore/8.2.0``, ``foss/2018b``
=========    =================================


.. _list_software_QTLtools_802:

*QTLtools*
++++++++++

QTLtools is a tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs)  in few easy-to-perform steps.

*homepage*: https://qtltools.github.io/qtltools/

=======    ===============
version    toolchain      
=======    ===============
``1.1``    ``intel/2016b``
=======    ===============


.. _list_software_qtop_452:

*qtop*
++++++

qtop is a nifty command-line tool for monitoring queueing systems, esp. PBS/torque.  It tries to fit as much information as possible in your screen's real estate,  by stitching together the output of commands like pbsnodes -a, qstat & qstat -q.  It is possible to write wrappers for other platforms -people have done so for SGE, OAR etc-  or, even examine traces offline and present the sampled information.

*homepage*: http://cern.ch/fotis/QTOP/

=======    =============    ==========
version    versionsuffix    toolchain 
=======    =============    ==========
``53``     ``-1``           ``system``
=======    =============    ==========


.. _list_software_Quandl_613:

*Quandl*
++++++++

A Python library for Quandl’s RESTful API.

*homepage*: https://pypi.python.org/pypi/Quandl

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``3.4.2``    ``-Python-3.6.4``    ``intel/2018a``
``3.4.8``                         ``foss/2019a`` 
=========    =================    ===============


.. _list_software_QuantumESPRESSO_1375:

*QuantumESPRESSO*
+++++++++++++++++

Quantum ESPRESSO  is an integrated suite of computer codes  for electronic-structure calculations and materials modeling at the nanoscale.  It is based on density-functional theory, plane waves, and pseudopotentials   (both norm-conserving and ultrasoft). 

*homepage*: https://www.quantum-espresso.org

=========    =============    ===============================
version      versionsuffix    toolchain                      
=========    =============    ===============================
``5.3.0``                     ``intel/2016.02-GCC-4.9``      
``5.4.0``                     ``intel/2016.02-GCC-4.9``      
``5.4.0``    ``-hybrid``      ``foss/2016b``                 
``6.0``                       ``intel/2016b``                
``6.1``                       ``intel/2017a``                
``6.2``                       ``intel/2017b``                
``6.2.1``                     ``iomkl/2017b``                
``6.3``                       ``foss/2018b``, ``intel/2018b``
``6.4.1``                     ``intel/2019a``                
=========    =============    ===============================


.. _list_software_QUAST_398:

*QUAST*
+++++++

QUAST evaluates genome assemblies by computing various metrics.  It works both with and without reference genomes. The tool accepts multiple  assemblies, thus is suitable for comparison.

*homepage*: https://github.com/ablab/quast

=========    ==================    ==============================
version      versionsuffix         toolchain                     
=========    ==================    ==============================
``4.6.0``    ``-Python-3.5.2``     ``foss/2016b``                
``4.6.3``    ``-Python-3.6.4``     ``foss/2018a``                
``5.0.2``    ``-Python-2.7.15``    ``foss/2018b``, ``foss/2019a``
=========    ==================    ==============================


.. _list_software_QuaZIP_538:

*QuaZIP*
++++++++

QuaZIP is the C++ wrapper for Gilles Vollant's ZIP/UNZIP   package (AKA Minizip) using Trolltech's Qt library.

*homepage*: https://stachenov.github.io/quazip

=========    =================
version      toolchain        
=========    =================
``0.8.1``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_QuickFF_649:

*QuickFF*
+++++++++

QuickFF is a Python package developed at the Center for Molecular Modeling (CMM) to quickly derive accurate force fields from ab initio calculations.

*homepage*: http://molmod.github.io/QuickFF/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.1.4``    ``-Python-2.7.12``    ``intel/2016b``
``2.2.0``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_Quip_415:

*Quip*
++++++

Quip compresses next-generation sequencing data with extreme prejudice.  It supports input and output in the FASTQ and SAM/BAM formats,  compressing large datasets to as little as 15% of their original size.

*homepage*: http://homes.cs.washington.edu/~dcjones/quip

=========    =============
version      toolchain    
=========    =============
``1.1.8``    ``GCC/4.8.2``
=========    =============


.. _list_software_Quorum_649:

*Quorum*
++++++++

QuorUM is an error corrector for Illumina reads

*homepage*: http://www.genome.umd.edu/quorum.html

=========    ===============
version      toolchain      
=========    ===============
``1.1.1``    ``intel/2017a``
=========    ===============


.. _list_software_QuTiP_467:

*QuTiP*
+++++++

QuTiP is open-source software for simulating the dynamics of open quantum systems.

*homepage*: http://qutip.org

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``4.1.0``    ``-Python-2.7.12``    ``intel/2016b``                
``4.3.1``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
=========    ==================    ===============================


.. _list_software_Qwt_316:

*Qwt*
+++++

The Qwt library contains GUI Components and utility classes which are primarily useful for programs  with a technical background.

*homepage*: http://qwt.sourceforge.net/

=========    ===============================
version      toolchain                      
=========    ===============================
``6.1.2``    ``intel/2016a``                
``6.1.3``    ``foss/2016b``, ``intel/2016b``
``6.1.4``    ``GCCcore/8.2.0``              
=========    ===============================


.. _list_software_QwtPolar_826:

*QwtPolar*
++++++++++

The QwtPolar library contains classes for displaying values on a polar coordinate system.

*homepage*: http://qwtpolar.sourceforge.net/

=========    ==================================================
version      toolchain                                         
=========    ==================================================
``1.1.1``    ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2016b``
=========    ==================================================


.. _list_software_letter_r:

*R*
---


:ref:`list_software_R_82` - :ref:`list_software_R-bundle-Bioconductor_2065` - :ref:`list_software_R-keras_661` - :ref:`list_software_R-tesseract_1101` - :ref:`list_software_Racon_499` - :ref:`list_software_Ragout_626` - :ref:`list_software_randfold_842` - :ref:`list_software_randrproto_1099` - :ref:`list_software_RapidJSON_810` - :ref:`list_software_rapidtide_950` - :ref:`list_software_Rascaf_592` - :ref:`list_software_RASPA2_425` - :ref:`list_software_RAxML_420` - :ref:`list_software_RBFOpt_525` - :ref:`list_software_Rcorrector_1061` - :ref:`list_software_rCUDA_399` - :ref:`list_software_RDFlib_531` - :ref:`list_software_RDKit_446` - :ref:`list_software_RDP-Classifier_1304` - :ref:`list_software_re2c_364` - :ref:`list_software_Reads2snp_882` - :ref:`list_software_Redundans_932` - :ref:`list_software_ReFrame_674` - :ref:`list_software_RELION_457` - :ref:`list_software_REMORA_454` - :ref:`list_software_renderproto_1204` - :ref:`list_software_RepastHPC_842` - :ref:`list_software_RepeatMasker_1220` - :ref:`list_software_requests_892` - :ref:`list_software_RERconverge_1090` - :ref:`list_software_rgdal_522` - :ref:`list_software_rgeos_544` - :ref:`list_software_rhdf5_473` - :ref:`list_software_rioxarray_993` - :ref:`list_software_rjags_535` - :ref:`list_software_Rmath_508` - :ref:`list_software_RMBlast_661` - :ref:`list_software_RNA-SeQC_602` - :ref:`list_software_RNAclust_780` - :ref:`list_software_RNAcode_636` - :ref:`list_software_RNAIndel_717` - :ref:`list_software_rnaQUAST_719` - :ref:`list_software_RNAz_347` - :ref:`list_software_Roary_525` - :ref:`list_software_ROI_PAC_541` - :ref:`list_software_ROME_307` - :ref:`list_software_ROOT_324` - :ref:`list_software_rootpy_685` - :ref:`list_software_Rosetta_738` - :ref:`list_software_rpmrebuild_1078` - :ref:`list_software_rpy2_397` - :ref:`list_software_RQGIS3_441` - :ref:`list_software_RSEM_311` - :ref:`list_software_RSeQC_414` - :ref:`list_software_RStan_488` - :ref:`list_software_RTG-Tools_811` - :ref:`list_software_Rtree_514` - :ref:`list_software_Ruby_418` - :ref:`list_software_Rust_430`


.. _list_software_R_82:

*R*
+++

R is a free software environment for statistical computing  and graphics.

*homepage*: https://www.r-project.org/

=========    ==================    ===================================================
version      versionsuffix         toolchain                                          
=========    ==================    ===================================================
``3.2.3``                          ``foss/2016a``, ``foss/2016b``, ``intel/2016a``    
``3.2.3``    ``-bare``             ``foss/2016a``, ``intel/2016a``                    
``3.2.3``    ``-libX11-1.6.3``     ``intel/2016a``                                    
``3.3.1``                          ``foss/2016a``, ``foss/2016b``, ``intel/2016b``    
``3.3.3``    ``-X11-20160819``     ``foss/2016b``, ``intel/2016b``                    
``3.3.3``    ``-X11-20170314``     ``intel/2017a``                                    
``3.4.0``    ``-X11-20170314``     ``intel/2017a``                                    
``3.4.1``    ``-X11-20160819``     ``foss/2016b``                                     
``3.4.3``    ``-X11-20171023``     ``foss/2017b``, ``intel/2017b``                    
``3.4.4``    ``-X11-20180131``     ``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``   
``3.5.0``    ``-X11-20180131``     ``iomkl/2018a``                                    
``3.5.1``                          ``foss/2018b``, ``intel/2018b``                    
``3.5.1``    ``-Python-2.7.15``    ``foss/2018b``                                     
``3.6.0``                          ``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``
=========    ==================    ===================================================


.. _list_software_R-bundle-Bioconductor_2065:

*R-bundle-Bioconductor*
+++++++++++++++++++++++

Bioconductor provides tools for the analysis and coprehension  of high-throughput genomic data.

*homepage*: https://bioconductor.org

=======    =============    ===============================
version    versionsuffix    toolchain                      
=======    =============    ===============================
``3.2``    ``-R-3.2.3``     ``foss/2016a``, ``intel/2016a``
``3.3``    ``-R-3.3.1``     ``intel/2016b``                
``3.5``    ``-R-3.4.0``     ``intel/2017a``                
``3.6``    ``-R-3.4.3``     ``foss/2017b``, ``intel/2017b``
``3.6``    ``-R-3.4.4``     ``intel/2018a``                
``3.7``    ``-R-3.5.0``     ``iomkl/2018a``                
``3.7``    ``-R-3.5.1``     ``foss/2018b``                 
``3.8``    ``-R-3.5.1``     ``foss/2018b``                 
``3.9``    ``-R-3.6.0``     ``foss/2019a``                 
=======    =============    ===============================


.. _list_software_R-keras_661:

*R-keras*
+++++++++

Interface to 'Keras' <https://keras.io>, a high-level neural networks 'API'. 

*homepage*: https://cran.r-project.org/web/packages/keras

===========    =========================    ==================================
version        versionsuffix                toolchain                         
===========    =========================    ==================================
``2.1.6``      ``-R-3.4.4``                 ``foss/2018a``                    
``2.2.5.0``    ``-Python-3.7.2-R-3.6.0``    ``foss/2019a``, ``fosscuda/2019a``
===========    =========================    ==================================


.. _list_software_R-tesseract_1101:

*R-tesseract*
+++++++++++++

The R extension for using tesseract

*homepage*: https://cran.r-project.org/package=tesseract

=======    =============    ==============
version    versionsuffix    toolchain     
=======    =============    ==============
``4.0``    ``-R-3.5.1``     ``foss/2018b``
=======    =============    ==============


.. _list_software_Racon_499:

*Racon*
+++++++

Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.

*homepage*: https://github.com/lbcb-sci/racon

=========    =================
version      toolchain        
=========    =================
``1.3.2``    ``GCCcore/8.2.0``
``1.4.7``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_Ragout_626:

*Ragout*
++++++++

Ragout (Reference-Assisted Genome Ordering UTility) is a tool for chromosome assembly using multiple  references. Given a set of assembly fragments (contigs/scaffolds) and one or multiple related references (complete or  draft), it produces a chromosome-scale assembly (as a set of scaffolds).

*homepage*: http://fenderglass.github.io/Ragout/

=======    ==================    ==============
version    versionsuffix         toolchain     
=======    ==================    ==============
``2.0``    ``-Python-2.7.12``    ``foss/2016b``
=======    ==================    ==============


.. _list_software_randfold_842:

*randfold*
++++++++++

Minimum free energy of folding randomization test software

*homepage*: http://bioinformatics.psb.ugent.be/software/details/Randfold

=========    ===============================
version      toolchain                      
=========    ===============================
``2.0.1``    ``foss/2018b``, ``intel/2016b``
=========    ===============================


.. _list_software_randrproto_1099:

*randrproto*
++++++++++++

Xrandr protocol and ancillary headers

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.5.0``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_RapidJSON_810:

*RapidJSON*
+++++++++++

A fast JSON parser/generator for C++ with both SAX/DOM style API

*homepage*: https://rapidjson.org

=========    =================
version      toolchain        
=========    =================
``1.1.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_rapidtide_950:

*rapidtide*
+++++++++++

Rapidtide is a suite of python programs used to perform time delay analysis on functional imaging  data to find time lagged correlations between the voxelwise time series and other time series.

*homepage*: https://rapidtide.readthedocs.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.8.0``    ``-Python-3.7.2``    ``intel/2019a``
=========    =================    ===============


.. _list_software_Rascaf_592:

*Rascaf*
++++++++

Rascaf (RnA-seq SCAFfolder) uses continuity and order information from paired-end RNA-seq reads  to improve a draft assembly, particularly in the gene regions.

*homepage*: https://github.com/mourisl/Rascaf

=========    ===============
version      toolchain      
=========    ===============
``1.0.2``    ``intel/2017a``
=========    ===============


.. _list_software_RASPA2_425:

*RASPA2*
++++++++

A general purpose classical simulation package that can be used for the simulation of  molecules in gases, fluids, zeolites, aluminosilicates,  metal-organic frameworks, carbon nanotubes and external fields.

*homepage*: https://github.com/numat/RASPA2

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.0.3``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_RAxML_420:

*RAxML*
+++++++

RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.

*homepage*: https://github.com/stamatak/standard-RAxML

==========    ================    ================================================
version       versionsuffix       toolchain                                       
==========    ================    ================================================
``8.2.4``     ``-hybrid-avx2``    ``foss/2016a``                                  
``8.2.9``     ``-hybrid-avx2``    ``foss/2016a``                                  
``8.2.10``    ``-hybrid-avx2``    ``intel/2017a``                                 
``8.2.11``    ``-hybrid-avx``     ``foss/2017b``, ``intel/2017b``                 
``8.2.11``    ``-hybrid-avx2``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``8.2.11``    ``-hybrid-sse3``    ``foss/2017b``, ``intel/2017b``                 
``8.2.12``    ``-hybrid-avx2``    ``intel/2019a``                                 
==========    ================    ================================================


.. _list_software_RBFOpt_525:

*RBFOpt*
++++++++

RBFOpt is a Python library for black-box optimization (also known as derivative-free optimization).

*homepage*: https://github.com/coin-or/rbfopt

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``4.1.1``                         ``intel/2019a``
``4.1.1``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_Rcorrector_1061:

*Rcorrector*
++++++++++++

Rcorrector(RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data.

*homepage*: https://github.com/mourisl/Rcorrector

=========    ===============
version      toolchain      
=========    ===============
``1.0.2``    ``intel/2017a``
=========    ===============


.. _list_software_rCUDA_399:

*rCUDA*
+++++++

The rCUDA Framework enables the concurrent usage of CUDA-compatible devices remotely.

*homepage*: http://www.rcuda.net/

=========    ==============================    ==========
version      versionsuffix                     toolchain 
=========    ==============================    ==========
``4.0.1``    ``_linux_64_Ubuntu10.04``         ``system``
``5.0``      ``_linux_64_scientificLinux6``    ``system``
=========    ==============================    ==========


.. _list_software_RDFlib_531:

*RDFlib*
++++++++

RDFLib is a Python library for working with RDF, a simple yet powerful language  for representing information.

*homepage*: https://github.com/RDFLib/rdflib

=========    ==============
version      toolchain     
=========    ==============
``4.2.2``    ``foss/2019a``
=========    ==============


.. _list_software_RDKit_446:

*RDKit*
+++++++

RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.

*homepage*: https://www.rdkit.org

=============    =================    ===============
version          versionsuffix        toolchain      
=============    =================    ===============
``2018.09.3``    ``-Python-3.6.6``    ``intel/2018b``
=============    =================    ===============


.. _list_software_RDP-Classifier_1304:

*RDP-Classifier*
++++++++++++++++

The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic  assignments from domain to genus, with confidence estimates for each assignment.

*homepage*: http://sourceforge.net/projects/rdp-classifier

========    ==================    ==========
version     versionsuffix         toolchain 
========    ==================    ==========
``2.7``     ``-Java-1.7.0_60``    ``system``
``2.12``    ``-Java-1.8``         ``system``
========    ==================    ==========


.. _list_software_re2c_364:

*re2c*
++++++

re2c is a free and open-source lexer generator for C and C++. Its main goal is generating fast lexers: at least as fast as their reasonably optimized hand-coded counterparts. Instead of using traditional table-driven approach, re2c encodes the generated finite state automata directly in the form of conditional jumps and comparisons.

*homepage*: https://re2c.org/

=========    =================
version      toolchain        
=========    =================
``1.1.1``    ``GCCcore/8.2.0``
``1.2.1``    ``GCCcore/8.3.0``
=========    =================


.. _list_software_Reads2snp_882:

*Reads2snp*
+++++++++++

reads2snp is a SNP and genotype caller: it predicts the genotype of distinct individuals   at distinct positions of a set of sequences based on read mapping / read counts. Its typical input is a bam file.   Its typical output is a vcf file. It is written in C++, based on the bio++ libraries, multi-threaded with openMP,   available under Linux and MacOS

*homepage*: http://kimura.univ-montp2.fr/PopPhyl/index.php?section=tools

=======    ==========
version    toolchain 
=======    ==========
``2.0``    ``system``
=======    ==========


.. _list_software_Redundans_932:

*Redundans*
+++++++++++

Redundans is a pipeline that assists an assembly of heterozygous/polymorphic genomes.

*homepage*: https://github.com/lpryszcz/redundans

=========    ===============
version      toolchain      
=========    ===============
``0.13c``    ``intel/2017b``
=========    ===============


.. _list_software_ReFrame_674:

*ReFrame*
+++++++++

ReFrame is a framework for writing regression tests for HPC systems.

*homepage*: https://github.com/eth-cscs/reframe

========    ==========
version     toolchain 
========    ==========
``2.18``    ``system``
``2.19``    ``system``
========    ==========


.. _list_software_RELION_457:

*RELION*
++++++++

RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer  program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class  averages in electron cryo-microscopy (cryo-EM).

*homepage*: http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page

=======================    ================    ============================================================================================================
version                    versionsuffix       toolchain                                                                                                   
=======================    ================    ============================================================================================================
``1.4``                                        ``intel/2016b``                                                                                             
``1.4``                    ``-single``         ``intel/2016b``                                                                                             
``2.0.1``                                      ``intel/2016b``                                                                                             
``2.1``                                        ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``fosscuda/2018a``, ``intel/2017b``, ``intelcuda/2017b``
``2.1``                    ``-CUDA-9.1.85``    ``foss/2018a``                                                                                              
``3.0.4``                                      ``foss/2017b``, ``intel/2017b``                                                                             
``3.0_beta.2018.08.02``                        ``fosscuda/2018a``, ``intel/2018a``                                                                         
=======================    ================    ============================================================================================================


.. _list_software_REMORA_454:

*REMORA*
++++++++

REsource MOnitoring for Remote Applications

*homepage*: https://github.com/TACC/remora

=========    ================================================================
version      toolchain                                                       
=========    ================================================================
``1.8.2``    ``foss/2017a``, ``foss/2018a``, ``intel/2017a``, ``intel/2018a``
``1.8.3``    ``gompi/2019a``                                                 
=========    ================================================================


.. _list_software_renderproto_1204:

*renderproto*
+++++++++++++

Xrender protocol and ancillary headers

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

========    ==================================================================
version     toolchain                                                         
========    ==================================================================
``0.11``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``
========    ==================================================================


.. _list_software_RepastHPC_842:

*RepastHPC*
+++++++++++

The Repast Suite is a family of advanced, free, and open source agent-based modeling and simulation platforms that have collectively been under continuous development for over 15 years:  Repast for High Performance Computing 2.2.0, released on 30 September 2016, is a lean and expert-focused C++-based  modeling system that is designed for use on large computing clusters and supercomputers.

*homepage*: https://repast.github.io/

=========    ==============
version      toolchain     
=========    ==============
``2.2.0``    ``foss/2016a``
=========    ==============


.. _list_software_RepeatMasker_1220:

*RepeatMasker*
++++++++++++++

RepeatMasker is a program that screens DNA sequences for interspersed repeats  and low complexity DNA sequences.

*homepage*: http://www.repeatmasker.org/

============    ======================    ==========================================================
version         versionsuffix             toolchain                                                 
============    ======================    ==========================================================
``4.0.8``       ``-Perl-5.26.0-HMMER``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
``4.0.8``       ``-Perl-5.28.0-HMMER``    ``intel/2018b``                                           
``4.0.9-p2``    ``-HMMER``                ``gompi/2019b``                                           
============    ======================    ==========================================================


.. _list_software_requests_892:

*requests*
++++++++++

Python http for humans

*homepage*: https://pypi.python.org/pypi/requests

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``2.10.0``    ``-Python-2.7.11``    ``foss/2016a``                 
``2.10.0``    ``-Python-3.5.1``     ``foss/2016a``                 
``2.11.1``    ``-Python-2.7.12``    ``intel/2016b``                
``2.11.1``    ``-Python-3.5.2``     ``intel/2016b``                
``2.13.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
==========    ==================    ===============================


.. _list_software_RERconverge_1090:

*RERconverge*
+++++++++++++

RERconverge is a set of software written in R that  estimates the correlation between relative evolutionary rates of  genes and the evolution of a convergent binary or continuous trait  across a phylogeny. 

*homepage*: https://github.com/nclark-lab/RERconverge

=========    =============    ===============================
version      versionsuffix    toolchain                      
=========    =============    ===============================
``0.1.0``    ``-R-3.4.3``     ``foss/2017b``, ``intel/2017b``
=========    =============    ===============================


.. _list_software_rgdal_522:

*rgdal*
+++++++

Provides bindings to the 'Geospatial' Data Abstraction Library ('GDAL') (>= 1.11.4 and <= 2.5.0) and  access to projection/transformation operations from the 'PROJ.4' library.

*homepage*: http://rgdal.r-forge.r-project.org/

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``1.4-4``    ``-R-3.6.0``     ``foss/2019a``
=========    =============    ==============


.. _list_software_rgeos_544:

*rgeos*
+++++++

R interface to Geometry Engine - Open Source (GEOS) using the C API for topology operations  on geometries

*homepage*: https://cran.r-project.org/web/packages/rgeos/

==========    =============    ===============
version       versionsuffix    toolchain      
==========    =============    ===============
``0.3-17``    ``-R-3.2.3``     ``intel/2016a``
``0.5-1``     ``-R-3.6.0``     ``foss/2019a`` 
==========    =============    ===============


.. _list_software_rhdf5_473:

*rhdf5*
+++++++

This R/Bioconductor package provides an interface between HDF5 and R.

*homepage*: https://bioconductor.org/packages/release/bioc/html/rhdf5.html

==========    =============    ===============
version       versionsuffix    toolchain      
==========    =============    ===============
``2.16.0``    ``-R-3.2.3``     ``intel/2016a``
``2.18.0``    ``-R-3.3.1``     ``intel/2016b``
==========    =============    ===============


.. _list_software_rioxarray_993:

*rioxarray*
+++++++++++

geospatial xarray extension powered by rasterio

*homepage*: https://github.com/corteva/rioxarray

==========    =================    ===============
version       versionsuffix        toolchain      
==========    =================    ===============
``0.0.12``    ``-Python-3.7.2``    ``intel/2019a``
==========    =================    ===============


.. _list_software_rjags_535:

*rjags*
+++++++

The rjags package is an interface to the JAGS library.

*homepage*: http://cran.r-project.org/web/packages/rjags

=======    =============    ===============
version    versionsuffix    toolchain      
=======    =============    ===============
``4-6``    ``-R-3.4.0``     ``intel/2017a``
``4-6``    ``-R-3.4.3``     ``intel/2017b``
``4-8``    ``-R-3.5.1``     ``foss/2018b`` 
=======    =============    ===============


.. _list_software_Rmath_508:

*Rmath*
+++++++

Rmath is the standalone version of the R math library.  Rmath can be used in your own C/C++ routines.

*homepage*: https://www.r-project.org/

=========    ===============
version      toolchain      
=========    ===============
``3.3.1``    ``intel/2016b``
=========    ===============


.. _list_software_RMBlast_661:

*RMBlast*
+++++++++

RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary difference between  this distribution and the NCBI distribution is the addition of a new program 'rmblastn' for use with RepeatMasker and  RepeatModeler.

*homepage*: http://www.repeatmasker.org/RMBlast.html

==========    ==================    ==============
version       versionsuffix         toolchain     
==========    ==================    ==============
``2.2.28``    ``-Python-2.7.11``    ``foss/2016a``
==========    ==================    ==============


.. _list_software_RNA-SeQC_602:

*RNA-SeQC*
++++++++++

RNA-SeQC is a java program which computes a series of quality control metrics for  RNA-seq data. The input can be one or more BAM files. The output consists of HTML reports and tab  delimited files of metrics data. This program can be valuable for comparing sequencing quality  across different samples or experiments to evaluate different experimental parameters. It can  also be run on individual samples as a means of quality control before continuing with downstream  analysis.

*homepage*: https://software.broadinstitute.org/cancer/cga/rna-seqc

=========    ===================    ==============
version      versionsuffix          toolchain     
=========    ===================    ==============
``1.1.8``    ``-Java-1.8``          ``foss/2018b``
``1.1.8``    ``-Java-1.8.0_121``    ``foss/2016b``
=========    ===================    ==============


.. _list_software_RNAclust_780:

*RNAclust*
++++++++++

RNAclust is a perl script summarizing all the single steps required for clustering  of structured RNA motifs, i.e. identifying groups of RNA sequences sharing a secondary structure  motif. It requires as input a multiple FASTA file.

*homepage*: http://www.bioinf.uni-leipzig.de/~kristin/Software/RNAclust/

=======    =================    ==============
version    versionsuffix        toolchain     
=======    =================    ==============
``1.3``    ``-Perl-5.24.0``     ``foss/2016b``
``1.3``    ``-Python-3.6.6``    ``foss/2018b``
=======    =================    ==============


.. _list_software_RNAcode_636:

*RNAcode*
+++++++++

RNAcode - Analyze the protein coding potential in multiple sequence alignments

*homepage*: https://wash.github.io/rnacode/

=======    ==============
version    toolchain     
=======    ==============
``0.3``    ``foss/2017a``
=======    ==============


.. _list_software_RNAIndel_717:

*RNAIndel*
++++++++++

RNAIndel calls coding indels and classifies them into somatic, germline,  and artifact from tumor RNA-Seq data.

*homepage*: https://github.com/stjude/RNAIndel

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.3.0``    ``-Python-3.6.6``    ``intel/2018b``
``1.0.0``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_rnaQUAST_719:

*rnaQUAST*
++++++++++

rnaQUAST is a tool for evaluating RNA-Seq assemblies using reference genome and gene database. In addition, rnaQUAST is also capable of estimating gene database coverage by raw reads and de novo quality assessment using third-party software.

*homepage*: http://cab.spbu.ru/software/rnaquast/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.5.2``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_RNAz_347:

*RNAz*
++++++

RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.

*homepage*: http://www.tbi.univie.ac.at/~wash/RNAz/

=======    ==============================
version    toolchain                     
=======    ==============================
``2.1``    ``foss/2016b``, ``foss/2018b``
=======    ==============================


.. _list_software_Roary_525:

*Roary*
+++++++

Rapid large-scale prokaryote pan genome analysis

*homepage*: https://github.com/sanger-pathogens/Roary

==========    ================    ===============
version       versionsuffix       toolchain      
==========    ================    ===============
``3.12.0``                        ``system``     
``3.12.0``    ``-Perl-5.26.1``    ``intel/2018a``
==========    ================    ===============


.. _list_software_ROI_PAC_541:

*ROI_PAC*
+++++++++

Repeat Orbit Interferometry PACkage (ROI_PAC), software for processing synthetic aperture radar data  to produce differential interferograms

*homepage*: http://roipac.org/

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``3.0.1``    ``-Perl-5.24.1``    ``intel/2017a``
=========    ================    ===============


.. _list_software_ROME_307:

*ROME*
++++++

The ROME (Refinement and Optimization via Machine Learning for cryo-EM)  Software package is one of the major research products at the Intel® PCCSB.

*homepage*: https://ipccsb.dfci.harvard.edu/rome

=========    =================
version      toolchain        
=========    =================
``1.1.2``    ``intel/2019.02``
=========    =================


.. _list_software_ROOT_324:

*ROOT*
++++++

The ROOT system provides a set of OO frameworks with all the functionality     needed to handle and analyze large amounts of data in a very efficient way.

*homepage*: http://root.cern.ch/drupal/

============    ==================    ===============================
version         versionsuffix         toolchain                      
============    ==================    ===============================
``6.10.02``     ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``6.10.04``     ``-Python-2.7.13``    ``intel/2017a``                
``6.10.08``     ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``6.14.06``     ``-Python-2.7.15``    ``foss/2018b``                 
``v5.34.34``    ``-Python-2.7.11``    ``intel/2016a``                
``v5.34.36``    ``-Python-2.7.11``    ``intel/2016a``                
``v6.06.02``    ``-Python-2.7.12``    ``intel/2016b``                
``v6.08.02``    ``-Python-2.7.11``    ``foss/2016a``                 
============    ==================    ===============================


.. _list_software_rootpy_685:

*rootpy*
++++++++

The rootpy project is a community-driven initiative aiming to provide a more pythonic interface with  ROOT on top of the existing PyROOT bindings. Given Python’s reflective and dynamic nature, rootpy also aims to  improve ROOT design flaws and supplement existing ROOT functionality. The scientific Python community also offers a  multitude of powerful packages such as SciPy, NumPy, matplotlib, scikit-learn, and PyTables, but a suitable interface  between them and ROOT has been lacking. rootpy provides the interfaces and conversion mechanisms required to liberate  your data and to take advantage of these alternatives if needed.

*homepage*: http://www.rootpy.org/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``0.8.0``    ``-Python-2.7.11``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_Rosetta_738:

*Rosetta*
+++++++++

Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids.

*homepage*: https://www.rosettacommons.org

=================    ==============
version              toolchain     
=================    ==============
``3.7``              ``foss/2016b``
``2016.13.58602``    ``foss/2016a``
``2016.46.59086``    ``foss/2016b``
=================    ==============


.. _list_software_rpmrebuild_1078:

*rpmrebuild*
++++++++++++

rpmrebuild is a tool to build an RPM file from a package that has already been     installed in a basic use

*homepage*: http://rpmrebuild.sourceforge.net/

========    ==========
version     toolchain 
========    ==========
``2.11``    ``system``
========    ==========


.. _list_software_rpy2_397:

*rpy2*
++++++

rpy2 is a redesign and rewrite of rpy. It is providing a low-level  interface to R from Python, a proposed high-level interface, including wrappers to  graphical libraries, as well as R-like structures and functions. 

*homepage*: http://rpy.sourceforge.net/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``2.7.9``    ``-Python-2.7.12``    ``foss/2016b`` 
``2.8.2``    ``-Python-2.7.13``    ``intel/2017a``
=========    ==================    ===============


.. _list_software_RQGIS3_441:

*RQGIS3*
++++++++

RQGIS3 establishes an interface between R and QGIS3, i.e.,  it allows the user to access QGIS3 functionalities from within R.

*homepage*: https://github.com/r-spatial/RQGIS3

============    =============    ==============
version         versionsuffix    toolchain     
============    =============    ==============
``20190903``    ``-R-3.6.0``     ``foss/2019a``
============    =============    ==============


.. _list_software_RSEM_311:

*RSEM*
++++++

RNA-Seq by Expectation-Maximization

*homepage*: http://deweylab.github.io/RSEM/

==========    ================================================
version       toolchain                                       
==========    ================================================
``1.2.26``    ``GNU/4.9.3-2.25``                              
``1.2.30``    ``foss/2016a``, ``intel/2016b``                 
``1.3.0``     ``foss/2016b``, ``intel/2017a``                 
``1.3.1``     ``foss/2017b``, ``intel/2017b``, ``intel/2018a``
``1.3.2``     ``foss/2018b``                                  
==========    ================================================


.. _list_software_RSeQC_414:

*RSeQC*
+++++++

RSeQC provides a number of useful modules that can   comprehensively evaluate high throughput sequence data especially RNA-seq   data. Some basic modules quickly inspect sequence quality, nucleotide   composition bias, PCR bias and GC bias, while RNA-seq specific modules   evaluate sequencing saturation, mapped reads distribution, coverage   uniformity, strand specificity, transcript level RNA integrity etc.

*homepage*: http://rseqc.sourceforge.net/

=========    ==========================    ===============
version      versionsuffix                 toolchain      
=========    ==========================    ===============
``2.6.4``    ``-Python-2.7.12-R-3.3.1``    ``foss/2016b`` 
``2.6.4``    ``-Python-2.7.14``            ``intel/2018a``
``3.0.0``    ``-Python-3.6.6``             ``foss/2018b`` 
=========    ==========================    ===============


.. _list_software_RStan_488:

*RStan*
+++++++

RStan is the R interface to Stan. Stan is a state-of-the-art   platform for statistical modeling and high-performance statistical computation.

*homepage*: https://github.com/stan-dev/rstan

==========    =============    ===============================
version       versionsuffix    toolchain                      
==========    =============    ===============================
``2.18.2``    ``-R-3.4.3``     ``foss/2017b``, ``intel/2017b``
``2.18.2``    ``-R-3.5.1``     ``foss/2018b``                 
==========    =============    ===============================


.. _list_software_RTG-Tools_811:

*RTG-Tools*
+++++++++++

RTG Tools contains utilities to easily manipulate and accurately compare multiple VCF files,  as well as utilities for processing other common NGS data formats. 

*homepage*: https://www.realtimegenomics.com/products/rtg-tools

=========    =============    ==========
version      versionsuffix    toolchain 
=========    =============    ==========
``3.9.1``    ``-Java-1.8``    ``system``
=========    =============    ==========


.. _list_software_Rtree_514:

*Rtree*
+++++++

Rtree is a ctypes Python wrapper of libspatialindex that provides a number of advanced spatial  indexing features for the spatially curious Python user.

*homepage*: http://toblerity.org/rtree/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.8.3``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_Ruby_418:

*Ruby*
++++++

Ruby is a dynamic, open source programming language with   a focus on simplicity and productivity. It has an elegant syntax that is   natural to read and easy to write.

*homepage*: https://www.ruby-lang.org

=========    ================================================
version      toolchain                                       
=========    ================================================
``2.1.6``    ``system``                                      
``2.3.1``    ``intel/2016b``, ``system``                     
``2.3.3``    ``system``                                      
``2.3.4``    ``system``                                      
``2.4.2``    ``foss/2017b``                                  
``2.5.0``    ``foss/2018a``, ``intel/2017a``, ``intel/2017b``
``2.5.1``    ``intel/2018a``                                 
``2.6.1``    ``GCCcore/7.3.0``                               
``2.6.3``    ``GCCcore/8.2.0``                               
=========    ================================================


.. _list_software_Rust_430:

*Rust*
++++++

Rust is a systems programming language that runs blazingly fast, prevents segfaults,  and guarantees thread safety.

*homepage*: https://www.rust-lang.org

==========    =================
version       toolchain        
==========    =================
``1.8.0``     ``foss/2016a``   
``1.12.0``    ``foss/2016b``   
``1.12.1``    ``foss/2016b``   
``1.18.0``    ``foss/2017a``   
``1.21.0``    ``foss/2017b``   
``1.22.1``    ``GCCcore/6.4.0``
``1.29.2``    ``GCCcore/7.3.0``
``1.30.1``    ``GCCcore/6.4.0``
``1.35.0``    ``GCCcore/8.2.0``
``1.37.0``    ``GCCcore/8.3.0``
==========    =================


.. _list_software_letter_s:

*S*
---


:ref:`list_software_S-Lang_514` - :ref:`list_software_S4_135` - :ref:`list_software_SAGE_288` - :ref:`list_software_Sailfish_819` - :ref:`list_software_SALib_427` - :ref:`list_software_Salmon_618` - :ref:`list_software_SALMON-TDDFT_877` - :ref:`list_software_Sambamba_788` - :ref:`list_software_Samcef_591` - :ref:`list_software_samclip_745` - :ref:`list_software_SAMtools_786` - :ref:`list_software_SAS_231` - :ref:`list_software_savvy_569` - :ref:`list_software_Saxon-HE_707` - :ref:`list_software_sbt_329` - :ref:`list_software_ScaLAPACK_707` - :ref:`list_software_Scalasca_795` - :ref:`list_software_SCALCE_427` - :ref:`list_software_sciClone_816` - :ref:`list_software_ScientificPython_1667` - :ref:`list_software_scikit-allel_1214` - :ref:`list_software_scikit-image_1207` - :ref:`list_software_scikit-learn_1222` - :ref:`list_software_scikit-multilearn_1777` - :ref:`list_software_scikit-optimize_1573` - :ref:`list_software_SCIPhI_480` - :ref:`list_software_scipy_552` - :ref:`list_software_SciPy-bundle_1167` - :ref:`list_software_SCnorm_594` - :ref:`list_software_Scoary_625` - :ref:`list_software_SCons_486` - :ref:`list_software_SCOOP_388` - :ref:`list_software_Score-P_633` - :ref:`list_software_SCOTCH_452` - :ref:`list_software_scp_326` - :ref:`list_software_Scrappie_823` - :ref:`list_software_SDCC_285` - :ref:`list_software_SDL2_277` - :ref:`list_software_Seaborn_714` - :ref:`list_software_SearchGUI_827` - :ref:`list_software_segemehl_842` - :ref:`list_software_SelEstim_806` - :ref:`list_software_sep_328` - :ref:`list_software_Seq-Gen_624` - :ref:`list_software_seq2HLA_592` - :ref:`list_software_SeqAn_472` - :ref:`list_software_SeqAn3_523` - :ref:`list_software_Seqmagick_917` - :ref:`list_software_SeqPrep_704` - :ref:`list_software_seqtk_552` - :ref:`list_software_Serf_400` - :ref:`list_software_setuptools_1122` - :ref:`list_software_Seurat_628` - :ref:`list_software_Shannon_725` - :ref:`list_software_shapAAR_640` - :ref:`list_software_SHAPEIT_526` - :ref:`list_software_sharutils_991` - :ref:`list_software_shift_542` - :ref:`list_software_SHORE_385` - :ref:`list_software_Short-Pair_969` - :ref:`list_software_shovill_769` - :ref:`list_software_shrinkwrap_1097` - :ref:`list_software_Sibelia_697` - :ref:`list_software_sickle_635` - :ref:`list_software_Siesta_617` - :ref:`list_software_SimPEG_517` - :ref:`list_software_SIMPLE_458` - :ref:`list_software_SimpleElastix_1348` - :ref:`list_software_SimpleITK_850` - :ref:`list_software_simpy_562` - :ref:`list_software_SimVascular_1130` - :ref:`list_software_SingleM_687` - :ref:`list_software_Singularity_1179` - :ref:`list_software_SIONlib_624` - :ref:`list_software_SIP_236` - :ref:`list_software_SKESA_375` - :ref:`list_software_sketchmap_960` - :ref:`list_software_skewer_657` - :ref:`list_software_SLEPc_407` - :ref:`list_software_slepc4py_820` - :ref:`list_software_sleuth_661` - :ref:`list_software_slidingwindow_1410` - :ref:`list_software_smafa_520` - :ref:`list_software_smallgenomeutilities_2160` - :ref:`list_software_SMARTdenovo_1042` - :ref:`list_software_Smoldyn_742` - :ref:`list_software_SMRT-Link_769` - :ref:`list_software_snakemake_944` - :ref:`list_software_SNAP_306` - :ref:`list_software_SNAPE-pooled_1063` - :ref:`list_software_snaphu_655` - :ref:`list_software_snappy_667` - :ref:`list_software_snippy_675` - :ref:`list_software_snpEff_610` - :ref:`list_software_SNPhylo_685` - :ref:`list_software_SOAPaligner_1045` - :ref:`list_software_SOAPdenovo-Trans_1523` - :ref:`list_software_SOAPdenovo2_1008` - :ref:`list_software_SOAPfuse_742` - :ref:`list_software_socat_538` - :ref:`list_software_SolexaQA++_852` - :ref:`list_software_SortMeRNA_827` - :ref:`list_software_Spack_498` - :ref:`list_software_SPAdes_544` - :ref:`list_software_spaln_542` - :ref:`list_software_Spark_513` - :ref:`list_software_sparsehash_1074` - :ref:`list_software_spectral.methods_1664` - :ref:`list_software_spglib_641` - :ref:`list_software_spglib-python_1360` - :ref:`list_software_Sphinx_634` - :ref:`list_software_SpiceyPy_822` - :ref:`list_software_SpiecEasi_886` - :ref:`list_software_SPLASH_459` - :ref:`list_software_SpliceMap_894` - :ref:`list_software_splitRef_841` - :ref:`list_software_SPRNG_394` - :ref:`list_software_Spyder_631` - :ref:`list_software_SQLite_562` - :ref:`list_software_SqueezeMeta_1129` - :ref:`list_software_SRA-Toolkit_1017` - :ref:`list_software_SRPRISM_560` - :ref:`list_software_SSAHA2_418` - :ref:`list_software_SSPACE_Basic_1024` - :ref:`list_software_Stacks_617` - :ref:`list_software_STAMP_389` - :ref:`list_software_StaMPS_536` - :ref:`list_software_Stampy_638` - :ref:`list_software_STAR_314` - :ref:`list_software_STAR-Fusion_987` - :ref:`list_software_Stata_509` - :ref:`list_software_statsmodels_1203` - :ref:`list_software_STIR_322` - :ref:`list_software_STREAM_460` - :ref:`list_software_strelka_758` - :ref:`list_software_StringTie_921` - :ref:`list_software_Structure_977` - :ref:`list_software_Subread_710` - :ref:`list_software_Subversion_1072` - :ref:`list_software_suds_447` - :ref:`list_software_SuiteSparse_1144` - :ref:`list_software_SUMACLUST_705` - :ref:`list_software_SUMATRA_541` - :ref:`list_software_SUNDIALS_611` - :ref:`list_software_SuperLU_688` - :ref:`list_software_supermagic_1072` - :ref:`list_software_supernova_995` - :ref:`list_software_SVDetect_770` - :ref:`list_software_SVDquest_799` - :ref:`list_software_swarm_554` - :ref:`list_software_SWASH_390` - :ref:`list_software_SWIG_314` - :ref:`list_software_SymEngine_911` - :ref:`list_software_sympy_578` - :ref:`list_software_Szip_422`


.. _list_software_S-Lang_514:

*S-Lang*
++++++++

S-Lang is a multi-platform programmer's library designed to allow  a developer to create robust multi-platform software. It provides facilities  required by interactive applications such as display/screen management, keyboard  input, keymaps, and so on.

*homepage*: http://www.jedsoft.org/slang/

=========    =============
version      toolchain    
=========    =============
``2.3.0``    ``GCC/4.9.2``
=========    =============


.. _list_software_S4_135:

*S4*
++++

S4 stands for Stanford Stratified Structure Solver, a frequency domain code  to solve the linear Maxwell’s equations in layered periodic structures. Internally, it uses  Rigorous Coupled Wave Analysis (RCWA, also called the Fourier Modal Method (FMM)) and the  S-matrix algorithm.

*homepage*: https://web.stanford.edu/group/fan/S4

==================    ==============
version               toolchain     
==================    ==============
``1.1.1-20180610``    ``foss/2017b``
==================    ==============


.. _list_software_SAGE_288:

*SAGE*
++++++

S.A.G.E. (Statistical Analysis for Genetic  Epidemiology) is free software package containing programs for  use in the genetic analysis of family, pedigree and individual data.

*homepage*: http://darwin.cwru.edu/sage/

=======    ==========
version    toolchain 
=======    ==========
``6.3``    ``system``
``6.4``    ``system``
=======    ==========


.. _list_software_Sailfish_819:

*Sailfish*
++++++++++

Sailfish is a software tool that implements a novel, alignment-free algorithm for the estimation of  isoform abundances directly from a set of reference sequences and RNA-seq reads. 

*homepage*: http://www.cs.cmu.edu/~ckingsf/software/sailfish/

==========    ==================    ==============
version       versionsuffix         toolchain     
==========    ==================    ==============
``0.10.1``    ``-Python-2.7.12``    ``foss/2016b``
==========    ==================    ==============


.. _list_software_SALib_427:

*SALib*
+++++++

Sensitivity Analysis Library in Python (Numpy). Contains Sobol, Morris, Fractional Factorial and FAST methods.

*homepage*: http://SALib.github.io/SALib/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.1.3``    ``-Python-2.7.14``    ``intel/2017b``
=========    ==================    ===============


.. _list_software_Salmon_618:

*Salmon*
++++++++

Salmon is a wicked-fast program to produce a highly-accurate,  transcript-level quantification estimates from RNA-seq data.

*homepage*: https://github.com/COMBINE-lab/salmon

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.8.2``                           ``system``     
``0.8.2``     ``-Python-2.7.12``    ``foss/2016b`` 
``0.11.2``                          ``intel/2018a``
``0.12.0``                          ``foss/2018b`` 
``0.14.1``                          ``foss/2018b`` 
``0.14.2``                          ``gompi/2019a``
==========    ==================    ===============


.. _list_software_SALMON-TDDFT_877:

*SALMON-TDDFT*
++++++++++++++

SALMON is an open-source computer program for ab-initio quantum-mechanical calculations of  electron dynamics at the nanoscale that takes place in various situations of light-matter interactions.  It is based on time-dependent density functional theory, solving time-dependent Kohn-Sham equation in  real time and real space with norm-conserving pseudopotentials.

*homepage*: https://salmon-tddft.jp

=========    ===============================
version      toolchain                      
=========    ===============================
``1.2.1``    ``foss/2018b``, ``intel/2018b``
=========    ===============================


.. _list_software_Sambamba_788:

*Sambamba*
++++++++++

Sambamba is a high performance modern robust and fast tool (and library),   written in the D programming language, for working with SAM and BAM files. Current functionality   is an important subset of samtools functionality, including view, index, sort, markdup, and depth

*homepage*: http://lomereiter.github.io/sambamba/

=========    ==========
version      toolchain 
=========    ==========
``0.6.6``    ``system``
=========    ==========


.. _list_software_Samcef_591:

*Samcef*
++++++++

FEM solver solution suite for basic linear structures to advanced, flexible nonlinear mechanisms and  thermal applications.

*homepage*: http://www.plm.automation.siemens.com/en_us/products/lms/samtech/samcef-solver-suite/

===========    ==========
version        toolchain 
===========    ==========
``17.0-03``    ``system``
===========    ==========


.. _list_software_samclip_745:

*samclip*
+++++++++

Filter SAM file for soft and hard clipped alignments

*homepage*: https://github.com/tseemann/samclip

=======    ================    =================
version    versionsuffix       toolchain        
=======    ================    =================
``0.2``    ``-Perl-5.28.0``    ``GCCcore/7.3.0``
=======    ================    =================


.. _list_software_SAMtools_786:

*SAMtools*
++++++++++

SAM Tools provide various utilities for manipulating alignments in the SAM format,   including sorting, merging, indexing and generating alignments in a per-position format.

*homepage*: http://www.htslib.org/

==========    =================    =======================================================================================================================================
version       versionsuffix        toolchain                                                                                                                              
==========    =================    =======================================================================================================================================
``0.1.17``                         ``intel/2017a``                                                                                                                        
``0.1.19``                         ``foss/2016a``, ``foss/2016b``                                                                                                         
``0.1.20``                         ``foss/2018b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``                                                     
``1.2``                            ``foss/2016b``                                                                                                                         
``1.3``                            ``foss/2016a``, ``intel/2016a``                                                                                                        
``1.3.1``                          ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                                                                       
``1.3.1``     ``-HTSlib-1.3.2``    ``intel/2016b``                                                                                                                        
``1.4``                            ``foss/2016b``, ``intel/2016b``, ``intel/2017a``                                                                                       
``1.4.1``                          ``intel/2017a``                                                                                                                        
``1.5``                            ``foss/2016b``, ``intel/2017a``                                                                                                        
``1.6``                            ``GCC/6.4.0-2.28``, ``foss/2017a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2017b``                                            
``1.7``                            ``foss/2018a``, ``intel/2018a``                                                                                                        
``1.8``                            ``intel/2018a``                                                                                                                        
``1.9``                            ``GCC/6.4.0-2.28``, ``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``, ``intel/2018b``
==========    =================    =======================================================================================================================================


.. _list_software_SAS_231:

*SAS*
+++++

SAS is a software suite for advanced analytics, multivariate analyses, business intelligence,  data management, and predictive analytics.

*homepage*: https://www.sas.com

=======    ==========
version    toolchain 
=======    ==========
``9.4``    ``system``
=======    ==========


.. _list_software_savvy_569:

*savvy*
+++++++

Interface to various variant calling formats.

*homepage*: https://github.com/statgen/savvy

=========    ====================
version      toolchain           
=========    ====================
``1.3.0``    ``GCC/8.2.0-2.31.1``
=========    ====================


.. _list_software_Saxon-HE_707:

*Saxon-HE*
++++++++++

Open Source SAXON XSLT processor developed by Saxonica Limited.

*homepage*: http://saxon.sourceforge.net

============    ===================    ==========
version         versionsuffix          toolchain 
============    ===================    ==========
``9.7.0.4``     ``-Java-1.7.0_79``     ``system``
``9.7.0.21``    ``-Java-1.8.0_162``    ``system``
============    ===================    ==========


.. _list_software_sbt_329:

*sbt*
+++++

sbt is a build tool for Scala, Java, and more.

*homepage*: http://www.scala-sbt.org/

=========    ===================    ==========
version      versionsuffix          toolchain 
=========    ===================    ==========
``1.0.2``    ``-Java-1.8.0_152``    ``system``
=========    ===================    ==========


.. _list_software_ScaLAPACK_707:

*ScaLAPACK*
+++++++++++

The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines  redesigned for distributed memory MIMD parallel computers.

*homepage*: https://www.netlib.org/scalapack/

=========    =================================    ============================================================================================================================================
version      versionsuffix                        toolchain                                                                                                                                   
=========    =================================    ============================================================================================================================================
``2.0.2``                                         ``gompi/2019b``, ``gompic/2019b``                                                                                                           
``2.0.2``    ``-OpenBLAS-0.2.13-LAPACK-3.5.0``    ``gmvapich2/1.7.20``                                                                                                                        
``2.0.2``    ``-OpenBLAS-0.2.15-LAPACK-3.6.0``    ``gmpich/2016a``, ``gmvapich2/2016a``, ``gompi/2016a``                                                                                      
``2.0.2``    ``-OpenBLAS-0.2.18-LAPACK-3.6.0``    ``gompi/2016.04``, ``gompi/2016.06``                                                                                                        
``2.0.2``    ``-OpenBLAS-0.2.18-LAPACK-3.6.1``    ``gompi/2016.07``, ``gompi/2016b``                                                                                                          
``2.0.2``    ``-OpenBLAS-0.2.19-LAPACK-3.6.1``    ``gompi/2016.09``                                                                                                                           
``2.0.2``    ``-OpenBLAS-0.2.19-LAPACK-3.7.0``    ``gompi/2017a``                                                                                                                             
``2.0.2``    ``-OpenBLAS-0.2.20``                 ``gimpi/2017b``, ``gimpi/2018a``, ``gimpic/2017b``, ``gmpich/2017.08``, ``gompi/2017b``, ``gompi/2018a``, ``gompic/2017b``, ``gompic/2018a``
``2.0.2``    ``-OpenBLAS-0.3.1``                  ``gompi/2018b``, ``gompic/2018b``                                                                                                           
``2.0.2``    ``-OpenBLAS-0.3.3``                  ``gompi/2018.08``                                                                                                                           
``2.0.2``    ``-OpenBLAS-0.3.5``                  ``gompi/2019a``, ``gompic/2019a``                                                                                                           
=========    =================================    ============================================================================================================================================


.. _list_software_Scalasca_795:

*Scalasca*
++++++++++

Scalasca is a software tool that supports the performance optimization of  parallel programs by measuring and analyzing their runtime behavior. The  analysis identifies potential performance bottlenecks -- in particular  those concerning communication and synchronization -- and offers guidance  in exploring their causes. 

*homepage*: https://www.scalasca.org/

=======    ===============
version    toolchain      
=======    ===============
``2.3``    ``foss/2016a`` 
``2.5``    ``gompi/2019a``
=======    ===============


.. _list_software_SCALCE_427:

*SCALCE*
++++++++

SCALCE [skeɪlz] is a FASTQ compression tool that uses locally consistet parsing  to obtain better compression rate. SCALCE has been specifically designed for Illumina reads but it can  handle other technologies (that generate base pair reads) if the read length is the same throughout the file.

*homepage*: http://scalce.sourceforge.net/Home

=======    =============
version    toolchain    
=======    =============
``2.7``    ``GCC/4.8.2``
=======    =============


.. _list_software_sciClone_816:

*sciClone*
++++++++++

An R package for inferring the subclonal architecture of tumors 

*homepage*: https://github.com/genome/sciclone

=======    =============    ==============
version    versionsuffix    toolchain     
=======    =============    ==============
``1.1``    ``-R-3.5.1``     ``foss/2018b``
=======    =============    ==============


.. _list_software_ScientificPython_1667:

*ScientificPython*
++++++++++++++++++

ScientificPython is a collection of Python modules for scientific computing.  It contains support for geometry, mathematical functions, statistics, physical units, IO, visualization,  and parallelization.

*homepage*: https://sourcesup.cru.fr/projects/scientific-py/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``2.9.4``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
=========    ==================    ===============================


.. _list_software_scikit-allel_1214:

*scikit-allel*
++++++++++++++

This package provides utilities for exploratory analysis of large scale genetic variation data.   It is based on numpy, scipy and other general-purpose Python scientific libraries.

*homepage*: http://scikit-allel.readthedocs.io/en/latest/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.1.8``    ``-Python-2.7.13``    ``foss/2017a``
=========    ==================    ==============


.. _list_software_scikit-image_1207:

*scikit-image*
++++++++++++++

scikit-image is a collection of algorithms for image processing.

*homepage*: https://scikit-image.org/

==========    ==================    ===================================================
version       versionsuffix         toolchain                                          
==========    ==================    ===================================================
``0.12.3``    ``-Python-2.7.11``    ``foss/2016a``                                     
``0.12.3``    ``-Python-2.7.12``    ``intel/2016b``                                    
``0.12.3``    ``-Python-3.5.2``     ``foss/2016b``, ``intel/2016b``                    
``0.13.0``    ``-Python-2.7.13``    ``intel/2017a``                                    
``0.13.0``    ``-Python-3.6.1``     ``intel/2017a``                                    
``0.13.0``    ``-Python-3.6.3``     ``foss/2017b``                                     
``0.13.1``    ``-Python-2.7.14``    ``intel/2017b``                                    
``0.13.1``    ``-Python-3.6.3``     ``foss/2017b``                                     
``0.13.1``    ``-Python-3.6.4``     ``foss/2018a``                                     
``0.14.0``    ``-Python-3.6.4``     ``intel/2018a``                                    
``0.14.1``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
``0.15.0``    ``-Python-3.7.2``     ``foss/2019a``                                     
``0.16.2``    ``-Python-3.7.4``     ``foss/2019b``                                     
==========    ==================    ===================================================


.. _list_software_scikit-learn_1222:

*scikit-learn*
++++++++++++++

Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world, building upon numpy, scipy, and matplotlib. As a machine-learning module, it provides versatile tools for data mining and analysis in any field of science and engineering. It strives to be simple and efficient, accessible to everybody, and reusable in various contexts.

*homepage*: https://scikit-learn.org/stable/index.html

==========    =================================    ========================================================================
version       versionsuffix                        toolchain                                                               
==========    =================================    ========================================================================
``0.16.1``    ``-Python-2.7.13``                   ``foss/2017a``                                                          
``0.17.1``    ``-Python-2.7.11``                   ``foss/2016a``, ``intel/2016a``                                         
``0.17.1``    ``-Python-2.7.11-freetype-2.6.3``    ``intel/2016a``                                                         
``0.17.1``    ``-Python-2.7.12``                   ``intel/2016b``                                                         
``0.17.1``    ``-Python-3.5.1``                    ``foss/2016a``, ``intel/2016a``                                         
``0.18``      ``-Python-2.7.12``                   ``intel/2016b``                                                         
``0.18``      ``-Python-3.5.2``                    ``intel/2016b``                                                         
``0.18.1``    ``-Python-2.7.12``                   ``foss/2016b``, ``intel/2016b``                                         
``0.18.1``    ``-Python-2.7.13``                   ``intel/2017a``                                                         
``0.18.1``    ``-Python-3.5.2``                    ``foss/2016b``, ``intel/2016b``                                         
``0.18.1``    ``-Python-3.6.1``                    ``intel/2017a``                                                         
``0.18.2``    ``-Python-3.6.1``                    ``intel/2017a``                                                         
``0.19.0``    ``-Python-3.6.1``                    ``intel/2017a``                                                         
``0.19.1``    ``-Python-2.7.13``                   ``foss/2017a``                                                          
``0.19.1``    ``-Python-2.7.14``                   ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.19.1``    ``-Python-3.6.3``                    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.19.1``    ``-Python-3.6.4``                    ``foss/2018a``, ``intel/2018a``                                         
``0.20.0``    ``-Python-2.7.15``                   ``foss/2018b``, ``intel/2018b``                                         
``0.20.0``    ``-Python-3.6.6``                    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                     
``0.20.2``    ``-Python-2.7.15``                   ``foss/2018b``                                                          
``0.20.2``    ``-Python-3.6.6``                    ``foss/2018b``                                                          
``0.20.3``                                         ``foss/2019a``, ``intel/2019a``                                         
``0.21.3``    ``-Python-3.7.4``                    ``foss/2019b``, ``intel/2019b``                                         
==========    =================================    ========================================================================


.. _list_software_scikit-multilearn_1777:

*scikit-multilearn*
+++++++++++++++++++

Scikit-multilearn is a BSD-licensed library for multi-label classification that is built on top of the well-known scikit-learn ecosystem.

*homepage*: http://scikit.ml

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.2.0``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_scikit-optimize_1573:

*scikit-optimize*
+++++++++++++++++

Scikit-Optimize, or skopt, is a simple and efficient library to minimize (very) expensive  and noisy black-box functions.

*homepage*: https://scikit-optimize.github.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``0.5.2``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_SCIPhI_480:

*SCIPhI*
++++++++

Single-cell mutation identification via phylogenetic inference (SCIPhI) is a new approach  to mutation detection in individual tumor cells by leveraging the evolutionary relationship among cells.

*homepage*: https://github.com/cbg-ethz/SCIPhI

=========    ===============
version      toolchain      
=========    ===============
``0.1.3``    ``intel/2018b``
=========    ===============


.. _list_software_scipy_552:

*scipy*
+++++++

SciPy is a collection of mathematical algorithms and convenience  functions built on the Numpy extension for Python.

*homepage*: http://www.scipy.org

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.16.0``    ``-Python-2.7.12``    ``intel/2016b``
``0.17.0``    ``-Python-2.7.11``    ``intel/2016a``
``0.19.0``    ``-Python-3.5.2``     ``intel/2016b``
==========    ==================    ===============


.. _list_software_SciPy-bundle_1167:

*SciPy-bundle*
++++++++++++++

Bundle of Python packages for scientific software

*homepage*: https://python.org/

===========    ==================    ========================================================================
version        versionsuffix         toolchain                                                               
===========    ==================    ========================================================================
``2019.03``                          ``foss/2019a``, ``fosscuda/2019a``, ``intel/2019a``, ``intelcuda/2019a``
``2019.10``    ``-Python-2.7.16``    ``foss/2019b``                                                          
``2019.10``    ``-Python-3.7.4``     ``foss/2019b``, ``fosscuda/2019b``, ``intel/2019b``                     
===========    ==================    ========================================================================


.. _list_software_SCnorm_594:

*SCnorm*
++++++++

This package implements SCnorm — a method to normalize single-cell RNA-seq data.

*homepage*: https://bioconductor.org/packages/SCnorm

==========    =============    ===============
version       versionsuffix    toolchain      
==========    =============    ===============
``0.99.7``    ``-R-3.4.0``     ``intel/2017a``
==========    =============    ===============


.. _list_software_Scoary_625:

*Scoary*
++++++++

Microbial pan-GWAS using the output from Roary

*homepage*: https://github.com/AdmiralenOla/Scoary

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.6.16``    ``-Python-2.7.14``    ``intel/2018a``
==========    ==================    ===============


.. _list_software_SCons_486:

*SCons*
+++++++

SCons is a software construction tool.

*homepage*: https://www.scons.org/

=========    =======================    =================================================================================
version      versionsuffix              toolchain                                                                        
=========    =======================    =================================================================================
``2.4.1``    ``-Python-2.7.11``         ``foss/2016a``                                                                   
``2.5.0``    ``-Python-2.7.12``         ``foss/2016b``, ``intel/2016b``                                                  
``2.5.1``                               ``intel/2017a``                                                                  
``2.5.1``    ``-Python-2.7.12``         ``foss/2016b``, ``intel/2016b``                                                  
``2.5.1``    ``-Python-2.7.13``         ``intel/2017a``                                                                  
``3.0.1``    ``-Python-2.7.14``         ``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``, ``iomkl/2018a``
``3.0.1``    ``-Python-2.7.15``         ``fosscuda/2018b``                                                               
``3.0.1``    ``-Python-2.7.15-bare``    ``GCCcore/7.3.0``                                                                
``3.0.1``    ``-Python-3.6.3``          ``foss/2017b``, ``intel/2017b``                                                  
``3.0.1``    ``-Python-3.6.4``          ``foss/2018a``, ``intel/2018a``                                                  
``3.0.4``    ``-Python-2.7.15``         ``foss/2018b``                                                                   
``3.0.5``                               ``GCCcore/8.2.0``                                                                
``3.0.5``    ``-Python-2.7.15``         ``GCCcore/8.2.0``                                                                
``3.0.5``    ``-Python-3.7.2``          ``GCCcore/8.2.0``                                                                
``3.1.1``                               ``GCCcore/8.3.0``                                                                
=========    =======================    =================================================================================


.. _list_software_SCOOP_388:

*SCOOP*
+++++++

SCOOP (Scalable COncurrent Operations in Python) is a distributed task module  allowing concurrent parallel programming on various environments, from heterogeneous grids to supercomputers.

*homepage*: http://code.google.com/p/scoop/

===========    ==================    =================
version        versionsuffix         toolchain        
===========    ==================    =================
``0.6.2``      ``-Python-2.7.12``    ``intel/2016b``  
``0.7.1.1``                          ``GCCcore/8.2.0``
``0.7.1.1``    ``-Python-2.7.11``    ``intel/2016a``  
``0.7.1.1``    ``-Python-2.7.14``    ``intel/2017b``  
``0.7.1.1``    ``-Python-3.5.1``     ``intel/2016a``  
===========    ==================    =================


.. _list_software_Score-P_633:

*Score-P*
+++++++++

The Score-P measurement infrastructure is a highly scalable and easy-to-use  tool suite for profiling, event tracing, and online analysis of HPC  applications. 

*homepage*: https://www.score-p.org

=========    ===============
version      toolchain      
=========    ===============
``2.0.1``    ``foss/2016a`` 
``6.0``      ``gompi/2019a``
=========    ===============


.. _list_software_SCOTCH_452:

*SCOTCH*
++++++++

Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning.

*homepage*: https://gforge.inria.fr/projects/scotch/

===================    =============    =====================================================================================================================================================
version                versionsuffix    toolchain                                                                                                                                            
===================    =============    =====================================================================================================================================================
``5.1.12b_esmumps``                     ``foss/2017b``                                                                                                                                       
``6.0.4``                               ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``6.0.4``              ``-64bitint``    ``foss/2017b``                                                                                                                                       
``6.0.6``                               ``foss/2018a``, ``foss/2018b``, ``gompi/2019a``, ``iimpi/2019a``, ``intel/2018a``                                                                    
``6.0.9``                               ``gompi/2019b``                                                                                                                                      
===================    =============    =====================================================================================================================================================


.. _list_software_scp_326:

*scp*
+++++

The scp.py module uses a paramiko transport to send and recieve files via the scp1 protocol.

*homepage*: https://github.com/jbardin/scp.py

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.10.2``    ``-Python-2.7.12``    ``intel/2016b``
``0.13.1``    ``-Python-2.7.15``    ``intel/2018b``
``0.13.2``    ``-Python-2.7.15``    ``intel/2018b``
==========    ==================    ===============


.. _list_software_Scrappie_823:

*Scrappie*
++++++++++

Scrappie is a technology demonstrator for the Oxford Nanopore Research Algorithms group.

*homepage*: https://github.com/nanoporetech/scrappie

=========    ===============
version      toolchain      
=========    ===============
``1.3.2``    ``intel/2018a``
=========    ===============


.. _list_software_SDCC_285:

*SDCC*
++++++

SDCC is a retargettable, optimizing ANSI - C compiler suite that targets  the Intel MCS51 based microprocessors (8031, 8032, 8051, 8052, etc.), Maxim (formerly Dallas)  DS80C390 variants, Freescale (formerly Motorola) HC08 based (hc08, s08) and Zilog Z80 based  MCUs (z80, z180, gbz80, Rabbit 2000/3000, Rabbit 3000A). Work is in progress on supporting the  Microchip PIC16 and PIC18 targets. It can be retargeted for other microprocessors.

*homepage*: http://sdcc.sourceforge.net/

=========    ==========
version      toolchain 
=========    ==========
``3.3.0``    ``system``
=========    ==========


.. _list_software_SDL2_277:

*SDL2*
++++++

SDL: Simple DirectMedia Layer, a cross-platform multimedia library

*homepage*: http://www.libsdl.org/

=========    ======================================================================
version      toolchain                                                             
=========    ======================================================================
``2.0.4``    ``intel/2016b``                                                       
``2.0.8``    ``foss/2017b``, ``intel/2017b``, ``intel/2018a``                      
``2.0.9``    ``GCCcore/8.2.0``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``
=========    ======================================================================


.. _list_software_Seaborn_714:

*Seaborn*
+++++++++

Seaborn is a Python visualization library based on matplotlib.   It provides a high-level interface for drawing attractive statistical graphics. 

*homepage*: http://stanford.edu/~mwaskom/software/seaborn/

=========    ==================    =========================================================================================
version      versionsuffix         toolchain                                                                                
=========    ==================    =========================================================================================
``0.7.1``    ``-Python-2.7.12``    ``intel/2016b``                                                                          
``0.8.1``    ``-Python-2.7.13``    ``foss/2017a``                                                                           
``0.8.1``    ``-Python-2.7.14``    ``intel/2018a``                                                                          
``0.9.0``    ``-Python-2.7.14``    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``, ``intelcuda/2017b``
``0.9.0``    ``-Python-2.7.15``    ``foss/2018b``, ``foss/2019a``, ``intel/2018b``, ``intel/2019a``                         
``0.9.0``    ``-Python-3.6.3``     ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``                 
``0.9.0``    ``-Python-3.6.4``     ``intel/2018a``                                                                          
``0.9.0``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``                                                          
``0.9.0``    ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``                                                          
=========    ==================    =========================================================================================


.. _list_software_SearchGUI_827:

*SearchGUI*
+++++++++++

SearchGUI is a user-friendly open-source graphical user interface for configuring and  running proteomics identification search engines and de novo sequencing algorithms,  currently supporting X! Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA,  Novor and DirecTag.

*homepage*: https://github.com/compomics/searchgui

=========    ===================    ==========
version      versionsuffix          toolchain 
=========    ===================    ==========
``3.3.3``    ``-Java-1.8.0_152``    ``system``
=========    ===================    ==========


.. _list_software_segemehl_842:

*segemehl*
++++++++++

segemehl is a software to map short sequencer reads to reference genomes.   Unlike other methods, segemehl is able to detect not only mismatches but also insertions   and deletions. Furthermore, segemehl is not limited to a specific read length and is able   to mapprimer- or polyadenylation contaminated reads correctly. segemehl implements a matching   strategy based on enhanced suffix arrays (ESA). Segemehl now supports the SAM format, reads   gziped queries to save both disk and memory space and allows bisulfite sequencing mapping   and split read mapping. 

*homepage*: http://www.bioinf.uni-leipzig.de/Software/segemehl/

=========    ================================================
version      toolchain                                       
=========    ================================================
``0.2.0``    ``foss/2016b``, ``intel/2017b``, ``intel/2018a``
``0.3.4``    ``foss/2018b``                                  
=========    ================================================


.. _list_software_SelEstim_806:

*SelEstim*
++++++++++

SelEstim is aimed at distinguishing neutral from selected polymorphisms and estimate   the intensity of selection at the latter. The SelEstim model accounts explicitly for positive selection,   and it is assumed that all marker loci in the dataset are responding to selection, to some extent

*homepage*: http://www1.montpellier.inra.fr/CBGP/software/selestim/index.html

=========    =================    ==========
version      versionsuffix        toolchain 
=========    =================    ==========
``1.1.4``    ``-Linux-64bits``    ``system``
=========    =================    ==========


.. _list_software_sep_328:

*sep*
+++++

Python and C library for Source Extraction and Photometry.  (this easyconfig provides python library only)

*homepage*: https://sep.readthedocs.io

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.0.3``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
=========    ==================    ===============================


.. _list_software_Seq-Gen_624:

*Seq-Gen*
+++++++++

Seq-Gen is a program that will simulate the evolution of nucleotide or amino acid sequences  along a phylogeny, using common models of the substitution process.

*homepage*: http://tree.bio.ed.ac.uk/software/Seq-Gen

=========    ===============
version      toolchain      
=========    ===============
``1.3.4``    ``intel/2017b``
=========    ===============


.. _list_software_seq2HLA_592:

*seq2HLA*
+++++++++

In-silico method written in Python and R to determine HLA genotypes of a sample.

*homepage*: https://github.com/TRON-Bioinformatics/seq2HLA

=======    ==================    ===============
version    versionsuffix         toolchain      
=======    ==================    ===============
``2.3``    ``-Python-2.7.14``    ``intel/2018a``
``2.3``    ``-Python-2.7.15``    ``foss/2018b`` 
=======    ==================    ===============


.. _list_software_SeqAn_472:

*SeqAn*
+++++++

SeqAn is an open source C++ library of efficient algorithms and data structures  for the analysis of sequences with the focus on biological data

*homepage*: https://www.seqan.de/

=========    =====================================================
version      toolchain                                            
=========    =====================================================
``2.3.2``    ``foss/2016b``                                       
``2.4.0``    ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``intel/2018b``
=========    =====================================================


.. _list_software_SeqAn3_523:

*SeqAn3*
++++++++

SeqAn is an open source C++ library   of efficient algorithms and data structures for the   analysis of sequences with the focus on biological   data. Our library applies a unique generic design   that guarantees high performance, generality,   extensibility, and integration with other libraries.   SeqAn is easy to use and simplifies the development   of new software tools with a minimal loss of   performance. 

*homepage*: https://github.com/seqan/seqan3

=========    ==========
version      toolchain 
=========    ==========
``3.0.0``    ``system``
=========    ==========


.. _list_software_Seqmagick_917:

*Seqmagick*
+++++++++++

We often have to convert between sequence formats and do  little tasks on them, and it's not worth writing scripts for that. Seqmagick  is a kickass little utility built in the spirit of imagemagick to expose the  file format conversion in Biopython in a convenient way. Instead of having a  big mess of scripts, there is one that takes arguments.

*homepage*: https://fhcrc.github.io/seqmagick/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``0.6.1``    ``-Python-2.7.11``    ``foss/2016a``
``0.6.2``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_SeqPrep_704:

*SeqPrep*
+++++++++

Tool for stripping adaptors and/or merging paired reads with overlap into single reads.

*homepage*: https://github.com/jstjohn/SeqPrep

=========    =================
version      toolchain        
=========    =================
``1.3.2``    ``GCCcore/7.3.0``
=========    =================


.. _list_software_seqtk_552:

*seqtk*
+++++++

Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.   It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.

*homepage*: https://github.com/lh3/seqtk/

=======    ===============================
version    toolchain                      
=======    ===============================
``1.2``    ``foss/2016b``, ``intel/2017a``
``1.3``    ``foss/2018a``, ``foss/2018b`` 
=======    ===============================


.. _list_software_Serf_400:

*Serf*
++++++

The serf library is a high performance C-based HTTP client library  built upon the Apache Portable Runtime (APR) library

*homepage*: http://serf.apache.org/

=========    ======================================================================================
version      toolchain                                                                             
=========    ======================================================================================
``1.3.9``    ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2017b``, ``intel/2017b``, ``iomkl/2018a``
=========    ======================================================================================


.. _list_software_setuptools_1122:

*setuptools*
++++++++++++

Easily download, build, install, upgrade, and uninstall Python packages

*homepage*: https://pypi.org/project/setuptools

==========    =============    ==========
version       versionsuffix    toolchain 
==========    =============    ==========
``1.4.2``                      ``system``
``41.0.1``    ``-py3``         ``system``
==========    =============    ==========


.. _list_software_Seurat_628:

*Seurat*
++++++++

Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data.

*homepage*: http://satijalab.org/seurat

============    =============    ===============
version         versionsuffix    toolchain      
============    =============    ===============
``1.4.0.16``    ``-R-3.4.0``     ``intel/2017a``
``2.3.4``       ``-R-3.4.4``     ``intel/2018a``
``2.3.4``       ``-R-3.5.1``     ``foss/2018b`` 
============    =============    ===============


.. _list_software_Shannon_725:

*Shannon*
+++++++++

Shannon is a program for assembling transcripts from RNA-Seq data

*homepage*: https://sreeramkannan.github.io/Shannon/

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20170511``    ``-Python-2.7.13``    ``intel/2017a``
============    ==================    ===============


.. _list_software_shapAAR_640:

*shapAAR*
+++++++++

An R package for the extraction, analysis and classification of (not only) archaeological objects  from scanned images.

*homepage*: https://github.com/ISAAKiel/shapAAR

==================    =============    ==============
version               versionsuffix    toolchain     
==================    =============    ==============
``0.1.0-20180425``    ``-R-3.6.0``     ``foss/2019a``
==================    =============    ==============


.. _list_software_SHAPEIT_526:

*SHAPEIT*
+++++++++

SHAPEIT is a fast and accurate method for estimation of  haplotypes (aka phasing) from genotype or sequencing data.

*homepage*: https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html

==========    ===============    ==========
version       versionsuffix      toolchain 
==========    ===============    ==========
``2.r837``    ``.GLIBCv2.12``    ``system``
``2.r904``    ``.glibcv2.17``    ``system``
==========    ===============    ==========


.. _list_software_sharutils_991:

*sharutils*
+++++++++++

GNU shar makes so-called shell archives out of many files, preparing them for transmission by  electronic mail services, while unshar helps unpacking shell archives after reception.

*homepage*: https://www.gnu.org/software/sharutils/

========    =================
version     toolchain        
========    =================
``4.15``    ``GCCcore/6.3.0``
========    =================


.. _list_software_shift_542:

*shift*
+++++++

Shift is a framework for Self-Healing Independent File Transfer that provides high performance and resilience for local and remote transfers through a variety of techniques. 

*homepage*: http://people.nas.nasa.gov/~kolano/projects/shift.html

=======    ===============
version    toolchain      
=======    ===============
``4.0``    ``intel/2016a``
=======    ===============


.. _list_software_SHORE_385:

*SHORE*
+++++++

SHORE, for Short Read, is a mapping and analysis pipeline for short read data produced on the Illumina platform. 

*homepage*: http://1001genomes.org/software/shore.html

=========    ==============
version      toolchain     
=========    ==============
``0.9.3``    ``foss/2016a``
=========    ==============


.. _list_software_Short-Pair_969:

*Short-Pair*
++++++++++++

Sensitive Short Read Homology Search for Paired-End Reads

*homepage*: https://sourceforge.net/projects/short-pair

============    ==================    ==============
version         versionsuffix         toolchain     
============    ==================    ==============
``20170125``    ``-Python-2.7.15``    ``foss/2018b``
============    ==================    ==============


.. _list_software_shovill_769:

*shovill*
+++++++++

Faster SPAdes assembly of Illumina reads

*homepage*: https://github.com/tseemann/shovill

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``0.9.0``    ``-Python-2.7.14``    ``foss/2018a``
``1.0.4``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_shrinkwrap_1097:

*shrinkwrap*
++++++++++++

A std::streambuf wrapper for compression formats.

*homepage*: https://github.com/jonathonl/shrinkwrap

==============    =================
version           toolchain        
==============    =================
``1.0.0-beta``    ``GCCcore/8.2.0``
==============    =================


.. _list_software_Sibelia_697:

*Sibelia*
+++++++++

Sibelia: A comparative genomics tool: It assists biologists in analysing   the genomic variations that correlate with pathogens, or the genomic changes that help   microorganisms adapt in different environments. Sibelia will also be helpful for the   evolutionary and genome rearrangement studies for multiple strains of microorganisms.

*homepage*: http://bioinf.spbau.ru/en/sibelia

=========    ==============
version      toolchain     
=========    ==============
``3.0.6``    ``foss/2016b``
``3.0.7``    ``foss/2018b``
=========    ==============


.. _list_software_sickle_635:

*sickle*
++++++++

Windowed Adaptive Trimming for fastq files using quality 

*homepage*: https://github.com/najoshi/sickle

========    ===============================
version     toolchain                      
========    ===============================
``1.33``    ``foss/2017a``, ``intel/2018a``
========    ===============================


.. _list_software_Siesta_617:

*Siesta*
++++++++

SIESTA is both a method and its computer program implementation, to perform efficient electronic  structure calculations and ab initio molecular dynamics simulations of molecules and solids.

*homepage*: http://departments.icmab.es/leem/siesta

==========    ===============================
version       toolchain                      
==========    ===============================
``4.0``       ``intel/2017a``                
``4.0.1``     ``intel/2017a``                
``4.1-b2``    ``intel/2017a``                
``4.1-b3``    ``intel/2017a``                
``4.1-b4``    ``foss/2018b``, ``intel/2018b``
==========    ===============================


.. _list_software_SimPEG_517:

*SimPEG*
++++++++

Simulation and Parameter Estimation in Geophysics: a python package for simulation and gradient based parameter estimation in the context of geophysical applications.

*homepage*: http://simpeg.xyz

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.3.1``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_SIMPLE_458:

*SIMPLE*
++++++++

Single-particle IMage Processing Linux Engine is a program package  for cryo-EM image processing, focusing on ab initio 3D reconstruction of low-symmetry single-particles. 

*homepage*: http://simplecryoem.com/

=======    ==============
version    toolchain     
=======    ==============
``2.5``    ``foss/2018a``
=======    ==============


.. _list_software_SimpleElastix_1348:

*SimpleElastix*
+++++++++++++++

Multi-lingual medical image registration library. 

*homepage*: http://simpleelastix.github.io/

==========    =================    ==============
version       versionsuffix        toolchain     
==========    =================    ==============
``0.10.0``    ``-Python-3.6.3``    ``foss/2017b``
``0.10.0``    ``-Python-3.6.4``    ``foss/2018a``
==========    =================    ==============


.. _list_software_SimpleITK_850:

*SimpleITK*
+++++++++++

imbalanced-learn is a Python package offering a number of re-sampling techniques commonly used in  datasets showing strong between-class imbalance.

*homepage*: https://github.com/scikit-learn-contrib/imbalanced-learn

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.1.0``    ``-Python-2.7.15``    ``foss/2018b``
``1.1.0``    ``-Python-3.6.4``     ``foss/2018a``
``1.1.0``    ``-Python-3.6.6``     ``foss/2018b``
=========    ==================    ==============


.. _list_software_simpy_562:

*simpy*
+++++++

SimPy is a process-based discrete-event simulation framework based on standard Python.

*homepage*: https://simpy.readthedocs.io

==========    =================    ===============
version       versionsuffix        toolchain      
==========    =================    ===============
``3.0.11``                         ``intel/2019a``
``3.0.11``    ``-Python-3.6.6``    ``intel/2018b``
==========    =================    ===============


.. _list_software_SimVascular_1130:

*SimVascular*
+++++++++++++

SimVascular is an open source software suite for cardiovascular simulation, providing a complete  pipeline from medical image data to 3D model construction, meshing, and blood flow simulation.

*homepage*: http://simvascular.github.io/

=============    ===============
version          toolchain      
=============    ===============
``2.16.0406``    ``intel/2016b``
=============    ===============


.. _list_software_SingleM_687:

*SingleM*
+++++++++

SingleM is a tool to find the abundances of discrete operational taxonomic units (OTUs) directly  from shotgun metagenome data, without heavy reliance on reference sequence databases.

*homepage*: https://github.com/wwood/singlem

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.12.1``    ``-Python-2.7.15``    ``intel/2019a``
==========    ==================    ===============


.. _list_software_Singularity_1179:

*Singularity*
+++++++++++++

Singularity is a portable application stack packaging and runtime utility.

*homepage*: http://singularity.lbl.gov

=========    ==================
version      toolchain         
=========    ==================
``1.0``      ``GCC/4.9.3-2.25``
``2.2.1``    ``GCC/6.3.0-2.27``
``2.3.1``    ``GCC/5.4.0-2.26``
``2.4.2``    ``GCC/5.4.0-2.26``
=========    ==================


.. _list_software_SIONlib_624:

*SIONlib*
+++++++++

SIONlib is a scalable I/O library for parallel access to task-local files.  The library not only supports writing and reading binary data to or from  several thousands of processors into a single or a small number of physical  files, but also provides global open and close functions to access SIONlib  files in parallel. This package provides a stripped-down installation of  SIONlib for use with performance tools (e.g., Score-P), with renamed symbols  to avoid conflicts when an application using SIONlib itself is linked against  a tool requiring a different SIONlib version. 

*homepage*: http://www.fz-juelich.de/ias/jsc/EN/Expertise/Support/Software/SIONlib/_node.html

=========    =============    =================
version      versionsuffix    toolchain        
=========    =============    =================
``1.6.1``                     ``foss/2016a``   
``1.6.1``    ``-tools``       ``foss/2016a``   
``1.7.1``                     ``foss/2017a``   
``1.7.1``    ``-tools``       ``foss/2017a``   
``1.7.4``    ``-tools``       ``GCCcore/8.2.0``
=========    =============    =================


.. _list_software_SIP_236:

*SIP*
+++++

SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries.

*homepage*: http://www.riverbankcomputing.com/software/sip/

===========    ==================    ===============================================
version        versionsuffix         toolchain                                      
===========    ==================    ===============================================
``4.18``       ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``                
``4.18.1``     ``-Python-2.7.11``    ``foss/2016a``                                 
``4.18.1``     ``-Python-2.7.12``    ``intel/2016b``                                
``4.19``       ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``                
``4.19.2``     ``-Python-2.7.13``    ``intel/2017a``                                
``4.19.8``     ``-Python-2.7.14``    ``foss/2017b``, ``foss/2018a``, ``intel/2017b``
``4.19.8``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                
``4.19.13``    ``-Python-2.7.15``    ``fosscuda/2018b``                             
===========    ==================    ===============================================


.. _list_software_SKESA_375:

*SKESA*
+++++++

SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs.

*homepage*: https://github.com/ncbi/SKESA

=========    ==============
version      toolchain     
=========    ==============
``2.2``      ``foss/2018a``
``2.3.0``    ``foss/2018b``
=========    ==============


.. _list_software_sketchmap_960:

*sketchmap*
+++++++++++

Sketch-map is a dimensionality reduction algorithm that is particularly well suited to examining  the high-dimensionality data that is routinely produced in atomistic simulations.

*homepage*: http://sketchmap.org/

============    ===============
version         toolchain      
============    ===============
``20170130``    ``intel/2016b``
============    ===============


.. _list_software_skewer_657:

*skewer*
++++++++

skewer implements the bit-masked k-difference matching algorithm dedicated to the task of adapter  trimming and it is specially designed for processing next-generation sequencing (NGS) paired-end sequences.

*homepage*: https://github.com/relipmoc/skewer

=========    ===============
version      toolchain      
=========    ===============
``0.2.2``    ``intel/2016b``
=========    ===============


.. _list_software_SLEPc_407:

*SLEPc*
+++++++

SLEPc (Scalable Library for Eigenvalue Problem Computations) is a software library for the solution  of large scale sparse eigenvalue problems on parallel computers. It is an extension of PETSc and can be used for  either standard or generalized eigenproblems, with real or complex arithmetic. It can also be used for computing a  partial SVD of a large, sparse, rectangular matrix, and to solve quadratic eigenvalue problems.

*homepage*: http://www.grycap.upv.es/slepc/

==========    ==============
version       toolchain     
==========    ==============
``3.8.3``     ``foss/2017b``
``3.9.2``     ``foss/2018a``
``3.11.0``    ``foss/2018b``
==========    ==============


.. _list_software_slepc4py_820:

*slepc4py*
++++++++++

Python bindings for SLEPc, the Scalable Library for Eigenvalue Problem Computations.

*homepage*: https://bitbucket.org/slepc/slepc4py

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``3.9.0``    ``-Python-3.6.4``    ``foss/2018a``
=========    =================    ==============


.. _list_software_sleuth_661:

*sleuth*
++++++++

Investigate RNA-Seq transcript abundance from kallisto and perform differential expression analysis.

*homepage*: http://pachterlab.github.io/sleuth

==========    =============    ===============
version       versionsuffix    toolchain      
==========    =============    ===============
``0.29.0``    ``-R-3.4.0``     ``intel/2017a``
``0.30.0``    ``-R-3.5.1``     ``foss/2018b`` 
==========    =============    ===============


.. _list_software_slidingwindow_1410:

*slidingwindow*
+++++++++++++++

slidingwindow is a simple little Python library for computing a set of windows into a larger dataset,  designed for use with image-processing algorithms that utilise a sliding window to break the processing up  into a series of smaller chunks.

*homepage*: https://github.com/adamrehn/slidingwindow

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.0.13``    ``-Python-2.7.15``    ``intel/2018b``
``0.0.13``    ``-Python-3.6.6``     ``intel/2018b``
==========    ==================    ===============


.. _list_software_smafa_520:

*smafa*
+++++++

Smafa attempts to align or cluster pre-aligned biological sequences, handling sequences  which are all the same length.

*homepage*: https://github.com/wwood/smafa

=========    =================
version      toolchain        
=========    =================
``0.4.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_smallgenomeutilities_2160:

*smallgenomeutilities*
++++++++++++++++++++++

The smallgenomeutilities are a collection of scripts that is  useful for dealing and manipulating NGS data of small viral genomes. They  are written in Python 3 with a small number of dependencies.

*homepage*: https://github.com/cbg-ethz/smallgenomeutilities

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.2.1``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_SMARTdenovo_1042:

*SMARTdenovo*
+++++++++++++

SMARTdenovo is a de novo assembler for PacBio and Oxford Nanopore (ONT) data. It produces an assembly from all-vs-all raw read alignments without an error correction stage. It also provides tools to generate accurate consensus sequences, though a platform dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish for ONT) are still required for higher accuracy.

*homepage*: https://github.com/ruanjue/smartdenovo

============    ==============
version         toolchain     
============    ==============
``20180219``    ``foss/2018b``
============    ==============


.. _list_software_Smoldyn_742:

*Smoldyn*
+++++++++

Smoldyn is a computer program for cell-scale biochemical simulations. It simulates each molecule of  interest individually to capture natural stochasticity and to yield nanometer-scale spatial resolution. It treats  other molecules implicitly, enabling it to simulate hundreds of thousands of molecules over several minutes of real  time. Simulated molecules diffuse, react, are confined by surfaces, and bind to membranes much as they would in a real  biological system. 

*homepage*: http://www.smoldyn.org/

========    ==============
version     toolchain     
========    ==============
``2.48``    ``foss/2016a``
========    ==============


.. _list_software_SMRT-Link_769:

*SMRT-Link*
+++++++++++

PacBio’s open-source SMRT Analysis software suite is designed for use with Single Molecule,   Real-Time (SMRT) Sequencing data. You can analyze, visualize, and manage your data through an intuitive GUI   or command-line interface. You can also integrate SMRT Analysis in your existing data workflow through   the extensive set of APIs provided

*homepage*: https://www.pacb.com/support/software-downloads/

===============    ===================    ==========
version            versionsuffix          toolchain 
===============    ===================    ==========
``5.1.0.26412``    ``-cli-tools-only``    ``system``
``6.0.0.47841``    ``-cli-tools-only``    ``system``
===============    ===================    ==========


.. _list_software_snakemake_944:

*snakemake*
+++++++++++

The Snakemake workflow management system is a tool to create reproducible and scalable data analyses.

*homepage*: https://snakemake.readthedocs.io

=========    =================    ===============================
version      versionsuffix        toolchain                      
=========    =================    ===============================
``5.2.2``    ``-Python-3.6.4``    ``intel/2018a``                
``5.2.4``    ``-Python-3.6.6``    ``foss/2018b``, ``intel/2018b``
=========    =================    ===============================


.. _list_software_SNAP_306:

*SNAP*
++++++

SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.  SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser.

*homepage*: http://korflab.ucdavis.edu/software.html

==============    =========================================================================
version           toolchain                                                                
==============    =========================================================================
``1.0beta.23``    ``intel/2017b``                                                          
``2013-11-29``    ``GCC/6.4.0-2.28``, ``GCC/8.3.0``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
==============    =========================================================================


.. _list_software_SNAPE-pooled_1063:

*SNAPE-pooled*
++++++++++++++

SNAPE-pooled computes the probability distribution for the frequency of the minor allele  in a certain population, at a certain position in the genome.

*homepage*: https://code.google.com/archive/p/snape-pooled/

=======    ==============
version    toolchain     
=======    ==============
``r32``    ``foss/2016a``
=======    ==============


.. _list_software_snaphu_655:

*snaphu*
++++++++

SNAPHU is an implementation of the Statistical-cost, Network-flow Algorithm for Phase Unwrapping  proposed by Chen and Zebker

*homepage*: https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/

=========    ===================================================
version      toolchain                                          
=========    ===================================================
``1.4.2``    ``GCCcore/6.3.0``, ``intel/2016b``, ``intel/2017a``
=========    ===================================================


.. _list_software_snappy_667:

*snappy*
++++++++

Snappy is a compression/decompression library. It does not aim for maximum compression, or compatibility with any other compression library; instead, it aims for very high speeds and reasonable compression.

*homepage*: https://github.com/google/snappy

=========    ============================================================================================================
version      toolchain                                                                                                   
=========    ============================================================================================================
``1.1.2``    ``GCC/4.9.2``                                                                                               
``1.1.3``    ``GCC/4.9.3``, ``GCC/4.9.3-2.25``                                                                           
``1.1.6``    ``system``                                                                                                  
``1.1.7``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``intel/2017a``, ``intel/2017b``
=========    ============================================================================================================


.. _list_software_snippy_675:

*snippy*
++++++++

Snippy finds SNPs between a haploid reference genome and your NGS sequence  reads. It will find both substitutions (snps) and insertions/deletions (indels). Rapid haploid variant calling and core genome alignment.

*homepage*: https://github.com/tseemann/snippy

=========    ================    ==============
version      versionsuffix       toolchain     
=========    ================    ==============
``4.4.1``    ``-Perl-5.28.0``    ``foss/2018b``
=========    ================    ==============


.. _list_software_snpEff_610:

*snpEff*
++++++++

SnpEff is a variant annotation and effect prediction tool.   It annotates and predicts the effects of genetic variants (such as amino acid changes).

*homepage*: http://sourceforge.net/projects/snpeff/

========    ==================    ==========
version     versionsuffix         toolchain 
========    ==================    ==========
``3.6``     ``-Java-1.7.0_80``    ``system``
``4.1d``    ``-Java-1.7.0_80``    ``system``
========    ==================    ==========


.. _list_software_SNPhylo_685:

*SNPhylo*
+++++++++

SNPhylo: a pipeline to generate a phylogenetic tree from huge SNP data

*homepage*: http://chibba.pgml.uga.edu/snphylo/

============    ==========================    ===============================
version         versionsuffix                 toolchain                      
============    ==========================    ===============================
``20140701``                                  ``foss/2016a``, ``intel/2016a``
``20160204``    ``-Python-2.7.14-R-3.4.3``    ``foss/2017b``, ``intel/2017b``
============    ==========================    ===============================


.. _list_software_SOAPaligner_1045:

*SOAPaligner*
+++++++++++++

SOAPaligner/soap2 is a member of the SOAP (Short Oligonucleotide   Analysis Package). It is an updated version of SOAP software for short oligonucleotide   alignment. 

*homepage*: http://soap.genomics.org.cn/soapaligner.html

========    =================    ==========
version     versionsuffix        toolchain 
========    =================    ==========
``2.21``    ``_Linux-x86_64``    ``system``
========    =================    ==========


.. _list_software_SOAPdenovo-Trans_1523:

*SOAPdenovo-Trans*
++++++++++++++++++

SOAPdenovo-Trans is a de novo transcriptome assembler basing on the SOAPdenovo framework, adapt to alternative splicing and different expression level among transcripts. 

*homepage*: http://soap.genomics.org.cn/SOAPdenovo-Trans.html

=========    ===============
version      toolchain      
=========    ===============
``1.0.4``    ``intel/2017a``
=========    ===============


.. _list_software_SOAPdenovo2_1008:

*SOAPdenovo2*
+++++++++++++

SOAPdenovo is a novel short-read assembly method that can build a   de novo draft assembly for human-sized genomes. The program is specially designed to   assemble Illumina short reads. It creates new opportunities for building reference   sequences and carrying out accurate analyses of unexplored genomes in a cost effective way.   SOAPdenovo2 is the successor of SOAPdenovo.

*homepage*: http://soap.genomics.org.cn/index.html

========    ===========================================================================================
version     toolchain                                                                                  
========    ===========================================================================================
``r240``    ``GCC/5.4.0-2.26``                                                                         
``r241``    ``GCC/6.4.0-2.28``, ``foss/2018a``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``, ``intel/2018a``
========    ===========================================================================================


.. _list_software_SOAPfuse_742:

*SOAPfuse*
++++++++++

SOAPfuse is an open source tool developed for genome-wide detection of fusion  transcripts from paired-end RNA-Seq data.

*homepage*: https://sourceforge.net/p/soapfuse/wiki/Home/

========    ================    ==============
version     versionsuffix       toolchain     
========    ================    ==============
``1.27``    ``-Perl-5.24.0``    ``foss/2016b``
``1.27``    ``-Perl-5.28.0``    ``foss/2018b``
========    ================    ==============


.. _list_software_socat_538:

*socat*
+++++++

socat is a relay for bidirectional data transfer between two independent data channels.

*homepage*: http://www.dest-unreach.org/socat

===========    =================
version        toolchain        
===========    =================
``1.7.3.3``    ``GCCcore/8.2.0``
===========    =================


.. _list_software_SolexaQA++_852:

*SolexaQA++*
++++++++++++

SolexaQA calculates sequence quality statistics and creates visual representations  of data quality for second-generation sequencing data. Originally developed for the Illumina system  (historically known as “Solexa”), SolexaQA now also supports Ion Torrent and 454 data.

*homepage*: http://solexaqa.sourceforge.net

=========    ==============
version      toolchain     
=========    ==============
``3.1.5``    ``foss/2016b``
=========    ==============


.. _list_software_SortMeRNA_827:

*SortMeRNA*
+++++++++++

SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.

*homepage*: http://bioinfo.lifl.fr/RNA/sortmerna/

=======    ==============
version    toolchain     
=======    ==============
``2.1``    ``foss/2016a``
=======    ==============


.. _list_software_Spack_498:

*Spack*
+++++++

Spack is a package manager for supercomputers, Linux, and macOS. It makes installing scientific  software easy. With Spack, you can build a package with multiple versions, configurations, platforms, and compilers,  and all of these builds can coexist on the same machine.

*homepage*: https://spack.io/

==========    ==========
version       toolchain 
==========    ==========
``0.10.0``    ``system``
``0.11.2``    ``system``
``0.12.1``    ``system``
==========    ==========


.. _list_software_SPAdes_544:

*SPAdes*
++++++++

Genome assembler for single-cell and isolates data sets

*homepage*: http://bioinf.spbau.ru/en/spades

==========    ==============================================
version       toolchain                                     
==========    ==============================================
``3.9.0``     ``foss/2016a``, ``foss/2016b``                
``3.10.1``    ``foss/2016b``, ``foss/2017a``                
``3.11.1``    ``foss/2017b``, ``foss/2018a``                
``3.12.0``    ``foss/2016b``, ``foss/2018a``, ``foss/2018b``
``3.13.0``    ``foss/2018b``                                
``3.13.1``    ``GCC/8.2.0-2.31.1``                          
==========    ==============================================


.. _list_software_spaln_542:

*spaln*
+++++++

Spaln (space-efficient spliced alignment) is a stand-alone program that maps  and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job.

*homepage*: https://github.com/ogotoh/spaln

==========    ========================================
version       toolchain                               
==========    ========================================
``2.3.3c``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
==========    ========================================


.. _list_software_Spark_513:

*Spark*
+++++++

Spark is Hadoop MapReduce done in memory

*homepage*: https://spark.apache.org

=========    ===========================================    ===============================
version      versionsuffix                                  toolchain                      
=========    ===========================================    ===============================
``1.3.0``                                                   ``system``                     
``1.4.1``                                                   ``system``                     
``1.5.0``                                                   ``system``                     
``1.6.0``                                                   ``system``                     
``1.6.1``                                                   ``system``                     
``2.0.0``                                                   ``system``                     
``2.0.2``                                                   ``system``                     
``2.2.0``    ``-Hadoop-2.6-Java-1.8.0_144``                 ``system``                     
``2.2.0``    ``-Hadoop-2.6-Java-1.8.0_152``                 ``system``                     
``2.2.0``    ``-Hadoop-2.6-Java-1.8.0_152-Python-3.6.3``    ``intel/2017b``                
``2.3.0``    ``-Hadoop-2.7-Java-1.8.0_162``                 ``system``                     
``2.4.0``    ``-Hadoop-2.7-Java-1.8``                       ``system``                     
``2.4.0``    ``-Hadoop-2.7-Java-1.8-Python-3.6.6``          ``intel/2018b``                
``2.4.0``    ``-Python-2.7.15``                             ``foss/2018b``, ``intel/2018b``
``2.4.0``    ``-Python-3.6.6``                              ``intel/2018b``                
=========    ===========================================    ===============================


.. _list_software_sparsehash_1074:

*sparsehash*
++++++++++++

An extremely memory-efficient hash_map  implementation. 2 bits/entry overhead! The SparseHash library  contains several hash-map implementations, including  implementations that optimize for space or speed.

*homepage*: https://github.com/sparsehash/sparsehash

=========    ===========================================================================================================
version      toolchain                                                                                                  
=========    ===========================================================================================================
``2.0.2``    ``foss/2016a``                                                                                             
``2.0.3``    ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``foss/2016b``, ``intel/2017a``
=========    ===========================================================================================================


.. _list_software_spectral.methods_1664:

*spectral.methods*
++++++++++++++++++

Contains some implementations of Singular Spectrum Analysis (SSA) for the gapfilling and  spectral decomposition of time series.

*homepage*: https://cran.r-project.org/web/packages/spectral.methods

=============    =============    ===============
version          versionsuffix    toolchain      
=============    =============    ===============
``0.7.2.133``    ``-R-3.4.3``     ``intel/2017b``
``0.7.2.133``    ``-R-3.4.4``     ``intel/2018a``
=============    =============    ===============


.. _list_software_spglib_641:

*spglib*
++++++++

Spglib is a library for finding and handling crystal symmetries written in C.

*homepage*: https://atztogo.github.io/spglib/

=========    =========================
version      toolchain                
=========    =========================
``1.9.2``    ``intel/2016.02-GCC-4.9``
``1.9.9``    ``intel/2017b``          
=========    =========================


.. _list_software_spglib-python_1360:

*spglib-python*
+++++++++++++++

Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C.

*homepage*: https://pypi.python.org/pypi/spglib

================    ==================    ===============
version             versionsuffix         toolchain      
================    ==================    ===============
``1.9.4.2``         ``-Python-2.7.12``    ``intel/2016b``
``1.9.5``           ``-Python-2.7.12``    ``intel/2016b``
``1.9.9.38``        ``-Python-2.7.13``    ``intel/2017a``
``1.10.0.2``        ``-Python-2.7.14``    ``intel/2017b``
``1.10.0.2``        ``-Python-3.6.3``     ``intel/2017b``
``1.14.1.post0``    ``-Python-3.7.2``     ``intel/2019a``
================    ==================    ===============


.. _list_software_Sphinx_634:

*Sphinx*
++++++++

Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.  It was originally created for the new Python documentation, and it has excellent facilities for the   documentation of Python projects, but C/C++ is already supported as well, and it is planned to add   special support for other languages as well.

*homepage*: http://sphinx.pocoo.org/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.4.8``    ``-Python-2.7.11``    ``foss/2016a``                 
``1.4.8``    ``-Python-3.5.1``     ``foss/2016a``                 
``1.8.1``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``1.8.1``    ``-Python-2.7.15``    ``foss/2018b``                 
``1.8.1``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``1.8.1``    ``-Python-3.6.4``     ``foss/2018a``                 
``1.8.1``    ``-Python-3.6.6``     ``foss/2018b``, ``intel/2018b``
``1.8.3``    ``-Python-3.6.4``     ``intel/2018a``                
=========    ==================    ===============================


.. _list_software_SpiceyPy_822:

*SpiceyPy*
++++++++++

SpiceyPy is a Python wrapper for the NAIF C SPICE Toolkit (N65)

*homepage*: https://github.com/AndrewAnnex/SpiceyPy

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.1.0``    ``-Python-2.7.12``    ``intel/2016b``                
``1.1.1``    ``-Python-3.6.1``     ``intel/2017a``                
``2.1.0``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``
``2.1.0``    ``-Python-3.6.4``     ``foss/2018a``                 
=========    ==================    ===============================


.. _list_software_SpiecEasi_886:

*SpiecEasi*
+++++++++++

Sparse InversE Covariance estimation for Ecological Association and Statistical Inference

*homepage*: https://github.com/zdk123/SpiecEasi

============    =============    ===============
version         versionsuffix    toolchain      
============    =============    ===============
``1.0.0``       ``-R-3.4.4``     ``intel/2018a``
``20160830``    ``-R-3.3.1``     ``intel/2016b``
============    =============    ===============


.. _list_software_SPLASH_459:

*SPLASH*
++++++++

SPLASH is a free and open source visualisation tool  for Smoothed Particle Hydrodynamics (SPH) simulations.

*homepage*: http://users.monash.edu.au/~dprice/splash/index.html

=========    ==============
version      toolchain     
=========    ==============
``2.8.0``    ``foss/2018b``
=========    ==============


.. _list_software_SpliceMap_894:

*SpliceMap*
+++++++++++

SpliceMap is a de novo splice junction discovery and alignment tool.  It offers high sensitivity and support for arbitrary RNA-seq read lengths.

*homepage*: https://web.stanford.edu/group/wonglab/SpliceMap/

===========    ==================
version        toolchain         
===========    ==================
``3.3.5.2``    ``GCC/7.3.0-2.30``
===========    ==================


.. _list_software_splitRef_841:

*splitRef*
++++++++++

splitRef splits a reference haplotype file into smaller files   with subsets of markers. The current version is a pre-release. 

*homepage*: http://www.sph.umich.edu/csg/yli/splitRef/index.html

=========    ==========
version      toolchain 
=========    ==========
``0.0.2``    ``system``
=========    ==========


.. _list_software_SPRNG_394:

*SPRNG*
+++++++

Scalable Parallel Pseudo Random Number Generators Library

*homepage*: http://www.sprng.org/

========    ==============
version     toolchain     
========    ==============
``2.0b``    ``foss/2016a``
========    ==============


.. _list_software_Spyder_631:

*Spyder*
++++++++

Spyder is an interactive Python development environment providing MATLAB-like features in a simple  and light-weighted software.

*homepage*: https://github.com/spyder-ide/spyder

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.1.4``    ``-Python-2.7.13``    ``intel/2017a``
``3.3.1``    ``-Python-3.6.4``     ``foss/2018a`` 
``3.3.2``    ``-Python-3.6.6``     ``foss/2018b`` 
=========    ==================    ===============


.. _list_software_SQLite_562:

*SQLite*
++++++++

SQLite: SQL Database Engine in a C Library

*homepage*: http://www.sqlite.org/

============    ====================================================================================================================================================================
version         toolchain                                                                                                                                                           
============    ====================================================================================================================================================================
``3.8.8.1``     ``GCC/4.8.4``, ``GCC/4.9.2``                                                                                                                                        
``3.8.10.2``    ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``gimkl/2.11.5``                                                                                                            
``3.9.2``       ``GCC/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``               
``3.13.0``      ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``3.14.1``      ``GCCcore/4.9.3``                                                                                                                                                   
``3.17.0``      ``GCCcore/6.3.0``                                                                                                                                                   
``3.20.1``      ``GCCcore/6.4.0``                                                                                                                                                   
``3.21.0``      ``GCCcore/6.4.0``                                                                                                                                                   
``3.23.0``      ``GCCcore/6.4.0``                                                                                                                                                   
``3.24.0``      ``GCCcore/7.2.0``, ``GCCcore/7.3.0``                                                                                                                                
``3.26.0``      ``GCCcore/8.2.0``                                                                                                                                                   
``3.27.2``      ``GCCcore/8.2.0``                                                                                                                                                   
``3.29.0``      ``GCCcore/8.3.0``                                                                                                                                                   
============    ====================================================================================================================================================================


.. _list_software_SqueezeMeta_1129:

*SqueezeMeta*
+++++++++++++

SqueezeMeta is a full automatic pipeline for metagenomics/metatranscriptomics,  covering all steps of the analysis.

*homepage*: https://github.com/jtamames/SqueezeMeta

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``0.4.3``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_SRA-Toolkit_1017:

*SRA-Toolkit*
+++++++++++++

The SRA Toolkit, and the source-code SRA System Development  Kit (SDK), will allow you to programmatically access data housed within SRA  and convert it from the SRA format

*homepage*: http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software

===========    ===================    ==========
version        versionsuffix          toolchain 
===========    ===================    ==========
``2.3.5``      ``-centos_linux64``    ``system``
``2.5.4-1``    ``-centos_linux64``    ``system``
``2.5.7``      ``-centos_linux64``    ``system``
``2.8.2-1``    ``-centos_linux64``    ``system``
``2.9.0``      ``-centos_linux64``    ``system``
``2.9.2``      ``-ubuntu64``          ``system``
``2.9.4``      ``-centos_linux64``    ``system``
``2.9.6-1``    ``-centos_linux64``    ``system``
===========    ===================    ==========


.. _list_software_SRPRISM_560:

*SRPRISM*
+++++++++

Single Read Paired Read Indel Substitution Minimizer

*homepage*: https://github.com/ncbi/SRPRISM

=========    =============    =================
version      versionsuffix    toolchain        
=========    =============    =================
``3.0.0``                     ``foss/2018b``   
``3.1.1``    ``-Java-11``     ``GCCcore/8.2.0``
=========    =============    =================


.. _list_software_SSAHA2_418:

*SSAHA2*
++++++++

SSAHA2 (Sequence Search and Alignment by Hashing Algorithm) is a   pairwise sequence alignment program designed for the efficient mapping of sequencing   reads onto genomic reference sequences. SSAHA2 reads of most sequencing platforms   (ABI-Sanger, Roche 454, Illumina-Solexa) and a range of output formats (SAM, CIGAR, PSL etc.)   are supported. A pile-up pipeline for analysis and genotype calling is available as   a separate package. 

*homepage*: http://www.sanger.ac.uk/resources/software/ssaha2/

=========    =============    ==========
version      versionsuffix    toolchain 
=========    =============    ==========
``2.5.5``    ``-i686``        ``system``
``2.5.5``    ``-x86_64``      ``system``
=========    =============    ==========


.. _list_software_SSPACE_Basic_1024:

*SSPACE_Basic*
++++++++++++++

SSPACE Basic, SSAKE-based Scaffolding of Pre-Assembled Contigs after Extension

*homepage*: https://github.com/nsoranzo/sspace_basic

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``2.1.1``    ``-Perl-5.24.1``    ``intel/2017a``
``2.1.1``    ``-Perl-5.26.0``    ``intel/2017b``
=========    ================    ===============


.. _list_software_Stacks_617:

*Stacks*
++++++++

Stacks is a software pipeline for building loci from short-read sequences, such as those generated on  the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,  for the purpose of building genetic maps and conducting population genomics and phylogeography. 

*homepage*: http://catchenlab.life.illinois.edu/stacks

==============    ==============================================================================
version           toolchain                                                                     
==============    ==============================================================================
``1.40``          ``foss/2016a``                                                                
``1.42``          ``foss/2016a``                                                                
``1.44``          ``foss/2016a``                                                                
``1.45``          ``foss/2016a``                                                                
``1.46``          ``intel/2017a``                                                               
``1.47``          ``foss/2016a``                                                                
``1.48``          ``intel/2017b``, ``intel/2018b``                                              
``2.0``           ``foss/2018a``, ``intel/2018a``                                               
``2.0Beta7c``     ``intel/2017b``                                                               
``2.0Beta8c``     ``intel/2017b``                                                               
``2.0Beta9``      ``intel/2018a``                                                               
``2.0Beta10a``    ``foss/2018a``                                                                
``2.2``           ``foss/2018a``                                                                
``2.3b``          ``foss/2018a``                                                                
``2.3e``          ``foss/2018b``                                                                
``2.41``          ``GCC/8.2.0-2.31.1``, ``foss/2018b``, ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
==============    ==============================================================================


.. _list_software_STAMP_389:

*STAMP*
+++++++

STAMP is a tool for characterizing similarities between transcription factor binding motifs

*homepage*: http://www.benoslab.pitt.edu/stamp/

=======    ================================
version    toolchain                       
=======    ================================
``1.2``    ``intel/2016a``                 
``1.3``    ``intel/2016a``, ``intel/2016b``
=======    ================================


.. _list_software_StaMPS_536:

*StaMPS*
++++++++

A software package to extract ground displacements from time series of synthetic aperture radar  (SAR) acquisitions.

*homepage*: http://homepages.see.leeds.ac.uk/~earahoo/stamps/

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``3.3b1``    ``-Perl-5.24.1``    ``intel/2017a``
=========    ================    ===============


.. _list_software_Stampy_638:

*Stampy*
++++++++

Stampy is a package for the mapping of short reads from illumina sequencing machines onto  a reference genome.

*homepage*: http://www.well.ox.ac.uk/stampy

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.0.31``    ``-Python-2.7.12``    ``intel/2016b``
``1.0.32``    ``-Python-2.7.14``    ``intel/2018a``
==========    ==================    ===============


.. _list_software_STAR_314:

*STAR*
++++++

STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays.

*homepage*: https://github.com/alexdobin/STAR

==========    ===============================================
version       toolchain                                      
==========    ===============================================
``2.4.2a``    ``foss/2018b``                                 
``2.5.0a``    ``GNU/4.9.3-2.25``                             
``2.5.2a``    ``foss/2016a``                                 
``2.5.2b``    ``intel/2016b``                                
``2.5.3a``    ``intel/2017a``, ``intel/2017b``               
``2.5.4b``    ``foss/2016b``, ``foss/2017b``, ``intel/2017b``
``2.6.0c``    ``foss/2018a``, ``intel/2018a``                
``2.6.1c``    ``foss/2018b``                                 
``2.7.0d``    ``foss/2018b``                                 
``2.7.0f``    ``GCC/8.2.0-2.31.1``, ``foss/2018b``           
``2.7.1a``    ``GCC/8.2.0-2.31.1``, ``foss/2018b``           
==========    ===============================================


.. _list_software_STAR-Fusion_987:

*STAR-Fusion*
+++++++++++++

STAR-Fusion uses the STAR aligner to identify candidate fusion transcripts  supported by Illumina reads. STAR-Fusion further processes the output generated by the STAR aligner  to map junction reads and spanning reads to a reference annotation set.

*homepage*: https://github.com/STAR-Fusion/STAR-Fusion

=========    =============================    ====================
version      versionsuffix                    toolchain           
=========    =============================    ====================
``1.5.0``    ``-Perl-5.28.0``                 ``foss/2018b``      
``1.6.0``    ``-Perl-5.28.1-Python-3.7.2``    ``GCC/8.2.0-2.31.1``
=========    =============================    ====================


.. _list_software_Stata_509:

*Stata*
+++++++

Stata is a complete, integrated statistical software package that provides everything you need  for data analysis, data management, and graphics.

*homepage*: https://www.stata.com/

=======    ==========
version    toolchain 
=======    ==========
``15``     ``system``
=======    ==========


.. _list_software_statsmodels_1203:

*statsmodels*
+++++++++++++

Statsmodels is a Python module that provides classes and functions  for the estimation of many different statistical models, as well as for conducting   statistical tests, and statistical data exploration.

*homepage*: https://www.statsmodels.org

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``0.6.1``     ``-Python-2.7.13``    ``foss/2017a``                 
``0.6.1``     ``-Python-3.5.1``     ``intel/2016a``                
``0.6.1``     ``-Python-3.5.2``     ``intel/2016b``                
``0.8.0``     ``-Python-2.7.13``    ``intel/2017a``                
``0.9.0``     ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
``0.10.1``                          ``foss/2019a``                 
==========    ==================    ===============================


.. _list_software_STIR_322:

*STIR*
++++++

Software for Tomographic Image Reconstruction

*homepage*: http://stir.sourceforge.net/

=======    ===============
version    toolchain      
=======    ===============
``3.0``    ``intel/2018a``
=======    ===============


.. _list_software_STREAM_460:

*STREAM*
++++++++

The STREAM benchmark is a simple synthetic benchmark program that measures sustainable  memory bandwidth (in MB/s) and the corresponding computation rate for simple vector kernels.

*homepage*: https://www.cs.virginia.edu/stream/

========    ==========================================================================
version     toolchain                                                                 
========    ==========================================================================
``5.10``    ``GCC/7.3.0-2.30``, ``GCC/8.2.0-2.31.1``, ``intel/2016b``, ``intel/2018b``
========    ==========================================================================


.. _list_software_strelka_758:

*strelka*
+++++++++

Strelka2 is a fast and accurate small variant caller  optimized for analysis of germline variation in small cohorts and  somatic variation in tumor/normal sample pairs.

*homepage*: https://github.com/Illumina/strelka

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``2.9.7``                           ``intel/2018a``                
``2.9.9``                           ``foss/2018b``                 
``2.9.10``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
==========    ==================    ===============================


.. _list_software_StringTie_921:

*StringTie*
+++++++++++

StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.

*homepage*: http://ccb.jhu.edu/software/%(namelower)/

==========    ==================================
version       toolchain                         
==========    ==================================
``1.3.0``     ``intel/2016b``                   
``1.3.3``     ``GCCcore/6.4.0``, ``intel/2017a``
``1.3.3b``    ``foss/2016b``                    
``1.3.5``     ``GCCcore/8.2.0``, ``foss/2018b`` 
``2.0.3``     ``GCCcore/7.3.0``                 
==========    ==================================


.. _list_software_Structure_977:

*Structure*
+++++++++++

The program structure is a free software package for using multi-locus genotype data to investigate  population structure.

*homepage*: https://web.stanford.edu/group/pritchardlab/structure.html

=========    ======================================
version      toolchain                             
=========    ======================================
``2.3.4``    ``iccifort/2019.3.199-GCC-8.3.0-2.32``
=========    ======================================


.. _list_software_Subread_710:

*Subread*
+++++++++

High performance read alignment, quantification and mutation discovery

*homepage*: http://subread.sourceforge.net/

============    ==============================
version         toolchain                     
============    ==============================
``1.5.0-p1``    ``foss/2016a``, ``foss/2016b``
``1.6.3``       ``foss/2018b``                
``1.6.4``       ``foss/2018b``                
``2.0.0``       ``GCC/7.3.0-2.30``            
============    ==============================


.. _list_software_Subversion_1072:

*Subversion*
++++++++++++

Subversion is an open source version control system.

*homepage*: http://subversion.apache.org/

==========    ===============================
version       toolchain                      
==========    ===============================
``1.9.7``     ``iomkl/2018a``                
``1.9.9``     ``GCCcore/7.3.0``              
``1.10.0``    ``foss/2017b``, ``intel/2017b``
``1.12.0``    ``GCCcore/8.2.0``              
==========    ===============================


.. _list_software_suds_447:

*suds*
++++++

Lightweight SOAP client

*homepage*: https://pypi.python.org/pypi/suds-py3

===========    =================    ===============
version        versionsuffix        toolchain      
===========    =================    ===============
``1.3.3.0``    ``-Python-3.6.4``    ``intel/2018a``
===========    =================    ===============


.. _list_software_SuiteSparse_1144:

*SuiteSparse*
+++++++++++++

SuiteSparse is a collection of libraries manipulate sparse matrices.

*homepage*: http://faculty.cse.tamu.edu/davis/suitesparse.html

=========    ===================    =================================================================================================
version      versionsuffix          toolchain                                                                                        
=========    ===================    =================================================================================================
``4.5.1``    ``-METIS-5.1.0``       ``foss/2016a``, ``intel/2016a``                                                                  
``4.5.2``    ``-METIS-5.1.0``       ``foss/2016a``, ``intel/2016a``                                                                  
``4.5.3``    ``-METIS-5.1.0``       ``foss/2016a``, ``foss/2016b``, ``intel/2016b``                                                  
``4.5.3``    ``-ParMETIS-4.0.3``    ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``                                 
``4.5.5``    ``-METIS-5.1.0``       ``foss/2017a``, ``intel/2017a``                                                                  
``4.5.5``    ``-ParMETIS-4.0.3``    ``foss/2017a``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``                                 
``4.5.6``    ``-METIS-5.1.0``       ``foss/2017b``                                                                                   
``5.1.2``    ``-METIS-5.1.0``       ``foss/2017b``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``
``5.1.2``    ``-ParMETIS-4.0.3``    ``foss/2017b``, ``intel/2017b``                                                                  
``5.4.0``    ``-METIS-5.1.0``       ``foss/2019a``, ``intel/2018b``, ``intel/2019a``                                                 
=========    ===================    =================================================================================================


.. _list_software_SUMACLUST_705:

*SUMACLUST*
+++++++++++

SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.

*homepage*: http://metabarcoding.org/sumatra

==========    ==============
version       toolchain     
==========    ==============
``1.0.20``    ``foss/2016a``
==========    ==============


.. _list_software_SUMATRA_541:

*SUMATRA*
+++++++++

SUMATRA and SUMACLUST: fast and exact comparison and clustering of sequences.

*homepage*: http://metabarcoding.org/sumatra

==========    ==============
version       toolchain     
==========    ==============
``1.0.20``    ``foss/2016a``
==========    ==============


.. _list_software_SUNDIALS_611:

*SUNDIALS*
++++++++++

SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers

*homepage*: http://computation.llnl.gov/projects/sundials

=========    ==================================================================================================================
version      toolchain                                                                                                         
=========    ==================================================================================================================
``2.6.2``    ``intel/2016b``, ``intel/2018b``                                                                                  
``2.7.0``    ``foss/2016b``, ``foss/2017b``, ``foss/2018a``, ``intel/2016b``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
=========    ==================================================================================================================


.. _list_software_SuperLU_688:

*SuperLU*
+++++++++

SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems  of linear equations on high performance machines.

*homepage*: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/

=========    ===============================
version      toolchain                      
=========    ===============================
``5.1.1``    ``foss/2016a``, ``intel/2016a``
``5.2.1``    ``foss/2017b``, ``intel/2017b``
=========    ===============================


.. _list_software_supermagic_1072:

*supermagic*
++++++++++++

Very simple MPI sanity code. Nothing more, nothing less.

*homepage*: https://github.com/hpc/supermagic

============    ==============
version         toolchain     
============    ==============
``20170824``    ``foss/2017a``
============    ==============


.. _list_software_supernova_995:

*supernova*
+++++++++++

Supernova is a software package for de novo assembly from Chromium Linked-Reads  that are made from a single whole-genome library from an individual DNA source

*homepage*: https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome

=========    ==========
version      toolchain 
=========    ==========
``2.0.1``    ``system``
``2.1.1``    ``system``
=========    ==========


.. _list_software_SVDetect_770:

*SVDetect*
++++++++++

SVDetect is a application for the isolation and the type prediction of intra- and inter-chromosomal  rearrangements from paired-end/mate-pair sequencing data provided by the high-throughput sequencing technologies.  This tool aims to identifying structural variations with both clustering and sliding-window strategies, and  helping in their visualization at the genome scale.

*homepage*: http://svdetect.sourceforge.net/Site/Home.html

========    ================    ==========================================================
version     versionsuffix       toolchain                                                 
========    ================    ==========================================================
``0.8b``    ``-Perl-5.26.0``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
========    ================    ==========================================================


.. _list_software_SVDquest_799:

*SVDquest*
++++++++++

SVDquartets-based species trees

*homepage*: https://github.com/pranjalv123/SVDquest

============    ===============
version         toolchain      
============    ===============
``20190627``    ``gompi/2019a``
============    ===============


.. _list_software_swarm_554:

*swarm*
+++++++

A robust and fast clustering method for amplicon-based studies 

*homepage*: https://github.com/torognes/swarm

=========    ==============
version      toolchain     
=========    ==============
``2.2.2``    ``foss/2018b``
=========    ==============


.. _list_software_SWASH_390:

*SWASH*
+++++++

SWASH is a general-purpose numerical tool for simulating unsteady, non-hydrostatic, free-surface,  rotational flow and transport phenomena in coastal waters as driven by waves, tides, buoyancy and wind forces.

*homepage*: http://swash.sourceforge.net/

========    =============    ================================
version     versionsuffix    toolchain                       
========    =============    ================================
``3.14``    ``-mpi``         ``intel/2016b``, ``intel/2017a``
``4.01``    ``-mpi``         ``intel/2017a``                 
========    =============    ================================


.. _list_software_SWIG_314:

*SWIG*
++++++

SWIG is a software development tool that connects programs written in C and C++ with  a variety of high-level programming languages.

*homepage*: http://www.swig.org/

==========    ============================    ================================================================
version       versionsuffix                   toolchain                                                       
==========    ============================    ================================================================
``3.0.8``     ``-Python-2.7.11``              ``foss/2016a``, ``intel/2016a``                                 
``3.0.8``     ``-Python-3.5.1``               ``foss/2016a``                                                  
``3.0.10``    ``-Python-2.7.12``              ``foss/2016b``, ``intel/2016b``                                 
``3.0.10``    ``-Python-2.7.12-PCRE-8.39``    ``intel/2016b``                                                 
``3.0.11``    ``-Python-2.7.12``              ``foss/2016b``, ``intel/2016b``                                 
``3.0.12``                                    ``GCCcore/8.2.0``                                               
``3.0.12``    ``-Python-2.7.13``              ``intel/2017a``                                                 
``3.0.12``    ``-Python-2.7.14``              ``foss/2017b``, ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``3.0.12``    ``-Python-2.7.15``              ``GCCcore/8.2.0``, ``foss/2018b``, ``intel/2018b``              
``3.0.12``    ``-Python-3.6.1``               ``intel/2017a``                                                 
``3.0.12``    ``-Python-3.6.2``               ``foss/2017b``, ``intel/2018.00``                               
``3.0.12``    ``-Python-3.6.3``               ``foss/2017b``, ``intel/2017b``, ``intel/2018.01``              
``3.0.12``    ``-Python-3.6.4``               ``foss/2018a``, ``intel/2018a``, ``iomkl/2018a``                
``3.0.12``    ``-Python-3.6.6``               ``foss/2018b``, ``intel/2018b``                                 
``3.0.12``    ``-Python-3.7.2``               ``GCCcore/8.2.0``                                               
==========    ============================    ================================================================


.. _list_software_SymEngine_911:

*SymEngine*
+++++++++++

SymEngine is a standalone fast C++ symbolic manipulation library.

*homepage*: https://github.com/symengine/symengine

=========    =============    ====================
version      versionsuffix    toolchain           
=========    =============    ====================
``0.3.0``    ``-20181006``    ``intel/2018a``     
``0.4.0``                     ``GCC/8.2.0-2.31.1``
=========    =============    ====================


.. _list_software_sympy_578:

*sympy*
+++++++

SymPy is a Python library for symbolic mathematics. It aims to  become a full-featured computer algebra system (CAS) while keeping the code as  simple as possible in order to be comprehensible and easily extensible. SymPy  is written entirely in Python and does not require any external libraries.

*homepage*: http://sympy.org/

=======    ==================    ===============================
version    versionsuffix         toolchain                      
=======    ==================    ===============================
``1.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``1.3``    ``-Python-2.7.14``    ``intel/2018a``                
``1.3``    ``-Python-2.7.15``    ``foss/2018b``                 
``1.3``    ``-Python-3.6.4``     ``intel/2018a``                
``1.3``    ``-Python-3.6.6``     ``foss/2018b``                 
``1.4``                          ``foss/2019a``, ``intel/2019a``
=======    ==================    ===============================


.. _list_software_Szip_422:

*Szip*
++++++

Szip compression software, providing lossless compression of scientific data 

*homepage*: https://www.hdfgroup.org/doc_resource/SZIP/

=========    =========================================================================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                                                                                
=========    =========================================================================================================================================================================================================================================================================
``2.1``      ``GCC/4.8.1``, ``GCCcore/5.4.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``intel/2017.01``, ``intel/2017a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``2.1.1``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                                                                                                            
=========    =========================================================================================================================================================================================================================================================================


.. _list_software_letter_t:

*T*
---


:ref:`list_software_tabix_536` - :ref:`list_software_taco_423` - :ref:`list_software_TagLib_563` - :ref:`list_software_TAMkin_548` - :ref:`list_software_taxator-tk_1039` - :ref:`list_software_tbb_312` - :ref:`list_software_tbl2asn_694` - :ref:`list_software_TCC_218` - :ref:`list_software_Tcl_291` - :ref:`list_software_tcsh_434` - :ref:`list_software_tecplot360ex_1137` - :ref:`list_software_TensorFlow_1043` - :ref:`list_software_TensorRT_801` - :ref:`list_software_Tesla-Deployment-Kit_1948` - :ref:`list_software_tesseract_974` - :ref:`list_software_testpath_877` - :ref:`list_software_TetGen_583` - :ref:`list_software_TEToolkit_895` - :ref:`list_software_texinfo_765` - :ref:`list_software_Text-CSV_702` - :ref:`list_software_Theano_607` - :ref:`list_software_THetA_438` - :ref:`list_software_thurstonianIRT_1454` - :ref:`list_software_TiCCutils_884` - :ref:`list_software_Tika_393` - :ref:`list_software_TiMBL_408` - :ref:`list_software_time_431` - :ref:`list_software_TINKER_461` - :ref:`list_software_Tk_191` - :ref:`list_software_Tkinter_737` - :ref:`list_software_TM-align_729` - :ref:`list_software_tmux_462` - :ref:`list_software_ToFu_382` - :ref:`list_software_TopHat_592` - :ref:`list_software_torchvision_1208` - :ref:`list_software_TotalView_927` - :ref:`list_software_tqdm_438` - :ref:`list_software_Tracer_609` - :ref:`list_software_TransDecoder_1214` - :ref:`list_software_Transrate_948` - :ref:`list_software_travis_665` - :ref:`list_software_TreeMix_702` - :ref:`list_software_TRF_236` - :ref:`list_software_Triangle_822` - :ref:`list_software_Trilinos_852` - :ref:`list_software_Trim_Galore_1109` - :ref:`list_software_trimAl_617` - :ref:`list_software_Trimmomatic_1158` - :ref:`list_software_Trinity_755` - :ref:`list_software_TRIQS_403` - :ref:`list_software_TRIQS-cthyb_986` - :ref:`list_software_TRIQS-dft_tools_1422` - :ref:`list_software_TRIQS-tprf_892` - :ref:`list_software_TRUST_418` - :ref:`list_software_TurboVNC_755` - :ref:`list_software_TVB_236` - :ref:`list_software_tvb-data_787` - :ref:`list_software_TVB-deps_709` - :ref:`list_software_tvb-framework_1351` - :ref:`list_software_tvb-library_1134`


.. _list_software_tabix_536:

*tabix*
+++++++

Generic indexer for TAB-delimited genome position files 

*homepage*: http://samtools.sourceforge.net

=========    =========================================================================================================
version      toolchain                                                                                                
=========    =========================================================================================================
``0.2.6``    ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2016a``, ``intel/2016a``, ``intel/2016b``
=========    =========================================================================================================


.. _list_software_taco_423:

*taco*
++++++

Multi-sample transcriptome assembly from RNA-Seq

*homepage*: http://tacorna.github.io

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.5.1``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_TagLib_563:

*TagLib*
++++++++

TagLib is a library for reading and editing the meta-data of several popular audio formats.

*homepage*: https://taglib.org/

==========    =================
version       toolchain        
==========    =================
``1.11.1``    ``GCCcore/8.2.0``
==========    =================


.. _list_software_TAMkin_548:

*TAMkin*
++++++++

TAMkin is a post-processing toolkit for normal mode analysis,  thermochemistry and reaction kinetics. It uses a Hessian computation from a  standard computational chemistry program as its input.

*homepage*: https://molmod.github.io/tamkin/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.0.9``    ``-Python-2.7.11``    ``intel/2016a``
``1.2.4``    ``-Python-2.7.14``    ``intel/2017b``
``1.2.6``    ``-Python-3.7.2``     ``intel/2019a``
=========    ==================    ===============


.. _list_software_taxator-tk_1039:

*taxator-tk*
++++++++++++

A set of programs for the taxonomic analysis of nucleotide sequence data

*homepage*: https://github.com/fungs/taxator-tk

=========    ===============================
version      toolchain                      
=========    ===============================
``1.3.3``    ``foss/2018b``, ``gompi/2019a``
=========    ===============================


.. _list_software_tbb_312:

*tbb*
+++++

Intel Threading Building Blocks (Intel TBB)  is a widely used, award-winning C++ template library for creating reliable,  portable, and scalable parallel applications.  Use Intel TBB for a simple and rapid way of developing robust task-based  parallel applications that scale to available processor cores, are compatible  with multiple environments, and are easier to maintain.  Intel TBB is the most proficient way to implement future-proof parallel applications  that tap into the power and performance of multicore and manycore hardware platforms.

*homepage*: http://software.intel.com/en-us/articles/intel-tbb/

==============    ==================================================
version           toolchain                                         
==============    ==================================================
``4.0.0.233``     ``system``                                        
``4.0.5.339``     ``system``                                        
``4.3.6.211``     ``system``                                        
``4.4.2.152``     ``system``                                        
``2017.2.132``    ``system``                                        
``2017.4.174``    ``system``                                        
``2017.6.196``    ``system``                                        
``2017_U5``       ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2017a``
``2017_U6``       ``GCCcore/6.3.0``, ``intel/2017a``                
``2018_U1``       ``GCCcore/6.4.0``                                 
``2018_U2``       ``GCCcore/6.4.0``                                 
``2018_U3``       ``GCCcore/6.4.0``                                 
``2018_U5``       ``GCCcore/6.4.0``, ``GCCcore/7.3.0``              
``2019_U4``       ``GCCcore/8.2.0``                                 
``2019_U9``       ``GCCcore/8.3.0``                                 
==============    ==================================================


.. _list_software_tbl2asn_694:

*tbl2asn*
+++++++++

Tbl2asn is a command-line program that automates the creation of  sequence records for submission to GenBank

*homepage*: https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/

============    =============    ==========
version         versionsuffix    toolchain 
============    =============    ==========
``20180227``    ``-linux64``     ``system``
============    =============    ==========


.. _list_software_TCC_218:

*TCC*
+++++

The Tiny C Compiler (aka TCC, tCc, or TinyCC) is an x86 and x86-64 C compiler  created by Fabrice Bellard. It is designed to work for slow computers with little disk space  and can run shebang style !/usr/bin/tcc . TCC is distributed under the LGPL. TCC claims to  implement all of ANSI C (C89/C90),[1] much of the new ISO C99 standard,  and many GNU C extensions including inline assembly.

*homepage*: http://bellard.org/tcc/

==========    ==========
version       toolchain 
==========    ==========
``0.9.26``    ``system``
==========    ==========


.. _list_software_Tcl_291:

*Tcl*
+++++

Tcl (Tool Command Language) is a very powerful but easy to learn dynamic  programming language, suitable for a very wide range of uses, including web  and desktop applications, networking, administration, testing and many more. 

*homepage*: https://www.tcl.tk/

==========    =========================================================================================================================================================================
version       toolchain                                                                                                                                                                
==========    =========================================================================================================================================================================
``8.5.19``    ``foss/2017a``                                                                                                                                                           
``8.6.3``     ``GCC/4.8.4``, ``GCC/4.9.2``                                                                                                                                             
``8.6.4``     ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``
``8.6.5``     ``GCC/4.9.3-2.25``, ``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``     
``8.6.6``     ``GCCcore/4.9.3``, ``GCCcore/6.3.0``                                                                                                                                     
``8.6.7``     ``GCCcore/6.4.0``                                                                                                                                                        
``8.6.8``     ``GCCcore/6.4.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``                                                                                                                  
``8.6.9``     ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                                                                     
==========    =========================================================================================================================================================================


.. _list_software_tcsh_434:

*tcsh*
++++++

Tcsh is an enhanced, but completely compatible version of the Berkeley UNIX C shell (csh).   It is a command language interpreter usable both as an interactive login shell and a shell script command   processor. It includes a command-line editor, programmable word completion, spelling correction, a history   mechanism, job control and a C-like syntax.

*homepage*: http://www.tcsh.org

===========    =======================================================
version        toolchain                                              
===========    =======================================================
``6.19.00``    ``intel/2016a``                                        
``6.20.00``    ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
===========    =======================================================


.. _list_software_tecplot360ex_1137:

*tecplot360ex*
++++++++++++++

Quickly plot and animate your CFD results exactly the way you want. Analyze complex solutions, arrange multiple layouts, and communicate your results with professional images and animations.

*homepage*: http://www.tecplot.com/products/tecplot-360/

===========    ==========
version        toolchain 
===========    ==========
``linux64``    ``system``
===========    ==========


.. _list_software_TensorFlow_1043:

*TensorFlow*
++++++++++++

An open-source software library for Machine Intelligence

*homepage*: https://www.tensorflow.org/

==========    =============================    ===================================================
version       versionsuffix                    toolchain                                          
==========    =============================    ===================================================
``0.12.1``    ``-Python-2.7.12``               ``foss/2016b``, ``intel/2016b``                    
``0.12.1``    ``-Python-3.5.2``                ``foss/2016b``, ``intel/2016b``                    
``1.0.1``     ``-Python-2.7.12``               ``intel/2016b``                                    
``1.0.1``     ``-Python-3.5.2``                ``intel/2016b``                                    
``1.1.0``     ``-Python-2.7.13``               ``intel/2017a``                                    
``1.1.0``     ``-Python-3.6.1``                ``intel/2017a``                                    
``1.2.0``     ``-Python-3.6.1``                ``intel/2017a``                                    
``1.2.1``     ``-GPU-Python-3.5.2``            ``foss/2016b``                                     
``1.2.1``     ``-Python-3.5.2``                ``foss/2016b``                                     
``1.3.0``     ``-Python-2.7.13``               ``intel/2017a``                                    
``1.3.0``     ``-Python-3.6.1``                ``intel/2017a``                                    
``1.3.0``     ``-Python-3.6.3``                ``intel/2017b``                                    
``1.4.0``     ``-Python-3.6.3``                ``foss/2017b``, ``intel/2017b``                    
``1.4.1``     ``-Python-3.6.3``                ``foss/2017b``                                     
``1.5.0``     ``-Python-3.6.3``                ``foss/2017b``, ``intel/2017b``                    
``1.6.0``     ``-Python-3.6.4``                ``foss/2018a``, ``intel/2018a``                    
``1.6.0``     ``-Python-3.6.4-CUDA-9.1.85``    ``foss/2018a``                                     
``1.7.0``     ``-Python-3.6.4``                ``foss/2018a``                                     
``1.7.0``     ``-Python-3.6.4-CUDA-9.1.85``    ``foss/2018a``                                     
``1.8.0``     ``-Python-2.7.14``               ``foss/2017b``, ``fosscuda/2017b``                 
``1.8.0``     ``-Python-3.6.3``                ``foss/2017b``, ``fosscuda/2017b``                 
``1.8.0``     ``-Python-3.6.4``                ``foss/2018a``, ``fosscuda/2018a``, ``intel/2018a``
``1.10.0``    ``-Python-2.7.15``               ``fosscuda/2018b``                                 
``1.10.1``    ``-Python-2.7.15``               ``fosscuda/2018b``                                 
``1.10.1``    ``-Python-3.6.6``                ``foss/2018b``                                     
``1.11.0``    ``-Python-3.6.6``                ``foss/2018b``                                     
``1.12.0``    ``-Python-2.7.15``               ``fosscuda/2018b``                                 
``1.12.0``    ``-Python-3.6.6``                ``foss/2018b``, ``fosscuda/2018b``                 
``1.13.1``    ``-Python-3.6.6``                ``foss/2018b``                                     
``1.13.1``    ``-Python-3.7.2``                ``foss/2019a``, ``fosscuda/2019a``                 
``1.14.0``    ``-Python-3.7.2``                ``foss/2019a``, ``fosscuda/2019a``                 
``1.15.0``    ``-Python-3.7.4``                ``foss/2019b``, ``fosscuda/2019b``                 
``2.0.0``     ``-Python-3.7.2``                ``foss/2019a``                                     
``2.0.0``     ``-Python-3.7.4``                ``fosscuda/2019b``                                 
==========    =============================    ===================================================


.. _list_software_TensorRT_801:

*TensorRT*
++++++++++

NVIDIA TensorRT is a platform for high-performance deep learning inference

*homepage*: https://developer.nvidia.com/tensorrt

===========    ==================    ==================
version        versionsuffix         toolchain         
===========    ==================    ==================
``4.0.1.6``    ``-Python-2.7.15``    ``fosscuda/2018b``
===========    ==================    ==================


.. _list_software_Tesla-Deployment-Kit_1948:

*Tesla-Deployment-Kit*
++++++++++++++++++++++

The Tesla Deployment Kit is a set of tools provided primarily for the NVIDIA Tesla  range of GPUs. They aim to empower users to better manage their NVIDIA GPUs by providing a broad  range of functionalities. The kit contains:  * NVIDIA Management Library (NVML),  * Tesla Deployment Kit - Linux Edition (Aug 1st, 2013)

*homepage*: https://developer.nvidia.com/tesla-deployment-kit

============    ==========
version         toolchain 
============    ==========
``5.319.43``    ``system``
============    ==========


.. _list_software_tesseract_974:

*tesseract*
+++++++++++

Tesseract is an optical character recognition engine

*homepage*: https://github.com/tesseract-ocr/tesseract

=========    =================
version      toolchain        
=========    =================
``4.0.0``    ``GCCcore/7.3.0``
``4.1.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_testpath_877:

*testpath*
++++++++++

Test utilities for code working with files and commands

*homepage*: https://github.com/jupyter/testpath

=======    ==================    ===============================
version    versionsuffix         toolchain                      
=======    ==================    ===============================
``0.3``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``0.3``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``0.3``    ``-Python-3.5.1``     ``foss/2016a``                 
``0.3``    ``-Python-3.5.2``     ``intel/2016b``                
=======    ==================    ===============================


.. _list_software_TetGen_583:

*TetGen*
++++++++

A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator 

*homepage*: http://wias-berlin.de/software/index.jsp?id=TetGen

=========    =================
version      toolchain        
=========    =================
``1.5.0``    ``GCCcore/6.4.0``
=========    =================


.. _list_software_TEToolkit_895:

*TEToolkit*
+++++++++++

Tools for estimating differential enrichment of Transposable  Elements and other highly repetitive regions

*homepage*: http://hammelllab.labsites.cshl.edu/software/#TEToolkit

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``1.5.1``    ``-Python-2.7.11``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_texinfo_765:

*texinfo*
+++++++++

Texinfo is the official documentation format of the GNU project.

*homepage*: https://www.gnu.org/software/texinfo/

=========    ====================================
version      toolchain                           
=========    ====================================
``4.13a``    ``system``                          
``5.2``      ``GCC/4.8.2``                       
``6.4``      ``GCCcore/5.4.0``                   
``6.5``      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``6.6``      ``GCCcore/8.2.0``                   
=========    ====================================


.. _list_software_Text-CSV_702:

*Text-CSV*
++++++++++

Text-CSV parser

*homepage*: https://metacpan.org/pod/Text::CSV

========    ================    ==============
version     versionsuffix       toolchain     
========    ================    ==============
``1.33``    ``-Perl-5.22.1``    ``foss/2016a``
========    ================    ==============


.. _list_software_Theano_607:

*Theano*
++++++++

Theano is a Python library that allows you to define, optimize, and evaluate mathematical expressions involving multi-dimensional arrays efficiently.

*homepage*: https://deeplearning.net/software/theano

=========    ==================    =======================================
version      versionsuffix         toolchain                              
=========    ==================    =======================================
``0.8.2``    ``-Python-2.7.11``    ``foss/2016a``                         
``0.8.2``    ``-Python-3.5.1``     ``foss/2016a``                         
``0.8.2``    ``-Python-3.5.2``     ``intel/2016b``                        
``0.9.0``    ``-Python-2.7.13``    ``intel/2017a``                        
``0.9.0``    ``-Python-3.6.1``     ``intel/2017a``                        
``1.0.0``    ``-Python-2.7.14``    ``intel/2017b``                        
``1.0.0``    ``-Python-3.6.3``     ``intel/2017b``                        
``1.0.1``    ``-Python-3.6.3``     ``foss/2017b``, ``intel/2017b``        
``1.0.2``    ``-Python-2.7.14``    ``fosscuda/2017b``, ``intelcuda/2017b``
``1.0.2``    ``-Python-2.7.15``    ``fosscuda/2018b``                     
``1.0.2``    ``-Python-3.6.3``     ``fosscuda/2017b``, ``intelcuda/2017b``
``1.0.2``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``        
``1.0.3``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``     
``1.0.4``                          ``foss/2019a``, ``fosscuda/2019a``     
``1.0.4``    ``-Python-3.6.4``     ``foss/2018a``                         
``1.0.4``    ``-Python-3.6.6``     ``fosscuda/2018b``                     
=========    ==================    =======================================


.. _list_software_THetA_438:

*THetA*
+++++++

Tumor Heterogeneity Analysis (THetA) and THetA2 are algorithms  that estimate the tumor purity and clonal/subclonal copy number aberrations  directly from high-throughput DNA sequencing data.

*homepage*: https://github.com/raphael-group/THetA

=======    ==================    ==============
version    versionsuffix         toolchain     
=======    ==================    ==============
``0.7``    ``-Python-2.7.15``    ``foss/2018b``
=======    ==================    ==============


.. _list_software_thurstonianIRT_1454:

*thurstonianIRT*
++++++++++++++++

Fit Thurstonian IRT models in R using Stan, lavaan, or Mplus

*homepage*: https://github.com/paul-buerkner/thurstonianIRT

=========    =============    ==============
version      versionsuffix    toolchain     
=========    =============    ==============
``0.9.0``    ``-R-3.6.0``     ``foss/2019a``
=========    =============    ==============


.. _list_software_TiCCutils_884:

*TiCCutils*
+++++++++++

TiCC utils is a collection of generic C++ software which is used in a lot of programs produced at  Tilburg centre for Cognition and Communication (TiCC) at Tilburg University and  Centre for Dutch Language and Speech at University of Antwerp.

*homepage*: https://github.com/LanguageMachines/ticcutils

========    ================================
version     toolchain                       
========    ================================
``0.11``    ``foss/2016a``                  
``0.21``    ``iimpi/2019a``, ``intel/2018b``
========    ================================


.. _list_software_Tika_393:

*Tika*
++++++

The Apache Tika toolkit detects and extracts metadata and text from over a thousand different  file types (such as PPT, XLS, and PDF).

*homepage*: https://tika.apache.org/

========    ==========
version     toolchain 
========    ==========
``1.16``    ``system``
========    ==========


.. _list_software_TiMBL_408:

*TiMBL*
+++++++

TiMBL (Tilburg Memory Based Learner)  is an open source software package implementing several memory-based learning algorithms,  among which IB1-IG, an implementation of k-nearest neighbor classification with feature weighting suitable for  symbolic feature spaces, and IGTree, a decision-tree approximation of IB1-IG. All implemented algorithms have in  common that they store some representation of the training set explicitly in memory. During testing, new cases are  classified by extrapolation from the most similar stored cases.

*homepage*: http://ilk.uvt.nl/timbl/

==========    ================================
version       toolchain                       
==========    ================================
``6.4.7``     ``foss/2016a``                  
``6.4.13``    ``iimpi/2019a``, ``intel/2018b``
==========    ================================


.. _list_software_time_431:

*time*
++++++

The `time' command runs another program, then displays information about the resources used by that  program, collected by the system while the program was running.

*homepage*: http://www.gnu.org/software/time/

=======    ==========
version    toolchain 
=======    ==========
``1.7``    ``system``
=======    ==========


.. _list_software_TINKER_461:

*TINKER*
++++++++

The TINKER molecular modeling software is a complete and general package for molecular mechanics  and dynamics, with some special features for biopolymers.

*homepage*: http://dasher.wustl.edu/tinker

=========    ==============
version      toolchain     
=========    ==============
``8.6.1``    ``foss/2018b``
=========    ==============


.. _list_software_Tk_191:

*Tk*
++++

Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for  building a graphical user interface (GUI) in many different programming languages.

*homepage*: https://www.tcl.tk/

=========    =================    ================================================================================================================================================
version      versionsuffix        toolchain                                                                                                                                       
=========    =================    ================================================================================================================================================
``8.6.3``    ``-no-X11``          ``GCC/4.8.4``, ``GCC/4.9.2``                                                                                                                    
``8.6.4``                         ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``                                                                                             
``8.6.4``    ``-libX11-1.6.3``    ``intel/2016a``                                                                                                                                 
``8.6.4``    ``-no-X11``          ``GCC/4.9.3-2.25``, ``GNU/4.9.3-2.25``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``                            
``8.6.5``                         ``GCC/5.4.0-2.26``, ``GCCcore/6.3.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``iccifort/2016.3.210-GCC-5.4.0-2.26``, ``intel/2016b``
``8.6.6``                         ``foss/2017a``, ``intel/2017a``                                                                                                                 
``8.6.7``                         ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``                                                                        
``8.6.8``                         ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``foss/2018a``, ``iomkl/2018a``                                                                           
``8.6.9``                         ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                                            
=========    =================    ================================================================================================================================================


.. _list_software_Tkinter_737:

*Tkinter*
+++++++++

Tkinter module, built with the Python buildsystem

*homepage*: https://python.org/

==========    ==================    =========================================================================================================
version       versionsuffix         toolchain                                                                                                
==========    ==================    =========================================================================================================
``2.7.13``    ``-Python-2.7.13``    ``foss/2017a``, ``intel/2017a``                                                                          
``2.7.14``    ``-Python-2.7.14``    ``foss/2017b``, ``foss/2018a``, ``fosscuda/2017b``, ``intel/2017b``, ``intel/2018a``, ``intelcuda/2017b``
``2.7.15``                          ``GCCcore/8.2.0``                                                                                        
``2.7.15``    ``-Python-2.7.15``    ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``                                                      
``3.6.1``     ``-Python-3.6.1``     ``foss/2017a``, ``intel/2017a``                                                                          
``3.6.2``     ``-Python-3.6.2``     ``foss/2017b``                                                                                           
``3.6.3``     ``-Python-3.6.3``     ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``                                 
``3.6.4``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``, ``iomkl/2018.02``, ``iomkl/2018a``                                      
``3.6.6``     ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018b``, ``iomkl/2018b``                                     
``3.7.2``                           ``GCCcore/8.2.0``                                                                                        
``3.7.4``                           ``GCCcore/8.3.0``                                                                                        
==========    ==================    =========================================================================================================


.. _list_software_TM-align_729:

*TM-align*
++++++++++

This package unifies protein structure alignment and RNA structure alignment  into the standard TM-align program for single chain structure alignment, MM-align program for  multi-chain structure alignment, and TM-score program for sequence dependent structure superposition.

*homepage*: https://zhanglab.ccmb.med.umich.edu/RNA-align/TMalign/

============    ===============================
version         toolchain                      
============    ===============================
``20180426``    ``foss/2018b``, ``intel/2019a``
============    ===============================


.. _list_software_tmux_462:

*tmux*
++++++

tmux is a terminal multiplexer. It lets you switch easily  between several programs in one terminal, detach them (they keep  running in the background) and reattach them to a different terminal.

*homepage*: http://tmux.sourceforge.net/

========    ==============================
version     toolchain                     
========    ==============================
``1.9a``    ``GCC/4.9.2``                 
``2.2``     ``GCCcore/4.9.3``             
``2.3``     ``GCC/5.4.0-2.26``, ``system``
========    ==============================


.. _list_software_ToFu_382:

*ToFu*
++++++

Tomography for Fusion.

*homepage*: https://github.com/ToFuProject/tofu

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``1.3.17``    ``-Python-2.7.14``    ``foss/2018a``                 
``1.3.17``    ``-Python-3.6.4``     ``foss/2018a``                 
``1.4.0``     ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``
``1.4.0``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``
``1.4.1``     ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``
==========    ==================    ===============================


.. _list_software_TopHat_592:

*TopHat*
++++++++

TopHat is a fast splice junction mapper for RNA-Seq reads.

*homepage*: http://ccb.jhu.edu/software/tophat/

=========    ==================    ================================================================================
version      versionsuffix         toolchain                                                                       
=========    ==================    ================================================================================
``2.1.0``    ``-Python-2.7.14``    ``intel/2018a``                                                                 
``2.1.1``                          ``foss/2016a``, ``foss/2016b``, ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``2.1.2``                          ``foss/2018b``                                                                  
=========    ==================    ================================================================================


.. _list_software_torchvision_1208:

*torchvision*
+++++++++++++

Datasets, Transforms and Models specific to Computer Vision

*homepage*: https://github.com/pytorch/vision

=========    =============================    ==================================
version      versionsuffix                    toolchain                         
=========    =============================    ==================================
``0.2.0``    ``-Python-3.6.4``                ``intel/2018a``                   
``0.2.0``    ``-Python-3.6.4-CUDA-9.1.85``    ``foss/2018a``                    
``0.2.2``    ``-Python-3.6.6``                ``foss/2018b``, ``fosscuda/2018b``
``0.3.0``    ``-Python-3.7.2``                ``foss/2019a``                    
=========    =============================    ==================================


.. _list_software_TotalView_927:

*TotalView*
+++++++++++

TotalView is a GUI-based source code defect analysis tool that gives you unprecedented  control over processes and thread execution and visibility into program state and variables. It allows  you to debug one or many processes and/or threads in a single window with complete control over program  execution. This allows you to set breakpoints, stepping line by line through the code on a single thread,  or with coordinated groups of processes or threads, and run or halt arbitrary sets of processes or threads.  You can reproduce and troubleshoot difficult problems that can occur in concurrent programs that take  advantage of threads, OpenMP, MPI, GPUs or coprocessors.

*homepage*: http://www.roguewave.com/products/totalview.aspx

============    =================    ==========
version         versionsuffix        toolchain 
============    =================    ==========
``8.11.0-0``    ``-linux-x86-64``    ``system``
``8.11.0-2``    ``-linux-x86-64``    ``system``
``8.12.0-0``    ``-linux-x86-64``    ``system``
============    =================    ==========


.. _list_software_tqdm_438:

*tqdm*
++++++

A fast, extensible progress bar for Python and CLI

*homepage*: https://github.com/tqdm/tqdm

==========    ==================    =================
version       versionsuffix         toolchain        
==========    ==================    =================
``4.24.0``    ``-Python-2.7.15``    ``foss/2018b``   
``4.24.0``    ``-Python-3.5.1``     ``foss/2016a``   
``4.29.0``    ``-Python-3.6.4``     ``intel/2018a``  
``4.32.1``                          ``GCCcore/8.2.0``
==========    ==================    =================


.. _list_software_Tracer_609:

*Tracer*
++++++++

Tracer is a graphical tool for visualization and diagnostics of MCMC output.

*homepage*: https://github.com/beast-dev/tracer

=========    =================
version      toolchain        
=========    =================
``1.7.1``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_TransDecoder_1214:

*TransDecoder*
++++++++++++++

TransDecoder identifies candidate coding regions within transcript sequences,  such as those generated by de novo RNA-Seq transcript assembly using Trinity,  or constructed based on RNA-Seq alignments to the genome using  Tophat and Cufflinks.

*homepage*: https://github.com/TransDecoder/TransDecoder/wiki

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``2.1.0``    ``-Perl-5.24.1``    ``intel/2017a``
=========    ================    ===============


.. _list_software_Transrate_948:

*Transrate*
+++++++++++

Transrate is software for de-novo transcriptome assembly quality analysis.  It examines your assembly in detail and compares it to experimental evidence  such as the sequencing reads, reporting quality scores for contigs and  assemblies. This allows you to choose between assemblers and parameters,  filter out the bad contigs from an assembly, and help decide when  to stop trying to improve the assembly.

*homepage*: http://hibberdlab.com/transrate/

=========    ==========
version      toolchain 
=========    ==========
``1.0.3``    ``system``
=========    ==========


.. _list_software_travis_665:

*travis*
++++++++

Travis CI Client (CLI and Ruby library)

*homepage*: https://github.com/travis-ci/travis.rb

=========    ===============    ==========
version      versionsuffix      toolchain 
=========    ===============    ==========
``1.8.2``    ``-Ruby-2.3.1``    ``system``
``1.8.4``    ``-Ruby-2.3.3``    ``system``
=========    ===============    ==========


.. _list_software_TreeMix_702:

*TreeMix*
+++++++++

TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a  set of populations.

*homepage*: http://bitbucket.org/nygcresearch/treemix

========    ===============
version     toolchain      
========    ===============
``1.13``    ``intel/2018a``
========    ===============


.. _list_software_TRF_236:

*TRF*
+++++

Tandem repeats finder: a program to analyze DNA sequences. Legacy version.

*homepage*: https://tandem.bu.edu/trf/trf.html

========    =============    ==========
version     versionsuffix    toolchain 
========    =============    ==========
``4.09``    ``-linux64``     ``system``
========    =============    ==========


.. _list_software_Triangle_822:

*Triangle*
++++++++++

Triangle generates exact Delaunay triangulations, constrained Delaunay triangulations,  conforming Delaunay triangulations, Voronoi diagrams, and high-quality triangular meshes.  The latter can be generated with no small or large angles,  and are thus suitable for finite element analysis.

*homepage*: http://www.cs.cmu.edu/~quake/triangle.html

=======    ====================================================================================================
version    toolchain                                                                                           
=======    ====================================================================================================
``1.6``    ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2018a``, ``intel/2016b``, ``intel/2017a``, ``intel/2018a``
=======    ====================================================================================================


.. _list_software_Trilinos_852:

*Trilinos*
++++++++++

The Trilinos Project is an effort to develop algorithms and enabling technologies  within an object-oriented software framework for the solution of large-scale, complex multi-physics  engineering and scientific problems. A unique design feature of Trilinos is its focus on packages.

*homepage*: https://trilinos.org

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``12.12.1``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``
``12.12.1``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``
===========    ==================    ===============================


.. _list_software_Trim_Galore_1109:

*Trim_Galore*
+++++++++++++

Trim Galore is a wrapper around Cutadapt and FastQC to  consistently apply adapter and quality trimming to FastQ files, with extra  functionality for RRBS data.

*homepage*: https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.4.2``                          ``foss/2016b``                 
``0.4.4``                          ``foss/2016b``, ``intel/2017a``
``0.5.0``                          ``foss/2018b``                 
``0.5.0``    ``-Python-3.6.6``     ``intel/2018b``                
``0.6.0``    ``-Python-2.7.15``    ``foss/2018b``                 
``0.6.0``    ``-Python-3.6.6``     ``foss/2018b``                 
``0.6.1``    ``-Python-2.7.15``    ``foss/2018b``                 
``0.6.1``    ``-Python-3.6.6``     ``foss/2018b``                 
``0.6.2``    ``-Java-11``          ``GCCcore/8.2.0``              
``0.6.2``    ``-Python-2.7.15``    ``foss/2018b``                 
``0.6.2``    ``-Python-3.6.6``     ``foss/2018b``                 
=========    ==================    ===============================


.. _list_software_trimAl_617:

*trimAl*
++++++++

EVB, FEP and LIE simulator.

*homepage*: https://github.com/scapella/trimal

=========    ====================
version      toolchain           
=========    ====================
``1.4.1``    ``GCC/8.2.0-2.31.1``
=========    ====================


.. _list_software_Trimmomatic_1158:

*Trimmomatic*
+++++++++++++

Trimmomatic performs a variety of useful trimming tasks for illumina   paired-end and single ended data.The selection of trimming steps and their associated   parameters are supplied on the command line. 

*homepage*: http://www.usadellab.org/cms/?page=trimmomatic

========    ===================    ==========
version     versionsuffix          toolchain 
========    ===================    ==========
``0.32``    ``-Java-1.7.0_80``     ``system``
``0.36``    ``-Java-1.8.0_92``     ``system``
``0.38``    ``-Java-1.8``          ``system``
``0.38``    ``-Java-1.8.0_162``    ``system``
========    ===================    ==========


.. _list_software_Trinity_755:

*Trinity*
+++++++++

Trinity represents a novel method for the efficient and robust de novo reconstruction  of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,  Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.

*homepage*: https://trinityrnaseq.github.io

=========    ========================================
version      toolchain                               
=========    ========================================
``2.2.0``    ``foss/2016a``                          
``2.4.0``    ``foss/2017a``                          
``2.5.1``    ``intel/2017a``, ``intel/2017b``        
``2.6.6``    ``intel/2018a``                         
``2.8.4``    ``foss/2018b``                          
``2.8.5``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
=========    ========================================


.. _list_software_TRIQS_403:

*TRIQS*
+++++++

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems. 

*homepage*: https://triqs.github.io/triqs

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.2.0``    ``-Python-2.7.15``    ``foss/2019a``
``2.2.1``    ``-Python-2.7.15``    ``foss/2019a``
=========    ==================    ==============


.. _list_software_TRIQS-cthyb_986:

*TRIQS-cthyb*
+++++++++++++

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   cthyb = continuous-time hybridisation-expansion quantum Monte Carlo   The TRIQS-based hybridization-expansion solver allows to solve the  generic problem of a quantum impurity embedded in a conduction bath  for an arbitrary local interaction vertex. The “impurity” can be any  set of orbitals, on one or several atoms. 

*homepage*: https://triqs.github.io/cthyb/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.2.0``    ``-Python-2.7.15``    ``foss/2019a``
=========    ==================    ==============


.. _list_software_TRIQS-dft_tools_1422:

*TRIQS-dft_tools*
+++++++++++++++++

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   This TRIQS-based-based application is aimed at ab-initio calculations   for correlated materials, combining realistic DFT band-structure calculations   with the dynamical mean-field theory. Together with the necessary tools to   perform the DMFT self-consistency loop for realistic multi-band problems,   the package provides a full-fledged charge self-consistent interface to the   Wien2K package. In addition, if Wien2k is not available, it provides a generic  interface for one-shot DFT+DMFT calculations, where only the single-particle   Hamiltonian in orbital space has to be provided. 

*homepage*: https://triqs.github.io/dft_tools

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.2.0``    ``-Python-2.7.15``    ``foss/2019a``
=========    ==================    ==============


.. _list_software_TRIQS-tprf_892:

*TRIQS-tprf*
++++++++++++

TRIQS (Toolbox for Research on Interacting Quantum Systems) is a  scientific project providing a set of C++ and Python libraries to  develop new tools for the study of interacting quantum systems.   TPRF is a TRIQS-based two-particle response function tool box that  implements basic operations for higher order response functions such  as inversion, products, the random phase approximation, the bethe  salpeter equation (in the local vertex approximation), etc..   The aim is to provide efficient (C++/OpenMP/MPI) implementations of  the basic operations needed to compute the two-particle response in  the different two-particle channels (particle-hole, particle-particle). 

*homepage*: https://triqs.github.io/tprf

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.2.0``    ``-Python-2.7.15``    ``foss/2019a``
=========    ==================    ==============


.. _list_software_TRUST_418:

*TRUST*
+++++++

Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze  TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. 

*homepage*: https://bitbucket.org/liulab/trust

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.0.2``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_TurboVNC_755:

*TurboVNC*
++++++++++

TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide  peak performance for 3D and video workloads.

*homepage*: https://www.turbovnc.org

=========    =================
version      toolchain        
=========    =================
``2.2.3``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_TVB_236:

*TVB*
+++++

The Virtual Brain will deliver the first open simulation of the human brain  based on individual large-scale connectivity.

*homepage*: http://thevirtualbrain.org

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.4.1``    ``-Python-2.7.11``    ``intel/2016a``
``1.5``      ``-Python-2.7.11``    ``intel/2016a``
=========    ==================    ===============


.. _list_software_tvb-data_787:

*tvb-data*
++++++++++

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  Various demonstration data for use with TVB.

*homepage*: http://www.thevirtualbrain.org/

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``1.5``         ``-Python-2.7.11``    ``intel/2016a``
``20150915``    ``-Python-2.7.11``    ``intel/2016a``
============    ==================    ===============


.. _list_software_TVB-deps_709:

*TVB-deps*
++++++++++

Bundle of dependency Python packages for TVB (The Virtual Brain)

*homepage*: http://thevirtualbrain.org

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20160618``    ``-Python-2.7.11``    ``intel/2016a``
============    ==================    ===============


.. _list_software_tvb-framework_1351:

*tvb-framework*
+++++++++++++++

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.

*homepage*: http://www.thevirtualbrain.org/

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``1.5``         ``-Python-2.7.11``    ``intel/2016a``
``20150921``    ``-Python-2.7.11``    ``intel/2016a``
============    ==================    ===============


.. _list_software_tvb-library_1134:

*tvb-library*
+++++++++++++

The Virtual Brain Project (TVB Project) has the purpose of offering some modern tools to the  Neurosciences community, for computing, simulating and analyzing functional and structural data of human brains.  TVB Scientific Library is the most important scientific contribution of TVB Project.

*homepage*: http://www.thevirtualbrain.org/

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``1.5``         ``-Python-2.7.11``    ``intel/2016a``
``20150922``    ``-Python-2.7.11``    ``intel/2016a``
============    ==================    ===============


.. _list_software_letter_u:

*U*
---


:ref:`list_software_UCLUST_480` - :ref:`list_software_UCX_240` - :ref:`list_software_udocker_749` - :ref:`list_software_UDUNITS_556` - :ref:`list_software_UFL_231` - :ref:`list_software_UMI-tools_841` - :ref:`list_software_umis_446` - :ref:`list_software_UNAFold_617` - :ref:`list_software_Unicycler_942` - :ref:`list_software_unixODBC_732` - :ref:`list_software_unrar_552` - :ref:`list_software_UnZip_502` - :ref:`list_software_USEARCH_523` - :ref:`list_software_USPEX_405` - :ref:`list_software_utf8proc_827` - :ref:`list_software_util-linux_1051`


.. _list_software_UCLUST_480:

*UCLUST*
++++++++

UCLUST: Extreme high-speed sequence clustering, alignment and database search.

*homepage*: http://www.drive5.com/usearch/index.html

===========    ===============    ==========
version        versionsuffix      toolchain 
===========    ===============    ==========
``1.2.22q``    ``-i86linux64``    ``system``
===========    ===============    ==========


.. _list_software_UCX_240:

*UCX*
+++++

Unified Communication X An open-source production grade communication framework for data centric and high-performance applications 

*homepage*: http://www.openucx.org/

============    =============    =======================================================
version         versionsuffix    toolchain                                              
============    =============    =======================================================
``1.3.1``                        ``GCCcore/6.4.0``, ``GCCcore/7.3.0``                   
``1.5.0``                        ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
``1.5.0rc1``    ``-hpcx``        ``GCCcore/8.2.0``                                      
``1.5.1``                        ``GCCcore/8.2.0``                                      
``1.6.1``                        ``GCCcore/8.3.0``                                      
============    =============    =======================================================


.. _list_software_udocker_749:

*udocker*
+++++++++

A basic user tool to execute simple docker containers in batch  or interactive systems without root privileges.

*homepage*: https://github.com/indigo-dc/udocker

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``1.1.1``    ``-Python-2.7.14``    ``foss/2017b`` 
``1.1.3``    ``-Python-2.7.14``    ``intel/2018a``
=========    ==================    ===============


.. _list_software_UDUNITS_556:

*UDUNITS*
+++++++++

UDUNITS supports conversion of unit specifications between formatted and binary forms,  arithmetic manipulation of units, and conversion of values between compatible scales of measurement.

*homepage*: http://www.unidata.ucar.edu/software/udunits/

==========    =====================================================================================================================
version       toolchain                                                                                                            
==========    =====================================================================================================================
``2.2.20``    ``foss/2016a``, ``intel/2016b``                                                                                      
``2.2.24``    ``intel/2017a``                                                                                                      
``2.2.25``    ``foss/2017b``, ``intel/2017b``                                                                                      
``2.2.26``    ``GCCcore/8.2.0``, ``foss/2018a``, ``foss/2018b``, ``intel/2017b``, ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
==========    =====================================================================================================================


.. _list_software_UFL_231:

*UFL*
+++++

The Unified Form Language (UFL) is a domain specific language   for declaration of finite element discretizations of variational forms.   More precisely, it defines a flexible interface for choosing finite element   spaces and defining expressions for weak forms in a notation close to   mathematical notation.

*homepage*: https://bitbucket.org/fenics-project/ufl

============    =================    ==============
version         versionsuffix        toolchain     
============    =================    ==============
``2018.1.0``    ``-Python-3.6.4``    ``foss/2018a``
============    =================    ==============


.. _list_software_UMI-tools_841:

*UMI-tools*
+++++++++++

Tools for handling Unique Molecular Identifiers in NGS data sets

*homepage*: https://umi-tools.readthedocs.io

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``1.0.0``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_umis_446:

*umis*
++++++

Package for estimating UMI counts in Transcript Tag Counting data.

*homepage*: https://github.com/vals/umis

=========    ===============
version      toolchain      
=========    ===============
``1.0.3``    ``intel/2019a``
=========    ===============


.. _list_software_UNAFold_617:

*UNAFold*
+++++++++

The UNAFold package contains several programs for performing energy minimization and partition  function calculations on nucleic acid sequences.

*homepage*: http://mfold.rna.albany.edu/?q=DINAMelt/software

=======    ================    ===============
version    versionsuffix       toolchain      
=======    ================    ===============
``3.8``    ``-Perl-5.24.1``    ``intel/2017a``
=======    ================    ===============


.. _list_software_Unicycler_942:

*Unicycler*
+++++++++++

Unicycler is an assembly pipeline for bacterial genomes. It can assemble Illumina-only read sets  where it functions as a SPAdes-optimiser. It can also assembly long-read-only sets (PacBio or Nanopore)  where it runs a miniasm+Racon pipeline. 

*homepage*: 

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``0.4.7``    ``-Python-3.7.2``    ``foss/2019a``
=========    =================    ==============


.. _list_software_unixODBC_732:

*unixODBC*
++++++++++

unixODBC provides a uniform interface between application and database driver

*homepage*: https://www.unixodbc.org

=========    ==============
version      toolchain     
=========    ==============
``2.3.7``    ``foss/2018b``
=========    ==============


.. _list_software_unrar_552:

*unrar*
+++++++

RAR is a powerful archive manager.

*homepage*: https://www.rarlab.com/

=========    =================
version      toolchain        
=========    =================
``5.6.1``    ``GCCcore/7.3.0``
``5.7.3``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_UnZip_502:

*UnZip*
+++++++

UnZip is an extraction utility for archives compressed in .zip format (also called "zipfiles"). Although highly compatible both with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's own Zip program, our primary objectives have been portability and non-MSDOS functionality.

*homepage*: http://www.info-zip.org/UnZip.html

=======    =======================================================
version    toolchain                                              
=======    =======================================================
``6.0``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``
=======    =======================================================


.. _list_software_USEARCH_523:

*USEARCH*
+++++++++

USEARCH is a unique sequence analysis tool which offers search and clustering algorithms that are  often orders of magnitude faster than BLAST.

*homepage*: http://www.drive5.com/usearch/index.html

===================    ===============    ==========
version                versionsuffix      toolchain 
===================    ===============    ==========
``5.2.236``            ``-i86linux32``    ``system``
``5.2.236-6.1.544``    ``-i86linux32``    ``system``
``6.1.544``            ``-i86linux32``    ``system``
===================    ===============    ==========


.. _list_software_USPEX_405:

*USPEX*
+++++++

USPEX (Universal Structure Predictor: Evolutionary Xtallography...  and in Russian "uspekh" means "success" - owing to the high success rate and many  useful results produced by this method) is a method developed by the Oganov  laboratory since 2004. The problem of crystal structure prediction is very old and  does, in fact, constitute the central problem of theoretical crystal chemistry.  USPEX can also be used for finding low-energy metastable phases, as well as stable  structures of nanoparticles, surface reconstructions, molecular packings in organic  crystals, and for searching for materials with desired physical (mechanical,  electronic) properties. The USPEX code is based on an efficient evolutionary  algorithm developed by A.R. Oganov's group, but also has options for using  alternative methods (random sampling, metadynamics, corrected particle swarm  optimization algorithms). USPEX is interfaced with many ab initio codes, such as  VASP, SIESTA, GULP, Quantum Espresso, CP2K, CASTEP, LAMMPS, and so on. 

*homepage*: http://uspex-team.org/uspex/overview

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``9.4.4``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_utf8proc_827:

*utf8proc*
++++++++++

utf8proc is a small, clean C library that provides Unicode normalization, case-folding,  and other operations for data in the UTF-8 encoding.

*homepage*: https://github.com/JuliaStrings/utf8proc

=========    =================
version      toolchain        
=========    =================
``2.2.0``    ``GCCcore/6.4.0``
``2.3.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_util-linux_1051:

*util-linux*
++++++++++++

Set of Linux utilities

*homepage*: https://www.kernel.org/pub/linux/utils/util-linux

==========    ====================================
version       toolchain                           
==========    ====================================
``2.27.1``    ``foss/2016a``, ``intel/2016a``     
``2.28``      ``foss/2016a``, ``intel/2016a``     
``2.28.1``    ``intel/2016b``                     
``2.29``      ``foss/2016b``, ``intel/2016b``     
``2.29.2``    ``GCCcore/6.3.0``, ``intel/2017a``  
``2.30``      ``GCCcore/6.4.0``                   
``2.30.1``    ``GCCcore/6.3.0``, ``foss/2017a``   
``2.31``      ``GCCcore/6.4.0``                   
``2.31.1``    ``GCCcore/6.4.0``                   
``2.32``      ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
``2.32.1``    ``GCCcore/7.3.0``                   
``2.33``      ``GCCcore/8.2.0``                   
``2.34``      ``GCCcore/8.3.0``                   
==========    ====================================


.. _list_software_letter_v:

*V*
---


:ref:`list_software_V8_142` - :ref:`list_software_Valgrind_823` - :ref:`list_software_Vampir_623` - :ref:`list_software_VarDict_685` - :ref:`list_software_VariantMetaCaller_1711` - :ref:`list_software_VarScan_686` - :ref:`list_software_vartools_890` - :ref:`list_software_VASP_314` - :ref:`list_software_VAtools_712` - :ref:`list_software_VCF-kit_596` - :ref:`list_software_vcfnp_541` - :ref:`list_software_VCFtools_784` - :ref:`list_software_VEGAS_374` - :ref:`list_software_Velvet_630` - :ref:`list_software_VEP_235` - :ref:`list_software_verifyBamID_1074` - :ref:`list_software_VERSE_389` - :ref:`list_software_ViennaRNA_834` - :ref:`list_software_Vim_300` - :ref:`list_software_VirSorter_944` - :ref:`list_software_VirtualGL_890` - :ref:`list_software_VMD_231` - :ref:`list_software_Voro++_508` - :ref:`list_software_vsc-base_788` - :ref:`list_software_vsc-install_1136` - :ref:`list_software_vsc-mympirun_1274` - :ref:`list_software_vsc-mympirun-scoop_1867` - :ref:`list_software_vsc-processcontrol_1913` - :ref:`list_software_VSEARCH_524` - :ref:`list_software_vt_234` - :ref:`list_software_VTK_245` - :ref:`list_software_VTune_498` - :ref:`list_software_VV_172` - :ref:`list_software_VXL_250`


.. _list_software_V8_142:

*V8*
++++

R interface to Google's open source JavaScript engine

*homepage*: https://cran.r-project.org/web/packages/V8/

=======    =============    ==============
version    versionsuffix    toolchain     
=======    =============    ==============
``2.2``    ``-R-3.5.1``     ``foss/2018b``
=======    =============    ==============


.. _list_software_Valgrind_823:

*Valgrind*
++++++++++

Valgrind: Debugging and profiling tools

*homepage*: http://valgrind.org

==========    =================================================================================
version       toolchain                                                                        
==========    =================================================================================
``3.11.0``    ``foss/2016a``, ``intel/2016a``                                                  
``3.13.0``    ``foss/2017b``, ``foss/2018a``, ``intel/2017a``, ``intel/2017b``, ``intel/2018a``
``3.14.0``    ``foss/2018b``                                                                   
==========    =================================================================================


.. _list_software_Vampir_623:

*Vampir*
++++++++

The Vampir software tool provides an easy-to-use framework that enables  developers to quickly display and analyze arbitrary program behavior at any level of detail.  The tool suite implements optimized event analysis algorithms and customizable displays that  enable fast and interactive rendering of very complex performance monitoring data.

*homepage*: http://www.vampir.eu

=========    =============    ==========
version      versionsuffix    toolchain 
=========    =============    ==========
``8.4.1``                     ``system``
``8.4.1``    ``-demo``        ``system``
=========    =============    ==========


.. _list_software_VarDict_685:

*VarDict*
+++++++++

VarDict is an ultra sensitive variant caller for both single  and paired sample variant calling from BAM files.

*homepage*: https://github.com/AstraZeneca-NGS/VarDict

=========    ================    ==============
version      versionsuffix       toolchain     
=========    ================    ==============
``1.5.7``    ``-Perl-5.28.0``    ``foss/2018b``
=========    ================    ==============


.. _list_software_VariantMetaCaller_1711:

*VariantMetaCaller*
+++++++++++++++++++

VariantMetaCaller automatically integrates variant calling pipelines into a better performing  overall model that also predicts accurate variant probabilities.

*homepage*: http://bioinformatics.mit.bme.hu/VariantMetaCaller/

=======    ===============
version    toolchain      
=======    ===============
``1.0``    ``intel/2017a``
=======    ===============


.. _list_software_VarScan_686:

*VarScan*
+++++++++

Variant calling and somatic mutation/CNV detection for next-generation sequencing data

*homepage*: https://github.com/dkoboldt/varscan

=========    ==================    ==========
version      versionsuffix         toolchain 
=========    ==================    ==========
``2.3.6``    ``-Java-1.7.0_80``    ``system``
``2.4.1``    ``-Java-1.7.0_80``    ``system``
=========    ==================    ==========


.. _list_software_vartools_890:

*vartools*
++++++++++

Command line utility that provides tools for processing   and analyzing astronomical time series data.

*homepage*: https://www.astro.princeton.edu/~jhartman/vartools.html

========    ===============================
version     toolchain                      
========    ===============================
``1.35``    ``foss/2016b``, ``intel/2016b``
========    ===============================


.. _list_software_VASP_314:

*VASP*
++++++

The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.

*homepage*: http://www.vasp.at

=========    =========================
version      toolchain                
=========    =========================
``5.4.1``    ``intel/2016.02-GCC-4.9``
=========    =========================


.. _list_software_VAtools_712:

*VAtools*
+++++++++

VAtools is a python package that includes several tools to annotate VCF files  with data from other tools. 

*homepage*: https://vatools.readthedocs.io/en/latest/

=========    =================    ==============
version      versionsuffix        toolchain     
=========    =================    ==============
``3.0.1``    ``-Python-3.6.6``    ``foss/2018b``
=========    =================    ==============


.. _list_software_VCF-kit_596:

*VCF-kit*
+++++++++

VCF-kit is a command-line based collection of utilities for performing analysis on  Variant Call Format (VCF) files.

*homepage*: https://github.com/AndersenLab/VCF-kit

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``0.1.6``    ``-Python-2.7.15``    ``intel/2018b``
=========    ==================    ===============


.. _list_software_vcfnp_541:

*vcfnp*
+++++++

Load data from a VCF (variant call format) file into numpy arrays,   and (optionally) from there into an HDF5 file.

*homepage*: https://github.com/alimanfoo/vcfnp

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.3.0``    ``-Python-2.7.11``    ``foss/2016a``
=========    ==================    ==============


.. _list_software_VCFtools_784:

*VCFtools*
++++++++++

The aim of VCFtools is to provide   easily accessible methods for working with complex   genetic variation data in the form of VCF files.

*homepage*: https://vcftools.github.io

==========    ================    ===============================
version       versionsuffix       toolchain                      
==========    ================    ===============================
``0.1.14``    ``-Perl-5.22.1``    ``foss/2016a``, ``intel/2016a``
``0.1.15``    ``-Perl-5.24.0``    ``foss/2016b``                 
``0.1.15``    ``-Perl-5.26.0``    ``foss/2017b``, ``intel/2017b``
``0.1.15``    ``-Perl-5.26.1``    ``foss/2018a``                 
``0.1.16``    ``-Perl-5.28.0``    ``foss/2018b``, ``intel/2018b``
==========    ================    ===============================


.. _list_software_VEGAS_374:

*VEGAS*
+++++++

VEGAS (Versatile Gene-based Association Study) is  a free program for performing gene-based tests for association using  the results from genetic association studies 

*homepage*: http://gump.qimr.edu.au/VEGAS/

==========    ==========
version       toolchain 
==========    ==========
``0.8.27``    ``system``
==========    ==========


.. _list_software_Velvet_630:

*Velvet*
++++++++

Sequence assembler for very short reads

*homepage*: http://www.ebi.ac.uk/~zerbino/velvet/

==========    ================    ==============================
version       versionsuffix       toolchain                     
==========    ================    ==============================
``1.2.10``    ``-mt-kmer_191``    ``foss/2018a``, ``foss/2018b``
``1.2.10``    ``-mt-kmer_37``     ``intel/2017a``               
==========    ================    ==============================


.. _list_software_VEP_235:

*VEP*
+++++

Variant Effect Predictor (VEP) determines the effect of your  variants (SNPs, insertions, deletions, CNVs or structural variants) on genes,  transcripts, and protein sequence, as well as regulatory regions.  Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently  used routines in VEP.

*homepage*: https://www.ensembl.org/info/docs/tools/vep

========    ================    ===============================
version     versionsuffix       toolchain                      
========    ================    ===============================
``93.4``    ``-Perl-5.26.1``    ``intel/2018a``                
``94.0``    ``-Perl-5.28.0``    ``foss/2018b``                 
``94.5``    ``-Perl-5.26.0``    ``foss/2017b``, ``intel/2017b``
``95.0``    ``-Perl-5.28.0``    ``foss/2018b``                 
``96.0``    ``-Perl-5.28.1``    ``foss/2019a``                 
========    ================    ===============================


.. _list_software_verifyBamID_1074:

*verifyBamID*
+++++++++++++

verifyBamID is a software that verifies whether the reads in particular file match previously known   genotypes for an individual (or group of individuals), and checks whether the reads are contaminated as a mixture of   two samples. verifyBamID can detect sample contamination and swaps when external genotypes are available. When   external genotypes are not available, verifyBamID still robustly detects sample swaps. 

*homepage*: http://csg.sph.umich.edu/kang/verifyBamID/index.html

=========    ==============
version      toolchain     
=========    ==============
``1.1.3``    ``foss/2016a``
=========    ==============


.. _list_software_VERSE_389:

*VERSE*
+++++++

A versatile and efficient RNA-Seq read counting tool

*homepage*: https://github.com/qinzhu/VERSE

=========    ==============
version      toolchain     
=========    ==============
``0.1.5``    ``foss/2016b``
=========    ==============


.. _list_software_ViennaRNA_834:

*ViennaRNA*
+++++++++++

The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures.

*homepage*: https://www.tbi.univie.ac.at/RNA/

==========    ==================    ===============================
version       versionsuffix         toolchain                      
==========    ==================    ===============================
``2.2.3``                           ``intel/2016b``                
``2.3.4``                           ``foss/2016b``                 
``2.3.5``                           ``intel/2017a``                
``2.4.10``    ``-Python-2.7.15``    ``foss/2018b``, ``intel/2018b``
``2.4.11``    ``-Python-3.6.6``     ``foss/2018b``                 
``2.4.14``    ``-Python-3.6.6``     ``foss/2018b``                 
==========    ==================    ===============================


.. _list_software_Vim_300:

*Vim*
+++++

Vim is an advanced text editor that seeks to provide the power   of the de-facto Unix editor 'Vi', with a more complete feature set. 

*homepage*: http://www.vim.org

============    ==================    =================
version         versionsuffix         toolchain        
============    ==================    =================
``8.0``         ``-Python-2.7.11``    ``foss/2016a``   
``8.1.0483``    ``-Python-2.7.15``    ``foss/2018b``   
``8.1.1209``    ``-Python-3.7.2``     ``GCCcore/8.2.0``
============    ==================    =================


.. _list_software_VirSorter_944:

*VirSorter*
+++++++++++

VirSorter: mining viral signal from microbial genomic data.

*homepage*: https://github.com/simroux/VirSorter

============    ================    ==============
version         versionsuffix       toolchain     
============    ================    ==============
``20160601``    ``-Perl-5.22.1``    ``foss/2016a``
============    ================    ==============


.. _list_software_VirtualGL_890:

*VirtualGL*
+++++++++++

VirtualGL is an open source toolkit that gives any Linux or  Unix remote display software the ability to run OpenGL applications with full  hardware acceleration.

*homepage*: https://virtualgl.org/

=========    ==============
version      toolchain     
=========    ==============
``2.6.1``    ``foss/2018b``
=========    ==============


.. _list_software_VMD_231:

*VMD*
+++++

VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular  systems using 3-D graphics and built-in scripting.

*homepage*: http://www.ks.uiuc.edu/Research/vmd

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``1.9.3``    ``-Python-2.7.12``    ``intel/2016b``                
``1.9.3``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2018a``
``1.9.3``    ``-Python-2.7.15``    ``intel/2018b``                
=========    ==================    ===============================


.. _list_software_Voro++_508:

*Voro++*
++++++++

Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation. A distinguishing feature of the Voro++ library is that it carries out cell-based calculations, computing the Voronoi cell for each particle individually. It is particularly well-suited for applications that rely on cell-based statistics, where features of Voronoi cells (eg. volume, centroid, number of faces) can be used to analyze a system of particles.

*homepage*: http://math.lbl.gov/voro++/

=========    ==================================================
version      toolchain                                         
=========    ==================================================
``0.4.6``    ``GCCcore/6.4.0``, ``foss/2016a``, ``intel/2016a``
=========    ==================================================


.. _list_software_vsc-base_788:

*vsc-base*
++++++++++

Basic Python libraries used by UGent's HPC group

*homepage*: http://hpcugent.github.com/vsc-base/

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``1.6.9``                           ``system``     
``1.7.3``                           ``system``     
``2.0.4``                           ``system``     
``2.1.2``                           ``system``     
``2.4.2``                           ``system``     
``2.4.17``    ``-Python-2.7.11``    ``intel/2016a``
``2.5.1``                           ``system``     
``2.5.1``     ``-Python-2.7.11``    ``intel/2016a``
``2.5.1``     ``-Python-2.7.12``    ``intel/2016b``
``2.5.8``                           ``system``     
``2.8.0``                           ``system``     
``2.8.1``                           ``system``     
``2.8.3``                           ``system``     
==========    ==================    ===============


.. _list_software_vsc-install_1136:

*vsc-install*
+++++++++++++

vsc-install provides shared setuptools functions and classes for Python libraries  developed by UGent's HPC group

*homepage*: https://pypi.python.org/pypi/vsc-install

===========    ==================    ===============
version        versionsuffix         toolchain      
===========    ==================    ===============
``0.9.18``     ``-Python-2.7.11``    ``intel/2016a``
``0.10.6``                           ``system``     
``0.10.6``     ``-Python-2.7.11``    ``intel/2016a``
``0.10.11``    ``-Python-2.7.11``    ``intel/2016a``
``0.10.11``    ``-Python-2.7.12``    ``intel/2016b``
``0.10.25``                          ``system``     
``0.10.26``                          ``system``     
``0.11.1``                           ``system``     
``0.11.2``                           ``system``     
===========    ==================    ===============


.. _list_software_vsc-mympirun_1274:

*vsc-mympirun*
++++++++++++++

mympirun is a tool to make it easier for users of HPC clusters to run MPI programs with good performance.

*homepage*: https://github.com/hpcugent/vsc-mympirun

===========    ==================================    ===============
version        versionsuffix                         toolchain      
===========    ==================================    ===============
``3.2.1``                                            ``system``     
``3.3.0``                                            ``system``     
``3.4.2``                                            ``system``     
``3.4.2``      ``-Python-2.7.11-vsc-base-2.4.17``    ``intel/2016a``
``3.4.2``      ``-Python-2.7.11-vsc-base-2.5.1``     ``intel/2016a``
``3.4.2``      ``-vsc-base-2.4.2``                   ``system``     
``3.4.3``      ``-Python-2.7.12``                    ``intel/2016b``
``4.0.0``                                            ``system``     
``4.0.0b0``                                          ``system``     
``4.0.1``                                            ``system``     
``4.0.2``                                            ``system``     
``4.1.4``                                            ``system``     
``4.1.5``                                            ``system``     
``4.1.6``                                            ``system``     
``4.1.8``                                            ``system``     
``4.1.9``                                            ``system``     
===========    ==================================    ===============


.. _list_software_vsc-mympirun-scoop_1867:

*vsc-mympirun-scoop*
++++++++++++++++++++

VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent.

*homepage*: http://hpcugent.github.com/vsc-mympirun/

=========    ==================    ===============
version      versionsuffix         toolchain      
=========    ==================    ===============
``3.4.1``    ``-Python-2.7.12``    ``intel/2016b``
=========    ==================    ===============


.. _list_software_vsc-processcontrol_1913:

*vsc-processcontrol*
++++++++++++++++++++

vsc-processcontrol is a module to abstract process control like scheduler settings and affinity  from actual implementations like vsc.affinity and psutil.

*homepage*: http://hpcugent.github.com/vsc-processcontrol/

=======    ===================    ==========
version    versionsuffix          toolchain 
=======    ===================    ==========
``1.0``                           ``system``
``1.0``    ``-vsc-base-2.1.2``    ``system``
=======    ===================    ==========


.. _list_software_VSEARCH_524:

*VSEARCH*
+++++++++

VSEARCH supports de novo and reference based chimera detection,  clustering, full-length and prefix dereplication, rereplication,  reverse complementation, masking, all-vs-all pairwise global alignment,  exact and global alignment searching, shuffling, subsampling and sorting.  It also supports FASTQ file analysis, filtering,  conversion and merging of paired-end reads.

*homepage*: https://github.com/torognes/vsearch

==========    ========================================
version       toolchain                               
==========    ========================================
``2.9.1``     ``foss/2018b``                          
``2.13.4``    ``iccifort/2019.1.144-GCC-8.2.0-2.31.1``
==========    ========================================


.. _list_software_vt_234:

*vt*
++++

A tool set for short variant discovery in genetic sequence data.

*homepage*: http://genome.sph.umich.edu/wiki/Vt

===========    ==============
version        toolchain     
===========    ==============
``0.57721``    ``foss/2018b``
===========    ==============


.. _list_software_VTK_245:

*VTK*
+++++

The Visualization Toolkit (VTK) is an open-source, freely available software system for  3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several  interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization  algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques  such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.

*homepage*: https://www.vtk.org

=========    ==================    ==================================
version      versionsuffix         toolchain                         
=========    ==================    ==================================
``6.3.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``   
``6.3.0``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``   
``7.0.0``    ``-Python-2.7.12``    ``intel/2016b``                   
``7.1.0``    ``-Python-2.7.12``    ``intel/2016b``                   
``7.1.1``    ``-Python-2.7.13``    ``intel/2017a``                   
``8.0.1``    ``-Python-2.7.14``    ``foss/2017b``, ``intel/2017b``   
``8.1.0``    ``-Python-2.7.14``    ``foss/2018a``, ``intel/2018a``   
``8.1.0``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``   
``8.1.1``    ``-Python-2.7.14``    ``intel/2018a``                   
``8.1.1``    ``-Python-2.7.15``    ``foss/2018b``                    
``8.1.1``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``
``8.2.0``    ``-Python-2.7.15``    ``foss/2019a``                    
``8.2.0``    ``-Python-3.7.2``     ``foss/2019a``, ``intel/2019a``   
``8.2.0``    ``-Python-3.7.4``     ``foss/2019b``                    
=========    ==================    ==================================


.. _list_software_VTune_498:

*VTune*
+++++++

Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,  Assembly and Java.

*homepage*: https://software.intel.com/en-us/vtune

=================    ==========
version              toolchain 
=================    ==========
``2013_update6``     ``system``
``2013_update8``     ``system``
``2013_update9``     ``system``
``2013_update10``    ``system``
``2013_update11``    ``system``
``2013_update12``    ``system``
``2016_update3``     ``system``
``2017``             ``system``
``2017_update1``     ``system``
``2017_update2``     ``system``
``2017_update3``     ``system``
``2018_update1``     ``system``
``2018_update2``     ``system``
``2018_update3``     ``system``
``2019_update2``     ``system``
``2019_update3``     ``system``
``2019_update5``     ``system``
=================    ==========


.. _list_software_VV_172:

*VV*
++++

VV is an open-source and cross platform image viewer, designed for fast and  simple visualization of spatio-temporal images: 2D, 2D+t, 3D and 3D+t (or 4D)  images. Only the command-line (clitk) tools are build.

*homepage*: https://www.creatis.insa-lyon.fr/rio/vv

==============    =================    ==============
version           versionsuffix        toolchain     
==============    =================    ==============
``2018.09.19``    ``-Python-3.6.6``    ``foss/2018b``
==============    =================    ==============


.. _list_software_VXL_250:

*VXL*
+++++

A multi-platform collection of C++ software libraries for Computer Vision and Image Understanding.

*homepage*: https://sf.net/projects/vxl

==========    ==============
version       toolchain     
==========    ==============
``1.18.0``    ``foss/2018a``
==========    ==============


.. _list_software_letter_w:

*W*
---


:ref:`list_software_Wannier90_829` - :ref:`list_software_WannierTools_1253` - :ref:`list_software_wcwidth_762` - :ref:`list_software_WebKitGTK+_855` - :ref:`list_software_WebSocket++_989` - :ref:`list_software_WEKA_296` - :ref:`list_software_wget_439` - :ref:`list_software_wheel_533` - :ref:`list_software_WIEN2k_464` - :ref:`list_software_WISExome_753` - :ref:`list_software_wkhtmltopdf_1204` - :ref:`list_software_worker_666` - :ref:`list_software_WPS_250` - :ref:`list_software_WRF_239` - :ref:`list_software_WRF-Fire_674` - :ref:`list_software_wrf-python_1054` - :ref:`list_software_wxPropertyGrid_1498` - :ref:`list_software_wxPython_881` - :ref:`list_software_wxWidgets_966`


.. _list_software_Wannier90_829:

*Wannier90*
+++++++++++

A tool for obtaining maximally-localised Wannier functions

*homepage*: http://www.wannier.org

===========    =============    ================================================
version        versionsuffix    toolchain                                       
===========    =============    ================================================
``1.2``                         ``intel/2016.02-GCC-4.9``                       
``2.0.1``                       ``intel/2016.02-GCC-4.9``, ``intel/2016a``      
``2.0.1.1``    ``-abinit``      ``intel/2018b``                                 
``2.1.0``                       ``foss/2017b``, ``intel/2017a``, ``intel/2017b``
``3.0.0``                       ``intel/2018b``                                 
===========    =============    ================================================


.. _list_software_WannierTools_1253:

*WannierTools*
++++++++++++++

an open-source software package for novel topological materials

*homepage*: http://www.wanniertools.com/

=========    ===============
version      toolchain      
=========    ===============
``2.3.0``    ``intel/2018a``
=========    ===============


.. _list_software_wcwidth_762:

*wcwidth*
+++++++++

wcwidth is a low-level Python library to simplify Terminal emulation.

*homepage*: https://github.com/jquast/wcwidth

=========    ==================    ===============================
version      versionsuffix         toolchain                      
=========    ==================    ===============================
``0.1.7``    ``-Python-2.7.11``    ``foss/2016a``                 
``0.1.7``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``0.1.7``    ``-Python-3.5.1``     ``foss/2016a``                 
``0.1.7``    ``-Python-3.5.2``     ``intel/2016b``                
=========    ==================    ===============================


.. _list_software_WebKitGTK+_855:

*WebKitGTK+*
++++++++++++

WebKitGTK+ is a full-featured port of the WebKit rendering engine, suitable for projects requiring any kind of web integration, from hybrid HTML/CSS applications to full-fledged web browsers. It offers WebKit’s full functionality and is useful in a wide range of systems from desktop computers to embedded systems like phones, tablets, and televisions.

*homepage*: https://webkitgtk.org/

==========    ====================
version       toolchain           
==========    ====================
``2.24.1``    ``GCC/8.2.0-2.31.1``
==========    ====================


.. _list_software_WebSocket++_989:

*WebSocket++*
+++++++++++++

WebSocket++ is an open source (BSD license) header only C++ library  that implements RFC6455 The WebSocket Protocol. 

*homepage*: https://www.zaphoyd.com/websocketpp/

=========    ===============
version      toolchain      
=========    ===============
``0.8.1``    ``gompi/2019a``
=========    ===============


.. _list_software_WEKA_296:

*WEKA*
++++++

Weka is a collection of machine learning algorithms for data mining tasks.   The algorithms can either be applied directly to a dataset or called from your own Java code.   Weka contains tools for data pre-processing, classification, regression, clustering,   association rules, and visualization. It is also well-suited for developing new machine   learning schemes.

*homepage*: http://www.cs.waikato.ac.nz/ml/weka/index.html

==========    ==================    ==========
version       versionsuffix         toolchain 
==========    ==================    ==========
``3.6.12``    ``-Java-1.7.0_80``    ``system``
``3.7.0``     ``-Java-1.7.0_80``    ``system``
==========    ==================    ==========


.. _list_software_wget_439:

*wget*
++++++

GNU Wget is a free software package for retrieving files using HTTP, HTTPS and FTP,  the most widely-used Internet protocols. It is a non-interactive commandline tool,  so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc.

*homepage*: https://www.gnu.org/software/wget/

==========    ==============================
version       toolchain                     
==========    ==============================
``1.17.1``    ``foss/2016a``, ``foss/2016b``
``1.19.4``    ``GCCcore/6.4.0``             
``1.20.1``    ``GCCcore/7.3.0``             
==========    ==============================


.. _list_software_wheel_533:

*wheel*
+++++++

A built-package format for Python.

*homepage*: https://pypi.python.org/pypi/wheel

==========    ==================    ========================================================================
version       versionsuffix         toolchain                                                               
==========    ==================    ========================================================================
``0.29.0``    ``-Python-2.7.11``    ``foss/2016a``                                                          
``0.29.0``    ``-Python-3.5.1``     ``foss/2016a``                                                          
``0.30.0``    ``-Python-2.7.14``    ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.30.0``    ``-Python-3.6.3``     ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017b``, ``intelcuda/2017b``
``0.30.0``    ``-Python-3.6.4``     ``foss/2018a``, ``intel/2018a``                                         
``0.31.0``    ``-Python-2.7.14``    ``fosscuda/2018a``, ``intel/2018a``                                     
``0.31.0``    ``-Python-3.6.4``     ``foss/2018a``, ``fosscuda/2018a``                                      
``0.31.1``    ``-Python-2.7.15``    ``fosscuda/2018b``                                                      
``0.31.1``    ``-Python-3.6.6``     ``foss/2018b``, ``fosscuda/2018b``                                      
==========    ==================    ========================================================================


.. _list_software_WIEN2k_464:

*WIEN2k*
++++++++

The program package WIEN2k allows to perform electronic structure calculations of solids using density functional theory (DFT). It is based on the full-potential (linearized) augmented plane-wave ((L)APW) + local orbitals (lo) method, one among the most accurate schemes for band structure calculations. WIEN2k is an all-electron scheme including relativistic effects and has many features.

*homepage*: http://www.wien2k.at/

========    ================================================
version     toolchain                                       
========    ================================================
``17.1``    ``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``18.1``    ``foss/2018a``, ``gimkl/2017a``, ``intel/2018a``
``19.1``    ``intel/2019a``                                 
========    ================================================


.. _list_software_WISExome_753:

*WISExome*
++++++++++

A within-sample comparison approach to detect copy number variations in whole exome sequencing data

*homepage*: https://github.com/VUmcCGP/wisexome

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20180517``    ``-Python-2.7.14``    ``intel/2018a``
============    ==================    ===============


.. _list_software_wkhtmltopdf_1204:

*wkhtmltopdf*
+++++++++++++

wkhtmltopdf and wkhtmltoimage are open source (LGPLv3) command line tools to render   HTML into PDF and various image formats using the Qt WebKit rendering engine. These run entirely headless   and do not require a display or display service.

*homepage*: http://wkhtmltopdf.org/

==========    =================    ==========
version       versionsuffix        toolchain 
==========    =================    ==========
``0.12.3``    ``-Linux-x86_64``    ``system``
==========    =================    ==========


.. _list_software_worker_666:

*worker*
++++++++

The Worker framework has been developed to help deal with parameter exploration experiments  that would otherwise result in many jobs, forcing the user resort to scripting to retain her sanity;  see also https://vscentrum.be/neutral/documentation/cluster-doc/running-jobs/worker-framework.

*homepage*: https://github.com/gjbex/worker

=========    ================    ===============
version      versionsuffix       toolchain      
=========    ================    ===============
``1.6.4``                        ``intel/2016a``
``1.6.5``                        ``intel/2016a``
``1.6.6``                        ``intel/2016b``
``1.6.7``    ``-intel-2016b``    ``system``     
``1.6.7``    ``-intel-2017a``    ``system``     
``1.6.7``    ``-intel-2017b``    ``system``     
``1.6.8``    ``-intel-2018a``    ``system``     
``1.6.8``    ``-intel-2018b``    ``system``     
=========    ================    ===============


.. _list_software_WPS_250:

*WPS*
+++++

WRF Preprocessing System (WPS) for WRF. The Weather Research and Forecasting (WRF) Model is   a next-generation mesoscale numerical weather prediction system designed to serve both operational   forecasting and atmospheric research needs.

*homepage*: http://www.wrf-model.org

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``3.8.0``    ``-dmpar``       ``intel/2016a``
=========    =============    ===============


.. _list_software_WRF_239:

*WRF*
+++++

The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale  numerical weather prediction system designed to serve both operational forecasting and atmospheric  research needs.

*homepage*: http://www.wrf-model.org

=========    =============    ================================
version      versionsuffix    toolchain                       
=========    =============    ================================
``3.8.0``    ``-dmpar``       ``intel/2016a``, ``intel/2016b``
=========    =============    ================================


.. _list_software_WRF-Fire_674:

*WRF-Fire*
++++++++++

WRF-Fire combines the Weather Research and Forecasting model (WRF) with a fire code implementing a  surface fire behavior model, called SFIRE, based on semi-empirical formulas calculate the rate of spread of the fire  line (the interface between burning and unignited fuel) based on fuel properties, wind velocities from WRF, and  terrain slope. The fire spread is implemented by the level set method.

*homepage*: http://www.openwfm.org/wiki/WRF-SFIRE

============    =============    ===============
version         versionsuffix    toolchain      
============    =============    ===============
``20170221``    ``-dmpar``       ``intel/2016b``
============    =============    ===============


.. _list_software_wrf-python_1054:

*wrf-python*
++++++++++++

A collection of diagnostic and interpolation routines for use with output from  the Weather Research and Forecasting (WRF-ARW) Model.

*homepage*: https://wrf-python.readthedocs.io

=========    =================    ===============
version      versionsuffix        toolchain      
=========    =================    ===============
``1.2.0``    ``-Python-3.6.4``    ``intel/2018a``
``1.3.1``    ``-Python-3.6.6``    ``intel/2018b``
=========    =================    ===============


.. _list_software_wxPropertyGrid_1498:

*wxPropertyGrid*
++++++++++++++++

wxPropertyGrid is a property sheet control for wxWidgets. In other words, it is  a specialized two-column grid for editing properties such as strings, numbers, flagsets, string arrays, and colours.

*homepage*: http://wxpropgrid.sourceforge.net/

==========    =============
version       toolchain    
==========    =============
``1.4.15``    ``GCC/4.9.2``
==========    =============


.. _list_software_wxPython_881:

*wxPython*
++++++++++

wxPython is a GUI toolkit for the Python programming language.  It allows Python programmers to create programs with a robust,  highly functional graphical user interface, simply and easily.  It is implemented as a Python extension module (native code)  that wraps the popular wxWidgets cross platform GUI library,  which is written in C++.

*homepage*: http://www.wxpython.org

===========    ==================    ===============================
version        versionsuffix         toolchain                      
===========    ==================    ===============================
``3.0.2.0``    ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``3.0.2.0``    ``-Python-2.7.12``    ``intel/2016b``                
``3.0.2.0``    ``-Python-2.7.13``    ``intel/2017a``                
``3.0.2.0``    ``-Python-2.7.14``    ``intel/2017b``                
``3.0.2.0``    ``-Python-2.7.15``    ``foss/2018b``                 
``4.0.4``      ``-Python-2.7.15``    ``GCC/8.2.0-2.31.1``           
``4.0.4``      ``-Python-3.7.2``     ``GCC/8.2.0-2.31.1``           
===========    ==================    ===============================


.. _list_software_wxWidgets_966:

*wxWidgets*
+++++++++++

wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a single code base. It has popular language bindings for Python, Perl, Ruby and many other languages, and unlike other cross-platform toolkits, wxWidgets gives applications a truly native look and feel because it uses the platform's native API rather than emulating the GUI.

*homepage*: https://www.wxwidgets.org

=========    ====================
version      toolchain           
=========    ====================
``3.0.3``    ``foss/2018a``      
``3.0.4``    ``GCC/8.2.0-2.31.1``
=========    ====================


.. _list_software_letter_x:

*X*
---


:ref:`list_software_X11_186` - :ref:`list_software_x264_276` - :ref:`list_software_x265_277` - :ref:`list_software_xarray_663` - :ref:`list_software_xbitmaps_872` - :ref:`list_software_xcb-proto_926` - :ref:`list_software_xcb-util_808` - :ref:`list_software_xcb-util-image_1368` - :ref:`list_software_xcb-util-keysyms_1642` - :ref:`list_software_xcb-util-renderutil_1939` - :ref:`list_software_xcb-util-wm_1081` - :ref:`list_software_xCell_504` - :ref:`list_software_XCfun_484` - :ref:`list_software_XCrySDen_752` - :ref:`list_software_Xerces-C++_816` - :ref:`list_software_xextproto_1021` - :ref:`list_software_xf86vidmodeproto_1640` - :ref:`list_software_XGBoost_678` - :ref:`list_software_xineramaproto_1417` - :ref:`list_software_XKeyboardConfig_1503` - :ref:`list_software_XMDS2_366` - :ref:`list_software_Xmipp_526` - :ref:`list_software_XML-LibXML_806` - :ref:`list_software_XML-Parser_907` - :ref:`list_software_xmlf90_544` - :ref:`list_software_XMLSec_524` - :ref:`list_software_XMLStarlet_976` - :ref:`list_software_xonsh_560` - :ref:`list_software_xorg-macros_1138` - :ref:`list_software_xprop_569` - :ref:`list_software_xproto_684` - :ref:`list_software_XSD_239` - :ref:`list_software_XTandem_689` - :ref:`list_software_xtrans_672` - :ref:`list_software_XZ_178`


.. _list_software_X11_186:

*X11*
+++++

The X Window System (X11) is a windowing system for bitmap displays

*homepage*: https://www.x.org

============    ==================================================
version         toolchain                                         
============    ==================================================
``20160819``    ``GCCcore/5.4.0``, ``foss/2016b``, ``intel/2016b``
``20170129``    ``GCCcore/6.3.0``, ``gimkl/2017a``                
``20170314``    ``GCCcore/6.3.0``                                 
``20171023``    ``GCCcore/6.4.0``                                 
``20180131``    ``GCCcore/6.4.0``                                 
``20180604``    ``GCCcore/7.3.0``                                 
``20190311``    ``GCCcore/8.2.0``                                 
``20190717``    ``GCCcore/8.3.0``                                 
============    ==================================================


.. _list_software_x264_276:

*x264*
++++++

x264 is a free software library and application for encoding video streams  into the H.264/MPEG-4 AVC compression format, and is released under the  terms of the GNU GPL. 

*homepage*: https://www.videolan.org/developers/x264.html

============    =================================
version         toolchain                        
============    =================================
``20160114``    ``gimkl/2.11.5``, ``intel/2016a``
``20160430``    ``foss/2016a``, ``intel/2016a``  
``20160614``    ``foss/2016b``, ``intel/2016b``  
``20170406``    ``gimkl/2017a``                  
``20170721``    ``GCCcore/6.4.0``                
``20170913``    ``intel/2017a``                  
``20171217``    ``foss/2017b``, ``intel/2017b``  
``20180128``    ``GCCcore/6.4.0``                
``20180325``    ``GCCcore/6.4.0``                
``20181203``    ``GCCcore/7.3.0``                
``20190413``    ``GCCcore/8.2.0``                
``20190925``    ``GCCcore/8.3.0``                
============    =================================


.. _list_software_x265_277:

*x265*
++++++

x265 is a free software library and application for encoding video streams  into the H.265 AVC compression format, and is released under the terms of  the GNU GPL. 

*homepage*: https://x265.org/

=======    ==================================
version    toolchain                         
=======    ==================================
``2.4``    ``foss/2016b``                    
``2.5``    ``intel/2017a``                   
``2.6``    ``GCCcore/6.4.0``, ``intel/2017b``
``2.7``    ``GCCcore/6.4.0``                 
``2.9``    ``GCCcore/7.3.0``                 
``3.0``    ``GCCcore/8.2.0``                 
``3.2``    ``GCCcore/8.3.0``                 
=======    ==================================


.. _list_software_xarray_663:

*xarray*
++++++++

xarray (formerly xray) is an open source project and Python package that aims to bring  the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the  core pandas data structures.

*homepage*: https://github.com/pydata/xarray

==========    ==================    ===============
version       versionsuffix         toolchain      
==========    ==================    ===============
``0.9.5``     ``-Python-2.7.13``    ``intel/2017a``
``0.9.5``     ``-Python-3.6.1``     ``intel/2017a``
``0.9.6``     ``-Python-2.7.13``    ``intel/2017a``
``0.10.0``    ``-Python-2.7.14``    ``intel/2017b``
``0.10.0``    ``-Python-3.6.3``     ``intel/2017b``
``0.10.3``    ``-Python-3.6.4``     ``intel/2018a``
``0.10.4``    ``-Python-3.6.4``     ``intel/2018a``
``0.10.8``    ``-Python-3.6.4``     ``intel/2018a``
``0.12.1``    ``-Python-3.6.6``     ``intel/2018b``
``0.13.0``    ``-Python-3.7.2``     ``intel/2019a``
==========    ==================    ===============


.. _list_software_xbitmaps_872:

*xbitmaps*
++++++++++

provides bitmaps for x

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===========================================
version      toolchain                                  
=========    ===========================================
``1.1.1``    ``foss/2016a``, ``intel/2016a``, ``system``
``1.1.2``    ``system``                                 
=========    ===========================================


.. _list_software_xcb-proto_926:

*xcb-proto*
+++++++++++

The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint, latency hiding, direct access to the protocol, improved threading support, and extensibility.

*homepage*: http://xcb.freedesktop.org/

========    ==========
version     toolchain 
========    ==========
``1.11``    ``system``
========    ==========


.. _list_software_xcb-util_808:

*xcb-util*
++++++++++

The xcb-util package provides additional extensions to the XCB library,  many that were previously found in Xlib, but are not part of core X protocol

*homepage*: http://xcb.freedesktop.org/

=========    ===============================
version      toolchain                      
=========    ===============================
``0.4.0``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_xcb-util-image_1368:

*xcb-util-image*
++++++++++++++++

The xcb-util-image package provides additional extensions to the XCB library.

*homepage*: http://xcb.freedesktop.org/

=========    ===============================
version      toolchain                      
=========    ===============================
``0.4.0``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_xcb-util-keysyms_1642:

*xcb-util-keysyms*
++++++++++++++++++

The xcb-util-keysyms package contains a library for  handling standard X key constants and conversion to/from keycodes.

*homepage*: http://xcb.freedesktop.org/

=========    ===============================
version      toolchain                      
=========    ===============================
``0.4.0``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_xcb-util-renderutil_1939:

*xcb-util-renderutil*
+++++++++++++++++++++

The xcb-util-renderutil package provides additional extensions to the XCB library.

*homepage*: http://xcb.freedesktop.org/

=========    ===============================
version      toolchain                      
=========    ===============================
``0.3.9``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_xcb-util-wm_1081:

*xcb-util-wm*
+++++++++++++

The xcb-util-wm package contains libraries which provide client and window-manager helpers  for EWMH and ICCCM.

*homepage*: http://xcb.freedesktop.org/

=========    ===============================
version      toolchain                      
=========    ===============================
``0.4.1``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_xCell_504:

*xCell*
+++++++

xCell is a gene signatures-based method learned from thousands of pure cell types from various sources.

*homepage*: https://github.com/dviraran/xCell

========    =============    ==============
version     versionsuffix    toolchain     
========    =============    ==============
``1.12``    ``-R-3.5.1``     ``foss/2018b``
========    =============    ==============


.. _list_software_XCfun_484:

*XCfun*
+++++++

XCFun is a library of DFT exchange-correlation (XC) functionals.  It is based on automatic differentiation and can therefore generate arbitrary order  derivatives of these functionals. 

*homepage*: http://dftlibs.org/xcfun/

============    ==================    ===============
version         versionsuffix         toolchain      
============    ==================    ===============
``20180122``    ``-Python-2.7.14``    ``intel/2017b``
``20190127``    ``-Python-3.7.2``     ``foss/2019a`` 
============    ==================    ===============


.. _list_software_XCrySDen_752:

*XCrySDen*
++++++++++

XCrySDen is a crystalline and molecular structure visualisation program aiming  at display of isosurfaces and contours, which can be superimposed on crystalline structures and  interactively rotated and manipulated.

*homepage*: http://www.xcrysden.org/

==========    ===============
version       toolchain      
==========    ===============
``1.5.60``    ``intel/2019a``
==========    ===============


.. _list_software_Xerces-C++_816:

*Xerces-C++*
++++++++++++

Xerces-C++ is a validating XML parser written in a portable subset of C++. Xerces-C++ makes it easy to give your application the ability to read and write XML data. A shared library is provided for parsing, generating, manipulating, and validating XML documents using the DOM, SAX, and SAX2 APIs.

*homepage*: https://xerces.apache.org/xerces-c/

=========    =================
version      toolchain        
=========    =================
``3.1.4``    ``GCCcore/6.4.0``
``3.2.0``    ``GCCcore/7.3.0``
``3.2.2``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_xextproto_1021:

*xextproto*
+++++++++++

XExtProto protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ==================================================================
version      toolchain                                                         
=========    ==================================================================
``7.3.0``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2017b``
=========    ==================================================================


.. _list_software_xf86vidmodeproto_1640:

*xf86vidmodeproto*
++++++++++++++++++

X11 XFree86 video mode extension protocol headers.

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``2.3.1``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_XGBoost_678:

*XGBoost*
+++++++++

XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,  flexible and portable.

*homepage*: https://github.com/dmlc/xgboost

============    ===================    ===============
version         versionsuffix          toolchain      
============    ===================    ===============
``0.6a2``       ``-Python-2.7.12``     ``intel/2016b``
``0.6a2``       ``-Python-3.5.2``      ``intel/2016b``
``0.6a2``       ``-Python-3.6.1``      ``intel/2017a``
``0.72.1``      ``-Python-2.7.14``     ``intel/2017b``
``20171120``    ``-Java-1.8.0_152``    ``intel/2017b``
============    ===================    ===============


.. _list_software_xineramaproto_1417:

*xineramaproto*
+++++++++++++++

X protocol and ancillary headers for xinerama

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    ===============================
version      toolchain                      
=========    ===============================
``1.2.1``    ``foss/2016a``, ``intel/2016a``
=========    ===============================


.. _list_software_XKeyboardConfig_1503:

*XKeyboardConfig*
+++++++++++++++++

The non-arch keyboard configuration database for X Window.  The goal is to provide the consistent, well-structured,  frequently released open source of X keyboard configuration data  for X Window System implementations (free, open source and commercial).  The project is targeted to XKB-based systems.

*homepage*: http://www.freedesktop.org/wiki/Software/XKeyboardConfig/

========    ===============================
version     toolchain                      
========    ===============================
``2.17``    ``foss/2016a``, ``intel/2016a``
========    ===============================


.. _list_software_XMDS2_366:

*XMDS2*
+++++++

The purpose of XMDS2 is to simplify the process of creating simulations that  solve systems of initial-value first-order partial and ordinary differential equations.

*homepage*: http://www.xmds.org/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.2.3``    ``-Python-2.7.15``    ``foss/2018b``
=========    ==================    ==============


.. _list_software_Xmipp_526:

*Xmipp*
+++++++

Scipion is an image processing framework to obtain 3D models of macromolecular complexes using Electron Microscopy (3DEM). It integrates several software packages and presents an unified interface for both biologists and developers. Scipion allows to execute workflows combining different software tools, while taking care of formats and conversions. Additionally, all steps are tracked and can be reproduced later on.  

*homepage*: https://github.com/I2PC/scipion/wiki

==================    ==================    ==================================
version               versionsuffix         toolchain                         
==================    ==================    ==================================
``3.19.04-Apollo``    ``-Python-2.7.15``    ``foss/2019a``, ``fosscuda/2019a``
==================    ==================    ==================================


.. _list_software_XML-LibXML_806:

*XML-LibXML*
++++++++++++

Perl binding for libxml2

*homepage*: https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod

==========    ================    ===============================
version       versionsuffix       toolchain                      
==========    ================    ===============================
``2.0132``    ``-Perl-5.24.0``    ``foss/2016b``, ``intel/2016b``
``2.0132``    ``-Perl-5.24.1``    ``intel/2017a``                
``2.0132``    ``-Perl-5.26.0``    ``foss/2017b``, ``intel/2017b``
``2.0132``    ``-Perl-5.26.1``    ``GCCcore/6.4.0``              
``2.0132``    ``-Perl-5.28.0``    ``GCCcore/7.3.0``              
``2.0200``    ``-Perl-5.28.1``    ``GCCcore/8.2.0``              
``2.0201``                        ``GCCcore/8.3.0``              
==========    ================    ===============================


.. _list_software_XML-Parser_907:

*XML-Parser*
++++++++++++

This is a Perl extension interface to James Clark's XML parser, expat.

*homepage*: http://search.cpan.org/~toddr/XML-Parser-2.41/

===========    ================    ======================================================================================
version        versionsuffix       toolchain                                                                             
===========    ================    ======================================================================================
``2.41``       ``-Perl-5.20.3``    ``intel/2016a``                                                                       
``2.44``       ``-Perl-5.22.1``    ``foss/2016a``, ``intel/2016a``                                                       
``2.44_01``    ``-Perl-5.24.0``    ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016b``, ``gimkl/2017a``, ``intel/2016b``
``2.44_01``    ``-Perl-5.24.1``    ``GCCcore/6.3.0``, ``intel/2017a``                                                    
``2.44_01``    ``-Perl-5.26.0``    ``GCCcore/6.4.0``                                                                     
``2.44_01``    ``-Perl-5.26.1``    ``GCCcore/6.4.0``                                                                     
``2.44_01``    ``-Perl-5.28.0``    ``GCCcore/7.3.0``                                                                     
===========    ================    ======================================================================================


.. _list_software_xmlf90_544:

*xmlf90*
++++++++

xmlf90 is a basic XML parsing library written in Fortran.

*homepage*: https://launchpad.net/xmlf90

=========    ==============================
version      toolchain                     
=========    ==============================
``1.5.3``    ``foss/2016b``, ``foss/2017a``
=========    ==============================


.. _list_software_XMLSec_524:

*XMLSec*
++++++++

XML Security Library is a C library based on LibXML2,  supporting major XML security standards.

*homepage*: https://www.aleksey.com/xmlsec/index.html

==========    =================
version       toolchain        
==========    =================
``1.2.26``    ``GCCcore/6.4.0``
==========    =================


.. _list_software_XMLStarlet_976:

*XMLStarlet*
++++++++++++

Command line XML tool

*homepage*: http://xmlstar.sourceforge.net

=========    =================================
version      toolchain                        
=========    =================================
``1.6.1``    ``GCCcore/6.4.0``, ``foss/2016a``
=========    =================================


.. _list_software_xonsh_560:

*xonsh*
+++++++

Xonsh is a Python-ish, BASHwards-looking shell language and command prompt.

*homepage*: http://xon.sh/

=========    ===============
version      toolchain      
=========    ===============
``0.3.2``    ``intel/2016a``
=========    ===============


.. _list_software_xorg-macros_1138:

*xorg-macros*
+++++++++++++

X.org macros utilities.

*homepage*: http://cgit.freedesktop.org/xorg/util/macros

==========    ==================================================================================
version       toolchain                                                                         
==========    ==================================================================================
``1.19.0``    ``foss/2016a``, ``foss/2016b``, ``gimkl/2.11.5``, ``intel/2016a``, ``intel/2016b``
``1.19.1``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``                                              
``1.19.2``    ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``        
==========    ==================================================================================


.. _list_software_xprop_569:

*xprop*
+++++++

The xprop utility is for displaying window and font properties in an X server.  One window or font is selected using the command line arguments or possibly  in the case of a window, by clicking on the desired window. A list of  properties is then given, possibly with formatting information.

*homepage*: http://www.x.org/wiki/

=========    =================    =======================================================================================================================
version      versionsuffix        toolchain                                                                                                              
=========    =================    =======================================================================================================================
``1.2.2``                         ``GCCcore/5.4.0``, ``GCCcore/6.4.0``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
``1.2.2``    ``-X11-20180131``    ``GCCcore/6.4.0``                                                                                                      
``1.2.3``                         ``GCCcore/7.3.0``                                                                                                      
``1.2.4``                         ``GCCcore/8.2.0``                                                                                                      
=========    =================    =======================================================================================================================


.. _list_software_xproto_684:

*xproto*
++++++++

X protocol and ancillary headers

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

==========    =======================================================
version       toolchain                                              
==========    =======================================================
``7.0.28``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``      
``7.0.29``    ``intel/2016a``                                        
``7.0.31``    ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.3.0``
==========    =======================================================


.. _list_software_XSD_239:

*XSD*
+++++

CodeSynthesis XSD is an open-source, cross-platform W3C XML Schema to C++ data binding compiler.

*homepage*: https://www.codesynthesis.com/products/xsd/

=========    =================
version      toolchain        
=========    =================
``4.0.0``    ``GCCcore/8.2.0``
=========    =================


.. _list_software_XTandem_689:

*XTandem*
+++++++++

X!Tandem open source is software that can match tandem mass  spectra with peptide sequences, in a process that has come to be known as  protein identification.

*homepage*: https://www.thegpm.org/tandem

==============    ==========================================================
version           toolchain                                                 
==============    ==========================================================
``17.02.01.4``    ``GCC/6.4.0-2.28``, ``iccifort/2017.4.196-GCC-6.4.0-2.28``
==============    ==========================================================


.. _list_software_xtrans_672:

*xtrans*
++++++++

xtrans includes a number of routines to make X implementations transport-independent;  at time of writing, it includes support for UNIX sockets, IPv4, IPv6, and DECnet. 

*homepage*: http://www.freedesktop.org/wiki/Software/xlibs

=========    =================================================
version      toolchain                                        
=========    =================================================
``1.3.5``    ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016a``
=========    =================================================


.. _list_software_XZ_178:

*XZ*
++++

xz: XZ utilities

*homepage*: https://tukaani.org/xz/

=========    ===================    ===========================================================================================================================================================
version      versionsuffix          toolchain                                                                                                                                                  
=========    ===================    ===========================================================================================================================================================
``5.0.5``                           ``GCC/4.8.2``                                                                                                                                              
``5.2.0``                           ``GCC/4.9.2``                                                                                                                                              
``5.2.2``                           ``GCC/4.9.2``, ``GCC/5.4.0-2.26``, ``GCCcore/4.9.3``, ``GCCcore/5.4.0``, ``foss/2016.04``, ``foss/2016a``, ``foss/2016b``, ``intel/2016a``, ``intel/2016b``
``5.2.2``    ``-gettext-0.19.7``    ``foss/2016a``, ``intel/2016a``                                                                                                                            
``5.2.3``                           ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``gimkl/2017a``                                                                                                      
``5.2.4``                           ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                                 
=========    ===================    ===========================================================================================================================================================


.. _list_software_letter_y:

*Y*
---


:ref:`list_software_Yade_387` - :ref:`list_software_yaff_422` - :ref:`list_software_Yambo_504` - :ref:`list_software_YAPS_317` - :ref:`list_software_Yasm_410` - :ref:`list_software_YAXT_326`


.. _list_software_Yade_387:

*Yade*
++++++

Yade is an extensible open-source framework for discrete numerical models,   focused on Discrete Element Method. The computation parts are written in c++ using flexible object model,   allowing independent implementation of new alogrithms and interfaces.   Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. 

*homepage*: https://yade-dem.org/doc/

============    ==================    ===============================
version         versionsuffix         toolchain                      
============    ==================    ===============================
``1.20.0``      ``-Python-2.7.11``    ``foss/2016a``, ``intel/2016a``
``2016.06a``    ``-Python-2.7.12``    ``foss/2016b``, ``intel/2016b``
``2017.01a``    ``-Python-2.7.12``    ``intel/2016b``                
``2018.02b``    ``-Python-2.7.14``    ``intel/2018a``                
============    ==================    ===============================


.. _list_software_yaff_422:

*yaff*
++++++

Yaff stands for 'Yet another force field'. It is a pythonic force-field code.

*homepage*: https://molmod.github.io/yaff/

====================    ==============================    ================================================
version                 versionsuffix                     toolchain                                       
====================    ==============================    ================================================
``1.0.develop.2.15``    ``-Python-2.7.12-HDF5-1.8.18``    ``intel/2016b``                                 
``1.1.2``               ``-Python-2.7.13``                ``intel/2017a``                                 
``1.1.3``               ``-Python-2.7.13``                ``intel/2017a``                                 
``1.4.2``               ``-Python-2.7.14``                ``foss/2018a``, ``intel/2017b``, ``intel/2018a``
``1.4.2``               ``-Python-3.6.3``                 ``intel/2017b``                                 
``1.4.5``               ``-Python-2.7.15``                ``intel/2018b``                                 
``1.5.0``               ``-Python-2.7.15``                ``intel/2018b``                                 
``1.5.0``               ``-Python-3.7.2``                 ``intel/2019a``                                 
``1.6.0``               ``-Python-3.7.2``                 ``intel/2019a``                                 
====================    ==============================    ================================================


.. _list_software_Yambo_504:

*Yambo*
+++++++

Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.  Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf.

*homepage*: http://www.yambo-code.org

=========    =========================
version      toolchain                
=========    =========================
``3.4.2``    ``intel/2016.02-GCC-4.9``
=========    =========================


.. _list_software_YAPS_317:

*YAPS*
++++++

YAPS - Yet Another Positioning Solver

*homepage*: https://github.com/baktoft/yaps

=========    =============    ===============
version      versionsuffix    toolchain      
=========    =============    ===============
``1.1.0``    ``-R-3.5.1``     ``foss/2018b`` 
``1.1.0``    ``-R-3.6.0``     ``intel/2019a``
=========    =============    ===============


.. _list_software_Yasm_410:

*Yasm*
++++++

Yasm: Complete rewrite of the NASM assembler with BSD license

*homepage*: http://www.tortall.net/projects/yasm/

=========    ================================================================================================================================================================================================================
version      toolchain                                                                                                                                                                                                       
=========    ================================================================================================================================================================================================================
``1.3.0``    ``GCCcore/6.4.0``, ``GCCcore/7.3.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``foss/2016a``, ``foss/2016b``, ``foss/2017a``, ``gimkl/2.11.5``, ``gimkl/2017a``, ``intel/2016a``, ``intel/2016b``, ``intel/2017a``
=========    ================================================================================================================================================================================================================


.. _list_software_YAXT_326:

*YAXT*
++++++

Yet Another eXchange Tool

*homepage*: https://www.dkrz.de/redmine/projects/yaxt

=========    =================================================
version      toolchain                                        
=========    =================================================
``0.5.1``    ``intel/2016b``, ``intel/2017a``, ``intel/2017b``
``0.6.0``    ``intel/2018a``, ``intel/2018b``, ``iomkl/2018b``
=========    =================================================


.. _list_software_letter_z:

*Z*
---


:ref:`list_software_zarr_447` - :ref:`list_software_ZeroMQ_574` - :ref:`list_software_zingeR_623` - :ref:`list_software_zlib_433` - :ref:`list_software_zlibbioc_846` - :ref:`list_software_ZPAQ_316` - :ref:`list_software_zsh_341` - :ref:`list_software_zstd_453`


.. _list_software_zarr_447:

*zarr*
++++++

Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays,   designed for use in parallel computing.

*homepage*: http://zarr.readthedocs.io/en/stable/

=========    ==================    ==============
version      versionsuffix         toolchain     
=========    ==================    ==============
``2.1.4``    ``-Python-2.7.13``    ``foss/2017a``
=========    ==================    ==============


.. _list_software_ZeroMQ_574:

*ZeroMQ*
++++++++

ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.  It gives you sockets that carry atomic messages across various transports like in-process,  inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,  pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered  products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous  message-processing tasks. It has a score of language APIs and runs on most operating systems.

*homepage*: https://www.zeromq.org/

=========    =========================================================================================================
version      toolchain                                                                                                
=========    =========================================================================================================
``4.1.4``    ``foss/2016a``, ``intel/2016a``                                                                          
``4.1.5``    ``intel/2016b``                                                                                          
``4.2.0``    ``foss/2016b``, ``intel/2016b``                                                                          
``4.2.2``    ``foss/2017a``, ``foss/2017b``, ``fosscuda/2017b``, ``intel/2017a``, ``intel/2017b``, ``intelcuda/2017b``
``4.2.5``    ``foss/2018a``, ``foss/2018b``, ``fosscuda/2018b``, ``intel/2018a``, ``intel/2018b``                     
``4.3.2``    ``GCCcore/8.2.0``, ``GCCcore/8.3.0``                                                                     
=========    =========================================================================================================


.. _list_software_zingeR_623:

*zingeR*
++++++++

Zero-Inflated Negative binomial Gene Expression in R

*homepage*: https://github.com/statOmics/zingeR

============    =============    ==============
version         versionsuffix    toolchain     
============    =============    ==============
``20180131``    ``-R-3.5.1``     ``foss/2018b``
============    =============    ==============


.. _list_software_zlib_433:

*zlib*
++++++

zlib is designed to be a free, general-purpose, legally unencumbered -- that is,  not covered by any patents -- lossless data-compression library for use on virtually any  computer hardware and operating system.

*homepage*: http://www.zlib.net/

==========    =====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
version       toolchain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
==========    =====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
``1.2.7``     ``GCC/4.8.1``, ``GCC/4.8.2``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         
``1.2.8``     ``GCC/4.8.2``, ``GCC/4.8.3``, ``GCC/4.8.4``, ``GCC/4.9.2``, ``GCC/4.9.2-binutils-2.25``, ``GCC/4.9.3``, ``GCC/4.9.3-2.25``, ``GCC/4.9.3-binutils-2.25``, ``GCC/5.1.0-binutils-2.25``, ``GCCcore/4.9.2``, ``GCCcore/4.9.3``, ``GCCcore/4.9.4``, ``GCCcore/5.3.0``, ``GCCcore/5.4.0``, ``GCCcore/6.1.0``, ``GCCcore/6.2.0``, ``GCCcore/6.3.0``, ``GNU/4.9.3-2.25``, ``foss/2016.04``, ``foss/2016a``, ``gimkl/2.11.5``, ``intel/2016.02-GCC-4.9``, ``intel/2016a``, ``intel/2016b``, ``intel/2017.01``, ``iomkl/2016.07``, ``iomkl/2016.09-GCC-4.9.3-2.25``, ``system``
``1.2.10``    ``system``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
``1.2.11``    ``GCCcore/5.4.0``, ``GCCcore/5.5.0``, ``GCCcore/6.3.0``, ``GCCcore/6.4.0``, ``GCCcore/7.1.0``, ``GCCcore/7.2.0``, ``GCCcore/7.3.0``, ``GCCcore/7.4.0``, ``GCCcore/8.1.0``, ``GCCcore/8.2.0``, ``GCCcore/8.3.0``, ``GCCcore/9.1.0``, ``GCCcore/9.2.0``, ``GCCcore/system``, ``gimkl/2017a``, ``system``                                                                                                                                                                                                                                                               
==========    =====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================


.. _list_software_zlibbioc_846:

*zlibbioc*
++++++++++

This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have  these available via other means.

*homepage*: https://bioconductor.org/packages/release/bioc/html/zlibbioc.html

==========    =============    ===============
version       versionsuffix    toolchain      
==========    =============    ===============
``1.18.0``    ``-R-3.2.3``     ``intel/2016a``
``1.20.0``    ``-R-3.3.1``     ``intel/2016b``
==========    =============    ===============


.. _list_software_ZPAQ_316:

*ZPAQ*
++++++

zpaq is a free and open source (GPL v3) incremental,  journaling command-line archiver for Windows, Linux and Mac OS/X

*homepage*: http://mattmahoney.net/dc/zpaq.html

========    =============
version     toolchain    
========    =============
``7.00``    ``GCC/4.8.2``
========    =============


.. _list_software_zsh_341:

*zsh*
+++++

Zsh is a shell designed for interactive use, although it is also a powerful scripting language.

*homepage*: http://www.zsh.org/

=========    ==================
version      toolchain         
=========    ==================
``5.1.1``    ``GNU/4.9.3-2.25``
``5.2``      ``foss/2016b``    
=========    ==================


.. _list_software_zstd_453:

*zstd*
++++++

Zstandard is a real-time compression algorithm, providing high compression ratios.   It offers a very wide range of compression/speed trade-off, while being backed by a very fast decoder.  It also offers a special mode for small data, called dictionary compression, and can create dictionaries  from any sample set.

*homepage*: https://facebook.github.io/zstd

=========    =================================
version      toolchain                        
=========    =================================
``1.3.4``    ``foss/2016b``                   
``1.4.0``    ``GCCcore/8.2.0``, ``foss/2018b``
=========    =================================

